<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <title>Tool Test Results (powered by Planemo)</title>

    <!-- Bootstrap -->
    <style>/*!
 * Bootstrap v3.3.1 (http://getbootstrap.com)
 * Copyright 2011-2014 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 *//*! normalize.css v3.0.2 | MIT License | git.io/normalize */html{font-family:sans-serif;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%}body{margin:0}article,aside,details,figcaption,figure,footer,header,hgroup,main,menu,nav,section,summary{display:block}audio,canvas,progress,video{display:inline-block;vertical-align:baseline}audio:not([controls]){display:none;height:0}[hidden],template{display:none}a{background-color:transparent}a:active,a:hover{outline:0}abbr[title]{border-bottom:1px dotted}b,strong{font-weight:700}dfn{font-style:italic}h1{margin:.67em 0;font-size:2em}mark{color:#000;background:#ff0}small{font-size:80%}sub,sup{position:relative;font-size:75%;line-height:0;vertical-align:baseline}sup{top:-.5em}sub{bottom:-.25em}img{border:0}svg:not(:root){overflow:hidden}figure{margin:1em 40px}hr{height:0;-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box}pre{overflow:auto}code,kbd,pre,samp{font-family:monospace,monospace;font-size:1em}button,input,optgroup,select,textarea{margin:0;font:inherit;color:inherit}button{overflow:visible}button,select{text-transform:none}button,html input[type=button],input[type=reset],input[type=submit]{-webkit-appearance:button;cursor:pointer}button[disabled],html input[disabled]{cursor:default}button::-moz-focus-inner,input::-moz-focus-inner{padding:0;border:0}input{line-height:normal}input[type=checkbox],input[type=radio]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;padding:0}input[type=number]::-webkit-inner-spin-button,input[type=number]::-webkit-outer-spin-button{height:auto}input[type=search]{-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;-webkit-appearance:textfield}input[type=search]::-webkit-search-cancel-button,input[type=search]::-webkit-search-decoration{-webkit-appearance:none}fieldset{padding:.35em .625em .75em;margin:0 2px;border:1px solid silver}legend{padding:0;border:0}textarea{overflow:auto}optgroup{font-weight:700}table{border-spacing:0;border-collapse:collapse}td,th{padding:0}/*! Source: https://github.com/h5bp/html5-boilerplate/blob/master/src/css/main.css */@media print{*,:before,:after{color:#000!important;text-shadow:none!important;background:transparent!important;-webkit-box-shadow:none!important;box-shadow:none!important}a,a:visited{text-decoration:underline}a[href]:after{content:" (" attr(href) ")"}abbr[title]:after{content:" (" attr(title) ")"}a[href^="#"]:after,a[href^="javascript:"]:after{content:""}pre,blockquote{border:1px solid #999;page-break-inside:avoid}thead{display:table-header-group}tr,img{page-break-inside:avoid}img{max-width:100%!important}p,h2,h3{orphans:3;widows:3}h2,h3{page-break-after:avoid}select{background:#fff!important}.navbar{display:none}.btn>.caret,.dropup>.btn>.caret{border-top-color:#000!important}.label{border:1px solid #000}.table{border-collapse:collapse!important}.table td,.table th{background-color:#fff!important}.table-bordered th,.table-bordered td{border:1px solid #ddd!important}}@font-face{font-family:'Glyphicons Halflings';src:url(../fonts/glyphicons-halflings-regular.eot);src:url(../fonts/glyphicons-halflings-regular.eot?#iefix) format('embedded-opentype'),url(../fonts/glyphicons-halflings-regular.woff) format('woff'),url(../fonts/glyphicons-halflings-regular.ttf) format('truetype'),url(../fonts/glyphicons-halflings-regular.svg#glyphicons_halflingsregular) format('svg')}.glyphicon{position:relative;top:1px;display:inline-block;font-family:'Glyphicons Halflings';font-style:normal;font-weight:400;line-height:1;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale}.glyphicon-asterisk:before{content:"\2a"}.glyphicon-plus:before{content:"\2b"}.glyphicon-euro:before,.glyphicon-eur:before{content:"\20ac"}.glyphicon-minus:before{content:"\2212"}.glyphicon-cloud:before{content:"\2601"}.glyphicon-envelope:before{content:"\2709"}.glyphicon-pencil:before{content:"\270f"}.glyphicon-glass:before{content:"\e001"}.glyphicon-music:before{content:"\e002"}.glyphicon-search:before{content:"\e003"}.glyphicon-heart:before{content:"\e005"}.glyphicon-star:before{content:"\e006"}.glyphicon-star-empty:before{content:"\e007"}.glyphicon-user:before{content:"\e008"}.glyphicon-film:before{content:"\e009"}.glyphicon-th-large:before{content:"\e010"}.glyphicon-th:before{content:"\e011"}.glyphicon-th-list:before{content:"\e012"}.glyphicon-ok:before{content:"\e013"}.glyphicon-remove:before{content:"\e014"}.glyphicon-zoom-in:before{content:"\e015"}.glyphicon-zoom-out:before{content:"\e016"}.glyphicon-off:before{content:"\e017"}.glyphicon-signal:before{content:"\e018"}.glyphicon-cog:before{content:"\e019"}.glyphicon-trash:before{content:"\e020"}.glyphicon-home:before{content:"\e021"}.glyphicon-file:before{content:"\e022"}.glyphicon-time:before{content:"\e023"}.glyphicon-road:before{content:"\e024"}.glyphicon-download-alt:before{content:"\e025"}.glyphicon-download:before{content:"\e026"}.glyphicon-upload:before{content:"\e027"}.glyphicon-inbox:before{content:"\e028"}.glyphicon-play-circle:before{content:"\e029"}.glyphicon-repeat:before{content:"\e030"}.glyphicon-refresh:before{content:"\e031"}.glyphicon-list-alt:before{content:"\e032"}.glyphicon-lock:before{content:"\e033"}.glyphicon-flag:before{content:"\e034"}.glyphicon-headphones:before{content:"\e035"}.glyphicon-volume-off:before{content:"\e036"}.glyphicon-volume-down:before{content:"\e037"}.glyphicon-volume-up:before{content:"\e038"}.glyphicon-qrcode:before{content:"\e039"}.glyphicon-barcode:before{content:"\e040"}.glyphicon-tag:before{content:"\e041"}.glyphicon-tags:before{content:"\e042"}.glyphicon-book:before{content:"\e043"}.glyphicon-bookmark:before{content:"\e044"}.glyphicon-print:before{content:"\e045"}.glyphicon-camera:before{content:"\e046"}.glyphicon-font:before{content:"\e047"}.glyphicon-bold:before{content:"\e048"}.glyphicon-italic:before{content:"\e049"}.glyphicon-text-height:before{content:"\e050"}.glyphicon-text-width:before{content:"\e051"}.glyphicon-align-left:before{content:"\e052"}.glyphicon-align-center:before{content:"\e053"}.glyphicon-align-right:before{content:"\e054"}.glyphicon-align-justify:before{content:"\e055"}.glyphicon-list:before{content:"\e056"}.glyphicon-indent-left:before{content:"\e057"}.glyphicon-indent-right:before{content:"\e058"}.glyphicon-facetime-video:before{content:"\e059"}.glyphicon-picture:before{content:"\e060"}.glyphicon-map-marker:before{content:"\e062"}.glyphicon-adjust:before{content:"\e063"}.glyphicon-tint:before{content:"\e064"}.glyphicon-edit:before{content:"\e065"}.glyphicon-share:before{content:"\e066"}.glyphicon-check:before{content:"\e067"}.glyphicon-move:before{content:"\e068"}.glyphicon-step-backward:before{content:"\e069"}.glyphicon-fast-backward:before{content:"\e070"}.glyphicon-backward:before{content:"\e071"}.glyphicon-play:before{content:"\e072"}.glyphicon-pause:before{content:"\e073"}.glyphicon-stop:before{content:"\e074"}.glyphicon-forward:before{content:"\e075"}.glyphicon-fast-forward:before{content:"\e076"}.glyphicon-step-forward:before{content:"\e077"}.glyphicon-eject:before{content:"\e078"}.glyphicon-chevron-left:before{content:"\e079"}.glyphicon-chevron-right:before{content:"\e080"}.glyphicon-plus-sign:before{content:"\e081"}.glyphicon-minus-sign:before{content:"\e082"}.glyphicon-remove-sign:before{content:"\e083"}.glyphicon-ok-sign:before{content:"\e084"}.glyphicon-question-sign:before{content:"\e085"}.glyphicon-info-sign:before{content:"\e086"}.glyphicon-screenshot:before{content:"\e087"}.glyphicon-remove-circle:before{content:"\e088"}.glyphicon-ok-circle:before{content:"\e089"}.glyphicon-ban-circle:before{content:"\e090"}.glyphicon-arrow-left:before{content:"\e091"}.glyphicon-arrow-right:before{content:"\e092"}.glyphicon-arrow-up:before{content:"\e093"}.glyphicon-arrow-down:before{content:"\e094"}.glyphicon-share-alt:before{content:"\e095"}.glyphicon-resize-full:before{content:"\e096"}.glyphicon-resize-small:before{content:"\e097"}.glyphicon-exclamation-sign:before{content:"\e101"}.glyphicon-gift:before{content:"\e102"}.glyphicon-leaf:before{content:"\e103"}.glyphicon-fire:before{content:"\e104"}.glyphicon-eye-open:before{content:"\e105"}.glyphicon-eye-close:before{content:"\e106"}.glyphicon-warning-sign:before{content:"\e107"}.glyphicon-plane:before{content:"\e108"}.glyphicon-calendar:before{content:"\e109"}.glyphicon-random:before{content:"\e110"}.glyphicon-comment:before{content:"\e111"}.glyphicon-magnet:before{content:"\e112"}.glyphicon-chevron-up:before{content:"\e113"}.glyphicon-chevron-down:before{content:"\e114"}.glyphicon-retweet:before{content:"\e115"}.glyphicon-shopping-cart:before{content:"\e116"}.glyphicon-folder-close:before{content:"\e117"}.glyphicon-folder-open:before{content:"\e118"}.glyphicon-resize-vertical:before{content:"\e119"}.glyphicon-resize-horizontal:before{content:"\e120"}.glyphicon-hdd:before{content:"\e121"}.glyphicon-bullhorn:before{content:"\e122"}.glyphicon-bell:before{content:"\e123"}.glyphicon-certificate:before{content:"\e124"}.glyphicon-thumbs-up:before{content:"\e125"}.glyphicon-thumbs-down:before{content:"\e126"}.glyphicon-hand-right:before{content:"\e127"}.glyphicon-hand-left:before{content:"\e128"}.glyphicon-hand-up:before{content:"\e129"}.glyphicon-hand-down:before{content:"\e130"}.glyphicon-circle-arrow-right:before{content:"\e131"}.glyphicon-circle-arrow-left:before{content:"\e132"}.glyphicon-circle-arrow-up:before{content:"\e133"}.glyphicon-circle-arrow-down:before{content:"\e134"}.glyphicon-globe:before{content:"\e135"}.glyphicon-wrench:before{content:"\e136"}.glyphicon-tasks:before{content:"\e137"}.glyphicon-filter:before{content:"\e138"}.glyphicon-briefcase:before{content:"\e139"}.glyphicon-fullscreen:before{content:"\e140"}.glyphicon-dashboard:before{content:"\e141"}.glyphicon-paperclip:before{content:"\e142"}.glyphicon-heart-empty:before{content:"\e143"}.glyphicon-link:before{content:"\e144"}.glyphicon-phone:before{content:"\e145"}.glyphicon-pushpin:before{content:"\e146"}.glyphicon-usd:before{content:"\e148"}.glyphicon-gbp:before{content:"\e149"}.glyphicon-sort:before{content:"\e150"}.glyphicon-sort-by-alphabet:before{content:"\e151"}.glyphicon-sort-by-alphabet-alt:before{content:"\e152"}.glyphicon-sort-by-order:before{content:"\e153"}.glyphicon-sort-by-order-alt:before{content:"\e154"}.glyphicon-sort-by-attributes:before{content:"\e155"}.glyphicon-sort-by-attributes-alt:before{content:"\e156"}.glyphicon-unchecked:before{content:"\e157"}.glyphicon-expand:before{content:"\e158"}.glyphicon-collapse-down:before{content:"\e159"}.glyphicon-collapse-up:before{content:"\e160"}.glyphicon-log-in:before{content:"\e161"}.glyphicon-flash:before{content:"\e162"}.glyphicon-log-out:before{content:"\e163"}.glyphicon-new-window:before{content:"\e164"}.glyphicon-record:before{content:"\e165"}.glyphicon-save:before{content:"\e166"}.glyphicon-open:before{content:"\e167"}.glyphicon-saved:before{content:"\e168"}.glyphicon-import:before{content:"\e169"}.glyphicon-export:before{content:"\e170"}.glyphicon-send:before{content:"\e171"}.glyphicon-floppy-disk:before{content:"\e172"}.glyphicon-floppy-saved:before{content:"\e173"}.glyphicon-floppy-remove:before{content:"\e174"}.glyphicon-floppy-save:before{content:"\e175"}.glyphicon-floppy-open:before{content:"\e176"}.glyphicon-credit-card:before{content:"\e177"}.glyphicon-transfer:before{content:"\e178"}.glyphicon-cutlery:before{content:"\e179"}.glyphicon-header:before{content:"\e180"}.glyphicon-compressed:before{content:"\e181"}.glyphicon-earphone:before{content:"\e182"}.glyphicon-phone-alt:before{content:"\e183"}.glyphicon-tower:before{content:"\e184"}.glyphicon-stats:before{content:"\e185"}.glyphicon-sd-video:before{content:"\e186"}.glyphicon-hd-video:before{content:"\e187"}.glyphicon-subtitles:before{content:"\e188"}.glyphicon-sound-stereo:before{content:"\e189"}.glyphicon-sound-dolby:before{content:"\e190"}.glyphicon-sound-5-1:before{content:"\e191"}.glyphicon-sound-6-1:before{content:"\e192"}.glyphicon-sound-7-1:before{content:"\e193"}.glyphicon-copyright-mark:before{content:"\e194"}.glyphicon-registration-mark:before{content:"\e195"}.glyphicon-cloud-download:before{content:"\e197"}.glyphicon-cloud-upload:before{content:"\e198"}.glyphicon-tree-conifer:before{content:"\e199"}.glyphicon-tree-deciduous:before{content:"\e200"}*{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}:before,:after{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}html{font-size:10px;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;line-height:1.42857143;color:#333;background-color:#fff}input,button,select,textarea{font-family:inherit;font-size:inherit;line-height:inherit}a{color:#337ab7;text-decoration:none}a:hover,a:focus{color:#23527c;text-decoration:underline}a:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}figure{margin:0}img{vertical-align:middle}.img-responsive,.thumbnail>img,.thumbnail a>img,.carousel-inner>.item>img,.carousel-inner>.item>a>img{display:block;max-width:100%;height:auto}.img-rounded{border-radius:6px}.img-thumbnail{display:inline-block;max-width:100%;height:auto;padding:4px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:all .2s ease-in-out;-o-transition:all .2s ease-in-out;transition:all .2s ease-in-out}.img-circle{border-radius:50%}hr{margin-top:20px;margin-bottom:20px;border:0;border-top:1px solid #eee}.sr-only{position:absolute;width:1px;height:1px;padding:0;margin:-1px;overflow:hidden;clip:rect(0,0,0,0);border:0}.sr-only-focusable:active,.sr-only-focusable:focus{position:static;width:auto;height:auto;margin:0;overflow:visible;clip:auto}h1,h2,h3,h4,h5,h6,.h1,.h2,.h3,.h4,.h5,.h6{font-family:inherit;font-weight:500;line-height:1.1;color:inherit}h1 small,h2 small,h3 small,h4 small,h5 small,h6 small,.h1 small,.h2 small,.h3 small,.h4 small,.h5 small,.h6 small,h1 .small,h2 .small,h3 .small,h4 .small,h5 .small,h6 .small,.h1 .small,.h2 .small,.h3 .small,.h4 .small,.h5 .small,.h6 .small{font-weight:400;line-height:1;color:#777}h1,.h1,h2,.h2,h3,.h3{margin-top:20px;margin-bottom:10px}h1 small,.h1 small,h2 small,.h2 small,h3 small,.h3 small,h1 .small,.h1 .small,h2 .small,.h2 .small,h3 .small,.h3 .small{font-size:65%}h4,.h4,h5,.h5,h6,.h6{margin-top:10px;margin-bottom:10px}h4 small,.h4 small,h5 small,.h5 small,h6 small,.h6 small,h4 .small,.h4 .small,h5 .small,.h5 .small,h6 .small,.h6 .small{font-size:75%}h1,.h1{font-size:36px}h2,.h2{font-size:30px}h3,.h3{font-size:24px}h4,.h4{font-size:18px}h5,.h5{font-size:14px}h6,.h6{font-size:12px}p{margin:0 0 10px}.lead{margin-bottom:20px;font-size:16px;font-weight:300;line-height:1.4}@media (min-width:768px){.lead{font-size:21px}}small,.small{font-size:85%}mark,.mark{padding:.2em;background-color:#fcf8e3}.text-left{text-align:left}.text-right{text-align:right}.text-center{text-align:center}.text-justify{text-align:justify}.text-nowrap{white-space:nowrap}.text-lowercase{text-transform:lowercase}.text-uppercase{text-transform:uppercase}.text-capitalize{text-transform:capitalize}.text-muted{color:#777}.text-primary{color:#337ab7}a.text-primary:hover{color:#286090}.text-success{color:#3c763d}a.text-success:hover{color:#2b542c}.text-info{color:#31708f}a.text-info:hover{color:#245269}.text-warning{color:#8a6d3b}a.text-warning:hover{color:#66512c}.text-danger{color:#a94442}a.text-danger:hover{color:#843534}.bg-primary{color:#fff;background-color:#337ab7}a.bg-primary:hover{background-color:#286090}.bg-success{background-color:#dff0d8}a.bg-success:hover{background-color:#c1e2b3}.bg-info{background-color:#d9edf7}a.bg-info:hover{background-color:#afd9ee}.bg-warning{background-color:#fcf8e3}a.bg-warning:hover{background-color:#f7ecb5}.bg-danger{background-color:#f2dede}a.bg-danger:hover{background-color:#e4b9b9}.page-header{padding-bottom:9px;margin:40px 0 20px;border-bottom:1px solid #eee}ul,ol{margin-top:0;margin-bottom:10px}ul ul,ol ul,ul ol,ol ol{margin-bottom:0}.list-unstyled{padding-left:0;list-style:none}.list-inline{padding-left:0;margin-left:-5px;list-style:none}.list-inline>li{display:inline-block;padding-right:5px;padding-left:5px}dl{margin-top:0;margin-bottom:20px}dt,dd{line-height:1.42857143}dt{font-weight:700}dd{margin-left:0}@media (min-width:768px){.dl-horizontal dt{float:left;width:160px;overflow:hidden;clear:left;text-align:right;text-overflow:ellipsis;white-space:nowrap}.dl-horizontal dd{margin-left:180px}}abbr[title],abbr[data-original-title]{cursor:help;border-bottom:1px dotted #777}.initialism{font-size:90%;text-transform:uppercase}blockquote{padding:10px 20px;margin:0 0 20px;font-size:17.5px;border-left:5px solid #eee}blockquote p:last-child,blockquote ul:last-child,blockquote ol:last-child{margin-bottom:0}blockquote footer,blockquote small,blockquote .small{display:block;font-size:80%;line-height:1.42857143;color:#777}blockquote footer:before,blockquote small:before,blockquote .small:before{content:'\2014 \00A0'}.blockquote-reverse,blockquote.pull-right{padding-right:15px;padding-left:0;text-align:right;border-right:5px solid #eee;border-left:0}.blockquote-reverse footer:before,blockquote.pull-right footer:before,.blockquote-reverse small:before,blockquote.pull-right small:before,.blockquote-reverse .small:before,blockquote.pull-right .small:before{content:''}.blockquote-reverse footer:after,blockquote.pull-right footer:after,.blockquote-reverse small:after,blockquote.pull-right small:after,.blockquote-reverse .small:after,blockquote.pull-right .small:after{content:'\00A0 \2014'}address{margin-bottom:20px;font-style:normal;line-height:1.42857143}code,kbd,pre,samp{font-family:Menlo,Monaco,Consolas,"Courier New",monospace}code{padding:2px 4px;font-size:90%;color:#c7254e;background-color:#f9f2f4;border-radius:4px}kbd{padding:2px 4px;font-size:90%;color:#fff;background-color:#333;border-radius:3px;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.25);box-shadow:inset 0 -1px 0 rgba(0,0,0,.25)}kbd kbd{padding:0;font-size:100%;font-weight:700;-webkit-box-shadow:none;box-shadow:none}pre{display:block;padding:9.5px;margin:0 0 10px;font-size:13px;line-height:1.42857143;color:#333;word-break:break-all;word-wrap:break-word;background-color:#f5f5f5;border:1px solid #ccc;border-radius:4px}pre code{padding:0;font-size:inherit;color:inherit;white-space:pre-wrap;background-color:transparent;border-radius:0}.pre-scrollable{max-height:340px;overflow-y:scroll}.container{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}@media (min-width:768px){.container{width:750px}}@media (min-width:992px){.container{width:970px}}@media (min-width:1200px){.container{width:1170px}}.container-fluid{padding-right:15px;padding-left:15px;margin-right:auto;margin-left:auto}.row{margin-right:-15px;margin-left:-15px}.col-xs-1,.col-sm-1,.col-md-1,.col-lg-1,.col-xs-2,.col-sm-2,.col-md-2,.col-lg-2,.col-xs-3,.col-sm-3,.col-md-3,.col-lg-3,.col-xs-4,.col-sm-4,.col-md-4,.col-lg-4,.col-xs-5,.col-sm-5,.col-md-5,.col-lg-5,.col-xs-6,.col-sm-6,.col-md-6,.col-lg-6,.col-xs-7,.col-sm-7,.col-md-7,.col-lg-7,.col-xs-8,.col-sm-8,.col-md-8,.col-lg-8,.col-xs-9,.col-sm-9,.col-md-9,.col-lg-9,.col-xs-10,.col-sm-10,.col-md-10,.col-lg-10,.col-xs-11,.col-sm-11,.col-md-11,.col-lg-11,.col-xs-12,.col-sm-12,.col-md-12,.col-lg-12{position:relative;min-height:1px;padding-right:15px;padding-left:15px}.col-xs-1,.col-xs-2,.col-xs-3,.col-xs-4,.col-xs-5,.col-xs-6,.col-xs-7,.col-xs-8,.col-xs-9,.col-xs-10,.col-xs-11,.col-xs-12{float:left}.col-xs-12{width:100%}.col-xs-11{width:91.66666667%}.col-xs-10{width:83.33333333%}.col-xs-9{width:75%}.col-xs-8{width:66.66666667%}.col-xs-7{width:58.33333333%}.col-xs-6{width:50%}.col-xs-5{width:41.66666667%}.col-xs-4{width:33.33333333%}.col-xs-3{width:25%}.col-xs-2{width:16.66666667%}.col-xs-1{width:8.33333333%}.col-xs-pull-12{right:100%}.col-xs-pull-11{right:91.66666667%}.col-xs-pull-10{right:83.33333333%}.col-xs-pull-9{right:75%}.col-xs-pull-8{right:66.66666667%}.col-xs-pull-7{right:58.33333333%}.col-xs-pull-6{right:50%}.col-xs-pull-5{right:41.66666667%}.col-xs-pull-4{right:33.33333333%}.col-xs-pull-3{right:25%}.col-xs-pull-2{right:16.66666667%}.col-xs-pull-1{right:8.33333333%}.col-xs-pull-0{right:auto}.col-xs-push-12{left:100%}.col-xs-push-11{left:91.66666667%}.col-xs-push-10{left:83.33333333%}.col-xs-push-9{left:75%}.col-xs-push-8{left:66.66666667%}.col-xs-push-7{left:58.33333333%}.col-xs-push-6{left:50%}.col-xs-push-5{left:41.66666667%}.col-xs-push-4{left:33.33333333%}.col-xs-push-3{left:25%}.col-xs-push-2{left:16.66666667%}.col-xs-push-1{left:8.33333333%}.col-xs-push-0{left:auto}.col-xs-offset-12{margin-left:100%}.col-xs-offset-11{margin-left:91.66666667%}.col-xs-offset-10{margin-left:83.33333333%}.col-xs-offset-9{margin-left:75%}.col-xs-offset-8{margin-left:66.66666667%}.col-xs-offset-7{margin-left:58.33333333%}.col-xs-offset-6{margin-left:50%}.col-xs-offset-5{margin-left:41.66666667%}.col-xs-offset-4{margin-left:33.33333333%}.col-xs-offset-3{margin-left:25%}.col-xs-offset-2{margin-left:16.66666667%}.col-xs-offset-1{margin-left:8.33333333%}.col-xs-offset-0{margin-left:0}@media (min-width:768px){.col-sm-1,.col-sm-2,.col-sm-3,.col-sm-4,.col-sm-5,.col-sm-6,.col-sm-7,.col-sm-8,.col-sm-9,.col-sm-10,.col-sm-11,.col-sm-12{float:left}.col-sm-12{width:100%}.col-sm-11{width:91.66666667%}.col-sm-10{width:83.33333333%}.col-sm-9{width:75%}.col-sm-8{width:66.66666667%}.col-sm-7{width:58.33333333%}.col-sm-6{width:50%}.col-sm-5{width:41.66666667%}.col-sm-4{width:33.33333333%}.col-sm-3{width:25%}.col-sm-2{width:16.66666667%}.col-sm-1{width:8.33333333%}.col-sm-pull-12{right:100%}.col-sm-pull-11{right:91.66666667%}.col-sm-pull-10{right:83.33333333%}.col-sm-pull-9{right:75%}.col-sm-pull-8{right:66.66666667%}.col-sm-pull-7{right:58.33333333%}.col-sm-pull-6{right:50%}.col-sm-pull-5{right:41.66666667%}.col-sm-pull-4{right:33.33333333%}.col-sm-pull-3{right:25%}.col-sm-pull-2{right:16.66666667%}.col-sm-pull-1{right:8.33333333%}.col-sm-pull-0{right:auto}.col-sm-push-12{left:100%}.col-sm-push-11{left:91.66666667%}.col-sm-push-10{left:83.33333333%}.col-sm-push-9{left:75%}.col-sm-push-8{left:66.66666667%}.col-sm-push-7{left:58.33333333%}.col-sm-push-6{left:50%}.col-sm-push-5{left:41.66666667%}.col-sm-push-4{left:33.33333333%}.col-sm-push-3{left:25%}.col-sm-push-2{left:16.66666667%}.col-sm-push-1{left:8.33333333%}.col-sm-push-0{left:auto}.col-sm-offset-12{margin-left:100%}.col-sm-offset-11{margin-left:91.66666667%}.col-sm-offset-10{margin-left:83.33333333%}.col-sm-offset-9{margin-left:75%}.col-sm-offset-8{margin-left:66.66666667%}.col-sm-offset-7{margin-left:58.33333333%}.col-sm-offset-6{margin-left:50%}.col-sm-offset-5{margin-left:41.66666667%}.col-sm-offset-4{margin-left:33.33333333%}.col-sm-offset-3{margin-left:25%}.col-sm-offset-2{margin-left:16.66666667%}.col-sm-offset-1{margin-left:8.33333333%}.col-sm-offset-0{margin-left:0}}@media (min-width:992px){.col-md-1,.col-md-2,.col-md-3,.col-md-4,.col-md-5,.col-md-6,.col-md-7,.col-md-8,.col-md-9,.col-md-10,.col-md-11,.col-md-12{float:left}.col-md-12{width:100%}.col-md-11{width:91.66666667%}.col-md-10{width:83.33333333%}.col-md-9{width:75%}.col-md-8{width:66.66666667%}.col-md-7{width:58.33333333%}.col-md-6{width:50%}.col-md-5{width:41.66666667%}.col-md-4{width:33.33333333%}.col-md-3{width:25%}.col-md-2{width:16.66666667%}.col-md-1{width:8.33333333%}.col-md-pull-12{right:100%}.col-md-pull-11{right:91.66666667%}.col-md-pull-10{right:83.33333333%}.col-md-pull-9{right:75%}.col-md-pull-8{right:66.66666667%}.col-md-pull-7{right:58.33333333%}.col-md-pull-6{right:50%}.col-md-pull-5{right:41.66666667%}.col-md-pull-4{right:33.33333333%}.col-md-pull-3{right:25%}.col-md-pull-2{right:16.66666667%}.col-md-pull-1{right:8.33333333%}.col-md-pull-0{right:auto}.col-md-push-12{left:100%}.col-md-push-11{left:91.66666667%}.col-md-push-10{left:83.33333333%}.col-md-push-9{left:75%}.col-md-push-8{left:66.66666667%}.col-md-push-7{left:58.33333333%}.col-md-push-6{left:50%}.col-md-push-5{left:41.66666667%}.col-md-push-4{left:33.33333333%}.col-md-push-3{left:25%}.col-md-push-2{left:16.66666667%}.col-md-push-1{left:8.33333333%}.col-md-push-0{left:auto}.col-md-offset-12{margin-left:100%}.col-md-offset-11{margin-left:91.66666667%}.col-md-offset-10{margin-left:83.33333333%}.col-md-offset-9{margin-left:75%}.col-md-offset-8{margin-left:66.66666667%}.col-md-offset-7{margin-left:58.33333333%}.col-md-offset-6{margin-left:50%}.col-md-offset-5{margin-left:41.66666667%}.col-md-offset-4{margin-left:33.33333333%}.col-md-offset-3{margin-left:25%}.col-md-offset-2{margin-left:16.66666667%}.col-md-offset-1{margin-left:8.33333333%}.col-md-offset-0{margin-left:0}}@media (min-width:1200px){.col-lg-1,.col-lg-2,.col-lg-3,.col-lg-4,.col-lg-5,.col-lg-6,.col-lg-7,.col-lg-8,.col-lg-9,.col-lg-10,.col-lg-11,.col-lg-12{float:left}.col-lg-12{width:100%}.col-lg-11{width:91.66666667%}.col-lg-10{width:83.33333333%}.col-lg-9{width:75%}.col-lg-8{width:66.66666667%}.col-lg-7{width:58.33333333%}.col-lg-6{width:50%}.col-lg-5{width:41.66666667%}.col-lg-4{width:33.33333333%}.col-lg-3{width:25%}.col-lg-2{width:16.66666667%}.col-lg-1{width:8.33333333%}.col-lg-pull-12{right:100%}.col-lg-pull-11{right:91.66666667%}.col-lg-pull-10{right:83.33333333%}.col-lg-pull-9{right:75%}.col-lg-pull-8{right:66.66666667%}.col-lg-pull-7{right:58.33333333%}.col-lg-pull-6{right:50%}.col-lg-pull-5{right:41.66666667%}.col-lg-pull-4{right:33.33333333%}.col-lg-pull-3{right:25%}.col-lg-pull-2{right:16.66666667%}.col-lg-pull-1{right:8.33333333%}.col-lg-pull-0{right:auto}.col-lg-push-12{left:100%}.col-lg-push-11{left:91.66666667%}.col-lg-push-10{left:83.33333333%}.col-lg-push-9{left:75%}.col-lg-push-8{left:66.66666667%}.col-lg-push-7{left:58.33333333%}.col-lg-push-6{left:50%}.col-lg-push-5{left:41.66666667%}.col-lg-push-4{left:33.33333333%}.col-lg-push-3{left:25%}.col-lg-push-2{left:16.66666667%}.col-lg-push-1{left:8.33333333%}.col-lg-push-0{left:auto}.col-lg-offset-12{margin-left:100%}.col-lg-offset-11{margin-left:91.66666667%}.col-lg-offset-10{margin-left:83.33333333%}.col-lg-offset-9{margin-left:75%}.col-lg-offset-8{margin-left:66.66666667%}.col-lg-offset-7{margin-left:58.33333333%}.col-lg-offset-6{margin-left:50%}.col-lg-offset-5{margin-left:41.66666667%}.col-lg-offset-4{margin-left:33.33333333%}.col-lg-offset-3{margin-left:25%}.col-lg-offset-2{margin-left:16.66666667%}.col-lg-offset-1{margin-left:8.33333333%}.col-lg-offset-0{margin-left:0}}table{background-color:transparent}caption{padding-top:8px;padding-bottom:8px;color:#777;text-align:left}th{text-align:left}.table{width:100%;max-width:100%;margin-bottom:20px}.table>thead>tr>th,.table>tbody>tr>th,.table>tfoot>tr>th,.table>thead>tr>td,.table>tbody>tr>td,.table>tfoot>tr>td{padding:8px;line-height:1.42857143;vertical-align:top;border-top:1px solid #ddd}.table>thead>tr>th{vertical-align:bottom;border-bottom:2px solid #ddd}.table>caption+thead>tr:first-child>th,.table>colgroup+thead>tr:first-child>th,.table>thead:first-child>tr:first-child>th,.table>caption+thead>tr:first-child>td,.table>colgroup+thead>tr:first-child>td,.table>thead:first-child>tr:first-child>td{border-top:0}.table>tbody+tbody{border-top:2px solid #ddd}.table .table{background-color:#fff}.table-condensed>thead>tr>th,.table-condensed>tbody>tr>th,.table-condensed>tfoot>tr>th,.table-condensed>thead>tr>td,.table-condensed>tbody>tr>td,.table-condensed>tfoot>tr>td{padding:5px}.table-bordered{border:1px solid #ddd}.table-bordered>thead>tr>th,.table-bordered>tbody>tr>th,.table-bordered>tfoot>tr>th,.table-bordered>thead>tr>td,.table-bordered>tbody>tr>td,.table-bordered>tfoot>tr>td{border:1px solid #ddd}.table-bordered>thead>tr>th,.table-bordered>thead>tr>td{border-bottom-width:2px}.table-striped>tbody>tr:nth-child(odd){background-color:#f9f9f9}.table-hover>tbody>tr:hover{background-color:#f5f5f5}table col[class*=col-]{position:static;display:table-column;float:none}table td[class*=col-],table th[class*=col-]{position:static;display:table-cell;float:none}.table>thead>tr>td.active,.table>tbody>tr>td.active,.table>tfoot>tr>td.active,.table>thead>tr>th.active,.table>tbody>tr>th.active,.table>tfoot>tr>th.active,.table>thead>tr.active>td,.table>tbody>tr.active>td,.table>tfoot>tr.active>td,.table>thead>tr.active>th,.table>tbody>tr.active>th,.table>tfoot>tr.active>th{background-color:#f5f5f5}.table-hover>tbody>tr>td.active:hover,.table-hover>tbody>tr>th.active:hover,.table-hover>tbody>tr.active:hover>td,.table-hover>tbody>tr:hover>.active,.table-hover>tbody>tr.active:hover>th{background-color:#e8e8e8}.table>thead>tr>td.success,.table>tbody>tr>td.success,.table>tfoot>tr>td.success,.table>thead>tr>th.success,.table>tbody>tr>th.success,.table>tfoot>tr>th.success,.table>thead>tr.success>td,.table>tbody>tr.success>td,.table>tfoot>tr.success>td,.table>thead>tr.success>th,.table>tbody>tr.success>th,.table>tfoot>tr.success>th{background-color:#dff0d8}.table-hover>tbody>tr>td.success:hover,.table-hover>tbody>tr>th.success:hover,.table-hover>tbody>tr.success:hover>td,.table-hover>tbody>tr:hover>.success,.table-hover>tbody>tr.success:hover>th{background-color:#d0e9c6}.table>thead>tr>td.info,.table>tbody>tr>td.info,.table>tfoot>tr>td.info,.table>thead>tr>th.info,.table>tbody>tr>th.info,.table>tfoot>tr>th.info,.table>thead>tr.info>td,.table>tbody>tr.info>td,.table>tfoot>tr.info>td,.table>thead>tr.info>th,.table>tbody>tr.info>th,.table>tfoot>tr.info>th{background-color:#d9edf7}.table-hover>tbody>tr>td.info:hover,.table-hover>tbody>tr>th.info:hover,.table-hover>tbody>tr.info:hover>td,.table-hover>tbody>tr:hover>.info,.table-hover>tbody>tr.info:hover>th{background-color:#c4e3f3}.table>thead>tr>td.warning,.table>tbody>tr>td.warning,.table>tfoot>tr>td.warning,.table>thead>tr>th.warning,.table>tbody>tr>th.warning,.table>tfoot>tr>th.warning,.table>thead>tr.warning>td,.table>tbody>tr.warning>td,.table>tfoot>tr.warning>td,.table>thead>tr.warning>th,.table>tbody>tr.warning>th,.table>tfoot>tr.warning>th{background-color:#fcf8e3}.table-hover>tbody>tr>td.warning:hover,.table-hover>tbody>tr>th.warning:hover,.table-hover>tbody>tr.warning:hover>td,.table-hover>tbody>tr:hover>.warning,.table-hover>tbody>tr.warning:hover>th{background-color:#faf2cc}.table>thead>tr>td.danger,.table>tbody>tr>td.danger,.table>tfoot>tr>td.danger,.table>thead>tr>th.danger,.table>tbody>tr>th.danger,.table>tfoot>tr>th.danger,.table>thead>tr.danger>td,.table>tbody>tr.danger>td,.table>tfoot>tr.danger>td,.table>thead>tr.danger>th,.table>tbody>tr.danger>th,.table>tfoot>tr.danger>th{background-color:#f2dede}.table-hover>tbody>tr>td.danger:hover,.table-hover>tbody>tr>th.danger:hover,.table-hover>tbody>tr.danger:hover>td,.table-hover>tbody>tr:hover>.danger,.table-hover>tbody>tr.danger:hover>th{background-color:#ebcccc}.table-responsive{min-height:.01%;overflow-x:auto}@media screen and (max-width:767px){.table-responsive{width:100%;margin-bottom:15px;overflow-y:hidden;-ms-overflow-style:-ms-autohiding-scrollbar;border:1px solid #ddd}.table-responsive>.table{margin-bottom:0}.table-responsive>.table>thead>tr>th,.table-responsive>.table>tbody>tr>th,.table-responsive>.table>tfoot>tr>th,.table-responsive>.table>thead>tr>td,.table-responsive>.table>tbody>tr>td,.table-responsive>.table>tfoot>tr>td{white-space:nowrap}.table-responsive>.table-bordered{border:0}.table-responsive>.table-bordered>thead>tr>th:first-child,.table-responsive>.table-bordered>tbody>tr>th:first-child,.table-responsive>.table-bordered>tfoot>tr>th:first-child,.table-responsive>.table-bordered>thead>tr>td:first-child,.table-responsive>.table-bordered>tbody>tr>td:first-child,.table-responsive>.table-bordered>tfoot>tr>td:first-child{border-left:0}.table-responsive>.table-bordered>thead>tr>th:last-child,.table-responsive>.table-bordered>tbody>tr>th:last-child,.table-responsive>.table-bordered>tfoot>tr>th:last-child,.table-responsive>.table-bordered>thead>tr>td:last-child,.table-responsive>.table-bordered>tbody>tr>td:last-child,.table-responsive>.table-bordered>tfoot>tr>td:last-child{border-right:0}.table-responsive>.table-bordered>tbody>tr:last-child>th,.table-responsive>.table-bordered>tfoot>tr:last-child>th,.table-responsive>.table-bordered>tbody>tr:last-child>td,.table-responsive>.table-bordered>tfoot>tr:last-child>td{border-bottom:0}}fieldset{min-width:0;padding:0;margin:0;border:0}legend{display:block;width:100%;padding:0;margin-bottom:20px;font-size:21px;line-height:inherit;color:#333;border:0;border-bottom:1px solid #e5e5e5}label{display:inline-block;max-width:100%;margin-bottom:5px;font-weight:700}input[type=search]{-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box}input[type=radio],input[type=checkbox]{margin:4px 0 0;margin-top:1px \9;line-height:normal}input[type=file]{display:block}input[type=range]{display:block;width:100%}select[multiple],select[size]{height:auto}input[type=file]:focus,input[type=radio]:focus,input[type=checkbox]:focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}output{display:block;padding-top:7px;font-size:14px;line-height:1.42857143;color:#555}.form-control{display:block;width:100%;height:34px;padding:6px 12px;font-size:14px;line-height:1.42857143;color:#555;background-color:#fff;background-image:none;border:1px solid #ccc;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075);-webkit-transition:border-color ease-in-out .15s,-webkit-box-shadow ease-in-out .15s;-o-transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s;transition:border-color ease-in-out .15s,box-shadow ease-in-out .15s}.form-control:focus{border-color:#66afe9;outline:0;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6);box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 8px rgba(102,175,233,.6)}.form-control::-moz-placeholder{color:#999;opacity:1}.form-control:-ms-input-placeholder{color:#999}.form-control::-webkit-input-placeholder{color:#999}.form-control[disabled],.form-control[readonly],fieldset[disabled] .form-control{cursor:not-allowed;background-color:#eee;opacity:1}textarea.form-control{height:auto}input[type=search]{-webkit-appearance:none}@media screen and (-webkit-min-device-pixel-ratio:0){input[type=date],input[type=time],input[type=datetime-local],input[type=month]{line-height:34px}input[type=date].input-sm,input[type=time].input-sm,input[type=datetime-local].input-sm,input[type=month].input-sm{line-height:30px}input[type=date].input-lg,input[type=time].input-lg,input[type=datetime-local].input-lg,input[type=month].input-lg{line-height:46px}}.form-group{margin-bottom:15px}.radio,.checkbox{position:relative;display:block;margin-top:10px;margin-bottom:10px}.radio label,.checkbox label{min-height:20px;padding-left:20px;margin-bottom:0;font-weight:400;cursor:pointer}.radio input[type=radio],.radio-inline input[type=radio],.checkbox input[type=checkbox],.checkbox-inline input[type=checkbox]{position:absolute;margin-top:4px \9;margin-left:-20px}.radio+.radio,.checkbox+.checkbox{margin-top:-5px}.radio-inline,.checkbox-inline{display:inline-block;padding-left:20px;margin-bottom:0;font-weight:400;vertical-align:middle;cursor:pointer}.radio-inline+.radio-inline,.checkbox-inline+.checkbox-inline{margin-top:0;margin-left:10px}input[type=radio][disabled],input[type=checkbox][disabled],input[type=radio].disabled,input[type=checkbox].disabled,fieldset[disabled] input[type=radio],fieldset[disabled] input[type=checkbox]{cursor:not-allowed}.radio-inline.disabled,.checkbox-inline.disabled,fieldset[disabled] .radio-inline,fieldset[disabled] .checkbox-inline{cursor:not-allowed}.radio.disabled label,.checkbox.disabled label,fieldset[disabled] .radio label,fieldset[disabled] .checkbox label{cursor:not-allowed}.form-control-static{padding-top:7px;padding-bottom:7px;margin-bottom:0}.form-control-static.input-lg,.form-control-static.input-sm{padding-right:0;padding-left:0}.input-sm,.form-group-sm .form-control{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-sm,select.form-group-sm .form-control{height:30px;line-height:30px}textarea.input-sm,textarea.form-group-sm .form-control,select[multiple].input-sm,select[multiple].form-group-sm .form-control{height:auto}.input-lg,.form-group-lg .form-control{height:46px;padding:10px 16px;font-size:18px;line-height:1.33;border-radius:6px}select.input-lg,select.form-group-lg .form-control{height:46px;line-height:46px}textarea.input-lg,textarea.form-group-lg .form-control,select[multiple].input-lg,select[multiple].form-group-lg .form-control{height:auto}.has-feedback{position:relative}.has-feedback .form-control{padding-right:42.5px}.form-control-feedback{position:absolute;top:0;right:0;z-index:2;display:block;width:34px;height:34px;line-height:34px;text-align:center;pointer-events:none}.input-lg+.form-control-feedback{width:46px;height:46px;line-height:46px}.input-sm+.form-control-feedback{width:30px;height:30px;line-height:30px}.has-success .help-block,.has-success .control-label,.has-success .radio,.has-success .checkbox,.has-success .radio-inline,.has-success .checkbox-inline,.has-success.radio label,.has-success.checkbox label,.has-success.radio-inline label,.has-success.checkbox-inline label{color:#3c763d}.has-success .form-control{border-color:#3c763d;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-success .form-control:focus{border-color:#2b542c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #67b168}.has-success .input-group-addon{color:#3c763d;background-color:#dff0d8;border-color:#3c763d}.has-success .form-control-feedback{color:#3c763d}.has-warning .help-block,.has-warning .control-label,.has-warning .radio,.has-warning .checkbox,.has-warning .radio-inline,.has-warning .checkbox-inline,.has-warning.radio label,.has-warning.checkbox label,.has-warning.radio-inline label,.has-warning.checkbox-inline label{color:#8a6d3b}.has-warning .form-control{border-color:#8a6d3b;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-warning .form-control:focus{border-color:#66512c;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #c0a16b}.has-warning .input-group-addon{color:#8a6d3b;background-color:#fcf8e3;border-color:#8a6d3b}.has-warning .form-control-feedback{color:#8a6d3b}.has-error .help-block,.has-error .control-label,.has-error .radio,.has-error .checkbox,.has-error .radio-inline,.has-error .checkbox-inline,.has-error.radio label,.has-error.checkbox label,.has-error.radio-inline label,.has-error.checkbox-inline label{color:#a94442}.has-error .form-control{border-color:#a94442;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075);box-shadow:inset 0 1px 1px rgba(0,0,0,.075)}.has-error .form-control:focus{border-color:#843534;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483;box-shadow:inset 0 1px 1px rgba(0,0,0,.075),0 0 6px #ce8483}.has-error .input-group-addon{color:#a94442;background-color:#f2dede;border-color:#a94442}.has-error .form-control-feedback{color:#a94442}.has-feedback label~.form-control-feedback{top:25px}.has-feedback label.sr-only~.form-control-feedback{top:0}.help-block{display:block;margin-top:5px;margin-bottom:10px;color:#737373}@media (min-width:768px){.form-inline .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.form-inline .form-control{display:inline-block;width:auto;vertical-align:middle}.form-inline .form-control-static{display:inline-block}.form-inline .input-group{display:inline-table;vertical-align:middle}.form-inline .input-group .input-group-addon,.form-inline .input-group .input-group-btn,.form-inline .input-group .form-control{width:auto}.form-inline .input-group>.form-control{width:100%}.form-inline .control-label{margin-bottom:0;vertical-align:middle}.form-inline .radio,.form-inline .checkbox{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.form-inline .radio label,.form-inline .checkbox label{padding-left:0}.form-inline .radio input[type=radio],.form-inline .checkbox input[type=checkbox]{position:relative;margin-left:0}.form-inline .has-feedback .form-control-feedback{top:0}}.form-horizontal .radio,.form-horizontal .checkbox,.form-horizontal .radio-inline,.form-horizontal .checkbox-inline{padding-top:7px;margin-top:0;margin-bottom:0}.form-horizontal .radio,.form-horizontal .checkbox{min-height:27px}.form-horizontal .form-group{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.form-horizontal .control-label{padding-top:7px;margin-bottom:0;text-align:right}}.form-horizontal .has-feedback .form-control-feedback{right:15px}@media (min-width:768px){.form-horizontal .form-group-lg .control-label{padding-top:14.3px}}@media (min-width:768px){.form-horizontal .form-group-sm .control-label{padding-top:6px}}.btn{display:inline-block;padding:6px 12px;margin-bottom:0;font-size:14px;font-weight:400;line-height:1.42857143;text-align:center;white-space:nowrap;vertical-align:middle;-ms-touch-action:manipulation;touch-action:manipulation;cursor:pointer;-webkit-user-select:none;-moz-user-select:none;-ms-user-select:none;user-select:none;background-image:none;border:1px solid transparent;border-radius:4px}.btn:focus,.btn:active:focus,.btn.active:focus,.btn.focus,.btn:active.focus,.btn.active.focus{outline:thin dotted;outline:5px auto -webkit-focus-ring-color;outline-offset:-2px}.btn:hover,.btn:focus,.btn.focus{color:#333;text-decoration:none}.btn:active,.btn.active{background-image:none;outline:0;-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn.disabled,.btn[disabled],fieldset[disabled] .btn{pointer-events:none;cursor:not-allowed;filter:alpha(opacity=65);-webkit-box-shadow:none;box-shadow:none;opacity:.65}.btn-default{color:#333;background-color:#fff;border-color:#ccc}.btn-default:hover,.btn-default:focus,.btn-default.focus,.btn-default:active,.btn-default.active,.open>.dropdown-toggle.btn-default{color:#333;background-color:#e6e6e6;border-color:#adadad}.btn-default:active,.btn-default.active,.open>.dropdown-toggle.btn-default{background-image:none}.btn-default.disabled,.btn-default[disabled],fieldset[disabled] .btn-default,.btn-default.disabled:hover,.btn-default[disabled]:hover,fieldset[disabled] .btn-default:hover,.btn-default.disabled:focus,.btn-default[disabled]:focus,fieldset[disabled] .btn-default:focus,.btn-default.disabled.focus,.btn-default[disabled].focus,fieldset[disabled] .btn-default.focus,.btn-default.disabled:active,.btn-default[disabled]:active,fieldset[disabled] .btn-default:active,.btn-default.disabled.active,.btn-default[disabled].active,fieldset[disabled] .btn-default.active{background-color:#fff;border-color:#ccc}.btn-default .badge{color:#fff;background-color:#333}.btn-primary{color:#fff;background-color:#337ab7;border-color:#2e6da4}.btn-primary:hover,.btn-primary:focus,.btn-primary.focus,.btn-primary:active,.btn-primary.active,.open>.dropdown-toggle.btn-primary{color:#fff;background-color:#286090;border-color:#204d74}.btn-primary:active,.btn-primary.active,.open>.dropdown-toggle.btn-primary{background-image:none}.btn-primary.disabled,.btn-primary[disabled],fieldset[disabled] .btn-primary,.btn-primary.disabled:hover,.btn-primary[disabled]:hover,fieldset[disabled] .btn-primary:hover,.btn-primary.disabled:focus,.btn-primary[disabled]:focus,fieldset[disabled] .btn-primary:focus,.btn-primary.disabled.focus,.btn-primary[disabled].focus,fieldset[disabled] .btn-primary.focus,.btn-primary.disabled:active,.btn-primary[disabled]:active,fieldset[disabled] .btn-primary:active,.btn-primary.disabled.active,.btn-primary[disabled].active,fieldset[disabled] .btn-primary.active{background-color:#337ab7;border-color:#2e6da4}.btn-primary .badge{color:#337ab7;background-color:#fff}.btn-success{color:#fff;background-color:#5cb85c;border-color:#4cae4c}.btn-success:hover,.btn-success:focus,.btn-success.focus,.btn-success:active,.btn-success.active,.open>.dropdown-toggle.btn-success{color:#fff;background-color:#449d44;border-color:#398439}.btn-success:active,.btn-success.active,.open>.dropdown-toggle.btn-success{background-image:none}.btn-success.disabled,.btn-success[disabled],fieldset[disabled] .btn-success,.btn-success.disabled:hover,.btn-success[disabled]:hover,fieldset[disabled] .btn-success:hover,.btn-success.disabled:focus,.btn-success[disabled]:focus,fieldset[disabled] .btn-success:focus,.btn-success.disabled.focus,.btn-success[disabled].focus,fieldset[disabled] .btn-success.focus,.btn-success.disabled:active,.btn-success[disabled]:active,fieldset[disabled] .btn-success:active,.btn-success.disabled.active,.btn-success[disabled].active,fieldset[disabled] .btn-success.active{background-color:#5cb85c;border-color:#4cae4c}.btn-success .badge{color:#5cb85c;background-color:#fff}.btn-info{color:#fff;background-color:#5bc0de;border-color:#46b8da}.btn-info:hover,.btn-info:focus,.btn-info.focus,.btn-info:active,.btn-info.active,.open>.dropdown-toggle.btn-info{color:#fff;background-color:#31b0d5;border-color:#269abc}.btn-info:active,.btn-info.active,.open>.dropdown-toggle.btn-info{background-image:none}.btn-info.disabled,.btn-info[disabled],fieldset[disabled] .btn-info,.btn-info.disabled:hover,.btn-info[disabled]:hover,fieldset[disabled] .btn-info:hover,.btn-info.disabled:focus,.btn-info[disabled]:focus,fieldset[disabled] .btn-info:focus,.btn-info.disabled.focus,.btn-info[disabled].focus,fieldset[disabled] .btn-info.focus,.btn-info.disabled:active,.btn-info[disabled]:active,fieldset[disabled] .btn-info:active,.btn-info.disabled.active,.btn-info[disabled].active,fieldset[disabled] .btn-info.active{background-color:#5bc0de;border-color:#46b8da}.btn-info .badge{color:#5bc0de;background-color:#fff}.btn-warning{color:#fff;background-color:#f0ad4e;border-color:#eea236}.btn-warning:hover,.btn-warning:focus,.btn-warning.focus,.btn-warning:active,.btn-warning.active,.open>.dropdown-toggle.btn-warning{color:#fff;background-color:#ec971f;border-color:#d58512}.btn-warning:active,.btn-warning.active,.open>.dropdown-toggle.btn-warning{background-image:none}.btn-warning.disabled,.btn-warning[disabled],fieldset[disabled] .btn-warning,.btn-warning.disabled:hover,.btn-warning[disabled]:hover,fieldset[disabled] .btn-warning:hover,.btn-warning.disabled:focus,.btn-warning[disabled]:focus,fieldset[disabled] .btn-warning:focus,.btn-warning.disabled.focus,.btn-warning[disabled].focus,fieldset[disabled] .btn-warning.focus,.btn-warning.disabled:active,.btn-warning[disabled]:active,fieldset[disabled] .btn-warning:active,.btn-warning.disabled.active,.btn-warning[disabled].active,fieldset[disabled] .btn-warning.active{background-color:#f0ad4e;border-color:#eea236}.btn-warning .badge{color:#f0ad4e;background-color:#fff}.btn-danger{color:#fff;background-color:#d9534f;border-color:#d43f3a}.btn-danger:hover,.btn-danger:focus,.btn-danger.focus,.btn-danger:active,.btn-danger.active,.open>.dropdown-toggle.btn-danger{color:#fff;background-color:#c9302c;border-color:#ac2925}.btn-danger:active,.btn-danger.active,.open>.dropdown-toggle.btn-danger{background-image:none}.btn-danger.disabled,.btn-danger[disabled],fieldset[disabled] .btn-danger,.btn-danger.disabled:hover,.btn-danger[disabled]:hover,fieldset[disabled] .btn-danger:hover,.btn-danger.disabled:focus,.btn-danger[disabled]:focus,fieldset[disabled] .btn-danger:focus,.btn-danger.disabled.focus,.btn-danger[disabled].focus,fieldset[disabled] .btn-danger.focus,.btn-danger.disabled:active,.btn-danger[disabled]:active,fieldset[disabled] .btn-danger:active,.btn-danger.disabled.active,.btn-danger[disabled].active,fieldset[disabled] .btn-danger.active{background-color:#d9534f;border-color:#d43f3a}.btn-danger .badge{color:#d9534f;background-color:#fff}.btn-link{font-weight:400;color:#337ab7;border-radius:0}.btn-link,.btn-link:active,.btn-link.active,.btn-link[disabled],fieldset[disabled] .btn-link{background-color:transparent;-webkit-box-shadow:none;box-shadow:none}.btn-link,.btn-link:hover,.btn-link:focus,.btn-link:active{border-color:transparent}.btn-link:hover,.btn-link:focus{color:#23527c;text-decoration:underline;background-color:transparent}.btn-link[disabled]:hover,fieldset[disabled] .btn-link:hover,.btn-link[disabled]:focus,fieldset[disabled] .btn-link:focus{color:#777;text-decoration:none}.btn-lg,.btn-group-lg>.btn{padding:10px 16px;font-size:18px;line-height:1.33;border-radius:6px}.btn-sm,.btn-group-sm>.btn{padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}.btn-xs,.btn-group-xs>.btn{padding:1px 5px;font-size:12px;line-height:1.5;border-radius:3px}.btn-block{display:block;width:100%}.btn-block+.btn-block{margin-top:5px}input[type=submit].btn-block,input[type=reset].btn-block,input[type=button].btn-block{width:100%}.fade{opacity:0;-webkit-transition:opacity .15s linear;-o-transition:opacity .15s linear;transition:opacity .15s linear}.fade.in{opacity:1}.collapse{display:none;visibility:hidden}.collapse.in{display:block;visibility:visible}tr.collapse.in{display:table-row}tbody.collapse.in{display:table-row-group}.collapsing{position:relative;height:0;overflow:hidden;-webkit-transition-timing-function:ease;-o-transition-timing-function:ease;transition-timing-function:ease;-webkit-transition-duration:.35s;-o-transition-duration:.35s;transition-duration:.35s;-webkit-transition-property:height,visibility;-o-transition-property:height,visibility;transition-property:height,visibility}.caret{display:inline-block;width:0;height:0;margin-left:2px;vertical-align:middle;border-top:4px solid;border-right:4px solid transparent;border-left:4px solid transparent}.dropdown{position:relative}.dropdown-toggle:focus{outline:0}.dropdown-menu{position:absolute;top:100%;left:0;z-index:1000;display:none;float:left;min-width:160px;padding:5px 0;margin:2px 0 0;font-size:14px;text-align:left;list-style:none;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.15);border-radius:4px;-webkit-box-shadow:0 6px 12px rgba(0,0,0,.175);box-shadow:0 6px 12px rgba(0,0,0,.175)}.dropdown-menu.pull-right{right:0;left:auto}.dropdown-menu .divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.dropdown-menu>li>a{display:block;padding:3px 20px;clear:both;font-weight:400;line-height:1.42857143;color:#333;white-space:nowrap}.dropdown-menu>li>a:hover,.dropdown-menu>li>a:focus{color:#262626;text-decoration:none;background-color:#f5f5f5}.dropdown-menu>.active>a,.dropdown-menu>.active>a:hover,.dropdown-menu>.active>a:focus{color:#fff;text-decoration:none;background-color:#337ab7;outline:0}.dropdown-menu>.disabled>a,.dropdown-menu>.disabled>a:hover,.dropdown-menu>.disabled>a:focus{color:#777}.dropdown-menu>.disabled>a:hover,.dropdown-menu>.disabled>a:focus{text-decoration:none;cursor:not-allowed;background-color:transparent;background-image:none;filter:progid:DXImageTransform.Microsoft.gradient(enabled=false)}.open>.dropdown-menu{display:block}.open>a{outline:0}.dropdown-menu-right{right:0;left:auto}.dropdown-menu-left{right:auto;left:0}.dropdown-header{display:block;padding:3px 20px;font-size:12px;line-height:1.42857143;color:#777;white-space:nowrap}.dropdown-backdrop{position:fixed;top:0;right:0;bottom:0;left:0;z-index:990}.pull-right>.dropdown-menu{right:0;left:auto}.dropup .caret,.navbar-fixed-bottom .dropdown .caret{content:"";border-top:0;border-bottom:4px solid}.dropup .dropdown-menu,.navbar-fixed-bottom .dropdown .dropdown-menu{top:auto;bottom:100%;margin-bottom:1px}@media (min-width:768px){.navbar-right .dropdown-menu{right:0;left:auto}.navbar-right .dropdown-menu-left{right:auto;left:0}}.btn-group,.btn-group-vertical{position:relative;display:inline-block;vertical-align:middle}.btn-group>.btn,.btn-group-vertical>.btn{position:relative;float:left}.btn-group>.btn:hover,.btn-group-vertical>.btn:hover,.btn-group>.btn:focus,.btn-group-vertical>.btn:focus,.btn-group>.btn:active,.btn-group-vertical>.btn:active,.btn-group>.btn.active,.btn-group-vertical>.btn.active{z-index:2}.btn-group .btn+.btn,.btn-group .btn+.btn-group,.btn-group .btn-group+.btn,.btn-group .btn-group+.btn-group{margin-left:-1px}.btn-toolbar{margin-left:-5px}.btn-toolbar .btn-group,.btn-toolbar .input-group{float:left}.btn-toolbar>.btn,.btn-toolbar>.btn-group,.btn-toolbar>.input-group{margin-left:5px}.btn-group>.btn:not(:first-child):not(:last-child):not(.dropdown-toggle){border-radius:0}.btn-group>.btn:first-child{margin-left:0}.btn-group>.btn:first-child:not(:last-child):not(.dropdown-toggle){border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn:last-child:not(:first-child),.btn-group>.dropdown-toggle:not(:first-child){border-top-left-radius:0;border-bottom-left-radius:0}.btn-group>.btn-group{float:left}.btn-group>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group>.btn-group:first-child>.btn:last-child,.btn-group>.btn-group:first-child>.dropdown-toggle{border-top-right-radius:0;border-bottom-right-radius:0}.btn-group>.btn-group:last-child>.btn:first-child{border-top-left-radius:0;border-bottom-left-radius:0}.btn-group .dropdown-toggle:active,.btn-group.open .dropdown-toggle{outline:0}.btn-group>.btn+.dropdown-toggle{padding-right:8px;padding-left:8px}.btn-group>.btn-lg+.dropdown-toggle{padding-right:12px;padding-left:12px}.btn-group.open .dropdown-toggle{-webkit-box-shadow:inset 0 3px 5px rgba(0,0,0,.125);box-shadow:inset 0 3px 5px rgba(0,0,0,.125)}.btn-group.open .dropdown-toggle.btn-link{-webkit-box-shadow:none;box-shadow:none}.btn .caret{margin-left:0}.btn-lg .caret{border-width:5px 5px 0;border-bottom-width:0}.dropup .btn-lg .caret{border-width:0 5px 5px}.btn-group-vertical>.btn,.btn-group-vertical>.btn-group,.btn-group-vertical>.btn-group>.btn{display:block;float:none;width:100%;max-width:100%}.btn-group-vertical>.btn-group>.btn{float:none}.btn-group-vertical>.btn+.btn,.btn-group-vertical>.btn+.btn-group,.btn-group-vertical>.btn-group+.btn,.btn-group-vertical>.btn-group+.btn-group{margin-top:-1px;margin-left:0}.btn-group-vertical>.btn:not(:first-child):not(:last-child){border-radius:0}.btn-group-vertical>.btn:first-child:not(:last-child){border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn:last-child:not(:first-child){border-top-left-radius:0;border-top-right-radius:0;border-bottom-left-radius:4px}.btn-group-vertical>.btn-group:not(:first-child):not(:last-child)>.btn{border-radius:0}.btn-group-vertical>.btn-group:first-child:not(:last-child)>.btn:last-child,.btn-group-vertical>.btn-group:first-child:not(:last-child)>.dropdown-toggle{border-bottom-right-radius:0;border-bottom-left-radius:0}.btn-group-vertical>.btn-group:last-child:not(:first-child)>.btn:first-child{border-top-left-radius:0;border-top-right-radius:0}.btn-group-justified{display:table;width:100%;table-layout:fixed;border-collapse:separate}.btn-group-justified>.btn,.btn-group-justified>.btn-group{display:table-cell;float:none;width:1%}.btn-group-justified>.btn-group .btn{width:100%}.btn-group-justified>.btn-group .dropdown-menu{left:auto}[data-toggle=buttons]>.btn input[type=radio],[data-toggle=buttons]>.btn-group>.btn input[type=radio],[data-toggle=buttons]>.btn input[type=checkbox],[data-toggle=buttons]>.btn-group>.btn input[type=checkbox]{position:absolute;clip:rect(0,0,0,0);pointer-events:none}.input-group{position:relative;display:table;border-collapse:separate}.input-group[class*=col-]{float:none;padding-right:0;padding-left:0}.input-group .form-control{position:relative;z-index:2;float:left;width:100%;margin-bottom:0}.input-group-lg>.form-control,.input-group-lg>.input-group-addon,.input-group-lg>.input-group-btn>.btn{height:46px;padding:10px 16px;font-size:18px;line-height:1.33;border-radius:6px}select.input-group-lg>.form-control,select.input-group-lg>.input-group-addon,select.input-group-lg>.input-group-btn>.btn{height:46px;line-height:46px}textarea.input-group-lg>.form-control,textarea.input-group-lg>.input-group-addon,textarea.input-group-lg>.input-group-btn>.btn,select[multiple].input-group-lg>.form-control,select[multiple].input-group-lg>.input-group-addon,select[multiple].input-group-lg>.input-group-btn>.btn{height:auto}.input-group-sm>.form-control,.input-group-sm>.input-group-addon,.input-group-sm>.input-group-btn>.btn{height:30px;padding:5px 10px;font-size:12px;line-height:1.5;border-radius:3px}select.input-group-sm>.form-control,select.input-group-sm>.input-group-addon,select.input-group-sm>.input-group-btn>.btn{height:30px;line-height:30px}textarea.input-group-sm>.form-control,textarea.input-group-sm>.input-group-addon,textarea.input-group-sm>.input-group-btn>.btn,select[multiple].input-group-sm>.form-control,select[multiple].input-group-sm>.input-group-addon,select[multiple].input-group-sm>.input-group-btn>.btn{height:auto}.input-group-addon,.input-group-btn,.input-group .form-control{display:table-cell}.input-group-addon:not(:first-child):not(:last-child),.input-group-btn:not(:first-child):not(:last-child),.input-group .form-control:not(:first-child):not(:last-child){border-radius:0}.input-group-addon,.input-group-btn{width:1%;white-space:nowrap;vertical-align:middle}.input-group-addon{padding:6px 12px;font-size:14px;font-weight:400;line-height:1;color:#555;text-align:center;background-color:#eee;border:1px solid #ccc;border-radius:4px}.input-group-addon.input-sm{padding:5px 10px;font-size:12px;border-radius:3px}.input-group-addon.input-lg{padding:10px 16px;font-size:18px;border-radius:6px}.input-group-addon input[type=radio],.input-group-addon input[type=checkbox]{margin-top:0}.input-group .form-control:first-child,.input-group-addon:first-child,.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group>.btn,.input-group-btn:first-child>.dropdown-toggle,.input-group-btn:last-child>.btn:not(:last-child):not(.dropdown-toggle),.input-group-btn:last-child>.btn-group:not(:last-child)>.btn{border-top-right-radius:0;border-bottom-right-radius:0}.input-group-addon:first-child{border-right:0}.input-group .form-control:last-child,.input-group-addon:last-child,.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group>.btn,.input-group-btn:last-child>.dropdown-toggle,.input-group-btn:first-child>.btn:not(:first-child),.input-group-btn:first-child>.btn-group:not(:first-child)>.btn{border-top-left-radius:0;border-bottom-left-radius:0}.input-group-addon:last-child{border-left:0}.input-group-btn{position:relative;font-size:0;white-space:nowrap}.input-group-btn>.btn{position:relative}.input-group-btn>.btn+.btn{margin-left:-1px}.input-group-btn>.btn:hover,.input-group-btn>.btn:focus,.input-group-btn>.btn:active{z-index:2}.input-group-btn:first-child>.btn,.input-group-btn:first-child>.btn-group{margin-right:-1px}.input-group-btn:last-child>.btn,.input-group-btn:last-child>.btn-group{margin-left:-1px}.nav{padding-left:0;margin-bottom:0;list-style:none}.nav>li{position:relative;display:block}.nav>li>a{position:relative;display:block;padding:10px 15px}.nav>li>a:hover,.nav>li>a:focus{text-decoration:none;background-color:#eee}.nav>li.disabled>a{color:#777}.nav>li.disabled>a:hover,.nav>li.disabled>a:focus{color:#777;text-decoration:none;cursor:not-allowed;background-color:transparent}.nav .open>a,.nav .open>a:hover,.nav .open>a:focus{background-color:#eee;border-color:#337ab7}.nav .nav-divider{height:1px;margin:9px 0;overflow:hidden;background-color:#e5e5e5}.nav>li>a>img{max-width:none}.nav-tabs{border-bottom:1px solid #ddd}.nav-tabs>li{float:left;margin-bottom:-1px}.nav-tabs>li>a{margin-right:2px;line-height:1.42857143;border:1px solid transparent;border-radius:4px 4px 0 0}.nav-tabs>li>a:hover{border-color:#eee #eee #ddd}.nav-tabs>li.active>a,.nav-tabs>li.active>a:hover,.nav-tabs>li.active>a:focus{color:#555;cursor:default;background-color:#fff;border:1px solid #ddd;border-bottom-color:transparent}.nav-tabs.nav-justified{width:100%;border-bottom:0}.nav-tabs.nav-justified>li{float:none}.nav-tabs.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-tabs.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-tabs.nav-justified>li{display:table-cell;width:1%}.nav-tabs.nav-justified>li>a{margin-bottom:0}}.nav-tabs.nav-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:hover,.nav-tabs.nav-justified>.active>a:focus{border:1px solid #ddd}@media (min-width:768px){.nav-tabs.nav-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs.nav-justified>.active>a,.nav-tabs.nav-justified>.active>a:hover,.nav-tabs.nav-justified>.active>a:focus{border-bottom-color:#fff}}.nav-pills>li{float:left}.nav-pills>li>a{border-radius:4px}.nav-pills>li+li{margin-left:2px}.nav-pills>li.active>a,.nav-pills>li.active>a:hover,.nav-pills>li.active>a:focus{color:#fff;background-color:#337ab7}.nav-stacked>li{float:none}.nav-stacked>li+li{margin-top:2px;margin-left:0}.nav-justified{width:100%}.nav-justified>li{float:none}.nav-justified>li>a{margin-bottom:5px;text-align:center}.nav-justified>.dropdown .dropdown-menu{top:auto;left:auto}@media (min-width:768px){.nav-justified>li{display:table-cell;width:1%}.nav-justified>li>a{margin-bottom:0}}.nav-tabs-justified{border-bottom:0}.nav-tabs-justified>li>a{margin-right:0;border-radius:4px}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:hover,.nav-tabs-justified>.active>a:focus{border:1px solid #ddd}@media (min-width:768px){.nav-tabs-justified>li>a{border-bottom:1px solid #ddd;border-radius:4px 4px 0 0}.nav-tabs-justified>.active>a,.nav-tabs-justified>.active>a:hover,.nav-tabs-justified>.active>a:focus{border-bottom-color:#fff}}.tab-content>.tab-pane{display:none;visibility:hidden}.tab-content>.active{display:block;visibility:visible}.nav-tabs .dropdown-menu{margin-top:-1px;border-top-left-radius:0;border-top-right-radius:0}.navbar{position:relative;min-height:50px;margin-bottom:20px;border:1px solid transparent}@media (min-width:768px){.navbar{border-radius:4px}}@media (min-width:768px){.navbar-header{float:left}}.navbar-collapse{padding-right:15px;padding-left:15px;overflow-x:visible;-webkit-overflow-scrolling:touch;border-top:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1)}.navbar-collapse.in{overflow-y:auto}@media (min-width:768px){.navbar-collapse{width:auto;border-top:0;-webkit-box-shadow:none;box-shadow:none}.navbar-collapse.collapse{display:block!important;height:auto!important;padding-bottom:0;overflow:visible!important;visibility:visible!important}.navbar-collapse.in{overflow-y:visible}.navbar-fixed-top .navbar-collapse,.navbar-static-top .navbar-collapse,.navbar-fixed-bottom .navbar-collapse{padding-right:0;padding-left:0}}.navbar-fixed-top .navbar-collapse,.navbar-fixed-bottom .navbar-collapse{max-height:340px}@media (max-device-width:480px) and (orientation:landscape){.navbar-fixed-top .navbar-collapse,.navbar-fixed-bottom .navbar-collapse{max-height:200px}}.container>.navbar-header,.container-fluid>.navbar-header,.container>.navbar-collapse,.container-fluid>.navbar-collapse{margin-right:-15px;margin-left:-15px}@media (min-width:768px){.container>.navbar-header,.container-fluid>.navbar-header,.container>.navbar-collapse,.container-fluid>.navbar-collapse{margin-right:0;margin-left:0}}.navbar-static-top{z-index:1000;border-width:0 0 1px}@media (min-width:768px){.navbar-static-top{border-radius:0}}.navbar-fixed-top,.navbar-fixed-bottom{position:fixed;right:0;left:0;z-index:1030}@media (min-width:768px){.navbar-fixed-top,.navbar-fixed-bottom{border-radius:0}}.navbar-fixed-top{top:0;border-width:0 0 1px}.navbar-fixed-bottom{bottom:0;margin-bottom:0;border-width:1px 0 0}.navbar-brand{float:left;height:50px;padding:15px 15px;font-size:18px;line-height:20px}.navbar-brand:hover,.navbar-brand:focus{text-decoration:none}.navbar-brand>img{display:block}@media (min-width:768px){.navbar>.container .navbar-brand,.navbar>.container-fluid .navbar-brand{margin-left:-15px}}.navbar-toggle{position:relative;float:right;padding:9px 10px;margin-top:8px;margin-right:15px;margin-bottom:8px;background-color:transparent;background-image:none;border:1px solid transparent;border-radius:4px}.navbar-toggle:focus{outline:0}.navbar-toggle .icon-bar{display:block;width:22px;height:2px;border-radius:1px}.navbar-toggle .icon-bar+.icon-bar{margin-top:4px}@media (min-width:768px){.navbar-toggle{display:none}}.navbar-nav{margin:7.5px -15px}.navbar-nav>li>a{padding-top:10px;padding-bottom:10px;line-height:20px}@media (max-width:767px){.navbar-nav .open .dropdown-menu{position:static;float:none;width:auto;margin-top:0;background-color:transparent;border:0;-webkit-box-shadow:none;box-shadow:none}.navbar-nav .open .dropdown-menu>li>a,.navbar-nav .open .dropdown-menu .dropdown-header{padding:5px 15px 5px 25px}.navbar-nav .open .dropdown-menu>li>a{line-height:20px}.navbar-nav .open .dropdown-menu>li>a:hover,.navbar-nav .open .dropdown-menu>li>a:focus{background-image:none}}@media (min-width:768px){.navbar-nav{float:left;margin:0}.navbar-nav>li{float:left}.navbar-nav>li>a{padding-top:15px;padding-bottom:15px}}.navbar-form{padding:10px 15px;margin-top:8px;margin-right:-15px;margin-bottom:8px;margin-left:-15px;border-top:1px solid transparent;border-bottom:1px solid transparent;-webkit-box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1);box-shadow:inset 0 1px 0 rgba(255,255,255,.1),0 1px 0 rgba(255,255,255,.1)}@media (min-width:768px){.navbar-form .form-group{display:inline-block;margin-bottom:0;vertical-align:middle}.navbar-form .form-control{display:inline-block;width:auto;vertical-align:middle}.navbar-form .form-control-static{display:inline-block}.navbar-form .input-group{display:inline-table;vertical-align:middle}.navbar-form .input-group .input-group-addon,.navbar-form .input-group .input-group-btn,.navbar-form .input-group .form-control{width:auto}.navbar-form .input-group>.form-control{width:100%}.navbar-form .control-label{margin-bottom:0;vertical-align:middle}.navbar-form .radio,.navbar-form .checkbox{display:inline-block;margin-top:0;margin-bottom:0;vertical-align:middle}.navbar-form .radio label,.navbar-form .checkbox label{padding-left:0}.navbar-form .radio input[type=radio],.navbar-form .checkbox input[type=checkbox]{position:relative;margin-left:0}.navbar-form .has-feedback .form-control-feedback{top:0}}@media (max-width:767px){.navbar-form .form-group{margin-bottom:5px}.navbar-form .form-group:last-child{margin-bottom:0}}@media (min-width:768px){.navbar-form{width:auto;padding-top:0;padding-bottom:0;margin-right:0;margin-left:0;border:0;-webkit-box-shadow:none;box-shadow:none}}.navbar-nav>li>.dropdown-menu{margin-top:0;border-top-left-radius:0;border-top-right-radius:0}.navbar-fixed-bottom .navbar-nav>li>.dropdown-menu{border-top-left-radius:4px;border-top-right-radius:4px;border-bottom-right-radius:0;border-bottom-left-radius:0}.navbar-btn{margin-top:8px;margin-bottom:8px}.navbar-btn.btn-sm{margin-top:10px;margin-bottom:10px}.navbar-btn.btn-xs{margin-top:14px;margin-bottom:14px}.navbar-text{margin-top:15px;margin-bottom:15px}@media (min-width:768px){.navbar-text{float:left;margin-right:15px;margin-left:15px}}@media (min-width:768px){.navbar-left{float:left!important}.navbar-right{float:right!important;margin-right:-15px}.navbar-right~.navbar-right{margin-right:0}}.navbar-default{background-color:#f8f8f8;border-color:#e7e7e7}.navbar-default .navbar-brand{color:#777}.navbar-default .navbar-brand:hover,.navbar-default .navbar-brand:focus{color:#5e5e5e;background-color:transparent}.navbar-default .navbar-text{color:#777}.navbar-default .navbar-nav>li>a{color:#777}.navbar-default .navbar-nav>li>a:hover,.navbar-default .navbar-nav>li>a:focus{color:#333;background-color:transparent}.navbar-default .navbar-nav>.active>a,.navbar-default .navbar-nav>.active>a:hover,.navbar-default .navbar-nav>.active>a:focus{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav>.disabled>a,.navbar-default .navbar-nav>.disabled>a:hover,.navbar-default .navbar-nav>.disabled>a:focus{color:#ccc;background-color:transparent}.navbar-default .navbar-toggle{border-color:#ddd}.navbar-default .navbar-toggle:hover,.navbar-default .navbar-toggle:focus{background-color:#ddd}.navbar-default .navbar-toggle .icon-bar{background-color:#888}.navbar-default .navbar-collapse,.navbar-default .navbar-form{border-color:#e7e7e7}.navbar-default .navbar-nav>.open>a,.navbar-default .navbar-nav>.open>a:hover,.navbar-default .navbar-nav>.open>a:focus{color:#555;background-color:#e7e7e7}@media (max-width:767px){.navbar-default .navbar-nav .open .dropdown-menu>li>a{color:#777}.navbar-default .navbar-nav .open .dropdown-menu>li>a:hover,.navbar-default .navbar-nav .open .dropdown-menu>li>a:focus{color:#333;background-color:transparent}.navbar-default .navbar-nav .open .dropdown-menu>.active>a,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:hover,.navbar-default .navbar-nav .open .dropdown-menu>.active>a:focus{color:#555;background-color:#e7e7e7}.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:hover,.navbar-default .navbar-nav .open .dropdown-menu>.disabled>a:focus{color:#ccc;background-color:transparent}}.navbar-default .navbar-link{color:#777}.navbar-default .navbar-link:hover{color:#333}.navbar-default .btn-link{color:#777}.navbar-default .btn-link:hover,.navbar-default .btn-link:focus{color:#333}.navbar-default .btn-link[disabled]:hover,fieldset[disabled] .navbar-default .btn-link:hover,.navbar-default .btn-link[disabled]:focus,fieldset[disabled] .navbar-default .btn-link:focus{color:#ccc}.navbar-inverse{background-color:#222;border-color:#080808}.navbar-inverse .navbar-brand{color:#9d9d9d}.navbar-inverse .navbar-brand:hover,.navbar-inverse .navbar-brand:focus{color:#fff;background-color:transparent}.navbar-inverse .navbar-text{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav>li>a:hover,.navbar-inverse .navbar-nav>li>a:focus{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav>.active>a,.navbar-inverse .navbar-nav>.active>a:hover,.navbar-inverse .navbar-nav>.active>a:focus{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav>.disabled>a,.navbar-inverse .navbar-nav>.disabled>a:hover,.navbar-inverse .navbar-nav>.disabled>a:focus{color:#444;background-color:transparent}.navbar-inverse .navbar-toggle{border-color:#333}.navbar-inverse .navbar-toggle:hover,.navbar-inverse .navbar-toggle:focus{background-color:#333}.navbar-inverse .navbar-toggle .icon-bar{background-color:#fff}.navbar-inverse .navbar-collapse,.navbar-inverse .navbar-form{border-color:#101010}.navbar-inverse .navbar-nav>.open>a,.navbar-inverse .navbar-nav>.open>a:hover,.navbar-inverse .navbar-nav>.open>a:focus{color:#fff;background-color:#080808}@media (max-width:767px){.navbar-inverse .navbar-nav .open .dropdown-menu>.dropdown-header{border-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu .divider{background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a{color:#9d9d9d}.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:hover,.navbar-inverse .navbar-nav .open .dropdown-menu>li>a:focus{color:#fff;background-color:transparent}.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:hover,.navbar-inverse .navbar-nav .open .dropdown-menu>.active>a:focus{color:#fff;background-color:#080808}.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:hover,.navbar-inverse .navbar-nav .open .dropdown-menu>.disabled>a:focus{color:#444;background-color:transparent}}.navbar-inverse .navbar-link{color:#9d9d9d}.navbar-inverse .navbar-link:hover{color:#fff}.navbar-inverse .btn-link{color:#9d9d9d}.navbar-inverse .btn-link:hover,.navbar-inverse .btn-link:focus{color:#fff}.navbar-inverse .btn-link[disabled]:hover,fieldset[disabled] .navbar-inverse .btn-link:hover,.navbar-inverse .btn-link[disabled]:focus,fieldset[disabled] .navbar-inverse .btn-link:focus{color:#444}.breadcrumb{padding:8px 15px;margin-bottom:20px;list-style:none;background-color:#f5f5f5;border-radius:4px}.breadcrumb>li{display:inline-block}.breadcrumb>li+li:before{padding:0 5px;color:#ccc;content:"/\00a0"}.breadcrumb>.active{color:#777}.pagination{display:inline-block;padding-left:0;margin:20px 0;border-radius:4px}.pagination>li{display:inline}.pagination>li>a,.pagination>li>span{position:relative;float:left;padding:6px 12px;margin-left:-1px;line-height:1.42857143;color:#337ab7;text-decoration:none;background-color:#fff;border:1px solid #ddd}.pagination>li:first-child>a,.pagination>li:first-child>span{margin-left:0;border-top-left-radius:4px;border-bottom-left-radius:4px}.pagination>li:last-child>a,.pagination>li:last-child>span{border-top-right-radius:4px;border-bottom-right-radius:4px}.pagination>li>a:hover,.pagination>li>span:hover,.pagination>li>a:focus,.pagination>li>span:focus{color:#23527c;background-color:#eee;border-color:#ddd}.pagination>.active>a,.pagination>.active>span,.pagination>.active>a:hover,.pagination>.active>span:hover,.pagination>.active>a:focus,.pagination>.active>span:focus{z-index:2;color:#fff;cursor:default;background-color:#337ab7;border-color:#337ab7}.pagination>.disabled>span,.pagination>.disabled>span:hover,.pagination>.disabled>span:focus,.pagination>.disabled>a,.pagination>.disabled>a:hover,.pagination>.disabled>a:focus{color:#777;cursor:not-allowed;background-color:#fff;border-color:#ddd}.pagination-lg>li>a,.pagination-lg>li>span{padding:10px 16px;font-size:18px}.pagination-lg>li:first-child>a,.pagination-lg>li:first-child>span{border-top-left-radius:6px;border-bottom-left-radius:6px}.pagination-lg>li:last-child>a,.pagination-lg>li:last-child>span{border-top-right-radius:6px;border-bottom-right-radius:6px}.pagination-sm>li>a,.pagination-sm>li>span{padding:5px 10px;font-size:12px}.pagination-sm>li:first-child>a,.pagination-sm>li:first-child>span{border-top-left-radius:3px;border-bottom-left-radius:3px}.pagination-sm>li:last-child>a,.pagination-sm>li:last-child>span{border-top-right-radius:3px;border-bottom-right-radius:3px}.pager{padding-left:0;margin:20px 0;text-align:center;list-style:none}.pager li{display:inline}.pager li>a,.pager li>span{display:inline-block;padding:5px 14px;background-color:#fff;border:1px solid #ddd;border-radius:15px}.pager li>a:hover,.pager li>a:focus{text-decoration:none;background-color:#eee}.pager .next>a,.pager .next>span{float:right}.pager .previous>a,.pager .previous>span{float:left}.pager .disabled>a,.pager .disabled>a:hover,.pager .disabled>a:focus,.pager .disabled>span{color:#777;cursor:not-allowed;background-color:#fff}.label{display:inline;padding:.2em .6em .3em;font-size:75%;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;border-radius:.25em}a.label:hover,a.label:focus{color:#fff;text-decoration:none;cursor:pointer}.label:empty{display:none}.btn .label{position:relative;top:-1px}.label-default{background-color:#777}.label-default[href]:hover,.label-default[href]:focus{background-color:#5e5e5e}.label-primary{background-color:#337ab7}.label-primary[href]:hover,.label-primary[href]:focus{background-color:#286090}.label-success{background-color:#5cb85c}.label-success[href]:hover,.label-success[href]:focus{background-color:#449d44}.label-info{background-color:#5bc0de}.label-info[href]:hover,.label-info[href]:focus{background-color:#31b0d5}.label-warning{background-color:#f0ad4e}.label-warning[href]:hover,.label-warning[href]:focus{background-color:#ec971f}.label-danger{background-color:#d9534f}.label-danger[href]:hover,.label-danger[href]:focus{background-color:#c9302c}.badge{display:inline-block;min-width:10px;padding:3px 7px;font-size:12px;font-weight:700;line-height:1;color:#fff;text-align:center;white-space:nowrap;vertical-align:baseline;background-color:#777;border-radius:10px}.badge:empty{display:none}.btn .badge{position:relative;top:-1px}.btn-xs .badge{top:0;padding:1px 5px}a.badge:hover,a.badge:focus{color:#fff;text-decoration:none;cursor:pointer}.list-group-item.active>.badge,.nav-pills>.active>a>.badge{color:#337ab7;background-color:#fff}.list-group-item>.badge{float:right}.list-group-item>.badge+.badge{margin-right:5px}.nav-pills>li>a>.badge{margin-left:3px}.jumbotron{padding:30px 15px;margin-bottom:30px;color:inherit;background-color:#eee}.jumbotron h1,.jumbotron .h1{color:inherit}.jumbotron p{margin-bottom:15px;font-size:21px;font-weight:200}.jumbotron>hr{border-top-color:#d5d5d5}.container .jumbotron,.container-fluid .jumbotron{border-radius:6px}.jumbotron .container{max-width:100%}@media screen and (min-width:768px){.jumbotron{padding:48px 0}.container .jumbotron,.container-fluid .jumbotron{padding-right:60px;padding-left:60px}.jumbotron h1,.jumbotron .h1{font-size:63px}}.thumbnail{display:block;padding:4px;margin-bottom:20px;line-height:1.42857143;background-color:#fff;border:1px solid #ddd;border-radius:4px;-webkit-transition:border .2s ease-in-out;-o-transition:border .2s ease-in-out;transition:border .2s ease-in-out}.thumbnail>img,.thumbnail a>img{margin-right:auto;margin-left:auto}a.thumbnail:hover,a.thumbnail:focus,a.thumbnail.active{border-color:#337ab7}.thumbnail .caption{padding:9px;color:#333}.alert{padding:15px;margin-bottom:20px;border:1px solid transparent;border-radius:4px}.alert h4{margin-top:0;color:inherit}.alert .alert-link{font-weight:700}.alert>p,.alert>ul{margin-bottom:0}.alert>p+p{margin-top:5px}.alert-dismissable,.alert-dismissible{padding-right:35px}.alert-dismissable .close,.alert-dismissible .close{position:relative;top:-2px;right:-21px;color:inherit}.alert-success{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.alert-success hr{border-top-color:#c9e2b3}.alert-success .alert-link{color:#2b542c}.alert-info{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.alert-info hr{border-top-color:#a6e1ec}.alert-info .alert-link{color:#245269}.alert-warning{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.alert-warning hr{border-top-color:#f7e1b5}.alert-warning .alert-link{color:#66512c}.alert-danger{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.alert-danger hr{border-top-color:#e4b9c0}.alert-danger .alert-link{color:#843534}@-webkit-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@-o-keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}@keyframes progress-bar-stripes{from{background-position:40px 0}to{background-position:0 0}}.progress{height:20px;margin-bottom:20px;overflow:hidden;background-color:#f5f5f5;border-radius:4px;-webkit-box-shadow:inset 0 1px 2px rgba(0,0,0,.1);box-shadow:inset 0 1px 2px rgba(0,0,0,.1)}.progress-bar{float:left;width:0;height:100%;font-size:12px;line-height:20px;color:#fff;text-align:center;background-color:#337ab7;-webkit-box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);box-shadow:inset 0 -1px 0 rgba(0,0,0,.15);-webkit-transition:width .6s ease;-o-transition:width .6s ease;transition:width .6s ease}.progress-striped .progress-bar,.progress-bar-striped{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);-webkit-background-size:40px 40px;background-size:40px 40px}.progress.active .progress-bar,.progress-bar.active{-webkit-animation:progress-bar-stripes 2s linear infinite;-o-animation:progress-bar-stripes 2s linear infinite;animation:progress-bar-stripes 2s linear infinite}.progress-bar-success{background-color:#5cb85c}.progress-striped .progress-bar-success{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-info{background-color:#5bc0de}.progress-striped .progress-bar-info{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-warning{background-color:#f0ad4e}.progress-striped .progress-bar-warning{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.progress-bar-danger{background-color:#d9534f}.progress-striped .progress-bar-danger{background-image:-webkit-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:-o-linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent);background-image:linear-gradient(45deg,rgba(255,255,255,.15) 25%,transparent 25%,transparent 50%,rgba(255,255,255,.15) 50%,rgba(255,255,255,.15) 75%,transparent 75%,transparent)}.media{margin-top:15px}.media:first-child{margin-top:0}.media-right,.media>.pull-right{padding-left:10px}.media-left,.media>.pull-left{padding-right:10px}.media-left,.media-right,.media-body{display:table-cell;vertical-align:top}.media-middle{vertical-align:middle}.media-bottom{vertical-align:bottom}.media-heading{margin-top:0;margin-bottom:5px}.media-list{padding-left:0;list-style:none}.list-group{padding-left:0;margin-bottom:20px}.list-group-item{position:relative;display:block;padding:10px 15px;margin-bottom:-1px;background-color:#fff;border:1px solid #ddd}.list-group-item:first-child{border-top-left-radius:4px;border-top-right-radius:4px}.list-group-item:last-child{margin-bottom:0;border-bottom-right-radius:4px;border-bottom-left-radius:4px}a.list-group-item{color:#555}a.list-group-item .list-group-item-heading{color:#333}a.list-group-item:hover,a.list-group-item:focus{color:#555;text-decoration:none;background-color:#f5f5f5}.list-group-item.disabled,.list-group-item.disabled:hover,.list-group-item.disabled:focus{color:#777;cursor:not-allowed;background-color:#eee}.list-group-item.disabled .list-group-item-heading,.list-group-item.disabled:hover .list-group-item-heading,.list-group-item.disabled:focus .list-group-item-heading{color:inherit}.list-group-item.disabled .list-group-item-text,.list-group-item.disabled:hover .list-group-item-text,.list-group-item.disabled:focus .list-group-item-text{color:#777}.list-group-item.active,.list-group-item.active:hover,.list-group-item.active:focus{z-index:2;color:#fff;background-color:#337ab7;border-color:#337ab7}.list-group-item.active .list-group-item-heading,.list-group-item.active:hover .list-group-item-heading,.list-group-item.active:focus .list-group-item-heading,.list-group-item.active .list-group-item-heading>small,.list-group-item.active:hover .list-group-item-heading>small,.list-group-item.active:focus .list-group-item-heading>small,.list-group-item.active .list-group-item-heading>.small,.list-group-item.active:hover .list-group-item-heading>.small,.list-group-item.active:focus .list-group-item-heading>.small{color:inherit}.list-group-item.active .list-group-item-text,.list-group-item.active:hover .list-group-item-text,.list-group-item.active:focus .list-group-item-text{color:#c7ddef}.list-group-item-success{color:#3c763d;background-color:#dff0d8}a.list-group-item-success{color:#3c763d}a.list-group-item-success .list-group-item-heading{color:inherit}a.list-group-item-success:hover,a.list-group-item-success:focus{color:#3c763d;background-color:#d0e9c6}a.list-group-item-success.active,a.list-group-item-success.active:hover,a.list-group-item-success.active:focus{color:#fff;background-color:#3c763d;border-color:#3c763d}.list-group-item-info{color:#31708f;background-color:#d9edf7}a.list-group-item-info{color:#31708f}a.list-group-item-info .list-group-item-heading{color:inherit}a.list-group-item-info:hover,a.list-group-item-info:focus{color:#31708f;background-color:#c4e3f3}a.list-group-item-info.active,a.list-group-item-info.active:hover,a.list-group-item-info.active:focus{color:#fff;background-color:#31708f;border-color:#31708f}.list-group-item-warning{color:#8a6d3b;background-color:#fcf8e3}a.list-group-item-warning{color:#8a6d3b}a.list-group-item-warning .list-group-item-heading{color:inherit}a.list-group-item-warning:hover,a.list-group-item-warning:focus{color:#8a6d3b;background-color:#faf2cc}a.list-group-item-warning.active,a.list-group-item-warning.active:hover,a.list-group-item-warning.active:focus{color:#fff;background-color:#8a6d3b;border-color:#8a6d3b}.list-group-item-danger{color:#a94442;background-color:#f2dede}a.list-group-item-danger{color:#a94442}a.list-group-item-danger .list-group-item-heading{color:inherit}a.list-group-item-danger:hover,a.list-group-item-danger:focus{color:#a94442;background-color:#ebcccc}a.list-group-item-danger.active,a.list-group-item-danger.active:hover,a.list-group-item-danger.active:focus{color:#fff;background-color:#a94442;border-color:#a94442}.list-group-item-heading{margin-top:0;margin-bottom:5px}.list-group-item-text{margin-bottom:0;line-height:1.3}.panel{margin-bottom:20px;background-color:#fff;border:1px solid transparent;border-radius:4px;-webkit-box-shadow:0 1px 1px rgba(0,0,0,.05);box-shadow:0 1px 1px rgba(0,0,0,.05)}.panel-body{padding:15px}.panel-heading{padding:10px 15px;border-bottom:1px solid transparent;border-top-left-radius:3px;border-top-right-radius:3px}.panel-heading>.dropdown .dropdown-toggle{color:inherit}.panel-title{margin-top:0;margin-bottom:0;font-size:16px;color:inherit}.panel-title>a{color:inherit}.panel-footer{padding:10px 15px;background-color:#f5f5f5;border-top:1px solid #ddd;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.list-group,.panel>.panel-collapse>.list-group{margin-bottom:0}.panel>.list-group .list-group-item,.panel>.panel-collapse>.list-group .list-group-item{border-width:1px 0;border-radius:0}.panel>.list-group:first-child .list-group-item:first-child,.panel>.panel-collapse>.list-group:first-child .list-group-item:first-child{border-top:0;border-top-left-radius:3px;border-top-right-radius:3px}.panel>.list-group:last-child .list-group-item:last-child,.panel>.panel-collapse>.list-group:last-child .list-group-item:last-child{border-bottom:0;border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel-heading+.list-group .list-group-item:first-child{border-top-width:0}.list-group+.panel-footer{border-top-width:0}.panel>.table,.panel>.table-responsive>.table,.panel>.panel-collapse>.table{margin-bottom:0}.panel>.table caption,.panel>.table-responsive>.table caption,.panel>.panel-collapse>.table caption{padding-right:15px;padding-left:15px}.panel>.table:first-child,.panel>.table-responsive:first-child>.table:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table:first-child>thead:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child{border-top-left-radius:3px;border-top-right-radius:3px}.panel>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:first-child,.panel>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:first-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:first-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:first-child{border-top-left-radius:3px}.panel>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child td:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child td:last-child,.panel>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>thead:first-child>tr:first-child th:last-child,.panel>.table:first-child>tbody:first-child>tr:first-child th:last-child,.panel>.table-responsive:first-child>.table:first-child>tbody:first-child>tr:first-child th:last-child{border-top-right-radius:3px}.panel>.table:last-child,.panel>.table-responsive:last-child>.table:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table:last-child>tbody:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child{border-bottom-right-radius:3px;border-bottom-left-radius:3px}.panel>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:first-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:first-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:first-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:first-child{border-bottom-left-radius:3px}.panel>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child td:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child td:last-child,.panel>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tbody:last-child>tr:last-child th:last-child,.panel>.table:last-child>tfoot:last-child>tr:last-child th:last-child,.panel>.table-responsive:last-child>.table:last-child>tfoot:last-child>tr:last-child th:last-child{border-bottom-right-radius:3px}.panel>.panel-body+.table,.panel>.panel-body+.table-responsive,.panel>.table+.panel-body,.panel>.table-responsive+.panel-body{border-top:1px solid #ddd}.panel>.table>tbody:first-child>tr:first-child th,.panel>.table>tbody:first-child>tr:first-child td{border-top:0}.panel>.table-bordered,.panel>.table-responsive>.table-bordered{border:0}.panel>.table-bordered>thead>tr>th:first-child,.panel>.table-responsive>.table-bordered>thead>tr>th:first-child,.panel>.table-bordered>tbody>tr>th:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:first-child,.panel>.table-bordered>tfoot>tr>th:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:first-child,.panel>.table-bordered>thead>tr>td:first-child,.panel>.table-responsive>.table-bordered>thead>tr>td:first-child,.panel>.table-bordered>tbody>tr>td:first-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:first-child,.panel>.table-bordered>tfoot>tr>td:first-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:first-child{border-left:0}.panel>.table-bordered>thead>tr>th:last-child,.panel>.table-responsive>.table-bordered>thead>tr>th:last-child,.panel>.table-bordered>tbody>tr>th:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>th:last-child,.panel>.table-bordered>tfoot>tr>th:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>th:last-child,.panel>.table-bordered>thead>tr>td:last-child,.panel>.table-responsive>.table-bordered>thead>tr>td:last-child,.panel>.table-bordered>tbody>tr>td:last-child,.panel>.table-responsive>.table-bordered>tbody>tr>td:last-child,.panel>.table-bordered>tfoot>tr>td:last-child,.panel>.table-responsive>.table-bordered>tfoot>tr>td:last-child{border-right:0}.panel>.table-bordered>thead>tr:first-child>td,.panel>.table-responsive>.table-bordered>thead>tr:first-child>td,.panel>.table-bordered>tbody>tr:first-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>td,.panel>.table-bordered>thead>tr:first-child>th,.panel>.table-responsive>.table-bordered>thead>tr:first-child>th,.panel>.table-bordered>tbody>tr:first-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:first-child>th{border-bottom:0}.panel>.table-bordered>tbody>tr:last-child>td,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>td,.panel>.table-bordered>tfoot>tr:last-child>td,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>td,.panel>.table-bordered>tbody>tr:last-child>th,.panel>.table-responsive>.table-bordered>tbody>tr:last-child>th,.panel>.table-bordered>tfoot>tr:last-child>th,.panel>.table-responsive>.table-bordered>tfoot>tr:last-child>th{border-bottom:0}.panel>.table-responsive{margin-bottom:0;border:0}.panel-group{margin-bottom:20px}.panel-group .panel{margin-bottom:0;border-radius:4px}.panel-group .panel+.panel{margin-top:5px}.panel-group .panel-heading{border-bottom:0}.panel-group .panel-heading+.panel-collapse>.panel-body,.panel-group .panel-heading+.panel-collapse>.list-group{border-top:1px solid #ddd}.panel-group .panel-footer{border-top:0}.panel-group .panel-footer+.panel-collapse .panel-body{border-bottom:1px solid #ddd}.panel-default{border-color:#ddd}.panel-default>.panel-heading{color:#333;background-color:#f5f5f5;border-color:#ddd}.panel-default>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ddd}.panel-default>.panel-heading .badge{color:#f5f5f5;background-color:#333}.panel-default>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ddd}.panel-primary{border-color:#337ab7}.panel-primary>.panel-heading{color:#fff;background-color:#337ab7;border-color:#337ab7}.panel-primary>.panel-heading+.panel-collapse>.panel-body{border-top-color:#337ab7}.panel-primary>.panel-heading .badge{color:#337ab7;background-color:#fff}.panel-primary>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#337ab7}.panel-success{border-color:#d6e9c6}.panel-success>.panel-heading{color:#3c763d;background-color:#dff0d8;border-color:#d6e9c6}.panel-success>.panel-heading+.panel-collapse>.panel-body{border-top-color:#d6e9c6}.panel-success>.panel-heading .badge{color:#dff0d8;background-color:#3c763d}.panel-success>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#d6e9c6}.panel-info{border-color:#bce8f1}.panel-info>.panel-heading{color:#31708f;background-color:#d9edf7;border-color:#bce8f1}.panel-info>.panel-heading+.panel-collapse>.panel-body{border-top-color:#bce8f1}.panel-info>.panel-heading .badge{color:#d9edf7;background-color:#31708f}.panel-info>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#bce8f1}.panel-warning{border-color:#faebcc}.panel-warning>.panel-heading{color:#8a6d3b;background-color:#fcf8e3;border-color:#faebcc}.panel-warning>.panel-heading+.panel-collapse>.panel-body{border-top-color:#faebcc}.panel-warning>.panel-heading .badge{color:#fcf8e3;background-color:#8a6d3b}.panel-warning>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#faebcc}.panel-danger{border-color:#ebccd1}.panel-danger>.panel-heading{color:#a94442;background-color:#f2dede;border-color:#ebccd1}.panel-danger>.panel-heading+.panel-collapse>.panel-body{border-top-color:#ebccd1}.panel-danger>.panel-heading .badge{color:#f2dede;background-color:#a94442}.panel-danger>.panel-footer+.panel-collapse>.panel-body{border-bottom-color:#ebccd1}.embed-responsive{position:relative;display:block;height:0;padding:0;overflow:hidden}.embed-responsive .embed-responsive-item,.embed-responsive iframe,.embed-responsive embed,.embed-responsive object,.embed-responsive video{position:absolute;top:0;bottom:0;left:0;width:100%;height:100%;border:0}.embed-responsive.embed-responsive-16by9{padding-bottom:56.25%}.embed-responsive.embed-responsive-4by3{padding-bottom:75%}.well{min-height:20px;padding:19px;margin-bottom:20px;background-color:#f5f5f5;border:1px solid #e3e3e3;border-radius:4px;-webkit-box-shadow:inset 0 1px 1px rgba(0,0,0,.05);box-shadow:inset 0 1px 1px rgba(0,0,0,.05)}.well blockquote{border-color:#ddd;border-color:rgba(0,0,0,.15)}.well-lg{padding:24px;border-radius:6px}.well-sm{padding:9px;border-radius:3px}.close{float:right;font-size:21px;font-weight:700;line-height:1;color:#000;text-shadow:0 1px 0 #fff;filter:alpha(opacity=20);opacity:.2}.close:hover,.close:focus{color:#000;text-decoration:none;cursor:pointer;filter:alpha(opacity=50);opacity:.5}button.close{-webkit-appearance:none;padding:0;cursor:pointer;background:0 0;border:0}.modal-open{overflow:hidden}.modal{position:fixed;top:0;right:0;bottom:0;left:0;z-index:1040;display:none;overflow:hidden;-webkit-overflow-scrolling:touch;outline:0}.modal.fade .modal-dialog{-webkit-transition:-webkit-transform .3s ease-out;-o-transition:-o-transform .3s ease-out;transition:transform .3s ease-out;-webkit-transform:translate(0,-25%);-ms-transform:translate(0,-25%);-o-transform:translate(0,-25%);transform:translate(0,-25%)}.modal.in .modal-dialog{-webkit-transform:translate(0,0);-ms-transform:translate(0,0);-o-transform:translate(0,0);transform:translate(0,0)}.modal-open .modal{overflow-x:hidden;overflow-y:auto}.modal-dialog{position:relative;width:auto;margin:10px}.modal-content{position:relative;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #999;border:1px solid rgba(0,0,0,.2);border-radius:6px;outline:0;-webkit-box-shadow:0 3px 9px rgba(0,0,0,.5);box-shadow:0 3px 9px rgba(0,0,0,.5)}.modal-backdrop{position:absolute;top:0;right:0;left:0;background-color:#000}.modal-backdrop.fade{filter:alpha(opacity=0);opacity:0}.modal-backdrop.in{filter:alpha(opacity=50);opacity:.5}.modal-header{min-height:16.43px;padding:15px;border-bottom:1px solid #e5e5e5}.modal-header .close{margin-top:-2px}.modal-title{margin:0;line-height:1.42857143}.modal-body{position:relative;padding:15px}.modal-footer{padding:15px;text-align:right;border-top:1px solid #e5e5e5}.modal-footer .btn+.btn{margin-bottom:0;margin-left:5px}.modal-footer .btn-group .btn+.btn{margin-left:-1px}.modal-footer .btn-block+.btn-block{margin-left:0}.modal-scrollbar-measure{position:absolute;top:-9999px;width:50px;height:50px;overflow:scroll}@media (min-width:768px){.modal-dialog{width:600px;margin:30px auto}.modal-content{-webkit-box-shadow:0 5px 15px rgba(0,0,0,.5);box-shadow:0 5px 15px rgba(0,0,0,.5)}.modal-sm{width:300px}}@media (min-width:992px){.modal-lg{width:900px}}.tooltip{position:absolute;z-index:1070;display:block;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:12px;font-weight:400;line-height:1.4;visibility:visible;filter:alpha(opacity=0);opacity:0}.tooltip.in{filter:alpha(opacity=90);opacity:.9}.tooltip.top{padding:5px 0;margin-top:-3px}.tooltip.right{padding:0 5px;margin-left:3px}.tooltip.bottom{padding:5px 0;margin-top:3px}.tooltip.left{padding:0 5px;margin-left:-3px}.tooltip-inner{max-width:200px;padding:3px 8px;color:#fff;text-align:center;text-decoration:none;background-color:#000;border-radius:4px}.tooltip-arrow{position:absolute;width:0;height:0;border-color:transparent;border-style:solid}.tooltip.top .tooltip-arrow{bottom:0;left:50%;margin-left:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-left .tooltip-arrow{right:5px;bottom:0;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.top-right .tooltip-arrow{bottom:0;left:5px;margin-bottom:-5px;border-width:5px 5px 0;border-top-color:#000}.tooltip.right .tooltip-arrow{top:50%;left:0;margin-top:-5px;border-width:5px 5px 5px 0;border-right-color:#000}.tooltip.left .tooltip-arrow{top:50%;right:0;margin-top:-5px;border-width:5px 0 5px 5px;border-left-color:#000}.tooltip.bottom .tooltip-arrow{top:0;left:50%;margin-left:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-left .tooltip-arrow{top:0;right:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.tooltip.bottom-right .tooltip-arrow{top:0;left:5px;margin-top:-5px;border-width:0 5px 5px;border-bottom-color:#000}.popover{position:absolute;top:0;left:0;z-index:1060;display:none;max-width:276px;padding:1px;font-family:"Helvetica Neue",Helvetica,Arial,sans-serif;font-size:14px;font-weight:400;line-height:1.42857143;text-align:left;white-space:normal;background-color:#fff;-webkit-background-clip:padding-box;background-clip:padding-box;border:1px solid #ccc;border:1px solid rgba(0,0,0,.2);border-radius:6px;-webkit-box-shadow:0 5px 10px rgba(0,0,0,.2);box-shadow:0 5px 10px rgba(0,0,0,.2)}.popover.top{margin-top:-10px}.popover.right{margin-left:10px}.popover.bottom{margin-top:10px}.popover.left{margin-left:-10px}.popover-title{padding:8px 14px;margin:0;font-size:14px;background-color:#f7f7f7;border-bottom:1px solid #ebebeb;border-radius:5px 5px 0 0}.popover-content{padding:9px 14px}.popover>.arrow,.popover>.arrow:after{position:absolute;display:block;width:0;height:0;border-color:transparent;border-style:solid}.popover>.arrow{border-width:11px}.popover>.arrow:after{content:"";border-width:10px}.popover.top>.arrow{bottom:-11px;left:50%;margin-left:-11px;border-top-color:#999;border-top-color:rgba(0,0,0,.25);border-bottom-width:0}.popover.top>.arrow:after{bottom:1px;margin-left:-10px;content:" ";border-top-color:#fff;border-bottom-width:0}.popover.right>.arrow{top:50%;left:-11px;margin-top:-11px;border-right-color:#999;border-right-color:rgba(0,0,0,.25);border-left-width:0}.popover.right>.arrow:after{bottom:-10px;left:1px;content:" ";border-right-color:#fff;border-left-width:0}.popover.bottom>.arrow{top:-11px;left:50%;margin-left:-11px;border-top-width:0;border-bottom-color:#999;border-bottom-color:rgba(0,0,0,.25)}.popover.bottom>.arrow:after{top:1px;margin-left:-10px;content:" ";border-top-width:0;border-bottom-color:#fff}.popover.left>.arrow{top:50%;right:-11px;margin-top:-11px;border-right-width:0;border-left-color:#999;border-left-color:rgba(0,0,0,.25)}.popover.left>.arrow:after{right:1px;bottom:-10px;content:" ";border-right-width:0;border-left-color:#fff}.carousel{position:relative}.carousel-inner{position:relative;width:100%;overflow:hidden}.carousel-inner>.item{position:relative;display:none;-webkit-transition:.6s ease-in-out left;-o-transition:.6s ease-in-out left;transition:.6s ease-in-out left}.carousel-inner>.item>img,.carousel-inner>.item>a>img{line-height:1}@media all and (transform-3d),(-webkit-transform-3d){.carousel-inner>.item{-webkit-transition:-webkit-transform .6s ease-in-out;-o-transition:-o-transform .6s ease-in-out;transition:transform .6s ease-in-out;-webkit-backface-visibility:hidden;backface-visibility:hidden;-webkit-perspective:1000;perspective:1000}.carousel-inner>.item.next,.carousel-inner>.item.active.right{left:0;-webkit-transform:translate3d(100%,0,0);transform:translate3d(100%,0,0)}.carousel-inner>.item.prev,.carousel-inner>.item.active.left{left:0;-webkit-transform:translate3d(-100%,0,0);transform:translate3d(-100%,0,0)}.carousel-inner>.item.next.left,.carousel-inner>.item.prev.right,.carousel-inner>.item.active{left:0;-webkit-transform:translate3d(0,0,0);transform:translate3d(0,0,0)}}.carousel-inner>.active,.carousel-inner>.next,.carousel-inner>.prev{display:block}.carousel-inner>.active{left:0}.carousel-inner>.next,.carousel-inner>.prev{position:absolute;top:0;width:100%}.carousel-inner>.next{left:100%}.carousel-inner>.prev{left:-100%}.carousel-inner>.next.left,.carousel-inner>.prev.right{left:0}.carousel-inner>.active.left{left:-100%}.carousel-inner>.active.right{left:100%}.carousel-control{position:absolute;top:0;bottom:0;left:0;width:15%;font-size:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6);filter:alpha(opacity=50);opacity:.5}.carousel-control.left{background-image:-webkit-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.5)),to(rgba(0,0,0,.0001)));background-image:linear-gradient(to right,rgba(0,0,0,.5) 0,rgba(0,0,0,.0001) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#80000000', endColorstr='#00000000', GradientType=1);background-repeat:repeat-x}.carousel-control.right{right:0;left:auto;background-image:-webkit-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-o-linear-gradient(left,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);background-image:-webkit-gradient(linear,left top,right top,from(rgba(0,0,0,.0001)),to(rgba(0,0,0,.5)));background-image:linear-gradient(to right,rgba(0,0,0,.0001) 0,rgba(0,0,0,.5) 100%);filter:progid:DXImageTransform.Microsoft.gradient(startColorstr='#00000000', endColorstr='#80000000', GradientType=1);background-repeat:repeat-x}.carousel-control:hover,.carousel-control:focus{color:#fff;text-decoration:none;filter:alpha(opacity=90);outline:0;opacity:.9}.carousel-control .icon-prev,.carousel-control .icon-next,.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right{position:absolute;top:50%;z-index:5;display:inline-block}.carousel-control .icon-prev,.carousel-control .glyphicon-chevron-left{left:50%;margin-left:-10px}.carousel-control .icon-next,.carousel-control .glyphicon-chevron-right{right:50%;margin-right:-10px}.carousel-control .icon-prev,.carousel-control .icon-next{width:20px;height:20px;margin-top:-10px;font-family:serif}.carousel-control .icon-prev:before{content:'\2039'}.carousel-control .icon-next:before{content:'\203a'}.carousel-indicators{position:absolute;bottom:10px;left:50%;z-index:15;width:60%;padding-left:0;margin-left:-30%;text-align:center;list-style:none}.carousel-indicators li{display:inline-block;width:10px;height:10px;margin:1px;text-indent:-999px;cursor:pointer;background-color:#000 \9;background-color:rgba(0,0,0,0);border:1px solid #fff;border-radius:10px}.carousel-indicators .active{width:12px;height:12px;margin:0;background-color:#fff}.carousel-caption{position:absolute;right:15%;bottom:20px;left:15%;z-index:10;padding-top:20px;padding-bottom:20px;color:#fff;text-align:center;text-shadow:0 1px 2px rgba(0,0,0,.6)}.carousel-caption .btn{text-shadow:none}@media screen and (min-width:768px){.carousel-control .glyphicon-chevron-left,.carousel-control .glyphicon-chevron-right,.carousel-control .icon-prev,.carousel-control .icon-next{width:30px;height:30px;margin-top:-15px;font-size:30px}.carousel-control .glyphicon-chevron-left,.carousel-control .icon-prev{margin-left:-15px}.carousel-control .glyphicon-chevron-right,.carousel-control .icon-next{margin-right:-15px}.carousel-caption{right:20%;left:20%;padding-bottom:30px}.carousel-indicators{bottom:20px}}.clearfix:before,.clearfix:after,.dl-horizontal dd:before,.dl-horizontal dd:after,.container:before,.container:after,.container-fluid:before,.container-fluid:after,.row:before,.row:after,.form-horizontal .form-group:before,.form-horizontal .form-group:after,.btn-toolbar:before,.btn-toolbar:after,.btn-group-vertical>.btn-group:before,.btn-group-vertical>.btn-group:after,.nav:before,.nav:after,.navbar:before,.navbar:after,.navbar-header:before,.navbar-header:after,.navbar-collapse:before,.navbar-collapse:after,.pager:before,.pager:after,.panel-body:before,.panel-body:after,.modal-footer:before,.modal-footer:after{display:table;content:" "}.clearfix:after,.dl-horizontal dd:after,.container:after,.container-fluid:after,.row:after,.form-horizontal .form-group:after,.btn-toolbar:after,.btn-group-vertical>.btn-group:after,.nav:after,.navbar:after,.navbar-header:after,.navbar-collapse:after,.pager:after,.panel-body:after,.modal-footer:after{clear:both}.center-block{display:block;margin-right:auto;margin-left:auto}.pull-right{float:right!important}.pull-left{float:left!important}.hide{display:none!important}.show{display:block!important}.invisible{visibility:hidden}.text-hide{font:0/0 a;color:transparent;text-shadow:none;background-color:transparent;border:0}.hidden{display:none!important;visibility:hidden!important}.affix{position:fixed}@-ms-viewport{width:device-width}.visible-xs,.visible-sm,.visible-md,.visible-lg{display:none!important}.visible-xs-block,.visible-xs-inline,.visible-xs-inline-block,.visible-sm-block,.visible-sm-inline,.visible-sm-inline-block,.visible-md-block,.visible-md-inline,.visible-md-inline-block,.visible-lg-block,.visible-lg-inline,.visible-lg-inline-block{display:none!important}@media (max-width:767px){.visible-xs{display:block!important}table.visible-xs{display:table}tr.visible-xs{display:table-row!important}th.visible-xs,td.visible-xs{display:table-cell!important}}@media (max-width:767px){.visible-xs-block{display:block!important}}@media (max-width:767px){.visible-xs-inline{display:inline!important}}@media (max-width:767px){.visible-xs-inline-block{display:inline-block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm{display:block!important}table.visible-sm{display:table}tr.visible-sm{display:table-row!important}th.visible-sm,td.visible-sm{display:table-cell!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-block{display:block!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline{display:inline!important}}@media (min-width:768px) and (max-width:991px){.visible-sm-inline-block{display:inline-block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md{display:block!important}table.visible-md{display:table}tr.visible-md{display:table-row!important}th.visible-md,td.visible-md{display:table-cell!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-block{display:block!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline{display:inline!important}}@media (min-width:992px) and (max-width:1199px){.visible-md-inline-block{display:inline-block!important}}@media (min-width:1200px){.visible-lg{display:block!important}table.visible-lg{display:table}tr.visible-lg{display:table-row!important}th.visible-lg,td.visible-lg{display:table-cell!important}}@media (min-width:1200px){.visible-lg-block{display:block!important}}@media (min-width:1200px){.visible-lg-inline{display:inline!important}}@media (min-width:1200px){.visible-lg-inline-block{display:inline-block!important}}@media (max-width:767px){.hidden-xs{display:none!important}}@media (min-width:768px) and (max-width:991px){.hidden-sm{display:none!important}}@media (min-width:992px) and (max-width:1199px){.hidden-md{display:none!important}}@media (min-width:1200px){.hidden-lg{display:none!important}}.visible-print{display:none!important}@media print{.visible-print{display:block!important}table.visible-print{display:table}tr.visible-print{display:table-row!important}th.visible-print,td.visible-print{display:table-cell!important}}.visible-print-block{display:none!important}@media print{.visible-print-block{display:block!important}}.visible-print-inline{display:none!important}@media print{.visible-print-inline{display:inline!important}}.visible-print-inline-block{display:none!important}@media print{.visible-print-inline-block{display:inline-block!important}}@media print{.hidden-print{display:none!important}}</style>
    <style>/* bootstrap custom style stuff - taken from demo template. */
/*
 * Base structure
 */

/* Move down content because we have a fixed navbar that is 50px tall */
body {
  padding-top: 50px;
}


/*
 * Global add-ons
 */

.sub-header {
  padding-bottom: 10px;
  border-bottom: 1px solid #eee;
}

/*
 * Top navigation
 * Hide default border to remove 1px line.
 */
.navbar-fixed-top {
  border: 0;
}

/*
 * Sidebar
 */

/* Hide for mobile, show later */
.sidebar {
  display: none;
}
@media (min-width: 768px) {
  .sidebar {
    position: fixed;
    top: 51px;
    bottom: 0;
    left: 0;
    z-index: 1000;
    display: block;
    padding: 20px;
    overflow-x: hidden;
    overflow-y: auto; /* Scrollable contents if viewport is shorter than content. */
    background-color: #f5f5f5;
    border-right: 1px solid #eee;
  }
}

/* Sidebar navigation */
.nav-sidebar {
  margin-right: -21px; /* 20px padding + 1px border */
  margin-bottom: 20px;
  margin-left: -20px;
}
.nav-sidebar > li > a {
  padding-right: 20px;
  padding-left: 20px;
}
.nav-sidebar > .active > a,
.nav-sidebar > .active > a:hover,
.nav-sidebar > .active > a:focus {
  color: #fff;
  background-color: #428bca;
}


/*
 * Main content
 */

.main {
  padding: 20px;
}
@media (min-width: 768px) {
  .main {
    padding-right: 40px;
    padding-left: 40px;
  }
}
.main .page-header {
  margin-top: 0;
}


/*
 * Placeholder dashboard ideas
 */

.placeholders {
  margin-bottom: 30px;
  text-align: center;
}
.placeholders h4 {
  margin-bottom: 0;
}
.placeholder {
  margin-bottom: 20px;
}
.placeholder img {
  display: inline-block;
  border-radius: 50%;
}

.text-success-custom{
    color: rgba(6, 108, 8, 1);
}
.text-danger-custom{
    font-weight: bold;
    color: rgba(213, 24, 20, 1);
}
.panel-danger-custom .panel-title{
    font-weight: bold;
    color: rgba(213, 24, 20, 1);
}t status

.panel-success-custom .panel-title{
    color: rgba(6, 108, 8, 1);
}t status
</style>

    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
      <script src="https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js"></script>
      <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->

  </head>
  <body>

    <nav class="navbar navbar-inverse navbar-fixed-top" role="navigation">
      <div class="container-fluid">
        <div class="navbar-header">

          <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
            <span class="sr-only">Toggle navigation</span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
            <span class="icon-bar"></span>
          </button>
          <a class="navbar-brand" href="#">Tool Test Results (powered by Planemo)</a>
        </div>
        <div id="navbar" class="navbar-collapse collapse">
          <ul class="nav navbar-nav navbar-right">
            <li><a href="https://galaxyproject.org">Galaxy</a></li>
            <li><a href="https://planemo.readthedocs.org">Planemo</a></li>
          </ul>
          <div class="navbar-form navbar-right">
          </div>
        </div>
      </div>
    </nav>

    <div class="container-fluid">
      <div class="row">
        <div class="col-sm-3 col-md-2 sidebar">
          <ul class="nav nav-sidebar">
            <li><a href="#overview" class="text-success"><strong>Overview</strong></a></li>
          </ul>
          <ul class="nav nav-sidebar" id="nav-sidebar-tests">
          </ul>
        </div>
        <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
          <!-- <h1 class="page-header">Tests</h1> -->
          <h2 id="overview">Overview</h2>
          <div id="overview-content"></div>
          <div class="progress">
          </div>
          <h2 id="tests">Tests</h2>
          <p>The remainder of this contains a description for each test executed to run these jobs.</p>
        </div>
      </div>
    </div>

    <script>/*! jQuery v2.1.1 | (c) 2005, 2014 jQuery Foundation, Inc. | jquery.org/license */
!function(a,b){"object"==typeof module&&"object"==typeof module.exports?module.exports=a.document?b(a,!0):function(a){if(!a.document)throw new Error("jQuery requires a window with a document");return b(a)}:b(a)}("undefined"!=typeof window?window:this,function(a,b){var c=[],d=c.slice,e=c.concat,f=c.push,g=c.indexOf,h={},i=h.toString,j=h.hasOwnProperty,k={},l=a.document,m="2.1.1",n=function(a,b){return new n.fn.init(a,b)},o=/^[\s\uFEFF\xA0]+|[\s\uFEFF\xA0]+$/g,p=/^-ms-/,q=/-([\da-z])/gi,r=function(a,b){return b.toUpperCase()};n.fn=n.prototype={jquery:m,constructor:n,selector:"",length:0,toArray:function(){return d.call(this)},get:function(a){return null!=a?0>a?this[a+this.length]:this[a]:d.call(this)},pushStack:function(a){var b=n.merge(this.constructor(),a);return b.prevObject=this,b.context=this.context,b},each:function(a,b){return n.each(this,a,b)},map:function(a){return this.pushStack(n.map(this,function(b,c){return a.call(b,c,b)}))},slice:function(){return this.pushStack(d.apply(this,arguments))},first:function(){return this.eq(0)},last:function(){return this.eq(-1)},eq:function(a){var b=this.length,c=+a+(0>a?b:0);return this.pushStack(c>=0&&b>c?[this[c]]:[])},end:function(){return this.prevObject||this.constructor(null)},push:f,sort:c.sort,splice:c.splice},n.extend=n.fn.extend=function(){var a,b,c,d,e,f,g=arguments[0]||{},h=1,i=arguments.length,j=!1;for("boolean"==typeof g&&(j=g,g=arguments[h]||{},h++),"object"==typeof g||n.isFunction(g)||(g={}),h===i&&(g=this,h--);i>h;h++)if(null!=(a=arguments[h]))for(b in a)c=g[b],d=a[b],g!==d&&(j&&d&&(n.isPlainObject(d)||(e=n.isArray(d)))?(e?(e=!1,f=c&&n.isArray(c)?c:[]):f=c&&n.isPlainObject(c)?c:{},g[b]=n.extend(j,f,d)):void 0!==d&&(g[b]=d));return g},n.extend({expando:"jQuery"+(m+Math.random()).replace(/\D/g,""),isReady:!0,error:function(a){throw new Error(a)},noop:function(){},isFunction:function(a){return"function"===n.type(a)},isArray:Array.isArray,isWindow:function(a){return null!=a&&a===a.window},isNumeric:function(a){return!n.isArray(a)&&a-parseFloat(a)>=0},isPlainObject:function(a){return"object"!==n.type(a)||a.nodeType||n.isWindow(a)?!1:a.constructor&&!j.call(a.constructor.prototype,"isPrototypeOf")?!1:!0},isEmptyObject:function(a){var b;for(b in a)return!1;return!0},type:function(a){return null==a?a+"":"object"==typeof a||"function"==typeof a?h[i.call(a)]||"object":typeof a},globalEval:function(a){var b,c=eval;a=n.trim(a),a&&(1===a.indexOf("use strict")?(b=l.createElement("script"),b.text=a,l.head.appendChild(b).parentNode.removeChild(b)):c(a))},camelCase:function(a){return a.replace(p,"ms-").replace(q,r)},nodeName:function(a,b){return a.nodeName&&a.nodeName.toLowerCase()===b.toLowerCase()},each:function(a,b,c){var d,e=0,f=a.length,g=s(a);if(c){if(g){for(;f>e;e++)if(d=b.apply(a[e],c),d===!1)break}else for(e in a)if(d=b.apply(a[e],c),d===!1)break}else if(g){for(;f>e;e++)if(d=b.call(a[e],e,a[e]),d===!1)break}else for(e in a)if(d=b.call(a[e],e,a[e]),d===!1)break;return a},trim:function(a){return null==a?"":(a+"").replace(o,"")},makeArray:function(a,b){var c=b||[];return null!=a&&(s(Object(a))?n.merge(c,"string"==typeof a?[a]:a):f.call(c,a)),c},inArray:function(a,b,c){return null==b?-1:g.call(b,a,c)},merge:function(a,b){for(var c=+b.length,d=0,e=a.length;c>d;d++)a[e++]=b[d];return a.length=e,a},grep:function(a,b,c){for(var d,e=[],f=0,g=a.length,h=!c;g>f;f++)d=!b(a[f],f),d!==h&&e.push(a[f]);return e},map:function(a,b,c){var d,f=0,g=a.length,h=s(a),i=[];if(h)for(;g>f;f++)d=b(a[f],f,c),null!=d&&i.push(d);else for(f in a)d=b(a[f],f,c),null!=d&&i.push(d);return e.apply([],i)},guid:1,proxy:function(a,b){var c,e,f;return"string"==typeof b&&(c=a[b],b=a,a=c),n.isFunction(a)?(e=d.call(arguments,2),f=function(){return a.apply(b||this,e.concat(d.call(arguments)))},f.guid=a.guid=a.guid||n.guid++,f):void 0},now:Date.now,support:k}),n.each("Boolean Number String Function Array Date RegExp Object Error".split(" "),function(a,b){h["[object "+b+"]"]=b.toLowerCase()});function s(a){var b=a.length,c=n.type(a);return"function"===c||n.isWindow(a)?!1:1===a.nodeType&&b?!0:"array"===c||0===b||"number"==typeof b&&b>0&&b-1 in a}var t=function(a){var b,c,d,e,f,g,h,i,j,k,l,m,n,o,p,q,r,s,t,u="sizzle"+-new Date,v=a.document,w=0,x=0,y=gb(),z=gb(),A=gb(),B=function(a,b){return a===b&&(l=!0),0},C="undefined",D=1<<31,E={}.hasOwnProperty,F=[],G=F.pop,H=F.push,I=F.push,J=F.slice,K=F.indexOf||function(a){for(var b=0,c=this.length;c>b;b++)if(this[b]===a)return b;return-1},L="checked|selected|async|autofocus|autoplay|controls|defer|disabled|hidden|ismap|loop|multiple|open|readonly|required|scoped",M="[\\x20\\t\\r\\n\\f]",N="(?:\\\\.|[\\w-]|[^\\x00-\\xa0])+",O=N.replace("w","w#"),P="\\["+M+"*("+N+")(?:"+M+"*([*^$|!~]?=)"+M+"*(?:'((?:\\\\.|[^\\\\'])*)'|\"((?:\\\\.|[^\\\\\"])*)\"|("+O+"))|)"+M+"*\\]",Q=":("+N+")(?:\\((('((?:\\\\.|[^\\\\'])*)'|\"((?:\\\\.|[^\\\\\"])*)\")|((?:\\\\.|[^\\\\()[\\]]|"+P+")*)|.*)\\)|)",R=new RegExp("^"+M+"+|((?:^|[^\\\\])(?:\\\\.)*)"+M+"+$","g"),S=new RegExp("^"+M+"*,"+M+"*"),T=new RegExp("^"+M+"*([>+~]|"+M+")"+M+"*"),U=new RegExp("="+M+"*([^\\]'\"]*?)"+M+"*\\]","g"),V=new RegExp(Q),W=new RegExp("^"+O+"$"),X={ID:new RegExp("^#("+N+")"),CLASS:new RegExp("^\\.("+N+")"),TAG:new RegExp("^("+N.replace("w","w*")+")"),ATTR:new RegExp("^"+P),PSEUDO:new RegExp("^"+Q),CHILD:new RegExp("^:(only|first|last|nth|nth-last)-(child|of-type)(?:\\("+M+"*(even|odd|(([+-]|)(\\d*)n|)"+M+"*(?:([+-]|)"+M+"*(\\d+)|))"+M+"*\\)|)","i"),bool:new RegExp("^(?:"+L+")$","i"),needsContext:new RegExp("^"+M+"*[>+~]|:(even|odd|eq|gt|lt|nth|first|last)(?:\\("+M+"*((?:-\\d)?\\d*)"+M+"*\\)|)(?=[^-]|$)","i")},Y=/^(?:input|select|textarea|button)$/i,Z=/^h\d$/i,$=/^[^{]+\{\s*\[native \w/,_=/^(?:#([\w-]+)|(\w+)|\.([\w-]+))$/,ab=/[+~]/,bb=/'|\\/g,cb=new RegExp("\\\\([\\da-f]{1,6}"+M+"?|("+M+")|.)","ig"),db=function(a,b,c){var d="0x"+b-65536;return d!==d||c?b:0>d?String.fromCharCode(d+65536):String.fromCharCode(d>>10|55296,1023&d|56320)};try{I.apply(F=J.call(v.childNodes),v.childNodes),F[v.childNodes.length].nodeType}catch(eb){I={apply:F.length?function(a,b){H.apply(a,J.call(b))}:function(a,b){var c=a.length,d=0;while(a[c++]=b[d++]);a.length=c-1}}}function fb(a,b,d,e){var f,h,j,k,l,o,r,s,w,x;if((b?b.ownerDocument||b:v)!==n&&m(b),b=b||n,d=d||[],!a||"string"!=typeof a)return d;if(1!==(k=b.nodeType)&&9!==k)return[];if(p&&!e){if(f=_.exec(a))if(j=f[1]){if(9===k){if(h=b.getElementById(j),!h||!h.parentNode)return d;if(h.id===j)return d.push(h),d}else if(b.ownerDocument&&(h=b.ownerDocument.getElementById(j))&&t(b,h)&&h.id===j)return d.push(h),d}else{if(f[2])return I.apply(d,b.getElementsByTagName(a)),d;if((j=f[3])&&c.getElementsByClassName&&b.getElementsByClassName)return I.apply(d,b.getElementsByClassName(j)),d}if(c.qsa&&(!q||!q.test(a))){if(s=r=u,w=b,x=9===k&&a,1===k&&"object"!==b.nodeName.toLowerCase()){o=g(a),(r=b.getAttribute("id"))?s=r.replace(bb,"\\$&"):b.setAttribute("id",s),s="[id='"+s+"'] ",l=o.length;while(l--)o[l]=s+qb(o[l]);w=ab.test(a)&&ob(b.parentNode)||b,x=o.join(",")}if(x)try{return I.apply(d,w.querySelectorAll(x)),d}catch(y){}finally{r||b.removeAttribute("id")}}}return i(a.replace(R,"$1"),b,d,e)}function gb(){var a=[];function b(c,e){return a.push(c+" ")>d.cacheLength&&delete b[a.shift()],b[c+" "]=e}return b}function hb(a){return a[u]=!0,a}function ib(a){var b=n.createElement("div");try{return!!a(b)}catch(c){return!1}finally{b.parentNode&&b.parentNode.removeChild(b),b=null}}function jb(a,b){var c=a.split("|"),e=a.length;while(e--)d.attrHandle[c[e]]=b}function kb(a,b){var c=b&&a,d=c&&1===a.nodeType&&1===b.nodeType&&(~b.sourceIndex||D)-(~a.sourceIndex||D);if(d)return d;if(c)while(c=c.nextSibling)if(c===b)return-1;return a?1:-1}function lb(a){return function(b){var c=b.nodeName.toLowerCase();return"input"===c&&b.type===a}}function mb(a){return function(b){var c=b.nodeName.toLowerCase();return("input"===c||"button"===c)&&b.type===a}}function nb(a){return hb(function(b){return b=+b,hb(function(c,d){var e,f=a([],c.length,b),g=f.length;while(g--)c[e=f[g]]&&(c[e]=!(d[e]=c[e]))})})}function ob(a){return a&&typeof a.getElementsByTagName!==C&&a}c=fb.support={},f=fb.isXML=function(a){var b=a&&(a.ownerDocument||a).documentElement;return b?"HTML"!==b.nodeName:!1},m=fb.setDocument=function(a){var b,e=a?a.ownerDocument||a:v,g=e.defaultView;return e!==n&&9===e.nodeType&&e.documentElement?(n=e,o=e.documentElement,p=!f(e),g&&g!==g.top&&(g.addEventListener?g.addEventListener("unload",function(){m()},!1):g.attachEvent&&g.attachEvent("onunload",function(){m()})),c.attributes=ib(function(a){return a.className="i",!a.getAttribute("className")}),c.getElementsByTagName=ib(function(a){return a.appendChild(e.createComment("")),!a.getElementsByTagName("*").length}),c.getElementsByClassName=$.test(e.getElementsByClassName)&&ib(function(a){return a.innerHTML="<div class='a'></div><div class='a i'></div>",a.firstChild.className="i",2===a.getElementsByClassName("i").length}),c.getById=ib(function(a){return o.appendChild(a).id=u,!e.getElementsByName||!e.getElementsByName(u).length}),c.getById?(d.find.ID=function(a,b){if(typeof b.getElementById!==C&&p){var c=b.getElementById(a);return c&&c.parentNode?[c]:[]}},d.filter.ID=function(a){var b=a.replace(cb,db);return function(a){return a.getAttribute("id")===b}}):(delete d.find.ID,d.filter.ID=function(a){var b=a.replace(cb,db);return function(a){var c=typeof a.getAttributeNode!==C&&a.getAttributeNode("id");return c&&c.value===b}}),d.find.TAG=c.getElementsByTagName?function(a,b){return typeof b.getElementsByTagName!==C?b.getElementsByTagName(a):void 0}:function(a,b){var c,d=[],e=0,f=b.getElementsByTagName(a);if("*"===a){while(c=f[e++])1===c.nodeType&&d.push(c);return d}return f},d.find.CLASS=c.getElementsByClassName&&function(a,b){return typeof b.getElementsByClassName!==C&&p?b.getElementsByClassName(a):void 0},r=[],q=[],(c.qsa=$.test(e.querySelectorAll))&&(ib(function(a){a.innerHTML="<select msallowclip=''><option selected=''></option></select>",a.querySelectorAll("[msallowclip^='']").length&&q.push("[*^$]="+M+"*(?:''|\"\")"),a.querySelectorAll("[selected]").length||q.push("\\["+M+"*(?:value|"+L+")"),a.querySelectorAll(":checked").length||q.push(":checked")}),ib(function(a){var b=e.createElement("input");b.setAttribute("type","hidden"),a.appendChild(b).setAttribute("name","D"),a.querySelectorAll("[name=d]").length&&q.push("name"+M+"*[*^$|!~]?="),a.querySelectorAll(":enabled").length||q.push(":enabled",":disabled"),a.querySelectorAll("*,:x"),q.push(",.*:")})),(c.matchesSelector=$.test(s=o.matches||o.webkitMatchesSelector||o.mozMatchesSelector||o.oMatchesSelector||o.msMatchesSelector))&&ib(function(a){c.disconnectedMatch=s.call(a,"div"),s.call(a,"[s!='']:x"),r.push("!=",Q)}),q=q.length&&new RegExp(q.join("|")),r=r.length&&new RegExp(r.join("|")),b=$.test(o.compareDocumentPosition),t=b||$.test(o.contains)?function(a,b){var c=9===a.nodeType?a.documentElement:a,d=b&&b.parentNode;return a===d||!(!d||1!==d.nodeType||!(c.contains?c.contains(d):a.compareDocumentPosition&&16&a.compareDocumentPosition(d)))}:function(a,b){if(b)while(b=b.parentNode)if(b===a)return!0;return!1},B=b?function(a,b){if(a===b)return l=!0,0;var d=!a.compareDocumentPosition-!b.compareDocumentPosition;return d?d:(d=(a.ownerDocument||a)===(b.ownerDocument||b)?a.compareDocumentPosition(b):1,1&d||!c.sortDetached&&b.compareDocumentPosition(a)===d?a===e||a.ownerDocument===v&&t(v,a)?-1:b===e||b.ownerDocument===v&&t(v,b)?1:k?K.call(k,a)-K.call(k,b):0:4&d?-1:1)}:function(a,b){if(a===b)return l=!0,0;var c,d=0,f=a.parentNode,g=b.parentNode,h=[a],i=[b];if(!f||!g)return a===e?-1:b===e?1:f?-1:g?1:k?K.call(k,a)-K.call(k,b):0;if(f===g)return kb(a,b);c=a;while(c=c.parentNode)h.unshift(c);c=b;while(c=c.parentNode)i.unshift(c);while(h[d]===i[d])d++;return d?kb(h[d],i[d]):h[d]===v?-1:i[d]===v?1:0},e):n},fb.matches=function(a,b){return fb(a,null,null,b)},fb.matchesSelector=function(a,b){if((a.ownerDocument||a)!==n&&m(a),b=b.replace(U,"='$1']"),!(!c.matchesSelector||!p||r&&r.test(b)||q&&q.test(b)))try{var d=s.call(a,b);if(d||c.disconnectedMatch||a.document&&11!==a.document.nodeType)return d}catch(e){}return fb(b,n,null,[a]).length>0},fb.contains=function(a,b){return(a.ownerDocument||a)!==n&&m(a),t(a,b)},fb.attr=function(a,b){(a.ownerDocument||a)!==n&&m(a);var e=d.attrHandle[b.toLowerCase()],f=e&&E.call(d.attrHandle,b.toLowerCase())?e(a,b,!p):void 0;return void 0!==f?f:c.attributes||!p?a.getAttribute(b):(f=a.getAttributeNode(b))&&f.specified?f.value:null},fb.error=function(a){throw new Error("Syntax error, unrecognized expression: "+a)},fb.uniqueSort=function(a){var b,d=[],e=0,f=0;if(l=!c.detectDuplicates,k=!c.sortStable&&a.slice(0),a.sort(B),l){while(b=a[f++])b===a[f]&&(e=d.push(f));while(e--)a.splice(d[e],1)}return k=null,a},e=fb.getText=function(a){var b,c="",d=0,f=a.nodeType;if(f){if(1===f||9===f||11===f){if("string"==typeof a.textContent)return a.textContent;for(a=a.firstChild;a;a=a.nextSibling)c+=e(a)}else if(3===f||4===f)return a.nodeValue}else while(b=a[d++])c+=e(b);return c},d=fb.selectors={cacheLength:50,createPseudo:hb,match:X,attrHandle:{},find:{},relative:{">":{dir:"parentNode",first:!0}," ":{dir:"parentNode"},"+":{dir:"previousSibling",first:!0},"~":{dir:"previousSibling"}},preFilter:{ATTR:function(a){return a[1]=a[1].replace(cb,db),a[3]=(a[3]||a[4]||a[5]||"").replace(cb,db),"~="===a[2]&&(a[3]=" "+a[3]+" "),a.slice(0,4)},CHILD:function(a){return a[1]=a[1].toLowerCase(),"nth"===a[1].slice(0,3)?(a[3]||fb.error(a[0]),a[4]=+(a[4]?a[5]+(a[6]||1):2*("even"===a[3]||"odd"===a[3])),a[5]=+(a[7]+a[8]||"odd"===a[3])):a[3]&&fb.error(a[0]),a},PSEUDO:function(a){var b,c=!a[6]&&a[2];return X.CHILD.test(a[0])?null:(a[3]?a[2]=a[4]||a[5]||"":c&&V.test(c)&&(b=g(c,!0))&&(b=c.indexOf(")",c.length-b)-c.length)&&(a[0]=a[0].slice(0,b),a[2]=c.slice(0,b)),a.slice(0,3))}},filter:{TAG:function(a){var b=a.replace(cb,db).toLowerCase();return"*"===a?function(){return!0}:function(a){return a.nodeName&&a.nodeName.toLowerCase()===b}},CLASS:function(a){var b=y[a+" "];return b||(b=new RegExp("(^|"+M+")"+a+"("+M+"|$)"))&&y(a,function(a){return b.test("string"==typeof a.className&&a.className||typeof a.getAttribute!==C&&a.getAttribute("class")||"")})},ATTR:function(a,b,c){return function(d){var e=fb.attr(d,a);return null==e?"!="===b:b?(e+="","="===b?e===c:"!="===b?e!==c:"^="===b?c&&0===e.indexOf(c):"*="===b?c&&e.indexOf(c)>-1:"$="===b?c&&e.slice(-c.length)===c:"~="===b?(" "+e+" ").indexOf(c)>-1:"|="===b?e===c||e.slice(0,c.length+1)===c+"-":!1):!0}},CHILD:function(a,b,c,d,e){var f="nth"!==a.slice(0,3),g="last"!==a.slice(-4),h="of-type"===b;return 1===d&&0===e?function(a){return!!a.parentNode}:function(b,c,i){var j,k,l,m,n,o,p=f!==g?"nextSibling":"previousSibling",q=b.parentNode,r=h&&b.nodeName.toLowerCase(),s=!i&&!h;if(q){if(f){while(p){l=b;while(l=l[p])if(h?l.nodeName.toLowerCase()===r:1===l.nodeType)return!1;o=p="only"===a&&!o&&"nextSibling"}return!0}if(o=[g?q.firstChild:q.lastChild],g&&s){k=q[u]||(q[u]={}),j=k[a]||[],n=j[0]===w&&j[1],m=j[0]===w&&j[2],l=n&&q.childNodes[n];while(l=++n&&l&&l[p]||(m=n=0)||o.pop())if(1===l.nodeType&&++m&&l===b){k[a]=[w,n,m];break}}else if(s&&(j=(b[u]||(b[u]={}))[a])&&j[0]===w)m=j[1];else while(l=++n&&l&&l[p]||(m=n=0)||o.pop())if((h?l.nodeName.toLowerCase()===r:1===l.nodeType)&&++m&&(s&&((l[u]||(l[u]={}))[a]=[w,m]),l===b))break;return m-=e,m===d||m%d===0&&m/d>=0}}},PSEUDO:function(a,b){var c,e=d.pseudos[a]||d.setFilters[a.toLowerCase()]||fb.error("unsupported pseudo: "+a);return e[u]?e(b):e.length>1?(c=[a,a,"",b],d.setFilters.hasOwnProperty(a.toLowerCase())?hb(function(a,c){var d,f=e(a,b),g=f.length;while(g--)d=K.call(a,f[g]),a[d]=!(c[d]=f[g])}):function(a){return e(a,0,c)}):e}},pseudos:{not:hb(function(a){var b=[],c=[],d=h(a.replace(R,"$1"));return d[u]?hb(function(a,b,c,e){var f,g=d(a,null,e,[]),h=a.length;while(h--)(f=g[h])&&(a[h]=!(b[h]=f))}):function(a,e,f){return b[0]=a,d(b,null,f,c),!c.pop()}}),has:hb(function(a){return function(b){return fb(a,b).length>0}}),contains:hb(function(a){return function(b){return(b.textContent||b.innerText||e(b)).indexOf(a)>-1}}),lang:hb(function(a){return W.test(a||"")||fb.error("unsupported lang: "+a),a=a.replace(cb,db).toLowerCase(),function(b){var c;do if(c=p?b.lang:b.getAttribute("xml:lang")||b.getAttribute("lang"))return c=c.toLowerCase(),c===a||0===c.indexOf(a+"-");while((b=b.parentNode)&&1===b.nodeType);return!1}}),target:function(b){var c=a.location&&a.location.hash;return c&&c.slice(1)===b.id},root:function(a){return a===o},focus:function(a){return a===n.activeElement&&(!n.hasFocus||n.hasFocus())&&!!(a.type||a.href||~a.tabIndex)},enabled:function(a){return a.disabled===!1},disabled:function(a){return a.disabled===!0},checked:function(a){var b=a.nodeName.toLowerCase();return"input"===b&&!!a.checked||"option"===b&&!!a.selected},selected:function(a){return a.parentNode&&a.parentNode.selectedIndex,a.selected===!0},empty:function(a){for(a=a.firstChild;a;a=a.nextSibling)if(a.nodeType<6)return!1;return!0},parent:function(a){return!d.pseudos.empty(a)},header:function(a){return Z.test(a.nodeName)},input:function(a){return Y.test(a.nodeName)},button:function(a){var b=a.nodeName.toLowerCase();return"input"===b&&"button"===a.type||"button"===b},text:function(a){var b;return"input"===a.nodeName.toLowerCase()&&"text"===a.type&&(null==(b=a.getAttribute("type"))||"text"===b.toLowerCase())},first:nb(function(){return[0]}),last:nb(function(a,b){return[b-1]}),eq:nb(function(a,b,c){return[0>c?c+b:c]}),even:nb(function(a,b){for(var c=0;b>c;c+=2)a.push(c);return a}),odd:nb(function(a,b){for(var c=1;b>c;c+=2)a.push(c);return a}),lt:nb(function(a,b,c){for(var d=0>c?c+b:c;--d>=0;)a.push(d);return a}),gt:nb(function(a,b,c){for(var d=0>c?c+b:c;++d<b;)a.push(d);return a})}},d.pseudos.nth=d.pseudos.eq;for(b in{radio:!0,checkbox:!0,file:!0,password:!0,image:!0})d.pseudos[b]=lb(b);for(b in{submit:!0,reset:!0})d.pseudos[b]=mb(b);function pb(){}pb.prototype=d.filters=d.pseudos,d.setFilters=new pb,g=fb.tokenize=function(a,b){var c,e,f,g,h,i,j,k=z[a+" "];if(k)return b?0:k.slice(0);h=a,i=[],j=d.preFilter;while(h){(!c||(e=S.exec(h)))&&(e&&(h=h.slice(e[0].length)||h),i.push(f=[])),c=!1,(e=T.exec(h))&&(c=e.shift(),f.push({value:c,type:e[0].replace(R," ")}),h=h.slice(c.length));for(g in d.filter)!(e=X[g].exec(h))||j[g]&&!(e=j[g](e))||(c=e.shift(),f.push({value:c,type:g,matches:e}),h=h.slice(c.length));if(!c)break}return b?h.length:h?fb.error(a):z(a,i).slice(0)};function qb(a){for(var b=0,c=a.length,d="";c>b;b++)d+=a[b].value;return d}function rb(a,b,c){var d=b.dir,e=c&&"parentNode"===d,f=x++;return b.first?function(b,c,f){while(b=b[d])if(1===b.nodeType||e)return a(b,c,f)}:function(b,c,g){var h,i,j=[w,f];if(g){while(b=b[d])if((1===b.nodeType||e)&&a(b,c,g))return!0}else while(b=b[d])if(1===b.nodeType||e){if(i=b[u]||(b[u]={}),(h=i[d])&&h[0]===w&&h[1]===f)return j[2]=h[2];if(i[d]=j,j[2]=a(b,c,g))return!0}}}function sb(a){return a.length>1?function(b,c,d){var e=a.length;while(e--)if(!a[e](b,c,d))return!1;return!0}:a[0]}function tb(a,b,c){for(var d=0,e=b.length;e>d;d++)fb(a,b[d],c);return c}function ub(a,b,c,d,e){for(var f,g=[],h=0,i=a.length,j=null!=b;i>h;h++)(f=a[h])&&(!c||c(f,d,e))&&(g.push(f),j&&b.push(h));return g}function vb(a,b,c,d,e,f){return d&&!d[u]&&(d=vb(d)),e&&!e[u]&&(e=vb(e,f)),hb(function(f,g,h,i){var j,k,l,m=[],n=[],o=g.length,p=f||tb(b||"*",h.nodeType?[h]:h,[]),q=!a||!f&&b?p:ub(p,m,a,h,i),r=c?e||(f?a:o||d)?[]:g:q;if(c&&c(q,r,h,i),d){j=ub(r,n),d(j,[],h,i),k=j.length;while(k--)(l=j[k])&&(r[n[k]]=!(q[n[k]]=l))}if(f){if(e||a){if(e){j=[],k=r.length;while(k--)(l=r[k])&&j.push(q[k]=l);e(null,r=[],j,i)}k=r.length;while(k--)(l=r[k])&&(j=e?K.call(f,l):m[k])>-1&&(f[j]=!(g[j]=l))}}else r=ub(r===g?r.splice(o,r.length):r),e?e(null,g,r,i):I.apply(g,r)})}function wb(a){for(var b,c,e,f=a.length,g=d.relative[a[0].type],h=g||d.relative[" "],i=g?1:0,k=rb(function(a){return a===b},h,!0),l=rb(function(a){return K.call(b,a)>-1},h,!0),m=[function(a,c,d){return!g&&(d||c!==j)||((b=c).nodeType?k(a,c,d):l(a,c,d))}];f>i;i++)if(c=d.relative[a[i].type])m=[rb(sb(m),c)];else{if(c=d.filter[a[i].type].apply(null,a[i].matches),c[u]){for(e=++i;f>e;e++)if(d.relative[a[e].type])break;return vb(i>1&&sb(m),i>1&&qb(a.slice(0,i-1).concat({value:" "===a[i-2].type?"*":""})).replace(R,"$1"),c,e>i&&wb(a.slice(i,e)),f>e&&wb(a=a.slice(e)),f>e&&qb(a))}m.push(c)}return sb(m)}function xb(a,b){var c=b.length>0,e=a.length>0,f=function(f,g,h,i,k){var l,m,o,p=0,q="0",r=f&&[],s=[],t=j,u=f||e&&d.find.TAG("*",k),v=w+=null==t?1:Math.random()||.1,x=u.length;for(k&&(j=g!==n&&g);q!==x&&null!=(l=u[q]);q++){if(e&&l){m=0;while(o=a[m++])if(o(l,g,h)){i.push(l);break}k&&(w=v)}c&&((l=!o&&l)&&p--,f&&r.push(l))}if(p+=q,c&&q!==p){m=0;while(o=b[m++])o(r,s,g,h);if(f){if(p>0)while(q--)r[q]||s[q]||(s[q]=G.call(i));s=ub(s)}I.apply(i,s),k&&!f&&s.length>0&&p+b.length>1&&fb.uniqueSort(i)}return k&&(w=v,j=t),r};return c?hb(f):f}return h=fb.compile=function(a,b){var c,d=[],e=[],f=A[a+" "];if(!f){b||(b=g(a)),c=b.length;while(c--)f=wb(b[c]),f[u]?d.push(f):e.push(f);f=A(a,xb(e,d)),f.selector=a}return f},i=fb.select=function(a,b,e,f){var i,j,k,l,m,n="function"==typeof a&&a,o=!f&&g(a=n.selector||a);if(e=e||[],1===o.length){if(j=o[0]=o[0].slice(0),j.length>2&&"ID"===(k=j[0]).type&&c.getById&&9===b.nodeType&&p&&d.relative[j[1].type]){if(b=(d.find.ID(k.matches[0].replace(cb,db),b)||[])[0],!b)return e;n&&(b=b.parentNode),a=a.slice(j.shift().value.length)}i=X.needsContext.test(a)?0:j.length;while(i--){if(k=j[i],d.relative[l=k.type])break;if((m=d.find[l])&&(f=m(k.matches[0].replace(cb,db),ab.test(j[0].type)&&ob(b.parentNode)||b))){if(j.splice(i,1),a=f.length&&qb(j),!a)return I.apply(e,f),e;break}}}return(n||h(a,o))(f,b,!p,e,ab.test(a)&&ob(b.parentNode)||b),e},c.sortStable=u.split("").sort(B).join("")===u,c.detectDuplicates=!!l,m(),c.sortDetached=ib(function(a){return 1&a.compareDocumentPosition(n.createElement("div"))}),ib(function(a){return a.innerHTML="<a href='#'></a>","#"===a.firstChild.getAttribute("href")})||jb("type|href|height|width",function(a,b,c){return c?void 0:a.getAttribute(b,"type"===b.toLowerCase()?1:2)}),c.attributes&&ib(function(a){return a.innerHTML="<input/>",a.firstChild.setAttribute("value",""),""===a.firstChild.getAttribute("value")})||jb("value",function(a,b,c){return c||"input"!==a.nodeName.toLowerCase()?void 0:a.defaultValue}),ib(function(a){return null==a.getAttribute("disabled")})||jb(L,function(a,b,c){var d;return c?void 0:a[b]===!0?b.toLowerCase():(d=a.getAttributeNode(b))&&d.specified?d.value:null}),fb}(a);n.find=t,n.expr=t.selectors,n.expr[":"]=n.expr.pseudos,n.unique=t.uniqueSort,n.text=t.getText,n.isXMLDoc=t.isXML,n.contains=t.contains;var u=n.expr.match.needsContext,v=/^<(\w+)\s*\/?>(?:<\/\1>|)$/,w=/^.[^:#\[\.,]*$/;function x(a,b,c){if(n.isFunction(b))return n.grep(a,function(a,d){return!!b.call(a,d,a)!==c});if(b.nodeType)return n.grep(a,function(a){return a===b!==c});if("string"==typeof b){if(w.test(b))return n.filter(b,a,c);b=n.filter(b,a)}return n.grep(a,function(a){return g.call(b,a)>=0!==c})}n.filter=function(a,b,c){var d=b[0];return c&&(a=":not("+a+")"),1===b.length&&1===d.nodeType?n.find.matchesSelector(d,a)?[d]:[]:n.find.matches(a,n.grep(b,function(a){return 1===a.nodeType}))},n.fn.extend({find:function(a){var b,c=this.length,d=[],e=this;if("string"!=typeof a)return this.pushStack(n(a).filter(function(){for(b=0;c>b;b++)if(n.contains(e[b],this))return!0}));for(b=0;c>b;b++)n.find(a,e[b],d);return d=this.pushStack(c>1?n.unique(d):d),d.selector=this.selector?this.selector+" "+a:a,d},filter:function(a){return this.pushStack(x(this,a||[],!1))},not:function(a){return this.pushStack(x(this,a||[],!0))},is:function(a){return!!x(this,"string"==typeof a&&u.test(a)?n(a):a||[],!1).length}});var y,z=/^(?:\s*(<[\w\W]+>)[^>]*|#([\w-]*))$/,A=n.fn.init=function(a,b){var c,d;if(!a)return this;if("string"==typeof a){if(c="<"===a[0]&&">"===a[a.length-1]&&a.length>=3?[null,a,null]:z.exec(a),!c||!c[1]&&b)return!b||b.jquery?(b||y).find(a):this.constructor(b).find(a);if(c[1]){if(b=b instanceof n?b[0]:b,n.merge(this,n.parseHTML(c[1],b&&b.nodeType?b.ownerDocument||b:l,!0)),v.test(c[1])&&n.isPlainObject(b))for(c in b)n.isFunction(this[c])?this[c](b[c]):this.attr(c,b[c]);return this}return d=l.getElementById(c[2]),d&&d.parentNode&&(this.length=1,this[0]=d),this.context=l,this.selector=a,this}return a.nodeType?(this.context=this[0]=a,this.length=1,this):n.isFunction(a)?"undefined"!=typeof y.ready?y.ready(a):a(n):(void 0!==a.selector&&(this.selector=a.selector,this.context=a.context),n.makeArray(a,this))};A.prototype=n.fn,y=n(l);var B=/^(?:parents|prev(?:Until|All))/,C={children:!0,contents:!0,next:!0,prev:!0};n.extend({dir:function(a,b,c){var d=[],e=void 0!==c;while((a=a[b])&&9!==a.nodeType)if(1===a.nodeType){if(e&&n(a).is(c))break;d.push(a)}return d},sibling:function(a,b){for(var c=[];a;a=a.nextSibling)1===a.nodeType&&a!==b&&c.push(a);return c}}),n.fn.extend({has:function(a){var b=n(a,this),c=b.length;return this.filter(function(){for(var a=0;c>a;a++)if(n.contains(this,b[a]))return!0})},closest:function(a,b){for(var c,d=0,e=this.length,f=[],g=u.test(a)||"string"!=typeof a?n(a,b||this.context):0;e>d;d++)for(c=this[d];c&&c!==b;c=c.parentNode)if(c.nodeType<11&&(g?g.index(c)>-1:1===c.nodeType&&n.find.matchesSelector(c,a))){f.push(c);break}return this.pushStack(f.length>1?n.unique(f):f)},index:function(a){return a?"string"==typeof a?g.call(n(a),this[0]):g.call(this,a.jquery?a[0]:a):this[0]&&this[0].parentNode?this.first().prevAll().length:-1},add:function(a,b){return this.pushStack(n.unique(n.merge(this.get(),n(a,b))))},addBack:function(a){return this.add(null==a?this.prevObject:this.prevObject.filter(a))}});function D(a,b){while((a=a[b])&&1!==a.nodeType);return a}n.each({parent:function(a){var b=a.parentNode;return b&&11!==b.nodeType?b:null},parents:function(a){return n.dir(a,"parentNode")},parentsUntil:function(a,b,c){return n.dir(a,"parentNode",c)},next:function(a){return D(a,"nextSibling")},prev:function(a){return D(a,"previousSibling")},nextAll:function(a){return n.dir(a,"nextSibling")},prevAll:function(a){return n.dir(a,"previousSibling")},nextUntil:function(a,b,c){return n.dir(a,"nextSibling",c)},prevUntil:function(a,b,c){return n.dir(a,"previousSibling",c)},siblings:function(a){return n.sibling((a.parentNode||{}).firstChild,a)},children:function(a){return n.sibling(a.firstChild)},contents:function(a){return a.contentDocument||n.merge([],a.childNodes)}},function(a,b){n.fn[a]=function(c,d){var e=n.map(this,b,c);return"Until"!==a.slice(-5)&&(d=c),d&&"string"==typeof d&&(e=n.filter(d,e)),this.length>1&&(C[a]||n.unique(e),B.test(a)&&e.reverse()),this.pushStack(e)}});var E=/\S+/g,F={};function G(a){var b=F[a]={};return n.each(a.match(E)||[],function(a,c){b[c]=!0}),b}n.Callbacks=function(a){a="string"==typeof a?F[a]||G(a):n.extend({},a);var b,c,d,e,f,g,h=[],i=!a.once&&[],j=function(l){for(b=a.memory&&l,c=!0,g=e||0,e=0,f=h.length,d=!0;h&&f>g;g++)if(h[g].apply(l[0],l[1])===!1&&a.stopOnFalse){b=!1;break}d=!1,h&&(i?i.length&&j(i.shift()):b?h=[]:k.disable())},k={add:function(){if(h){var c=h.length;!function g(b){n.each(b,function(b,c){var d=n.type(c);"function"===d?a.unique&&k.has(c)||h.push(c):c&&c.length&&"string"!==d&&g(c)})}(arguments),d?f=h.length:b&&(e=c,j(b))}return this},remove:function(){return h&&n.each(arguments,function(a,b){var c;while((c=n.inArray(b,h,c))>-1)h.splice(c,1),d&&(f>=c&&f--,g>=c&&g--)}),this},has:function(a){return a?n.inArray(a,h)>-1:!(!h||!h.length)},empty:function(){return h=[],f=0,this},disable:function(){return h=i=b=void 0,this},disabled:function(){return!h},lock:function(){return i=void 0,b||k.disable(),this},locked:function(){return!i},fireWith:function(a,b){return!h||c&&!i||(b=b||[],b=[a,b.slice?b.slice():b],d?i.push(b):j(b)),this},fire:function(){return k.fireWith(this,arguments),this},fired:function(){return!!c}};return k},n.extend({Deferred:function(a){var b=[["resolve","done",n.Callbacks("once memory"),"resolved"],["reject","fail",n.Callbacks("once memory"),"rejected"],["notify","progress",n.Callbacks("memory")]],c="pending",d={state:function(){return c},always:function(){return e.done(arguments).fail(arguments),this},then:function(){var a=arguments;return n.Deferred(function(c){n.each(b,function(b,f){var g=n.isFunction(a[b])&&a[b];e[f[1]](function(){var a=g&&g.apply(this,arguments);a&&n.isFunction(a.promise)?a.promise().done(c.resolve).fail(c.reject).progress(c.notify):c[f[0]+"With"](this===d?c.promise():this,g?[a]:arguments)})}),a=null}).promise()},promise:function(a){return null!=a?n.extend(a,d):d}},e={};return d.pipe=d.then,n.each(b,function(a,f){var g=f[2],h=f[3];d[f[1]]=g.add,h&&g.add(function(){c=h},b[1^a][2].disable,b[2][2].lock),e[f[0]]=function(){return e[f[0]+"With"](this===e?d:this,arguments),this},e[f[0]+"With"]=g.fireWith}),d.promise(e),a&&a.call(e,e),e},when:function(a){var b=0,c=d.call(arguments),e=c.length,f=1!==e||a&&n.isFunction(a.promise)?e:0,g=1===f?a:n.Deferred(),h=function(a,b,c){return function(e){b[a]=this,c[a]=arguments.length>1?d.call(arguments):e,c===i?g.notifyWith(b,c):--f||g.resolveWith(b,c)}},i,j,k;if(e>1)for(i=new Array(e),j=new Array(e),k=new Array(e);e>b;b++)c[b]&&n.isFunction(c[b].promise)?c[b].promise().done(h(b,k,c)).fail(g.reject).progress(h(b,j,i)):--f;return f||g.resolveWith(k,c),g.promise()}});var H;n.fn.ready=function(a){return n.ready.promise().done(a),this},n.extend({isReady:!1,readyWait:1,holdReady:function(a){a?n.readyWait++:n.ready(!0)},ready:function(a){(a===!0?--n.readyWait:n.isReady)||(n.isReady=!0,a!==!0&&--n.readyWait>0||(H.resolveWith(l,[n]),n.fn.triggerHandler&&(n(l).triggerHandler("ready"),n(l).off("ready"))))}});function I(){l.removeEventListener("DOMContentLoaded",I,!1),a.removeEventListener("load",I,!1),n.ready()}n.ready.promise=function(b){return H||(H=n.Deferred(),"complete"===l.readyState?setTimeout(n.ready):(l.addEventListener("DOMContentLoaded",I,!1),a.addEventListener("load",I,!1))),H.promise(b)},n.ready.promise();var J=n.access=function(a,b,c,d,e,f,g){var h=0,i=a.length,j=null==c;if("object"===n.type(c)){e=!0;for(h in c)n.access(a,b,h,c[h],!0,f,g)}else if(void 0!==d&&(e=!0,n.isFunction(d)||(g=!0),j&&(g?(b.call(a,d),b=null):(j=b,b=function(a,b,c){return j.call(n(a),c)})),b))for(;i>h;h++)b(a[h],c,g?d:d.call(a[h],h,b(a[h],c)));return e?a:j?b.call(a):i?b(a[0],c):f};n.acceptData=function(a){return 1===a.nodeType||9===a.nodeType||!+a.nodeType};function K(){Object.defineProperty(this.cache={},0,{get:function(){return{}}}),this.expando=n.expando+Math.random()}K.uid=1,K.accepts=n.acceptData,K.prototype={key:function(a){if(!K.accepts(a))return 0;var b={},c=a[this.expando];if(!c){c=K.uid++;try{b[this.expando]={value:c},Object.defineProperties(a,b)}catch(d){b[this.expando]=c,n.extend(a,b)}}return this.cache[c]||(this.cache[c]={}),c},set:function(a,b,c){var d,e=this.key(a),f=this.cache[e];if("string"==typeof b)f[b]=c;else if(n.isEmptyObject(f))n.extend(this.cache[e],b);else for(d in b)f[d]=b[d];return f},get:function(a,b){var c=this.cache[this.key(a)];return void 0===b?c:c[b]},access:function(a,b,c){var d;return void 0===b||b&&"string"==typeof b&&void 0===c?(d=this.get(a,b),void 0!==d?d:this.get(a,n.camelCase(b))):(this.set(a,b,c),void 0!==c?c:b)},remove:function(a,b){var c,d,e,f=this.key(a),g=this.cache[f];if(void 0===b)this.cache[f]={};else{n.isArray(b)?d=b.concat(b.map(n.camelCase)):(e=n.camelCase(b),b in g?d=[b,e]:(d=e,d=d in g?[d]:d.match(E)||[])),c=d.length;while(c--)delete g[d[c]]}},hasData:function(a){return!n.isEmptyObject(this.cache[a[this.expando]]||{})},discard:function(a){a[this.expando]&&delete this.cache[a[this.expando]]}};var L=new K,M=new K,N=/^(?:\{[\w\W]*\}|\[[\w\W]*\])$/,O=/([A-Z])/g;function P(a,b,c){var d;if(void 0===c&&1===a.nodeType)if(d="data-"+b.replace(O,"-$1").toLowerCase(),c=a.getAttribute(d),"string"==typeof c){try{c="true"===c?!0:"false"===c?!1:"null"===c?null:+c+""===c?+c:N.test(c)?n.parseJSON(c):c}catch(e){}M.set(a,b,c)}else c=void 0;return c}n.extend({hasData:function(a){return M.hasData(a)||L.hasData(a)},data:function(a,b,c){return M.access(a,b,c)},removeData:function(a,b){M.remove(a,b)
},_data:function(a,b,c){return L.access(a,b,c)},_removeData:function(a,b){L.remove(a,b)}}),n.fn.extend({data:function(a,b){var c,d,e,f=this[0],g=f&&f.attributes;if(void 0===a){if(this.length&&(e=M.get(f),1===f.nodeType&&!L.get(f,"hasDataAttrs"))){c=g.length;while(c--)g[c]&&(d=g[c].name,0===d.indexOf("data-")&&(d=n.camelCase(d.slice(5)),P(f,d,e[d])));L.set(f,"hasDataAttrs",!0)}return e}return"object"==typeof a?this.each(function(){M.set(this,a)}):J(this,function(b){var c,d=n.camelCase(a);if(f&&void 0===b){if(c=M.get(f,a),void 0!==c)return c;if(c=M.get(f,d),void 0!==c)return c;if(c=P(f,d,void 0),void 0!==c)return c}else this.each(function(){var c=M.get(this,d);M.set(this,d,b),-1!==a.indexOf("-")&&void 0!==c&&M.set(this,a,b)})},null,b,arguments.length>1,null,!0)},removeData:function(a){return this.each(function(){M.remove(this,a)})}}),n.extend({queue:function(a,b,c){var d;return a?(b=(b||"fx")+"queue",d=L.get(a,b),c&&(!d||n.isArray(c)?d=L.access(a,b,n.makeArray(c)):d.push(c)),d||[]):void 0},dequeue:function(a,b){b=b||"fx";var c=n.queue(a,b),d=c.length,e=c.shift(),f=n._queueHooks(a,b),g=function(){n.dequeue(a,b)};"inprogress"===e&&(e=c.shift(),d--),e&&("fx"===b&&c.unshift("inprogress"),delete f.stop,e.call(a,g,f)),!d&&f&&f.empty.fire()},_queueHooks:function(a,b){var c=b+"queueHooks";return L.get(a,c)||L.access(a,c,{empty:n.Callbacks("once memory").add(function(){L.remove(a,[b+"queue",c])})})}}),n.fn.extend({queue:function(a,b){var c=2;return"string"!=typeof a&&(b=a,a="fx",c--),arguments.length<c?n.queue(this[0],a):void 0===b?this:this.each(function(){var c=n.queue(this,a,b);n._queueHooks(this,a),"fx"===a&&"inprogress"!==c[0]&&n.dequeue(this,a)})},dequeue:function(a){return this.each(function(){n.dequeue(this,a)})},clearQueue:function(a){return this.queue(a||"fx",[])},promise:function(a,b){var c,d=1,e=n.Deferred(),f=this,g=this.length,h=function(){--d||e.resolveWith(f,[f])};"string"!=typeof a&&(b=a,a=void 0),a=a||"fx";while(g--)c=L.get(f[g],a+"queueHooks"),c&&c.empty&&(d++,c.empty.add(h));return h(),e.promise(b)}});var Q=/[+-]?(?:\d*\.|)\d+(?:[eE][+-]?\d+|)/.source,R=["Top","Right","Bottom","Left"],S=function(a,b){return a=b||a,"none"===n.css(a,"display")||!n.contains(a.ownerDocument,a)},T=/^(?:checkbox|radio)$/i;!function(){var a=l.createDocumentFragment(),b=a.appendChild(l.createElement("div")),c=l.createElement("input");c.setAttribute("type","radio"),c.setAttribute("checked","checked"),c.setAttribute("name","t"),b.appendChild(c),k.checkClone=b.cloneNode(!0).cloneNode(!0).lastChild.checked,b.innerHTML="<textarea>x</textarea>",k.noCloneChecked=!!b.cloneNode(!0).lastChild.defaultValue}();var U="undefined";k.focusinBubbles="onfocusin"in a;var V=/^key/,W=/^(?:mouse|pointer|contextmenu)|click/,X=/^(?:focusinfocus|focusoutblur)$/,Y=/^([^.]*)(?:\.(.+)|)$/;function Z(){return!0}function $(){return!1}function _(){try{return l.activeElement}catch(a){}}n.event={global:{},add:function(a,b,c,d,e){var f,g,h,i,j,k,l,m,o,p,q,r=L.get(a);if(r){c.handler&&(f=c,c=f.handler,e=f.selector),c.guid||(c.guid=n.guid++),(i=r.events)||(i=r.events={}),(g=r.handle)||(g=r.handle=function(b){return typeof n!==U&&n.event.triggered!==b.type?n.event.dispatch.apply(a,arguments):void 0}),b=(b||"").match(E)||[""],j=b.length;while(j--)h=Y.exec(b[j])||[],o=q=h[1],p=(h[2]||"").split(".").sort(),o&&(l=n.event.special[o]||{},o=(e?l.delegateType:l.bindType)||o,l=n.event.special[o]||{},k=n.extend({type:o,origType:q,data:d,handler:c,guid:c.guid,selector:e,needsContext:e&&n.expr.match.needsContext.test(e),namespace:p.join(".")},f),(m=i[o])||(m=i[o]=[],m.delegateCount=0,l.setup&&l.setup.call(a,d,p,g)!==!1||a.addEventListener&&a.addEventListener(o,g,!1)),l.add&&(l.add.call(a,k),k.handler.guid||(k.handler.guid=c.guid)),e?m.splice(m.delegateCount++,0,k):m.push(k),n.event.global[o]=!0)}},remove:function(a,b,c,d,e){var f,g,h,i,j,k,l,m,o,p,q,r=L.hasData(a)&&L.get(a);if(r&&(i=r.events)){b=(b||"").match(E)||[""],j=b.length;while(j--)if(h=Y.exec(b[j])||[],o=q=h[1],p=(h[2]||"").split(".").sort(),o){l=n.event.special[o]||{},o=(d?l.delegateType:l.bindType)||o,m=i[o]||[],h=h[2]&&new RegExp("(^|\\.)"+p.join("\\.(?:.*\\.|)")+"(\\.|$)"),g=f=m.length;while(f--)k=m[f],!e&&q!==k.origType||c&&c.guid!==k.guid||h&&!h.test(k.namespace)||d&&d!==k.selector&&("**"!==d||!k.selector)||(m.splice(f,1),k.selector&&m.delegateCount--,l.remove&&l.remove.call(a,k));g&&!m.length&&(l.teardown&&l.teardown.call(a,p,r.handle)!==!1||n.removeEvent(a,o,r.handle),delete i[o])}else for(o in i)n.event.remove(a,o+b[j],c,d,!0);n.isEmptyObject(i)&&(delete r.handle,L.remove(a,"events"))}},trigger:function(b,c,d,e){var f,g,h,i,k,m,o,p=[d||l],q=j.call(b,"type")?b.type:b,r=j.call(b,"namespace")?b.namespace.split("."):[];if(g=h=d=d||l,3!==d.nodeType&&8!==d.nodeType&&!X.test(q+n.event.triggered)&&(q.indexOf(".")>=0&&(r=q.split("."),q=r.shift(),r.sort()),k=q.indexOf(":")<0&&"on"+q,b=b[n.expando]?b:new n.Event(q,"object"==typeof b&&b),b.isTrigger=e?2:3,b.namespace=r.join("."),b.namespace_re=b.namespace?new RegExp("(^|\\.)"+r.join("\\.(?:.*\\.|)")+"(\\.|$)"):null,b.result=void 0,b.target||(b.target=d),c=null==c?[b]:n.makeArray(c,[b]),o=n.event.special[q]||{},e||!o.trigger||o.trigger.apply(d,c)!==!1)){if(!e&&!o.noBubble&&!n.isWindow(d)){for(i=o.delegateType||q,X.test(i+q)||(g=g.parentNode);g;g=g.parentNode)p.push(g),h=g;h===(d.ownerDocument||l)&&p.push(h.defaultView||h.parentWindow||a)}f=0;while((g=p[f++])&&!b.isPropagationStopped())b.type=f>1?i:o.bindType||q,m=(L.get(g,"events")||{})[b.type]&&L.get(g,"handle"),m&&m.apply(g,c),m=k&&g[k],m&&m.apply&&n.acceptData(g)&&(b.result=m.apply(g,c),b.result===!1&&b.preventDefault());return b.type=q,e||b.isDefaultPrevented()||o._default&&o._default.apply(p.pop(),c)!==!1||!n.acceptData(d)||k&&n.isFunction(d[q])&&!n.isWindow(d)&&(h=d[k],h&&(d[k]=null),n.event.triggered=q,d[q](),n.event.triggered=void 0,h&&(d[k]=h)),b.result}},dispatch:function(a){a=n.event.fix(a);var b,c,e,f,g,h=[],i=d.call(arguments),j=(L.get(this,"events")||{})[a.type]||[],k=n.event.special[a.type]||{};if(i[0]=a,a.delegateTarget=this,!k.preDispatch||k.preDispatch.call(this,a)!==!1){h=n.event.handlers.call(this,a,j),b=0;while((f=h[b++])&&!a.isPropagationStopped()){a.currentTarget=f.elem,c=0;while((g=f.handlers[c++])&&!a.isImmediatePropagationStopped())(!a.namespace_re||a.namespace_re.test(g.namespace))&&(a.handleObj=g,a.data=g.data,e=((n.event.special[g.origType]||{}).handle||g.handler).apply(f.elem,i),void 0!==e&&(a.result=e)===!1&&(a.preventDefault(),a.stopPropagation()))}return k.postDispatch&&k.postDispatch.call(this,a),a.result}},handlers:function(a,b){var c,d,e,f,g=[],h=b.delegateCount,i=a.target;if(h&&i.nodeType&&(!a.button||"click"!==a.type))for(;i!==this;i=i.parentNode||this)if(i.disabled!==!0||"click"!==a.type){for(d=[],c=0;h>c;c++)f=b[c],e=f.selector+" ",void 0===d[e]&&(d[e]=f.needsContext?n(e,this).index(i)>=0:n.find(e,this,null,[i]).length),d[e]&&d.push(f);d.length&&g.push({elem:i,handlers:d})}return h<b.length&&g.push({elem:this,handlers:b.slice(h)}),g},props:"altKey bubbles cancelable ctrlKey currentTarget eventPhase metaKey relatedTarget shiftKey target timeStamp view which".split(" "),fixHooks:{},keyHooks:{props:"char charCode key keyCode".split(" "),filter:function(a,b){return null==a.which&&(a.which=null!=b.charCode?b.charCode:b.keyCode),a}},mouseHooks:{props:"button buttons clientX clientY offsetX offsetY pageX pageY screenX screenY toElement".split(" "),filter:function(a,b){var c,d,e,f=b.button;return null==a.pageX&&null!=b.clientX&&(c=a.target.ownerDocument||l,d=c.documentElement,e=c.body,a.pageX=b.clientX+(d&&d.scrollLeft||e&&e.scrollLeft||0)-(d&&d.clientLeft||e&&e.clientLeft||0),a.pageY=b.clientY+(d&&d.scrollTop||e&&e.scrollTop||0)-(d&&d.clientTop||e&&e.clientTop||0)),a.which||void 0===f||(a.which=1&f?1:2&f?3:4&f?2:0),a}},fix:function(a){if(a[n.expando])return a;var b,c,d,e=a.type,f=a,g=this.fixHooks[e];g||(this.fixHooks[e]=g=W.test(e)?this.mouseHooks:V.test(e)?this.keyHooks:{}),d=g.props?this.props.concat(g.props):this.props,a=new n.Event(f),b=d.length;while(b--)c=d[b],a[c]=f[c];return a.target||(a.target=l),3===a.target.nodeType&&(a.target=a.target.parentNode),g.filter?g.filter(a,f):a},special:{load:{noBubble:!0},focus:{trigger:function(){return this!==_()&&this.focus?(this.focus(),!1):void 0},delegateType:"focusin"},blur:{trigger:function(){return this===_()&&this.blur?(this.blur(),!1):void 0},delegateType:"focusout"},click:{trigger:function(){return"checkbox"===this.type&&this.click&&n.nodeName(this,"input")?(this.click(),!1):void 0},_default:function(a){return n.nodeName(a.target,"a")}},beforeunload:{postDispatch:function(a){void 0!==a.result&&a.originalEvent&&(a.originalEvent.returnValue=a.result)}}},simulate:function(a,b,c,d){var e=n.extend(new n.Event,c,{type:a,isSimulated:!0,originalEvent:{}});d?n.event.trigger(e,null,b):n.event.dispatch.call(b,e),e.isDefaultPrevented()&&c.preventDefault()}},n.removeEvent=function(a,b,c){a.removeEventListener&&a.removeEventListener(b,c,!1)},n.Event=function(a,b){return this instanceof n.Event?(a&&a.type?(this.originalEvent=a,this.type=a.type,this.isDefaultPrevented=a.defaultPrevented||void 0===a.defaultPrevented&&a.returnValue===!1?Z:$):this.type=a,b&&n.extend(this,b),this.timeStamp=a&&a.timeStamp||n.now(),void(this[n.expando]=!0)):new n.Event(a,b)},n.Event.prototype={isDefaultPrevented:$,isPropagationStopped:$,isImmediatePropagationStopped:$,preventDefault:function(){var a=this.originalEvent;this.isDefaultPrevented=Z,a&&a.preventDefault&&a.preventDefault()},stopPropagation:function(){var a=this.originalEvent;this.isPropagationStopped=Z,a&&a.stopPropagation&&a.stopPropagation()},stopImmediatePropagation:function(){var a=this.originalEvent;this.isImmediatePropagationStopped=Z,a&&a.stopImmediatePropagation&&a.stopImmediatePropagation(),this.stopPropagation()}},n.each({mouseenter:"mouseover",mouseleave:"mouseout",pointerenter:"pointerover",pointerleave:"pointerout"},function(a,b){n.event.special[a]={delegateType:b,bindType:b,handle:function(a){var c,d=this,e=a.relatedTarget,f=a.handleObj;return(!e||e!==d&&!n.contains(d,e))&&(a.type=f.origType,c=f.handler.apply(this,arguments),a.type=b),c}}}),k.focusinBubbles||n.each({focus:"focusin",blur:"focusout"},function(a,b){var c=function(a){n.event.simulate(b,a.target,n.event.fix(a),!0)};n.event.special[b]={setup:function(){var d=this.ownerDocument||this,e=L.access(d,b);e||d.addEventListener(a,c,!0),L.access(d,b,(e||0)+1)},teardown:function(){var d=this.ownerDocument||this,e=L.access(d,b)-1;e?L.access(d,b,e):(d.removeEventListener(a,c,!0),L.remove(d,b))}}}),n.fn.extend({on:function(a,b,c,d,e){var f,g;if("object"==typeof a){"string"!=typeof b&&(c=c||b,b=void 0);for(g in a)this.on(g,b,c,a[g],e);return this}if(null==c&&null==d?(d=b,c=b=void 0):null==d&&("string"==typeof b?(d=c,c=void 0):(d=c,c=b,b=void 0)),d===!1)d=$;else if(!d)return this;return 1===e&&(f=d,d=function(a){return n().off(a),f.apply(this,arguments)},d.guid=f.guid||(f.guid=n.guid++)),this.each(function(){n.event.add(this,a,d,c,b)})},one:function(a,b,c,d){return this.on(a,b,c,d,1)},off:function(a,b,c){var d,e;if(a&&a.preventDefault&&a.handleObj)return d=a.handleObj,n(a.delegateTarget).off(d.namespace?d.origType+"."+d.namespace:d.origType,d.selector,d.handler),this;if("object"==typeof a){for(e in a)this.off(e,b,a[e]);return this}return(b===!1||"function"==typeof b)&&(c=b,b=void 0),c===!1&&(c=$),this.each(function(){n.event.remove(this,a,c,b)})},trigger:function(a,b){return this.each(function(){n.event.trigger(a,b,this)})},triggerHandler:function(a,b){var c=this[0];return c?n.event.trigger(a,b,c,!0):void 0}});var ab=/<(?!area|br|col|embed|hr|img|input|link|meta|param)(([\w:]+)[^>]*)\/>/gi,bb=/<([\w:]+)/,cb=/<|&#?\w+;/,db=/<(?:script|style|link)/i,eb=/checked\s*(?:[^=]|=\s*.checked.)/i,fb=/^$|\/(?:java|ecma)script/i,gb=/^true\/(.*)/,hb=/^\s*<!(?:\[CDATA\[|--)|(?:\]\]|--)>\s*$/g,ib={option:[1,"<select multiple='multiple'>","</select>"],thead:[1,"<table>","</table>"],col:[2,"<table><colgroup>","</colgroup></table>"],tr:[2,"<table><tbody>","</tbody></table>"],td:[3,"<table><tbody><tr>","</tr></tbody></table>"],_default:[0,"",""]};ib.optgroup=ib.option,ib.tbody=ib.tfoot=ib.colgroup=ib.caption=ib.thead,ib.th=ib.td;function jb(a,b){return n.nodeName(a,"table")&&n.nodeName(11!==b.nodeType?b:b.firstChild,"tr")?a.getElementsByTagName("tbody")[0]||a.appendChild(a.ownerDocument.createElement("tbody")):a}function kb(a){return a.type=(null!==a.getAttribute("type"))+"/"+a.type,a}function lb(a){var b=gb.exec(a.type);return b?a.type=b[1]:a.removeAttribute("type"),a}function mb(a,b){for(var c=0,d=a.length;d>c;c++)L.set(a[c],"globalEval",!b||L.get(b[c],"globalEval"))}function nb(a,b){var c,d,e,f,g,h,i,j;if(1===b.nodeType){if(L.hasData(a)&&(f=L.access(a),g=L.set(b,f),j=f.events)){delete g.handle,g.events={};for(e in j)for(c=0,d=j[e].length;d>c;c++)n.event.add(b,e,j[e][c])}M.hasData(a)&&(h=M.access(a),i=n.extend({},h),M.set(b,i))}}function ob(a,b){var c=a.getElementsByTagName?a.getElementsByTagName(b||"*"):a.querySelectorAll?a.querySelectorAll(b||"*"):[];return void 0===b||b&&n.nodeName(a,b)?n.merge([a],c):c}function pb(a,b){var c=b.nodeName.toLowerCase();"input"===c&&T.test(a.type)?b.checked=a.checked:("input"===c||"textarea"===c)&&(b.defaultValue=a.defaultValue)}n.extend({clone:function(a,b,c){var d,e,f,g,h=a.cloneNode(!0),i=n.contains(a.ownerDocument,a);if(!(k.noCloneChecked||1!==a.nodeType&&11!==a.nodeType||n.isXMLDoc(a)))for(g=ob(h),f=ob(a),d=0,e=f.length;e>d;d++)pb(f[d],g[d]);if(b)if(c)for(f=f||ob(a),g=g||ob(h),d=0,e=f.length;e>d;d++)nb(f[d],g[d]);else nb(a,h);return g=ob(h,"script"),g.length>0&&mb(g,!i&&ob(a,"script")),h},buildFragment:function(a,b,c,d){for(var e,f,g,h,i,j,k=b.createDocumentFragment(),l=[],m=0,o=a.length;o>m;m++)if(e=a[m],e||0===e)if("object"===n.type(e))n.merge(l,e.nodeType?[e]:e);else if(cb.test(e)){f=f||k.appendChild(b.createElement("div")),g=(bb.exec(e)||["",""])[1].toLowerCase(),h=ib[g]||ib._default,f.innerHTML=h[1]+e.replace(ab,"<$1></$2>")+h[2],j=h[0];while(j--)f=f.lastChild;n.merge(l,f.childNodes),f=k.firstChild,f.textContent=""}else l.push(b.createTextNode(e));k.textContent="",m=0;while(e=l[m++])if((!d||-1===n.inArray(e,d))&&(i=n.contains(e.ownerDocument,e),f=ob(k.appendChild(e),"script"),i&&mb(f),c)){j=0;while(e=f[j++])fb.test(e.type||"")&&c.push(e)}return k},cleanData:function(a){for(var b,c,d,e,f=n.event.special,g=0;void 0!==(c=a[g]);g++){if(n.acceptData(c)&&(e=c[L.expando],e&&(b=L.cache[e]))){if(b.events)for(d in b.events)f[d]?n.event.remove(c,d):n.removeEvent(c,d,b.handle);L.cache[e]&&delete L.cache[e]}delete M.cache[c[M.expando]]}}}),n.fn.extend({text:function(a){return J(this,function(a){return void 0===a?n.text(this):this.empty().each(function(){(1===this.nodeType||11===this.nodeType||9===this.nodeType)&&(this.textContent=a)})},null,a,arguments.length)},append:function(){return this.domManip(arguments,function(a){if(1===this.nodeType||11===this.nodeType||9===this.nodeType){var b=jb(this,a);b.appendChild(a)}})},prepend:function(){return this.domManip(arguments,function(a){if(1===this.nodeType||11===this.nodeType||9===this.nodeType){var b=jb(this,a);b.insertBefore(a,b.firstChild)}})},before:function(){return this.domManip(arguments,function(a){this.parentNode&&this.parentNode.insertBefore(a,this)})},after:function(){return this.domManip(arguments,function(a){this.parentNode&&this.parentNode.insertBefore(a,this.nextSibling)})},remove:function(a,b){for(var c,d=a?n.filter(a,this):this,e=0;null!=(c=d[e]);e++)b||1!==c.nodeType||n.cleanData(ob(c)),c.parentNode&&(b&&n.contains(c.ownerDocument,c)&&mb(ob(c,"script")),c.parentNode.removeChild(c));return this},empty:function(){for(var a,b=0;null!=(a=this[b]);b++)1===a.nodeType&&(n.cleanData(ob(a,!1)),a.textContent="");return this},clone:function(a,b){return a=null==a?!1:a,b=null==b?a:b,this.map(function(){return n.clone(this,a,b)})},html:function(a){return J(this,function(a){var b=this[0]||{},c=0,d=this.length;if(void 0===a&&1===b.nodeType)return b.innerHTML;if("string"==typeof a&&!db.test(a)&&!ib[(bb.exec(a)||["",""])[1].toLowerCase()]){a=a.replace(ab,"<$1></$2>");try{for(;d>c;c++)b=this[c]||{},1===b.nodeType&&(n.cleanData(ob(b,!1)),b.innerHTML=a);b=0}catch(e){}}b&&this.empty().append(a)},null,a,arguments.length)},replaceWith:function(){var a=arguments[0];return this.domManip(arguments,function(b){a=this.parentNode,n.cleanData(ob(this)),a&&a.replaceChild(b,this)}),a&&(a.length||a.nodeType)?this:this.remove()},detach:function(a){return this.remove(a,!0)},domManip:function(a,b){a=e.apply([],a);var c,d,f,g,h,i,j=0,l=this.length,m=this,o=l-1,p=a[0],q=n.isFunction(p);if(q||l>1&&"string"==typeof p&&!k.checkClone&&eb.test(p))return this.each(function(c){var d=m.eq(c);q&&(a[0]=p.call(this,c,d.html())),d.domManip(a,b)});if(l&&(c=n.buildFragment(a,this[0].ownerDocument,!1,this),d=c.firstChild,1===c.childNodes.length&&(c=d),d)){for(f=n.map(ob(c,"script"),kb),g=f.length;l>j;j++)h=c,j!==o&&(h=n.clone(h,!0,!0),g&&n.merge(f,ob(h,"script"))),b.call(this[j],h,j);if(g)for(i=f[f.length-1].ownerDocument,n.map(f,lb),j=0;g>j;j++)h=f[j],fb.test(h.type||"")&&!L.access(h,"globalEval")&&n.contains(i,h)&&(h.src?n._evalUrl&&n._evalUrl(h.src):n.globalEval(h.textContent.replace(hb,"")))}return this}}),n.each({appendTo:"append",prependTo:"prepend",insertBefore:"before",insertAfter:"after",replaceAll:"replaceWith"},function(a,b){n.fn[a]=function(a){for(var c,d=[],e=n(a),g=e.length-1,h=0;g>=h;h++)c=h===g?this:this.clone(!0),n(e[h])[b](c),f.apply(d,c.get());return this.pushStack(d)}});var qb,rb={};function sb(b,c){var d,e=n(c.createElement(b)).appendTo(c.body),f=a.getDefaultComputedStyle&&(d=a.getDefaultComputedStyle(e[0]))?d.display:n.css(e[0],"display");return e.detach(),f}function tb(a){var b=l,c=rb[a];return c||(c=sb(a,b),"none"!==c&&c||(qb=(qb||n("<iframe frameborder='0' width='0' height='0'/>")).appendTo(b.documentElement),b=qb[0].contentDocument,b.write(),b.close(),c=sb(a,b),qb.detach()),rb[a]=c),c}var ub=/^margin/,vb=new RegExp("^("+Q+")(?!px)[a-z%]+$","i"),wb=function(a){return a.ownerDocument.defaultView.getComputedStyle(a,null)};function xb(a,b,c){var d,e,f,g,h=a.style;return c=c||wb(a),c&&(g=c.getPropertyValue(b)||c[b]),c&&(""!==g||n.contains(a.ownerDocument,a)||(g=n.style(a,b)),vb.test(g)&&ub.test(b)&&(d=h.width,e=h.minWidth,f=h.maxWidth,h.minWidth=h.maxWidth=h.width=g,g=c.width,h.width=d,h.minWidth=e,h.maxWidth=f)),void 0!==g?g+"":g}function yb(a,b){return{get:function(){return a()?void delete this.get:(this.get=b).apply(this,arguments)}}}!function(){var b,c,d=l.documentElement,e=l.createElement("div"),f=l.createElement("div");if(f.style){f.style.backgroundClip="content-box",f.cloneNode(!0).style.backgroundClip="",k.clearCloneStyle="content-box"===f.style.backgroundClip,e.style.cssText="border:0;width:0;height:0;top:0;left:-9999px;margin-top:1px;position:absolute",e.appendChild(f);function g(){f.style.cssText="-webkit-box-sizing:border-box;-moz-box-sizing:border-box;box-sizing:border-box;display:block;margin-top:1%;top:1%;border:1px;padding:1px;width:4px;position:absolute",f.innerHTML="",d.appendChild(e);var g=a.getComputedStyle(f,null);b="1%"!==g.top,c="4px"===g.width,d.removeChild(e)}a.getComputedStyle&&n.extend(k,{pixelPosition:function(){return g(),b},boxSizingReliable:function(){return null==c&&g(),c},reliableMarginRight:function(){var b,c=f.appendChild(l.createElement("div"));return c.style.cssText=f.style.cssText="-webkit-box-sizing:content-box;-moz-box-sizing:content-box;box-sizing:content-box;display:block;margin:0;border:0;padding:0",c.style.marginRight=c.style.width="0",f.style.width="1px",d.appendChild(e),b=!parseFloat(a.getComputedStyle(c,null).marginRight),d.removeChild(e),b}})}}(),n.swap=function(a,b,c,d){var e,f,g={};for(f in b)g[f]=a.style[f],a.style[f]=b[f];e=c.apply(a,d||[]);for(f in b)a.style[f]=g[f];return e};var zb=/^(none|table(?!-c[ea]).+)/,Ab=new RegExp("^("+Q+")(.*)$","i"),Bb=new RegExp("^([+-])=("+Q+")","i"),Cb={position:"absolute",visibility:"hidden",display:"block"},Db={letterSpacing:"0",fontWeight:"400"},Eb=["Webkit","O","Moz","ms"];function Fb(a,b){if(b in a)return b;var c=b[0].toUpperCase()+b.slice(1),d=b,e=Eb.length;while(e--)if(b=Eb[e]+c,b in a)return b;return d}function Gb(a,b,c){var d=Ab.exec(b);return d?Math.max(0,d[1]-(c||0))+(d[2]||"px"):b}function Hb(a,b,c,d,e){for(var f=c===(d?"border":"content")?4:"width"===b?1:0,g=0;4>f;f+=2)"margin"===c&&(g+=n.css(a,c+R[f],!0,e)),d?("content"===c&&(g-=n.css(a,"padding"+R[f],!0,e)),"margin"!==c&&(g-=n.css(a,"border"+R[f]+"Width",!0,e))):(g+=n.css(a,"padding"+R[f],!0,e),"padding"!==c&&(g+=n.css(a,"border"+R[f]+"Width",!0,e)));return g}function Ib(a,b,c){var d=!0,e="width"===b?a.offsetWidth:a.offsetHeight,f=wb(a),g="border-box"===n.css(a,"boxSizing",!1,f);if(0>=e||null==e){if(e=xb(a,b,f),(0>e||null==e)&&(e=a.style[b]),vb.test(e))return e;d=g&&(k.boxSizingReliable()||e===a.style[b]),e=parseFloat(e)||0}return e+Hb(a,b,c||(g?"border":"content"),d,f)+"px"}function Jb(a,b){for(var c,d,e,f=[],g=0,h=a.length;h>g;g++)d=a[g],d.style&&(f[g]=L.get(d,"olddisplay"),c=d.style.display,b?(f[g]||"none"!==c||(d.style.display=""),""===d.style.display&&S(d)&&(f[g]=L.access(d,"olddisplay",tb(d.nodeName)))):(e=S(d),"none"===c&&e||L.set(d,"olddisplay",e?c:n.css(d,"display"))));for(g=0;h>g;g++)d=a[g],d.style&&(b&&"none"!==d.style.display&&""!==d.style.display||(d.style.display=b?f[g]||"":"none"));return a}n.extend({cssHooks:{opacity:{get:function(a,b){if(b){var c=xb(a,"opacity");return""===c?"1":c}}}},cssNumber:{columnCount:!0,fillOpacity:!0,flexGrow:!0,flexShrink:!0,fontWeight:!0,lineHeight:!0,opacity:!0,order:!0,orphans:!0,widows:!0,zIndex:!0,zoom:!0},cssProps:{"float":"cssFloat"},style:function(a,b,c,d){if(a&&3!==a.nodeType&&8!==a.nodeType&&a.style){var e,f,g,h=n.camelCase(b),i=a.style;return b=n.cssProps[h]||(n.cssProps[h]=Fb(i,h)),g=n.cssHooks[b]||n.cssHooks[h],void 0===c?g&&"get"in g&&void 0!==(e=g.get(a,!1,d))?e:i[b]:(f=typeof c,"string"===f&&(e=Bb.exec(c))&&(c=(e[1]+1)*e[2]+parseFloat(n.css(a,b)),f="number"),null!=c&&c===c&&("number"!==f||n.cssNumber[h]||(c+="px"),k.clearCloneStyle||""!==c||0!==b.indexOf("background")||(i[b]="inherit"),g&&"set"in g&&void 0===(c=g.set(a,c,d))||(i[b]=c)),void 0)}},css:function(a,b,c,d){var e,f,g,h=n.camelCase(b);return b=n.cssProps[h]||(n.cssProps[h]=Fb(a.style,h)),g=n.cssHooks[b]||n.cssHooks[h],g&&"get"in g&&(e=g.get(a,!0,c)),void 0===e&&(e=xb(a,b,d)),"normal"===e&&b in Db&&(e=Db[b]),""===c||c?(f=parseFloat(e),c===!0||n.isNumeric(f)?f||0:e):e}}),n.each(["height","width"],function(a,b){n.cssHooks[b]={get:function(a,c,d){return c?zb.test(n.css(a,"display"))&&0===a.offsetWidth?n.swap(a,Cb,function(){return Ib(a,b,d)}):Ib(a,b,d):void 0},set:function(a,c,d){var e=d&&wb(a);return Gb(a,c,d?Hb(a,b,d,"border-box"===n.css(a,"boxSizing",!1,e),e):0)}}}),n.cssHooks.marginRight=yb(k.reliableMarginRight,function(a,b){return b?n.swap(a,{display:"inline-block"},xb,[a,"marginRight"]):void 0}),n.each({margin:"",padding:"",border:"Width"},function(a,b){n.cssHooks[a+b]={expand:function(c){for(var d=0,e={},f="string"==typeof c?c.split(" "):[c];4>d;d++)e[a+R[d]+b]=f[d]||f[d-2]||f[0];return e}},ub.test(a)||(n.cssHooks[a+b].set=Gb)}),n.fn.extend({css:function(a,b){return J(this,function(a,b,c){var d,e,f={},g=0;if(n.isArray(b)){for(d=wb(a),e=b.length;e>g;g++)f[b[g]]=n.css(a,b[g],!1,d);return f}return void 0!==c?n.style(a,b,c):n.css(a,b)},a,b,arguments.length>1)},show:function(){return Jb(this,!0)},hide:function(){return Jb(this)},toggle:function(a){return"boolean"==typeof a?a?this.show():this.hide():this.each(function(){S(this)?n(this).show():n(this).hide()})}});function Kb(a,b,c,d,e){return new Kb.prototype.init(a,b,c,d,e)}n.Tween=Kb,Kb.prototype={constructor:Kb,init:function(a,b,c,d,e,f){this.elem=a,this.prop=c,this.easing=e||"swing",this.options=b,this.start=this.now=this.cur(),this.end=d,this.unit=f||(n.cssNumber[c]?"":"px")},cur:function(){var a=Kb.propHooks[this.prop];return a&&a.get?a.get(this):Kb.propHooks._default.get(this)},run:function(a){var b,c=Kb.propHooks[this.prop];return this.pos=b=this.options.duration?n.easing[this.easing](a,this.options.duration*a,0,1,this.options.duration):a,this.now=(this.end-this.start)*b+this.start,this.options.step&&this.options.step.call(this.elem,this.now,this),c&&c.set?c.set(this):Kb.propHooks._default.set(this),this}},Kb.prototype.init.prototype=Kb.prototype,Kb.propHooks={_default:{get:function(a){var b;return null==a.elem[a.prop]||a.elem.style&&null!=a.elem.style[a.prop]?(b=n.css(a.elem,a.prop,""),b&&"auto"!==b?b:0):a.elem[a.prop]},set:function(a){n.fx.step[a.prop]?n.fx.step[a.prop](a):a.elem.style&&(null!=a.elem.style[n.cssProps[a.prop]]||n.cssHooks[a.prop])?n.style(a.elem,a.prop,a.now+a.unit):a.elem[a.prop]=a.now}}},Kb.propHooks.scrollTop=Kb.propHooks.scrollLeft={set:function(a){a.elem.nodeType&&a.elem.parentNode&&(a.elem[a.prop]=a.now)}},n.easing={linear:function(a){return a},swing:function(a){return.5-Math.cos(a*Math.PI)/2}},n.fx=Kb.prototype.init,n.fx.step={};var Lb,Mb,Nb=/^(?:toggle|show|hide)$/,Ob=new RegExp("^(?:([+-])=|)("+Q+")([a-z%]*)$","i"),Pb=/queueHooks$/,Qb=[Vb],Rb={"*":[function(a,b){var c=this.createTween(a,b),d=c.cur(),e=Ob.exec(b),f=e&&e[3]||(n.cssNumber[a]?"":"px"),g=(n.cssNumber[a]||"px"!==f&&+d)&&Ob.exec(n.css(c.elem,a)),h=1,i=20;if(g&&g[3]!==f){f=f||g[3],e=e||[],g=+d||1;do h=h||".5",g/=h,n.style(c.elem,a,g+f);while(h!==(h=c.cur()/d)&&1!==h&&--i)}return e&&(g=c.start=+g||+d||0,c.unit=f,c.end=e[1]?g+(e[1]+1)*e[2]:+e[2]),c}]};function Sb(){return setTimeout(function(){Lb=void 0}),Lb=n.now()}function Tb(a,b){var c,d=0,e={height:a};for(b=b?1:0;4>d;d+=2-b)c=R[d],e["margin"+c]=e["padding"+c]=a;return b&&(e.opacity=e.width=a),e}function Ub(a,b,c){for(var d,e=(Rb[b]||[]).concat(Rb["*"]),f=0,g=e.length;g>f;f++)if(d=e[f].call(c,b,a))return d}function Vb(a,b,c){var d,e,f,g,h,i,j,k,l=this,m={},o=a.style,p=a.nodeType&&S(a),q=L.get(a,"fxshow");c.queue||(h=n._queueHooks(a,"fx"),null==h.unqueued&&(h.unqueued=0,i=h.empty.fire,h.empty.fire=function(){h.unqueued||i()}),h.unqueued++,l.always(function(){l.always(function(){h.unqueued--,n.queue(a,"fx").length||h.empty.fire()})})),1===a.nodeType&&("height"in b||"width"in b)&&(c.overflow=[o.overflow,o.overflowX,o.overflowY],j=n.css(a,"display"),k="none"===j?L.get(a,"olddisplay")||tb(a.nodeName):j,"inline"===k&&"none"===n.css(a,"float")&&(o.display="inline-block")),c.overflow&&(o.overflow="hidden",l.always(function(){o.overflow=c.overflow[0],o.overflowX=c.overflow[1],o.overflowY=c.overflow[2]}));for(d in b)if(e=b[d],Nb.exec(e)){if(delete b[d],f=f||"toggle"===e,e===(p?"hide":"show")){if("show"!==e||!q||void 0===q[d])continue;p=!0}m[d]=q&&q[d]||n.style(a,d)}else j=void 0;if(n.isEmptyObject(m))"inline"===("none"===j?tb(a.nodeName):j)&&(o.display=j);else{q?"hidden"in q&&(p=q.hidden):q=L.access(a,"fxshow",{}),f&&(q.hidden=!p),p?n(a).show():l.done(function(){n(a).hide()}),l.done(function(){var b;L.remove(a,"fxshow");for(b in m)n.style(a,b,m[b])});for(d in m)g=Ub(p?q[d]:0,d,l),d in q||(q[d]=g.start,p&&(g.end=g.start,g.start="width"===d||"height"===d?1:0))}}function Wb(a,b){var c,d,e,f,g;for(c in a)if(d=n.camelCase(c),e=b[d],f=a[c],n.isArray(f)&&(e=f[1],f=a[c]=f[0]),c!==d&&(a[d]=f,delete a[c]),g=n.cssHooks[d],g&&"expand"in g){f=g.expand(f),delete a[d];for(c in f)c in a||(a[c]=f[c],b[c]=e)}else b[d]=e}function Xb(a,b,c){var d,e,f=0,g=Qb.length,h=n.Deferred().always(function(){delete i.elem}),i=function(){if(e)return!1;for(var b=Lb||Sb(),c=Math.max(0,j.startTime+j.duration-b),d=c/j.duration||0,f=1-d,g=0,i=j.tweens.length;i>g;g++)j.tweens[g].run(f);return h.notifyWith(a,[j,f,c]),1>f&&i?c:(h.resolveWith(a,[j]),!1)},j=h.promise({elem:a,props:n.extend({},b),opts:n.extend(!0,{specialEasing:{}},c),originalProperties:b,originalOptions:c,startTime:Lb||Sb(),duration:c.duration,tweens:[],createTween:function(b,c){var d=n.Tween(a,j.opts,b,c,j.opts.specialEasing[b]||j.opts.easing);return j.tweens.push(d),d},stop:function(b){var c=0,d=b?j.tweens.length:0;if(e)return this;for(e=!0;d>c;c++)j.tweens[c].run(1);return b?h.resolveWith(a,[j,b]):h.rejectWith(a,[j,b]),this}}),k=j.props;for(Wb(k,j.opts.specialEasing);g>f;f++)if(d=Qb[f].call(j,a,k,j.opts))return d;return n.map(k,Ub,j),n.isFunction(j.opts.start)&&j.opts.start.call(a,j),n.fx.timer(n.extend(i,{elem:a,anim:j,queue:j.opts.queue})),j.progress(j.opts.progress).done(j.opts.done,j.opts.complete).fail(j.opts.fail).always(j.opts.always)}n.Animation=n.extend(Xb,{tweener:function(a,b){n.isFunction(a)?(b=a,a=["*"]):a=a.split(" ");for(var c,d=0,e=a.length;e>d;d++)c=a[d],Rb[c]=Rb[c]||[],Rb[c].unshift(b)},prefilter:function(a,b){b?Qb.unshift(a):Qb.push(a)}}),n.speed=function(a,b,c){var d=a&&"object"==typeof a?n.extend({},a):{complete:c||!c&&b||n.isFunction(a)&&a,duration:a,easing:c&&b||b&&!n.isFunction(b)&&b};return d.duration=n.fx.off?0:"number"==typeof d.duration?d.duration:d.duration in n.fx.speeds?n.fx.speeds[d.duration]:n.fx.speeds._default,(null==d.queue||d.queue===!0)&&(d.queue="fx"),d.old=d.complete,d.complete=function(){n.isFunction(d.old)&&d.old.call(this),d.queue&&n.dequeue(this,d.queue)},d},n.fn.extend({fadeTo:function(a,b,c,d){return this.filter(S).css("opacity",0).show().end().animate({opacity:b},a,c,d)},animate:function(a,b,c,d){var e=n.isEmptyObject(a),f=n.speed(b,c,d),g=function(){var b=Xb(this,n.extend({},a),f);(e||L.get(this,"finish"))&&b.stop(!0)};return g.finish=g,e||f.queue===!1?this.each(g):this.queue(f.queue,g)},stop:function(a,b,c){var d=function(a){var b=a.stop;delete a.stop,b(c)};return"string"!=typeof a&&(c=b,b=a,a=void 0),b&&a!==!1&&this.queue(a||"fx",[]),this.each(function(){var b=!0,e=null!=a&&a+"queueHooks",f=n.timers,g=L.get(this);if(e)g[e]&&g[e].stop&&d(g[e]);else for(e in g)g[e]&&g[e].stop&&Pb.test(e)&&d(g[e]);for(e=f.length;e--;)f[e].elem!==this||null!=a&&f[e].queue!==a||(f[e].anim.stop(c),b=!1,f.splice(e,1));(b||!c)&&n.dequeue(this,a)})},finish:function(a){return a!==!1&&(a=a||"fx"),this.each(function(){var b,c=L.get(this),d=c[a+"queue"],e=c[a+"queueHooks"],f=n.timers,g=d?d.length:0;for(c.finish=!0,n.queue(this,a,[]),e&&e.stop&&e.stop.call(this,!0),b=f.length;b--;)f[b].elem===this&&f[b].queue===a&&(f[b].anim.stop(!0),f.splice(b,1));for(b=0;g>b;b++)d[b]&&d[b].finish&&d[b].finish.call(this);delete c.finish})}}),n.each(["toggle","show","hide"],function(a,b){var c=n.fn[b];n.fn[b]=function(a,d,e){return null==a||"boolean"==typeof a?c.apply(this,arguments):this.animate(Tb(b,!0),a,d,e)}}),n.each({slideDown:Tb("show"),slideUp:Tb("hide"),slideToggle:Tb("toggle"),fadeIn:{opacity:"show"},fadeOut:{opacity:"hide"},fadeToggle:{opacity:"toggle"}},function(a,b){n.fn[a]=function(a,c,d){return this.animate(b,a,c,d)}}),n.timers=[],n.fx.tick=function(){var a,b=0,c=n.timers;for(Lb=n.now();b<c.length;b++)a=c[b],a()||c[b]!==a||c.splice(b--,1);c.length||n.fx.stop(),Lb=void 0},n.fx.timer=function(a){n.timers.push(a),a()?n.fx.start():n.timers.pop()},n.fx.interval=13,n.fx.start=function(){Mb||(Mb=setInterval(n.fx.tick,n.fx.interval))},n.fx.stop=function(){clearInterval(Mb),Mb=null},n.fx.speeds={slow:600,fast:200,_default:400},n.fn.delay=function(a,b){return a=n.fx?n.fx.speeds[a]||a:a,b=b||"fx",this.queue(b,function(b,c){var d=setTimeout(b,a);c.stop=function(){clearTimeout(d)}})},function(){var a=l.createElement("input"),b=l.createElement("select"),c=b.appendChild(l.createElement("option"));a.type="checkbox",k.checkOn=""!==a.value,k.optSelected=c.selected,b.disabled=!0,k.optDisabled=!c.disabled,a=l.createElement("input"),a.value="t",a.type="radio",k.radioValue="t"===a.value}();var Yb,Zb,$b=n.expr.attrHandle;n.fn.extend({attr:function(a,b){return J(this,n.attr,a,b,arguments.length>1)},removeAttr:function(a){return this.each(function(){n.removeAttr(this,a)})}}),n.extend({attr:function(a,b,c){var d,e,f=a.nodeType;if(a&&3!==f&&8!==f&&2!==f)return typeof a.getAttribute===U?n.prop(a,b,c):(1===f&&n.isXMLDoc(a)||(b=b.toLowerCase(),d=n.attrHooks[b]||(n.expr.match.bool.test(b)?Zb:Yb)),void 0===c?d&&"get"in d&&null!==(e=d.get(a,b))?e:(e=n.find.attr(a,b),null==e?void 0:e):null!==c?d&&"set"in d&&void 0!==(e=d.set(a,c,b))?e:(a.setAttribute(b,c+""),c):void n.removeAttr(a,b))
},removeAttr:function(a,b){var c,d,e=0,f=b&&b.match(E);if(f&&1===a.nodeType)while(c=f[e++])d=n.propFix[c]||c,n.expr.match.bool.test(c)&&(a[d]=!1),a.removeAttribute(c)},attrHooks:{type:{set:function(a,b){if(!k.radioValue&&"radio"===b&&n.nodeName(a,"input")){var c=a.value;return a.setAttribute("type",b),c&&(a.value=c),b}}}}}),Zb={set:function(a,b,c){return b===!1?n.removeAttr(a,c):a.setAttribute(c,c),c}},n.each(n.expr.match.bool.source.match(/\w+/g),function(a,b){var c=$b[b]||n.find.attr;$b[b]=function(a,b,d){var e,f;return d||(f=$b[b],$b[b]=e,e=null!=c(a,b,d)?b.toLowerCase():null,$b[b]=f),e}});var _b=/^(?:input|select|textarea|button)$/i;n.fn.extend({prop:function(a,b){return J(this,n.prop,a,b,arguments.length>1)},removeProp:function(a){return this.each(function(){delete this[n.propFix[a]||a]})}}),n.extend({propFix:{"for":"htmlFor","class":"className"},prop:function(a,b,c){var d,e,f,g=a.nodeType;if(a&&3!==g&&8!==g&&2!==g)return f=1!==g||!n.isXMLDoc(a),f&&(b=n.propFix[b]||b,e=n.propHooks[b]),void 0!==c?e&&"set"in e&&void 0!==(d=e.set(a,c,b))?d:a[b]=c:e&&"get"in e&&null!==(d=e.get(a,b))?d:a[b]},propHooks:{tabIndex:{get:function(a){return a.hasAttribute("tabindex")||_b.test(a.nodeName)||a.href?a.tabIndex:-1}}}}),k.optSelected||(n.propHooks.selected={get:function(a){var b=a.parentNode;return b&&b.parentNode&&b.parentNode.selectedIndex,null}}),n.each(["tabIndex","readOnly","maxLength","cellSpacing","cellPadding","rowSpan","colSpan","useMap","frameBorder","contentEditable"],function(){n.propFix[this.toLowerCase()]=this});var ac=/[\t\r\n\f]/g;n.fn.extend({addClass:function(a){var b,c,d,e,f,g,h="string"==typeof a&&a,i=0,j=this.length;if(n.isFunction(a))return this.each(function(b){n(this).addClass(a.call(this,b,this.className))});if(h)for(b=(a||"").match(E)||[];j>i;i++)if(c=this[i],d=1===c.nodeType&&(c.className?(" "+c.className+" ").replace(ac," "):" ")){f=0;while(e=b[f++])d.indexOf(" "+e+" ")<0&&(d+=e+" ");g=n.trim(d),c.className!==g&&(c.className=g)}return this},removeClass:function(a){var b,c,d,e,f,g,h=0===arguments.length||"string"==typeof a&&a,i=0,j=this.length;if(n.isFunction(a))return this.each(function(b){n(this).removeClass(a.call(this,b,this.className))});if(h)for(b=(a||"").match(E)||[];j>i;i++)if(c=this[i],d=1===c.nodeType&&(c.className?(" "+c.className+" ").replace(ac," "):"")){f=0;while(e=b[f++])while(d.indexOf(" "+e+" ")>=0)d=d.replace(" "+e+" "," ");g=a?n.trim(d):"",c.className!==g&&(c.className=g)}return this},toggleClass:function(a,b){var c=typeof a;return"boolean"==typeof b&&"string"===c?b?this.addClass(a):this.removeClass(a):this.each(n.isFunction(a)?function(c){n(this).toggleClass(a.call(this,c,this.className,b),b)}:function(){if("string"===c){var b,d=0,e=n(this),f=a.match(E)||[];while(b=f[d++])e.hasClass(b)?e.removeClass(b):e.addClass(b)}else(c===U||"boolean"===c)&&(this.className&&L.set(this,"__className__",this.className),this.className=this.className||a===!1?"":L.get(this,"__className__")||"")})},hasClass:function(a){for(var b=" "+a+" ",c=0,d=this.length;d>c;c++)if(1===this[c].nodeType&&(" "+this[c].className+" ").replace(ac," ").indexOf(b)>=0)return!0;return!1}});var bc=/\r/g;n.fn.extend({val:function(a){var b,c,d,e=this[0];{if(arguments.length)return d=n.isFunction(a),this.each(function(c){var e;1===this.nodeType&&(e=d?a.call(this,c,n(this).val()):a,null==e?e="":"number"==typeof e?e+="":n.isArray(e)&&(e=n.map(e,function(a){return null==a?"":a+""})),b=n.valHooks[this.type]||n.valHooks[this.nodeName.toLowerCase()],b&&"set"in b&&void 0!==b.set(this,e,"value")||(this.value=e))});if(e)return b=n.valHooks[e.type]||n.valHooks[e.nodeName.toLowerCase()],b&&"get"in b&&void 0!==(c=b.get(e,"value"))?c:(c=e.value,"string"==typeof c?c.replace(bc,""):null==c?"":c)}}}),n.extend({valHooks:{option:{get:function(a){var b=n.find.attr(a,"value");return null!=b?b:n.trim(n.text(a))}},select:{get:function(a){for(var b,c,d=a.options,e=a.selectedIndex,f="select-one"===a.type||0>e,g=f?null:[],h=f?e+1:d.length,i=0>e?h:f?e:0;h>i;i++)if(c=d[i],!(!c.selected&&i!==e||(k.optDisabled?c.disabled:null!==c.getAttribute("disabled"))||c.parentNode.disabled&&n.nodeName(c.parentNode,"optgroup"))){if(b=n(c).val(),f)return b;g.push(b)}return g},set:function(a,b){var c,d,e=a.options,f=n.makeArray(b),g=e.length;while(g--)d=e[g],(d.selected=n.inArray(d.value,f)>=0)&&(c=!0);return c||(a.selectedIndex=-1),f}}}}),n.each(["radio","checkbox"],function(){n.valHooks[this]={set:function(a,b){return n.isArray(b)?a.checked=n.inArray(n(a).val(),b)>=0:void 0}},k.checkOn||(n.valHooks[this].get=function(a){return null===a.getAttribute("value")?"on":a.value})}),n.each("blur focus focusin focusout load resize scroll unload click dblclick mousedown mouseup mousemove mouseover mouseout mouseenter mouseleave change select submit keydown keypress keyup error contextmenu".split(" "),function(a,b){n.fn[b]=function(a,c){return arguments.length>0?this.on(b,null,a,c):this.trigger(b)}}),n.fn.extend({hover:function(a,b){return this.mouseenter(a).mouseleave(b||a)},bind:function(a,b,c){return this.on(a,null,b,c)},unbind:function(a,b){return this.off(a,null,b)},delegate:function(a,b,c,d){return this.on(b,a,c,d)},undelegate:function(a,b,c){return 1===arguments.length?this.off(a,"**"):this.off(b,a||"**",c)}});var cc=n.now(),dc=/\?/;n.parseJSON=function(a){return JSON.parse(a+"")},n.parseXML=function(a){var b,c;if(!a||"string"!=typeof a)return null;try{c=new DOMParser,b=c.parseFromString(a,"text/xml")}catch(d){b=void 0}return(!b||b.getElementsByTagName("parsererror").length)&&n.error("Invalid XML: "+a),b};var ec,fc,gc=/#.*$/,hc=/([?&])_=[^&]*/,ic=/^(.*?):[ \t]*([^\r\n]*)$/gm,jc=/^(?:about|app|app-storage|.+-extension|file|res|widget):$/,kc=/^(?:GET|HEAD)$/,lc=/^\/\//,mc=/^([\w.+-]+:)(?:\/\/(?:[^\/?#]*@|)([^\/?#:]*)(?::(\d+)|)|)/,nc={},oc={},pc="*/".concat("*");try{fc=location.href}catch(qc){fc=l.createElement("a"),fc.href="",fc=fc.href}ec=mc.exec(fc.toLowerCase())||[];function rc(a){return function(b,c){"string"!=typeof b&&(c=b,b="*");var d,e=0,f=b.toLowerCase().match(E)||[];if(n.isFunction(c))while(d=f[e++])"+"===d[0]?(d=d.slice(1)||"*",(a[d]=a[d]||[]).unshift(c)):(a[d]=a[d]||[]).push(c)}}function sc(a,b,c,d){var e={},f=a===oc;function g(h){var i;return e[h]=!0,n.each(a[h]||[],function(a,h){var j=h(b,c,d);return"string"!=typeof j||f||e[j]?f?!(i=j):void 0:(b.dataTypes.unshift(j),g(j),!1)}),i}return g(b.dataTypes[0])||!e["*"]&&g("*")}function tc(a,b){var c,d,e=n.ajaxSettings.flatOptions||{};for(c in b)void 0!==b[c]&&((e[c]?a:d||(d={}))[c]=b[c]);return d&&n.extend(!0,a,d),a}function uc(a,b,c){var d,e,f,g,h=a.contents,i=a.dataTypes;while("*"===i[0])i.shift(),void 0===d&&(d=a.mimeType||b.getResponseHeader("Content-Type"));if(d)for(e in h)if(h[e]&&h[e].test(d)){i.unshift(e);break}if(i[0]in c)f=i[0];else{for(e in c){if(!i[0]||a.converters[e+" "+i[0]]){f=e;break}g||(g=e)}f=f||g}return f?(f!==i[0]&&i.unshift(f),c[f]):void 0}function vc(a,b,c,d){var e,f,g,h,i,j={},k=a.dataTypes.slice();if(k[1])for(g in a.converters)j[g.toLowerCase()]=a.converters[g];f=k.shift();while(f)if(a.responseFields[f]&&(c[a.responseFields[f]]=b),!i&&d&&a.dataFilter&&(b=a.dataFilter(b,a.dataType)),i=f,f=k.shift())if("*"===f)f=i;else if("*"!==i&&i!==f){if(g=j[i+" "+f]||j["* "+f],!g)for(e in j)if(h=e.split(" "),h[1]===f&&(g=j[i+" "+h[0]]||j["* "+h[0]])){g===!0?g=j[e]:j[e]!==!0&&(f=h[0],k.unshift(h[1]));break}if(g!==!0)if(g&&a["throws"])b=g(b);else try{b=g(b)}catch(l){return{state:"parsererror",error:g?l:"No conversion from "+i+" to "+f}}}return{state:"success",data:b}}n.extend({active:0,lastModified:{},etag:{},ajaxSettings:{url:fc,type:"GET",isLocal:jc.test(ec[1]),global:!0,processData:!0,async:!0,contentType:"application/x-www-form-urlencoded; charset=UTF-8",accepts:{"*":pc,text:"text/plain",html:"text/html",xml:"application/xml, text/xml",json:"application/json, text/javascript"},contents:{xml:/xml/,html:/html/,json:/json/},responseFields:{xml:"responseXML",text:"responseText",json:"responseJSON"},converters:{"* text":String,"text html":!0,"text json":n.parseJSON,"text xml":n.parseXML},flatOptions:{url:!0,context:!0}},ajaxSetup:function(a,b){return b?tc(tc(a,n.ajaxSettings),b):tc(n.ajaxSettings,a)},ajaxPrefilter:rc(nc),ajaxTransport:rc(oc),ajax:function(a,b){"object"==typeof a&&(b=a,a=void 0),b=b||{};var c,d,e,f,g,h,i,j,k=n.ajaxSetup({},b),l=k.context||k,m=k.context&&(l.nodeType||l.jquery)?n(l):n.event,o=n.Deferred(),p=n.Callbacks("once memory"),q=k.statusCode||{},r={},s={},t=0,u="canceled",v={readyState:0,getResponseHeader:function(a){var b;if(2===t){if(!f){f={};while(b=ic.exec(e))f[b[1].toLowerCase()]=b[2]}b=f[a.toLowerCase()]}return null==b?null:b},getAllResponseHeaders:function(){return 2===t?e:null},setRequestHeader:function(a,b){var c=a.toLowerCase();return t||(a=s[c]=s[c]||a,r[a]=b),this},overrideMimeType:function(a){return t||(k.mimeType=a),this},statusCode:function(a){var b;if(a)if(2>t)for(b in a)q[b]=[q[b],a[b]];else v.always(a[v.status]);return this},abort:function(a){var b=a||u;return c&&c.abort(b),x(0,b),this}};if(o.promise(v).complete=p.add,v.success=v.done,v.error=v.fail,k.url=((a||k.url||fc)+"").replace(gc,"").replace(lc,ec[1]+"//"),k.type=b.method||b.type||k.method||k.type,k.dataTypes=n.trim(k.dataType||"*").toLowerCase().match(E)||[""],null==k.crossDomain&&(h=mc.exec(k.url.toLowerCase()),k.crossDomain=!(!h||h[1]===ec[1]&&h[2]===ec[2]&&(h[3]||("http:"===h[1]?"80":"443"))===(ec[3]||("http:"===ec[1]?"80":"443")))),k.data&&k.processData&&"string"!=typeof k.data&&(k.data=n.param(k.data,k.traditional)),sc(nc,k,b,v),2===t)return v;i=k.global,i&&0===n.active++&&n.event.trigger("ajaxStart"),k.type=k.type.toUpperCase(),k.hasContent=!kc.test(k.type),d=k.url,k.hasContent||(k.data&&(d=k.url+=(dc.test(d)?"&":"?")+k.data,delete k.data),k.cache===!1&&(k.url=hc.test(d)?d.replace(hc,"$1_="+cc++):d+(dc.test(d)?"&":"?")+"_="+cc++)),k.ifModified&&(n.lastModified[d]&&v.setRequestHeader("If-Modified-Since",n.lastModified[d]),n.etag[d]&&v.setRequestHeader("If-None-Match",n.etag[d])),(k.data&&k.hasContent&&k.contentType!==!1||b.contentType)&&v.setRequestHeader("Content-Type",k.contentType),v.setRequestHeader("Accept",k.dataTypes[0]&&k.accepts[k.dataTypes[0]]?k.accepts[k.dataTypes[0]]+("*"!==k.dataTypes[0]?", "+pc+"; q=0.01":""):k.accepts["*"]);for(j in k.headers)v.setRequestHeader(j,k.headers[j]);if(k.beforeSend&&(k.beforeSend.call(l,v,k)===!1||2===t))return v.abort();u="abort";for(j in{success:1,error:1,complete:1})v[j](k[j]);if(c=sc(oc,k,b,v)){v.readyState=1,i&&m.trigger("ajaxSend",[v,k]),k.async&&k.timeout>0&&(g=setTimeout(function(){v.abort("timeout")},k.timeout));try{t=1,c.send(r,x)}catch(w){if(!(2>t))throw w;x(-1,w)}}else x(-1,"No Transport");function x(a,b,f,h){var j,r,s,u,w,x=b;2!==t&&(t=2,g&&clearTimeout(g),c=void 0,e=h||"",v.readyState=a>0?4:0,j=a>=200&&300>a||304===a,f&&(u=uc(k,v,f)),u=vc(k,u,v,j),j?(k.ifModified&&(w=v.getResponseHeader("Last-Modified"),w&&(n.lastModified[d]=w),w=v.getResponseHeader("etag"),w&&(n.etag[d]=w)),204===a||"HEAD"===k.type?x="nocontent":304===a?x="notmodified":(x=u.state,r=u.data,s=u.error,j=!s)):(s=x,(a||!x)&&(x="error",0>a&&(a=0))),v.status=a,v.statusText=(b||x)+"",j?o.resolveWith(l,[r,x,v]):o.rejectWith(l,[v,x,s]),v.statusCode(q),q=void 0,i&&m.trigger(j?"ajaxSuccess":"ajaxError",[v,k,j?r:s]),p.fireWith(l,[v,x]),i&&(m.trigger("ajaxComplete",[v,k]),--n.active||n.event.trigger("ajaxStop")))}return v},getJSON:function(a,b,c){return n.get(a,b,c,"json")},getScript:function(a,b){return n.get(a,void 0,b,"script")}}),n.each(["get","post"],function(a,b){n[b]=function(a,c,d,e){return n.isFunction(c)&&(e=e||d,d=c,c=void 0),n.ajax({url:a,type:b,dataType:e,data:c,success:d})}}),n.each(["ajaxStart","ajaxStop","ajaxComplete","ajaxError","ajaxSuccess","ajaxSend"],function(a,b){n.fn[b]=function(a){return this.on(b,a)}}),n._evalUrl=function(a){return n.ajax({url:a,type:"GET",dataType:"script",async:!1,global:!1,"throws":!0})},n.fn.extend({wrapAll:function(a){var b;return n.isFunction(a)?this.each(function(b){n(this).wrapAll(a.call(this,b))}):(this[0]&&(b=n(a,this[0].ownerDocument).eq(0).clone(!0),this[0].parentNode&&b.insertBefore(this[0]),b.map(function(){var a=this;while(a.firstElementChild)a=a.firstElementChild;return a}).append(this)),this)},wrapInner:function(a){return this.each(n.isFunction(a)?function(b){n(this).wrapInner(a.call(this,b))}:function(){var b=n(this),c=b.contents();c.length?c.wrapAll(a):b.append(a)})},wrap:function(a){var b=n.isFunction(a);return this.each(function(c){n(this).wrapAll(b?a.call(this,c):a)})},unwrap:function(){return this.parent().each(function(){n.nodeName(this,"body")||n(this).replaceWith(this.childNodes)}).end()}}),n.expr.filters.hidden=function(a){return a.offsetWidth<=0&&a.offsetHeight<=0},n.expr.filters.visible=function(a){return!n.expr.filters.hidden(a)};var wc=/%20/g,xc=/\[\]$/,yc=/\r?\n/g,zc=/^(?:submit|button|image|reset|file)$/i,Ac=/^(?:input|select|textarea|keygen)/i;function Bc(a,b,c,d){var e;if(n.isArray(b))n.each(b,function(b,e){c||xc.test(a)?d(a,e):Bc(a+"["+("object"==typeof e?b:"")+"]",e,c,d)});else if(c||"object"!==n.type(b))d(a,b);else for(e in b)Bc(a+"["+e+"]",b[e],c,d)}n.param=function(a,b){var c,d=[],e=function(a,b){b=n.isFunction(b)?b():null==b?"":b,d[d.length]=encodeURIComponent(a)+"="+encodeURIComponent(b)};if(void 0===b&&(b=n.ajaxSettings&&n.ajaxSettings.traditional),n.isArray(a)||a.jquery&&!n.isPlainObject(a))n.each(a,function(){e(this.name,this.value)});else for(c in a)Bc(c,a[c],b,e);return d.join("&").replace(wc,"+")},n.fn.extend({serialize:function(){return n.param(this.serializeArray())},serializeArray:function(){return this.map(function(){var a=n.prop(this,"elements");return a?n.makeArray(a):this}).filter(function(){var a=this.type;return this.name&&!n(this).is(":disabled")&&Ac.test(this.nodeName)&&!zc.test(a)&&(this.checked||!T.test(a))}).map(function(a,b){var c=n(this).val();return null==c?null:n.isArray(c)?n.map(c,function(a){return{name:b.name,value:a.replace(yc,"\r\n")}}):{name:b.name,value:c.replace(yc,"\r\n")}}).get()}}),n.ajaxSettings.xhr=function(){try{return new XMLHttpRequest}catch(a){}};var Cc=0,Dc={},Ec={0:200,1223:204},Fc=n.ajaxSettings.xhr();a.ActiveXObject&&n(a).on("unload",function(){for(var a in Dc)Dc[a]()}),k.cors=!!Fc&&"withCredentials"in Fc,k.ajax=Fc=!!Fc,n.ajaxTransport(function(a){var b;return k.cors||Fc&&!a.crossDomain?{send:function(c,d){var e,f=a.xhr(),g=++Cc;if(f.open(a.type,a.url,a.async,a.username,a.password),a.xhrFields)for(e in a.xhrFields)f[e]=a.xhrFields[e];a.mimeType&&f.overrideMimeType&&f.overrideMimeType(a.mimeType),a.crossDomain||c["X-Requested-With"]||(c["X-Requested-With"]="XMLHttpRequest");for(e in c)f.setRequestHeader(e,c[e]);b=function(a){return function(){b&&(delete Dc[g],b=f.onload=f.onerror=null,"abort"===a?f.abort():"error"===a?d(f.status,f.statusText):d(Ec[f.status]||f.status,f.statusText,"string"==typeof f.responseText?{text:f.responseText}:void 0,f.getAllResponseHeaders()))}},f.onload=b(),f.onerror=b("error"),b=Dc[g]=b("abort");try{f.send(a.hasContent&&a.data||null)}catch(h){if(b)throw h}},abort:function(){b&&b()}}:void 0}),n.ajaxSetup({accepts:{script:"text/javascript, application/javascript, application/ecmascript, application/x-ecmascript"},contents:{script:/(?:java|ecma)script/},converters:{"text script":function(a){return n.globalEval(a),a}}}),n.ajaxPrefilter("script",function(a){void 0===a.cache&&(a.cache=!1),a.crossDomain&&(a.type="GET")}),n.ajaxTransport("script",function(a){if(a.crossDomain){var b,c;return{send:function(d,e){b=n("<script>").prop({async:!0,charset:a.scriptCharset,src:a.url}).on("load error",c=function(a){b.remove(),c=null,a&&e("error"===a.type?404:200,a.type)}),l.head.appendChild(b[0])},abort:function(){c&&c()}}}});var Gc=[],Hc=/(=)\?(?=&|$)|\?\?/;n.ajaxSetup({jsonp:"callback",jsonpCallback:function(){var a=Gc.pop()||n.expando+"_"+cc++;return this[a]=!0,a}}),n.ajaxPrefilter("json jsonp",function(b,c,d){var e,f,g,h=b.jsonp!==!1&&(Hc.test(b.url)?"url":"string"==typeof b.data&&!(b.contentType||"").indexOf("application/x-www-form-urlencoded")&&Hc.test(b.data)&&"data");return h||"jsonp"===b.dataTypes[0]?(e=b.jsonpCallback=n.isFunction(b.jsonpCallback)?b.jsonpCallback():b.jsonpCallback,h?b[h]=b[h].replace(Hc,"$1"+e):b.jsonp!==!1&&(b.url+=(dc.test(b.url)?"&":"?")+b.jsonp+"="+e),b.converters["script json"]=function(){return g||n.error(e+" was not called"),g[0]},b.dataTypes[0]="json",f=a[e],a[e]=function(){g=arguments},d.always(function(){a[e]=f,b[e]&&(b.jsonpCallback=c.jsonpCallback,Gc.push(e)),g&&n.isFunction(f)&&f(g[0]),g=f=void 0}),"script"):void 0}),n.parseHTML=function(a,b,c){if(!a||"string"!=typeof a)return null;"boolean"==typeof b&&(c=b,b=!1),b=b||l;var d=v.exec(a),e=!c&&[];return d?[b.createElement(d[1])]:(d=n.buildFragment([a],b,e),e&&e.length&&n(e).remove(),n.merge([],d.childNodes))};var Ic=n.fn.load;n.fn.load=function(a,b,c){if("string"!=typeof a&&Ic)return Ic.apply(this,arguments);var d,e,f,g=this,h=a.indexOf(" ");return h>=0&&(d=n.trim(a.slice(h)),a=a.slice(0,h)),n.isFunction(b)?(c=b,b=void 0):b&&"object"==typeof b&&(e="POST"),g.length>0&&n.ajax({url:a,type:e,dataType:"html",data:b}).done(function(a){f=arguments,g.html(d?n("<div>").append(n.parseHTML(a)).find(d):a)}).complete(c&&function(a,b){g.each(c,f||[a.responseText,b,a])}),this},n.expr.filters.animated=function(a){return n.grep(n.timers,function(b){return a===b.elem}).length};var Jc=a.document.documentElement;function Kc(a){return n.isWindow(a)?a:9===a.nodeType&&a.defaultView}n.offset={setOffset:function(a,b,c){var d,e,f,g,h,i,j,k=n.css(a,"position"),l=n(a),m={};"static"===k&&(a.style.position="relative"),h=l.offset(),f=n.css(a,"top"),i=n.css(a,"left"),j=("absolute"===k||"fixed"===k)&&(f+i).indexOf("auto")>-1,j?(d=l.position(),g=d.top,e=d.left):(g=parseFloat(f)||0,e=parseFloat(i)||0),n.isFunction(b)&&(b=b.call(a,c,h)),null!=b.top&&(m.top=b.top-h.top+g),null!=b.left&&(m.left=b.left-h.left+e),"using"in b?b.using.call(a,m):l.css(m)}},n.fn.extend({offset:function(a){if(arguments.length)return void 0===a?this:this.each(function(b){n.offset.setOffset(this,a,b)});var b,c,d=this[0],e={top:0,left:0},f=d&&d.ownerDocument;if(f)return b=f.documentElement,n.contains(b,d)?(typeof d.getBoundingClientRect!==U&&(e=d.getBoundingClientRect()),c=Kc(f),{top:e.top+c.pageYOffset-b.clientTop,left:e.left+c.pageXOffset-b.clientLeft}):e},position:function(){if(this[0]){var a,b,c=this[0],d={top:0,left:0};return"fixed"===n.css(c,"position")?b=c.getBoundingClientRect():(a=this.offsetParent(),b=this.offset(),n.nodeName(a[0],"html")||(d=a.offset()),d.top+=n.css(a[0],"borderTopWidth",!0),d.left+=n.css(a[0],"borderLeftWidth",!0)),{top:b.top-d.top-n.css(c,"marginTop",!0),left:b.left-d.left-n.css(c,"marginLeft",!0)}}},offsetParent:function(){return this.map(function(){var a=this.offsetParent||Jc;while(a&&!n.nodeName(a,"html")&&"static"===n.css(a,"position"))a=a.offsetParent;return a||Jc})}}),n.each({scrollLeft:"pageXOffset",scrollTop:"pageYOffset"},function(b,c){var d="pageYOffset"===c;n.fn[b]=function(e){return J(this,function(b,e,f){var g=Kc(b);return void 0===f?g?g[c]:b[e]:void(g?g.scrollTo(d?a.pageXOffset:f,d?f:a.pageYOffset):b[e]=f)},b,e,arguments.length,null)}}),n.each(["top","left"],function(a,b){n.cssHooks[b]=yb(k.pixelPosition,function(a,c){return c?(c=xb(a,b),vb.test(c)?n(a).position()[b]+"px":c):void 0})}),n.each({Height:"height",Width:"width"},function(a,b){n.each({padding:"inner"+a,content:b,"":"outer"+a},function(c,d){n.fn[d]=function(d,e){var f=arguments.length&&(c||"boolean"!=typeof d),g=c||(d===!0||e===!0?"margin":"border");return J(this,function(b,c,d){var e;return n.isWindow(b)?b.document.documentElement["client"+a]:9===b.nodeType?(e=b.documentElement,Math.max(b.body["scroll"+a],e["scroll"+a],b.body["offset"+a],e["offset"+a],e["client"+a])):void 0===d?n.css(b,c,g):n.style(b,c,d,g)},b,f?d:void 0,f,null)}})}),n.fn.size=function(){return this.length},n.fn.andSelf=n.fn.addBack,"function"==typeof define&&define.amd&&define("jquery",[],function(){return n});var Lc=a.jQuery,Mc=a.$;return n.noConflict=function(b){return a.$===n&&(a.$=Mc),b&&a.jQuery===n&&(a.jQuery=Lc),n},typeof b===U&&(a.jQuery=a.$=n),n});
//# sourceMappingURL=jquery.min.map</script>
    <script>/*!
 * Bootstrap v3.3.1 (http://getbootstrap.com)
 * Copyright 2011-2014 Twitter, Inc.
 * Licensed under MIT (https://github.com/twbs/bootstrap/blob/master/LICENSE)
 */
if("undefined"==typeof jQuery)throw new Error("Bootstrap's JavaScript requires jQuery");+function(a){var b=a.fn.jquery.split(" ")[0].split(".");if(b[0]<2&&b[1]<9||1==b[0]&&9==b[1]&&b[2]<1)throw new Error("Bootstrap's JavaScript requires jQuery version 1.9.1 or higher")}(jQuery),+function(a){"use strict";function b(){var a=document.createElement("bootstrap"),b={WebkitTransition:"webkitTransitionEnd",MozTransition:"transitionend",OTransition:"oTransitionEnd otransitionend",transition:"transitionend"};for(var c in b)if(void 0!==a.style[c])return{end:b[c]};return!1}a.fn.emulateTransitionEnd=function(b){var c=!1,d=this;a(this).one("bsTransitionEnd",function(){c=!0});var e=function(){c||a(d).trigger(a.support.transition.end)};return setTimeout(e,b),this},a(function(){a.support.transition=b(),a.support.transition&&(a.event.special.bsTransitionEnd={bindType:a.support.transition.end,delegateType:a.support.transition.end,handle:function(b){return a(b.target).is(this)?b.handleObj.handler.apply(this,arguments):void 0}})})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var c=a(this),e=c.data("bs.alert");e||c.data("bs.alert",e=new d(this)),"string"==typeof b&&e[b].call(c)})}var c='[data-dismiss="alert"]',d=function(b){a(b).on("click",c,this.close)};d.VERSION="3.3.1",d.TRANSITION_DURATION=150,d.prototype.close=function(b){function c(){g.detach().trigger("closed.bs.alert").remove()}var e=a(this),f=e.attr("data-target");f||(f=e.attr("href"),f=f&&f.replace(/.*(?=#[^\s]*$)/,""));var g=a(f);b&&b.preventDefault(),g.length||(g=e.closest(".alert")),g.trigger(b=a.Event("close.bs.alert")),b.isDefaultPrevented()||(g.removeClass("in"),a.support.transition&&g.hasClass("fade")?g.one("bsTransitionEnd",c).emulateTransitionEnd(d.TRANSITION_DURATION):c())};var e=a.fn.alert;a.fn.alert=b,a.fn.alert.Constructor=d,a.fn.alert.noConflict=function(){return a.fn.alert=e,this},a(document).on("click.bs.alert.data-api",c,d.prototype.close)}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.button"),f="object"==typeof b&&b;e||d.data("bs.button",e=new c(this,f)),"toggle"==b?e.toggle():b&&e.setState(b)})}var c=function(b,d){this.$element=a(b),this.options=a.extend({},c.DEFAULTS,d),this.isLoading=!1};c.VERSION="3.3.1",c.DEFAULTS={loadingText:"loading..."},c.prototype.setState=function(b){var c="disabled",d=this.$element,e=d.is("input")?"val":"html",f=d.data();b+="Text",null==f.resetText&&d.data("resetText",d[e]()),setTimeout(a.proxy(function(){d[e](null==f[b]?this.options[b]:f[b]),"loadingText"==b?(this.isLoading=!0,d.addClass(c).attr(c,c)):this.isLoading&&(this.isLoading=!1,d.removeClass(c).removeAttr(c))},this),0)},c.prototype.toggle=function(){var a=!0,b=this.$element.closest('[data-toggle="buttons"]');if(b.length){var c=this.$element.find("input");"radio"==c.prop("type")&&(c.prop("checked")&&this.$element.hasClass("active")?a=!1:b.find(".active").removeClass("active")),a&&c.prop("checked",!this.$element.hasClass("active")).trigger("change")}else this.$element.attr("aria-pressed",!this.$element.hasClass("active"));a&&this.$element.toggleClass("active")};var d=a.fn.button;a.fn.button=b,a.fn.button.Constructor=c,a.fn.button.noConflict=function(){return a.fn.button=d,this},a(document).on("click.bs.button.data-api",'[data-toggle^="button"]',function(c){var d=a(c.target);d.hasClass("btn")||(d=d.closest(".btn")),b.call(d,"toggle"),c.preventDefault()}).on("focus.bs.button.data-api blur.bs.button.data-api",'[data-toggle^="button"]',function(b){a(b.target).closest(".btn").toggleClass("focus",/^focus(in)?$/.test(b.type))})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.carousel"),f=a.extend({},c.DEFAULTS,d.data(),"object"==typeof b&&b),g="string"==typeof b?b:f.slide;e||d.data("bs.carousel",e=new c(this,f)),"number"==typeof b?e.to(b):g?e[g]():f.interval&&e.pause().cycle()})}var c=function(b,c){this.$element=a(b),this.$indicators=this.$element.find(".carousel-indicators"),this.options=c,this.paused=this.sliding=this.interval=this.$active=this.$items=null,this.options.keyboard&&this.$element.on("keydown.bs.carousel",a.proxy(this.keydown,this)),"hover"==this.options.pause&&!("ontouchstart"in document.documentElement)&&this.$element.on("mouseenter.bs.carousel",a.proxy(this.pause,this)).on("mouseleave.bs.carousel",a.proxy(this.cycle,this))};c.VERSION="3.3.1",c.TRANSITION_DURATION=600,c.DEFAULTS={interval:5e3,pause:"hover",wrap:!0,keyboard:!0},c.prototype.keydown=function(a){if(!/input|textarea/i.test(a.target.tagName)){switch(a.which){case 37:this.prev();break;case 39:this.next();break;default:return}a.preventDefault()}},c.prototype.cycle=function(b){return b||(this.paused=!1),this.interval&&clearInterval(this.interval),this.options.interval&&!this.paused&&(this.interval=setInterval(a.proxy(this.next,this),this.options.interval)),this},c.prototype.getItemIndex=function(a){return this.$items=a.parent().children(".item"),this.$items.index(a||this.$active)},c.prototype.getItemForDirection=function(a,b){var c="prev"==a?-1:1,d=this.getItemIndex(b),e=(d+c)%this.$items.length;return this.$items.eq(e)},c.prototype.to=function(a){var b=this,c=this.getItemIndex(this.$active=this.$element.find(".item.active"));return a>this.$items.length-1||0>a?void 0:this.sliding?this.$element.one("slid.bs.carousel",function(){b.to(a)}):c==a?this.pause().cycle():this.slide(a>c?"next":"prev",this.$items.eq(a))},c.prototype.pause=function(b){return b||(this.paused=!0),this.$element.find(".next, .prev").length&&a.support.transition&&(this.$element.trigger(a.support.transition.end),this.cycle(!0)),this.interval=clearInterval(this.interval),this},c.prototype.next=function(){return this.sliding?void 0:this.slide("next")},c.prototype.prev=function(){return this.sliding?void 0:this.slide("prev")},c.prototype.slide=function(b,d){var e=this.$element.find(".item.active"),f=d||this.getItemForDirection(b,e),g=this.interval,h="next"==b?"left":"right",i="next"==b?"first":"last",j=this;if(!f.length){if(!this.options.wrap)return;f=this.$element.find(".item")[i]()}if(f.hasClass("active"))return this.sliding=!1;var k=f[0],l=a.Event("slide.bs.carousel",{relatedTarget:k,direction:h});if(this.$element.trigger(l),!l.isDefaultPrevented()){if(this.sliding=!0,g&&this.pause(),this.$indicators.length){this.$indicators.find(".active").removeClass("active");var m=a(this.$indicators.children()[this.getItemIndex(f)]);m&&m.addClass("active")}var n=a.Event("slid.bs.carousel",{relatedTarget:k,direction:h});return a.support.transition&&this.$element.hasClass("slide")?(f.addClass(b),f[0].offsetWidth,e.addClass(h),f.addClass(h),e.one("bsTransitionEnd",function(){f.removeClass([b,h].join(" ")).addClass("active"),e.removeClass(["active",h].join(" ")),j.sliding=!1,setTimeout(function(){j.$element.trigger(n)},0)}).emulateTransitionEnd(c.TRANSITION_DURATION)):(e.removeClass("active"),f.addClass("active"),this.sliding=!1,this.$element.trigger(n)),g&&this.cycle(),this}};var d=a.fn.carousel;a.fn.carousel=b,a.fn.carousel.Constructor=c,a.fn.carousel.noConflict=function(){return a.fn.carousel=d,this};var e=function(c){var d,e=a(this),f=a(e.attr("data-target")||(d=e.attr("href"))&&d.replace(/.*(?=#[^\s]+$)/,""));if(f.hasClass("carousel")){var g=a.extend({},f.data(),e.data()),h=e.attr("data-slide-to");h&&(g.interval=!1),b.call(f,g),h&&f.data("bs.carousel").to(h),c.preventDefault()}};a(document).on("click.bs.carousel.data-api","[data-slide]",e).on("click.bs.carousel.data-api","[data-slide-to]",e),a(window).on("load",function(){a('[data-ride="carousel"]').each(function(){var c=a(this);b.call(c,c.data())})})}(jQuery),+function(a){"use strict";function b(b){var c,d=b.attr("data-target")||(c=b.attr("href"))&&c.replace(/.*(?=#[^\s]+$)/,"");return a(d)}function c(b){return this.each(function(){var c=a(this),e=c.data("bs.collapse"),f=a.extend({},d.DEFAULTS,c.data(),"object"==typeof b&&b);!e&&f.toggle&&"show"==b&&(f.toggle=!1),e||c.data("bs.collapse",e=new d(this,f)),"string"==typeof b&&e[b]()})}var d=function(b,c){this.$element=a(b),this.options=a.extend({},d.DEFAULTS,c),this.$trigger=a(this.options.trigger).filter('[href="#'+b.id+'"], [data-target="#'+b.id+'"]'),this.transitioning=null,this.options.parent?this.$parent=this.getParent():this.addAriaAndCollapsedClass(this.$element,this.$trigger),this.options.toggle&&this.toggle()};d.VERSION="3.3.1",d.TRANSITION_DURATION=350,d.DEFAULTS={toggle:!0,trigger:'[data-toggle="collapse"]'},d.prototype.dimension=function(){var a=this.$element.hasClass("width");return a?"width":"height"},d.prototype.show=function(){if(!this.transitioning&&!this.$element.hasClass("in")){var b,e=this.$parent&&this.$parent.find("> .panel").children(".in, .collapsing");if(!(e&&e.length&&(b=e.data("bs.collapse"),b&&b.transitioning))){var f=a.Event("show.bs.collapse");if(this.$element.trigger(f),!f.isDefaultPrevented()){e&&e.length&&(c.call(e,"hide"),b||e.data("bs.collapse",null));var g=this.dimension();this.$element.removeClass("collapse").addClass("collapsing")[g](0).attr("aria-expanded",!0),this.$trigger.removeClass("collapsed").attr("aria-expanded",!0),this.transitioning=1;var h=function(){this.$element.removeClass("collapsing").addClass("collapse in")[g](""),this.transitioning=0,this.$element.trigger("shown.bs.collapse")};if(!a.support.transition)return h.call(this);var i=a.camelCase(["scroll",g].join("-"));this.$element.one("bsTransitionEnd",a.proxy(h,this)).emulateTransitionEnd(d.TRANSITION_DURATION)[g](this.$element[0][i])}}}},d.prototype.hide=function(){if(!this.transitioning&&this.$element.hasClass("in")){var b=a.Event("hide.bs.collapse");if(this.$element.trigger(b),!b.isDefaultPrevented()){var c=this.dimension();this.$element[c](this.$element[c]())[0].offsetHeight,this.$element.addClass("collapsing").removeClass("collapse in").attr("aria-expanded",!1),this.$trigger.addClass("collapsed").attr("aria-expanded",!1),this.transitioning=1;var e=function(){this.transitioning=0,this.$element.removeClass("collapsing").addClass("collapse").trigger("hidden.bs.collapse")};return a.support.transition?void this.$element[c](0).one("bsTransitionEnd",a.proxy(e,this)).emulateTransitionEnd(d.TRANSITION_DURATION):e.call(this)}}},d.prototype.toggle=function(){this[this.$element.hasClass("in")?"hide":"show"]()},d.prototype.getParent=function(){return a(this.options.parent).find('[data-toggle="collapse"][data-parent="'+this.options.parent+'"]').each(a.proxy(function(c,d){var e=a(d);this.addAriaAndCollapsedClass(b(e),e)},this)).end()},d.prototype.addAriaAndCollapsedClass=function(a,b){var c=a.hasClass("in");a.attr("aria-expanded",c),b.toggleClass("collapsed",!c).attr("aria-expanded",c)};var e=a.fn.collapse;a.fn.collapse=c,a.fn.collapse.Constructor=d,a.fn.collapse.noConflict=function(){return a.fn.collapse=e,this},a(document).on("click.bs.collapse.data-api",'[data-toggle="collapse"]',function(d){var e=a(this);e.attr("data-target")||d.preventDefault();var f=b(e),g=f.data("bs.collapse"),h=g?"toggle":a.extend({},e.data(),{trigger:this});c.call(f,h)})}(jQuery),+function(a){"use strict";function b(b){b&&3===b.which||(a(e).remove(),a(f).each(function(){var d=a(this),e=c(d),f={relatedTarget:this};e.hasClass("open")&&(e.trigger(b=a.Event("hide.bs.dropdown",f)),b.isDefaultPrevented()||(d.attr("aria-expanded","false"),e.removeClass("open").trigger("hidden.bs.dropdown",f)))}))}function c(b){var c=b.attr("data-target");c||(c=b.attr("href"),c=c&&/#[A-Za-z]/.test(c)&&c.replace(/.*(?=#[^\s]*$)/,""));var d=c&&a(c);return d&&d.length?d:b.parent()}function d(b){return this.each(function(){var c=a(this),d=c.data("bs.dropdown");d||c.data("bs.dropdown",d=new g(this)),"string"==typeof b&&d[b].call(c)})}var e=".dropdown-backdrop",f='[data-toggle="dropdown"]',g=function(b){a(b).on("click.bs.dropdown",this.toggle)};g.VERSION="3.3.1",g.prototype.toggle=function(d){var e=a(this);if(!e.is(".disabled, :disabled")){var f=c(e),g=f.hasClass("open");if(b(),!g){"ontouchstart"in document.documentElement&&!f.closest(".navbar-nav").length&&a('<div class="dropdown-backdrop"/>').insertAfter(a(this)).on("click",b);var h={relatedTarget:this};if(f.trigger(d=a.Event("show.bs.dropdown",h)),d.isDefaultPrevented())return;e.trigger("focus").attr("aria-expanded","true"),f.toggleClass("open").trigger("shown.bs.dropdown",h)}return!1}},g.prototype.keydown=function(b){if(/(38|40|27|32)/.test(b.which)&&!/input|textarea/i.test(b.target.tagName)){var d=a(this);if(b.preventDefault(),b.stopPropagation(),!d.is(".disabled, :disabled")){var e=c(d),g=e.hasClass("open");if(!g&&27!=b.which||g&&27==b.which)return 27==b.which&&e.find(f).trigger("focus"),d.trigger("click");var h=" li:not(.divider):visible a",i=e.find('[role="menu"]'+h+', [role="listbox"]'+h);if(i.length){var j=i.index(b.target);38==b.which&&j>0&&j--,40==b.which&&j<i.length-1&&j++,~j||(j=0),i.eq(j).trigger("focus")}}}};var h=a.fn.dropdown;a.fn.dropdown=d,a.fn.dropdown.Constructor=g,a.fn.dropdown.noConflict=function(){return a.fn.dropdown=h,this},a(document).on("click.bs.dropdown.data-api",b).on("click.bs.dropdown.data-api",".dropdown form",function(a){a.stopPropagation()}).on("click.bs.dropdown.data-api",f,g.prototype.toggle).on("keydown.bs.dropdown.data-api",f,g.prototype.keydown).on("keydown.bs.dropdown.data-api",'[role="menu"]',g.prototype.keydown).on("keydown.bs.dropdown.data-api",'[role="listbox"]',g.prototype.keydown)}(jQuery),+function(a){"use strict";function b(b,d){return this.each(function(){var e=a(this),f=e.data("bs.modal"),g=a.extend({},c.DEFAULTS,e.data(),"object"==typeof b&&b);f||e.data("bs.modal",f=new c(this,g)),"string"==typeof b?f[b](d):g.show&&f.show(d)})}var c=function(b,c){this.options=c,this.$body=a(document.body),this.$element=a(b),this.$backdrop=this.isShown=null,this.scrollbarWidth=0,this.options.remote&&this.$element.find(".modal-content").load(this.options.remote,a.proxy(function(){this.$element.trigger("loaded.bs.modal")},this))};c.VERSION="3.3.1",c.TRANSITION_DURATION=300,c.BACKDROP_TRANSITION_DURATION=150,c.DEFAULTS={backdrop:!0,keyboard:!0,show:!0},c.prototype.toggle=function(a){return this.isShown?this.hide():this.show(a)},c.prototype.show=function(b){var d=this,e=a.Event("show.bs.modal",{relatedTarget:b});this.$element.trigger(e),this.isShown||e.isDefaultPrevented()||(this.isShown=!0,this.checkScrollbar(),this.setScrollbar(),this.$body.addClass("modal-open"),this.escape(),this.resize(),this.$element.on("click.dismiss.bs.modal",'[data-dismiss="modal"]',a.proxy(this.hide,this)),this.backdrop(function(){var e=a.support.transition&&d.$element.hasClass("fade");d.$element.parent().length||d.$element.appendTo(d.$body),d.$element.show().scrollTop(0),d.options.backdrop&&d.adjustBackdrop(),d.adjustDialog(),e&&d.$element[0].offsetWidth,d.$element.addClass("in").attr("aria-hidden",!1),d.enforceFocus();var f=a.Event("shown.bs.modal",{relatedTarget:b});e?d.$element.find(".modal-dialog").one("bsTransitionEnd",function(){d.$element.trigger("focus").trigger(f)}).emulateTransitionEnd(c.TRANSITION_DURATION):d.$element.trigger("focus").trigger(f)}))},c.prototype.hide=function(b){b&&b.preventDefault(),b=a.Event("hide.bs.modal"),this.$element.trigger(b),this.isShown&&!b.isDefaultPrevented()&&(this.isShown=!1,this.escape(),this.resize(),a(document).off("focusin.bs.modal"),this.$element.removeClass("in").attr("aria-hidden",!0).off("click.dismiss.bs.modal"),a.support.transition&&this.$element.hasClass("fade")?this.$element.one("bsTransitionEnd",a.proxy(this.hideModal,this)).emulateTransitionEnd(c.TRANSITION_DURATION):this.hideModal())},c.prototype.enforceFocus=function(){a(document).off("focusin.bs.modal").on("focusin.bs.modal",a.proxy(function(a){this.$element[0]===a.target||this.$element.has(a.target).length||this.$element.trigger("focus")},this))},c.prototype.escape=function(){this.isShown&&this.options.keyboard?this.$element.on("keydown.dismiss.bs.modal",a.proxy(function(a){27==a.which&&this.hide()},this)):this.isShown||this.$element.off("keydown.dismiss.bs.modal")},c.prototype.resize=function(){this.isShown?a(window).on("resize.bs.modal",a.proxy(this.handleUpdate,this)):a(window).off("resize.bs.modal")},c.prototype.hideModal=function(){var a=this;this.$element.hide(),this.backdrop(function(){a.$body.removeClass("modal-open"),a.resetAdjustments(),a.resetScrollbar(),a.$element.trigger("hidden.bs.modal")})},c.prototype.removeBackdrop=function(){this.$backdrop&&this.$backdrop.remove(),this.$backdrop=null},c.prototype.backdrop=function(b){var d=this,e=this.$element.hasClass("fade")?"fade":"";if(this.isShown&&this.options.backdrop){var f=a.support.transition&&e;if(this.$backdrop=a('<div class="modal-backdrop '+e+'" />').prependTo(this.$element).on("click.dismiss.bs.modal",a.proxy(function(a){a.target===a.currentTarget&&("static"==this.options.backdrop?this.$element[0].focus.call(this.$element[0]):this.hide.call(this))},this)),f&&this.$backdrop[0].offsetWidth,this.$backdrop.addClass("in"),!b)return;f?this.$backdrop.one("bsTransitionEnd",b).emulateTransitionEnd(c.BACKDROP_TRANSITION_DURATION):b()}else if(!this.isShown&&this.$backdrop){this.$backdrop.removeClass("in");var g=function(){d.removeBackdrop(),b&&b()};a.support.transition&&this.$element.hasClass("fade")?this.$backdrop.one("bsTransitionEnd",g).emulateTransitionEnd(c.BACKDROP_TRANSITION_DURATION):g()}else b&&b()},c.prototype.handleUpdate=function(){this.options.backdrop&&this.adjustBackdrop(),this.adjustDialog()},c.prototype.adjustBackdrop=function(){this.$backdrop.css("height",0).css("height",this.$element[0].scrollHeight)},c.prototype.adjustDialog=function(){var a=this.$element[0].scrollHeight>document.documentElement.clientHeight;this.$element.css({paddingLeft:!this.bodyIsOverflowing&&a?this.scrollbarWidth:"",paddingRight:this.bodyIsOverflowing&&!a?this.scrollbarWidth:""})},c.prototype.resetAdjustments=function(){this.$element.css({paddingLeft:"",paddingRight:""})},c.prototype.checkScrollbar=function(){this.bodyIsOverflowing=document.body.scrollHeight>document.documentElement.clientHeight,this.scrollbarWidth=this.measureScrollbar()},c.prototype.setScrollbar=function(){var a=parseInt(this.$body.css("padding-right")||0,10);this.bodyIsOverflowing&&this.$body.css("padding-right",a+this.scrollbarWidth)},c.prototype.resetScrollbar=function(){this.$body.css("padding-right","")},c.prototype.measureScrollbar=function(){var a=document.createElement("div");a.className="modal-scrollbar-measure",this.$body.append(a);var b=a.offsetWidth-a.clientWidth;return this.$body[0].removeChild(a),b};var d=a.fn.modal;a.fn.modal=b,a.fn.modal.Constructor=c,a.fn.modal.noConflict=function(){return a.fn.modal=d,this},a(document).on("click.bs.modal.data-api",'[data-toggle="modal"]',function(c){var d=a(this),e=d.attr("href"),f=a(d.attr("data-target")||e&&e.replace(/.*(?=#[^\s]+$)/,"")),g=f.data("bs.modal")?"toggle":a.extend({remote:!/#/.test(e)&&e},f.data(),d.data());d.is("a")&&c.preventDefault(),f.one("show.bs.modal",function(a){a.isDefaultPrevented()||f.one("hidden.bs.modal",function(){d.is(":visible")&&d.trigger("focus")})}),b.call(f,g,this)})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.tooltip"),f="object"==typeof b&&b,g=f&&f.selector;(e||"destroy"!=b)&&(g?(e||d.data("bs.tooltip",e={}),e[g]||(e[g]=new c(this,f))):e||d.data("bs.tooltip",e=new c(this,f)),"string"==typeof b&&e[b]())})}var c=function(a,b){this.type=this.options=this.enabled=this.timeout=this.hoverState=this.$element=null,this.init("tooltip",a,b)};c.VERSION="3.3.1",c.TRANSITION_DURATION=150,c.DEFAULTS={animation:!0,placement:"top",selector:!1,template:'<div class="tooltip" role="tooltip"><div class="tooltip-arrow"></div><div class="tooltip-inner"></div></div>',trigger:"hover focus",title:"",delay:0,html:!1,container:!1,viewport:{selector:"body",padding:0}},c.prototype.init=function(b,c,d){this.enabled=!0,this.type=b,this.$element=a(c),this.options=this.getOptions(d),this.$viewport=this.options.viewport&&a(this.options.viewport.selector||this.options.viewport);for(var e=this.options.trigger.split(" "),f=e.length;f--;){var g=e[f];if("click"==g)this.$element.on("click."+this.type,this.options.selector,a.proxy(this.toggle,this));else if("manual"!=g){var h="hover"==g?"mouseenter":"focusin",i="hover"==g?"mouseleave":"focusout";this.$element.on(h+"."+this.type,this.options.selector,a.proxy(this.enter,this)),this.$element.on(i+"."+this.type,this.options.selector,a.proxy(this.leave,this))}}this.options.selector?this._options=a.extend({},this.options,{trigger:"manual",selector:""}):this.fixTitle()},c.prototype.getDefaults=function(){return c.DEFAULTS},c.prototype.getOptions=function(b){return b=a.extend({},this.getDefaults(),this.$element.data(),b),b.delay&&"number"==typeof b.delay&&(b.delay={show:b.delay,hide:b.delay}),b},c.prototype.getDelegateOptions=function(){var b={},c=this.getDefaults();return this._options&&a.each(this._options,function(a,d){c[a]!=d&&(b[a]=d)}),b},c.prototype.enter=function(b){var c=b instanceof this.constructor?b:a(b.currentTarget).data("bs."+this.type);return c&&c.$tip&&c.$tip.is(":visible")?void(c.hoverState="in"):(c||(c=new this.constructor(b.currentTarget,this.getDelegateOptions()),a(b.currentTarget).data("bs."+this.type,c)),clearTimeout(c.timeout),c.hoverState="in",c.options.delay&&c.options.delay.show?void(c.timeout=setTimeout(function(){"in"==c.hoverState&&c.show()},c.options.delay.show)):c.show())},c.prototype.leave=function(b){var c=b instanceof this.constructor?b:a(b.currentTarget).data("bs."+this.type);return c||(c=new this.constructor(b.currentTarget,this.getDelegateOptions()),a(b.currentTarget).data("bs."+this.type,c)),clearTimeout(c.timeout),c.hoverState="out",c.options.delay&&c.options.delay.hide?void(c.timeout=setTimeout(function(){"out"==c.hoverState&&c.hide()},c.options.delay.hide)):c.hide()},c.prototype.show=function(){var b=a.Event("show.bs."+this.type);if(this.hasContent()&&this.enabled){this.$element.trigger(b);var d=a.contains(this.$element[0].ownerDocument.documentElement,this.$element[0]);if(b.isDefaultPrevented()||!d)return;var e=this,f=this.tip(),g=this.getUID(this.type);this.setContent(),f.attr("id",g),this.$element.attr("aria-describedby",g),this.options.animation&&f.addClass("fade");var h="function"==typeof this.options.placement?this.options.placement.call(this,f[0],this.$element[0]):this.options.placement,i=/\s?auto?\s?/i,j=i.test(h);j&&(h=h.replace(i,"")||"top"),f.detach().css({top:0,left:0,display:"block"}).addClass(h).data("bs."+this.type,this),this.options.container?f.appendTo(this.options.container):f.insertAfter(this.$element);var k=this.getPosition(),l=f[0].offsetWidth,m=f[0].offsetHeight;if(j){var n=h,o=this.options.container?a(this.options.container):this.$element.parent(),p=this.getPosition(o);h="bottom"==h&&k.bottom+m>p.bottom?"top":"top"==h&&k.top-m<p.top?"bottom":"right"==h&&k.right+l>p.width?"left":"left"==h&&k.left-l<p.left?"right":h,f.removeClass(n).addClass(h)}var q=this.getCalculatedOffset(h,k,l,m);this.applyPlacement(q,h);var r=function(){var a=e.hoverState;e.$element.trigger("shown.bs."+e.type),e.hoverState=null,"out"==a&&e.leave(e)};a.support.transition&&this.$tip.hasClass("fade")?f.one("bsTransitionEnd",r).emulateTransitionEnd(c.TRANSITION_DURATION):r()}},c.prototype.applyPlacement=function(b,c){var d=this.tip(),e=d[0].offsetWidth,f=d[0].offsetHeight,g=parseInt(d.css("margin-top"),10),h=parseInt(d.css("margin-left"),10);isNaN(g)&&(g=0),isNaN(h)&&(h=0),b.top=b.top+g,b.left=b.left+h,a.offset.setOffset(d[0],a.extend({using:function(a){d.css({top:Math.round(a.top),left:Math.round(a.left)})}},b),0),d.addClass("in");var i=d[0].offsetWidth,j=d[0].offsetHeight;"top"==c&&j!=f&&(b.top=b.top+f-j);var k=this.getViewportAdjustedDelta(c,b,i,j);k.left?b.left+=k.left:b.top+=k.top;var l=/top|bottom/.test(c),m=l?2*k.left-e+i:2*k.top-f+j,n=l?"offsetWidth":"offsetHeight";d.offset(b),this.replaceArrow(m,d[0][n],l)},c.prototype.replaceArrow=function(a,b,c){this.arrow().css(c?"left":"top",50*(1-a/b)+"%").css(c?"top":"left","")},c.prototype.setContent=function(){var a=this.tip(),b=this.getTitle();a.find(".tooltip-inner")[this.options.html?"html":"text"](b),a.removeClass("fade in top bottom left right")},c.prototype.hide=function(b){function d(){"in"!=e.hoverState&&f.detach(),e.$element.removeAttr("aria-describedby").trigger("hidden.bs."+e.type),b&&b()}var e=this,f=this.tip(),g=a.Event("hide.bs."+this.type);return this.$element.trigger(g),g.isDefaultPrevented()?void 0:(f.removeClass("in"),a.support.transition&&this.$tip.hasClass("fade")?f.one("bsTransitionEnd",d).emulateTransitionEnd(c.TRANSITION_DURATION):d(),this.hoverState=null,this)},c.prototype.fixTitle=function(){var a=this.$element;(a.attr("title")||"string"!=typeof a.attr("data-original-title"))&&a.attr("data-original-title",a.attr("title")||"").attr("title","")},c.prototype.hasContent=function(){return this.getTitle()},c.prototype.getPosition=function(b){b=b||this.$element;var c=b[0],d="BODY"==c.tagName,e=c.getBoundingClientRect();null==e.width&&(e=a.extend({},e,{width:e.right-e.left,height:e.bottom-e.top}));var f=d?{top:0,left:0}:b.offset(),g={scroll:d?document.documentElement.scrollTop||document.body.scrollTop:b.scrollTop()},h=d?{width:a(window).width(),height:a(window).height()}:null;return a.extend({},e,g,h,f)},c.prototype.getCalculatedOffset=function(a,b,c,d){return"bottom"==a?{top:b.top+b.height,left:b.left+b.width/2-c/2}:"top"==a?{top:b.top-d,left:b.left+b.width/2-c/2}:"left"==a?{top:b.top+b.height/2-d/2,left:b.left-c}:{top:b.top+b.height/2-d/2,left:b.left+b.width}},c.prototype.getViewportAdjustedDelta=function(a,b,c,d){var e={top:0,left:0};if(!this.$viewport)return e;var f=this.options.viewport&&this.options.viewport.padding||0,g=this.getPosition(this.$viewport);if(/right|left/.test(a)){var h=b.top-f-g.scroll,i=b.top+f-g.scroll+d;h<g.top?e.top=g.top-h:i>g.top+g.height&&(e.top=g.top+g.height-i)}else{var j=b.left-f,k=b.left+f+c;j<g.left?e.left=g.left-j:k>g.width&&(e.left=g.left+g.width-k)}return e},c.prototype.getTitle=function(){var a,b=this.$element,c=this.options;return a=b.attr("data-original-title")||("function"==typeof c.title?c.title.call(b[0]):c.title)},c.prototype.getUID=function(a){do a+=~~(1e6*Math.random());while(document.getElementById(a));return a},c.prototype.tip=function(){return this.$tip=this.$tip||a(this.options.template)},c.prototype.arrow=function(){return this.$arrow=this.$arrow||this.tip().find(".tooltip-arrow")},c.prototype.enable=function(){this.enabled=!0},c.prototype.disable=function(){this.enabled=!1},c.prototype.toggleEnabled=function(){this.enabled=!this.enabled},c.prototype.toggle=function(b){var c=this;b&&(c=a(b.currentTarget).data("bs."+this.type),c||(c=new this.constructor(b.currentTarget,this.getDelegateOptions()),a(b.currentTarget).data("bs."+this.type,c))),c.tip().hasClass("in")?c.leave(c):c.enter(c)},c.prototype.destroy=function(){var a=this;clearTimeout(this.timeout),this.hide(function(){a.$element.off("."+a.type).removeData("bs."+a.type)})};var d=a.fn.tooltip;a.fn.tooltip=b,a.fn.tooltip.Constructor=c,a.fn.tooltip.noConflict=function(){return a.fn.tooltip=d,this}}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.popover"),f="object"==typeof b&&b,g=f&&f.selector;(e||"destroy"!=b)&&(g?(e||d.data("bs.popover",e={}),e[g]||(e[g]=new c(this,f))):e||d.data("bs.popover",e=new c(this,f)),"string"==typeof b&&e[b]())})}var c=function(a,b){this.init("popover",a,b)};if(!a.fn.tooltip)throw new Error("Popover requires tooltip.js");c.VERSION="3.3.1",c.DEFAULTS=a.extend({},a.fn.tooltip.Constructor.DEFAULTS,{placement:"right",trigger:"click",content:"",template:'<div class="popover" role="tooltip"><div class="arrow"></div><h3 class="popover-title"></h3><div class="popover-content"></div></div>'}),c.prototype=a.extend({},a.fn.tooltip.Constructor.prototype),c.prototype.constructor=c,c.prototype.getDefaults=function(){return c.DEFAULTS},c.prototype.setContent=function(){var a=this.tip(),b=this.getTitle(),c=this.getContent();a.find(".popover-title")[this.options.html?"html":"text"](b),a.find(".popover-content").children().detach().end()[this.options.html?"string"==typeof c?"html":"append":"text"](c),a.removeClass("fade top bottom left right in"),a.find(".popover-title").html()||a.find(".popover-title").hide()},c.prototype.hasContent=function(){return this.getTitle()||this.getContent()},c.prototype.getContent=function(){var a=this.$element,b=this.options;return a.attr("data-content")||("function"==typeof b.content?b.content.call(a[0]):b.content)},c.prototype.arrow=function(){return this.$arrow=this.$arrow||this.tip().find(".arrow")},c.prototype.tip=function(){return this.$tip||(this.$tip=a(this.options.template)),this.$tip};var d=a.fn.popover;a.fn.popover=b,a.fn.popover.Constructor=c,a.fn.popover.noConflict=function(){return a.fn.popover=d,this}}(jQuery),+function(a){"use strict";function b(c,d){var e=a.proxy(this.process,this);this.$body=a("body"),this.$scrollElement=a(a(c).is("body")?window:c),this.options=a.extend({},b.DEFAULTS,d),this.selector=(this.options.target||"")+" .nav li > a",this.offsets=[],this.targets=[],this.activeTarget=null,this.scrollHeight=0,this.$scrollElement.on("scroll.bs.scrollspy",e),this.refresh(),this.process()}function c(c){return this.each(function(){var d=a(this),e=d.data("bs.scrollspy"),f="object"==typeof c&&c;e||d.data("bs.scrollspy",e=new b(this,f)),"string"==typeof c&&e[c]()})}b.VERSION="3.3.1",b.DEFAULTS={offset:10},b.prototype.getScrollHeight=function(){return this.$scrollElement[0].scrollHeight||Math.max(this.$body[0].scrollHeight,document.documentElement.scrollHeight)},b.prototype.refresh=function(){var b="offset",c=0;a.isWindow(this.$scrollElement[0])||(b="position",c=this.$scrollElement.scrollTop()),this.offsets=[],this.targets=[],this.scrollHeight=this.getScrollHeight();var d=this;this.$body.find(this.selector).map(function(){var d=a(this),e=d.data("target")||d.attr("href"),f=/^#./.test(e)&&a(e);return f&&f.length&&f.is(":visible")&&[[f[b]().top+c,e]]||null}).sort(function(a,b){return a[0]-b[0]}).each(function(){d.offsets.push(this[0]),d.targets.push(this[1])})},b.prototype.process=function(){var a,b=this.$scrollElement.scrollTop()+this.options.offset,c=this.getScrollHeight(),d=this.options.offset+c-this.$scrollElement.height(),e=this.offsets,f=this.targets,g=this.activeTarget;if(this.scrollHeight!=c&&this.refresh(),b>=d)return g!=(a=f[f.length-1])&&this.activate(a);if(g&&b<e[0])return this.activeTarget=null,this.clear();for(a=e.length;a--;)g!=f[a]&&b>=e[a]&&(!e[a+1]||b<=e[a+1])&&this.activate(f[a])},b.prototype.activate=function(b){this.activeTarget=b,this.clear();var c=this.selector+'[data-target="'+b+'"],'+this.selector+'[href="'+b+'"]',d=a(c).parents("li").addClass("active");d.parent(".dropdown-menu").length&&(d=d.closest("li.dropdown").addClass("active")),d.trigger("activate.bs.scrollspy")},b.prototype.clear=function(){a(this.selector).parentsUntil(this.options.target,".active").removeClass("active")};var d=a.fn.scrollspy;a.fn.scrollspy=c,a.fn.scrollspy.Constructor=b,a.fn.scrollspy.noConflict=function(){return a.fn.scrollspy=d,this},a(window).on("load.bs.scrollspy.data-api",function(){a('[data-spy="scroll"]').each(function(){var b=a(this);c.call(b,b.data())})})}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.tab");e||d.data("bs.tab",e=new c(this)),"string"==typeof b&&e[b]()})}var c=function(b){this.element=a(b)};c.VERSION="3.3.1",c.TRANSITION_DURATION=150,c.prototype.show=function(){var b=this.element,c=b.closest("ul:not(.dropdown-menu)"),d=b.data("target");if(d||(d=b.attr("href"),d=d&&d.replace(/.*(?=#[^\s]*$)/,"")),!b.parent("li").hasClass("active")){var e=c.find(".active:last a"),f=a.Event("hide.bs.tab",{relatedTarget:b[0]}),g=a.Event("show.bs.tab",{relatedTarget:e[0]});if(e.trigger(f),b.trigger(g),!g.isDefaultPrevented()&&!f.isDefaultPrevented()){var h=a(d);this.activate(b.closest("li"),c),this.activate(h,h.parent(),function(){e.trigger({type:"hidden.bs.tab",relatedTarget:b[0]}),b.trigger({type:"shown.bs.tab",relatedTarget:e[0]})
})}}},c.prototype.activate=function(b,d,e){function f(){g.removeClass("active").find("> .dropdown-menu > .active").removeClass("active").end().find('[data-toggle="tab"]').attr("aria-expanded",!1),b.addClass("active").find('[data-toggle="tab"]').attr("aria-expanded",!0),h?(b[0].offsetWidth,b.addClass("in")):b.removeClass("fade"),b.parent(".dropdown-menu")&&b.closest("li.dropdown").addClass("active").end().find('[data-toggle="tab"]').attr("aria-expanded",!0),e&&e()}var g=d.find("> .active"),h=e&&a.support.transition&&(g.length&&g.hasClass("fade")||!!d.find("> .fade").length);g.length&&h?g.one("bsTransitionEnd",f).emulateTransitionEnd(c.TRANSITION_DURATION):f(),g.removeClass("in")};var d=a.fn.tab;a.fn.tab=b,a.fn.tab.Constructor=c,a.fn.tab.noConflict=function(){return a.fn.tab=d,this};var e=function(c){c.preventDefault(),b.call(a(this),"show")};a(document).on("click.bs.tab.data-api",'[data-toggle="tab"]',e).on("click.bs.tab.data-api",'[data-toggle="pill"]',e)}(jQuery),+function(a){"use strict";function b(b){return this.each(function(){var d=a(this),e=d.data("bs.affix"),f="object"==typeof b&&b;e||d.data("bs.affix",e=new c(this,f)),"string"==typeof b&&e[b]()})}var c=function(b,d){this.options=a.extend({},c.DEFAULTS,d),this.$target=a(this.options.target).on("scroll.bs.affix.data-api",a.proxy(this.checkPosition,this)).on("click.bs.affix.data-api",a.proxy(this.checkPositionWithEventLoop,this)),this.$element=a(b),this.affixed=this.unpin=this.pinnedOffset=null,this.checkPosition()};c.VERSION="3.3.1",c.RESET="affix affix-top affix-bottom",c.DEFAULTS={offset:0,target:window},c.prototype.getState=function(a,b,c,d){var e=this.$target.scrollTop(),f=this.$element.offset(),g=this.$target.height();if(null!=c&&"top"==this.affixed)return c>e?"top":!1;if("bottom"==this.affixed)return null!=c?e+this.unpin<=f.top?!1:"bottom":a-d>=e+g?!1:"bottom";var h=null==this.affixed,i=h?e:f.top,j=h?g:b;return null!=c&&c>=i?"top":null!=d&&i+j>=a-d?"bottom":!1},c.prototype.getPinnedOffset=function(){if(this.pinnedOffset)return this.pinnedOffset;this.$element.removeClass(c.RESET).addClass("affix");var a=this.$target.scrollTop(),b=this.$element.offset();return this.pinnedOffset=b.top-a},c.prototype.checkPositionWithEventLoop=function(){setTimeout(a.proxy(this.checkPosition,this),1)},c.prototype.checkPosition=function(){if(this.$element.is(":visible")){var b=this.$element.height(),d=this.options.offset,e=d.top,f=d.bottom,g=a("body").height();"object"!=typeof d&&(f=e=d),"function"==typeof e&&(e=d.top(this.$element)),"function"==typeof f&&(f=d.bottom(this.$element));var h=this.getState(g,b,e,f);if(this.affixed!=h){null!=this.unpin&&this.$element.css("top","");var i="affix"+(h?"-"+h:""),j=a.Event(i+".bs.affix");if(this.$element.trigger(j),j.isDefaultPrevented())return;this.affixed=h,this.unpin="bottom"==h?this.getPinnedOffset():null,this.$element.removeClass(c.RESET).addClass(i).trigger(i.replace("affix","affixed")+".bs.affix")}"bottom"==h&&this.$element.offset({top:g-b-f})}};var d=a.fn.affix;a.fn.affix=b,a.fn.affix.Constructor=c,a.fn.affix.noConflict=function(){return a.fn.affix=d,this},a(window).on("load",function(){a('[data-spy="affix"]').each(function(){var c=a(this),d=c.data();d.offset=d.offset||{},null!=d.offsetBottom&&(d.offset.bottom=d.offsetBottom),null!=d.offsetTop&&(d.offset.top=d.offsetTop),b.call(c,d)})})}(jQuery);</script>
    <script>
var renderTestResults = function(testData) {
	var summary = testData["summary"];
	var numTests = summary["num_tests"];
	var numProblems = summary["num_errors"] + summary["num_failures"] + summary["num_skips"];
	var $overview = $("#overview-content");
	var $progress = $(".progress");
	if(numTests == 0) {
		$overview.addClass("alert").addClass("alert-danger").text("No tests were executed.");
		$progress.append($('<div class="progress-bar progress-bar-warning" role="progressbar" style="width: 100%" />'));
	} else if(numProblems > 0) {
		$overview.addClass("alert").addClass("alert-danger").text("There were problems with " + numProblems + " test(s) out of " + numTests + ".");
		var problemPercent = (numProblems/(1.0 * numTests)) * 100.0;
		var successPercent = 100.0 - problemPercent;
		$progress.append($('<div class="progress-bar progress-bar-success" role="progressbar" style="width: ' + successPercent  +  '%" />'));
		$progress.append($('<div class="progress-bar progress-bar-danger" role="progressbar" style="width: ' + problemPercent  +  '%" />'));
	} else {
		$overview.addClass("alert").addClass("alert-success").text("All " + numTests + " test(s) successfully executed.");
		$progress.append($('<div class="progress-bar progress-bar-success" role="progressbar" style="width: 100%" />'));
	}

	var $sidebar = $("#nav-sidebar-tests");
	for(var index in testData["tests"]) {
		var test = testData["tests"][index];
		var testResult = new TestResult(index, test);
		var rawId = testResult.rawId;

		var panelType = testResult.passed ? "panel-success panel-success-custom" : "panel-danger panel-danger-custom";
		var $panel = $('<div class="panel">');
		$panel.addClass(panelType);

		var $panelHeading = $('<div class="panel-heading">');
		var $panelTitle = $('<div class="panel-title">');
		var $a = $('<a class="collapsed" data-toggle="collapse">');
		$a.attr("id", rawId);
		$a.attr("data-target", "#collapse"  + index);
		var testName = testResult.toolName + " (Test #" + (testResult.testIndex + 1) + (testResult.passed ? "" : ", Failed") + ")";
		$a.text(testName);
		var $navLink = $('<a>').attr('href', '#' + rawId).text(testName)
		if(!testResult.passed) {
			$navLink.addClass("text-danger text-danger-custom");
		} else {
			$navLink.addClass("text-success text-success-custom");
		}
		$sidebar.append($('<li>').append( $navLink ) );
		$panelTitle.append($a)
		$panelHeading.append($panelTitle);

		var $panelBody = $('<div class="panel-body panel-collapse collapse" >');
		$panelBody.attr("id", "collapse" + index);

		var $status = $('<div>').text("status: " + testResult.status);
		$panelBody.append($status);
		if(testResult.problems.length > 0) {
			var $problemsLabel = $('<div>').text("problems: ");
			var $problemsDiv = $('<div style="margin-left:10px;">');
			var $problemsUl = $('<ul>');
			for(var problemIndex in testResult.problems) {
				$problemsUl.append($('<li>').append($('<pre>').text(testResult.problems[problemIndex])));
			}
			$problemsDiv.append($problemsUl);
			$panelBody.append($problemsLabel).append($problemsDiv);
		}
		var $commandLabel = $('<div>command:</div>');
		var $stdoutLabel = $('<div>job standard output:</div>');
		var $stderrLabel = $('<div>job standard error:</div>');
		var $command;
		if(testResult.command !== null) {
			$command = $('<pre class="pre-scrollable" style="margin-left:10px;">').text(testResult.command);
		} else {
			$command = $('<div class="alert alert-warning" style="margin-left:10px;">').text("No command recorded.");
		}
		var $stdout;
		if(testResult.stdout !== null) {
			$stdout = $('<pre class="pre-scrollable" style="margin-left:10px;">').text(testResult.stdout);
		} else {
			$stdout = $('<div class="alert alert-warning" style="margin-left:10px;">').text("No standard output recorded.");
		}
		var $stderr;
		if(testResult.stderr !== null) {
			$stderr = $('<pre class="pre-scrollable" style="margin-left:10px;">').text(testResult.stderr);
		} else {
			$stderr = $('<div class="alert alert-warning" style="margin-left:10px;">').text("No standard error recorded.");
		}
		$panelBody
			.append($commandLabel)
			.append($command)
			.append($stdoutLabel)
			.append($stdout)
			.append($stderrLabel)
			.append($stderr);
		if(!testResult.passed) {
			var $logLabel = $('<div>log:</div>');
			var $log = $('<pre class="pre-scrollable" style="margin-left: 10px;">').text(testResult.problemLog);
			$panelBody.append($logLabel).append($log);
		}

		$panel.append($panelHeading).append($panelBody);
		$(".main").append($panel);
	}
}

var TestResult = function(index, data) {
	this.rawId = data["id"];

	var idParts = this.rawId.split("TestForTool_");
	var testMethod = idParts[idParts.length-1];
	var splitParts;
	if(testMethod.indexOf(".test_tool_") > -1) {
		splitParts = testMethod.split(".test_tool_");
	} else {
		splitParts = rSplit(testMethod, "-", 1);
	}
	var toolName = splitParts[0];
	var testIndex;
	if(data["data"]["test_index"] !== null) {
		testIndex = data["data"]["test_index"];
	} else {
		testIndex = splitParts[1];
	}
	this.toolName = toolName;
	this.testIndex = parseInt(testIndex === undefined ? index : testIndex);
	this.status = data["data"]["status"];
	var job = data["data"]["job"];
	if(job) {
		this.stdout = data["data"]["job"]["stdout"];
		this.stderr = data["data"]["job"]["stderr"];
		this.command = data["data"]["job"]["command_line"];
	} else {
		this.stdout = null;
		this.stderr = null;
		this.command = null;
	}
	this.problems = [];
	var outputProblems = data["data"]["output_problems"] || [];
	var executionProblem = data["data"]["execution_problem"];
	this.problems.push.apply(this.problems, outputProblems);
	if(executionProblem) {
		this.problems.push(executionProblem);
	}
	this.problemLog = data["data"]["problem_log"];
	this.passed = (this.status == "success");
}

// http://stackoverflow.com/questions/5202085/javascript-equivalent-of-pythons-rsplit
function rSplit(str, sep, maxsplit) {
    var split = str.split(sep);
    return maxsplit ? [ split.slice(0, -maxsplit).join(sep) ].concat(split.slice(-maxsplit)) : split;
}


// http://stackoverflow.com/questions/19491336/get-url-parameter-jquery
function getUrlParameter(sParam)
{
    var sPageURL = window.location.search.substring(1);
    var sURLVariables = sPageURL.split('&');
    for (var i = 0; i < sURLVariables.length; i++)
    {
        var sParameterName = sURLVariables[i].split('=');
        if (sParameterName[0] == sParam)
        {
            return sParameterName[1];
        }
    }
}
</script>
    <script>
      var testDataUrl = getUrlParameter("test_data_url");
      if(testDataUrl) {
      $.ajax(
        {'url': testDataUrl,
         'type': 'GET',
        }
        )
        .success(function(content) { renderTestResults( $.parseJSON(content) ); })
        .failure(function() { alert("Failed to load test data.")} );
      } else {
        var test_data = {"results": {"errors": 47, "failures": 0, "skips": 0, "total": 47}, "suitename": "Tool Test Results (powered by Planemo)", "summary": {"num_errors": 2, "num_failures": 45, "num_skips": 0, "num_tests": 47}, "tests": [{"data": {"inputs": {"input1": {"id": "39209c3a4b0cef4c", "src": "hda"}, "input2": {"id": "26b7e36460fca86e", "src": "hda"}}, "job": {"command_line": "perl \u0027/home/runner/work/tools-iuc/tools-iuc/tools/coverage_report/CoverageReport.pl\u0027 -b \u0027/tmp/tmplbs_i535/files/3/7/3/dataset_373512d6-547b-45cd-8b17-c68bd3899927.dat\u0027 -t \u0027/tmp/tmplbs_i535/files/a/c/0/dataset_ac03cbd4-6a5e-4fb8-a963-aa79f31c5eb7.dat\u0027  -o \u0027/tmp/tmplbs_i535/files/3/3/5/dataset_335bf5b2-07c2-4790-8f18-4b36d8231b86.dat\u0027  -r  -m 40 -f 0.2 -n \u0027ecoli.bam\u0027", "command_version": "", "create_time": "2020-01-23T17:18:40.371434", "exit_code": 25, "external_id": "26317", "galaxy_version": "20.01", "history_id": "27a22803849b1486", "id": "adcc13f516c5d573", "inputs": {"input1": {"id": "39209c3a4b0cef4c", "src": "hda", "uuid": "373512d6-547b-45cd-8b17-c68bd3899927"}, "input2": {"id": "eb21622d9b6e26b4", "src": "hda", "uuid": "ac03cbd4-6a5e-4fb8-a963-aa79f31c5eb7"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 25 ()", "error_level": 3, "exit_code": 25, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output1": {"id": "598efe8f302ea8ae", "src": "hda", "uuid": "335bf5b2-07c2-4790-8f18-4b36d8231b86"}}, "params": {"PositionLevel": "\"\"", "__input_ext": "\"bam\"", "chromInfo": "\"/tmp/tmplbs_i535/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "frac": "\"0.2\"", "perGene": "\"true\"", "threshold": "\"40\""}, "state": "error", "stderr": "Warning messages:\n1: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n  failed to load cairo DLL\nWarning messages:\n1: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n  failed to load cairo DLL\nWarning messages:\n1: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n  failed to load cairo DLL\nerror: Permission denied (os error 13)\ncaused by: Permission denied (os error 13)\nCould not run tectonic with following error \u0027Inappropriate ioctl for device\u0027\n", "stdout": "", "tool_id": "CoverageReport2", "tool_stderr": "Warning messages:\n1: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n  failed to load cairo DLL\nWarning messages:\n1: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n  failed to load cairo DLL\nWarning messages:\n1: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n  failed to load cairo DLL\nerror: Permission denied (os error 13)\ncaused by: Permission denied (os error 13)\nCould not run tectonic with following error \u0027Inappropriate ioctl for device\u0027\n", "tool_stdout": "", "update_time": "2020-01-23T17:24:13.908617", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmplbs_i535/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmplbs_i535/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmplbs_i535/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmplbs_i535/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmplbs_i535/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmplbs_i535/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027python=2.7,bx-python=0.8.6\u0027 -set REPO=\u0027quay.io/local/mulled-v2-b09526e9169de63faa8bb47ddeb97043262b8aaf:04420d7b2389f021cb64a7433bc34da369bb176b-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bioconda/extended-base-image:latest\u0027 -set TEST=true build-and-test\n.\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmplbs_i535/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027perl-number-format=1.75,R=3.4.1,bedtools=2.17.0,samtools=0.1.18,tectonic=0.1.6\u0027 -set REPO=\u0027quay.io/local/mulled-v2-228a46f17062005a43722459af55cb20987c4027:1e8da17ee583ef23cec66792c37939d09e660a8e-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n..\nProblem in history with id 27a22803849b1486 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - ecoli.bam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  277.1 KB\n| Dataset Info:\n|  uploaded bam file\n| Peek:\n|  Binary bam alignments file\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - ecoli_6.bed (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 region\n| Dataset Info:\n|  uploaded interval file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Chrom\u003c/th\u003e\u003cth\u003e2.Start\u003c/th\u003e\u003cth\u003e3.End\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003egi|49175990|ref|NC_000913.2|_Escherichia_coli_str._K-12_substr._MG1655,_complete_genome,_cropped_to_first_1000_nucleotides\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1000\u003c/td\u003e\u003ctd\u003egi|49175990|ref|NC_000913.2|_Escherichia_coli_str._K-12_substr._MG1655,_complete_genome,_cropped_to_first_1000_nucleotides\u003c/td\u003e\u003ctd\u003e5\u003c/td\u003e\u003ctd\u003e6\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - ecoli_6.bed (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 region\n| Dataset Info:\n|  1 regions converted to BED.\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Chrom\u003c/th\u003e\u003cth\u003e2.Start\u003c/th\u003e\u003cth\u003e3.End\u003c/th\u003e\u003cth\u003e4.Name\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6.Strand\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003egi|49175990|ref|NC_000913.2|_Escherichia_coli_str._K-12_substr._MG1655,_complete_genome,_cropped_to_first_1000_nucleotides\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1000\u003c/td\u003e\u003ctd\u003eregion_0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  1 regions converted to BED.\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Panel Coverage Report on data 2 and data 1: PDF Report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Warning messages:\n|  1: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or direct\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Warning messages:\n|  1: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n|  failed to load cairo DLL\n|  Warning messages:\n|  1: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n|  failed to load cairo DLL\n|  Warning messages:\n|  1: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n|  failed to load cairo DLL\n|  error: Permission denied (os error 13)\n|  caused by: Permission denied (os error 13)\n|  Could not run tectonic with following error \u0027Inappropriate ioctl for device\u0027\n|\n--------------------------------------\n| Job adcc13f516c5d573\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:24:13.908617\n| Create Time:\n|  2020-01-23T17:18:40.371434\n|\n--------------------------------------\n| Job 3d556e8e35b754dc\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:20:35.378528\n| Create Time:\n|  2020-01-23T17:18:40.286802\n|\n--------------------------------------\n| Job 330fe5822b24a25f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:18:39.062478\n| Create Time:\n|  2020-01-23T17:18:33.425944\n|\n--------------------------------------\n| Job 19ec6936a7590160\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:18:26.974441\n| Create Time:\n|  2020-01-23T17:18:20.206262\n|\n--------------------------------------\nProblem in history with id 27a22803849b1486 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - ecoli.bam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  277.1 KB\n| Dataset Info:\n|  uploaded bam file\n| Peek:\n|  Binary bam alignments file\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - ecoli_6.bed (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 region\n| Dataset Info:\n|  uploaded interval file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Chrom\u003c/th\u003e\u003cth\u003e2.Start\u003c/th\u003e\u003cth\u003e3.End\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003egi|49175990|ref|NC_000913.2|_Escherichia_coli_str._K-12_substr._MG1655,_complete_genome,_cropped_to_first_1000_nucleotides\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1000\u003c/td\u003e\u003ctd\u003egi|49175990|ref|NC_000913.2|_Escherichia_coli_str._K-12_substr._MG1655,_complete_genome,_cropped_to_first_1000_nucleotides\u003c/td\u003e\u003ctd\u003e5\u003c/td\u003e\u003ctd\u003e6\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - ecoli_6.bed (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 region\n| Dataset Info:\n|  1 regions converted to BED.\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Chrom\u003c/th\u003e\u003cth\u003e2.Start\u003c/th\u003e\u003cth\u003e3.End\u003c/th\u003e\u003cth\u003e4.Name\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6.Strand\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003egi|49175990|ref|NC_000913.2|_Escherichia_coli_str._K-12_substr._MG1655,_complete_genome,_cropped_to_first_1000_nucleotides\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1000\u003c/td\u003e\u003ctd\u003eregion_0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  1 regions converted to BED.\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Panel Coverage Report on data 2 and data 1: PDF Report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Warning messages:\n|  1: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or direct\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Warning messages:\n|  1: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In png(file = \"../Plots/CoverageBoxPlot.png\", bg = \"white\", width = 240,  :\n|  failed to load cairo DLL\n|  Warning messages:\n|  1: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In png(file = \"../Plots/CoverageNtPlot.png\", bg = \"white\", width = 540,  :\n|  failed to load cairo DLL\n|  Warning messages:\n|  1: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In png(file = \"../Plots/Coverage_region_0.png\", bg = \"white\", width = 480,  :\n|  failed to load cairo DLL\n|  error: Permission denied (os error 13)\n|  caused by: Permission denied (os error 13)\n|  Could not run tectonic with following error \u0027Inappropriate ioctl for device\u0027\n|\n--------------------------------------\n| Job adcc13f516c5d573\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:24:13.908617\n| Create Time:\n|  2020-01-23T17:18:40.371434\n|\n--------------------------------------\n| Job 3d556e8e35b754dc\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:20:35.378528\n| Create Time:\n|  2020-01-23T17:18:40.286802\n|\n--------------------------------------\n| Job 330fe5822b24a25f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:18:39.062478\n| Create Time:\n|  2020-01-23T17:18:33.425944\n|\n--------------------------------------\n| Job 19ec6936a7590160\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:18:26.974441\n| Create Time:\n|  2020-01-23T17:18:20.206262\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 431.50546622276306, "tool_id": "CoverageReport2", "tool_version": "0.0.4"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_CoverageReport2.test_tool_000000"}, {"data": {"inputs": {"first": "1", "input": {"id": "27ee89e2e3d631e0", "src": "hda"}, "last": "27"}, "job": {"command_line": "gunzip -c -f \u0027/tmp/tmpwaz4r6j_/files/b/b/c/dataset_bbc4538e-9c20-4759-a1da-a9d5621f6036.dat\u0027 | fastx_trimmer -v -f 1 -l 27   -Q 33     -z   \u003e \u0027/tmp/tmpwaz4r6j_/files/9/d/7/dataset_9d79af99-5b6f-4a00-b8e0-fe7abd9345b2.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:03:11.467366", "exit_code": 0, "external_id": "12893", "galaxy_version": "20.01", "history_id": "b842d972534ccb3e", "id": "61f03d5eef6f1538", "inputs": {"input": {"id": "27ee89e2e3d631e0", "src": "hda", "uuid": "bbc4538e-9c20-4759-a1da-a9d5621f6036"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output": {"id": "61f03d5eef6f1538", "src": "hda", "uuid": "9d79af99-5b6f-4a00-b8e0-fe7abd9345b2"}}, "params": {"__input_ext": "\"fastqsanger.gz\"", "chromInfo": "\"/tmp/tmpwaz4r6j_/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "first": "\"1\"", "last": "\"27\""}, "state": "ok", "stderr": "Trimming: base 1 to 27\nInput: 2 reads.\nOutput: 2 reads.\n", "stdout": "", "tool_id": "cshl_fastx_trimmer", "tool_stderr": "Trimming: base 1 to 27\nInput: 2 reads.\nOutput: 2 reads.\n", "tool_stdout": "", "update_time": "2020-01-23T17:03:16.621370", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmp44u_5yw7fastx_trimmer2.out.gz v. /tmp/tmp2418z6t1fastx_trimmer2.out.gz )\n--- local_file\n+++ history_data\n@@ -1,8 +0,0 @@\n-@CSHL__2_FC042NGABCD:8:1:120:202\n-ACGATAGATCGGAAGAGCTAGTATGCC\n-+CSHL__2_FC042NGABCD:8:1:120:202\n-40 40 40 40 20 40 40 40 40 6 40 40 28 40 40 25 40 20 40 -1 30 40 14 27 40 8 1\n-@CSHL__2_FC042NGABCD:8:1:103:1185\n-ATCACGATAGATCGGCAGAGCTCGTTT\n-+CSHL__2_FC042NGABCD:8:1:103:1185\n-40 40 40 40 40 35 33 31 40 40 40 32 30 22 40 0 9 22 17 14 8 36 15 34 22 12 23\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpwaz4r6j_/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpwaz4r6j_/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpwaz4r6j_/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpwaz4r6j_/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpwaz4r6j_/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmp44u_5yw7fastx_trimmer2.out.gz v. /tmp/tmp2418z6t1fastx_trimmer2.out.gz )\n--- local_file\n+++ history_data\n@@ -1,8 +0,0 @@\n-@CSHL__2_FC042NGABCD:8:1:120:202\n-ACGATAGATCGGAAGAGCTAGTATGCC\n-+CSHL__2_FC042NGABCD:8:1:120:202\n-40 40 40 40 20 40 40 40 40 6 40 40 28 40 40 25 40 20 40 -1 30 40 14 27 40 8 1\n-@CSHL__2_FC042NGABCD:8:1:103:1185\n-ATCACGATAGATCGGCAGAGCTCGTTT\n-+CSHL__2_FC042NGABCD:8:1:103:1185\n-40 40 40 40 40 35 33 31 40 40 40 32 30 22 40 0 9 22 17 14 8 36 15 34 22 12 23\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 2, "time_seconds": 13.471972227096558, "tool_id": "cshl_fastx_trimmer", "tool_version": "1.0.3"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_cshl_fastx_trimmer.test_tool_000002"}, {"data": {"inputs": {"information|to_download": "stats", "information|type": "qc-stats", "run_id": "ERR675640"}, "job": {"command_line": "\u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/download_ebi_metagenomics_run_data\u0027 \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\u0027 \u0027ERR675640\u0027 \u0027qc-stats\u0027 \u0027stats\u0027  \u0027single_chunk\u0027  \u0027output.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:18:16.193367", "exit_code": 1, "external_id": "19515", "galaxy_version": "20.01", "history_id": "5ca39fba41b940d1", "id": "161ef5576377f4a0", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output_file_tsv": {"id": "449eb427f528cea7", "src": "hda", "uuid": "183e4e1e-15c1-4cf6-be79-e968d7ecd5be"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmpei4of2lf/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "information": "{\"__current_case__\": 1, \"to_download\": \"stats\", \"type\": \"qc-stats\"}", "run_id": "afdb7af813b36b14d33414b49c0accbc"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n    from xmltramp2 import xmltramp\nModuleNotFoundError: No module named \u0027xmltramp2\u0027\n", "stdout": "", "tool_id": "ebi_metagenomics_run_downloader", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n    from xmltramp2 import xmltramp\nModuleNotFoundError: No module named \u0027xmltramp2\u0027\n", "tool_stdout": "", "update_time": "2020-01-23T17:20:37.941499", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027six=1.10,xmltramp2=3.1.1,urllib3=1.12,curl=7.49.0\u0027 -set REPO=\u0027quay.io/local/mulled-v2-8c125371ebff2e31321392313b2d9003cb1537ac:cf8eee7269d33855301c368470dbf2e02e848344-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n.\nProblem in history with id 5ca39fba41b940d1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - Download run data for ERR675640 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n|\n--------------------------------------\n| Job 161ef5576377f4a0\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:20:37.941499\n| Create Time:\n|  2020-01-23T17:18:16.193367\n|\n--------------------------------------\nProblem in history with id 5ca39fba41b940d1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - Download run data for ERR675640 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n|\n--------------------------------------\n| Job 161ef5576377f4a0\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:20:37.941499\n| Create Time:\n|  2020-01-23T17:18:16.193367\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 205.91529631614685, "tool_id": "ebi_metagenomics_run_downloader", "tool_version": "0.1.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ebi_metagenomics_run_downloader.test_tool_000000"}, {"data": {"inputs": {"information|to_download": "PredictedCDSWithAnnotation", "information|type": "sequences", "run_id": "ERR675573"}, "job": {"command_line": "\u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/download_ebi_metagenomics_run_data\u0027 \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\u0027 \u0027ERR675573\u0027 \u0027sequences\u0027 \u0027PredictedCDSWithAnnotation\u0027  \u0027multiple_chunks\u0027  \u0027output.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:21:42.138351", "exit_code": 1, "external_id": "21867", "galaxy_version": "20.01", "history_id": "3e4e8de146a035f5", "id": "ba60dc8fe2d987d5", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output_file_fasta": {"id": "05698cd4bb850c59", "src": "hda", "uuid": "3af09aef-05f5-4540-b981-90b78c81268c"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmpei4of2lf/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "information": "{\"__current_case__\": 0, \"to_download\": \"PredictedCDSWithAnnotation\", \"type\": \"sequences\"}", "run_id": "afdb7af813b36b14a7efd4de713baeaf"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n    from xmltramp2 import xmltramp\nModuleNotFoundError: No module named \u0027xmltramp2\u0027\n", "stdout": "", "tool_id": "ebi_metagenomics_run_downloader", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n    from xmltramp2 import xmltramp\nModuleNotFoundError: No module named \u0027xmltramp2\u0027\n", "tool_stdout": "", "update_time": "2020-01-23T17:23:47.598624", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027six=1.10,xmltramp2=3.1.1,urllib3=1.12,curl=7.49.0\u0027 -set REPO=\u0027quay.io/local/mulled-v2-8c125371ebff2e31321392313b2d9003cb1537ac:cf8eee7269d33855301c368470dbf2e02e848344-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n.\nProblem in history with id 3e4e8de146a035f5 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - Download run data for ERR675573 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n|\n--------------------------------------\n| Job ba60dc8fe2d987d5\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:23:47.598624\n| Create Time:\n|  2020-01-23T17:21:42.138351\n|\n--------------------------------------\nProblem in history with id 3e4e8de146a035f5 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - Download run data for ERR675573 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 22, in \u003cmodule\u003e\n|  from xmltramp2 import xmltramp\n|  ModuleNotFoundError: No module named \u0027xmltramp2\u0027\n|\n--------------------------------------\n| Job ba60dc8fe2d987d5\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:23:47.598624\n| Create Time:\n|  2020-01-23T17:21:42.138351\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 205.806640625, "tool_id": "ebi_metagenomics_run_downloader", "tool_version": "0.1.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ebi_metagenomics_run_downloader.test_tool_000001"}, {"data": {"inputs": {"searched_domain|domain": "metagenomics_runs", "searched_domain|fields": ["id", "experiment_type"], "searched_domain|queries_0|combination_operation": "AND", "searched_domain|queries_0|comp_operation|operation": "equal", "searched_domain|queries_0|comp_operation|query_text": "fecal", "searched_domain|queries_0|query_field": "biome"}, "job": {"command_line": "python \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\u0027 getResults \u0027metagenomics_runs\u0027 \u0027biome:(fecal)\u0027 \u0027id,experiment_type\u0027 \u003e \u0027/tmp/tmpei4of2lf/files/7/1/d/dataset_71dafdb3-7bca-4c64-95d5-c166f18b98be.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:25:07.987514", "exit_code": 1, "external_id": "23855", "galaxy_version": "20.01", "history_id": "a1e670e1ea0da54b", "id": "f21e60058e4146e2", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"search_results": {"id": "bfa55a0374b9a6bf", "src": "hda", "uuid": "71dafdb3-7bca-4c64-95d5-c166f18b98be"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmpei4of2lf/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "searched_domain": "{\"__current_case__\": 69, \"domain\": \"metagenomics_runs\", \"fields\": [\"id\", \"experiment_type\"], \"queries\": [{\"__index__\": 0, \"combination_operation\": \"AND\", \"comp_operation\": {\"__current_case__\": 0, \"operation\": \"equal\", \"query_text\": \"fecal\"}, \"query_field\": \"biome\"}]}"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n    getResults(args[1], args[2], args[3])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n    numberOfResults = getNumberOfResults(domain, query)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n    xmlDoc = restRequest(requestUrl)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n    raise ex\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n    resp = urlopen(req)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n    \u0027http\u0027, request, response, code, msg, hdrs)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 400: \n", "stdout": "", "tool_id": "ebi_search_rest_results", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n    getResults(args[1], args[2], args[3])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n    numberOfResults = getNumberOfResults(domain, query)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n    xmlDoc = restRequest(requestUrl)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n    raise ex\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n    resp = urlopen(req)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n    \u0027http\u0027, request, response, code, msg, hdrs)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 400: \n", "tool_stdout": "", "update_time": "2020-01-23T17:26:54.542581", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027six=1.10,xmltramp2=3.1.1,urllib3=1.12\u0027 -set REPO=\u0027quay.io/local/mulled-v2-427ef1db0aa596d761169a2b643488a2a807f397:fa06153a05361b233bbc80b465dc920622fd92e5-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n.\nProblem in history with id a1e670e1ea0da54b - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - EBI Search on metagenomics_runs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n|  numberOfResults = getNumberOfResults(domain, query)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n|  xmlDoc = restRequest(requestUrl)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n|  raise ex\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n|  resp = urlopen(req)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n|  return opener.open(url, data, timeout)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n|  response = meth(req, response)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n|  \u0027http\u0027, request, response, code, msg, hdrs)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n|  return self._call_chain(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n|  result = func(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n|  raise HTTPError(req.full_url, code, msg, hdrs, fp)\n|  urllib.error.HTTPError: HTTP Error 400:\n|\n--------------------------------------\n| Job f21e60058e4146e2\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:26:54.542581\n| Create Time:\n|  2020-01-23T17:25:07.987514\n|\n--------------------------------------\nProblem in history with id a1e670e1ea0da54b - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - EBI Search on metagenomics_runs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n|  numberOfResults = getNumberOfResults(domain, query)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n|  xmlDoc = restRequest(requestUrl)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n|  raise ex\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n|  resp = urlopen(req)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n|  return opener.open(url, data, timeout)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n|  response = meth(req, response)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n|  \u0027http\u0027, request, response, code, msg, hdrs)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n|  return self._call_chain(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n|  result = func(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n|  raise HTTPError(req.full_url, code, msg, hdrs, fp)\n|  urllib.error.HTTPError: HTTP Error 400:\n|\n--------------------------------------\n| Job f21e60058e4146e2\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:26:54.542581\n| Create Time:\n|  2020-01-23T17:25:07.987514\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 205.76421880722046, "tool_id": "ebi_search_rest_results", "tool_version": "0.1.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ebi_search_rest_results.test_tool_000000"}, {"data": {"inputs": {"searched_domain|domain": "metagenomics_runs", "searched_domain|fields": ["id", "experiment_type"], "searched_domain|queries_0|combination_operation": "AND", "searched_domain|queries_0|comp_operation|not_query_text": "microb", "searched_domain|queries_0|comp_operation|operation": "not", "searched_domain|queries_0|comp_operation|query_text": "cheese", "searched_domain|queries_0|query_field": "project_name"}, "job": {"command_line": "python \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\u0027 getResults \u0027metagenomics_runs\u0027 \u0027project_name:(cheese NOT microb)\u0027 \u0027id,experiment_type\u0027 \u003e \u0027/tmp/tmpei4of2lf/files/d/7/8/dataset_d7833150-ed2e-439b-9858-3db5b8cb4484.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:28:33.814457", "exit_code": 1, "external_id": "24223", "galaxy_version": "20.01", "history_id": "9a22496ee796fbfc", "id": "4826aef2afaebe4b", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"search_results": {"id": "4674f339b90935da", "src": "hda", "uuid": "d7833150-ed2e-439b-9858-3db5b8cb4484"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmpei4of2lf/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "searched_domain": "{\"__current_case__\": 69, \"domain\": \"metagenomics_runs\", \"fields\": [\"id\", \"experiment_type\"], \"queries\": [{\"__index__\": 0, \"combination_operation\": \"AND\", \"comp_operation\": {\"__current_case__\": 1, \"not_query_text\": \"microb\", \"operation\": \"not\", \"query_text\": \"cheese\"}, \"query_field\": \"project_name\"}]}"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n    getResults(args[1], args[2], args[3])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n    numberOfResults = getNumberOfResults(domain, query)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n    xmlDoc = restRequest(requestUrl)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n    raise ex\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n    resp = urlopen(req)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n    \u0027http\u0027, request, response, code, msg, hdrs)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 400: \n", "stdout": "", "tool_id": "ebi_search_rest_results", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n    getResults(args[1], args[2], args[3])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n    numberOfResults = getNumberOfResults(domain, query)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n    xmlDoc = restRequest(requestUrl)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n    raise ex\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n    resp = urlopen(req)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n    \u0027http\u0027, request, response, code, msg, hdrs)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 400: \n", "tool_stdout": "", "update_time": "2020-01-23T17:28:38.692421", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 9a22496ee796fbfc - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - EBI Search on metagenomics_runs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n|  numberOfResults = getNumberOfResults(domain, query)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n|  xmlDoc = restRequest(requestUrl)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n|  raise ex\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n|  resp = urlopen(req)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n|  return opener.open(url, data, timeout)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n|  response = meth(req, response)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n|  \u0027http\u0027, request, response, code, msg, hdrs)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n|  return self._call_chain(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n|  result = func(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n|  raise HTTPError(req.full_url, code, msg, hdrs, fp)\n|  urllib.error.HTTPError: HTTP Error 400:\n|\n--------------------------------------\n| Job 4826aef2afaebe4b\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:28:38.692421\n| Create Time:\n|  2020-01-23T17:28:33.814457\n|\n--------------------------------------\nProblem in history with id 9a22496ee796fbfc - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - EBI Search on metagenomics_runs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n|  numberOfResults = getNumberOfResults(domain, query)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n|  xmlDoc = restRequest(requestUrl)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n|  raise ex\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n|  resp = urlopen(req)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n|  return opener.open(url, data, timeout)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n|  response = meth(req, response)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n|  \u0027http\u0027, request, response, code, msg, hdrs)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n|  return self._call_chain(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n|  result = func(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n|  raise HTTPError(req.full_url, code, msg, hdrs, fp)\n|  urllib.error.HTTPError: HTTP Error 400:\n|\n--------------------------------------\n| Job 4826aef2afaebe4b\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:28:38.692421\n| Create Time:\n|  2020-01-23T17:28:33.814457\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 7.004012107849121, "tool_id": "ebi_search_rest_results", "tool_version": "0.1.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ebi_search_rest_results.test_tool_000001"}, {"data": {"inputs": {"searched_domain|domain": "metagenomics_runs", "searched_domain|fields": ["id", "experiment_type"], "searched_domain|queries_0|combination_operation": "AND", "searched_domain|queries_0|comp_operation|operation": "equal", "searched_domain|queries_0|comp_operation|query_text": "marine", "searched_domain|queries_0|query_field": "biome", "searched_domain|queries_1|combination_operation": "AND", "searched_domain|queries_1|comp_operation|max": "200", "searched_domain|queries_1|comp_operation|min": "0", "searched_domain|queries_1|comp_operation|operation": "range", "searched_domain|queries_1|query_field": "depth"}, "job": {"command_line": "python \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\u0027 getResults \u0027metagenomics_runs\u0027 \u0027biome:(marine) AND depth:[0 TO 200]\u0027 \u0027id,experiment_type\u0027 \u003e \u0027/tmp/tmpei4of2lf/files/e/e/d/dataset_eed6aab7-6e21-4fdb-9847-1511d6564c2e.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:28:40.870551", "exit_code": 1, "external_id": "24454", "galaxy_version": "20.01", "history_id": "0535d515f68130b3", "id": "c95281174db2641a", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"search_results": {"id": "8bfda5654e551614", "src": "hda", "uuid": "eed6aab7-6e21-4fdb-9847-1511d6564c2e"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmpei4of2lf/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "searched_domain": "{\"__current_case__\": 69, \"domain\": \"metagenomics_runs\", \"fields\": [\"id\", \"experiment_type\"], \"queries\": [{\"__index__\": 0, \"combination_operation\": \"AND\", \"comp_operation\": {\"__current_case__\": 0, \"operation\": \"equal\", \"query_text\": \"marine\"}, \"query_field\": \"biome\"}, {\"__index__\": 1, \"combination_operation\": \"AND\", \"comp_operation\": {\"__current_case__\": 2, \"max\": \"200\", \"min\": \"0\", \"operation\": \"range\"}, \"query_field\": \"depth\"}]}"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n    getResults(args[1], args[2], args[3])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n    numberOfResults = getNumberOfResults(domain, query)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n    xmlDoc = restRequest(requestUrl)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n    raise ex\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n    resp = urlopen(req)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n    \u0027http\u0027, request, response, code, msg, hdrs)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 400: \n", "stdout": "", "tool_id": "ebi_search_rest_results", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n    getResults(args[1], args[2], args[3])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n    numberOfResults = getNumberOfResults(domain, query)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n    xmlDoc = restRequest(requestUrl)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n    raise ex\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n    resp = urlopen(req)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n    return opener.open(url, data, timeout)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n    response = meth(req, response)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n    \u0027http\u0027, request, response, code, msg, hdrs)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n    return self._call_chain(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n    result = func(*args)\n  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n    raise HTTPError(req.full_url, code, msg, hdrs, fp)\nurllib.error.HTTPError: HTTP Error 400: \n", "tool_stdout": "", "update_time": "2020-01-23T17:28:45.954655", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpei4of2lf/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 0535d515f68130b3 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - EBI Search on metagenomics_runs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n|  numberOfResults = getNumberOfResults(domain, query)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n|  xmlDoc = restRequest(requestUrl)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n|  raise ex\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n|  resp = urlopen(req)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n|  return opener.open(url, data, timeout)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n|  response = meth(req, response)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n|  \u0027http\u0027, request, response, code, msg, hdrs)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n|  return self._call_chain(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n|  result = func(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n|  raise HTTPError(req.full_url, code, msg, hdrs, fp)\n|  urllib.error.HTTPError: HTTP Error 400:\n|\n--------------------------------------\n| Job c95281174db2641a\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:28:45.954655\n| Create Time:\n|  2020-01-23T17:28:40.870551\n|\n--------------------------------------\nProblem in history with id 0535d515f68130b3 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - EBI Search on metagenomics_runs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 299, in \u003cmodule\u003e\n|  getResults(args[1], args[2], args[3])\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 179, in getResults\n|  numberOfResults = getNumberOfResults(domain, query)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 162, in getNumberOfResults\n|  xmlDoc = restRequest(requestUrl)\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 84, in restRequest\n|  raise ex\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ebi_tools/ebeye_urllib.py\", line 68, in restRequest\n|  resp = urlopen(req)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 223, in urlopen\n|  return opener.open(url, data, timeout)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 532, in open\n|  response = meth(req, response)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 642, in http_response\n|  \u0027http\u0027, request, response, code, msg, hdrs)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 570, in error\n|  return self._call_chain(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 504, in _call_chain\n|  result = func(*args)\n|  File \"/usr/local/lib/python3.6/urllib/request.py\", line 650, in http_error_default\n|  raise HTTPError(req.full_url, code, msg, hdrs, fp)\n|  urllib.error.HTTPError: HTTP Error 400:\n|\n--------------------------------------\n| Job c95281174db2641a\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:28:45.954655\n| Create Time:\n|  2020-01-23T17:28:40.870551\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 2, "time_seconds": 7.001227140426636, "tool_id": "ebi_search_rest_results", "tool_version": "0.1.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ebi_search_rest_results.test_tool_000002"}, {"data": {"inputs": {"query_type|free_text_search": "--free_text_search", "query_type|query": "kinase+homo+sapiens", "query_type|res|display_opt|display": "fasta", "query_type|res|result": "sequence_update"}, "job": {"command_line": "enasearch search_data --free_text_search --query \u0027kinase+homo+sapiens\u0027 --result \u0027sequence_update\u0027   --display \u0027fasta\u0027   --file \u0027/tmp/tmpvh8hsjtk/files/a/d/e/dataset_adee6ae2-8a7c-496c-92d9-04e0a2ab6bfc.dat\u0027 --download \u0027txt\u0027", "command_version": "0.1.1\n", "create_time": "2020-01-23T18:14:14.711147", "exit_code": 0, "external_id": "57731", "galaxy_version": "20.01", "history_id": "f21e60058e4146e2", "id": "940151f28ccfd679", "inputs": {}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output": {"id": "7535296d23a37c60", "src": "hda", "uuid": "adee6ae2-8a7c-496c-92d9-04e0a2ab6bfc"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmpvh8hsjtk/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "query_type": "{\"__current_case__\": 0, \"free_text_search\": \"--free_text_search\", \"query\": \"kinase+homo+sapiens\", \"res\": {\"__current_case__\": 13, \"display_opt\": {\"__current_case__\": 5, \"display\": \"fasta\"}, \"result\": \"sequence_update\"}}"}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "enasearch_search_data", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T18:14:21.024416", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected\nOutput file did not contain expected text \u0027\u003eENA|\u0027 (output \u0027\u0027)"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpvh8hsjtk/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpvh8hsjtk/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpvh8hsjtk/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpvh8hsjtk/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpvh8hsjtk/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected\nOutput file did not contain expected text \u0027\u003eENA|\u0027 (output \u0027\u0027)\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 6.760035753250122, "tool_id": "enasearch_search_data", "tool_version": "0.1.1.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_enasearch_search_data.test_tool_000000"}, {"data": {"inputs": {"annotate_source": {"id": "0ff3854aee1a95dd", "src": "hda"}, "a|a_selector": "extract", "a|actions_0|column_name": "anno5", "a|actions_0|column_type": "text", "a|actions_0|element_to_extract": "1", "a|actions_0|operation": "first", "infile": {"id": "5e6ad6ca0afeaf24", "src": "hda"}}, "job": {"command_line": "bgzip -c \u0027/tmp/tmp1fsx_m6m/files/4/a/4/dataset_4a4b44c5-dfc3-428a-8d87-fa34c62cc98e.dat\u0027 \u003e \u0027tabixed.bed.gz\u0027 \u0026\u0026 tabix -p \u0027bed\u0027 \u0027tabixed.bed.gz\u0027 \u0026\u0026 cp \u0027/tmp/tmp1fsx_m6m/files/6/3/6/dataset_6362a03e-b0d8-47e7-bd02-aadb8e98b385.dat\u0027 \u0027/tmp/tmp1fsx_m6m/files/b/4/6/dataset_b4666113-6373-4b07-909b-1749d0417cb7.dat\u0027 \u0026\u0026  gemini annotate -f \u0027tabixed.bed.gz\u0027 -a extract      -t text -e \u00271\u0027 -o first -c \u0027anno5\u0027 \u0027/tmp/tmp1fsx_m6m/files/b/4/6/dataset_b4666113-6373-4b07-909b-1749d0417cb7.dat\u0027", "command_version": "gemini 0.20.1\n", "create_time": "2020-01-23T17:12:51.512382", "exit_code": 0, "external_id": "19641", "galaxy_version": "20.01", "history_id": "1ae74d26531588b0", "id": "616a01dfb111aa65", "inputs": {"annotate_source": {"id": "0ff3854aee1a95dd", "src": "hda", "uuid": "4a4b44c5-dfc3-428a-8d87-fa34c62cc98e"}, "infile": {"id": "5e6ad6ca0afeaf24", "src": "hda", "uuid": "6362a03e-b0d8-47e7-bd02-aadb8e98b385"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outfile": {"id": "0e797101c95aea67", "src": "hda", "uuid": "b4666113-6373-4b07-909b-1749d0417cb7"}}, "params": {"__input_ext": "\"gemini.sqlite\"", "a": "{\"__current_case__\": 0, \"a_selector\": \"extract\", \"actions\": [{\"__index__\": 0, \"column_name\": \"anno5\", \"column_type\": \"text\", \"element_to_extract\": \"1\", \"operation\": \"first\"}]}", "chromInfo": "\"/tmp/tmp1fsx_m6m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "region_only": "\"true\""}, "state": "ok", "stderr": "", "stdout": "finished updating 0 variants\n", "tool_id": "gemini_annotate", "tool_stderr": "", "tool_stdout": "finished updating 0 variants\n", "update_time": "2020-01-23T17:12:57.691089", "user_email": "test@bx.psu.edu"}, "output_problems": ["Command produced by the job different than expected\nNo text matching expression \u0027-e +1\u0027 was found in output file."], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: Command produced by the job different than expected\nNo text matching expression \u0027-e +1\u0027 was found in output file.\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 20.199344635009766, "tool_id": "gemini_annotate", "tool_version": "0.20.1+galaxy1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_gemini_annotate.test_tool_000001"}, {"data": {"inputs": {"annotate_source": {"id": "48f3e8809447c25a", "src": "hda"}, "a|a_selector": "extract", "a|actions_0|column_name": "anno5", "a|actions_0|column_type": "text", "a|actions_0|element_to_extract": "1", "a|actions_0|operation": "first", "a|actions_1|column_name": "anno6", "a|actions_1|column_type": "integer", "a|actions_1|element_to_extract": "2", "a|actions_1|operation": "first", "infile": {"id": "baa1080356b320b7", "src": "hda"}}, "job": {"command_line": "bgzip -c \u0027/tmp/tmp1fsx_m6m/files/7/4/2/dataset_742a90a1-43e8-4f8e-9caa-530f04967929.dat\u0027 \u003e \u0027tabixed.bed.gz\u0027 \u0026\u0026 tabix -p \u0027bed\u0027 \u0027tabixed.bed.gz\u0027 \u0026\u0026 cp \u0027/tmp/tmp1fsx_m6m/files/6/1/6/dataset_6168f2cd-eb2b-479b-854b-950e369675d4.dat\u0027 \u0027/tmp/tmp1fsx_m6m/files/b/6/5/dataset_b65091b9-fa87-4d96-b701-7a9c1b0131b2.dat\u0027 \u0026\u0026  gemini annotate -f \u0027tabixed.bed.gz\u0027 -a extract          -t text,integer -e \u00271,2\u0027 -o first,first -c \u0027anno5,anno6\u0027 \u0027/tmp/tmp1fsx_m6m/files/b/6/5/dataset_b65091b9-fa87-4d96-b701-7a9c1b0131b2.dat\u0027", "command_version": "gemini 0.20.1\n", "create_time": "2020-01-23T17:13:11.771930", "exit_code": 0, "external_id": "19979", "galaxy_version": "20.01", "history_id": "440a6c2b5d9efe20", "id": "a6b9cabbe61f73ca", "inputs": {"annotate_source": {"id": "48f3e8809447c25a", "src": "hda", "uuid": "742a90a1-43e8-4f8e-9caa-530f04967929"}, "infile": {"id": "baa1080356b320b7", "src": "hda", "uuid": "6168f2cd-eb2b-479b-854b-950e369675d4"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outfile": {"id": "85f590476b78f8ed", "src": "hda", "uuid": "b65091b9-fa87-4d96-b701-7a9c1b0131b2"}}, "params": {"__input_ext": "\"gemini.sqlite\"", "a": "{\"__current_case__\": 0, \"a_selector\": \"extract\", \"actions\": [{\"__index__\": 0, \"column_name\": \"anno5\", \"column_type\": \"text\", \"element_to_extract\": \"1\", \"operation\": \"first\"}, {\"__index__\": 1, \"column_name\": \"anno6\", \"column_type\": \"integer\", \"element_to_extract\": \"2\", \"operation\": \"first\"}]}", "chromInfo": "\"/tmp/tmp1fsx_m6m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "region_only": "\"true\""}, "state": "ok", "stderr": "", "stdout": "finished updating 0 variants\n", "tool_id": "gemini_annotate", "tool_stderr": "", "tool_stdout": "finished updating 0 variants\n", "update_time": "2020-01-23T17:13:17.565574", "user_email": "test@bx.psu.edu"}, "output_problems": ["Command produced by the job different than expected\nNo text matching expression \u0027-e +1,2\u0027 was found in output file."], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: Command produced by the job different than expected\nNo text matching expression \u0027-e +1,2\u0027 was found in output file.\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 2, "time_seconds": 20.28519058227539, "tool_id": "gemini_annotate", "tool_version": "0.20.1+galaxy1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_gemini_annotate.test_tool_000002"}, {"data": {"inputs": {"analysis|mode": "counts", "infile": {"id": "f8cc3c3467afd773", "src": "hda"}}, "job": {"command_line": "gemini burden  \u0027/tmp/tmp1fsx_m6m/files/d/c/b/dataset_dcb9db99-e8e8-4345-ba32-72a9840eed40.dat\u0027 \u003e \u0027/tmp/tmp1fsx_m6m/files/e/9/6/dataset_e96a8243-7636-4c2f-99c5-478f0df8fdf8.dat\u0027", "command_version": "gemini 0.20.1\n", "create_time": "2020-01-23T17:13:25.365759", "exit_code": 0, "external_id": "20283", "galaxy_version": "20.01", "history_id": "ea0b941dfbe636f8", "id": "dccb60510b24da2f", "inputs": {"infile": {"id": "f8cc3c3467afd773", "src": "hda", "uuid": "dcb9db99-e8e8-4345-ba32-72a9840eed40"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outfile": {"id": "b908d9d86f8a3d36", "src": "hda", "uuid": "e96a8243-7636-4c2f-99c5-478f0df8fdf8"}}, "params": {"__input_ext": "\"gemini.sqlite\"", "analysis": "{\"__current_case__\": 0, \"mode\": \"counts\"}", "chromInfo": "\"/tmp/tmp1fsx_m6m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "nonsynonymous": "\"\""}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "gemini_burden", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T17:13:30.932892", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpu8fiiod3gemini_burden_count_highimpact_result.tabular v. /tmp/tmpif9h5u1qgemini_burden_count_highimpact_result.tabular )\n--- local_file\n+++ history_data\n@@ -1,2 +0,0 @@\n-gene\t1_kid\t3_kid\n-WDR37\t1\t2\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmpu8fiiod3gemini_burden_count_highimpact_result.tabular v. /tmp/tmpif9h5u1qgemini_burden_count_highimpact_result.tabular )\n--- local_file\n+++ history_data\n@@ -1,2 +0,0 @@\n-gene\t1_kid\t3_kid\n-WDR37\t1\t2\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 13.564966917037964, "tool_id": "gemini_burden", "tool_version": "0.20.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_gemini_burden.test_tool_000000"}, {"data": {"inputs": {"analysis|mode": "counts", "infile": {"id": "4ec7f8db939dbc03", "src": "hda"}, "nonsynonymous": "--nonsynonymous"}, "job": {"command_line": "gemini burden --nonsynonymous \u0027/tmp/tmp1fsx_m6m/files/b/6/6/dataset_b668b108-f82e-4abe-974f-759d65e2b605.dat\u0027 \u003e \u0027/tmp/tmp1fsx_m6m/files/0/9/3/dataset_093d27ea-ed4e-4e66-9c88-6d48c721e3bf.dat\u0027", "command_version": "gemini 0.20.1\n", "create_time": "2020-01-23T17:13:38.963215", "exit_code": 0, "external_id": "20548", "galaxy_version": "20.01", "history_id": "27ee89e2e3d631e0", "id": "5e6ad6ca0afeaf24", "inputs": {"infile": {"id": "4ec7f8db939dbc03", "src": "hda", "uuid": "b668b108-f82e-4abe-974f-759d65e2b605"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outfile": {"id": "9268a0f969417b28", "src": "hda", "uuid": "093d27ea-ed4e-4e66-9c88-6d48c721e3bf"}}, "params": {"__input_ext": "\"gemini.sqlite\"", "analysis": "{\"__current_case__\": 0, \"mode\": \"counts\"}", "chromInfo": "\"/tmp/tmp1fsx_m6m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "nonsynonymous": "\"--nonsynonymous\""}, "state": "ok", "stderr": "/usr/local/lib/python2.7/site-packages/pandas/core/frame.py:6211: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version\nof pandas will change to not sort by default.\n\nTo accept the future behavior, pass \u0027sort=False\u0027.\n\nTo retain the current behavior and silence the warning, pass \u0027sort=True\u0027.\n\n  sort=sort)\n", "stdout": "", "tool_id": "gemini_burden", "tool_stderr": "/usr/local/lib/python2.7/site-packages/pandas/core/frame.py:6211: FutureWarning: Sorting because non-concatenation axis is not aligned. A future version\nof pandas will change to not sort by default.\n\nTo accept the future behavior, pass \u0027sort=False\u0027.\n\nTo retain the current behavior and silence the warning, pass \u0027sort=True\u0027.\n\n  sort=sort)\n", "tool_stdout": "", "update_time": "2020-01-23T17:13:44.443113", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmp_eixq7begemini_burden_count_nonsynonymous_result.tabular v. /tmp/tmp06xccwakgemini_burden_count_nonsynonymous_result.tabular )\n--- local_file\n+++ history_data\n@@ -1,5 +0,0 @@\n-gene\t1_dad\t1_kid\t1_mom\t2_dad\t2_kid\t2_mom\t3_dad\t3_kid\t3_mom\n-SYCE1\t0\t1\t0\t0\t1\t0\t0\t1\t0\n-SPRN\t0\t1\t0\t0\t1\t0\t1\t1\t1\n-WDR37\t0\t1\t0\t0\t0\t0\t0\t2\t0\n-ASAH2C\t2\t3\t2\t1\t3\t1\t1\t2\t1\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp1fsx_m6m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmp_eixq7begemini_burden_count_nonsynonymous_result.tabular v. /tmp/tmp06xccwakgemini_burden_count_nonsynonymous_result.tabular )\n--- local_file\n+++ history_data\n@@ -1,5 +0,0 @@\n-gene\t1_dad\t1_kid\t1_mom\t2_dad\t2_kid\t2_mom\t3_dad\t3_kid\t3_mom\n-SYCE1\t0\t1\t0\t0\t1\t0\t0\t1\t0\n-SPRN\t0\t1\t0\t0\t1\t0\t1\t1\t1\n-WDR37\t0\t1\t0\t0\t0\t0\t0\t2\t0\n-ASAH2C\t2\t3\t2\t1\t3\t1\t1\t2\t1\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 13.594095230102539, "tool_id": "gemini_burden", "tool_version": "0.20.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_gemini_burden.test_tool_000001"}, {"data": {"inputs": {"image_format|dpi": "100", "image_format|format": "png", "input_tree": {"id": "a86a1a479f13fd91", "src": "hda"}, "pad": "2", "size": "7"}, "job": {"command_line": "graphlan.py --format \u0027png\u0027 --size \u00277\u0027 --pad \u00272\u0027 --dpi \u0027100\u0027 \u0027/tmp/tmps547tb2u/files/3/f/d/dataset_3fd3a9cb-7f3f-4083-aa49-29c1470497c0.dat\u0027 \u0027/tmp/tmps547tb2u/files/0/1/4/dataset_01483967-3df5-4ebe-a1d7-6d560995d108.dat\u0027", "command_version": "/tmp/tmps547tb2u/job_working_directory/000/135/tool_script.sh: line 9: graphlan.py: command not found\n", "create_time": "2020-01-23T17:21:08.312047", "exit_code": 127, "external_id": "27377", "galaxy_version": "20.01", "history_id": "bab99c776d4b9391", "id": "4e27b888e0f89616", "inputs": {"input_tree": {"id": "a86a1a479f13fd91", "src": "hda", "uuid": "3fd3a9cb-7f3f-4083-aa49-29c1470497c0"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"png_output_image": {"id": "32b5b014fd8ff7d7", "src": "hda", "uuid": "01483967-3df5-4ebe-a1d7-6d560995d108"}}, "params": {"__input_ext": "\"phyloxml\"", "chromInfo": "\"/tmp/tmps547tb2u/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "image_format": "{\"__current_case__\": 0, \"dpi\": \"100\", \"format\": \"png\"}", "pad": "\"2\"", "size": "\"7\""}, "state": "ok", "stderr": "/tmp/tmps547tb2u/job_working_directory/000/135/tool_script.sh: line 9: graphlan.py: command not found\n", "stdout": "", "tool_id": "graphlan", "tool_stderr": "/tmp/tmps547tb2u/job_working_directory/000/135/tool_script.sh: line 9: graphlan.py: command not found\n", "tool_stdout": "", "update_time": "2020-01-23T17:22:37.892600", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmp1rwrg702png_image.png v. /tmp/tmpe2ih7ew9png_image.png )\nBinary data detected, not displaying diff"], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmps547tb2u/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmps547tb2u/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nHistory item  different than expected, difference (using diff):\n( /tmp/tmp1rwrg702png_image.png v. /tmp/tmpe2ih7ew9png_image.png )\nBinary data detected, not displaying diff\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027graphlan=1.0.0\u0027 -set REPO=\u0027quay.io/local/graphlan:1.0.0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n.\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 110.23856449127197, "tool_id": "graphlan", "tool_version": "1.0.0.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_graphlan.test_tool_000000"}, {"data": {"inputs": {"annot": {"id": "67ef09f1c4f8fd77", "src": "hda"}, "input_tree": {"id": "ab0b902fc4c6ecc1", "src": "hda"}}, "job": {"command_line": "graphlan_annotate.py --annot \u0027/tmp/tmps547tb2u/files/d/1/f/dataset_d1f8dd82-8a0f-4dc0-bf97-3076aa8bfc63.dat\u0027 \u0027/tmp/tmps547tb2u/files/7/4/4/dataset_74420e14-8da2-4b45-a08e-08bbf2013ca6.dat\u0027 \u0027/tmp/tmps547tb2u/files/b/c/d/dataset_bcd34907-d18f-43f8-8ddc-68c443bf2a30.dat\u0027", "command_version": "/tmp/tmps547tb2u/job_working_directory/000/138/tool_script.sh: line 9: graphlan_annotate.py: command not found\n", "create_time": "2020-01-23T17:23:05.176224", "exit_code": 127, "external_id": "30574", "galaxy_version": "20.01", "history_id": "dccb60510b24da2f", "id": "237cecdf0f984d5b", "inputs": {"annot": {"id": "67ef09f1c4f8fd77", "src": "hda", "uuid": "d1f8dd82-8a0f-4dc0-bf97-3076aa8bfc63"}, "input_tree": {"id": "ab0b902fc4c6ecc1", "src": "hda", "uuid": "74420e14-8da2-4b45-a08e-08bbf2013ca6"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output_tree": {"id": "b722e15494aaed88", "src": "hda", "uuid": "bcd34907-d18f-43f8-8ddc-68c443bf2a30"}}, "params": {"__input_ext": "\"txt\"", "chromInfo": "\"/tmp/tmps547tb2u/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\""}, "state": "ok", "stderr": "/tmp/tmps547tb2u/job_working_directory/000/138/tool_script.sh: line 9: graphlan_annotate.py: command not found\n", "stdout": "", "tool_id": "graphlan_annotate", "tool_stderr": "/tmp/tmps547tb2u/job_working_directory/000/138/tool_script.sh: line 9: graphlan_annotate.py: command not found\n", "tool_stdout": "", "update_time": "2020-01-23T17:24:20.177185", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpm404olifintermediary_tree v. /tmp/tmp2xlv9ozlintermediary_tree )\n--- local_file\n+++ history_data\n@@ -1,439 +0,0 @@\n-\u003cphyloxml xmlns=\"http://www.phyloxml.org\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://www.phyloxml.org http://www.phyloxml.org/1.10/phyloxml.xsd\"\u003e\n-  \u003cphylogeny rooted=\"true\"\u003e\n-    \u003cclade\u003e\n-      \u003cclade\u003e\n-        \u003cname\u003ek__Archaea\u003c/name\u003e\n-        \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-        \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-        \u003cclade\u003e\n-          \u003cname\u003ep__Euryarchaeota\u003c/name\u003e\n-          \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-          \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-          \u003cclade\u003e\n-            \u003cname\u003ec__Methanobacteria\u003c/name\u003e\n-            \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-            \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-            \u003cclade\u003e\n-              \u003cname\u003eo__Methanobacteriales\u003c/name\u003e\n-              \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-              \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-              \u003cclade\u003e\n-                \u003cname\u003ef__Methanobacteriaceae\u003c/name\u003e\n-                \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n********\n*SNIP *\n********\n-                      \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e20.5761787104\u003c/property\u003e\n-                    \u003c/clade\u003e\n-                  \u003c/clade\u003e\n-                \u003c/clade\u003e\n-              \u003c/clade\u003e\n-            \u003c/clade\u003e\n-          \u003c/clade\u003e\n-        \u003c/clade\u003e\n-      \u003c/clade\u003e\n-    \u003c/clade\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"class_legend_marker_size\" ref=\"A:1\"\u003e1.5\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"class_legend_font_size\" ref=\"A:1\"\u003e10\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"branch_bracket_depth\" ref=\"A:1\"\u003e0.8\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"clade_separation\" ref=\"A:1\"\u003e0.5\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"branch_bracket_width\" ref=\"A:1\"\u003e0.2\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"annotation_legend_font_size\" ref=\"A:1\"\u003e10\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_keys\" ref=\"A:1\"\u003eFIRMICUTES_._._PROTEOBACTERIA\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_annotation_background_color\" ref=\"A:1\"\u003e._._._.\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_annotation_background_edge_color\" ref=\"A:1\"\u003e._._._.\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_clade_marker_color\" ref=\"A:1\"\u003e#2d19ff_._._#29cc36\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_clade_marker_edge_width\" ref=\"A:1\"\u003e._._._.\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_clade_marker_size\" ref=\"A:1\"\u003e40_._._40\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"ignore_branch_len\" ref=\"A:1\"\u003e1\u003c/property\u003e\n-  \u003c/phylogeny\u003e\n-\u003c/phyloxml\u003e"], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmps547tb2u/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmps547tb2u/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nHistory item  different than expected, difference (using diff):\n( /tmp/tmpm404olifintermediary_tree v. /tmp/tmp2xlv9ozlintermediary_tree )\n--- local_file\n+++ history_data\n@@ -1,439 +0,0 @@\n-\u003cphyloxml xmlns=\"http://www.phyloxml.org\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsi:schemaLocation=\"http://www.phyloxml.org http://www.phyloxml.org/1.10/phyloxml.xsd\"\u003e\n-  \u003cphylogeny rooted=\"true\"\u003e\n-    \u003cclade\u003e\n-      \u003cclade\u003e\n-        \u003cname\u003ek__Archaea\u003c/name\u003e\n-        \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-        \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-        \u003cclade\u003e\n-          \u003cname\u003ep__Euryarchaeota\u003c/name\u003e\n-          \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-          \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-          \u003cclade\u003e\n-            \u003cname\u003ec__Methanobacteria\u003c/name\u003e\n-            \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-            \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-            \u003cclade\u003e\n-              \u003cname\u003eo__Methanobacteriales\u003c/name\u003e\n-              \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n-              \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e58.0748302504\u003c/property\u003e\n-              \u003cclade\u003e\n-                \u003cname\u003ef__Methanobacteriaceae\u003c/name\u003e\n-                \u003cbranch_length\u003e1.0\u003c/branch_length\u003e\n********\n*SNIP *\n********\n-                      \u003cproperty applies_to=\"clade\" datatype=\"xsd:string\" id_ref=\"clade_marker_size\" ref=\"A:1\"\u003e20.5761787104\u003c/property\u003e\n-                    \u003c/clade\u003e\n-                  \u003c/clade\u003e\n-                \u003c/clade\u003e\n-              \u003c/clade\u003e\n-            \u003c/clade\u003e\n-          \u003c/clade\u003e\n-        \u003c/clade\u003e\n-      \u003c/clade\u003e\n-    \u003c/clade\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"class_legend_marker_size\" ref=\"A:1\"\u003e1.5\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"class_legend_font_size\" ref=\"A:1\"\u003e10\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"branch_bracket_depth\" ref=\"A:1\"\u003e0.8\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"clade_separation\" ref=\"A:1\"\u003e0.5\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"branch_bracket_width\" ref=\"A:1\"\u003e0.2\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"annotation_legend_font_size\" ref=\"A:1\"\u003e10\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_keys\" ref=\"A:1\"\u003eFIRMICUTES_._._PROTEOBACTERIA\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_annotation_background_color\" ref=\"A:1\"\u003e._._._.\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_annotation_background_edge_color\" ref=\"A:1\"\u003e._._._.\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_clade_marker_color\" ref=\"A:1\"\u003e#2d19ff_._._#29cc36\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_clade_marker_edge_width\" ref=\"A:1\"\u003e._._._.\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"leg_clade_marker_size\" ref=\"A:1\"\u003e40_._._40\u003c/property\u003e\n-    \u003cproperty applies_to=\"phylogeny\" datatype=\"xsd:string\" id_ref=\"ignore_branch_len\" ref=\"A:1\"\u003e1\u003c/property\u003e\n-  \u003c/phylogeny\u003e\n-\u003c/phyloxml\u003e\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmps547tb2u/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027graphlan=1.0.0\u0027 -set REPO=\u0027quay.io/local/graphlan:1.0.0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n.\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 116.61629462242126, "tool_id": "graphlan_annotate", "tool_version": "1.0.0.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_graphlan_annotate.test_tool_000000"}, {"data": {"inputs": {"subnetwork": {"id": "2987a55ca6380b3a", "src": "hda"}}, "job": {"command_line": "python \u0027/home/runner/work/tools-iuc/tools-iuc/tools/heinz/visualization.py\u0027 -i \u0027/tmp/tmpwq4xd049/files/4/a/3/dataset_4a3d1135-f81f-40ab-8a3f-a706963dd63c.dat\u0027 -o /tmp/tmpwq4xd049/files/5/e/e/dataset_5ee70ad3-9c7d-4a90-b1ca-d24984b7b2eb.dat", "command_version": "", "create_time": "2020-01-23T17:07:19.360744", "exit_code": 0, "external_id": "19104", "galaxy_version": "20.01", "history_id": "3e4e8de146a035f5", "id": "a6b9cabbe61f73ca", "inputs": {"subnetwork": {"id": "2987a55ca6380b3a", "src": "hda", "uuid": "4a3d1135-f81f-40ab-8a3f-a706963dd63c"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"visualization": {"id": "fb9b740929ea3f54", "src": "hda", "uuid": "5ee70ad3-9c7d-4a90-b1ca-d24984b7b2eb"}}, "params": {"__input_ext": "\"txt\"", "chromInfo": "\"/tmp/tmpwq4xd049/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\""}, "state": "ok", "stderr": "", "stdout": "The visualization is saved as PDF!\n", "tool_id": "heinz_visualization", "tool_stderr": "", "tool_stdout": "The visualization is saved as PDF!\n", "update_time": "2020-01-23T17:07:38.436495", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using sim_size):\n( /tmp/tmpzy0vifxnheinz_graph.pdf v. /tmp/tmpp_p0atsaheinz_graph.pdf )\nFiles /tmp/tmpp_p0atsaheinz_graph.pdf=1765b but /tmp/tmpzy0vifxnheinz_graph.pdf=14246b - compare by size (delta=2000) failed"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpwq4xd049/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpwq4xd049/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpwq4xd049/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpwq4xd049/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpwq4xd049/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using sim_size):\n( /tmp/tmpzy0vifxnheinz_graph.pdf v. /tmp/tmpp_p0atsaheinz_graph.pdf )\nFiles /tmp/tmpp_p0atsaheinz_graph.pdf=1765b but /tmp/tmpzy0vifxnheinz_graph.pdf=14246b - compare by size (delta=2000) failed\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 32.86890196800232, "tool_id": "heinz_visualization", "tool_version": "0.1.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_heinz_visualization.test_tool_000000"}, {"data": {"inputs": {"hmmfile": {"id": "f16a54a3ef316466", "src": "hda"}, "notextw": true, "seed": "4", "seqfile": {"id": "e511a12dc2cf12e1", "src": "hda"}}, "output_problems": ["list index out of range"], "problem_log": "Traceback (most recent call last):\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 887, in _verify_outputs\n    output_data = data_list[name]\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 71, in __getitem__\n    raise KeyError(item)\nKeyError: None\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 792, in verify_tool\n    raise e\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 893, in _verify_outputs\n    output_data = list(data_list.values())[output_index]\nIndexError: list index out of range\n", "problem_type": "builtins.IndexError", "status": "error", "test_index": 0, "time_seconds": 20.343687295913696, "tool_id": "hmmer_nhmmer", "tool_version": "3.2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_hmmer_nhmmer.test_tool_000000"}, {"data": {"inputs": {"input_hmm_conditional|hmmfile": {"id": "8c9b639396181d1e", "src": "hda"}, "input_hmm_conditional|input_hmm_source": "history", "notextw": true, "seed": "4", "seqfile": {"id": "3f322b493a2c6407", "src": "hda"}}, "output_problems": ["list index out of range"], "problem_log": "Traceback (most recent call last):\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 887, in _verify_outputs\n    output_data = data_list[name]\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 71, in __getitem__\n    raise KeyError(item)\nKeyError: None\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 792, in verify_tool\n    raise e\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpqe2bq4_c/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 893, in _verify_outputs\n    output_data = list(data_list.values())[output_index]\nIndexError: list index out of range\n", "problem_type": "builtins.IndexError", "status": "error", "test_index": 0, "time_seconds": 20.49040126800537, "tool_id": "hmmer_nhmmscan", "tool_version": "3.2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_hmmer_nhmmscan.test_tool_000000"}, {"data": {"inputs": {"advanced_options|advanced_options_selector": "advanced", "advanced_options|samout_conditional|reference_source|ref_file": {"id": "0ed16744ccfa2592", "src": "hda"}, "advanced_options|samout_conditional|reference_source|reference_source_selector": "history", "advanced_options|samout_conditional|samout": "Yes", "gfffile": {"id": "87dca8aa4ee9d3de", "src": "hda"}, "samfile": {"id": "4163d05c3cc243fc", "src": "hda"}}, "job": {"command_line": "ln -s \"/tmp/tmpgs0ghv1y/files/2/1/2/dataset_212f7fd1-6810-4149-b355-8d140d548299.dat\" \"localref.fa\" \u0026\u0026 samtools faidx \u0027localref.fa\u0027 2\u003e\u00261 || echo \"Error running samtools faidx for htseq-count\" \u003e\u00262 \u0026\u0026  samtools view -Su -t \u0027localref.fa.fai\u0027 \u0027/tmp/tmpgs0ghv1y/files/0/6/c/dataset_06c9f7d4-63ee-4a0b-8a28-775914af56b7.dat\u0027 | samtools sort -n -T \"${TMPDIR:-.}\" --output-fmt=SAM -o \u0027name_sorted_alignment.sam\u0027 - \u0026\u0026  htseq-count --mode=union --stranded=yes --minaqual=10 --type=\u0027exon\u0027 --idattr=\u0027gene_id\u0027  --nonunique=none --secondary-alignments=score --supplementary-alignments=score --samout=\u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027  --order=name --format=sam \u0027name_sorted_alignment.sam\u0027  \"/tmp/tmpgs0ghv1y/files/c/b/2/dataset_cb2d1b74-3001-4519-b8c0-6a4344a590d2.dat\" | awk \u0027{if ($1 ~ \"no_feature|ambiguous|too_low_aQual|not_aligned|alignment_not_unique\") print $0 | \"cat 1\u003e\u00262\"; else print $0}\u0027 \u003e \u0027/tmp/tmpgs0ghv1y/files/e/6/a/dataset_e6ac16b1-262d-4b48-a13e-ba915eb09fad.dat\u0027 2\u003e \u0027/tmp/tmpgs0ghv1y/files/5/2/1/dataset_52140889-1d03-4eaf-af4e-5ccc65954b62.dat\u0027  \u0026\u0026 samtools view -Su -t \u0027localref.fa.fai\u0027 \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027 | samtools sort -T \"${TMPDIR:-.}\" -o \u0027/tmp/tmpgs0ghv1y/files/4/1/0/dataset_4104e12f-9b21-46cc-87e8-c2b06ab4621c.dat\u0027 -", "command_version": "version 0.9.1\n", "create_time": "2020-01-23T16:57:03.528396", "exit_code": 0, "external_id": "11604", "galaxy_version": "20.01", "history_id": "5ca39fba41b940d1", "id": "f8cc3c3467afd773", "inputs": {"advanced_options|samout_conditional|ref_file": {"id": "0ed16744ccfa2592", "src": "hda", "uuid": "212f7fd1-6810-4149-b355-8d140d548299"}, "gfffile": {"id": "87dca8aa4ee9d3de", "src": "hda", "uuid": "cb2d1b74-3001-4519-b8c0-6a4344a590d2"}, "samfile": {"id": "4163d05c3cc243fc", "src": "hda", "uuid": "06c9f7d4-63ee-4a0b-8a28-775914af56b7"}}, "job_messages": [{"code_desc": "Unknown error occured", "desc": "Fatal error: Unknown error occured", "error_level": 3, "match": "Error", "stream": "stderr", "type": "regex"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"counts": {"id": "a94ebc130565c5b1", "src": "hda", "uuid": "e6ac16b1-262d-4b48-a13e-ba915eb09fad"}, "othercounts": {"id": "c57c922a3f91222f", "src": "hda", "uuid": "52140889-1d03-4eaf-af4e-5ccc65954b62"}, "samoutfile": {"id": "315992a54eac5ecc", "src": "hda", "uuid": "4104e12f-9b21-46cc-87e8-c2b06ab4621c"}}, "params": {"__input_ext": "\"input\"", "advanced_options": "{\"__current_case__\": 1, \"advanced_options_selector\": \"advanced\", \"nonunique\": \"none\", \"samout_conditional\": {\"__current_case__\": 0, \"reference_source\": {\"__current_case__\": 1, \"ref_file\": {\"values\": [{\"id\": 77, \"src\": \"hda\"}]}, \"reference_source_selector\": \"history\"}, \"samout\": \"Yes\"}, \"secondary_alignments\": \"false\", \"supplementary_alignments\": \"false\"}", "chromInfo": "\"/tmp/tmpgs0ghv1y/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "featuretype": "\"exon\"", "idattr": "\"gene_id\"", "minaqual": "\"10\"", "mode": "\"union\"", "stranded": "\"yes\""}, "state": "error", "stderr": "  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n  [Exception type: FileNotFoundError, raised in count.py:52]\n[E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\nsamtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n", "stdout": "", "tool_id": "htseq_count", "tool_stderr": "  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n  [Exception type: FileNotFoundError, raised in count.py:52]\n[E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\nsamtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n", "tool_stdout": "", "update_time": "2020-01-23T16:57:09.471391", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state.", "Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpgs0ghv1y/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpgs0ghv1y/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpgs0ghv1y/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpgs0ghv1y/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpgs0ghv1y/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 5ca39fba41b940d1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - htseq-test.sam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,698 lines, 19 comments\n| Dataset Info:\n|  uploaded sam file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.QNAME\u003c/th\u003e\u003cth\u003e2.FLAG\u003c/th\u003e\u003cth\u003e3.RNAME\u003c/th\u003e\u003cth\u003e4.POS\u003c/th\u003e\u003cth\u003e5.MAPQ\u003c/th\u003e\u003cth\u003e6.CIGAR\u003c/th\u003e\u003cth\u003e7.MRNM\u003c/th\u003e\u003cth\u003e8.MPOS\u003c/th\u003e\u003cth\u003e9.ISIZE\u003c/th\u003e\u003cth\u003e10.SEQ\u003c/th\u003e\u003cth\u003e11.QUAL\u003c/th\u003e\u003cth\u003e12.OPT\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@HD\u003c/td\u003e\u003ctd\u003eVN:1.0\u003c/td\u003e\u003ctd\u003eSO:coordinate\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:2-micron\u003c/td\u003e\u003ctd\u003eLN:6318\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:MT\u003c/td\u003e\u003ctd\u003eLN:85779\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:I\u003c/td\u003e\u003ctd\u003eLN:230208\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:VI\u003c/td\u003e\u003ctd\u003eLN:270148\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - htseq-test.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  492 lines\n| Dataset Info:\n|  uploaded gtf file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqname\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Feature\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Frame\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eCDS\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e646\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;; protein_id \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estart_codon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e337\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estop_codon\u003c/td\u003e\u003ctd\u003e647\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e538\u003c/td\u003e\u003ctd\u003e792\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL068W-A\u0026#34;; transcript_id \u0026#34;YAL068W-A\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL068W-A\u0026#34;; transcript_name \u0026#34;YAL068W-A\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - htseq-test_reference.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  18 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;2-micron\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;MT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;I\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - htseq-count on data 3, data 2, and data 1 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 5 - htseq-count on data 3, data 2, and data 1 (no feature) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 6 - htseq-count on data 3, data 2, and data 1 (BAM) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| Job f8cc3c3467afd773\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:57:09.471391\n| Create Time:\n|  2020-01-23T16:57:03.528396\n|\n--------------------------------------\n| Job 85f590476b78f8ed\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:02.157505\n| Create Time:\n|  2020-01-23T16:56:56.731714\n|\n--------------------------------------\n| Job 48f3e8809447c25a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:55.801479\n| Create Time:\n|  2020-01-23T16:56:49.987394\n|\n--------------------------------------\n| Job baa1080356b320b7\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:48.528630\n| Create Time:\n|  2020-01-23T16:56:43.273645\n|\n--------------------------------------\nProblem in history with id 5ca39fba41b940d1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - htseq-test.sam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,698 lines, 19 comments\n| Dataset Info:\n|  uploaded sam file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.QNAME\u003c/th\u003e\u003cth\u003e2.FLAG\u003c/th\u003e\u003cth\u003e3.RNAME\u003c/th\u003e\u003cth\u003e4.POS\u003c/th\u003e\u003cth\u003e5.MAPQ\u003c/th\u003e\u003cth\u003e6.CIGAR\u003c/th\u003e\u003cth\u003e7.MRNM\u003c/th\u003e\u003cth\u003e8.MPOS\u003c/th\u003e\u003cth\u003e9.ISIZE\u003c/th\u003e\u003cth\u003e10.SEQ\u003c/th\u003e\u003cth\u003e11.QUAL\u003c/th\u003e\u003cth\u003e12.OPT\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@HD\u003c/td\u003e\u003ctd\u003eVN:1.0\u003c/td\u003e\u003ctd\u003eSO:coordinate\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:2-micron\u003c/td\u003e\u003ctd\u003eLN:6318\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:MT\u003c/td\u003e\u003ctd\u003eLN:85779\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:I\u003c/td\u003e\u003ctd\u003eLN:230208\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:VI\u003c/td\u003e\u003ctd\u003eLN:270148\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - htseq-test.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  492 lines\n| Dataset Info:\n|  uploaded gtf file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqname\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Feature\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Frame\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eCDS\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e646\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;; protein_id \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estart_codon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e337\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estop_codon\u003c/td\u003e\u003ctd\u003e647\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e538\u003c/td\u003e\u003ctd\u003e792\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL068W-A\u0026#34;; transcript_id \u0026#34;YAL068W-A\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL068W-A\u0026#34;; transcript_name \u0026#34;YAL068W-A\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - htseq-test_reference.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  18 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;2-micron\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;MT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;I\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - htseq-count on data 3, data 2, and data 1 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 5 - htseq-count on data 3, data 2, and data 1 (no feature) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 6 - htseq-count on data 3, data 2, and data 1 (BAM) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| Job f8cc3c3467afd773\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:57:09.471391\n| Create Time:\n|  2020-01-23T16:57:03.528396\n|\n--------------------------------------\n| Job 85f590476b78f8ed\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:02.157505\n| Create Time:\n|  2020-01-23T16:56:56.731714\n|\n--------------------------------------\n| Job 48f3e8809447c25a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:55.801479\n| Create Time:\n|  2020-01-23T16:56:49.987394\n|\n--------------------------------------\n| Job baa1080356b320b7\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:48.528630\n| Create Time:\n|  2020-01-23T16:56:43.273645\n|\n--------------------------------------\nProblem in history with id 5ca39fba41b940d1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - htseq-test.sam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,698 lines, 19 comments\n| Dataset Info:\n|  uploaded sam file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.QNAME\u003c/th\u003e\u003cth\u003e2.FLAG\u003c/th\u003e\u003cth\u003e3.RNAME\u003c/th\u003e\u003cth\u003e4.POS\u003c/th\u003e\u003cth\u003e5.MAPQ\u003c/th\u003e\u003cth\u003e6.CIGAR\u003c/th\u003e\u003cth\u003e7.MRNM\u003c/th\u003e\u003cth\u003e8.MPOS\u003c/th\u003e\u003cth\u003e9.ISIZE\u003c/th\u003e\u003cth\u003e10.SEQ\u003c/th\u003e\u003cth\u003e11.QUAL\u003c/th\u003e\u003cth\u003e12.OPT\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@HD\u003c/td\u003e\u003ctd\u003eVN:1.0\u003c/td\u003e\u003ctd\u003eSO:coordinate\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:2-micron\u003c/td\u003e\u003ctd\u003eLN:6318\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:MT\u003c/td\u003e\u003ctd\u003eLN:85779\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:I\u003c/td\u003e\u003ctd\u003eLN:230208\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:VI\u003c/td\u003e\u003ctd\u003eLN:270148\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - htseq-test.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  492 lines\n| Dataset Info:\n|  uploaded gtf file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqname\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Feature\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Frame\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eCDS\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e646\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;; protein_id \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estart_codon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e337\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estop_codon\u003c/td\u003e\u003ctd\u003e647\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e538\u003c/td\u003e\u003ctd\u003e792\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL068W-A\u0026#34;; transcript_id \u0026#34;YAL068W-A\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL068W-A\u0026#34;; transcript_name \u0026#34;YAL068W-A\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - htseq-test_reference.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  18 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;2-micron\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;MT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;I\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - htseq-count on data 3, data 2, and data 1 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 5 - htseq-count on data 3, data 2, and data 1 (no feature) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 6 - htseq-count on data 3, data 2, and data 1 (BAM) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| Job f8cc3c3467afd773\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:57:09.471391\n| Create Time:\n|  2020-01-23T16:57:03.528396\n|\n--------------------------------------\n| Job 85f590476b78f8ed\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:02.157505\n| Create Time:\n|  2020-01-23T16:56:56.731714\n|\n--------------------------------------\n| Job 48f3e8809447c25a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:55.801479\n| Create Time:\n|  2020-01-23T16:56:49.987394\n|\n--------------------------------------\n| Job baa1080356b320b7\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:48.528630\n| Create Time:\n|  2020-01-23T16:56:43.273645\n|\n--------------------------------------\nProblem in history with id 5ca39fba41b940d1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - htseq-test.sam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,698 lines, 19 comments\n| Dataset Info:\n|  uploaded sam file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.QNAME\u003c/th\u003e\u003cth\u003e2.FLAG\u003c/th\u003e\u003cth\u003e3.RNAME\u003c/th\u003e\u003cth\u003e4.POS\u003c/th\u003e\u003cth\u003e5.MAPQ\u003c/th\u003e\u003cth\u003e6.CIGAR\u003c/th\u003e\u003cth\u003e7.MRNM\u003c/th\u003e\u003cth\u003e8.MPOS\u003c/th\u003e\u003cth\u003e9.ISIZE\u003c/th\u003e\u003cth\u003e10.SEQ\u003c/th\u003e\u003cth\u003e11.QUAL\u003c/th\u003e\u003cth\u003e12.OPT\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@HD\u003c/td\u003e\u003ctd\u003eVN:1.0\u003c/td\u003e\u003ctd\u003eSO:coordinate\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:2-micron\u003c/td\u003e\u003ctd\u003eLN:6318\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:MT\u003c/td\u003e\u003ctd\u003eLN:85779\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:I\u003c/td\u003e\u003ctd\u003eLN:230208\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@SQ\u003c/td\u003e\u003ctd\u003eSN:VI\u003c/td\u003e\u003ctd\u003eLN:270148\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - htseq-test.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  492 lines\n| Dataset Info:\n|  uploaded gtf file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqname\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Feature\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Frame\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eCDS\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e646\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;; protein_id \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estart_codon\u003c/td\u003e\u003ctd\u003e335\u003c/td\u003e\u003ctd\u003e337\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003estop_codon\u003c/td\u003e\u003ctd\u003e647\u003c/td\u003e\u003ctd\u003e649\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL069W\u0026#34;; transcript_id \u0026#34;YAL069W\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL069W\u0026#34;; transcript_name \u0026#34;YAL069W\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eI\u003c/td\u003e\u003ctd\u003eprotein_coding\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e538\u003c/td\u003e\u003ctd\u003e792\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e gene_id \u0026#34;YAL068W-A\u0026#34;; transcript_id \u0026#34;YAL068W-A\u0026#34;; exon_number \u0026#34;1\u0026#34;; gene_name \u0026#34;YAL068W-A\u0026#34;; transcript_name \u0026#34;YAL068W-A\u0026#34;;\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - htseq-test_reference.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  18 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;2-micron\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;MT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eactg\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;I\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - htseq-count on data 3, data 2, and data 1 (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 5 - htseq-count on data 3, data 2, and data 1 (no feature) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| 6 - htseq-count on data 3, data 2, and data 1 (BAM) (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/t\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [Errno 2] No such file or directory: \u0027/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\u0027\n|  [Exception type: FileNotFoundError, raised in count.py:52]\n|  [E::hts_open_format] Failed to open file /tmp/tmpgs0ghv1y/tmp/80_tmp.sam\n|  samtools view: failed to open \"/tmp/tmpgs0ghv1y/tmp/80_tmp.sam\" for reading: No such file or directory\n|\n--------------------------------------\n| Job f8cc3c3467afd773\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:57:09.471391\n| Create Time:\n|  2020-01-23T16:57:03.528396\n|\n--------------------------------------\n| Job 85f590476b78f8ed\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:02.157505\n| Create Time:\n|  2020-01-23T16:56:56.731714\n|\n--------------------------------------\n| Job 48f3e8809447c25a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:55.801479\n| Create Time:\n|  2020-01-23T16:56:49.987394\n|\n--------------------------------------\n| Job baa1080356b320b7\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:56:48.528630\n| Create Time:\n|  2020-01-23T16:56:43.273645\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 5, "time_seconds": 29.378702640533447, "tool_id": "htseq_count", "tool_version": "0.9.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_htseq_count.test_tool_000005"}, {"data": {"inputs": {"adv|annotation_gene_index": "8", "adv|evalue": "1", "adv|gap_fill": false, "adv|identity_threshold": "50", "adv|minpath": true, "adv|output_format": "tsv", "adv|output_max_dec": "10", "adv|pick_frames": true, "adv|prescreen_threshold": "0.01", "adv|remove_column_description_output": false, "adv|remove_statified_output": false, "adv|translated_query_coverage_threshold": "50", "adv|translated_subject_coverage_threshold": "50", "adv|xipe": false, "bypass|bypass_nucleotide_index": false, "bypass|bypass_nucleotide_search": false, "bypass|bypass_prescreen": false, "bypass|bypass_translated_search": false, "input": {"id": "73de4610aed2490c", "src": "hda"}, "nucleotide_db|nucleotide_database": {"id": "2891970512fa2d5a", "src": "hdca"}, "nucleotide_db|nucleotide_db_selector": "history", "pathways": "metacyc", "protein_db|protein_database": {"id": "3d556e8e35b754dc", "src": "hda"}, "protein_db|protein_db_selector": "history", "search_mode": "uniref50", "taxo_profile|taxonomic_profile": {"id": "df545be7ad5cd764", "src": "hda"}, "taxo_profile|taxonomic_profile_test": "true", "translated_alignment": "diamond"}, "job": {"command_line": "mkdir nucleotide_db \u0026\u0026 cp \u0027/tmp/tmphzs4hnom/files/5/f/9/dataset_5f9f6030-5edc-4b96-878b-898699598c4a.dat\u0027 \u0027nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz\u0027 \u0026\u0026 cp \u0027/tmp/tmphzs4hnom/files/8/b/4/dataset_8b4cebc0-4015-4b24-aad9-ff67d88c92db.dat\u0027 \u0027nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\u0027 \u0026\u0026  mkdir protein_db \u0026\u0026 diamond makedb --in \u0027/tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\u0027 --db \u0027protein_db/protein_db\u0027 --threads \"${GALAXY_SLOTS:-4}\" \u0026\u0026   humann2 --input \u0027/tmp/tmphzs4hnom/files/8/d/b/dataset_8db6f23a-434c-4a7b-8c91-d8953aea8e3f.dat\u0027 -o \u0027output\u0027      --taxonomic-profile \u0027/tmp/tmphzs4hnom/files/4/a/4/dataset_4a4745a3-7722-4a3b-8640-78d579155f40.dat\u0027 --metaphlan-options=\"-t rel_ab\"  --nucleotide-database nucleotide_db  --translated-alignment \u0027diamond\u0027 --protein-database protein_db --search-mode \u0027uniref50\u0027 --pathways \u0027metacyc\u0027  --annotation-gene-index 8 --evalue \u00271.0\u0027 --threads \"${GALAXY_SLOTS:-4}\" --memory-use minimum --prescreen-threshold \u00270.01\u0027 --identity-threshold \u002750.0\u0027 --translated-subject-coverage-threshold \u002750.0\u0027 --translated-query-coverage-threshold \u002750.0\u0027 --xipe \u0027off\u0027 --minpath \u0027on\u0027 --pick-frames \u0027on\u0027 --gap-fill \u0027off\u0027 --output-format \u0027tsv\u0027 --output-max-decimals \u002710\u0027 --output-basename \u0027humann2\u0027", "command_version": "humann2 v0.11.1\n", "create_time": "2020-01-23T16:58:46.071121", "exit_code": 1, "external_id": "12555", "galaxy_version": "20.01", "history_id": "a1e670e1ea0da54b", "id": "288a67f7f32bb2fd", "inputs": {"input": {"id": "73de4610aed2490c", "src": "hda", "uuid": "8db6f23a-434c-4a7b-8c91-d8953aea8e3f"}, "nucleotide_database1": {"id": "19ec6936a7590160", "src": "hda", "uuid": "5f9f6030-5edc-4b96-878b-898699598c4a"}, "nucleotide_database2": {"id": "330fe5822b24a25f", "src": "hda", "uuid": "8b4cebc0-4015-4b24-aad9-ff67d88c92db"}, "protein_database": {"id": "3d556e8e35b754dc", "src": "hda", "uuid": "75dfcd53-d0f4-469d-9374-8c632c3a76c5"}, "taxonomic_profile": {"id": "df545be7ad5cd764", "src": "hda", "uuid": "4a4745a3-7722-4a3b-8640-78d579155f40"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"gene_families_tsv": {"id": "adcc13f516c5d573", "src": "hda", "uuid": "069c0063-0e22-4d83-be78-84521ce137ee"}, "pathabundance_tsv": {"id": "e5cd870596a22f3d", "src": "hda", "uuid": "7c2971d4-51ee-4ae4-a949-a7c29e6dbf28"}, "pathcoverage_tsv": {"id": "97f8596e58df898c", "src": "hda", "uuid": "94eab187-4e2d-477b-afd5-5e4ba5728faf"}}, "params": {"__input_ext": "\"fasta\"", "adv": "{\"annotation_gene_index\": \"8\", \"evalue\": \"1.0\", \"gap_fill\": \"false\", \"id_mapping\": null, \"identity_threshold\": \"50.0\", \"minpath\": \"true\", \"output_format\": \"tsv\", \"output_max_dec\": \"10\", \"pick_frames\": \"true\", \"prescreen_threshold\": \"0.01\", \"remove_column_description_output\": \"false\", \"remove_statified_output\": \"false\", \"translated_query_coverage_threshold\": \"50.0\", \"translated_subject_coverage_threshold\": \"50.0\", \"xipe\": \"false\"}", "bypass": "{\"bypass_nucleotide_index\": \"false\", \"bypass_nucleotide_search\": \"false\", \"bypass_prescreen\": \"false\", \"bypass_translated_search\": \"false\"}", "chromInfo": "\"/tmp/tmphzs4hnom/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "nucleotide_db": "{\"__current_case__\": 1, \"nucleotide_database\": {\"values\": [{\"id\": 1, \"src\": \"hdca\"}]}, \"nucleotide_db_selector\": \"history\"}", "pathways": "\"metacyc\"", "protein_db": "{\"__current_case__\": 1, \"protein_database\": {\"values\": [{\"id\": 89, \"src\": \"hda\"}]}, \"protein_db_selector\": \"history\"}", "search_mode": "\"uniref50\"", "taxo_profile": "{\"__current_case__\": 0, \"taxonomic_profile\": {\"values\": [{\"id\": 86, \"src\": \"hda\"}]}, \"taxonomic_profile_test\": \"true\"}", "translated_alignment": "\"diamond\""}, "state": "error", "stderr": "gunzip: invalid magic\ngunzip: invalid magic\nCRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n\n", "stdout": "diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\nCheck http://github.com/bbuchfink/diamond for updates.\n\n#CPU threads: 1\nDatabase file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\nOpening the database file...  [0.000141s]\nLoading sequence data (0 sequences processed)...  [0.000566s]\nWriting trailer...  [8e-06s]\nClosing the database file...  [2.5e-05s]\nProcessed 151 sequences, 74681 letters.\nTotal time = 0.000807s\nCreating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\nOutput files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\nRemoving spaces from identifiers in input file ...\n\nFound g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\nFound g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n\nTotal species selected from prescreen: 2\n\nSelected species explain 100.00% of predicted community composition\n\n\nCreating custom ChocoPhlAn database ........\n\n", "tool_id": "humann2", "tool_stderr": "gunzip: invalid magic\ngunzip: invalid magic\nCRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n\n", "tool_stdout": "diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\nCheck http://github.com/bbuchfink/diamond for updates.\n\n#CPU threads: 1\nDatabase file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\nOpening the database file...  [0.000141s]\nLoading sequence data (0 sequences processed)...  [0.000566s]\nWriting trailer...  [8e-06s]\nClosing the database file...  [2.5e-05s]\nProcessed 151 sequences, 74681 letters.\nTotal time = 0.000807s\nCreating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\nOutput files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\nRemoving spaces from identifiers in input file ...\n\nFound g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\nFound g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n\nTotal species selected from prescreen: 2\n\nSelected species explain 100.00% of predicted community composition\n\n\nCreating custom ChocoPhlAn database ........\n\n", "update_time": "2020-01-23T17:00:01.114100", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state.", "Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmphzs4hnom/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmphzs4hnom/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmphzs4hnom/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmphzs4hnom/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmphzs4hnom/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id a1e670e1ea0da54b - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - input_sequences.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  77 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r2|637000026.fna|5753889|5754040|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln383_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTCCGTCCGCTGGTAGAACGTCTGGTAGAAGCTCAAAAAGCATTGGCAACCAAATACCTTTCCGAAGCCAAACGACTGATTGCCTCCAACGACAAGAAGGAAGTGGAAGAAGGATTCCTTGCCCTTTATCGTAGCCACAAGTGTCTTCCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r3|637000026.fna|1749333|1749484|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln165_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAAGTAAAAGTCTACACACAGGAAGGAACGATAATGGAAAGTACGATTGAAGTAAATCGTCCGATGGAGATAGCAGGATGGAAAATTTATCAGCTTAGTTATGATGAGTCGAAAGGTCGCTGGAGCGATATCAGTGTCTTTGAGCTGGTTCG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r8|637000026.fna|2991814|2991965|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln402_#0/1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - taxonomic_profile.tabular (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  10 lines, 1 comments\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#SampleID\u003c/td\u003e\u003ctd\u003eMetaphlan2_Analysis\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,687 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  6,355 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - reduced_uniref50.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  151 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_B0NQY6 Uncharacterized protein n=1 Tax=Bacteroides stercoris ATCC 43183 RepID=B0NQY6_BACSE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMYFRAQEVKNTSKSKRGKIRFTLILIMQETIYELYVAKVLIFLINGIERKEFSFGLRIFA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eQS\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_F3PHD0 Uncharacterized protein n=2 Tax=Bacteroides RepID=F3PHD0_9BACE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMIQTEFQHLLKPVPTAGIGDTKHWYEEYQPLAQTIPSLGTSNTGRWYERYQALEREFFPG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - Unnamed Collection (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 2, \u0027elements\u0027: [{\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1\u0027, \u0027element_index\u0027: 0, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:25.275504\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 5128810, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 3, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u002719ec6936a7590160\u0027, \u0027metadata_data_lines\u0027: 81023, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 4687, \u0027misc_blurb\u0027: \u00274,687 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074711\u0027, \u0027uuid\u0027: \u00275f9f6030-5edc-4b96-878b-898699598c4a\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}, {\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1\u0027, \u0027element_index\u0027: 1, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00275729865256bc2525\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:32.116962\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 8221523, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 4, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u0027330fe5822b24a25f\u0027, \u0027metadata_data_lines\u0027: 130067, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 6355, \u0027misc_blurb\u0027: \u00276,355 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074713\u0027, \u0027uuid\u0027: \u00278b4cebc0-4015-4b24-aad9-ff67d88c92db\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}], \u0027hid\u0027: 6, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027job_source_id\u0027: None, \u0027job_source_type\u0027: None, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027Unnamed Collection\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/a1e670e1ea0da54b/contents/dataset_collections/2891970512fa2d5a\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 7 - HUMAnN2 on data 5, data 4, and others: Gene families and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 8 - HUMAnN2 on data 5, data 4, and others: Pathways and their coverage (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 9 - HUMAnN2 on data 5, data 4, and others: Pathways and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| Job 288a67f7f32bb2fd\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:00:01.114100\n| Create Time:\n|  2020-01-23T16:58:46.071121\n|\n--------------------------------------\n| Job 5617b66919498d02\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:44.562752\n| Create Time:\n|  2020-01-23T16:58:38.904791\n|\n--------------------------------------\n| Job fb9b740929ea3f54\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:37.394554\n| Create Time:\n|  2020-01-23T16:58:32.161922\n|\n--------------------------------------\n| Job 2987a55ca6380b3a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:31.119150\n| Create Time:\n|  2020-01-23T16:58:25.320793\n|\n--------------------------------------\n| Job 087a78e08fef416e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:23.619643\n| Create Time:\n|  2020-01-23T16:58:18.548732\n|\n--------------------------------------\n| Job 9649d2d367737693\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:17.372779\n| Create Time:\n|  2020-01-23T16:58:11.738714\n|\n--------------------------------------\nProblem in history with id a1e670e1ea0da54b - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - input_sequences.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  77 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r2|637000026.fna|5753889|5754040|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln383_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTCCGTCCGCTGGTAGAACGTCTGGTAGAAGCTCAAAAAGCATTGGCAACCAAATACCTTTCCGAAGCCAAACGACTGATTGCCTCCAACGACAAGAAGGAAGTGGAAGAAGGATTCCTTGCCCTTTATCGTAGCCACAAGTGTCTTCCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r3|637000026.fna|1749333|1749484|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln165_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAAGTAAAAGTCTACACACAGGAAGGAACGATAATGGAAAGTACGATTGAAGTAAATCGTCCGATGGAGATAGCAGGATGGAAAATTTATCAGCTTAGTTATGATGAGTCGAAAGGTCGCTGGAGCGATATCAGTGTCTTTGAGCTGGTTCG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r8|637000026.fna|2991814|2991965|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln402_#0/1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - taxonomic_profile.tabular (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  10 lines, 1 comments\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#SampleID\u003c/td\u003e\u003ctd\u003eMetaphlan2_Analysis\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,687 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  6,355 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - reduced_uniref50.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  151 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_B0NQY6 Uncharacterized protein n=1 Tax=Bacteroides stercoris ATCC 43183 RepID=B0NQY6_BACSE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMYFRAQEVKNTSKSKRGKIRFTLILIMQETIYELYVAKVLIFLINGIERKEFSFGLRIFA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eQS\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_F3PHD0 Uncharacterized protein n=2 Tax=Bacteroides RepID=F3PHD0_9BACE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMIQTEFQHLLKPVPTAGIGDTKHWYEEYQPLAQTIPSLGTSNTGRWYERYQALEREFFPG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - Unnamed Collection (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 2, \u0027elements\u0027: [{\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1\u0027, \u0027element_index\u0027: 0, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:25.275504\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 5128810, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 3, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u002719ec6936a7590160\u0027, \u0027metadata_data_lines\u0027: 81023, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 4687, \u0027misc_blurb\u0027: \u00274,687 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074711\u0027, \u0027uuid\u0027: \u00275f9f6030-5edc-4b96-878b-898699598c4a\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}, {\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1\u0027, \u0027element_index\u0027: 1, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00275729865256bc2525\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:32.116962\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 8221523, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 4, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u0027330fe5822b24a25f\u0027, \u0027metadata_data_lines\u0027: 130067, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 6355, \u0027misc_blurb\u0027: \u00276,355 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074713\u0027, \u0027uuid\u0027: \u00278b4cebc0-4015-4b24-aad9-ff67d88c92db\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}], \u0027hid\u0027: 6, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027job_source_id\u0027: None, \u0027job_source_type\u0027: None, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027Unnamed Collection\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/a1e670e1ea0da54b/contents/dataset_collections/2891970512fa2d5a\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 7 - HUMAnN2 on data 5, data 4, and others: Gene families and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 8 - HUMAnN2 on data 5, data 4, and others: Pathways and their coverage (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 9 - HUMAnN2 on data 5, data 4, and others: Pathways and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| Job 288a67f7f32bb2fd\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:00:01.114100\n| Create Time:\n|  2020-01-23T16:58:46.071121\n|\n--------------------------------------\n| Job 5617b66919498d02\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:44.562752\n| Create Time:\n|  2020-01-23T16:58:38.904791\n|\n--------------------------------------\n| Job fb9b740929ea3f54\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:37.394554\n| Create Time:\n|  2020-01-23T16:58:32.161922\n|\n--------------------------------------\n| Job 2987a55ca6380b3a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:31.119150\n| Create Time:\n|  2020-01-23T16:58:25.320793\n|\n--------------------------------------\n| Job 087a78e08fef416e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:23.619643\n| Create Time:\n|  2020-01-23T16:58:18.548732\n|\n--------------------------------------\n| Job 9649d2d367737693\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:17.372779\n| Create Time:\n|  2020-01-23T16:58:11.738714\n|\n--------------------------------------\nProblem in history with id a1e670e1ea0da54b - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - input_sequences.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  77 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r2|637000026.fna|5753889|5754040|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln383_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTCCGTCCGCTGGTAGAACGTCTGGTAGAAGCTCAAAAAGCATTGGCAACCAAATACCTTTCCGAAGCCAAACGACTGATTGCCTCCAACGACAAGAAGGAAGTGGAAGAAGGATTCCTTGCCCTTTATCGTAGCCACAAGTGTCTTCCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r3|637000026.fna|1749333|1749484|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln165_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAAGTAAAAGTCTACACACAGGAAGGAACGATAATGGAAAGTACGATTGAAGTAAATCGTCCGATGGAGATAGCAGGATGGAAAATTTATCAGCTTAGTTATGATGAGTCGAAAGGTCGCTGGAGCGATATCAGTGTCTTTGAGCTGGTTCG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r8|637000026.fna|2991814|2991965|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln402_#0/1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - taxonomic_profile.tabular (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  10 lines, 1 comments\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#SampleID\u003c/td\u003e\u003ctd\u003eMetaphlan2_Analysis\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,687 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  6,355 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - reduced_uniref50.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  151 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_B0NQY6 Uncharacterized protein n=1 Tax=Bacteroides stercoris ATCC 43183 RepID=B0NQY6_BACSE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMYFRAQEVKNTSKSKRGKIRFTLILIMQETIYELYVAKVLIFLINGIERKEFSFGLRIFA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eQS\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_F3PHD0 Uncharacterized protein n=2 Tax=Bacteroides RepID=F3PHD0_9BACE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMIQTEFQHLLKPVPTAGIGDTKHWYEEYQPLAQTIPSLGTSNTGRWYERYQALEREFFPG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - Unnamed Collection (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 2, \u0027elements\u0027: [{\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1\u0027, \u0027element_index\u0027: 0, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:25.275504\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 5128810, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 3, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u002719ec6936a7590160\u0027, \u0027metadata_data_lines\u0027: 81023, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 4687, \u0027misc_blurb\u0027: \u00274,687 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074711\u0027, \u0027uuid\u0027: \u00275f9f6030-5edc-4b96-878b-898699598c4a\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}, {\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1\u0027, \u0027element_index\u0027: 1, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00275729865256bc2525\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:32.116962\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 8221523, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 4, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u0027330fe5822b24a25f\u0027, \u0027metadata_data_lines\u0027: 130067, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 6355, \u0027misc_blurb\u0027: \u00276,355 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074713\u0027, \u0027uuid\u0027: \u00278b4cebc0-4015-4b24-aad9-ff67d88c92db\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}], \u0027hid\u0027: 6, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027job_source_id\u0027: None, \u0027job_source_type\u0027: None, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027Unnamed Collection\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/a1e670e1ea0da54b/contents/dataset_collections/2891970512fa2d5a\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 7 - HUMAnN2 on data 5, data 4, and others: Gene families and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 8 - HUMAnN2 on data 5, data 4, and others: Pathways and their coverage (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 9 - HUMAnN2 on data 5, data 4, and others: Pathways and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| Job 288a67f7f32bb2fd\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:00:01.114100\n| Create Time:\n|  2020-01-23T16:58:46.071121\n|\n--------------------------------------\n| Job 5617b66919498d02\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:44.562752\n| Create Time:\n|  2020-01-23T16:58:38.904791\n|\n--------------------------------------\n| Job fb9b740929ea3f54\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:37.394554\n| Create Time:\n|  2020-01-23T16:58:32.161922\n|\n--------------------------------------\n| Job 2987a55ca6380b3a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:31.119150\n| Create Time:\n|  2020-01-23T16:58:25.320793\n|\n--------------------------------------\n| Job 087a78e08fef416e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:23.619643\n| Create Time:\n|  2020-01-23T16:58:18.548732\n|\n--------------------------------------\n| Job 9649d2d367737693\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:17.372779\n| Create Time:\n|  2020-01-23T16:58:11.738714\n|\n--------------------------------------\nProblem in history with id a1e670e1ea0da54b - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - input_sequences.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  77 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r2|637000026.fna|5753889|5754040|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln383_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTCCGTCCGCTGGTAGAACGTCTGGTAGAAGCTCAAAAAGCATTGGCAACCAAATACCTTTCCGAAGCCAAACGACTGATTGCCTCCAACGACAAGAAGGAAGTGGAAGAAGGATTCCTTGCCCTTTATCGTAGCCACAAGTGTCTTCCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r3|637000026.fna|1749333|1749484|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln165_#0/1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAAGTAAAAGTCTACACACAGGAAGGAACGATAATGGAAAGTACGATTGAAGTAAATCGTCCGATGGAGATAGCAGGATGGAAAATTTATCAGCTTAGTTATGATGAGTCGAAAGGTCGCTGGAGCGATATCAGTGTCTTTGAGCTGGTTCG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;r8|637000026.fna|2991814|2991965|_from_ |NC_004663_Bacteroides_thetaiotaomicron_VPI-5482,_c|_ln402_#0/1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - taxonomic_profile.tabular (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  10 lines, 1 comments\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#SampleID\u003c/td\u003e\u003ctd\u003eMetaphlan2_Analysis\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ek__Bacteria|p__Bacteroidetes|c__Bacteroidia|o__Bacteroidales\u003c/td\u003e\u003ctd\u003e100.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,687 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  6,355 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - reduced_uniref50.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  151 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_B0NQY6 Uncharacterized protein n=1 Tax=Bacteroides stercoris ATCC 43183 RepID=B0NQY6_BACSE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMYFRAQEVKNTSKSKRGKIRFTLILIMQETIYELYVAKVLIFLINGIERKEFSFGLRIFA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eQS\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;UniRef50_F3PHD0 Uncharacterized protein n=2 Tax=Bacteroides RepID=F3PHD0_9BACE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eMIQTEFQHLLKPVPTAGIGDTKHWYEEYQPLAQTIPSLGTSNTGRWYERYQALEREFFPG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - Unnamed Collection (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 2, \u0027elements\u0027: [{\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1\u0027, \u0027element_index\u0027: 0, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:25.275504\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 5128810, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 3, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u002719ec6936a7590160\u0027, \u0027metadata_data_lines\u0027: 81023, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 4687, \u0027misc_blurb\u0027: \u00274,687 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|223955863|ref|NZ_DS499664.1|:c18274-17102|46506|g__Bacteroides.s__Bacteroides_stercoris|UniRef90_R6FG07|UniRef50_R6FG07|1173\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATGGCTGCTATGTTCCAAGACAAATACGTATTCTCTCAATTAACCGCTTTTCTGAACAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACTCAGTTCAACAACTATGTTCGCAAGTATGATGGCAACAGATATGTGAAACATTTCACT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGCTGGAATCAGATGCTCGCGATGATGTTTGGACAACTGAGTAACCGTGAGAGCCTGCGA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGACTTAGTCGTTGCTTTCGAGGCGCATAGGGCCAAGCAATATCATCTTGGTTTAGGTCGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074711\u0027, \u0027uuid\u0027: \u00275f9f6030-5edc-4b96-878b-898699598c4a\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}, {\u0027element_identifier\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1\u0027, \u0027element_index\u0027: 1, \u0027element_type\u0027: \u0027hda\u0027, \u0027id\u0027: \u00275729865256bc2525\u0027, \u0027model_class\u0027: \u0027DatasetCollectionElement\u0027, \u0027object\u0027: {\u0027create_time\u0027: \u00272020-01-23T16:58:32.116962\u0027, \u0027data_type\u0027: \u0027galaxy.datatypes.sequence.Fasta\u0027, \u0027deleted\u0027: False, \u0027file_ext\u0027: \u0027fasta\u0027, \u0027file_size\u0027: 8221523, \u0027genome_build\u0027: \u0027?\u0027, \u0027hda_ldda\u0027: \u0027hda\u0027, \u0027hid\u0027: 4, \u0027history_content_type\u0027: \u0027dataset\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u0027330fe5822b24a25f\u0027, \u0027metadata_data_lines\u0027: 130067, \u0027metadata_dbkey\u0027: \u0027?\u0027, \u0027metadata_sequences\u0027: 6355, \u0027misc_blurb\u0027: \u00276,355 sequences\u0027, \u0027misc_info\u0027: \u0027uploaded fasta file\u0027, \u0027model_class\u0027: \u0027HistoryDatasetAssociation\u0027, \u0027name\u0027: \u0027g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\u0027, \u0027peek\u0027: \u0027\u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;gi|29345410|ref|NC_004663.1|:3331979-3332872|818|g__Bacteroides.s__Bacteroides_thetaiotaomicron|UniRef90_Q877J3|UniRef50_F0R2U7|894\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGTGGTCTCTCTTTGCAAATTGTTTGGTGTAACAAAGCAAGCCTTTTATAAGTATGTTGAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCATAGTACGGATAGTTCGGCCCGTGAACGTTTTGTTCTTGAGTTTGTCAAACGTGTGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCAAAGGACCCTGGTATTGGGGGGATGAAACTTTGGCTGATGTACCGTAATGAGTTCGGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACCAGCCAGGCCTTTGTTGGGCGTGACTGTTTCTGTGCTATATTGTCCAAGTATAAGCTG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\u0027, \u0027purged\u0027: False, \u0027state\u0027: \u0027ok\u0027, \u0027tags\u0027: [], \u0027update_time\u0027: \u00272020-01-23T16:58:46.074713\u0027, \u0027uuid\u0027: \u00278b4cebc0-4015-4b24-aad9-ff67d88c92db\u0027, \u0027validated_state\u0027: \u0027unknown\u0027, \u0027validated_state_message\u0027: None, \u0027visible\u0027: True}}], \u0027hid\u0027: 6, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u0027a1e670e1ea0da54b\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027job_source_id\u0027: None, \u0027job_source_type\u0027: None, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027Unnamed Collection\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/a1e670e1ea0da54b/contents/dataset_collections/2891970512fa2d5a\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 7 - HUMAnN2 on data 5, data 4, and others: Gene families and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 8 - HUMAnN2 on data 5, data 4, and others: Pathways and their coverage (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| 9 - HUMAnN2 on data 5, data 4, and others: Pathways and their abundance (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  diamond v0.8.24.86 | by Benjamin Buchfink \u003cbuchfink@gmail.com\u003e\n|  Check http://github.com/bbuchfink/diamond for updates.\n|  #CPU threads: 1\n|  Database file: /tmp/tmphzs4hnom/files/7/5/d/dataset_75dfcd53-d0f4-469d-9374-8c632c3a76c5.dat\n|  Opening the database file...  [0.000141s]\n|  Loading sequence data (0 sequences processed)...  [0.000566s]\n|  Writing trailer...  [8e-06s]\n|  Closing the database file...  [2.5e-05s]\n|  Processed 151 sequences, 74681 letters.\n|  Total time = 0.000807s\n|  Creating output directory: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Output files will be written to: /tmp/tmphzs4hnom/job_working_directory/000/68/working/output\n|  Removing spaces from identifiers in input file ...\n|  Found g__Bacteroides.s__Bacteroides_stercoris : 68.44% of mapped reads\n|  Found g__Bacteroides.s__Bacteroides_thetaiotaomicron : 31.56% of mapped reads\n|  Total species selected from prescreen: 2\n|  Selected species explain 100.00% of predicted community composition\n|  Creating custom ChocoPhlAn database ........\n| Dataset Job Standard Error:\n|  gunzip: invalid magic\n|  gunzip: invalid magic\n|  CRITICAL ERROR: Error executing: /bin/gunzip -c /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_stercoris.centroids.v0.1.1.ffn.gz /tmp/tmphzs4hnom/job_working_directory/000/68/working/nucleotide_db/g__Bacteroides.s__Bacteroides_thetaiotaomicron.centroids.v0.1.1.ffn.gz\n|\n--------------------------------------\n| Job 288a67f7f32bb2fd\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:00:01.114100\n| Create Time:\n|  2020-01-23T16:58:46.071121\n|\n--------------------------------------\n| Job 5617b66919498d02\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:44.562752\n| Create Time:\n|  2020-01-23T16:58:38.904791\n|\n--------------------------------------\n| Job fb9b740929ea3f54\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:37.394554\n| Create Time:\n|  2020-01-23T16:58:32.161922\n|\n--------------------------------------\n| Job 2987a55ca6380b3a\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:31.119150\n| Create Time:\n|  2020-01-23T16:58:25.320793\n|\n--------------------------------------\n| Job 087a78e08fef416e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:23.619643\n| Create Time:\n|  2020-01-23T16:58:18.548732\n|\n--------------------------------------\n| Job 9649d2d367737693\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:17.372779\n| Create Time:\n|  2020-01-23T16:58:11.738714\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 140.30329513549805, "tool_id": "humann2", "tool_version": "0.11.1.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_humann2.test_tool_000000"}, {"data": {"inputs": {"gencode": "11", "reference_genome|genome": {"id": "087a78e08fef416e", "src": "hda"}, "reference_genome|genome_type_select": "history", "standalone": false, "track_groups_0|category": "Auto Coloured", "track_groups_0|data_tracks_0|data_format|annotation": [{"id": "2987a55ca6380b3a", "src": "hda"}, {"id": "fb9b740929ea3f54", "src": "hda"}, {"id": "5617b66919498d02", "src": "hda"}, {"id": "288a67f7f32bb2fd", "src": "hda"}], "track_groups_0|data_tracks_0|data_format|data_format_select": "gene_calls", "track_groups_0|data_tracks_0|data_format|jbcolor_scale|color_score|color_score_select": "none", "track_groups_0|data_tracks_0|data_format|match_part|match_part_select": false, "track_groups_1|category": "Ignore Scale", "track_groups_1|data_tracks_0|data_format|annotation": {"id": "fed9b1622ba4118e", "src": "hda"}, "track_groups_1|data_tracks_0|data_format|data_format_select": "gene_calls", "track_groups_1|data_tracks_0|data_format|jbcolor_scale|color_score|color_score_select": "none", "track_groups_1|data_tracks_0|data_format|jbcolor_scale|color_score|color|color_select": "manual", "track_groups_1|data_tracks_0|data_format|jbcolor_scale|color_score|color|style_color": "#ff00ff", "track_groups_1|data_tracks_0|data_format|match_part|match_part_select": false, "track_groups_2|category": "Scaled Colour", "track_groups_2|data_tracks_0|data_format|annotation": {"id": "fed9b1622ba4118e", "src": "hda"}, "track_groups_2|data_tracks_0|data_format|data_format_select": "gene_calls", "track_groups_2|data_tracks_0|data_format|jbcolor_scale|color_score|color_scheme|color|color_select": "automatic", "track_groups_2|data_tracks_0|data_format|jbcolor_scale|color_score|color_scheme|score_scheme": "opacity", "track_groups_2|data_tracks_0|data_format|jbcolor_scale|color_score|color_score_select": "score", "track_groups_2|data_tracks_0|data_format|jbcolor_scale|color_score|score_scales|scale_select": "automatic", "track_groups_2|data_tracks_0|data_format|jbcolor_scale|color_score|score_scaling": "linear", "track_groups_2|data_tracks_0|data_format|match_part|match_part_select": false, "track_groups_2|data_tracks_1|data_format|annotation": {"id": "fed9b1622ba4118e", "src": "hda"}, "track_groups_2|data_tracks_1|data_format|data_format_select": "gene_calls", "track_groups_2|data_tracks_1|data_format|jbcolor_scale|color_score|color_scheme|color|color_select": "manual", "track_groups_2|data_tracks_1|data_format|jbcolor_scale|color_score|color_scheme|color|style_color": "#0000ff", "track_groups_2|data_tracks_1|data_format|jbcolor_scale|color_score|color_scheme|score_scheme": "opacity", "track_groups_2|data_tracks_1|data_format|jbcolor_scale|color_score|color_score_select": "score", "track_groups_2|data_tracks_1|data_format|jbcolor_scale|color_score|score_scales|scale_select": "automatic", "track_groups_2|data_tracks_1|data_format|jbcolor_scale|color_score|score_scaling": "linear", "track_groups_2|data_tracks_1|data_format|match_part|match_part_select": false, "track_groups_2|data_tracks_2|data_format|annotation": {"id": "fed9b1622ba4118e", "src": "hda"}, "track_groups_2|data_tracks_2|data_format|data_format_select": "gene_calls", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|color_scheme|color|color_select": "automatic", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|color_scheme|score_scheme": "opacity", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|color_score_select": "score", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|score_scales|maximum": "1000", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|score_scales|minimum": "0", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|score_scales|scale_select": "manual", "track_groups_2|data_tracks_2|data_format|jbcolor_scale|color_score|score_scaling": "linear", "track_groups_2|data_tracks_2|data_format|match_part|match_part_select": false, "track_groups_2|data_tracks_3|data_format|annotation": {"id": "fed9b1622ba4118e", "src": "hda"}, "track_groups_2|data_tracks_3|data_format|data_format_select": "gene_calls", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|color_scheme|color|color_select": "manual", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|color_scheme|color|style_color": "#ff0000", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|color_scheme|score_scheme": "opacity", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|color_score_select": "score", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|score_scales|maximum": "1000", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|score_scales|minimum": "0", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|score_scales|scale_select": "manual", "track_groups_2|data_tracks_3|data_format|jbcolor_scale|color_score|score_scaling": "linear", "track_groups_2|data_tracks_3|data_format|match_part|match_part_select": false, "track_groups_3|category": "Realistic", "track_groups_3|data_tracks_0|data_format|annotation": {"id": "b323ca264f40001e", "src": "hda"}, "track_groups_3|data_tracks_0|data_format|data_format_select": "gene_calls", "track_groups_3|data_tracks_0|data_format|jbcolor_scale|color_score|color_score_select": "none", "track_groups_3|data_tracks_0|data_format|match_part|match_part_select": false, "track_groups_3|data_tracks_1|data_format|annotation": {"id": "08fbf3fbfe908f7c", "src": "hda"}, "track_groups_3|data_tracks_1|data_format|data_format_select": "gene_calls", "track_groups_3|data_tracks_1|data_format|jbcolor_scale|color_score|color_score_select": "none", "track_groups_3|data_tracks_1|data_format|match_part|match_part_select": true, "track_groups_3|data_tracks_1|data_format|match_part|name": "cDNA_match", "uglyTestingHack": "enabled"}, "job": {"command_line": "mkdir -p /tmp/tmp8uc5qy5s/job_working_directory/000/42/working/dataset_01c879c9-fc27-46eb-861e-02d14e7e2d20_files \u0026\u0026  cp /tmp/tmp8uc5qy5s/job_working_directory/000/42/configs/tmpac7s2fod /tmp/tmp8uc5qy5s/job_working_directory/000/42/working/dataset_01c879c9-fc27-46eb-861e-02d14e7e2d20_files/galaxy.xml \u0026\u0026  export JBROWSE_SOURCE_DIR=$(dirname $(which prepare-refseqs.pl))/../opt/jbrowse  \u0026\u0026  python \u0027/home/runner/work/tools-iuc/tools-iuc/tools/jbrowse/jbrowse.py\u0027  --jbrowse ${JBROWSE_SOURCE_DIR}  --outdir /tmp/tmp8uc5qy5s/job_working_directory/000/42/working/dataset_01c879c9-fc27-46eb-861e-02d14e7e2d20_files /tmp/tmp8uc5qy5s/job_working_directory/000/42/configs/tmpac7s2fod \u0026\u0026  cp /tmp/tmp8uc5qy5s/job_working_directory/000/42/configs/tmp8om7fk76 /tmp/tmp8uc5qy5s/files/0/1/c/dataset_01c879c9-fc27-46eb-861e-02d14e7e2d20.dat;  cp /tmp/tmp8uc5qy5s/job_working_directory/000/42/configs/tmpac7s2fod /tmp/tmp8uc5qy5s/files/0/1/c/dataset_01c879c9-fc27-46eb-861e-02d14e7e2d20.dat", "command_version": "jbrowse.py 0.8.0\n", "create_time": "2020-01-23T17:25:18.776373", "exit_code": 0, "external_id": "20432", "galaxy_version": "20.01", "history_id": "9ce08b2254e4d5ed", "id": "25f486d5fc93a1ce", "inputs": {"genome": {"id": "087a78e08fef416e", "src": "hda", "uuid": "eec7386f-eed4-4a58-bd5c-c19e297f902b"}, "track_groups_0|data_tracks_0|annotation": {"id": "2987a55ca6380b3a", "src": "hda", "uuid": "2ddf6be1-e330-46b4-95a5-70db766dd258"}, "track_groups_0|data_tracks_0|annotation1": {"id": "2987a55ca6380b3a", "src": "hda", "uuid": "2ddf6be1-e330-46b4-95a5-70db766dd258"}, "track_groups_0|data_tracks_0|annotation2": {"id": "fb9b740929ea3f54", "src": "hda", "uuid": "4d19d7bd-db0c-452d-a063-10f0cf069a1c"}, "track_groups_0|data_tracks_0|annotation3": {"id": "5617b66919498d02", "src": "hda", "uuid": "bcf814e8-ed58-4dda-bc06-d5285b67f6a9"}, "track_groups_0|data_tracks_0|annotation4": {"id": "288a67f7f32bb2fd", "src": "hda", "uuid": "0c5acb15-7338-4b07-844c-977899b3ccea"}, "track_groups_1|data_tracks_0|annotation": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "e8bb2805-4c77-4c8c-b8b1-1b1539022edd"}, "track_groups_2|data_tracks_0|annotation": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "e8bb2805-4c77-4c8c-b8b1-1b1539022edd"}, "track_groups_2|data_tracks_1|annotation": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "e8bb2805-4c77-4c8c-b8b1-1b1539022edd"}, "track_groups_2|data_tracks_2|annotation": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "e8bb2805-4c77-4c8c-b8b1-1b1539022edd"}, "track_groups_2|data_tracks_3|annotation": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "e8bb2805-4c77-4c8c-b8b1-1b1539022edd"}, "track_groups_3|data_tracks_0|annotation": {"id": "b323ca264f40001e", "src": "hda", "uuid": "f86c517a-793c-4a0a-bf40-326a9ecffb59"}, "track_groups_3|data_tracks_1|annotation": {"id": "08fbf3fbfe908f7c", "src": "hda", "uuid": "00b74af7-6b4a-4dea-80d8-2e7afa0d3257"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output": {"id": "ca4ebafec84c6506", "src": "hda", "uuid": "01c879c9-fc27-46eb-861e-02d14e7e2d20"}}, "params": {"__input_ext": "\"fasta\"", "action": "{\"__current_case__\": 0, \"action_select\": \"create\"}", "chromInfo": "\"/tmp/tmp8uc5qy5s/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "gencode": "\"11\"", "jbgen": "{\"aboutDescription\": \"\", \"defaultLocation\": \"\", \"hideGenomeOptions\": \"false\", \"shareLink\": \"true\", \"show_menu\": \"true\", \"show_nav\": \"true\", \"show_overview\": \"true\", \"show_tracklist\": \"true\", \"trackPadding\": \"20\"}", "plugins": "{\"BlastView\": \"true\", \"ComboTrackSelector\": \"false\", \"GCContent\": \"false\"}", "reference_genome": "{\"__current_case__\": 1, \"genome\": {\"values\": [{\"id\": 64, \"src\": \"hda\"}]}, \"genome_type_select\": \"history\"}", "standalone": "\"false\"", "track_groups": "[{\"__index__\": 0, \"category\": \"Auto Coloured\", \"data_tracks\": [{\"__index__\": 0, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 65, \"src\": \"hda\"}, {\"id\": 66, \"src\": \"hda\"}, {\"id\": 67, \"src\": \"hda\"}, {\"id\": 68, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 0, \"color_select\": \"automatic\"}, \"color_score_select\": \"none\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}]}, {\"__index__\": 1, \"category\": \"Ignore Scale\", \"data_tracks\": [{\"__index__\": 0, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 69, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 1, \"color_select\": \"manual\", \"style_color\": \"#ff00ff\"}, \"color_score_select\": \"none\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}]}, {\"__index__\": 2, \"category\": \"Scaled Colour\", \"data_tracks\": [{\"__index__\": 0, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 69, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 1, \"color_scheme\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 0, \"color_select\": \"automatic\"}, \"score_scheme\": \"opacity\"}, \"color_score_select\": \"score\", \"score_scales\": {\"__current_case__\": 0, \"scale_select\": \"automatic\"}, \"score_scaling\": \"linear\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}, {\"__index__\": 1, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 69, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 1, \"color_scheme\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 1, \"color_select\": \"manual\", \"style_color\": \"#0000ff\"}, \"score_scheme\": \"opacity\"}, \"color_score_select\": \"score\", \"score_scales\": {\"__current_case__\": 0, \"scale_select\": \"automatic\"}, \"score_scaling\": \"linear\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}, {\"__index__\": 2, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 69, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 1, \"color_scheme\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 0, \"color_select\": \"automatic\"}, \"score_scheme\": \"opacity\"}, \"color_score_select\": \"score\", \"score_scales\": {\"__current_case__\": 1, \"maximum\": \"1000\", \"minimum\": \"0\", \"scale_select\": \"manual\"}, \"score_scaling\": \"linear\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}, {\"__index__\": 3, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 69, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 1, \"color_scheme\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 1, \"color_select\": \"manual\", \"style_color\": \"#ff0000\"}, \"score_scheme\": \"opacity\"}, \"color_score_select\": \"score\", \"score_scales\": {\"__current_case__\": 1, \"maximum\": \"1000\", \"minimum\": \"0\", \"scale_select\": \"manual\"}, \"score_scaling\": \"linear\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}]}, {\"__index__\": 3, \"category\": \"Realistic\", \"data_tracks\": [{\"__index__\": 0, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 70, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 0, \"color_select\": \"automatic\"}, \"color_score_select\": \"none\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 1, \"match_part_select\": \"false\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}, {\"__index__\": 1, \"data_format\": {\"__current_case__\": 2, \"annotation\": {\"values\": [{\"id\": 71, \"src\": \"hda\"}]}, \"data_format_select\": \"gene_calls\", \"index\": \"false\", \"jb_custom_config\": {\"option\": []}, \"jbcolor_scale\": {\"color_score\": {\"__current_case__\": 0, \"color\": {\"__current_case__\": 0, \"color_select\": \"automatic\"}, \"color_score_select\": \"none\"}}, \"jbmenu\": {\"track_menu\": []}, \"jbstyle\": {\"max_height\": \"600\", \"style_classname\": \"feature\", \"style_description\": \"note,description\", \"style_height\": \"10px\", \"style_label\": \"product,name,id\"}, \"match_part\": {\"__current_case__\": 0, \"match_part_select\": \"true\", \"name\": \"cDNA_match\"}, \"override_apollo_drag\": \"False\", \"override_apollo_plugins\": \"False\", \"track_config\": {\"__current_case__\": 3, \"html_options\": {\"topLevelFeatures\": \"\"}, \"track_class\": \"NeatHTMLFeatures/View/Track/NeatFeatures\"}, \"track_visibility\": \"default_off\"}}]}]", "uglyTestingHack": "\"enabled\""}, "state": "ok", "stderr": "INFO:jbrowse:Processing Auto Coloured / A.gff\nINFO:jbrowse:Processing Auto Coloured / B.gff\nINFO:jbrowse:Processing Auto Coloured / C.gff\nINFO:jbrowse:Processing Auto Coloured / D.gff\nINFO:jbrowse:Processing Ignore Scale / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Realistic / interpro.gff\nINFO:jbrowse:Processing Realistic / 2.gff\nWARNING: No feature names found for indexing, only reference sequence names will be indexed.\n", "stdout": "", "tool_id": "jbrowse", "tool_stderr": "INFO:jbrowse:Processing Auto Coloured / A.gff\nINFO:jbrowse:Processing Auto Coloured / B.gff\nINFO:jbrowse:Processing Auto Coloured / C.gff\nINFO:jbrowse:Processing Auto Coloured / D.gff\nINFO:jbrowse:Processing Ignore Scale / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Scaled Colour / 1.gff\nINFO:jbrowse:Processing Realistic / interpro.gff\nINFO:jbrowse:Processing Realistic / 2.gff\nWARNING: No feature names found for indexing, only reference sequence names will be indexed.\n", "tool_stdout": "", "update_time": "2020-01-23T17:25:24.279444", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpds4thjl6test.xml v. /tmp/tmpro19ryrztest.xml )\n--- local_file\n+++ history_data\n@@ -3,13 +3,13 @@\n     \u003cmetadata\u003e\n         \u003cgencode\u003e11\u003c/gencode\u003e\n         \u003cgenomes\u003e\n-              \u003cgenome path=\"/tmp/tmpqkd9zdgq/files/b/f/a/dataset_bfad32fe-cc05-405c-965c-6fe9b7a36908.dat\"\u003e\n-                \u003cmetadata\u003e\n-                  \u003cdataset id=\"b842d972534ccb3e\" hid=\"1\"\n+              \u003cgenome path=\"/tmp/tmp8uc5qy5s/files/e/e/c/dataset_eec7386f-eed4-4a58-bd5c-c19e297f902b.dat\"\u003e\n+                \u003cmetadata\u003e\n+                  \u003cdataset id=\"087a78e08fef416e\" hid=\"1\"\n                       size=\"171.6 KB\"\n                       edam_format=\"format_1929\"\n                       file_ext=\"fasta\" /\u003e\n-                  \u003chistory id=\"8155e4b4bf1581ff\"\n+                  \u003chistory id=\"9ce08b2254e4d5ed\"\n                       user_email=\"test@bx.psu.edu\"\n                       user_id=\"2\"\n                       display_name=\"test_history\"/\u003e\n@@ -37,90 +37,90 @@\n             \u003cshow_menu\u003etrue\u003c/show_menu\u003e\n             \u003chideGenomeOptions\u003efalse\u003c/hideGenomeOptions\u003e\n         \u003c/general\u003e\n-        \u003cgalaxyUrl\u003ehttp://localhost\u003c/galaxyUrl\u003e\n********\n*SNIP *\n********\n                       edam_format=\"format_1975\"\n                       file_ext=\"gff3\" /\u003e\n-                  \u003chistory id=\"8155e4b4bf1581ff\"\n+                  \u003chistory id=\"9ce08b2254e4d5ed\"\n                       user_email=\"test@bx.psu.edu\"\n                       user_id=\"2\"\n                       display_name=\"test_history\"/\u003e\n@@ -527,13 +527,13 @@\n         \u003c/track\u003e\n         \u003ctrack cat=\"Realistic\" format=\"gene_calls\" visibility=\"default_off\"\u003e\n             \u003cfiles\u003e\n-              \u003ctrackFile path=\"/tmp/tmpqkd9zdgq/files/a/b/b/dataset_abb36b76-2a3a-4353-919e-13c94629303f.dat\" ext=\"gff3\" label=\"2.gff\"\u003e\n-                \u003cmetadata\u003e\n-                  \u003cdataset id=\"ea0b941dfbe636f8\" hid=\"8\"\n+              \u003ctrackFile path=\"/tmp/tmp8uc5qy5s/files/0/0/b/dataset_00b74af7-6b4a-4dea-80d8-2e7afa0d3257.dat\" ext=\"gff3\" label=\"2.gff\"\u003e\n+                \u003cmetadata\u003e\n+                  \u003cdataset id=\"08fbf3fbfe908f7c\" hid=\"8\"\n                       size=\"326 bytes\"\n                       edam_format=\"format_1975\"\n                       file_ext=\"gff3\" /\u003e\n-                  \u003chistory id=\"8155e4b4bf1581ff\"\n+                  \u003chistory id=\"9ce08b2254e4d5ed\"\n                       user_email=\"test@bx.psu.edu\"\n                       user_id=\"2\"\n                       display_name=\"test_history\"/\u003e\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8uc5qy5s/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8uc5qy5s/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8uc5qy5s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8uc5qy5s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8uc5qy5s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmpds4thjl6test.xml v. /tmp/tmpro19ryrztest.xml )\n--- local_file\n+++ history_data\n@@ -3,13 +3,13 @@\n     \u003cmetadata\u003e\n         \u003cgencode\u003e11\u003c/gencode\u003e\n         \u003cgenomes\u003e\n-              \u003cgenome path=\"/tmp/tmpqkd9zdgq/files/b/f/a/dataset_bfad32fe-cc05-405c-965c-6fe9b7a36908.dat\"\u003e\n-                \u003cmetadata\u003e\n-                  \u003cdataset id=\"b842d972534ccb3e\" hid=\"1\"\n+              \u003cgenome path=\"/tmp/tmp8uc5qy5s/files/e/e/c/dataset_eec7386f-eed4-4a58-bd5c-c19e297f902b.dat\"\u003e\n+                \u003cmetadata\u003e\n+                  \u003cdataset id=\"087a78e08fef416e\" hid=\"1\"\n                       size=\"171.6 KB\"\n                       edam_format=\"format_1929\"\n                       file_ext=\"fasta\" /\u003e\n-                  \u003chistory id=\"8155e4b4bf1581ff\"\n+                  \u003chistory id=\"9ce08b2254e4d5ed\"\n                       user_email=\"test@bx.psu.edu\"\n                       user_id=\"2\"\n                       display_name=\"test_history\"/\u003e\n@@ -37,90 +37,90 @@\n             \u003cshow_menu\u003etrue\u003c/show_menu\u003e\n             \u003chideGenomeOptions\u003efalse\u003c/hideGenomeOptions\u003e\n         \u003c/general\u003e\n-        \u003cgalaxyUrl\u003ehttp://localhost\u003c/galaxyUrl\u003e\n********\n*SNIP *\n********\n                       edam_format=\"format_1975\"\n                       file_ext=\"gff3\" /\u003e\n-                  \u003chistory id=\"8155e4b4bf1581ff\"\n+                  \u003chistory id=\"9ce08b2254e4d5ed\"\n                       user_email=\"test@bx.psu.edu\"\n                       user_id=\"2\"\n                       display_name=\"test_history\"/\u003e\n@@ -527,13 +527,13 @@\n         \u003c/track\u003e\n         \u003ctrack cat=\"Realistic\" format=\"gene_calls\" visibility=\"default_off\"\u003e\n             \u003cfiles\u003e\n-              \u003ctrackFile path=\"/tmp/tmpqkd9zdgq/files/a/b/b/dataset_abb36b76-2a3a-4353-919e-13c94629303f.dat\" ext=\"gff3\" label=\"2.gff\"\u003e\n-                \u003cmetadata\u003e\n-                  \u003cdataset id=\"ea0b941dfbe636f8\" hid=\"8\"\n+              \u003ctrackFile path=\"/tmp/tmp8uc5qy5s/files/0/0/b/dataset_00b74af7-6b4a-4dea-80d8-2e7afa0d3257.dat\" ext=\"gff3\" label=\"2.gff\"\u003e\n+                \u003cmetadata\u003e\n+                  \u003cdataset id=\"08fbf3fbfe908f7c\" hid=\"8\"\n                       size=\"326 bytes\"\n                       edam_format=\"format_1975\"\n                       file_ext=\"gff3\" /\u003e\n-                  \u003chistory id=\"8155e4b4bf1581ff\"\n+                  \u003chistory id=\"9ce08b2254e4d5ed\"\n                       user_email=\"test@bx.psu.edu\"\n                       user_id=\"2\"\n                       display_name=\"test_history\"/\u003e\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 3, "time_seconds": 59.8152961730957, "tool_id": "jbrowse", "tool_version": "1.16.5+galaxy6"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_jbrowse.test_tool_000003"}, {"data": {"inputs": {"delta": {"id": "61f03d5eef6f1538", "src": "hda"}, "extra_outs": "all", "query_sequence": {"id": "b28067233c6d9fa0", "src": "hda"}, "reference_sequence": {"id": "d5b3e51b7b790121", "src": "hda"}, "sequences|seq_input": "no"}, "job": {"command_line": "ln -s /tmp/tmp334qy1a1/files/f/2/5/dataset_f25706b2-e898-4aa1-b955-9fecd80cc10b.dat reference.fa \u0026\u0026 ln -s /tmp/tmp334qy1a1/files/8/7/7/dataset_87705487-a823-4920-8834-cc2e5504b3f6.dat query.fa \u0026\u0026 mummerplot -b \u002720\u0027     -s \u0027small\u0027 -terminal png -title \u0027Title\u0027  \u0027/tmp/tmp334qy1a1/files/b/7/f/dataset_b7f69f8b-d825-4142-bca6-4eac60f7d1b3.dat\u0027", "command_version": "", "create_time": "2020-01-23T16:49:46.932594", "exit_code": 0, "external_id": "8338", "galaxy_version": "20.01", "history_id": "8155e4b4bf1581ff", "id": "80b8022ff3f677b7", "inputs": {"delta": {"id": "61f03d5eef6f1538", "src": "hda", "uuid": "b7f69f8b-d825-4142-bca6-4eac60f7d1b3"}, "query_sequence": {"id": "b28067233c6d9fa0", "src": "hda", "uuid": "87705487-a823-4920-8834-cc2e5504b3f6"}, "reference_sequence": {"id": "d5b3e51b7b790121", "src": "hda", "uuid": "f25706b2-e898-4aa1-b955-9fecd80cc10b"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"fplot": {"id": "5ca39fba41b940d1", "src": "hda", "uuid": "bb8e2dcf-ea3d-4b3e-84c8-6a61e2a3617f"}, "gnuplot": {"id": "c18708cec3343c17", "src": "hda", "uuid": "8d02609d-f743-4d50-9a25-e8b0d8a19760"}, "hplot": {"id": "a1e670e1ea0da54b", "src": "hda", "uuid": "72559e48-6499-41aa-874a-88240910bbf1"}, "output_png": {"id": "9a22496ee796fbfc", "src": "hda", "uuid": "44c38854-9667-4fd2-ad4a-7b3589cd201f"}, "rplot": {"id": "3e4e8de146a035f5", "src": "hda", "uuid": "7a0bb4ce-ee24-435a-9576-ab4e9bdb7f6a"}}, "params": {"__input_ext": "\"txt\"", "breaklen": "\"20\"", "chromInfo": "\"/tmp/tmp334qy1a1/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "color": "\"\"", "coverage": "\"\"", "dbkey": "\"?\"", "extra_outs": "\"all\"", "fat": "\"false\"", "filter": "\"false\"", "labels": "{\"IDs\": \"no\", \"__current_case__\": 1}", "range": "{\"__current_case__\": 1, \"custom\": \"no\"}", "sequences": "{\"__current_case__\": 1, \"seq_input\": \"no\"}", "size": "\"small\"", "snp": "\"false\"", "title": "\"Title\""}, "state": "ok", "stderr": "gnuplot: error while loading shared libraries: libXrender.so.1: cannot open shared object file: No such file or directory\nWARNING: Using outdated gnuplot, use v4 or later for best results\nReading delta file /tmp/tmp334qy1a1/files/b/7/f/dataset_b7f69f8b-d825-4142-bca6-4eac60f7d1b3.dat\nWriting plot files out.fplot, out.rplot, out.hplot\nWriting gnuplot script out.gp\nRendering plot out.png\nWARNING: Unable to run \u0027false out.gp\u0027, Inappropriate ioctl for device\n", "stdout": "", "tool_id": "mummer_mummerplot", "tool_stderr": "gnuplot: error while loading shared libraries: libXrender.so.1: cannot open shared object file: No such file or directory\nWARNING: Using outdated gnuplot, use v4 or later for best results\nReading delta file /tmp/tmp334qy1a1/files/b/7/f/dataset_b7f69f8b-d825-4142-bca6-4eac60f7d1b3.dat\nWriting plot files out.fplot, out.rplot, out.hplot\nWriting gnuplot script out.gp\nRendering plot out.png\nWARNING: Unable to run \u0027false out.gp\u0027, Inappropriate ioctl for device\n", "tool_stdout": "", "update_time": "2020-01-23T16:49:51.470606", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpixahe38dgnuplot.txt v. /tmp/tmpj1rbjt14gnuplot.txt )\n--- local_file\n+++ history_data\n@@ -1,22 +1,19 @@\n-set terminal png tiny size 800,800\n+set terminal png small\n set output \"out.png\"\n set title \"Title\"\n set size 1,1\n set grid\n-unset key\n+set nokey\n set border 15\n set tics scale 0\n set xlabel \"NG_007476.1:4960-11439\"\n set ylabel \"NC_000070.6:c41098183-41092724\"\n set format \"%.0f\"\n-set mouse format \"%.0f\"\n-set mouse mouseformat \"[%.0f, %.0f]\"\n-if(GPVAL_VERSION \u003c 5) { set mouse clipboardformat \"[%.0f, %.0f]\" }\n set xrange [1:6480]\n set yrange [1:5460]\n-set style line 1  lt 2 lw 3 pt 6 ps 1\n-set style line 2  lt 2 lw 3 pt 6 ps 1\n-set style line 3  lt 1 lw 3 pt 6 ps 1\n+set linestyle 1  lt 2 lw 3 pt 6 ps 1\n+set linestyle 2  lt 2 lw 3 pt 6 ps 1\n+set linestyle 3  lt 1 lw 3 pt 6 ps 1\n plot \\\n  \"out.fplot\" title \"FWD\" w lp ls 1, \\\n  \"out.rplot\" title \"REV\" w lp ls 2, \\\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp334qy1a1/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp334qy1a1/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp334qy1a1/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp334qy1a1/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp334qy1a1/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmpixahe38dgnuplot.txt v. /tmp/tmpj1rbjt14gnuplot.txt )\n--- local_file\n+++ history_data\n@@ -1,22 +1,19 @@\n-set terminal png tiny size 800,800\n+set terminal png small\n set output \"out.png\"\n set title \"Title\"\n set size 1,1\n set grid\n-unset key\n+set nokey\n set border 15\n set tics scale 0\n set xlabel \"NG_007476.1:4960-11439\"\n set ylabel \"NC_000070.6:c41098183-41092724\"\n set format \"%.0f\"\n-set mouse format \"%.0f\"\n-set mouse mouseformat \"[%.0f, %.0f]\"\n-if(GPVAL_VERSION \u003c 5) { set mouse clipboardformat \"[%.0f, %.0f]\" }\n set xrange [1:6480]\n set yrange [1:5460]\n-set style line 1  lt 2 lw 3 pt 6 ps 1\n-set style line 2  lt 2 lw 3 pt 6 ps 1\n-set style line 3  lt 1 lw 3 pt 6 ps 1\n+set linestyle 1  lt 2 lw 3 pt 6 ps 1\n+set linestyle 2  lt 2 lw 3 pt 6 ps 1\n+set linestyle 3  lt 1 lw 3 pt 6 ps 1\n plot \\\n  \"out.fplot\" title \"FWD\" w lp ls 1, \\\n  \"out.rplot\" title \"REV\" w lp ls 2, \\\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 47.525187492370605, "tool_id": "mummer_mummerplot", "tool_version": "4.0.0beta2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_mummer_mummerplot.test_tool_000000"}, {"data": {"inputs": {"data_type|fastq_input1": {"id": "95a5ac7e4618a243", "src": "hda"}, "data_type|type": "single", "format": "json", "select_species|panel": "bradley-2015", "select_species|species": "tb"}, "job": {"command_line": "ln -s \u0027/tmp/tmp0jcugsuy/files/8/1/f/dataset_81f7a7c7-a408-47cd-ac3a-3239457efb87.dat\u0027 sample.fastq \u0026\u0026       mykrobe predict \u0027sample\u0027 tb  --panel \u0027bradley-2015\u0027      --kmer \u002721\u0027                    -q -t \"${GALAXY_SLOTS:-1}\"  -1 *.fastq  --format \u0027json\u0027  \u003e \u0027/tmp/tmp0jcugsuy/files/4/0/5/dataset_40519e1b-baf0-406a-9512-7b8c03ecc566.dat\u0027", "command_version": "", "create_time": "2020-01-23T16:52:46.878331", "exit_code": 1, "external_id": "9931", "galaxy_version": "20.01", "history_id": "a90a30fafe298e1e", "id": "3e4e8de146a035f5", "inputs": {"fastq_input1": {"id": "95a5ac7e4618a243", "src": "hda", "uuid": "81f7a7c7-a408-47cd-ac3a-3239457efb87"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output": {"id": "863b4912f43ba501", "src": "hda", "uuid": "40519e1b-baf0-406a-9512-7b8c03ecc566"}}, "params": {"__input_ext": "\"fastqsanger\"", "adv": "{\"custom_probe\": null, \"custom_variant_to_resistance\": null}", "chromInfo": "\"/tmp/tmp0jcugsuy/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "conf_percent_cutoff": "\"\"", "data_type": "{\"__current_case__\": 0, \"fastq_input1\": {\"values\": [{\"id\": 36, \"src\": \"hda\"}]}, \"type\": \"single\"}", "dbkey": "\"?\"", "expected_depth": "\"\"", "expected_error_rate": "\"\"", "format": "\"json\"", "guess_sequence_method": "\"false\"", "ignore_filtered": "null", "ignore_minor_calls": "\"false\"", "kmer": "\"21\"", "min_depth": "\"\"", "min_gene_conf": "\"\"", "min_gene_percent_covg_threshold": "\"\"", "min_proportion_expected_depth": "\"\"", "min_variant_conf": "\"\"", "model": "null", "ont": "\"false\"", "ploidy": "null", "report_all_calls": "\"false\"", "select_species": "{\"__current_case__\": 1, \"panel\": \"bradley-2015\", \"species\": \"tb\"}"}, "state": "error", "stderr": "INFO:mykrobe.cmds.amr:Running AMR prediction with panels data/panels/tb-species-170421.fasta.gz, data/panels/tb-bradley-probe-set-jan-2019.fasta.gz\n[23 Jan 2020 16:53:06-qAk][cmd] /usr/local/bin/mccortex31 geno -t 1 -m 1GB -k 21 -o tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.covgs -I mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx -s sample-21 -1 sample.fastq -c /usr/local/share/mykrobe-data/panels/tb-species-170421.fasta.gz -c /usr/local/share/mykrobe-data/panels/tb-bradley-probe-set-jan-2019.fasta.gz tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:06-qAk][cwd] /tmp/tmp0jcugsuy/job_working_directory/000/23/working\n[23 Jan 2020 16:53:06-qAk][version] mccortex=v0.0.3-486-g22203aa-dirty zlib=1.2.11 htslib=1.2.1-250-gfa6ed9a-dirty ASSERTS=ON hash=Lookup3 CHECKS=ON k=3..31\n[23 Jan 2020 16:53:09-qAk] Saving graph to: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:09-qAk][sample] 0: sample-21\n[23 Jan 2020 16:53:09-qAk][task] sample.fastq; FASTQ offset: auto-detect, threshold: off; cut homopolymers: off; remove PCR duplicates: no; colour: 0\n[23 Jan 2020 16:53:09-qAk][memory] 112 bits per kmer\n[23 Jan 2020 16:53:09-qAk][memory] graph: 464MB\n[23 Jan 2020 16:53:09-qAk][memory] total: 464MB of 6.8GB RAM\n[23 Jan 2020 16:53:09-qAk] Writing 1 colour graph to tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:09-qAk][hasht] Allocating table with 34,603,008 entries, using 266MB\n[23 Jan 2020 16:53:09-qAk][hasht]  number of buckets: 1,048,576, bucket size: 33\n[23 Jan 2020 16:53:09-qAk][graph] kmer-size: 21; colours: 1; capacity: 34,603,008\n[23 Jan 2020 16:53:09-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 0 / 34,603,008 (0.00%)\n[23 Jan 2020 16:53:09-qAk][FileFilter] Loading file mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx [1 colour]\n[23 Jan 2020 16:53:09-qAk][GReader] 25,481,095 kmers, 315.9MB filesize\n[23 Jan 2020 16:53:16-qAk][GReader] Loaded 25,481,095 / 25,481,095 (100.00%) of kmers parsed\n[23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 25,481,095 / 34,603,008 (73.64%)\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n[23 Jan 2020 16:53:16-qAk][asyncio] Inputs: 1; Threads: 1\n[23 Jan 2020 16:53:16-qAk][seq] Parsing sequence file sample.fastq\n[23 Jan 2020 16:53:16-qAk] sample.fastq: Qual scores: Sanger / Illumina 1.9+ (Phred+33) [offset: 33, range: [33,126], sample: [35,79]]\n[23 Jan 2020 16:53:16-qAk][seq] Loaded 1,035 reads and 0 reads pairs (file: sample.fastq)\n[23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 1,196 / 34,603,008 (0.00%)\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n[23 Jan 2020 16:53:16-qAk][task] input: sample.fastq colour: 0\n[23 Jan 2020 16:53:16-qAk]  SE reads: 1,035  PE reads: 0\n[23 Jan 2020 16:53:16-qAk]  good reads: 1,035  bad reads: 0\n[23 Jan 2020 16:53:16-qAk]  dup SE reads: 0  dup PE pairs: 0\n[23 Jan 2020 16:53:16-qAk]  bases read: 103,500  bases loaded: 103,426\n[23 Jan 2020 16:53:16-qAk]  num contigs: 1,035  num kmers: 17,170 novel kmers: 0\n[23 Jan 2020 16:53:16-qAk] Reading coverage...\n[23 Jan 2020 16:53:44-qAk] Printed graph coverage for 352,617 reads\n[23 Jan 2020 16:53:45-qAk] Dumping graph...\n[23 Jan 2020 16:53:45-qAk] Dumping graph colour 0 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:45-qAk][graph_writer_save] Writing colours 0-0 of 1 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:45-qAk] Dumped 1,196 kmers in 1 colour into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx (format version: 6)\n[23 Jan 2020 16:53:45-qAk][memory] We made 20 allocs\n[23 Jan 2020 16:53:45-qAk] Done.\n[23 Jan 2020 16:53:45-qAk][time] 39.00 seconds\nTraceback (most recent call last):\n  File \"/usr/local/bin/mykrobe\", line 11, in \u003cmodule\u003e\n    sys.exit(main())\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cli.py\", line 8, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/parser.py\", line 42, in run_subtool\n    run(parser, args)\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cmds/amr.py\", line 274, in run\n    phylogenetics = species_predictor.run()\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/metagenomics/phylo.py\", line 65, in run\n    return MykrobePredictorPhylogeneticsResult(self.out_json[\"phylogenetics\"])\n  File \"/usr/local/lib/python3.7/site-packages/mongoengine/base/document.py\", line 79, in __init__\n    \"Instantiating a document with positional arguments is not \"\nTypeError: Instantiating a document with positional arguments is not supported. Please use `field_name=value` keyword arguments.\n", "stdout": "", "tool_id": "mykrobe_predict", "tool_stderr": "INFO:mykrobe.cmds.amr:Running AMR prediction with panels data/panels/tb-species-170421.fasta.gz, data/panels/tb-bradley-probe-set-jan-2019.fasta.gz\n[23 Jan 2020 16:53:06-qAk][cmd] /usr/local/bin/mccortex31 geno -t 1 -m 1GB -k 21 -o tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.covgs -I mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx -s sample-21 -1 sample.fastq -c /usr/local/share/mykrobe-data/panels/tb-species-170421.fasta.gz -c /usr/local/share/mykrobe-data/panels/tb-bradley-probe-set-jan-2019.fasta.gz tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:06-qAk][cwd] /tmp/tmp0jcugsuy/job_working_directory/000/23/working\n[23 Jan 2020 16:53:06-qAk][version] mccortex=v0.0.3-486-g22203aa-dirty zlib=1.2.11 htslib=1.2.1-250-gfa6ed9a-dirty ASSERTS=ON hash=Lookup3 CHECKS=ON k=3..31\n[23 Jan 2020 16:53:09-qAk] Saving graph to: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:09-qAk][sample] 0: sample-21\n[23 Jan 2020 16:53:09-qAk][task] sample.fastq; FASTQ offset: auto-detect, threshold: off; cut homopolymers: off; remove PCR duplicates: no; colour: 0\n[23 Jan 2020 16:53:09-qAk][memory] 112 bits per kmer\n[23 Jan 2020 16:53:09-qAk][memory] graph: 464MB\n[23 Jan 2020 16:53:09-qAk][memory] total: 464MB of 6.8GB RAM\n[23 Jan 2020 16:53:09-qAk] Writing 1 colour graph to tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:09-qAk][hasht] Allocating table with 34,603,008 entries, using 266MB\n[23 Jan 2020 16:53:09-qAk][hasht]  number of buckets: 1,048,576, bucket size: 33\n[23 Jan 2020 16:53:09-qAk][graph] kmer-size: 21; colours: 1; capacity: 34,603,008\n[23 Jan 2020 16:53:09-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 0 / 34,603,008 (0.00%)\n[23 Jan 2020 16:53:09-qAk][FileFilter] Loading file mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx [1 colour]\n[23 Jan 2020 16:53:09-qAk][GReader] 25,481,095 kmers, 315.9MB filesize\n[23 Jan 2020 16:53:16-qAk][GReader] Loaded 25,481,095 / 25,481,095 (100.00%) of kmers parsed\n[23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 25,481,095 / 34,603,008 (73.64%)\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n[23 Jan 2020 16:53:16-qAk][asyncio] Inputs: 1; Threads: 1\n[23 Jan 2020 16:53:16-qAk][seq] Parsing sequence file sample.fastq\n[23 Jan 2020 16:53:16-qAk] sample.fastq: Qual scores: Sanger / Illumina 1.9+ (Phred+33) [offset: 33, range: [33,126], sample: [35,79]]\n[23 Jan 2020 16:53:16-qAk][seq] Loaded 1,035 reads and 0 reads pairs (file: sample.fastq)\n[23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 1,196 / 34,603,008 (0.00%)\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n[23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n[23 Jan 2020 16:53:16-qAk][task] input: sample.fastq colour: 0\n[23 Jan 2020 16:53:16-qAk]  SE reads: 1,035  PE reads: 0\n[23 Jan 2020 16:53:16-qAk]  good reads: 1,035  bad reads: 0\n[23 Jan 2020 16:53:16-qAk]  dup SE reads: 0  dup PE pairs: 0\n[23 Jan 2020 16:53:16-qAk]  bases read: 103,500  bases loaded: 103,426\n[23 Jan 2020 16:53:16-qAk]  num contigs: 1,035  num kmers: 17,170 novel kmers: 0\n[23 Jan 2020 16:53:16-qAk] Reading coverage...\n[23 Jan 2020 16:53:44-qAk] Printed graph coverage for 352,617 reads\n[23 Jan 2020 16:53:45-qAk] Dumping graph...\n[23 Jan 2020 16:53:45-qAk] Dumping graph colour 0 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:45-qAk][graph_writer_save] Writing colours 0-0 of 1 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n[23 Jan 2020 16:53:45-qAk] Dumped 1,196 kmers in 1 colour into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx (format version: 6)\n[23 Jan 2020 16:53:45-qAk][memory] We made 20 allocs\n[23 Jan 2020 16:53:45-qAk] Done.\n[23 Jan 2020 16:53:45-qAk][time] 39.00 seconds\nTraceback (most recent call last):\n  File \"/usr/local/bin/mykrobe\", line 11, in \u003cmodule\u003e\n    sys.exit(main())\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cli.py\", line 8, in main\n    args.func(parser, args)\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/parser.py\", line 42, in run_subtool\n    run(parser, args)\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cmds/amr.py\", line 274, in run\n    phylogenetics = species_predictor.run()\n  File \"/usr/local/lib/python3.7/site-packages/mykrobe/metagenomics/phylo.py\", line 65, in run\n    return MykrobePredictorPhylogeneticsResult(self.out_json[\"phylogenetics\"])\n  File \"/usr/local/lib/python3.7/site-packages/mongoengine/base/document.py\", line 79, in __init__\n    \"Instantiating a document with positional arguments is not \"\nTypeError: Instantiating a document with positional arguments is not supported. Please use `field_name=value` keyword arguments.\n", "tool_stdout": "", "update_time": "2020-01-23T16:53:52.056780", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp0jcugsuy/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp0jcugsuy/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp0jcugsuy/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp0jcugsuy/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp0jcugsuy/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id a90a30fafe298e1e - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - reads.fastq (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,035 sequences\n| Dataset Info:\n|  uploaded fastqsanger file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e@ERR117639.81080\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTGGCCGCGGCCAGTAGGAACATCCCCGGCCAGTCGTCCTCGGTCGGGCTACACAGGGTCACAGTCACAGAATCCGACTGTGGCATATGCCGCGGCCACG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e+\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCAFFHIIKKJKJKKJMJLKKKJKJJIKIKJMJHMJJHKHJLHMMJKKIKKKGJKJKML@KMJKJJJKLOILLLKII8HHFGKJJKJLJHHKHNGIGIGJE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@ERR117639.138514\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - prediction (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  INFO:mykrobe.cmds.amr:Running AMR prediction with panels data/panels/tb-species-170421.fasta.gz, data/panels/tb-bradley-probe-set-jan-2019.fasta.gz\n|  [23 Jan 2020 16:53:06-qAk][cmd] /usr/local/bin/mccortex31 geno -t 1 -m 1GB -k 21 -o tmp/sample-21_tb-speci\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  INFO:mykrobe.cmds.amr:Running AMR prediction with panels data/panels/tb-species-170421.fasta.gz, data/panels/tb-bradley-probe-set-jan-2019.fasta.gz\n|  [23 Jan 2020 16:53:06-qAk][cmd] /usr/local/bin/mccortex31 geno -t 1 -m 1GB -k 21 -o tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.covgs -I mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx -s sample-21 -1 sample.fastq -c /usr/local/share/mykrobe-data/panels/tb-species-170421.fasta.gz -c /usr/local/share/mykrobe-data/panels/tb-bradley-probe-set-jan-2019.fasta.gz tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:06-qAk][cwd] /tmp/tmp0jcugsuy/job_working_directory/000/23/working\n|  [23 Jan 2020 16:53:06-qAk][version] mccortex=v0.0.3-486-g22203aa-dirty zlib=1.2.11 htslib=1.2.1-250-gfa6ed9a-dirty ASSERTS=ON hash=Lookup3 CHECKS=ON k=3..31\n|  [23 Jan 2020 16:53:09-qAk] Saving graph to: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:09-qAk][sample] 0: sample-21\n|  [23 Jan 2020 16:53:09-qAk][task] sample.fastq; FASTQ offset: auto-detect, threshold: off; cut homopolymers: off; remove PCR duplicates: no; colour: 0\n|  [23 Jan 2020 16:53:09-qAk][memory] 112 bits per kmer\n|  [23 Jan 2020 16:53:09-qAk][memory] graph: 464MB\n|  [23 Jan 2020 16:53:09-qAk][memory] total: 464MB of 6.8GB RAM\n|  [23 Jan 2020 16:53:09-qAk] Writing 1 colour graph to tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:09-qAk][hasht] Allocating table with 34,603,008 entries, using 266MB\n|  [23 Jan 2020 16:53:09-qAk][hasht]  number of buckets: 1,048,576, bucket size: 33\n|  [23 Jan 2020 16:53:09-qAk][graph] kmer-size: 21; colours: 1; capacity: 34,603,008\n|  [23 Jan 2020 16:53:09-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 0 / 34,603,008 (0.00%)\n|  [23 Jan 2020 16:53:09-qAk][FileFilter] Loading file mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx [1 colour]\n|  [23 Jan 2020 16:53:09-qAk][GReader] 25,481,095 kmers, 315.9MB filesize\n|  [23 Jan 2020 16:53:16-qAk][GReader] Loaded 25,481,095 / 25,481,095 (100.00%) of kmers parsed\n|  [23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 25,481,095 / 34,603,008 (73.64%)\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n|  [23 Jan 2020 16:53:16-qAk][asyncio] Inputs: 1; Threads: 1\n|  [23 Jan 2020 16:53:16-qAk][seq] Parsing sequence file sample.fastq\n|  [23 Jan 2020 16:53:16-qAk] sample.fastq: Qual scores: Sanger / Illumina 1.9+ (Phred+33) [offset: 33, range: [33,126], sample: [35,79]]\n|  [23 Jan 2020 16:53:16-qAk][seq] Loaded 1,035 reads and 0 reads pairs (file: sample.fastq)\n|  [23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 1,196 / 34,603,008 (0.00%)\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n|  [23 Jan 2020 16:53:16-qAk][task] input: sample.fastq colour: 0\n|  [23 Jan 2020 16:53:16-qAk]  SE reads: 1,035  PE reads: 0\n|  [23 Jan 2020 16:53:16-qAk]  good reads: 1,035  bad reads: 0\n|  [23 Jan 2020 16:53:16-qAk]  dup SE reads: 0  dup PE pairs: 0\n|  [23 Jan 2020 16:53:16-qAk]  bases read: 103,500  bases loaded: 103,426\n|  [23 Jan 2020 16:53:16-qAk]  num contigs: 1,035  num kmers: 17,170 novel kmers: 0\n|  [23 Jan 2020 16:53:16-qAk] Reading coverage...\n|  [23 Jan 2020 16:53:44-qAk] Printed graph coverage for 352,617 reads\n|  [23 Jan 2020 16:53:45-qAk] Dumping graph...\n|  [23 Jan 2020 16:53:45-qAk] Dumping graph colour 0 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:45-qAk][graph_writer_save] Writing colours 0-0 of 1 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:45-qAk] Dumped 1,196 kmers in 1 colour into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx (format version: 6)\n|  [23 Jan 2020 16:53:45-qAk][memory] We made 20 allocs\n|  [23 Jan 2020 16:53:45-qAk] Done.\n|  [23 Jan 2020 16:53:45-qAk][time] 39.00 seconds\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/mykrobe\", line 11, in \u003cmodule\u003e\n|  sys.exit(main())\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cli.py\", line 8, in main\n|  args.func(parser, args)\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/parser.py\", line 42, in run_subtool\n|  run(parser, args)\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cmds/amr.py\", line 274, in run\n|  phylogenetics = species_predictor.run()\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/metagenomics/phylo.py\", line 65, in run\n|  return MykrobePredictorPhylogeneticsResult(self.out_json[\"phylogenetics\"])\n|  File \"/usr/local/lib/python3.7/site-packages/mongoengine/base/document.py\", line 79, in __init__\n|  \"Instantiating a document with positional arguments is not \"\n|  TypeError: Instantiating a document with positional arguments is not supported. Please use `field_name=value` keyword arguments.\n|\n--------------------------------------\n| Job 3e4e8de146a035f5\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:53:52.056780\n| Create Time:\n|  2020-01-23T16:52:46.878331\n|\n--------------------------------------\n| Job 5ca39fba41b940d1\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:52:45.443718\n| Create Time:\n|  2020-01-23T16:52:40.175128\n|\n--------------------------------------\nProblem in history with id a90a30fafe298e1e - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - reads.fastq (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,035 sequences\n| Dataset Info:\n|  uploaded fastqsanger file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e@ERR117639.81080\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTGGCCGCGGCCAGTAGGAACATCCCCGGCCAGTCGTCCTCGGTCGGGCTACACAGGGTCACAGTCACAGAATCCGACTGTGGCATATGCCGCGGCCACG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e+\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCAFFHIIKKJKJKKJMJLKKKJKJJIKIKJMJHMJJHKHJLHMMJKKIKKKGJKJKML@KMJKJJJKLOILLLKII8HHFGKJJKJLJHHKHNGIGIGJE\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e@ERR117639.138514\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - prediction (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  INFO:mykrobe.cmds.amr:Running AMR prediction with panels data/panels/tb-species-170421.fasta.gz, data/panels/tb-bradley-probe-set-jan-2019.fasta.gz\n|  [23 Jan 2020 16:53:06-qAk][cmd] /usr/local/bin/mccortex31 geno -t 1 -m 1GB -k 21 -o tmp/sample-21_tb-speci\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  INFO:mykrobe.cmds.amr:Running AMR prediction with panels data/panels/tb-species-170421.fasta.gz, data/panels/tb-bradley-probe-set-jan-2019.fasta.gz\n|  [23 Jan 2020 16:53:06-qAk][cmd] /usr/local/bin/mccortex31 geno -t 1 -m 1GB -k 21 -o tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.covgs -I mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx -s sample-21 -1 sample.fastq -c /usr/local/share/mykrobe-data/panels/tb-species-170421.fasta.gz -c /usr/local/share/mykrobe-data/panels/tb-bradley-probe-set-jan-2019.fasta.gz tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:06-qAk][cwd] /tmp/tmp0jcugsuy/job_working_directory/000/23/working\n|  [23 Jan 2020 16:53:06-qAk][version] mccortex=v0.0.3-486-g22203aa-dirty zlib=1.2.11 htslib=1.2.1-250-gfa6ed9a-dirty ASSERTS=ON hash=Lookup3 CHECKS=ON k=3..31\n|  [23 Jan 2020 16:53:09-qAk] Saving graph to: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:09-qAk][sample] 0: sample-21\n|  [23 Jan 2020 16:53:09-qAk][task] sample.fastq; FASTQ offset: auto-detect, threshold: off; cut homopolymers: off; remove PCR duplicates: no; colour: 0\n|  [23 Jan 2020 16:53:09-qAk][memory] 112 bits per kmer\n|  [23 Jan 2020 16:53:09-qAk][memory] graph: 464MB\n|  [23 Jan 2020 16:53:09-qAk][memory] total: 464MB of 6.8GB RAM\n|  [23 Jan 2020 16:53:09-qAk] Writing 1 colour graph to tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:09-qAk][hasht] Allocating table with 34,603,008 entries, using 266MB\n|  [23 Jan 2020 16:53:09-qAk][hasht]  number of buckets: 1,048,576, bucket size: 33\n|  [23 Jan 2020 16:53:09-qAk][graph] kmer-size: 21; colours: 1; capacity: 34,603,008\n|  [23 Jan 2020 16:53:09-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 0 / 34,603,008 (0.00%)\n|  [23 Jan 2020 16:53:09-qAk][FileFilter] Loading file mykrobe/data/skeletons/tb-species-170421-tb-bradley-probe-set-jan-2019_21.ctx [1 colour]\n|  [23 Jan 2020 16:53:09-qAk][GReader] 25,481,095 kmers, 315.9MB filesize\n|  [23 Jan 2020 16:53:16-qAk][GReader] Loaded 25,481,095 / 25,481,095 (100.00%) of kmers parsed\n|  [23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 25,481,095 / 34,603,008 (73.64%)\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n|  [23 Jan 2020 16:53:16-qAk][asyncio] Inputs: 1; Threads: 1\n|  [23 Jan 2020 16:53:16-qAk][seq] Parsing sequence file sample.fastq\n|  [23 Jan 2020 16:53:16-qAk] sample.fastq: Qual scores: Sanger / Illumina 1.9+ (Phred+33) [offset: 33, range: [33,126], sample: [35,79]]\n|  [23 Jan 2020 16:53:16-qAk][seq] Loaded 1,035 reads and 0 reads pairs (file: sample.fastq)\n|  [23 Jan 2020 16:53:16-qAk][hasht] buckets: 1,048,576 [2^20]; bucket size: 33; memory: 266MB; occupancy: 1,196 / 34,603,008 (0.00%)\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  0: 25373650\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  1: 104198\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  2: 3116\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  3: 126\n|  [23 Jan 2020 16:53:16-qAk][hasht]  collisions  4: 5\n|  [23 Jan 2020 16:53:16-qAk][task] input: sample.fastq colour: 0\n|  [23 Jan 2020 16:53:16-qAk]  SE reads: 1,035  PE reads: 0\n|  [23 Jan 2020 16:53:16-qAk]  good reads: 1,035  bad reads: 0\n|  [23 Jan 2020 16:53:16-qAk]  dup SE reads: 0  dup PE pairs: 0\n|  [23 Jan 2020 16:53:16-qAk]  bases read: 103,500  bases loaded: 103,426\n|  [23 Jan 2020 16:53:16-qAk]  num contigs: 1,035  num kmers: 17,170 novel kmers: 0\n|  [23 Jan 2020 16:53:16-qAk] Reading coverage...\n|  [23 Jan 2020 16:53:44-qAk] Printed graph coverage for 352,617 reads\n|  [23 Jan 2020 16:53:45-qAk] Dumping graph...\n|  [23 Jan 2020 16:53:45-qAk] Dumping graph colour 0 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:45-qAk][graph_writer_save] Writing colours 0-0 of 1 into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx\n|  [23 Jan 2020 16:53:45-qAk] Dumped 1,196 kmers in 1 colour into: tmp/sample-21_tb-species-170421-tb-bradley-probe-set-jan-2019.ctx (format version: 6)\n|  [23 Jan 2020 16:53:45-qAk][memory] We made 20 allocs\n|  [23 Jan 2020 16:53:45-qAk] Done.\n|  [23 Jan 2020 16:53:45-qAk][time] 39.00 seconds\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/mykrobe\", line 11, in \u003cmodule\u003e\n|  sys.exit(main())\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cli.py\", line 8, in main\n|  args.func(parser, args)\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/parser.py\", line 42, in run_subtool\n|  run(parser, args)\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/cmds/amr.py\", line 274, in run\n|  phylogenetics = species_predictor.run()\n|  File \"/usr/local/lib/python3.7/site-packages/mykrobe/metagenomics/phylo.py\", line 65, in run\n|  return MykrobePredictorPhylogeneticsResult(self.out_json[\"phylogenetics\"])\n|  File \"/usr/local/lib/python3.7/site-packages/mongoengine/base/document.py\", line 79, in __init__\n|  \"Instantiating a document with positional arguments is not \"\n|  TypeError: Instantiating a document with positional arguments is not supported. Please use `field_name=value` keyword arguments.\n|\n--------------------------------------\n| Job 3e4e8de146a035f5\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:53:52.056780\n| Create Time:\n|  2020-01-23T16:52:46.878331\n|\n--------------------------------------\n| Job 5ca39fba41b940d1\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:52:45.443718\n| Create Time:\n|  2020-01-23T16:52:40.175128\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 110.18529868125916, "tool_id": "mykrobe_predict", "tool_version": "0.7.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_mykrobe_predict.test_tool_000000"}, {"data": {"inputs": {"db|db_select": "taxonomy", "db|output_format": "full-xml", "query_source|id_list": "10239", "query_source|qss": "id_list"}, "job": {"command_line": "python /home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/efetch.py taxonomy   --id 10239   --retmode xml --rettype full   --user_email \"test@bx.psu.edu\" --admin_email \"planemo@galaxyproject.org;test@bx.psu.edu\"", "command_version": null, "create_time": "2020-01-23T17:06:17.701758", "exit_code": 1, "external_id": "17201", "galaxy_version": "20.01", "history_id": "3e4e8de146a035f5", "id": "25cc0305829779de", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}, {"code_desc": "", "desc": "Fatal error: ", "error_level": 3, "match": "Error:", "stream": "stderr", "type": "regex"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmppmodq1f4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "db": "{\"__current_case__\": 45, \"db_select\": \"taxonomy\", \"output_format\": \"full-xml\"}", "dbkey": "\"?\"", "query_source": "{\"__current_case__\": 2, \"id_list\": \"10239\", \"qss\": \"id_list\"}"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/efetch.py\", line 36, in \u003cmodule\u003e\n    c.fetch(args.db, ftype=args.retmode, **payload)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 52, in fetch\n    summary = self.id_summary(db, payload[\u0027id\u0027])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 71, in id_summary\n    return Entrez.read(Entrez.esummary(**payload))\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/__init__.py\", line 448, in read\n    from .Parser import DataHandler\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 164, in \u003cmodule\u003e\n    class DataHandler(object):\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 184, in DataHandler\n    raise exception\nOSError: [Errno 13] Permission denied: \u0027/.config\u0027\n", "stdout": "", "tool_id": "ncbi_eutils_efetch", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/efetch.py\", line 36, in \u003cmodule\u003e\n    c.fetch(args.db, ftype=args.retmode, **payload)\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 52, in fetch\n    summary = self.id_summary(db, payload[\u0027id\u0027])\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 71, in id_summary\n    return Entrez.read(Entrez.esummary(**payload))\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/__init__.py\", line 448, in read\n    from .Parser import DataHandler\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 164, in \u003cmodule\u003e\n    class DataHandler(object):\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 184, in DataHandler\n    raise exception\nOSError: [Errno 13] Permission denied: \u0027/.config\u0027\n", "tool_stdout": "", "update_time": "2020-01-23T17:06:22.447477", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Expected output collection [output1] to have 1 elements, but it had 0."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmppmodq1f4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nExpected output collection [output1] to have 1 elements, but it had 0.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 3e4e8de146a035f5 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - NCBI EFetch results (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 1, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u00273e4e8de146a035f5\u0027, \u0027id\u0027: \u00279ce08b2254e4d5ed\u0027, \u0027job_source_id\u0027: \u002725cc0305829779de\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027NCBI EFetch results\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/3e4e8de146a035f5/contents/dataset_collections/9ce08b2254e4d5ed\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| Job 25cc0305829779de\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:22.447477\n| Create Time:\n|  2020-01-23T17:06:17.701758\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 6.7485880851745605, "tool_id": "ncbi_eutils_efetch", "tool_version": "1.2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ncbi_eutils_efetch.test_tool_000000"}, {"data": {"inputs": {"db_select": "pubmed"}, "job": {"command_line": "python /home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/einfo.py --db pubmed   --user_email \"test@bx.psu.edu\" --admin_email \"planemo@galaxyproject.org;test@bx.psu.edu\"  \u003e /tmp/tmppmodq1f4/files/d/e/3/dataset_de3c56a0-7fa6-4e00-bb08-8d92837644bc.dat", "command_version": "python: can\u0027t open file \u0027einfo.py\u0027: [Errno 2] No such file or directory\n", "create_time": "2020-01-23T17:06:36.389571", "exit_code": 0, "external_id": "17653", "galaxy_version": "20.01", "history_id": "9a22496ee796fbfc", "id": "a6b9cabbe61f73ca", "inputs": {}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"default": {"id": "4923f2d7e29ffb4f", "src": "hda", "uuid": "de3c56a0-7fa6-4e00-bb08-8d92837644bc"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmppmodq1f4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "db_select": "\"pubmed\"", "dbkey": "\"?\""}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "ncbi_eutils_einfo", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T17:06:41.232978", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using contains):\n( /tmp/tmpr7wexc6npubmed.metadata.xml v. /tmp/tmpai_t_9q8pubmed.metadata.xml )\nFailed to find \u0027\u003c!DOCTYPE eInfoResult PUBLIC \"-//NLM//DTD einfo 20130322//EN\" \"https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20130322/einfo.dtd\"\u003e\u0027 in history data. (lines_diff=0):\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmppmodq1f4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using contains):\n( /tmp/tmpr7wexc6npubmed.metadata.xml v. /tmp/tmpai_t_9q8pubmed.metadata.xml )\nFailed to find \u0027\u003c!DOCTYPE eInfoResult PUBLIC \"-//NLM//DTD einfo 20130322//EN\" \"https://eutils.ncbi.nlm.nih.gov/eutils/dtd/20130322/einfo.dtd\"\u003e\u0027 in history data. (lines_diff=0):\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 6.69968843460083, "tool_id": "ncbi_eutils_einfo", "tool_version": "1.2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ncbi_eutils_einfo.test_tool_000000"}, {"data": {"inputs": {"db_select": "nuccore", "query_source|id_list": "126697566", "query_source|qss": "id_list"}, "job": {"command_line": "python /home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py nuccore   --user_email \"test@bx.psu.edu\" --admin_email \"planemo@galaxyproject.org;test@bx.psu.edu\"    --id 126697566   \u003e /tmp/tmppmodq1f4/files/d/7/6/dataset_d76cd71c-d369-4da9-8a1d-066739cc118b.dat", "command_version": "python: can\u0027t open file \u0027epost.py\u0027: [Errno 2] No such file or directory\n", "create_time": "2020-01-23T17:06:55.006016", "exit_code": 1, "external_id": "18100", "galaxy_version": "20.01", "history_id": "238993350abc9bfd", "id": "dccb60510b24da2f", "inputs": {}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}, {"code_desc": "", "desc": "Fatal error: ", "error_level": 3, "match": "Error:", "stream": "stderr", "type": "regex"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"history": {"id": "2fd653ff8fe916b4", "src": "hda", "uuid": "d76cd71c-d369-4da9-8a1d-066739cc118b"}}, "params": {"__input_ext": "\"data\"", "chromInfo": "\"/tmp/tmppmodq1f4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "db_select": "\"nuccore\"", "dbkey": "\"?\"", "query_source": "{\"__current_case__\": 2, \"id_list\": \"126697566\", \"qss\": \"id_list\"}"}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py\", line 30, in \u003cmodule\u003e\n    print(c.post(args.db, **payload))\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 46, in post\n    return json.dumps(Entrez.read(Entrez.epost(database, **payload)), indent=4)\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/__init__.py\", line 448, in read\n    from .Parser import DataHandler\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 164, in \u003cmodule\u003e\n    class DataHandler(object):\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 184, in DataHandler\n    raise exception\nOSError: [Errno 13] Permission denied: \u0027/.config\u0027\n", "stdout": "", "tool_id": "ncbi_eutils_epost", "tool_stderr": "Traceback (most recent call last):\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py\", line 30, in \u003cmodule\u003e\n    print(c.post(args.db, **payload))\n  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 46, in post\n    return json.dumps(Entrez.read(Entrez.epost(database, **payload)), indent=4)\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/__init__.py\", line 448, in read\n    from .Parser import DataHandler\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 164, in \u003cmodule\u003e\n    class DataHandler(object):\n  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 184, in DataHandler\n    raise exception\nOSError: [Errno 13] Permission denied: \u0027/.config\u0027\n", "tool_stdout": "", "update_time": "2020-01-23T17:06:59.975296", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmppmodq1f4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmppmodq1f4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 238993350abc9bfd - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - NCBI Entrez WebEnv History (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py\", line 30, in \u003cmodule\u003e\n|  print(c.post(args.db, **payload))\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py\", line 30, in \u003cmodule\u003e\n|  print(c.post(args.db, **payload))\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 46, in post\n|  return json.dumps(Entrez.read(Entrez.epost(database, **payload)), indent=4)\n|  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/__init__.py\", line 448, in read\n|  from .Parser import DataHandler\n|  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 164, in \u003cmodule\u003e\n|  class DataHandler(object):\n|  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 184, in DataHandler\n|  raise exception\n|  OSError: [Errno 13] Permission denied: \u0027/.config\u0027\n|\n--------------------------------------\n| Job dccb60510b24da2f\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:59.975296\n| Create Time:\n|  2020-01-23T17:06:55.006016\n|\n--------------------------------------\nProblem in history with id 238993350abc9bfd - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - NCBI Entrez WebEnv History (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py\", line 30, in \u003cmodule\u003e\n|  print(c.post(args.db, **payload))\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/epost.py\", line 30, in \u003cmodule\u003e\n|  print(c.post(args.db, **payload))\n|  File \"/home/runner/work/tools-iuc/tools-iuc/tools/ncbi_entrez_eutils/eutils.py\", line 46, in post\n|  return json.dumps(Entrez.read(Entrez.epost(database, **payload)), indent=4)\n|  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/__init__.py\", line 448, in read\n|  from .Parser import DataHandler\n|  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 164, in \u003cmodule\u003e\n|  class DataHandler(object):\n|  File \"/usr/local/lib/python2.7/site-packages/Bio/Entrez/Parser.py\", line 184, in DataHandler\n|  raise exception\n|  OSError: [Errno 13] Permission denied: \u0027/.config\u0027\n|\n--------------------------------------\n| Job dccb60510b24da2f\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:59.975296\n| Create Time:\n|  2020-01-23T17:06:55.006016\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 6.971858263015747, "tool_id": "ncbi_eutils_epost", "tool_version": "1.2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ncbi_eutils_epost.test_tool_000000"}, {"data": {"inputs": {"assemble": {"id": "a7e0eaa17f0249ff", "src": "hda"}, "cons1": {"id": "10a6a0683c3e243b", "src": "hda"}, "cons2": {"id": "2d6c63bdf033511b", "src": "hda"}, "cregion": {"id": "c8f3ca57d9826a1e", "src": "hda"}, "maskqual": {"id": "e4a1e6f57e384175", "src": "hda"}, "primer1": {"id": "2c1f3fe730d02717", "src": "hda"}, "primer2": {"id": "c6c2b27cd5eb370b", "src": "hda"}, "qual1": {"id": "f0074ae8e103c6b2", "src": "hda"}, "qual2": {"id": "01949e60fe484dc7", "src": "hda"}, "sample": "TestSample"}, "job": {"command_line": "ln -s \u0027/tmp/tmpflp8359h/files/2/c/4/dataset_2c4e1aa6-e2e5-44c3-afc1-6981a56a8893.dat\u0027    ./quality-1_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/6/e/a/dataset_6eab053f-94a1-472b-9976-406d8db1aba9.dat\u0027    ./quality-2_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/6/a/1/dataset_6a1b773d-a1f9-409b-9dad-f1ba74d55703.dat\u0027  ./primers-1_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/d/c/0/dataset_dc0f2a85-0d77-4d1a-b717-c613e8c107e7.dat\u0027  ./primers-2_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/3/8/c/dataset_38c6d052-018f-45f4-aed6-f0a9347f2253.dat\u0027    ./consensus-1_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/e/d/9/dataset_ed9216df-9a99-4946-8364-f177b3e732f4.dat\u0027    ./consensus-2_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/5/a/e/dataset_5ae69426-095a-4aea-918a-45f32b380719.dat\u0027 ./assemble_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/6/e/2/dataset_6e2e8489-1614-4071-a637-3faa11660799.dat\u0027 ./maskqual_table.tab \u0026\u0026 ln -s \u0027/tmp/tmpflp8359h/files/f/c/1/dataset_fc17b4c6-2c80-464e-bcd4-04853618b2aa.dat\u0027  ./cregion_table.tab \u0026\u0026 Rscript \u0027/tmp/tmpflp8359h/job_working_directory/000/209/configs/tmpl3ddt2c0\u0027", "command_version": "", "create_time": "2020-01-23T17:32:28.633935", "exit_code": 1, "external_id": "53136", "galaxy_version": "20.01", "history_id": "3d556e8e35b754dc", "id": "714401c2c9d376cc", "inputs": {"assemble": {"id": "a7e0eaa17f0249ff", "src": "hda", "uuid": "5ae69426-095a-4aea-918a-45f32b380719"}, "cons1": {"id": "10a6a0683c3e243b", "src": "hda", "uuid": "38c6d052-018f-45f4-aed6-f0a9347f2253"}, "cons2": {"id": "2d6c63bdf033511b", "src": "hda", "uuid": "ed9216df-9a99-4946-8364-f177b3e732f4"}, "cregion": {"id": "c8f3ca57d9826a1e", "src": "hda", "uuid": "fc17b4c6-2c80-464e-bcd4-04853618b2aa"}, "maskqual": {"id": "e4a1e6f57e384175", "src": "hda", "uuid": "6e2e8489-1614-4071-a637-3faa11660799"}, "primer1": {"id": "2c1f3fe730d02717", "src": "hda", "uuid": "6a1b773d-a1f9-409b-9dad-f1ba74d55703"}, "primer2": {"id": "c6c2b27cd5eb370b", "src": "hda", "uuid": "dc0f2a85-0d77-4d1a-b717-c613e8c107e7"}, "qual1": {"id": "f0074ae8e103c6b2", "src": "hda", "uuid": "2c4e1aa6-e2e5-44c3-afc1-6981a56a8893"}, "qual2": {"id": "01949e60fe484dc7", "src": "hda", "uuid": "6eab053f-94a1-472b-9976-406d8db1aba9"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output": {"id": "da62383fb67fc13a", "src": "hda", "uuid": "34724e6d-0c61-46f0-831b-da02f814d01e"}}, "params": {"__input_ext": "\"tabular\"", "chromInfo": "\"/tmp/tmpflp8359h/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "sample": "\"TestSample\""}, "state": "error", "stderr": "Loading required package: ggplot2\nQuitting from lines 40-71 (AbSeqV3.Rmd) \nError in dev.control(displaylist = if (record) \"enable\" else \"inhibit\") : \n  dev.control() called without an open graphics device\nCalls: report_abseq3 ... call_block -\u003e block_exec -\u003e chunk_device -\u003e dev.control\nIn addition: Warning messages:\n1: In grDevices::png(f) :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In grDevices::png(f) : failed to load cairo DLL\n3: In grDevices::png(f) : failed to load cairo DLL\n4: In (function (filename = \"Rplot%03d.png\", width = 480, height = 480,  :\n  failed to load cairo DLL\nExecution halted\n", "stdout": "", "tool_id": "prestor_abseq3", "tool_stderr": "Loading required package: ggplot2\nQuitting from lines 40-71 (AbSeqV3.Rmd) \nError in dev.control(displaylist = if (record) \"enable\" else \"inhibit\") : \n  dev.control() called without an open graphics device\nCalls: report_abseq3 ... call_block -\u003e block_exec -\u003e chunk_device -\u003e dev.control\nIn addition: Warning messages:\n1: In grDevices::png(f) :\n  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n  libXrender.so.1: cannot open shared object file: No such file or directory\n2: In grDevices::png(f) : failed to load cairo DLL\n3: In grDevices::png(f) : failed to load cairo DLL\n4: In (function (filename = \"Rplot%03d.png\", width = 480, height = 480,  :\n  failed to load cairo DLL\nExecution halted\n", "tool_stdout": "", "update_time": "2020-01-23T17:33:02.169771", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpflp8359h/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpflp8359h/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpflp8359h/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpflp8359h/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpflp8359h/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 3d556e8e35b754dc - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - quality-1_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eQUALITY\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003e33.166666666666664\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2103:2227:11139\u003c/td\u003e\u003ctd\u003e23.87\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1117:17250:9432\u003c/td\u003e\u003ctd\u003e35.35666666666667\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2108:14465:19705\u003c/td\u003e\u003ctd\u003e36.166666666666664\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - quality-2_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eQUALITY\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003e29.67\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1117:17250:9432\u003c/td\u003e\u003ctd\u003e25.023333333333333\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2108:14465:19705\u003c/td\u003e\u003ctd\u003e32.56666666666667\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2111:3554:13238\u003c/td\u003e\u003ctd\u003e24.566666666666666\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - primers-1_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-IGK\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2103:2227:11139\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-IGHA\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1117:17250:9432\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-IGL\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1112:28236:12457\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-TRA\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - primers-2_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1112:28236:12457\u003c/td\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2110:13535:17883\u003c/td\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003eTS-shift2\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1119:21261:11479\u003c/td\u003e\u003ctd\u003eCTTGATAGGAATGAATA\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - consensus-1_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003eSEQCOUNT\u003c/td\u003e\u003ctd\u003eCONSCOUNT\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003ePRCONS\u003c/td\u003e\u003ctd\u003ePRCOUNT\u003c/td\u003e\u003ctd\u003ePRFREQ\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGATAGGAATGAATA\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-TRA\u003c/td\u003e\u003ctd\u003eHuman-TRA\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-IGK\u003c/td\u003e\u003ctd\u003eHuman-IGK\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTTCTATGGCTTTGTT\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - consensus-2_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003eSEQCOUNT\u003c/td\u003e\u003ctd\u003eCONSCOUNT\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003ePRCONS\u003c/td\u003e\u003ctd\u003ePRCOUNT\u003c/td\u003e\u003ctd\u003ePRFREQ\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift2\u003c/td\u003e\u003ctd\u003eTS-shift2\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - assemble_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003cth\u003e12\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eREFID\u003c/td\u003e\u003ctd\u003eLENGTH\u003c/td\u003e\u003ctd\u003eOVERLAP\u003c/td\u003e\u003ctd\u003eGAP\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003ctd\u003ePVALUE\u003c/td\u003e\u003ctd\u003eEVALUE1\u003c/td\u003e\u003ctd\u003eEVALUE2\u003c/td\u003e\u003ctd\u003eIDENTITY\u003c/td\u003e\u003ctd\u003eFIELDS1\u003c/td\u003e\u003ctd\u003eFIELDS2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e435\u003c/td\u003e\u003ctd\u003e120\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0833\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-IGL\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e435\u003c/td\u003e\u003ctd\u003e119\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0168\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-TRA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e354\u003c/td\u003e\u003ctd\u003e200\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0500\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-IGK\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e414\u003c/td\u003e\u003ctd\u003e141\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0780\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-IGK\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - maskqual_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eMASKED\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG|CONSCOUNT=1,1|PRCONS=Human-IGL\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCCTTGTCCGAGATGTCT|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003e6\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTAGCTACATGTAATTG|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003e29\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTTCTATGGCTTTGTT|CONSCOUNT=1,1|PRCONS=Human-TRB\u003c/td\u003e\u003ctd\u003e200\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 9 - cregion_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG|CONSCOUNT=1,1|PRCONS=Human-IGL\u003c/td\u003e\u003ctd\u003eHuman-IGLC-2-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003eHuman-IGKC-IGKJ-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGGGACTCCAAAATTAAT|CONSCOUNT=1,1|PRCONS=Human-IGL\u003c/td\u003e\u003ctd\u003eHuman-IGLC-2-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCCTTGTCCGAGATGTCT|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003eHuman-IGKC-IGKJ-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 10 - pRESTOr AbSeq3 Report on data 9, data 8, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Loading required package: ggplot2\n|  Quitting from lines 40-71 (AbSeqV3.Rmd)\n|  Error in dev.control(displaylist = if (record) \"enable\" else \"inhibit\") :\n|  dev.control() called without an open graphics device\n|  Calls: report_abseq3 ... call_block -\u003e block_exec\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Loading required package: ggplot2\n|  Quitting from lines 40-71 (AbSeqV3.Rmd)\n|  Error in dev.control(displaylist = if (record) \"enable\" else \"inhibit\") :\n|  dev.control() called without an open graphics device\n|  Calls: report_abseq3 ... call_block -\u003e block_exec -\u003e chunk_device -\u003e dev.control\n|  In addition: Warning messages:\n|  1: In grDevices::png(f) :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In grDevices::png(f) : failed to load cairo DLL\n|  3: In grDevices::png(f) : failed to load cairo DLL\n|  4: In (function (filename = \"Rplot%03d.png\", width = 480, height = 480,  :\n|  failed to load cairo DLL\n|  Execution halted\n|\n--------------------------------------\n| Job 714401c2c9d376cc\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:33:02.169771\n| Create Time:\n|  2020-01-23T17:32:28.633935\n|\n--------------------------------------\n| Job a6b4a66ebc8d0103\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:26.956343\n| Create Time:\n|  2020-01-23T17:32:21.930886\n|\n--------------------------------------\n| Job 73f3912444e9fea9\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:20.548103\n| Create Time:\n|  2020-01-23T17:32:15.289195\n|\n--------------------------------------\n| Job 6f4d175e987c4a98\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:13.306820\n| Create Time:\n|  2020-01-23T17:32:08.585420\n|\n--------------------------------------\n| Job ca3d5ace97af4ba1\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:07.109973\n| Create Time:\n|  2020-01-23T17:32:01.936684\n|\n--------------------------------------\n| Job d64937030dbb3a90\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:00.847190\n| Create Time:\n|  2020-01-23T17:31:55.306843\n|\n--------------------------------------\n| Job 528acd0bdb59e47c\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:53.600362\n| Create Time:\n|  2020-01-23T17:31:48.611924\n|\n--------------------------------------\n| Job 1f643d6293754887\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:47.357340\n| Create Time:\n|  2020-01-23T17:31:41.970089\n|\n--------------------------------------\n| Job bccbc41c775b10ad\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:39.912218\n| Create Time:\n|  2020-01-23T17:31:35.248365\n|\n--------------------------------------\n| Job d278da44c7459418\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:33.714747\n| Create Time:\n|  2020-01-23T17:31:28.593012\n|\n--------------------------------------\nProblem in history with id 3d556e8e35b754dc - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - quality-1_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eQUALITY\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003e33.166666666666664\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2103:2227:11139\u003c/td\u003e\u003ctd\u003e23.87\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1117:17250:9432\u003c/td\u003e\u003ctd\u003e35.35666666666667\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2108:14465:19705\u003c/td\u003e\u003ctd\u003e36.166666666666664\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - quality-2_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eQUALITY\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003e29.67\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1117:17250:9432\u003c/td\u003e\u003ctd\u003e25.023333333333333\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2108:14465:19705\u003c/td\u003e\u003ctd\u003e32.56666666666667\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2111:3554:13238\u003c/td\u003e\u003ctd\u003e24.566666666666666\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - primers-1_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-IGK\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2103:2227:11139\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-IGHA\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1117:17250:9432\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-IGL\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1112:28236:12457\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eHuman-TRA\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - primers-2_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2116:20980:12498\u003c/td\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1112:28236:12457\u003c/td\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:2110:13535:17883\u003c/td\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003eTS-shift2\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eM01873:M01873:000000000-B9G3J:1:1119:21261:11479\u003c/td\u003e\u003ctd\u003eCTTGATAGGAATGAATA\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - consensus-1_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003eSEQCOUNT\u003c/td\u003e\u003ctd\u003eCONSCOUNT\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003ePRCONS\u003c/td\u003e\u003ctd\u003ePRCOUNT\u003c/td\u003e\u003ctd\u003ePRFREQ\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGATAGGAATGAATA\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-TRA\u003c/td\u003e\u003ctd\u003eHuman-TRA\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-IGK\u003c/td\u003e\u003ctd\u003eHuman-IGK\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTTCTATGGCTTTGTT\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003eHuman-TRB\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - consensus-2_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eBARCODE\u003c/td\u003e\u003ctd\u003eSEQCOUNT\u003c/td\u003e\u003ctd\u003eCONSCOUNT\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003ePRCONS\u003c/td\u003e\u003ctd\u003ePRCOUNT\u003c/td\u003e\u003ctd\u003ePRFREQ\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003eTS-shift4\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003eTS-shift3\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003eTS-shift2\u003c/td\u003e\u003ctd\u003eTS-shift2\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1.0\u003c/td\u003e\u003ctd\u003e0.0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - assemble_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003cth\u003e12\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eREFID\u003c/td\u003e\u003ctd\u003eLENGTH\u003c/td\u003e\u003ctd\u003eOVERLAP\u003c/td\u003e\u003ctd\u003eGAP\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003ctd\u003ePVALUE\u003c/td\u003e\u003ctd\u003eEVALUE1\u003c/td\u003e\u003ctd\u003eEVALUE2\u003c/td\u003e\u003ctd\u003eIDENTITY\u003c/td\u003e\u003ctd\u003eFIELDS1\u003c/td\u003e\u003ctd\u003eFIELDS2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e435\u003c/td\u003e\u003ctd\u003e120\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0833\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-IGL\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCGGTTTGGTGCATCTAG\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e435\u003c/td\u003e\u003ctd\u003e119\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0168\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-TRA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e354\u003c/td\u003e\u003ctd\u003e200\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0500\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-IGK\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTTCTTTAGTGAAATGC\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e414\u003c/td\u003e\u003ctd\u003e141\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e0.0780\u003c/td\u003e\u003ctd\u003e1.1102e-16\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1\u003c/td\u003e\u003ctd\u003eCONSCOUNT=1|PRCONS=Human-IGK\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - maskqual_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003eMASKED\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG|CONSCOUNT=1,1|PRCONS=Human-IGL\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCCTTGTCCGAGATGTCT|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003e6\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTAGCTACATGTAATTG|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003e29\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTTCTATGGCTTTGTT|CONSCOUNT=1,1|PRCONS=Human-TRB\u003c/td\u003e\u003ctd\u003e200\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 9 - cregion_table.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  100 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eID\u003c/td\u003e\u003ctd\u003ePRIMER\u003c/td\u003e\u003ctd\u003eERROR\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCTTGCTTATGATTCCAG|CONSCOUNT=1,1|PRCONS=Human-IGL\u003c/td\u003e\u003ctd\u003eHuman-IGLC-2-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATCTTTGCGGCTGGTTA|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003eHuman-IGKC-IGKJ-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGGGACTCCAAAATTAAT|CONSCOUNT=1,1|PRCONS=Human-IGL\u003c/td\u003e\u003ctd\u003eHuman-IGLC-2-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eCCTTGTCCGAGATGTCT|CONSCOUNT=1,1|PRCONS=Human-IGK\u003c/td\u003e\u003ctd\u003eHuman-IGKC-IGKJ-InternalC\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 10 - pRESTOr AbSeq3 Report on data 9, data 8, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Loading required package: ggplot2\n|  Quitting from lines 40-71 (AbSeqV3.Rmd)\n|  Error in dev.control(displaylist = if (record) \"enable\" else \"inhibit\") :\n|  dev.control() called without an open graphics device\n|  Calls: report_abseq3 ... call_block -\u003e block_exec\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Loading required package: ggplot2\n|  Quitting from lines 40-71 (AbSeqV3.Rmd)\n|  Error in dev.control(displaylist = if (record) \"enable\" else \"inhibit\") :\n|  dev.control() called without an open graphics device\n|  Calls: report_abseq3 ... call_block -\u003e block_exec -\u003e chunk_device -\u003e dev.control\n|  In addition: Warning messages:\n|  1: In grDevices::png(f) :\n|  unable to load shared object \u0027/usr/local/lib/R/library/grDevices/libs//cairo.so\u0027:\n|  libXrender.so.1: cannot open shared object file: No such file or directory\n|  2: In grDevices::png(f) : failed to load cairo DLL\n|  3: In grDevices::png(f) : failed to load cairo DLL\n|  4: In (function (filename = \"Rplot%03d.png\", width = 480, height = 480,  :\n|  failed to load cairo DLL\n|  Execution halted\n|\n--------------------------------------\n| Job 714401c2c9d376cc\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:33:02.169771\n| Create Time:\n|  2020-01-23T17:32:28.633935\n|\n--------------------------------------\n| Job a6b4a66ebc8d0103\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:26.956343\n| Create Time:\n|  2020-01-23T17:32:21.930886\n|\n--------------------------------------\n| Job 73f3912444e9fea9\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:20.548103\n| Create Time:\n|  2020-01-23T17:32:15.289195\n|\n--------------------------------------\n| Job 6f4d175e987c4a98\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:13.306820\n| Create Time:\n|  2020-01-23T17:32:08.585420\n|\n--------------------------------------\n| Job ca3d5ace97af4ba1\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:07.109973\n| Create Time:\n|  2020-01-23T17:32:01.936684\n|\n--------------------------------------\n| Job d64937030dbb3a90\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:32:00.847190\n| Create Time:\n|  2020-01-23T17:31:55.306843\n|\n--------------------------------------\n| Job 528acd0bdb59e47c\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:53.600362\n| Create Time:\n|  2020-01-23T17:31:48.611924\n|\n--------------------------------------\n| Job 1f643d6293754887\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:47.357340\n| Create Time:\n|  2020-01-23T17:31:41.970089\n|\n--------------------------------------\n| Job bccbc41c775b10ad\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:39.912218\n| Create Time:\n|  2020-01-23T17:31:35.248365\n|\n--------------------------------------\n| Job d278da44c7459418\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:31:33.714747\n| Create Time:\n|  2020-01-23T17:31:28.593012\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 113.18638610839844, "tool_id": "prestor_abseq3", "tool_version": "0.5.4"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_prestor_abseq3.test_tool_000000"}, {"data": {"inputs": {"al|ambiguity_score": "0.99", "al|ambiguity_usage": "one", "al|fragmented": true, "al|fragmented_max_indent": "50", "al|min_alignment": "65", "al|min_identity": "95.0", "al|upper_bound_assembly": true, "al|upper_bound_min_con": "2", "al|use_all_alignments": true, "assembly|orga_type": "", "assembly|ref|features": {"id": "67ef09f1c4f8fd77", "src": "hda"}, "assembly|ref|r": {"id": "ab0b902fc4c6ecc1", "src": "hda"}, "assembly|ref|use_ref": "true", "assembly|type": "genome", "circos": true, "contig_thresholds": "0,1000", "extensive_mis_size": "1000", "genes|conserved_genes_finding": true, "genes|rna_finding": true, "in|custom": "true", "in|inputs_0|input": {"id": "a86a1a479f13fd91", "src": "hda"}, "in|inputs_0|labels": "contig1", "in|inputs_1|input": {"id": "32b5b014fd8ff7d7", "src": "hda"}, "in|inputs_1|labels": "", "k_mer|k_mer_size": "101", "k_mer|k_mer_stats": "--k-mer-stats", "large": true, "min_contig": "500", "scaffold_gap_max_size": "1000", "skip_unaligned_mis_contigs": true, "split_scaffolds": false, "strict_NA": true, "unaligned_part_size": "500"}, "job": {"command_line": "quast -o outputdir  -r \u0027/tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat\u0027 --features \u0027/tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat\u0027  --min-contig 500 --threads ${GALAXY_SLOTS:-1}  --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage \u0027one\u0027 --fragmented --contig-thresholds \u00270,1000\u0027 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500   \u0027/tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat\u0027 \u0027/tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\u0027  \u0026\u0026 mkdir -p \u0027/tmp/tmpng6x1hvs/job_working_directory/000/133/working/dataset_6acfa4b6-e005-4788-819c-a30a46761503_files\u0027 \u0026\u0026 cp outputdir/*.html \u0027/tmp/tmpng6x1hvs/job_working_directory/000/133/working/dataset_6acfa4b6-e005-4788-819c-a30a46761503_files\u0027 \u0026\u0026 cp -R outputdir/icarus_viewers \u0027/tmp/tmpng6x1hvs/job_working_directory/000/133/working/dataset_6acfa4b6-e005-4788-819c-a30a46761503_files\u0027", "command_version": "", "create_time": "2020-01-23T17:14:51.266446", "exit_code": 1, "external_id": "21538", "galaxy_version": "20.01", "history_id": "f16a54a3ef316466", "id": "83e248eabb1b3d31", "inputs": {"assembly|features": {"id": "67ef09f1c4f8fd77", "src": "hda", "uuid": "ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d"}, "assembly|r": {"id": "ab0b902fc4c6ecc1", "src": "hda", "uuid": "e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6"}, "in|inputs_0|input": {"id": "a86a1a479f13fd91", "src": "hda", "uuid": "ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6"}, "in|inputs_1|input": {"id": "32b5b014fd8ff7d7", "src": "hda", "uuid": "508c0de4-813a-4bd4-9549-77bc67215ed7"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"kmers": {"id": "12eb47432b3fa224", "src": "hda", "uuid": "48e2d0da-b1db-41e7-9cd9-c90d07d42f94"}, "log": {"id": "3837873dd5f5f1f7", "src": "hda", "uuid": "38cb80ef-1586-494e-9cd0-aa561a42f6f1"}, "mis_ass": {"id": "3513607ccdf74186", "src": "hda", "uuid": "a198fb3a-8c47-48ad-ab1a-6b8faf600153"}, "quast_tabular": {"id": "b722e15494aaed88", "src": "hda", "uuid": "74ded3b5-65ff-4811-94c8-649a7cb1535b"}, "report_html": {"id": "efa1406254c179e5", "src": "hda", "uuid": "6acfa4b6-e005-4788-819c-a30a46761503"}, "report_pdf": {"id": "18031c6acd0dc05e", "src": "hda", "uuid": "c6d9ce4d-4d4e-47db-8b7d-917fe6074e10"}, "unalign": {"id": "c465dd32e1971030", "src": "hda", "uuid": "658926a3-edb5-475e-936a-7a3d869263a8"}}, "params": {"__input_ext": "\"fasta\"", "al": "{\"ambiguity_score\": \"0.99\", \"ambiguity_usage\": \"one\", \"fragmented\": \"true\", \"fragmented_max_indent\": \"50\", \"min_alignment\": \"65\", \"min_identity\": \"95.0\", \"upper_bound_assembly\": \"true\", \"upper_bound_min_con\": \"2\", \"use_all_alignments\": \"true\"}", "assembly": "{\"__current_case__\": 0, \"orga_type\": \"\", \"ref\": {\"__current_case__\": 0, \"features\": {\"values\": [{\"id\": 214, \"src\": \"hda\"}]}, \"operons\": null, \"r\": {\"values\": [{\"id\": 213, \"src\": \"hda\"}]}, \"use_ref\": \"true\"}, \"type\": \"genome\"}", "chromInfo": "\"/tmp/tmpng6x1hvs/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "circos": "\"true\"", "contig_thresholds": "\"0,1000\"", "dbkey": "\"?\"", "extensive_mis_size": "\"1000\"", "genes": "{\"conserved_genes_finding\": \"true\", \"gene_finding\": {\"__current_case__\": 0, \"tool\": \"none\"}, \"rna_finding\": \"true\"}", "in": "{\"__current_case__\": 0, \"custom\": \"true\", \"inputs\": [{\"__index__\": 0, \"input\": {\"values\": [{\"id\": 211, \"src\": \"hda\"}]}, \"labels\": \"contig1\"}, {\"__index__\": 1, \"input\": {\"values\": [{\"id\": 212, \"src\": \"hda\"}]}, \"labels\": \"\"}]}", "k_mer": "{\"__current_case__\": 0, \"k_mer_size\": \"101\", \"k_mer_stats\": \"--k-mer-stats\"}", "large": "\"true\"", "min_contig": "\"500\"", "scaffold_gap_max_size": "\"1000\"", "skip_unaligned_mis_contigs": "\"true\"", "split_scaffolds": "\"false\"", "strict_NA": "\"true\"", "unaligned_part_size": "\"500\""}, "state": "error", "stderr": "ERROR! exception caught!\n\nIn case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\nor report an issue on our GitHub repository https://github.com/ablab/quast/issues\nPlease provide us with quast.log file from the output directory.\n", "stdout": "WARNING: Python locale settings can\u0027t be changed\n/usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:15:52\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n\nCWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\nMain parameters: \n  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n\nReference:\n  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n\nContigs:\n  Pre-processing...\n  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n\n2020-01-23 17:15:53\nRunning Basic statistics processor...\n  Reference genome:\n    dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n  Contig files: \n    1  contig1\n    2  contigs2_fna\n\n\u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\nTraceback (most recent call last):\n  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n    return_code = main(sys.argv[1:])\n  File \"/usr/local/bin/quast\", line 149, in main\n    icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n    html_saver.save_reference_lengths(results_dir, reference_lengths)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n    append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n    init(html_fpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n    script_texts.append(js_html(aux_f_rel_path))\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n    return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n    return codecs.ascii_decode(input, self.errors)[0]\nUnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n", "tool_id": "quast", "tool_stderr": "ERROR! exception caught!\n\nIn case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\nor report an issue on our GitHub repository https://github.com/ablab/quast/issues\nPlease provide us with quast.log file from the output directory.\n", "tool_stdout": "WARNING: Python locale settings can\u0027t be changed\n/usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:15:52\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n\nCWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\nMain parameters: \n  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n\nReference:\n  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n\nContigs:\n  Pre-processing...\n  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n\n2020-01-23 17:15:53\nRunning Basic statistics processor...\n  Reference genome:\n    dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n  Contig files: \n    1  contig1\n    2  contigs2_fna\n\n\u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\nTraceback (most recent call last):\n  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n    return_code = main(sys.argv[1:])\n  File \"/usr/local/bin/quast\", line 149, in main\n    icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n    html_saver.save_reference_lengths(results_dir, reference_lengths)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n    append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n    init(html_fpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n    script_texts.append(js_html(aux_f_rel_path))\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n    return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n    return codecs.ascii_decode(input, self.errors)[0]\nUnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n", "update_time": "2020-01-23T17:15:57.164768", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state.", "Job in error state.", "Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\nJob in error state.\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id f16a54a3ef316466 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - reference.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - genes.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  14 lines, 8 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!gff-spec-version 1.21\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!processor NCBI annotwriter\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build ASM584v2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build-accession NCBI_Assembly:GCF_000005845.2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - Quast on data 4, data 3, and others: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 4, data 3, and others:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 7 - Quast on data 4, data 3, and others:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 8 - Quast on data 4, data 3, and others: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 9 - Quast on data 4, data 3, and others: Misassemblies (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 10 - Quast on data 4, data 3, and others: Unaligned contigs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 11 - Quast on data 4, data 3, and others: K-mer-based metrics (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 83e248eabb1b3d31\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:15:57.164768\n| Create Time:\n|  2020-01-23T17:14:51.266446\n|\n--------------------------------------\n| Job 2f2abd7e16b39826\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:49.373875\n| Create Time:\n|  2020-01-23T17:14:44.459710\n|\n--------------------------------------\n| Job 5e985d59d5900484\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:42.973134\n| Create Time:\n|  2020-01-23T17:14:37.787649\n|\n--------------------------------------\n| Job fc5c2ad70cbb4f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:35.631912\n| Create Time:\n|  2020-01-23T17:14:31.130317\n|\n--------------------------------------\n| Job e9cf7d8d4867436b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:29.433558\n| Create Time:\n|  2020-01-23T17:14:24.392833\n|\n--------------------------------------\nProblem in history with id f16a54a3ef316466 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - reference.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - genes.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  14 lines, 8 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!gff-spec-version 1.21\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!processor NCBI annotwriter\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build ASM584v2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build-accession NCBI_Assembly:GCF_000005845.2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - Quast on data 4, data 3, and others: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 4, data 3, and others:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 7 - Quast on data 4, data 3, and others:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 8 - Quast on data 4, data 3, and others: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 9 - Quast on data 4, data 3, and others: Misassemblies (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 10 - Quast on data 4, data 3, and others: Unaligned contigs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 11 - Quast on data 4, data 3, and others: K-mer-based metrics (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 83e248eabb1b3d31\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:15:57.164768\n| Create Time:\n|  2020-01-23T17:14:51.266446\n|\n--------------------------------------\n| Job 2f2abd7e16b39826\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:49.373875\n| Create Time:\n|  2020-01-23T17:14:44.459710\n|\n--------------------------------------\n| Job 5e985d59d5900484\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:42.973134\n| Create Time:\n|  2020-01-23T17:14:37.787649\n|\n--------------------------------------\n| Job fc5c2ad70cbb4f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:35.631912\n| Create Time:\n|  2020-01-23T17:14:31.130317\n|\n--------------------------------------\n| Job e9cf7d8d4867436b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:29.433558\n| Create Time:\n|  2020-01-23T17:14:24.392833\n|\n--------------------------------------\nProblem in history with id f16a54a3ef316466 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - reference.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - genes.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  14 lines, 8 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!gff-spec-version 1.21\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!processor NCBI annotwriter\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build ASM584v2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build-accession NCBI_Assembly:GCF_000005845.2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - Quast on data 4, data 3, and others: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 4, data 3, and others:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 7 - Quast on data 4, data 3, and others:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 8 - Quast on data 4, data 3, and others: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 9 - Quast on data 4, data 3, and others: Misassemblies (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 10 - Quast on data 4, data 3, and others: Unaligned contigs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 11 - Quast on data 4, data 3, and others: K-mer-based metrics (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 83e248eabb1b3d31\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:15:57.164768\n| Create Time:\n|  2020-01-23T17:14:51.266446\n|\n--------------------------------------\n| Job 2f2abd7e16b39826\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:49.373875\n| Create Time:\n|  2020-01-23T17:14:44.459710\n|\n--------------------------------------\n| Job 5e985d59d5900484\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:42.973134\n| Create Time:\n|  2020-01-23T17:14:37.787649\n|\n--------------------------------------\n| Job fc5c2ad70cbb4f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:35.631912\n| Create Time:\n|  2020-01-23T17:14:31.130317\n|\n--------------------------------------\n| Job e9cf7d8d4867436b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:29.433558\n| Create Time:\n|  2020-01-23T17:14:24.392833\n|\n--------------------------------------\nProblem in history with id f16a54a3ef316466 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - reference.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - genes.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  14 lines, 8 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!gff-spec-version 1.21\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!processor NCBI annotwriter\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build ASM584v2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build-accession NCBI_Assembly:GCF_000005845.2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - Quast on data 4, data 3, and others: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 4, data 3, and others:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 7 - Quast on data 4, data 3, and others:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 8 - Quast on data 4, data 3, and others: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 9 - Quast on data 4, data 3, and others: Misassemblies (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 10 - Quast on data 4, data 3, and others: Unaligned contigs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 11 - Quast on data 4, data 3, and others: K-mer-based metrics (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 83e248eabb1b3d31\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:15:57.164768\n| Create Time:\n|  2020-01-23T17:14:51.266446\n|\n--------------------------------------\n| Job 2f2abd7e16b39826\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:49.373875\n| Create Time:\n|  2020-01-23T17:14:44.459710\n|\n--------------------------------------\n| Job 5e985d59d5900484\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:42.973134\n| Create Time:\n|  2020-01-23T17:14:37.787649\n|\n--------------------------------------\n| Job fc5c2ad70cbb4f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:35.631912\n| Create Time:\n|  2020-01-23T17:14:31.130317\n|\n--------------------------------------\n| Job e9cf7d8d4867436b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:29.433558\n| Create Time:\n|  2020-01-23T17:14:24.392833\n|\n--------------------------------------\nProblem in history with id f16a54a3ef316466 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - reference.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - genes.gff (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  14 lines, 8 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!gff-spec-version 1.21\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!processor NCBI annotwriter\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build ASM584v2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e#!genome-build-accession NCBI_Assembly:GCF_000005845.2\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - Quast on data 4, data 3, and others: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 4, data 3, and others:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 7 - Quast on data 4, data 3, and others:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 8 - Quast on data 4, data 3, and others: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 9 - Quast on data 4, data 3, and others: Misassemblies (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 10 - Quast on data 4, data 3, and others: Unaligned contigs (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 11 - Quast on data 4, data 3, and others: K-mer-based metrics (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat -\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir -r /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat --features /tmp/tmpng6x1hvs/files/e/d/6/dataset_ed6ed73d-5a4d-46a0-a23f-b0bebd4f393d.dat --min-contig 500 --threads 1 --labels contig1,contigs2_fna --large --k-mer-stats --k-mer-size 101 --circos --rna-finding --conserved-genes-finding --use-all-alignments --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --fragmented --contig-thresholds 0,1000 --strict-NA --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:15:52\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/133/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/133/working\n|  Main parameters:\n|  MODE: large, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, use all alignments: true, threshold for extensive misassembly size: 1000\n|  Reference:\n|  /tmp/tmpng6x1hvs/files/e/4/4/dataset_e44ef5ff-3a6d-4626-8fc1-bcfdd1d966c6.dat ==\u003e dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/c/3/dataset_ac3978e3-fc6e-4b9e-b9df-fdfb5ddd8bc6.dat ==\u003e contig1\n|  2  /tmp/tmpng6x1hvs/files/5/0/8/dataset_508c0de4-813a-4bd4-9549-77bc67215ed7.dat ==\u003e contigs2_fna\n|  2020-01-23 17:15:53\n|  Running Basic statistics processor...\n|  Reference genome:\n|  dataset_e44ef5ff_3a6d_4626_8fc1_bcfdd1d966c6.dat, length = 6650, num fragments = 1, GC % = 52.00\n|  Contig files:\n|  1  contig1\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 259, in do\n|  html_saver.save_reference_lengths(results_dir, reference_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 463, in save_reference_lengths\n|  append(results_dirpath, json_fpath, \u0027referenceLength\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 83e248eabb1b3d31\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:15:57.164768\n| Create Time:\n|  2020-01-23T17:14:51.266446\n|\n--------------------------------------\n| Job 2f2abd7e16b39826\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:49.373875\n| Create Time:\n|  2020-01-23T17:14:44.459710\n|\n--------------------------------------\n| Job 5e985d59d5900484\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:42.973134\n| Create Time:\n|  2020-01-23T17:14:37.787649\n|\n--------------------------------------\n| Job fc5c2ad70cbb4f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:35.631912\n| Create Time:\n|  2020-01-23T17:14:31.130317\n|\n--------------------------------------\n| Job e9cf7d8d4867436b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:14:29.433558\n| Create Time:\n|  2020-01-23T17:14:24.392833\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 134.1225140094757, "tool_id": "quast", "tool_version": "5.0.2+galaxy0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_quast.test_tool_000000"}, {"data": {"inputs": {"al|ambiguity_score": "0.99", "al|ambiguity_usage": "one", "al|fragmented": false, "al|fragmented_max_indent": "50", "al|min_alignment": "65", "al|min_identity": "95.0", "al|upper_bound_assembly": false, "al|upper_bound_min_con": "2", "al|use_all_alignments": false, "assembly|orga_type": "--eukaryote", "assembly|ref|use_ref": "false", "assembly|type": "genome", "circos": false, "contig_thresholds": "0,1000, 500", "extensive_mis_size": "1000", "genes|conserved_genes_finding": false, "genes|gene_finding|tool": "none", "genes|rna_finding": false, "in|custom": "false", "in|inputs": [{"id": "88eca6cbe8e287ba", "src": "hda"}, {"id": "6194a3cf14a0177a", "src": "hda"}], "k_mer|k_mer_stats": "", "large": false, "min_contig": "500", "scaffold_gap_max_size": "1000", "skip_unaligned_mis_contigs": false, "split_scaffolds": false, "strict_NA": false, "unaligned_part_size": "500"}, "job": {"command_line": "quast -o outputdir  --eukaryote --min-contig 500 --threads ${GALAXY_SLOTS:-1}  --labels contigs1_fna,contigs2_fna       --min-alignment 65 --min-identity 95.0 --ambiguity-usage \u0027one\u0027  --contig-thresholds \u00270,1000,500\u0027  --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs  \u0027/tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat\u0027 \u0027/tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\u0027  \u0026\u0026 mkdir -p \u0027/tmp/tmpng6x1hvs/job_working_directory/000/136/working/dataset_b2fe543a-8797-4dc8-840f-84616eb787c3_files\u0027 \u0026\u0026 cp outputdir/*.html \u0027/tmp/tmpng6x1hvs/job_working_directory/000/136/working/dataset_b2fe543a-8797-4dc8-840f-84616eb787c3_files\u0027 \u0026\u0026 cp -R outputdir/icarus_viewers \u0027/tmp/tmpng6x1hvs/job_working_directory/000/136/working/dataset_b2fe543a-8797-4dc8-840f-84616eb787c3_files\u0027", "command_version": "", "create_time": "2020-01-23T17:16:51.843265", "exit_code": 1, "external_id": "22133", "galaxy_version": "20.01", "history_id": "e511a12dc2cf12e1", "id": "15839f6e830b1abe", "inputs": {"inputs": {"id": "88eca6cbe8e287ba", "src": "hda", "uuid": "afb5b9cd-b726-4da9-bf8e-7994b47f5a2c"}, "inputs1": {"id": "88eca6cbe8e287ba", "src": "hda", "uuid": "afb5b9cd-b726-4da9-bf8e-7994b47f5a2c"}, "inputs2": {"id": "6194a3cf14a0177a", "src": "hda", "uuid": "588a0510-75ba-4bf4-a76a-ee2aa3b1aedc"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"log": {"id": "c786be9a5c4573e3", "src": "hda", "uuid": "0f780bad-fb01-4437-9551-c7c15d7abbeb"}, "quast_tabular": {"id": "f2fbab8d8d8f0547", "src": "hda", "uuid": "ea96917f-4b30-4f97-bf33-66a7ec0eff50"}, "report_html": {"id": "ce239e512e8f2197", "src": "hda", "uuid": "b2fe543a-8797-4dc8-840f-84616eb787c3"}, "report_pdf": {"id": "494f12e6e7ea7cc7", "src": "hda", "uuid": "6abaa1be-151c-4027-9f0a-51cabf6d97a0"}}, "params": {"__input_ext": "\"fasta\"", "al": "{\"ambiguity_score\": \"0.99\", \"ambiguity_usage\": \"one\", \"fragmented\": \"false\", \"fragmented_max_indent\": \"50\", \"min_alignment\": \"65\", \"min_identity\": \"95.0\", \"upper_bound_assembly\": \"false\", \"upper_bound_min_con\": \"2\", \"use_all_alignments\": \"false\"}", "assembly": "{\"__current_case__\": 0, \"orga_type\": \"--eukaryote\", \"ref\": {\"__current_case__\": 1, \"est_ref_size\": \"\", \"use_ref\": \"false\"}, \"type\": \"genome\"}", "chromInfo": "\"/tmp/tmpng6x1hvs/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "circos": "\"false\"", "contig_thresholds": "\"0,1000, 500\"", "dbkey": "\"?\"", "extensive_mis_size": "\"1000\"", "genes": "{\"conserved_genes_finding\": \"false\", \"gene_finding\": {\"__current_case__\": 0, \"tool\": \"none\"}, \"rna_finding\": \"false\"}", "in": "{\"__current_case__\": 1, \"custom\": \"false\", \"inputs\": {\"values\": [{\"id\": 222, \"src\": \"hda\"}, {\"id\": 223, \"src\": \"hda\"}]}}", "k_mer": "{\"__current_case__\": 1, \"k_mer_stats\": \"\"}", "large": "\"false\"", "min_contig": "\"500\"", "scaffold_gap_max_size": "\"1000\"", "skip_unaligned_mis_contigs": "\"false\"", "split_scaffolds": "\"false\"", "strict_NA": "\"false\"", "unaligned_part_size": "\"500\""}, "state": "error", "stderr": "ERROR! exception caught!\n\nIn case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\nor report an issue on our GitHub repository https://github.com/ablab/quast/issues\nPlease provide us with quast.log file from the output directory.\n", "stdout": "WARNING: Python locale settings can\u0027t be changed\n/usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:16:53\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n\nCWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\nMain parameters: \n  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n  ambiguity: one, threshold for extensive misassembly size: 1000\n\nContigs:\n  Pre-processing...\n  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n\n2020-01-23 17:16:53\nRunning Basic statistics processor...\n  Contig files: \n    1  contigs1_fna\n    2  contigs2_fna\n\n\u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\nTraceback (most recent call last):\n  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n    return_code = main(sys.argv[1:])\n  File \"/usr/local/bin/quast\", line 149, in main\n    icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n    html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n    append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n    init(html_fpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n    script_texts.append(js_html(aux_f_rel_path))\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n    return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n    return codecs.ascii_decode(input, self.errors)[0]\nUnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n", "tool_id": "quast", "tool_stderr": "ERROR! exception caught!\n\nIn case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\nor report an issue on our GitHub repository https://github.com/ablab/quast/issues\nPlease provide us with quast.log file from the output directory.\n", "tool_stdout": "WARNING: Python locale settings can\u0027t be changed\n/usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:16:53\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n\nCWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\nMain parameters: \n  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n  ambiguity: one, threshold for extensive misassembly size: 1000\n\nContigs:\n  Pre-processing...\n  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n\n2020-01-23 17:16:53\nRunning Basic statistics processor...\n  Contig files: \n    1  contigs1_fna\n    2  contigs2_fna\n\n\u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\nTraceback (most recent call last):\n  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n    return_code = main(sys.argv[1:])\n  File \"/usr/local/bin/quast\", line 149, in main\n    icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n    html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n    append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n    init(html_fpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n    script_texts.append(js_html(aux_f_rel_path))\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n    return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n    return codecs.ascii_decode(input, self.errors)[0]\nUnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n", "update_time": "2020-01-23T17:16:56.844949", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state.", "Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id e511a12dc2cf12e1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Quast on data 2 and data 1: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 4 - Quast on data 2 and data 1:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 5 - Quast on data 2 and data 1:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 2 and data 1: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 15839f6e830b1abe\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:16:56.844949\n| Create Time:\n|  2020-01-23T17:16:51.843265\n|\n--------------------------------------\n| Job 4e27b888e0f89616\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:50.417232\n| Create Time:\n|  2020-01-23T17:16:45.229626\n|\n--------------------------------------\n| Job a01e20e32f0965d9\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:43.188125\n| Create Time:\n|  2020-01-23T17:16:38.529718\n|\n--------------------------------------\nProblem in history with id e511a12dc2cf12e1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Quast on data 2 and data 1: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 4 - Quast on data 2 and data 1:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 5 - Quast on data 2 and data 1:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 2 and data 1: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 15839f6e830b1abe\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:16:56.844949\n| Create Time:\n|  2020-01-23T17:16:51.843265\n|\n--------------------------------------\n| Job 4e27b888e0f89616\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:50.417232\n| Create Time:\n|  2020-01-23T17:16:45.229626\n|\n--------------------------------------\n| Job a01e20e32f0965d9\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:43.188125\n| Create Time:\n|  2020-01-23T17:16:38.529718\n|\n--------------------------------------\nProblem in history with id e511a12dc2cf12e1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Quast on data 2 and data 1: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 4 - Quast on data 2 and data 1:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 5 - Quast on data 2 and data 1:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 2 and data 1: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 15839f6e830b1abe\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:16:56.844949\n| Create Time:\n|  2020-01-23T17:16:51.843265\n|\n--------------------------------------\n| Job 4e27b888e0f89616\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:50.417232\n| Create Time:\n|  2020-01-23T17:16:45.229626\n|\n--------------------------------------\n| Job a01e20e32f0965d9\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:43.188125\n| Create Time:\n|  2020-01-23T17:16:38.529718\n|\n--------------------------------------\nProblem in history with id e511a12dc2cf12e1 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs1.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  3 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;contig1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTCTGATAGCAGC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGGTCACTAAATACTTTAACCAA\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTACACAACATCCATGAAACGCATTAGCACCACC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eATTACCACCACCATCACCATTACCACAGGTAACGGTGCGGGCTGACGCGTACAGGAAACACAGAAAAAAG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - contigs2.fna (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NC_000913.3:1-6650\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGCTTTTCATTCTGACTGCAACGGGCAATATGTCTCTGTGTGGATTAAAAAAAGAGTGTC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTGATAGCAGCTTCTGAACTGGTTACCTGCCGTGAGTAAATTAAAATTTTATTGACTTAGG\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTCACTAAATACTTTAACCAATATAGGCATAGCGCACAGACAGATAAAAATTACAGAGTAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eACAACATCCATGAAACGCATTAGCACCACCATTACCACCACCATCACCATTACCACAGGT\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Quast on data 2 and data 1: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 4 - Quast on data 2 and data 1:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 5 - Quast on data 2 and data 1:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 6 - Quast on data 2 and data 1: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --e\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/bin/quast -o outputdir --eukaryote --min-contig 500 --threads 1 --labels contigs1_fna,contigs2_fna --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:16:53\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/136/working/outputdir/quast.log\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/136/working\n|  Main parameters:\n|  MODE: default, threads: 1, eukaryotic: true, minimum contig length: 500, minimum alignment length: 65, \\\n|  ambiguity: one, threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  1  /tmp/tmpng6x1hvs/files/a/f/b/dataset_afb5b9cd-b726-4da9-bf8e-7994b47f5a2c.dat ==\u003e contigs1_fna\n|  2  /tmp/tmpng6x1hvs/files/5/8/8/dataset_588a0510-75ba-4bf4-a76a-ee2aa3b1aedc.dat ==\u003e contigs2_fna\n|  2020-01-23 17:16:53\n|  Running Basic statistics processor...\n|  Contig files:\n|  1  contigs1_fna\n|  2  contigs2_fna\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/quast\", line 307, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/quast\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 15839f6e830b1abe\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:16:56.844949\n| Create Time:\n|  2020-01-23T17:16:51.843265\n|\n--------------------------------------\n| Job 4e27b888e0f89616\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:50.417232\n| Create Time:\n|  2020-01-23T17:16:45.229626\n|\n--------------------------------------\n| Job a01e20e32f0965d9\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:16:43.188125\n| Create Time:\n|  2020-01-23T17:16:38.529718\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 22.22426700592041, "tool_id": "quast", "tool_version": "5.0.2+galaxy0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_quast.test_tool_000001"}, {"data": {"inputs": {"al|ambiguity_score": "0.99", "al|ambiguity_usage": "one", "al|fragmented": false, "al|fragmented_max_indent": "50", "al|min_alignment": "65", "al|min_identity": "95.0", "al|upper_bound_assembly": false, "al|upper_bound_min_con": "2", "al|use_all_alignments": false, "assembly|ref|origin": "none", "assembly|type": "metagenome", "circos": false, "contig_thresholds": "0,1000, 500", "extensive_mis_size": "1000", "genes|conserved_genes_finding": false, "genes|gene_finding|tool": "--mgm", "genes|rna_finding": false, "in|custom": "false", "in|inputs": {"id": "d8484e674200e2e2", "src": "hda"}, "k_mer|k_mer_stats": "", "large": false, "min_contig": "500", "scaffold_gap_max_size": "1000", "skip_unaligned_mis_contigs": false, "split_scaffolds": false, "strict_NA": false, "unaligned_part_size": "500"}, "job": {"command_line": "metaquast -o outputdir  --min-contig 500 --threads ${GALAXY_SLOTS:-1}  --labels contigs3_fasta    --mgm    --min-alignment 65 --min-identity 95.0 --ambiguity-usage \u0027one\u0027  --contig-thresholds \u00270,1000,500\u0027  --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs  \u0027/tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\u0027  \u0026\u0026 mkdir -p \u0027/tmp/tmpng6x1hvs/job_working_directory/000/138/working/dataset_29bd1f20-4a1e-48b6-8356-6ca24a2df86b_files\u0027 \u0026\u0026 cp outputdir/*.html \u0027/tmp/tmpng6x1hvs/job_working_directory/000/138/working/dataset_29bd1f20-4a1e-48b6-8356-6ca24a2df86b_files\u0027", "command_version": "", "create_time": "2020-01-23T17:17:07.562039", "exit_code": 1, "external_id": "22400", "galaxy_version": "20.01", "history_id": "25f486d5fc93a1ce", "id": "237cecdf0f984d5b", "inputs": {"inputs": {"id": "d8484e674200e2e2", "src": "hda", "uuid": "6a5d51b7-942c-4fa9-8ec6-678445f5050e"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"log": {"id": "3b3b4832b2563c85", "src": "hda", "uuid": "adba20d0-bb88-4a9d-b417-cbdfe61d3f2c"}, "quast_tabular": {"id": "90653ba0c3bbef25", "src": "hda", "uuid": "516d2df7-9ac6-4b71-9911-b4913d5e2278"}, "report_html": {"id": "6fccc2792c7809ba", "src": "hda", "uuid": "29bd1f20-4a1e-48b6-8356-6ca24a2df86b"}, "report_pdf": {"id": "06bb318fcaf8f1eb", "src": "hda", "uuid": "2879c119-00c9-4dad-955c-05f3ab21853a"}}, "params": {"__input_ext": "\"fasta\"", "al": "{\"ambiguity_score\": \"0.99\", \"ambiguity_usage\": \"one\", \"fragmented\": \"false\", \"fragmented_max_indent\": \"50\", \"min_alignment\": \"65\", \"min_identity\": \"95.0\", \"upper_bound_assembly\": \"false\", \"upper_bound_min_con\": \"2\", \"use_all_alignments\": \"false\"}", "assembly": "{\"__current_case__\": 1, \"ref\": {\"__current_case__\": 3, \"origin\": \"none\"}, \"type\": \"metagenome\"}", "chromInfo": "\"/tmp/tmpng6x1hvs/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "circos": "\"false\"", "contig_thresholds": "\"0,1000, 500\"", "dbkey": "\"?\"", "extensive_mis_size": "\"1000\"", "genes": "{\"conserved_genes_finding\": \"false\", \"gene_finding\": {\"__current_case__\": 2, \"tool\": \"--mgm\"}, \"rna_finding\": \"false\"}", "in": "{\"__current_case__\": 1, \"custom\": \"false\", \"inputs\": {\"values\": [{\"id\": 228, \"src\": \"hda\"}]}}", "k_mer": "{\"__current_case__\": 1, \"k_mer_stats\": \"\"}", "large": "\"false\"", "min_contig": "\"500\"", "scaffold_gap_max_size": "\"1000\"", "skip_unaligned_mis_contigs": "\"false\"", "split_scaffolds": "\"false\"", "strict_NA": "\"false\"", "unaligned_part_size": "\"500\""}, "state": "error", "stderr": "ERROR! exception caught!\n\nIn case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\nor report an issue on our GitHub repository https://github.com/ablab/quast/issues\nPlease provide us with quast.log file from the output directory.\n", "stdout": "/usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:17:08\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n\nContigs:\n  Pre-processing...\n  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n\nNo references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n\n2020-01-23 17:17:09\nNOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\nWARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\nReference genomes are not found.\n\nNOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n  WARNING: Python locale settings can\u0027t be changed\n/usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:17:09\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\nNOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n\nCWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\nMain parameters: \n  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n  threshold for extensive misassembly size: 1000\n\nContigs:\n  Pre-processing...\n  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n\n2020-01-23 17:17:09\nRunning Basic statistics processor...\n  Contig files: \n    contigs3_fasta\n\n\u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\nTraceback (most recent call last):\n  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n    return_code = main(sys.argv[1:])\n  File \"/usr/local/bin/metaquast\", line 202, in main\n    _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n    return_code = quast.main(args)\n  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n    icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n    html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n    append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n    init(html_fpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n    script_texts.append(js_html(aux_f_rel_path))\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n    return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n    return codecs.ascii_decode(input, self.errors)[0]\nUnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n", "tool_id": "quast", "tool_stderr": "ERROR! exception caught!\n\nIn case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\nor report an issue on our GitHub repository https://github.com/ablab/quast/issues\nPlease provide us with quast.log file from the output directory.\n", "tool_stdout": "/usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:17:08\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n\nContigs:\n  Pre-processing...\n  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n\nNo references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n\n2020-01-23 17:17:09\nNOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\nWARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\nReference genomes are not found.\n\nNOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n  WARNING: Python locale settings can\u0027t be changed\n/usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n\nVersion: 5.0.2\n\nSystem information:\n  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n  Python version: 3.5.5\n  CPUs number: 2\n\nStarted: 2020-01-23 17:17:09\n\nLogging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\nNOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n\nCWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\nMain parameters: \n  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n  threshold for extensive misassembly size: 1000\n\nContigs:\n  Pre-processing...\n  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n\n2020-01-23 17:17:09\nRunning Basic statistics processor...\n  Contig files: \n    contigs3_fasta\n\n\u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\nTraceback (most recent call last):\n  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n    return_code = main(sys.argv[1:])\n  File \"/usr/local/bin/metaquast\", line 202, in main\n    _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n    return_code = quast.main(args)\n  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n    icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n    html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n    append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n    init(html_fpath)\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n    script_texts.append(js_html(aux_f_rel_path))\n  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n    return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n    return codecs.ascii_decode(input, self.errors)[0]\nUnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n", "update_time": "2020-01-23T17:17:12.549231", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpng6x1hvs/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 25f486d5fc93a1ce - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs3.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  30 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NODE_1_length_78752_cov_2.35353\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGGTTGCCGGCAAGTCGGGCTAGTCGTTCCAGCCAGCGGTAGAAACCCGGCTCAAACTCT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGCTCTCGAAGGTACGGCTACCTGTATTCTTCTCAATGTATTCAACGGCTGTCTGATGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGCCATGATAATCTGGCGGCTCAATCCGTTGAAAAGACTCTGGTGGAATTCTCCCCAGAAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCGGCGAAACCTCAGGATGCATGTGCATTCCGATGATAATCTCAGAACATTGGAACTCG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - Quast on data 1: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 3 - Quast on data 1:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 4 - Quast on data 1:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 5 - Quast on data 1: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 237cecdf0f984d5b\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:17:12.549231\n| Create Time:\n|  2020-01-23T17:17:07.562039\n|\n--------------------------------------\n| Job d760e1d92679b730\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:17:06.249437\n| Create Time:\n|  2020-01-23T17:17:00.903993\n|\n--------------------------------------\nProblem in history with id 25f486d5fc93a1ce - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - contigs3.fasta (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  30 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;NODE_1_length_78752_cov_2.35353\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eAGGTTGCCGGCAAGTCGGGCTAGTCGTTCCAGCCAGCGGTAGAAACCCGGCTCAAACTCT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGCTCTCGAAGGTACGGCTACCTGTATTCTTCTCAATGTATTCAACGGCTGTCTGATGCGT\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGCCATGATAATCTGGCGGCTCAATCCGTTGAAAAGACTCTGGTGGAATTCTCCCCAGAAC\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTTCGGCGAAACCTCAGGATGCATGTGCATTCCGATGATAATCTCAGAACATTGGAACTCG\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - Quast on data 1: tabular report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 3 - Quast on data 1:  HTML report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 4 - Quast on data 1:  PDF report (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| 5 - Quast on data 1: Log (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  /usr/local/opt/quast-5.0.2/metaquast.py -o outputdir --min-contig 500 --threads 1 --labels contigs3_fasta --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:08\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/metaquast.log\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  No references are provided, starting to search for reference genomes in SILVA 16S rRNA database and to download them from NCBI...\n|  2020-01-23 17:17:09\n|  NOTICE: Permission denied accessing /usr/local/lib/python3.5/site-packages/quast_libs/silva. Silva will be downloaded to home directory /.quast\n|  WARNING: Permission denied accessing home directory /.quast. Silva cannot be downloaded.\n|  Reference genomes are not found.\n|  NOTICE: No references are provided, starting regular QUAST with MetaGeneMark gene finder\n|  WARNING: Python locale settings can\u0027t be changed\n|  /usr/local/opt/quast-5.0.2/quast.py --min-contig 500 --threads 1 --mgm --min-alignment 65 --min-identity 95.0 --ambiguity-usage one --contig-thresholds 0,1000,500 --extensive-mis-size 1000 --scaffold-gap-max-size 1000 --unaligned-part-size 500 --skip-unaligned-mis-contigs --mgm /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat -o /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir --labels contigs3_fasta\n|  Version: 5.0.2\n|  System information:\n|  OS: Linux-5.0.0-1028-azure-x86_64-with-glibc2.2.5 (linux_64)\n|  Python version: 3.5.5\n|  CPUs number: 2\n|  Started: 2020-01-23 17:17:09\n|  Logging to /tmp/tmpng6x1hvs/job_working_directory/000/138/working/outputdir/quast.log\n|  NOTICE: Output directory already exists and looks like a QUAST output dir. Existing results can be reused (e.g. previously generated alignments)!\n|  CWD: /tmp/tmpng6x1hvs/job_working_directory/000/138/working\n|  Main parameters:\n|  MODE: meta, threads: 1, minimum contig length: 500, minimum alignment length: 65, ambiguity: one, \\\n|  threshold for extensive misassembly size: 1000\n|  Contigs:\n|  Pre-processing...\n|  /tmp/tmpng6x1hvs/files/6/a/5/dataset_6a5d51b7-942c-4fa9-8ec6-678445f5050e.dat ==\u003e contigs3_fasta\n|  2020-01-23 17:17:09\n|  Running Basic statistics processor...\n|  Contig files:\n|  contigs3_fasta\n|  \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/metaquast\", line 406, in \u003cmodule\u003e\n|  return_code = main(sys.argv[1:])\n|  File \"/usr/local/bin/metaquast\", line 202, in main\n|  _start_quast_main(quast_py_args, assemblies=assemblies, output_dirpath=output_dirpath, run_regular_quast=True)\n|  File \"/usr/local/bin/metaquast\", line 72, in _start_quast_main\n|  return_code = quast.main(args)\n|  File \"/usr/local/opt/quast-5.0.2/quast.py\", line 149, in main\n|  icarus_gc_fpath, circos_gc_fpath = basic_stats.do(ref_fpath, contigs_fpaths, os.path.join(output_dirpath, \u0027basic_stats\u0027), output_dirpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/basic_stats.py\", line 263, in do\n|  html_saver.save_contigs_lengths(results_dir, contigs_fpaths, corr_lists_of_lengths)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 475, in save_contigs_lengths\n|  append(results_dirpath, json_fpath, \u0027contigsLengths\u0027)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 220, in append\n|  init(html_fpath)\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 115, in init\n|  script_texts.append(js_html(aux_f_rel_path))\n|  File \"/usr/local/opt/quast-5.0.2/quast_libs/html_saver/html_saver.py\", line 95, in js_html\n|  return \u0027\u003cscript type=\"text/javascript\"\u003e\\n\u0027 + open(get_real_path(script_rel_path)).read() + \u0027\\n\u003c/script\u003e\\n\u0027\n|  File \"/usr/local/lib/python3.5/encodings/ascii.py\", line 26, in decode\n|  return codecs.ascii_decode(input, self.errors)[0]\n|  UnicodeDecodeError: \u0027ascii\u0027 codec can\u0027t decode byte 0xef in position 14588: ordinal not in range(128)\n| Dataset Job Standard Error:\n|  ERROR! exception caught!\n|  In case you have troubles running QUAST, you can write to quast.support@cab.spbu.ru\n|  or report an issue on our GitHub repository https://github.com/ablab/quast/issues\n|  Please provide us with quast.log file from the output directory.\n|\n--------------------------------------\n| Job 237cecdf0f984d5b\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:17:12.549231\n| Create Time:\n|  2020-01-23T17:17:07.562039\n|\n--------------------------------------\n| Job d760e1d92679b730\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:17:06.249437\n| Create Time:\n|  2020-01-23T17:17:00.903993\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 2, "time_seconds": 14.29137372970581, "tool_id": "quast", "tool_version": "5.0.2+galaxy0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_quast.test_tool_000002"}, {"data": {"inputs": {"inputrds": {"id": "0e8c18395d13c78f", "src": "hda"}}, "job": {"command_line": "Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/raceid/scripts/cluster.R\u0027 \u0027/tmp/tmpha48x28m/job_working_directory/000/168/configs/tmpajzokmbv\u0027 \u0026\u003e \u0027/tmp/tmpha48x28m/files/f/1/0/dataset_f100cf5d-18b0-4f65-bd0f-88dda93f2c5e.dat\u0027", "command_version": "0.5\n", "create_time": "2020-01-23T17:22:58.696094", "exit_code": 0, "external_id": "36155", "galaxy_version": "20.01", "history_id": "0e797101c95aea67", "id": "c011a4b8aa5812a2", "inputs": {"inputrds": {"id": "0e8c18395d13c78f", "src": "hda", "uuid": "7a25e5c6-72ea-47eb-97cb-72b795962fd5"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outassignments": {"id": "7e438c048dc73282", "src": "hda", "uuid": "66755080-0d7d-4fbd-83c1-2ffa7033365a"}, "outgenelist": {"id": "7cd46ccec257d13d", "src": "hda", "uuid": "c0bab203-9059-4594-ba4f-b3bf6af6a8a6"}, "outlog": {"id": "5e8952dd122908bb", "src": "hda", "uuid": "f100cf5d-18b0-4f65-bd0f-88dda93f2c5e"}, "outpdf": {"id": "6776b4c51106726f", "src": "hda", "uuid": "854759f6-6ae3-41ab-8293-f4f4266a6711"}, "outrdat": {"id": "fbb974970ddf8fa9", "src": "hda", "uuid": "ca2f4de4-5d0d-474d-98c4-711f3cf431dd"}}, "params": {"__input_ext": "\"input\"", "chromInfo": "\"/tmp/tmpha48x28m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "clust": "{\"funcluster\": \"kmedoids\", \"metric\": \"pearson\", \"use\": {\"__current_case__\": 0, \"def\": \"yes\"}}", "dbkey": "\"?\"", "extra": "{\"foldchange\": \"1.0\", \"plotlim\": \"10\", \"pvalue\": \"0.01\", \"tablelim\": \"25\"}", "outlier": "{\"final\": \"false\", \"outlg\": \"2\", \"outminc\": \"5\", \"use\": {\"__current_case__\": 0, \"def\": \"yes\"}}", "tsne": "{\"knn\": \"10\", \"perplexity\": \"30\", \"use\": {\"__current_case__\": 0, \"def\": \"yes\"}}"}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "raceid_clustering", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T17:29:01.627876", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpt2hkur07matrix2.genelist v. /tmp/tmp2se_g5komatrix2.genelist )\n--- local_file\n+++ history_data\n@@ -1,201 +1,201 @@\n n\tmean.ncl\tmean.cl\tfc\tpv\tpadj\n-ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002094e-20\t3.43529216795677e-16\n-ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248461e-19\t7.56741291835933e-16\n-ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997524e-19\t8.37309204362968e-16\n-ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051255e-18\t5.43349850280376e-15\n+ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002071e-20\t3.43529216795663e-16\n+ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248451e-19\t7.56741291835904e-16\n+ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997506e-19\t8.37309204362931e-16\n+ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051225e-18\t5.4334985028033e-15\n ENSDARG00000107321\t1\t0.1\t1.94683879586459\t19.4683879586459\t1.39905888784082e-09\t1.7060124078331e-06\n-ENSDARG00000105564\t1\t0.1\t1.72137476189456\t17.2137476189456\t5.11677986670931e-09\t5.19950114122111e-06\n-ENSDARG00000090447\t1\t0.1\t1.4302236712519\t14.302236712519\t3.5162074291291e-08\t3.06261667077144e-05\n-ENSDARG00000079591\t1\t0.11578454541401\t1.47825950354764\t12.767330028907\t4.83701028926678e-08\t3.68640646670745e-05\n-ENSDARG00000107138\t1\t0.118643244515052\t1.63372529702772\t13.7700659123533\t8.59237265783772e-08\t5.36793599520253e-05\n-ENSDARG00000041060\t1\t0.11578454541401\t1.4382199783801\t12.421519411226\t8.84781911763513e-08\t5.36793599520253e-05\n-ENSDARG00000077004\t1\t0.1\t1.29299827494356\t12.9299827494356\t1.03643477990381e-07\t5.36793599520253e-05\n-ENSDARG00000100003\t1\t0.1\t1.29775216728225\t12.9775216728225\t1.05650700250009e-07\t5.36793599520253e-05\n-ENSDARG00000102310\t1\t0.129577145917004\t1.72085266498654\t13.2805260743185\t1.20446928962966e-07\t5.64896096836309e-05\n-ENSDARG00000068650\t1\t0.115601615820105\t1.27001296748153\t10.9861177845289\t2.26895176903168e-07\t9.88128495413296e-05\n-ENSDARG00000086678\t1\t0.128030253978187\t1.35647794130946\t10.5949796955067\t3.169107566406e-07\t0.000122948041506918\n-ENSDARG00000105857\t1\t0.105900327795989\t1.37647789967886\t12.9978625026598\t3.22645344285827e-07\t0.000122948041506918\n-ENSDARG00000105159\t1\t0.1\t1.16700756453584\t11.6700756453584\t3.44329675018855e-07\t0.000123492825211174\n********\n*SNIP *\n********\n-ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476857001\n-ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.0015819768462871\t0.618830476857001\n-ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505542\t0.618830476857001\n-ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094625\t0.618830476857001\n-ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760145\t0.618830476857001\n-ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255128\t0.678671506770645\n-ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064379\t0.678671506770645\n+ENSDARG00000026180\t8\t0.151758427938041\t1.41564289946672\t9.32826544595394\t0.000683459166751406\t0.347254211640277\n+ENSDARG00000104786\t8\t0.182483274181232\t1.61787042996763\t8.8658559927024\t0.000787624115882892\t0.369395710349077\n+ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476856999\n+ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.00158197684628707\t0.618830476856999\n+ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505541\t0.618830476856999\n+ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094622\t0.618830476856999\n+ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760144\t0.618830476856999\n+ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255127\t0.678671506770645\n+ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064377\t0.678671506770645\n ENSDARG00000078352\t8\t0.177346649454539\t1.15788746142844\t6.52895030715114\t0.00248762071413556\t0.678671506770645\n-ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994521\t0.678671506770645\n-ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475838\t0.678671506770645\n-ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697548\t0.678671506770645\n-ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336178\t0.678671506770645\n+ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994522\t0.678671506770645\n+ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475831\t0.678671506770645\n+ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697552\t0.678671506770645\n+ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336179\t0.678671506770645\n"], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nHistory item  different than expected, difference (using diff):\n( /tmp/tmpt2hkur07matrix2.genelist v. /tmp/tmp2se_g5komatrix2.genelist )\n--- local_file\n+++ history_data\n@@ -1,201 +1,201 @@\n n\tmean.ncl\tmean.cl\tfc\tpv\tpadj\n-ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002094e-20\t3.43529216795677e-16\n-ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248461e-19\t7.56741291835933e-16\n-ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997524e-19\t8.37309204362968e-16\n-ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051255e-18\t5.43349850280376e-15\n+ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002071e-20\t3.43529216795663e-16\n+ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248451e-19\t7.56741291835904e-16\n+ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997506e-19\t8.37309204362931e-16\n+ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051225e-18\t5.4334985028033e-15\n ENSDARG00000107321\t1\t0.1\t1.94683879586459\t19.4683879586459\t1.39905888784082e-09\t1.7060124078331e-06\n-ENSDARG00000105564\t1\t0.1\t1.72137476189456\t17.2137476189456\t5.11677986670931e-09\t5.19950114122111e-06\n-ENSDARG00000090447\t1\t0.1\t1.4302236712519\t14.302236712519\t3.5162074291291e-08\t3.06261667077144e-05\n-ENSDARG00000079591\t1\t0.11578454541401\t1.47825950354764\t12.767330028907\t4.83701028926678e-08\t3.68640646670745e-05\n-ENSDARG00000107138\t1\t0.118643244515052\t1.63372529702772\t13.7700659123533\t8.59237265783772e-08\t5.36793599520253e-05\n-ENSDARG00000041060\t1\t0.11578454541401\t1.4382199783801\t12.421519411226\t8.84781911763513e-08\t5.36793599520253e-05\n-ENSDARG00000077004\t1\t0.1\t1.29299827494356\t12.9299827494356\t1.03643477990381e-07\t5.36793599520253e-05\n-ENSDARG00000100003\t1\t0.1\t1.29775216728225\t12.9775216728225\t1.05650700250009e-07\t5.36793599520253e-05\n-ENSDARG00000102310\t1\t0.129577145917004\t1.72085266498654\t13.2805260743185\t1.20446928962966e-07\t5.64896096836309e-05\n-ENSDARG00000068650\t1\t0.115601615820105\t1.27001296748153\t10.9861177845289\t2.26895176903168e-07\t9.88128495413296e-05\n-ENSDARG00000086678\t1\t0.128030253978187\t1.35647794130946\t10.5949796955067\t3.169107566406e-07\t0.000122948041506918\n-ENSDARG00000105857\t1\t0.105900327795989\t1.37647789967886\t12.9978625026598\t3.22645344285827e-07\t0.000122948041506918\n-ENSDARG00000105159\t1\t0.1\t1.16700756453584\t11.6700756453584\t3.44329675018855e-07\t0.000123492825211174\n********\n*SNIP *\n********\n-ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476857001\n-ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.0015819768462871\t0.618830476857001\n-ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505542\t0.618830476857001\n-ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094625\t0.618830476857001\n-ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760145\t0.618830476857001\n-ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255128\t0.678671506770645\n-ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064379\t0.678671506770645\n+ENSDARG00000026180\t8\t0.151758427938041\t1.41564289946672\t9.32826544595394\t0.000683459166751406\t0.347254211640277\n+ENSDARG00000104786\t8\t0.182483274181232\t1.61787042996763\t8.8658559927024\t0.000787624115882892\t0.369395710349077\n+ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476856999\n+ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.00158197684628707\t0.618830476856999\n+ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505541\t0.618830476856999\n+ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094622\t0.618830476856999\n+ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760144\t0.618830476856999\n+ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255127\t0.678671506770645\n+ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064377\t0.678671506770645\n ENSDARG00000078352\t8\t0.177346649454539\t1.15788746142844\t6.52895030715114\t0.00248762071413556\t0.678671506770645\n-ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994521\t0.678671506770645\n-ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475838\t0.678671506770645\n-ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697548\t0.678671506770645\n-ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336178\t0.678671506770645\n+ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994522\t0.678671506770645\n+ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475831\t0.678671506770645\n+ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697552\t0.678671506770645\n+ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336179\t0.678671506770645\n\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027r-raceid=0.1.3,bioconductor-scran=1.10.1\u0027 -set REPO=\u0027quay.io/local/mulled-v2-ea4f5d4e734403fd876c56467cab8b1011d95d75:d531b53fe65484e335601875c966b614f8aecac5-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bioconda/extended-base-image:latest\u0027 -set TEST=true build-and-test\n....\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 417.9059154987335, "tool_id": "raceid_clustering", "tool_version": "3.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_raceid_clustering.test_tool_000000"}, {"data": {"inputs": {"clust|funcluster": "kmedoids", "clust|metric": "pearson", "clust|use|bootnr": "50", "clust|use|clustnr": "30", "clust|use|def": "no", "clust|use|fselect": true, "clust|use|rseed": "17000", "clust|use|sat": true, "inputrds": {"id": "daf78d18ceb29125", "src": "hda"}, "outlier|final": false, "outlier|outlg": "2", "outlier|outminc": "5", "outlier|use|def": "no", "outlier|use|nbfactor": "5", "outlier|use|outdistquant": "0.95", "outlier|use|probthr": "0.001", "outlier|use|rfseed": "12345", "tsne|knn": "10", "tsne|perplexity": "30", "tsne|use|def": "no", "tsne|use|initial_cmd": true, "tsne|use|rseed_tsne": "15555"}, "job": {"command_line": "Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/raceid/scripts/cluster.R\u0027 \u0027/tmp/tmpha48x28m/job_working_directory/000/170/configs/tmp8zralna4\u0027 \u0026\u003e \u0027/tmp/tmpha48x28m/files/a/3/f/dataset_a3f091e1-cd1e-4f89-8f83-8ceb41cabb46.dat\u0027", "command_version": "0.5\n", "create_time": "2020-01-23T17:29:56.373965", "exit_code": 0, "external_id": "36636", "galaxy_version": "20.01", "history_id": "baa1080356b320b7", "id": "e78965a26b62ba53", "inputs": {"inputrds": {"id": "daf78d18ceb29125", "src": "hda", "uuid": "b718da8a-6a32-4ef8-8924-15a55b2ff122"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outassignments": {"id": "5d952db0f0e41e99", "src": "hda", "uuid": "6dd87c19-a26e-476e-8d08-ba62a2666216"}, "outgenelist": {"id": "a419613b7d66b4dd", "src": "hda", "uuid": "8cf69c26-4495-4d85-a590-d009b47d43c6"}, "outlog": {"id": "24c5f63829c78a60", "src": "hda", "uuid": "a3f091e1-cd1e-4f89-8f83-8ceb41cabb46"}, "outpdf": {"id": "1d5ca124ec943537", "src": "hda", "uuid": "42fb98ef-0a20-45e4-aed7-ec244f8a82ac"}, "outrdat": {"id": "a658f0c7b599f85a", "src": "hda", "uuid": "d9570122-4ccb-4726-987c-08a630f5429a"}}, "params": {"__input_ext": "\"input\"", "chromInfo": "\"/tmp/tmpha48x28m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "clust": "{\"funcluster\": \"kmedoids\", \"metric\": \"pearson\", \"use\": {\"__current_case__\": 1, \"bootnr\": \"50\", \"cln\": \"\", \"clustnr\": \"30\", \"def\": \"no\", \"fselect\": \"true\", \"knn\": \"\", \"rseed\": \"17000\", \"samp\": \"\", \"sat\": \"true\"}}", "dbkey": "\"?\"", "extra": "{\"foldchange\": \"1.0\", \"plotlim\": \"10\", \"pvalue\": \"0.01\", \"tablelim\": \"25\"}", "outlier": "{\"final\": \"false\", \"outlg\": \"2\", \"outminc\": \"5\", \"use\": {\"__current_case__\": 1, \"def\": \"no\", \"nbfactor\": \"5\", \"nbtree\": \"\", \"outdistquant\": \"0.95\", \"probthr\": \"0.001\", \"rfseed\": \"12345\"}}", "tsne": "{\"knn\": \"10\", \"perplexity\": \"30\", \"use\": {\"__current_case__\": 1, \"def\": \"no\", \"initial_cmd\": \"true\", \"rseed_fr\": \"15555\", \"rseed_tsne\": \"15555\"}}"}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "raceid_clustering", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T17:31:11.852490", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmp7syaml_eintestinal.genelist v. /tmp/tmp6qey7yw4intestinal.genelist )\n--- local_file\n+++ history_data\n@@ -1,201 +1,201 @@\n n\tmean.ncl\tmean.cl\tfc\tpv\tpadj\n-ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002094e-20\t3.43529216795677e-16\n-ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248461e-19\t7.56741291835933e-16\n-ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997524e-19\t8.37309204362968e-16\n-ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051255e-18\t5.43349850280376e-15\n+ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002071e-20\t3.43529216795663e-16\n+ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248451e-19\t7.56741291835904e-16\n+ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997506e-19\t8.37309204362931e-16\n+ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051225e-18\t5.4334985028033e-15\n ENSDARG00000107321\t1\t0.1\t1.94683879586459\t19.4683879586459\t1.39905888784082e-09\t1.7060124078331e-06\n-ENSDARG00000105564\t1\t0.1\t1.72137476189456\t17.2137476189456\t5.11677986670931e-09\t5.19950114122111e-06\n-ENSDARG00000090447\t1\t0.1\t1.4302236712519\t14.302236712519\t3.5162074291291e-08\t3.06261667077144e-05\n-ENSDARG00000079591\t1\t0.11578454541401\t1.47825950354764\t12.767330028907\t4.83701028926678e-08\t3.68640646670745e-05\n-ENSDARG00000107138\t1\t0.118643244515052\t1.63372529702772\t13.7700659123533\t8.59237265783772e-08\t5.36793599520253e-05\n-ENSDARG00000041060\t1\t0.11578454541401\t1.4382199783801\t12.421519411226\t8.84781911763513e-08\t5.36793599520253e-05\n-ENSDARG00000077004\t1\t0.1\t1.29299827494356\t12.9299827494356\t1.03643477990381e-07\t5.36793599520253e-05\n-ENSDARG00000100003\t1\t0.1\t1.29775216728225\t12.9775216728225\t1.05650700250009e-07\t5.36793599520253e-05\n-ENSDARG00000102310\t1\t0.129577145917004\t1.72085266498654\t13.2805260743185\t1.20446928962966e-07\t5.64896096836309e-05\n-ENSDARG00000068650\t1\t0.115601615820105\t1.27001296748153\t10.9861177845289\t2.26895176903168e-07\t9.88128495413296e-05\n-ENSDARG00000086678\t1\t0.128030253978187\t1.35647794130946\t10.5949796955067\t3.169107566406e-07\t0.000122948041506918\n-ENSDARG00000105857\t1\t0.105900327795989\t1.37647789967886\t12.9978625026598\t3.22645344285827e-07\t0.000122948041506918\n-ENSDARG00000105159\t1\t0.1\t1.16700756453584\t11.6700756453584\t3.44329675018855e-07\t0.000123492825211174\n********\n*SNIP *\n********\n-ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476857001\n-ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.0015819768462871\t0.618830476857001\n-ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505542\t0.618830476857001\n-ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094625\t0.618830476857001\n-ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760145\t0.618830476857001\n-ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255128\t0.678671506770645\n-ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064379\t0.678671506770645\n+ENSDARG00000026180\t8\t0.151758427938041\t1.41564289946672\t9.32826544595394\t0.000683459166751406\t0.347254211640277\n+ENSDARG00000104786\t8\t0.182483274181232\t1.61787042996763\t8.8658559927024\t0.000787624115882892\t0.369395710349077\n+ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476856999\n+ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.00158197684628707\t0.618830476856999\n+ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505541\t0.618830476856999\n+ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094622\t0.618830476856999\n+ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760144\t0.618830476856999\n+ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255127\t0.678671506770645\n+ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064377\t0.678671506770645\n ENSDARG00000078352\t8\t0.177346649454539\t1.15788746142844\t6.52895030715114\t0.00248762071413556\t0.678671506770645\n-ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994521\t0.678671506770645\n-ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475838\t0.678671506770645\n-ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697548\t0.678671506770645\n-ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336178\t0.678671506770645\n+ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994522\t0.678671506770645\n+ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475831\t0.678671506770645\n+ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697552\t0.678671506770645\n+ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336179\t0.678671506770645\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmp7syaml_eintestinal.genelist v. /tmp/tmp6qey7yw4intestinal.genelist )\n--- local_file\n+++ history_data\n@@ -1,201 +1,201 @@\n n\tmean.ncl\tmean.cl\tfc\tpv\tpadj\n-ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002094e-20\t3.43529216795677e-16\n-ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248461e-19\t7.56741291835933e-16\n-ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997524e-19\t8.37309204362968e-16\n-ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051255e-18\t5.43349850280376e-15\n+ENSDARG00000108798\t1\t0.13225291542414\t22.6067842060994\t170.935998904815\t5.63439752002071e-20\t3.43529216795663e-16\n+ENSDARG00000109030\t1\t0.186711263967539\t34.6842724288799\t185.764220603798\t2.48233981248451e-19\t7.56741291835904e-16\n+ENSDARG00000108379\t1\t0.168199368419563\t24.3874482842581\t144.991319012715\t4.11994031997506e-19\t8.37309204362931e-16\n+ENSDARG00000107599\t1\t0.194980763882093\t20.9729826342942\t107.564367975175\t3.56470297051225e-18\t5.4334985028033e-15\n ENSDARG00000107321\t1\t0.1\t1.94683879586459\t19.4683879586459\t1.39905888784082e-09\t1.7060124078331e-06\n-ENSDARG00000105564\t1\t0.1\t1.72137476189456\t17.2137476189456\t5.11677986670931e-09\t5.19950114122111e-06\n-ENSDARG00000090447\t1\t0.1\t1.4302236712519\t14.302236712519\t3.5162074291291e-08\t3.06261667077144e-05\n-ENSDARG00000079591\t1\t0.11578454541401\t1.47825950354764\t12.767330028907\t4.83701028926678e-08\t3.68640646670745e-05\n-ENSDARG00000107138\t1\t0.118643244515052\t1.63372529702772\t13.7700659123533\t8.59237265783772e-08\t5.36793599520253e-05\n-ENSDARG00000041060\t1\t0.11578454541401\t1.4382199783801\t12.421519411226\t8.84781911763513e-08\t5.36793599520253e-05\n-ENSDARG00000077004\t1\t0.1\t1.29299827494356\t12.9299827494356\t1.03643477990381e-07\t5.36793599520253e-05\n-ENSDARG00000100003\t1\t0.1\t1.29775216728225\t12.9775216728225\t1.05650700250009e-07\t5.36793599520253e-05\n-ENSDARG00000102310\t1\t0.129577145917004\t1.72085266498654\t13.2805260743185\t1.20446928962966e-07\t5.64896096836309e-05\n-ENSDARG00000068650\t1\t0.115601615820105\t1.27001296748153\t10.9861177845289\t2.26895176903168e-07\t9.88128495413296e-05\n-ENSDARG00000086678\t1\t0.128030253978187\t1.35647794130946\t10.5949796955067\t3.169107566406e-07\t0.000122948041506918\n-ENSDARG00000105857\t1\t0.105900327795989\t1.37647789967886\t12.9978625026598\t3.22645344285827e-07\t0.000122948041506918\n-ENSDARG00000105159\t1\t0.1\t1.16700756453584\t11.6700756453584\t3.44329675018855e-07\t0.000123492825211174\n********\n*SNIP *\n********\n-ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476857001\n-ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.0015819768462871\t0.618830476857001\n-ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505542\t0.618830476857001\n-ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094625\t0.618830476857001\n-ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760145\t0.618830476857001\n-ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255128\t0.678671506770645\n-ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064379\t0.678671506770645\n+ENSDARG00000026180\t8\t0.151758427938041\t1.41564289946672\t9.32826544595394\t0.000683459166751406\t0.347254211640277\n+ENSDARG00000104786\t8\t0.182483274181232\t1.61787042996763\t8.8658559927024\t0.000787624115882892\t0.369395710349077\n+ENSDARG00000101061\t8\t0.255483815147335\t1.62613690189592\t6.36493118344166\t0.00150635451453945\t0.618830476856999\n+ENSDARG00000016584\t8\t0.135272462199637\t0.880366460381022\t6.50809814551733\t0.00158197684628707\t0.618830476856999\n+ENSDARG00000070256\t8\t0.203213690445973\t1.26976802854965\t6.24843742448165\t0.00178015471505541\t0.618830476856999\n+ENSDARG00000062745\t8\t0.130428963634169\t0.955091918456498\t7.32269805604966\t0.00182445112094622\t0.618830476856999\n+ENSDARG00000086037\t8\t0.19632007891506\t1.29918699186992\t6.61769799120764\t0.00182695564760144\t0.618830476856999\n+ENSDARG00000027310\t8\t0.193675103177481\t1.44901164174846\t7.48166190685131\t0.00211840042255127\t0.678671506770645\n+ENSDARG00000103109\t8\t0.14293938597264\t0.936026560054929\t6.54841598545899\t0.00240745117064377\t0.678671506770645\n ENSDARG00000078352\t8\t0.177346649454539\t1.15788746142844\t6.52895030715114\t0.00248762071413556\t0.678671506770645\n-ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994521\t0.678671506770645\n-ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475838\t0.678671506770645\n-ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697548\t0.678671506770645\n-ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336178\t0.678671506770645\n+ENSDARG00000035389\t8\t0.143574445102194\t1.00421593036345\t6.99439186164828\t0.00269958693994522\t0.678671506770645\n+ENSDARG00000103106\t8\t0.175526172559138\t1.23041862190788\t7.00988692437495\t0.00270171719475831\t0.678671506770645\n+ENSDARG00000007257\t8\t0.157933075856061\t0.866671316548008\t5.48758587680447\t0.00283846391697552\t0.678671506770645\n+ENSDARG00000010596\t8\t0.110516369140454\t0.836156159426777\t7.56590327686312\t0.00297745919336179\t0.678671506770645\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 109.92123985290527, "tool_id": "raceid_clustering", "tool_version": "3.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_raceid_clustering.test_tool_000001"}, {"data": {"inputs": {"clust|funcluster": "hclust", "clust|metric": "euclidean", "clust|use|bootnr": "30", "clust|use|cln": "10", "clust|use|clustnr": "10", "clust|use|def": "no", "clust|use|fselect": false, "clust|use|knn": "5", "clust|use|rseed": "17000", "clust|use|samp": "10", "clust|use|sat": false, "inputrds": {"id": "09790f3d9766ed9a", "src": "hda"}, "outlier|final": true, "outlier|outlg": "5", "outlier|outminc": "3", "outlier|use|def": "no", "outlier|use|nbfactor": "5", "outlier|use|nbtree": "10", "outlier|use|outdistquant": "0.5", "outlier|use|probthr": "0.01", "outlier|use|rfseed": "12345", "tsne|knn": "6", "tsne|perplexity": "20", "tsne|use|def": "no", "tsne|use|initial_cmd": false, "tsne|use|rseed_tsne": "15555"}, "job": {"command_line": "Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/raceid/scripts/cluster.R\u0027 \u0027/tmp/tmpha48x28m/job_working_directory/000/172/configs/tmp6l109wb6\u0027 \u0026\u003e \u0027/tmp/tmpha48x28m/files/7/4/9/dataset_749b8593-d573-4ce2-8c1d-8a6b2c29c17b.dat\u0027", "command_version": "0.5\n", "create_time": "2020-01-23T17:31:46.262347", "exit_code": 0, "external_id": "37096", "galaxy_version": "20.01", "history_id": "48f3e8809447c25a", "id": "6fbaeb4019cd0c5e", "inputs": {"inputrds": {"id": "09790f3d9766ed9a", "src": "hda", "uuid": "ed1885f9-772e-4652-88b2-2110c23bfe51"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outassignments": {"id": "e038b12fcc34712c", "src": "hda", "uuid": "d3c21365-f8d9-4294-98c3-6d92eb1ebf34"}, "outgenelist": {"id": "648ee5580acb394c", "src": "hda", "uuid": "3f16a151-af71-4bc9-82ad-e2631efb6259"}, "outlog": {"id": "57d21b82f66bdde1", "src": "hda", "uuid": "749b8593-d573-4ce2-8c1d-8a6b2c29c17b"}, "outpdf": {"id": "cda64434023ac031", "src": "hda", "uuid": "4ea4f7e3-c727-41f0-b53d-7065585d852e"}, "outrdat": {"id": "846dc31143592f08", "src": "hda", "uuid": "8c118d8e-35c2-42d3-a19f-a5350251e9fb"}}, "params": {"__input_ext": "\"input\"", "chromInfo": "\"/tmp/tmpha48x28m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "clust": "{\"funcluster\": \"hclust\", \"metric\": \"euclidean\", \"use\": {\"__current_case__\": 1, \"bootnr\": \"30\", \"cln\": \"10\", \"clustnr\": \"10\", \"def\": \"no\", \"fselect\": \"false\", \"knn\": \"5\", \"rseed\": \"17000\", \"samp\": \"10\", \"sat\": \"false\"}}", "dbkey": "\"?\"", "extra": "{\"foldchange\": \"1.0\", \"plotlim\": \"10\", \"pvalue\": \"0.01\", \"tablelim\": \"25\"}", "outlier": "{\"final\": \"true\", \"outlg\": \"5\", \"outminc\": \"3\", \"use\": {\"__current_case__\": 1, \"def\": \"no\", \"nbfactor\": \"5\", \"nbtree\": \"10\", \"outdistquant\": \"0.5\", \"probthr\": \"0.01\", \"rfseed\": \"12345\"}}", "tsne": "{\"knn\": \"6\", \"perplexity\": \"20\", \"use\": {\"__current_case__\": 1, \"def\": \"no\", \"initial_cmd\": \"false\", \"rseed_fr\": \"15555\", \"rseed_tsne\": \"15555\"}}"}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "raceid_clustering", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T17:35:17.014955", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpjkj3uuzsintestinal_advanced.genelist v. /tmp/tmp2cg_n_6aintestinal_advanced.genelist )\n--- local_file\n+++ history_data\n@@ -1,251 +1,251 @@\n n\tmean.ncl\tmean.cl\tfc\tpv\tpadj\n-ENSDARG00000109030\t1\t0.274654337828615\t39.4233419769949\t143.538027794031\t1.85453080866723e-17\t1.13070743404441e-13\n-ENSDARG00000108798\t1\t0.327573035068885\t24.802335939268\t75.7154383420246\t1.23506199101095e-15\t3.76508647959687e-12\n-ENSDARG00000107599\t1\t0.564182966938587\t21.783984071531\t38.6115592779005\t1.4324116853842e-12\t2.91113801526248e-09\n-ENSDARG00000108379\t1\t0.68986839145523\t24.7427021224644\t35.8658295247755\t7.5942061344251e-12\t1.15754687003975e-08\n-ENSDARG00000107321\t1\t0.1\t2.23096784138222\t22.3096784138222\t6.33189194610735e-10\t7.7211090390833e-07\n-ENSDARG00000105564\t1\t0.1\t1.97081703295526\t19.7081703295526\t2.35709418427634e-09\t2.39520054025547e-06\n-ENSDARG00000107138\t1\t0.12141391587535\t1.84891204400927\t15.22817241071\t3.76162108069493e-08\t3.27637196128528e-05\n+ENSDARG00000109030\t1\t0.274654337828615\t39.4233419769949\t143.538027794031\t1.85453080866709e-17\t1.13070743404433e-13\n+ENSDARG00000108798\t1\t0.327573035068885\t24.802335939268\t75.7154383420246\t1.23506199101086e-15\t3.7650864795966e-12\n+ENSDARG00000107599\t1\t0.564182966938587\t21.783984071531\t38.6115592779005\t1.43241168538408e-12\t2.91113801526224e-09\n+ENSDARG00000108379\t1\t0.68986839145523\t24.7427021224644\t35.8658295247755\t7.59420613442469e-12\t1.15754687003968e-08\n+ENSDARG00000107321\t1\t0.1\t2.23096784138222\t22.3096784138222\t6.33189194610729e-10\t7.72110903908323e-07\n+ENSDARG00000105564\t1\t0.1\t1.97081703295526\t19.7081703295526\t2.35709418427633e-09\t2.39520054025546e-06\n+ENSDARG00000107138\t1\t0.12141391587535\t1.84891204400927\t15.22817241071\t3.76162108069488e-08\t3.27637196128524e-05\n ENSDARG00000105857\t1\t0.105759843800846\t1.57285911501407\t14.8719878782715\t1.52193964569661e-07\t0.000115990825247653\n-ENSDARG00000090447\t1\t0.117044445524061\t1.5247401265198\t13.0270182381816\t2.86307072797427e-07\t0.000193957135871768\n-ENSDARG00000068650\t1\t0.118444706911236\t1.42924397145598\t12.0667610121833\t4.76963874982153e-07\t0.000290804874576619\n-ENSDARG00000029234\t1\t0.107704361452076\t1.2680785099502\t11.7736969316181\t7.26433141517907e-07\t0.000402642078530425\n-ENSDARG00000100003\t1\t0.120259003658528\t1.3511174001475\t11.2350623158659\t8.51760668849127e-07\t0.000432765399831094\n-ENSDARG00000105009\t1\t0.114133477721009\t1.43604387395694\t12.5821441931984\t1.07513161417503e-06\t0.00046841229720652\n-ENSDARG00000104049\t1\t0.14589886302348\t1.5016492143503\t10.2923983314978\t1.07557358715619e-06\t0.00046841229720652\n********\n*SNIP *\n********\n+ENSDARG00000101413\t10\t0.141224429692467\t1.80827014074558\t12.8042304343753\t1.53774069797471e-07\t0.00093756050355518\n+ENSDARG00000069269\t10\t0.136680427340329\t1.39111418584467\t10.177859499816\t7.94091542244339e-07\t0.00242078806653187\n+ENSDARG00000096457\t10\t0.108692378208486\t0.98849791094733\t9.09445471007417\t8.01600268921723e-06\t0.0136362534840028\n+ENSDARG00000058178\t10\t0.135168786411076\t1.1479669345012\t8.49284043292363\t8.94620533639676e-06\t0.0136362534840028\n+ENSDARG000001048361\t10\t2.19609208044308\t17.0930420303104\t7.78339040631744\t2.81321418960986e-05\t0.0308316937773497\n+ENSDARG00000093313\t10\t0.126396920024548\t0.901779535343549\t7.13450561270331\t3.03411780652941e-05\t0.0308316937773497\n+ENSDARG00000040705\t10\t0.156669586745304\t1.04105796280058\t6.64492697292305\t4.60453020521711e-05\t0.0389962833230345\n+ENSDARG00000010270\t10\t0.125615247910794\t0.792776147157419\t6.31114582300081\t5.1167831160288e-05\t0.0389962833230345\n+ENSDARG00000016348\t10\t0.141266983378295\t1.03650793650794\t7.33722708392734\t6.15803180656239e-05\t0.0391750861522651\n+ENSDARG00000104197\t10\t1.88512035781081\t12.4705558421044\t6.61525710569826\t6.81411845087906e-05\t0.0391750861522651\n+ENSDARG00000086471\t10\t0.185942018777691\t1.10768386676327\t5.95714660970523\t7.06783578276064e-05\t0.0391750861522651\n+ENSDARG00000102820\t10\t1.91136642367287\t11.7867769779747\t6.16667575196038\t0.000123177015995748\t0.049434996770106\n+ENSDARG00000076025\t10\t0.1358413706015\t0.865139546616693\t6.36874865724554\t0.000127769715977782\t0.049434996770106\n+ENSDARG00000060566\t10\t0.175513373838258\t1.10598275418622\t6.30141584085455\t0.000145658259770016\t0.049434996770106\n+ENSDARG00000077760\t10\t0.172527864230777\t0.981147246329388\t5.68689150998233\t0.000146283831624921\t0.049434996770106\n+ENSDARG000001056691\t10\t2.91736340526917\t19.0715434690697\t6.53725327280921\t0.000159343665923606\t0.049434996770106\n+ENSDARG00000097110\t10\t0.207251034236892\t1.10961177579611\t5.35395048754164\t0.000170684808645056\t0.049434996770106\n+ENSDARG00000017127\t10\t0.321847124787121\t1.71261097592378\t5.32119395833208\t0.000176225214760421\t0.049434996770106\n+ENSDARG00000016138\t10\t0.150588587792026\t0.814494514984075\t5.40873997775283\t0.000180783665507664\t0.049434996770106\n+ENSDARG00000057568\t10\t0.149007737391486\t0.825280606125021\t5.53850840615591\t0.000187775524813667\t0.049434996770106\n+ENSDARG000001042421\t10\t1.9682001003536\t11.3276355799035\t5.7553272037068\t0.000196708249110441\t0.049434996770106\n+ENSDARG00000018593\t10\t0.167994135041575\t0.898088080558678\t5.34594901385347\t0.000257112102487745\t0.0529172584853737\n+ENSDARG00000057206\t10\t0.170963878363685\t0.918781977468761\t5.37412923865865\t0.000277735324181066\t0.0529172584853737\n+ENSDARG000001051951\t10\t3.11026247187605\t18.5149051843724\t5.95284332167778\t0.00032019043858885\t0.0533324207357941\n+ENSDARG00000079934\t10\t0.198961295595687\t1.08503079710172\t5.45347673703652\t0.00032037005450949\t0.0533324207357941\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmpjkj3uuzsintestinal_advanced.genelist v. /tmp/tmp2cg_n_6aintestinal_advanced.genelist )\n--- local_file\n+++ history_data\n@@ -1,251 +1,251 @@\n n\tmean.ncl\tmean.cl\tfc\tpv\tpadj\n-ENSDARG00000109030\t1\t0.274654337828615\t39.4233419769949\t143.538027794031\t1.85453080866723e-17\t1.13070743404441e-13\n-ENSDARG00000108798\t1\t0.327573035068885\t24.802335939268\t75.7154383420246\t1.23506199101095e-15\t3.76508647959687e-12\n-ENSDARG00000107599\t1\t0.564182966938587\t21.783984071531\t38.6115592779005\t1.4324116853842e-12\t2.91113801526248e-09\n-ENSDARG00000108379\t1\t0.68986839145523\t24.7427021224644\t35.8658295247755\t7.5942061344251e-12\t1.15754687003975e-08\n-ENSDARG00000107321\t1\t0.1\t2.23096784138222\t22.3096784138222\t6.33189194610735e-10\t7.7211090390833e-07\n-ENSDARG00000105564\t1\t0.1\t1.97081703295526\t19.7081703295526\t2.35709418427634e-09\t2.39520054025547e-06\n-ENSDARG00000107138\t1\t0.12141391587535\t1.84891204400927\t15.22817241071\t3.76162108069493e-08\t3.27637196128528e-05\n+ENSDARG00000109030\t1\t0.274654337828615\t39.4233419769949\t143.538027794031\t1.85453080866709e-17\t1.13070743404433e-13\n+ENSDARG00000108798\t1\t0.327573035068885\t24.802335939268\t75.7154383420246\t1.23506199101086e-15\t3.7650864795966e-12\n+ENSDARG00000107599\t1\t0.564182966938587\t21.783984071531\t38.6115592779005\t1.43241168538408e-12\t2.91113801526224e-09\n+ENSDARG00000108379\t1\t0.68986839145523\t24.7427021224644\t35.8658295247755\t7.59420613442469e-12\t1.15754687003968e-08\n+ENSDARG00000107321\t1\t0.1\t2.23096784138222\t22.3096784138222\t6.33189194610729e-10\t7.72110903908323e-07\n+ENSDARG00000105564\t1\t0.1\t1.97081703295526\t19.7081703295526\t2.35709418427633e-09\t2.39520054025546e-06\n+ENSDARG00000107138\t1\t0.12141391587535\t1.84891204400927\t15.22817241071\t3.76162108069488e-08\t3.27637196128524e-05\n ENSDARG00000105857\t1\t0.105759843800846\t1.57285911501407\t14.8719878782715\t1.52193964569661e-07\t0.000115990825247653\n-ENSDARG00000090447\t1\t0.117044445524061\t1.5247401265198\t13.0270182381816\t2.86307072797427e-07\t0.000193957135871768\n-ENSDARG00000068650\t1\t0.118444706911236\t1.42924397145598\t12.0667610121833\t4.76963874982153e-07\t0.000290804874576619\n-ENSDARG00000029234\t1\t0.107704361452076\t1.2680785099502\t11.7736969316181\t7.26433141517907e-07\t0.000402642078530425\n-ENSDARG00000100003\t1\t0.120259003658528\t1.3511174001475\t11.2350623158659\t8.51760668849127e-07\t0.000432765399831094\n-ENSDARG00000105009\t1\t0.114133477721009\t1.43604387395694\t12.5821441931984\t1.07513161417503e-06\t0.00046841229720652\n-ENSDARG00000104049\t1\t0.14589886302348\t1.5016492143503\t10.2923983314978\t1.07557358715619e-06\t0.00046841229720652\n********\n*SNIP *\n********\n+ENSDARG00000101413\t10\t0.141224429692467\t1.80827014074558\t12.8042304343753\t1.53774069797471e-07\t0.00093756050355518\n+ENSDARG00000069269\t10\t0.136680427340329\t1.39111418584467\t10.177859499816\t7.94091542244339e-07\t0.00242078806653187\n+ENSDARG00000096457\t10\t0.108692378208486\t0.98849791094733\t9.09445471007417\t8.01600268921723e-06\t0.0136362534840028\n+ENSDARG00000058178\t10\t0.135168786411076\t1.1479669345012\t8.49284043292363\t8.94620533639676e-06\t0.0136362534840028\n+ENSDARG000001048361\t10\t2.19609208044308\t17.0930420303104\t7.78339040631744\t2.81321418960986e-05\t0.0308316937773497\n+ENSDARG00000093313\t10\t0.126396920024548\t0.901779535343549\t7.13450561270331\t3.03411780652941e-05\t0.0308316937773497\n+ENSDARG00000040705\t10\t0.156669586745304\t1.04105796280058\t6.64492697292305\t4.60453020521711e-05\t0.0389962833230345\n+ENSDARG00000010270\t10\t0.125615247910794\t0.792776147157419\t6.31114582300081\t5.1167831160288e-05\t0.0389962833230345\n+ENSDARG00000016348\t10\t0.141266983378295\t1.03650793650794\t7.33722708392734\t6.15803180656239e-05\t0.0391750861522651\n+ENSDARG00000104197\t10\t1.88512035781081\t12.4705558421044\t6.61525710569826\t6.81411845087906e-05\t0.0391750861522651\n+ENSDARG00000086471\t10\t0.185942018777691\t1.10768386676327\t5.95714660970523\t7.06783578276064e-05\t0.0391750861522651\n+ENSDARG00000102820\t10\t1.91136642367287\t11.7867769779747\t6.16667575196038\t0.000123177015995748\t0.049434996770106\n+ENSDARG00000076025\t10\t0.1358413706015\t0.865139546616693\t6.36874865724554\t0.000127769715977782\t0.049434996770106\n+ENSDARG00000060566\t10\t0.175513373838258\t1.10598275418622\t6.30141584085455\t0.000145658259770016\t0.049434996770106\n+ENSDARG00000077760\t10\t0.172527864230777\t0.981147246329388\t5.68689150998233\t0.000146283831624921\t0.049434996770106\n+ENSDARG000001056691\t10\t2.91736340526917\t19.0715434690697\t6.53725327280921\t0.000159343665923606\t0.049434996770106\n+ENSDARG00000097110\t10\t0.207251034236892\t1.10961177579611\t5.35395048754164\t0.000170684808645056\t0.049434996770106\n+ENSDARG00000017127\t10\t0.321847124787121\t1.71261097592378\t5.32119395833208\t0.000176225214760421\t0.049434996770106\n+ENSDARG00000016138\t10\t0.150588587792026\t0.814494514984075\t5.40873997775283\t0.000180783665507664\t0.049434996770106\n+ENSDARG00000057568\t10\t0.149007737391486\t0.825280606125021\t5.53850840615591\t0.000187775524813667\t0.049434996770106\n+ENSDARG000001042421\t10\t1.9682001003536\t11.3276355799035\t5.7553272037068\t0.000196708249110441\t0.049434996770106\n+ENSDARG00000018593\t10\t0.167994135041575\t0.898088080558678\t5.34594901385347\t0.000257112102487745\t0.0529172584853737\n+ENSDARG00000057206\t10\t0.170963878363685\t0.918781977468761\t5.37412923865865\t0.000277735324181066\t0.0529172584853737\n+ENSDARG000001051951\t10\t3.11026247187605\t18.5149051843724\t5.95284332167778\t0.00032019043858885\t0.0533324207357941\n+ENSDARG00000079934\t10\t0.198961295595687\t1.08503079710172\t5.45347673703652\t0.00032037005450949\t0.0533324207357941\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 2, "time_seconds": 417.34253120422363, "tool_id": "raceid_clustering", "tool_version": "3.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_raceid_clustering.test_tool_000002"}, {"data": {"inputs": {"filt|minexpr": "1", "filt|minnumber": "3", "filt|mintotal": "1050", "intable": {"id": "a96b54cfbf34da19", "src": "hda"}}, "job": {"command_line": "Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/raceid/scripts/cluster.R\u0027 \u0027/tmp/tmpha48x28m/job_working_directory/000/176/configs/tmpqm3e9usf\u0027 2\u003e \u0027/tmp/tmpha48x28m/files/5/0/3/dataset_503f61fb-9bea-4906-9258-01fb6e7da7fd.dat\u0027 \u003e /dev/null", "command_version": "0.5\n", "create_time": "2020-01-23T17:39:42.801739", "exit_code": 0, "external_id": "38612", "galaxy_version": "20.01", "history_id": "f8cc3c3467afd773", "id": "a6edf8654f66f4d6", "inputs": {"intable": {"id": "a96b54cfbf34da19", "src": "hda", "uuid": "42b3f008-fa11-4feb-bf50-dc2e106ac700"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outlog": {"id": "0407161672f94bd4", "src": "hda", "uuid": "503f61fb-9bea-4906-9258-01fb6e7da7fd"}, "outpdf": {"id": "4b73dece6b8202b2", "src": "hda", "uuid": "435a53cc-9b6f-413c-a309-e5f6e0c3d3de"}, "outrdat": {"id": "3eabac299c8a6493", "src": "hda", "uuid": "740b8ad1-006c-4682-ad64-a7e3e7030513"}, "outtable": {"id": "8960b7e82c319407", "src": "hda", "uuid": "c7165164-57ca-4479-871e-f87d431b8e1a"}}, "params": {"__input_ext": "\"input\"", "chromInfo": "\"/tmp/tmpha48x28m/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "filt": "{\"hist_geq_one\": \"false\", \"minexpr\": \"1\", \"minnumber\": \"3\", \"mintotal\": \"1050\", \"use\": {\"__current_case__\": 0, \"def\": \"yes\"}}"}, "state": "ok", "stderr": "", "stdout": "", "tool_id": "raceid_filtnormconf", "tool_stderr": "", "tool_stdout": "", "update_time": "2020-01-23T17:40:04.105686", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using sim_size):\n( /tmp/tmpvaw76_rnmatrix.filter.rdat v. /tmp/tmps_srf_ttmatrix.filter.rdat )\nFiles /tmp/tmps_srf_ttmatrix.filter.rdat=1395196b but /tmp/tmpvaw76_rnmatrix.filter.rdat=1394799b - compare by size (delta=300) failed"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpha48x28m/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpha48x28m/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using sim_size):\n( /tmp/tmpvaw76_rnmatrix.filter.rdat v. /tmp/tmps_srf_ttmatrix.filter.rdat )\nFiles /tmp/tmps_srf_ttmatrix.filter.rdat=1395196b but /tmp/tmpvaw76_rnmatrix.filter.rdat=1394799b - compare by size (delta=300) failed\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 33.24210500717163, "tool_id": "raceid_filtnormconf", "tool_version": "3.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_raceid_filtnormconf.test_tool_000001"}, {"data": {"inputs": {"pdf": true, "rep_factorLevel_0|countsFile": [{"id": "598efe8f302ea8ae", "src": "hda"}, {"id": "5c14fa19c3fe9aa2", "src": "hda"}, {"id": "e9cf7d8d4867436b", "src": "hda"}], "rep_factorLevel_0|factorLevel": "Treated", "rep_factorLevel_1|countsFile": [{"id": "fc5c2ad70cbb4f9f", "src": "hda"}, {"id": "5e985d59d5900484", "src": "hda"}, {"id": "2f2abd7e16b39826", "src": "hda"}, {"id": "83e248eabb1b3d31", "src": "hda"}], "rep_factorLevel_1|factorLevel": "Untreated"}, "job": {"command_line": "Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/ruvseq.R\u0027 -p \u0027/tmp/tmp8t9xs33n/files/2/5/2/dataset_252eec59-0550-4834-8438-2a9845d419d8.dat\u0027  --sample_json \u0027/tmp/tmp8t9xs33n/job_working_directory/000/97/configs/tmp1yty76c_\u0027  -H  --min_k 1 --max_k 1 --min_mean_count 5", "command_version": "R version 3.5.1 (2018-07-02) -- \"Feather Spray\", RUVSeq version 1.16.0\n", "create_time": "2020-01-23T16:59:24.161096", "exit_code": 1, "external_id": "16851", "galaxy_version": "20.01", "history_id": "27a22803849b1486", "id": "27a98c5f1b1968ef", "inputs": {"rep_factorLevel_0|countsFile": {"id": "598efe8f302ea8ae", "src": "hda", "uuid": "241ffdb1-2ea8-4e66-80ba-39c72f078242"}, "rep_factorLevel_0|countsFile1": {"id": "598efe8f302ea8ae", "src": "hda", "uuid": "241ffdb1-2ea8-4e66-80ba-39c72f078242"}, "rep_factorLevel_0|countsFile2": {"id": "5c14fa19c3fe9aa2", "src": "hda", "uuid": "93689d74-64cb-4873-a93c-de192e54ff13"}, "rep_factorLevel_0|countsFile3": {"id": "e9cf7d8d4867436b", "src": "hda", "uuid": "0595e0b5-d3ca-449d-a08e-74d84d8c1951"}, "rep_factorLevel_1|countsFile": {"id": "fc5c2ad70cbb4f9f", "src": "hda", "uuid": "57b70cea-cc44-4a9f-88ce-599551a154a2"}, "rep_factorLevel_1|countsFile1": {"id": "fc5c2ad70cbb4f9f", "src": "hda", "uuid": "57b70cea-cc44-4a9f-88ce-599551a154a2"}, "rep_factorLevel_1|countsFile2": {"id": "5e985d59d5900484", "src": "hda", "uuid": "248db368-acd5-4711-af39-5b9ed88c6286"}, "rep_factorLevel_1|countsFile3": {"id": "2f2abd7e16b39826", "src": "hda", "uuid": "e2885406-1ecc-4c6f-84d5-1a80457b840b"}, "rep_factorLevel_1|countsFile4": {"id": "83e248eabb1b3d31", "src": "hda", "uuid": "c0888394-c799-4502-80cb-3a3e1e511225"}}, "job_messages": [{"code_desc": "An undefined error occurred, please check your input carefully and contact your administrator.", "desc": "Fatal error: An undefined error occurred, please check your input carefully and contact your administrator.", "error_level": 3, "match": "Error in", "stream": "stderr", "type": "regex"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"plots": {"id": "a01e20e32f0965d9", "src": "hda", "uuid": "252eec59-0550-4834-8438-2a9845d419d8"}}, "params": {"__input_ext": "\"tabular\"", "chromInfo": "\"/tmp/tmp8t9xs33n/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "header": "\"true\"", "max_k": "\"1\"", "min_k": "\"1\"", "min_mean_count": "\"5\"", "pdf": "\"true\"", "rep_factorLevel": "[{\"__index__\": 0, \"countsFile\": {\"values\": [{\"id\": 127, \"src\": \"hda\"}, {\"id\": 128, \"src\": \"hda\"}, {\"id\": 129, \"src\": \"hda\"}]}, \"factorLevel\": \"Treated\"}, {\"__index__\": 1, \"countsFile\": {\"values\": [{\"id\": 130, \"src\": \"hda\"}, {\"id\": 131, \"src\": \"hda\"}, {\"id\": 132, \"src\": \"hda\"}, {\"id\": 133, \"src\": \"hda\"}]}, \"factorLevel\": \"Untreated\"}]", "tximport": "{\"__current_case__\": 1, \"tximport_selector\": \"count\"}"}, "state": "error", "stderr": "Error in file(filename, \"r\", encoding = encoding) : \n  cannot open the connection\nCalls: source_local -\u003e source -\u003e file\nWarning message:\nIn file(filename, \"r\", encoding = encoding) :\n  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n", "stdout": "", "tool_id": "ruvseq", "tool_stderr": "Error in file(filename, \"r\", encoding = encoding) : \n  cannot open the connection\nCalls: source_local -\u003e source -\u003e file\nWarning message:\nIn file(filename, \"r\", encoding = encoding) :\n  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n", "tool_stdout": "", "update_time": "2020-01-23T17:00:08.323144", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state.", "Failed to find identifier [batch_effects_control_method_k1] for testing, tool generated collection elements [[]]"], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\nFailed to find identifier [batch_effects_control_method_k1] for testing, tool generated collection elements [[]]\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 27a22803849b1486 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - GSM461179_treat_single.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003etreat1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e19\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - GSM461180_treat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003etreat2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e69\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - GSM461181_treat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003etreat3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e59\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - GSM461176_untreat_single.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e15\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - GSM461177_untreat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e73\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e4\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - GSM461178_untreat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e57\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - GSM461182_untreat_single.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat4\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e29\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - RUVSeq covariate files on data 7, data 6, and others (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 8, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u002727a22803849b1486\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027job_source_id\u0027: \u002727a98c5f1b1968ef\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027RUVSeq covariate files on data 7, data 6, and others\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/27a22803849b1486/contents/dataset_collections/2891970512fa2d5a\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 9 - RUVSeq diagonstic plots on data 7, data 6, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n|\n--------------------------------------\n| Job 27a98c5f1b1968ef\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:00:08.323144\n| Create Time:\n|  2020-01-23T16:59:24.161096\n|\n--------------------------------------\n| Job facf9cd8e60ad8cf\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:22.036246\n| Create Time:\n|  2020-01-23T16:59:17.481489\n|\n--------------------------------------\n| Job 2fd653ff8fe916b4\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:15.866892\n| Create Time:\n|  2020-01-23T16:59:10.840257\n|\n--------------------------------------\n| Job 8e7a170d28bd3e46\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:09.540818\n| Create Time:\n|  2020-01-23T16:59:04.231000\n|\n--------------------------------------\n| Job 4923f2d7e29ffb4f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:02.238248\n| Create Time:\n|  2020-01-23T16:58:57.539204\n|\n--------------------------------------\n| Job e5cd870596a22f3d\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:56.189267\n| Create Time:\n|  2020-01-23T16:58:50.868478\n|\n--------------------------------------\n| Job 97f8596e58df898c\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:49.802815\n| Create Time:\n|  2020-01-23T16:58:44.046638\n|\n--------------------------------------\n| Job adcc13f516c5d573\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:42.723901\n| Create Time:\n|  2020-01-23T16:58:37.325478\n|\n--------------------------------------\nProblem in history with id 27a22803849b1486 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - GSM461179_treat_single.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003etreat1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e19\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - GSM461180_treat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003etreat2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e69\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - GSM461181_treat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003etreat3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e59\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - GSM461176_untreat_single.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e15\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - GSM461177_untreat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e73\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e4\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - GSM461178_untreat_paired.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e57\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - GSM461182_untreat_single.counts (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,005 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eGeneID\u003c/td\u003e\u003ctd\u003euntreat4\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e29\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - RUVSeq covariate files on data 7, data 6, and others (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 8, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u002727a22803849b1486\u0027, \u0027id\u0027: \u00272891970512fa2d5a\u0027, \u0027job_source_id\u0027: \u002727a98c5f1b1968ef\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027RUVSeq covariate files on data 7, data 6, and others\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/27a22803849b1486/contents/dataset_collections/2891970512fa2d5a\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 9 - RUVSeq diagonstic plots on data 7, data 6, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n|\n--------------------------------------\n| Job 27a98c5f1b1968ef\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:00:08.323144\n| Create Time:\n|  2020-01-23T16:59:24.161096\n|\n--------------------------------------\n| Job facf9cd8e60ad8cf\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:22.036246\n| Create Time:\n|  2020-01-23T16:59:17.481489\n|\n--------------------------------------\n| Job 2fd653ff8fe916b4\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:15.866892\n| Create Time:\n|  2020-01-23T16:59:10.840257\n|\n--------------------------------------\n| Job 8e7a170d28bd3e46\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:09.540818\n| Create Time:\n|  2020-01-23T16:59:04.231000\n|\n--------------------------------------\n| Job 4923f2d7e29ffb4f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:59:02.238248\n| Create Time:\n|  2020-01-23T16:58:57.539204\n|\n--------------------------------------\n| Job e5cd870596a22f3d\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:56.189267\n| Create Time:\n|  2020-01-23T16:58:50.868478\n|\n--------------------------------------\n| Job 97f8596e58df898c\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:49.802815\n| Create Time:\n|  2020-01-23T16:58:44.046638\n|\n--------------------------------------\n| Job adcc13f516c5d573\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:42.723901\n| Create Time:\n|  2020-01-23T16:58:37.325478\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 99.64249062538147, "tool_id": "ruvseq", "tool_version": "1.16.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ruvseq.test_tool_000000"}, {"data": {"inputs": {"header": false, "pdf": true, "rep_factorLevel_0|countsFile": [{"id": "4e27b888e0f89616", "src": "hda"}, {"id": "15839f6e830b1abe", "src": "hda"}, {"id": "d760e1d92679b730", "src": "hda"}], "rep_factorLevel_0|factorLevel": "Treated", "rep_factorLevel_1|countsFile": [{"id": "237cecdf0f984d5b", "src": "hda"}, {"id": "3995f2aeb4b1e9b8", "src": "hda"}, {"id": "43c20e86f8351b08", "src": "hda"}, {"id": "449eb427f528cea7", "src": "hda"}], "rep_factorLevel_1|factorLevel": "Untreated"}, "job": {"command_line": "Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/ruvseq.R\u0027 -p \u0027/tmp/tmp8t9xs33n/files/0/8/9/dataset_089111de-4dbc-4b8a-8964-82708dd29070.dat\u0027  --sample_json \u0027/tmp/tmp8t9xs33n/job_working_directory/000/105/configs/tmptiyn4cxm\u0027    --min_k 1 --max_k 1 --min_mean_count 5", "command_version": "R version 3.5.1 (2018-07-02) -- \"Feather Spray\", RUVSeq version 1.16.0\n", "create_time": "2020-01-23T17:01:03.632256", "exit_code": 1, "external_id": "17638", "galaxy_version": "20.01", "history_id": "3777da040b354424", "id": "7f39b8b663207ef1", "inputs": {"rep_factorLevel_0|countsFile": {"id": "4e27b888e0f89616", "src": "hda", "uuid": "e6a6a1ac-45b4-4871-b723-cc425514c8de"}, "rep_factorLevel_0|countsFile1": {"id": "4e27b888e0f89616", "src": "hda", "uuid": "e6a6a1ac-45b4-4871-b723-cc425514c8de"}, "rep_factorLevel_0|countsFile2": {"id": "15839f6e830b1abe", "src": "hda", "uuid": "7f44e074-bf81-4ae2-9a3c-d0c879e20255"}, "rep_factorLevel_0|countsFile3": {"id": "d760e1d92679b730", "src": "hda", "uuid": "747fa8f1-cd95-418c-8f19-8b707c5e571f"}, "rep_factorLevel_1|countsFile": {"id": "237cecdf0f984d5b", "src": "hda", "uuid": "adb112d2-62e0-4dc7-8ea7-115ae254e8ce"}, "rep_factorLevel_1|countsFile1": {"id": "237cecdf0f984d5b", "src": "hda", "uuid": "adb112d2-62e0-4dc7-8ea7-115ae254e8ce"}, "rep_factorLevel_1|countsFile2": {"id": "3995f2aeb4b1e9b8", "src": "hda", "uuid": "4c169751-d090-4195-9f25-f196e603a573"}, "rep_factorLevel_1|countsFile3": {"id": "43c20e86f8351b08", "src": "hda", "uuid": "1197506b-448c-4e8f-b158-c7e30e6a7708"}, "rep_factorLevel_1|countsFile4": {"id": "449eb427f528cea7", "src": "hda", "uuid": "b586b2be-facd-438a-a2b6-80250599cd98"}}, "job_messages": [{"code_desc": "An undefined error occurred, please check your input carefully and contact your administrator.", "desc": "Fatal error: An undefined error occurred, please check your input carefully and contact your administrator.", "error_level": 3, "match": "Error in", "stream": "stderr", "type": "regex"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"plots": {"id": "05698cd4bb850c59", "src": "hda", "uuid": "089111de-4dbc-4b8a-8964-82708dd29070"}}, "params": {"__input_ext": "\"tabular\"", "chromInfo": "\"/tmp/tmp8t9xs33n/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "header": "\"false\"", "max_k": "\"1\"", "min_k": "\"1\"", "min_mean_count": "\"5\"", "pdf": "\"true\"", "rep_factorLevel": "[{\"__index__\": 0, \"countsFile\": {\"values\": [{\"id\": 135, \"src\": \"hda\"}, {\"id\": 136, \"src\": \"hda\"}, {\"id\": 137, \"src\": \"hda\"}]}, \"factorLevel\": \"Treated\"}, {\"__index__\": 1, \"countsFile\": {\"values\": [{\"id\": 138, \"src\": \"hda\"}, {\"id\": 139, \"src\": \"hda\"}, {\"id\": 140, \"src\": \"hda\"}, {\"id\": 141, \"src\": \"hda\"}]}, \"factorLevel\": \"Untreated\"}]", "tximport": "{\"__current_case__\": 1, \"tximport_selector\": \"count\"}"}, "state": "error", "stderr": "Error in file(filename, \"r\", encoding = encoding) : \n  cannot open the connection\nCalls: source_local -\u003e source -\u003e file\nWarning message:\nIn file(filename, \"r\", encoding = encoding) :\n  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n", "stdout": "", "tool_id": "ruvseq", "tool_stderr": "Error in file(filename, \"r\", encoding = encoding) : \n  cannot open the connection\nCalls: source_local -\u003e source -\u003e file\nWarning message:\nIn file(filename, \"r\", encoding = encoding) :\n  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n", "tool_stdout": "", "update_time": "2020-01-23T17:01:17.546576", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 3777da040b354424 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - GSM461179_treat_single.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e19\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e980\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - GSM461180_treat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e69\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e2375\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - GSM461181_treat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e59\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e3169\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - GSM461176_untreat_single.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e15\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e1227\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - GSM461177_untreat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e73\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e4\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e3526\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - GSM461178_untreat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e57\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e2834\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - GSM461182_untreat_single.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e29\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e1213\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - RUVSeq covariate files on data 7, data 6, and others (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 8, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u00273777da040b354424\u0027, \u0027id\u0027: \u00275729865256bc2525\u0027, \u0027job_source_id\u0027: \u00277f39b8b663207ef1\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027RUVSeq covariate files on data 7, data 6, and others\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/3777da040b354424/contents/dataset_collections/5729865256bc2525\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 9 - RUVSeq diagonstic plots on data 7, data 6, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n|\n--------------------------------------\n| Job 7f39b8b663207ef1\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:01:17.546576\n| Create Time:\n|  2020-01-23T17:01:03.632256\n|\n--------------------------------------\n| Job 0a3452e55f15c193\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:01.865812\n| Create Time:\n|  2020-01-23T17:00:56.959341\n|\n--------------------------------------\n| Job 20de7af3d288c35f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:55.660625\n| Create Time:\n|  2020-01-23T17:00:50.311476\n|\n--------------------------------------\n| Job 2aec3223e9c2f754\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:49.140920\n| Create Time:\n|  2020-01-23T17:00:43.701458\n|\n--------------------------------------\n| Job b028110e32789f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:41.940142\n| Create Time:\n|  2020-01-23T17:00:37.004355\n|\n--------------------------------------\n| Job 7e4191b982566cec\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:35.600888\n| Create Time:\n|  2020-01-23T17:00:30.337295\n|\n--------------------------------------\n| Job c0ae89d2a79fffb2\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:28.436656\n| Create Time:\n|  2020-01-23T17:00:23.679913\n|\n--------------------------------------\n| Job 4f5573cc87ed03c8\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:22.322305\n| Create Time:\n|  2020-01-23T17:00:16.975518\n|\n--------------------------------------\nProblem in history with id 3777da040b354424 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - GSM461179_treat_single.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e19\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e980\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - GSM461180_treat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e69\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e2375\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - GSM461181_treat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e59\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e3169\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - GSM461176_untreat_single.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e15\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e1227\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - GSM461177_untreat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e73\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e4\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e3526\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - GSM461178_untreat_paired.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e57\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e2834\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - GSM461182_untreat_single.counts.noheader (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  4,004 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000003\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000008\u003c/td\u003e\u003ctd\u003e29\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000014\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000015\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eFBgn0000017\u003c/td\u003e\u003ctd\u003e1213\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - RUVSeq covariate files on data 7, data 6, and others (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 8, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u00273777da040b354424\u0027, \u0027id\u0027: \u00275729865256bc2525\u0027, \u0027job_source_id\u0027: \u00277f39b8b663207ef1\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027RUVSeq covariate files on data 7, data 6, and others\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/3777da040b354424/contents/dataset_collections/5729865256bc2525\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 9 - RUVSeq diagonstic plots on data 7, data 6, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n|\n--------------------------------------\n| Job 7f39b8b663207ef1\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:01:17.546576\n| Create Time:\n|  2020-01-23T17:01:03.632256\n|\n--------------------------------------\n| Job 0a3452e55f15c193\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:01.865812\n| Create Time:\n|  2020-01-23T17:00:56.959341\n|\n--------------------------------------\n| Job 20de7af3d288c35f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:55.660625\n| Create Time:\n|  2020-01-23T17:00:50.311476\n|\n--------------------------------------\n| Job 2aec3223e9c2f754\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:49.140920\n| Create Time:\n|  2020-01-23T17:00:43.701458\n|\n--------------------------------------\n| Job b028110e32789f9f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:41.940142\n| Create Time:\n|  2020-01-23T17:00:37.004355\n|\n--------------------------------------\n| Job 7e4191b982566cec\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:35.600888\n| Create Time:\n|  2020-01-23T17:00:30.337295\n|\n--------------------------------------\n| Job c0ae89d2a79fffb2\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:28.436656\n| Create Time:\n|  2020-01-23T17:00:23.679913\n|\n--------------------------------------\n| Job 4f5573cc87ed03c8\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:00:22.322305\n| Create Time:\n|  2020-01-23T17:00:16.975518\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 73.80086541175842, "tool_id": "ruvseq", "tool_version": "1.16.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ruvseq.test_tool_000001"}, {"data": {"inputs": {"min_mean_count": "0", "pdf": true, "rep_factorLevel_0|countsFile": [{"id": "bfa55a0374b9a6bf", "src": "hda"}, {"id": "4674f339b90935da", "src": "hda"}, {"id": "8bfda5654e551614", "src": "hda"}], "rep_factorLevel_0|factorLevel": "Treated", "rep_factorLevel_1|countsFile": [{"id": "7959e4ab20d6a14d", "src": "hda"}, {"id": "54ea706d2daa88f1", "src": "hda"}, {"id": "0b8510c6952c9474", "src": "hda"}], "rep_factorLevel_1|factorLevel": "Untreated", "tximport|mapping_format|mapping_format_selector": "tabular", "tximport|mapping_format|tabular_file": {"id": "c48fc7b200585e1b", "src": "hda"}, "tximport|tximport_selector": "tximport", "tximport|txtype": "sailfish"}, "job": {"command_line": "ln -s \u0027/tmp/tmp8t9xs33n/files/c/3/e/dataset_c3e5a5d1-621e-4ddb-abf1-8036b0b20744.dat\u0027 mapping.txt \u0026\u0026  Rscript \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/ruvseq.R\u0027 -p \u0027/tmp/tmp8t9xs33n/files/3/0/3/dataset_30390059-d224-45f7-adbc-0b2e2d6233ab.dat\u0027  --sample_json \u0027/tmp/tmp8t9xs33n/job_working_directory/000/113/configs/tmpo5hqxebf\u0027  -H  --min_k 1 --max_k 1 --min_mean_count 0  --txtype sailfish --tx2gene mapping.txt", "command_version": "R version 3.5.1 (2018-07-02) -- \"Feather Spray\", RUVSeq version 1.16.0\n", "create_time": "2020-01-23T17:02:17.611079", "exit_code": 1, "external_id": "18213", "galaxy_version": "20.01", "history_id": "2d785e48ad6d0148", "id": "e7d42a199e1d2420", "inputs": {"rep_factorLevel_0|countsFile": {"id": "bfa55a0374b9a6bf", "src": "hda", "uuid": "96c37344-86d9-4c4e-a1f6-3a4fffbc2e49"}, "rep_factorLevel_0|countsFile1": {"id": "bfa55a0374b9a6bf", "src": "hda", "uuid": "96c37344-86d9-4c4e-a1f6-3a4fffbc2e49"}, "rep_factorLevel_0|countsFile2": {"id": "4674f339b90935da", "src": "hda", "uuid": "c9ffbf81-869c-48eb-8b8d-d9ea29510302"}, "rep_factorLevel_0|countsFile3": {"id": "8bfda5654e551614", "src": "hda", "uuid": "d5cdc2cd-bc89-4a36-a7b4-ed438566d359"}, "rep_factorLevel_1|countsFile": {"id": "7959e4ab20d6a14d", "src": "hda", "uuid": "938f9ba0-2477-49a2-8ca2-e8e534dcb31b"}, "rep_factorLevel_1|countsFile1": {"id": "7959e4ab20d6a14d", "src": "hda", "uuid": "938f9ba0-2477-49a2-8ca2-e8e534dcb31b"}, "rep_factorLevel_1|countsFile2": {"id": "54ea706d2daa88f1", "src": "hda", "uuid": "69ab832b-f6ee-45b4-b0c2-522f5b8e80e3"}, "rep_factorLevel_1|countsFile3": {"id": "0b8510c6952c9474", "src": "hda", "uuid": "3f354632-12d9-4034-9065-f7a49f481622"}, "tximport|tabular_file": {"id": "c48fc7b200585e1b", "src": "hda", "uuid": "c3e5a5d1-621e-4ddb-abf1-8036b0b20744"}}, "job_messages": [{"code_desc": "An undefined error occurred, please check your input carefully and contact your administrator.", "desc": "Fatal error: An undefined error occurred, please check your input carefully and contact your administrator.", "error_level": 3, "match": "Error in", "stream": "stderr", "type": "regex"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"plots": {"id": "db1aec3269b07642", "src": "hda", "uuid": "30390059-d224-45f7-adbc-0b2e2d6233ab"}}, "params": {"__input_ext": "\"tabular\"", "chromInfo": "\"/tmp/tmp8t9xs33n/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "header": "\"true\"", "max_k": "\"1\"", "min_k": "\"1\"", "min_mean_count": "\"0\"", "pdf": "\"true\"", "rep_factorLevel": "[{\"__index__\": 0, \"countsFile\": {\"values\": [{\"id\": 143, \"src\": \"hda\"}, {\"id\": 144, \"src\": \"hda\"}, {\"id\": 145, \"src\": \"hda\"}]}, \"factorLevel\": \"Treated\"}, {\"__index__\": 1, \"countsFile\": {\"values\": [{\"id\": 146, \"src\": \"hda\"}, {\"id\": 147, \"src\": \"hda\"}, {\"id\": 148, \"src\": \"hda\"}]}, \"factorLevel\": \"Untreated\"}]", "tximport": "{\"__current_case__\": 0, \"mapping_format\": {\"__current_case__\": 1, \"mapping_format_selector\": \"tabular\", \"tabular_file\": {\"values\": [{\"id\": 149, \"src\": \"hda\"}]}}, \"tximport_selector\": \"tximport\", \"txtype\": \"sailfish\"}"}, "state": "error", "stderr": "Error in file(filename, \"r\", encoding = encoding) : \n  cannot open the connection\nCalls: source_local -\u003e source -\u003e file\nWarning message:\nIn file(filename, \"r\", encoding = encoding) :\n  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n", "stdout": "", "tool_id": "ruvseq", "tool_stderr": "Error in file(filename, \"r\", encoding = encoding) : \n  cannot open the connection\nCalls: source_local -\u003e source -\u003e file\nWarning message:\nIn file(filename, \"r\", encoding = encoding) :\n  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n", "tool_stdout": "", "update_time": "2020-01-23T17:02:31.268295", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8t9xs33n/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 2d785e48ad6d0148 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - sailfish_quant.sf1.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1759.2\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1837.2\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e649.348\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2555.05\u003c/td\u003e\u003ctd\u003e0.0173509\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - sailfish_quant.sf2.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1764.64\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e2171.69\u003c/td\u003e\u003ctd\u003e0.0161522\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e655.215\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2567.91\u003c/td\u003e\u003ctd\u003e0.01366\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - sailfish_quant.sf3.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1760.44\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e2127.85\u003c/td\u003e\u003ctd\u003e0.0280759\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e650.573\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2198.44\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - sailfish_quant.sf4.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1738.38\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1816.38\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e628.444\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2176.38\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - sailfish_quant.sf5.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1781.23\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1859.23\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e671.305\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2219.23\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - sailfish_quant.sf6.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1747.9\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1825.9\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e638.159\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2562.99\u003c/td\u003e\u003ctd\u003e0.0171188\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - tx2gene.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTXNAME\u003c/td\u003e\u003ctd\u003eGENEID\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_001168316\u003c/td\u003e\u003ctd\u003eDDX11L1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_174914\u003c/td\u003e\u003ctd\u003eDDX11L1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNR_031764\u003c/td\u003e\u003ctd\u003eDDX11L1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_004503\u003c/td\u003e\u003ctd\u003eWASH7P\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - RUVSeq covariate files on data 7, data 6, and others (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 8, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u00272d785e48ad6d0148\u0027, \u0027id\u0027: \u002754f2a3a23292eb07\u0027, \u0027job_source_id\u0027: \u0027e7d42a199e1d2420\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027RUVSeq covariate files on data 7, data 6, and others\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/2d785e48ad6d0148/contents/dataset_collections/54f2a3a23292eb07\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 9 - RUVSeq diagonstic plots on data 7, data 6, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n|\n--------------------------------------\n| Job e7d42a199e1d2420\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:02:31.268295\n| Create Time:\n|  2020-01-23T17:02:17.611079\n|\n--------------------------------------\n| Job 566489874e32c504\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:02:16.170291\n| Create Time:\n|  2020-01-23T17:02:10.968286\n|\n--------------------------------------\n| Job 00c35829965cb81b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:02:08.836402\n| Create Time:\n|  2020-01-23T17:02:04.323938\n|\n--------------------------------------\n| Job a46b181036e98e92\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:02:02.554903\n| Create Time:\n|  2020-01-23T17:01:57.459556\n|\n--------------------------------------\n| Job 3654d500199ce491\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:55.637499\n| Create Time:\n|  2020-01-23T17:01:50.783782\n|\n--------------------------------------\n| Job 4942bca139ae755c\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:49.084638\n| Create Time:\n|  2020-01-23T17:01:44.118619\n|\n--------------------------------------\n| Job 69ce05044577833e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:43.060176\n| Create Time:\n|  2020-01-23T17:01:37.493433\n|\n--------------------------------------\n| Job bd73887c155f3ede\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:35.847313\n| Create Time:\n|  2020-01-23T17:01:30.818461\n|\n--------------------------------------\nProblem in history with id 2d785e48ad6d0148 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - sailfish_quant.sf1.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1759.2\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1837.2\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e649.348\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2555.05\u003c/td\u003e\u003ctd\u003e0.0173509\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - sailfish_quant.sf2.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1764.64\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e2171.69\u003c/td\u003e\u003ctd\u003e0.0161522\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e655.215\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2567.91\u003c/td\u003e\u003ctd\u003e0.01366\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - sailfish_quant.sf3.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1760.44\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e2127.85\u003c/td\u003e\u003ctd\u003e0.0280759\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e650.573\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2198.44\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - sailfish_quant.sf4.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1738.38\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1816.38\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e628.444\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2176.38\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - sailfish_quant.sf5.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1781.23\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1859.23\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e671.305\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2219.23\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - sailfish_quant.sf6.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eName\u003c/td\u003e\u003ctd\u003eLength\u003c/td\u003e\u003ctd\u003eEffectiveLength\u003c/td\u003e\u003ctd\u003eTPM\u003c/td\u003e\u003ctd\u003eNumReads\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017409\u003c/td\u003e\u003ctd\u003e1957\u003c/td\u003e\u003ctd\u003e1747.9\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_014212\u003c/td\u003e\u003ctd\u003e2035\u003c/td\u003e\u003ctd\u003e1825.9\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_173860\u003c/td\u003e\u003ctd\u003e847\u003c/td\u003e\u003ctd\u003e638.159\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003ctd\u003e0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_017410\u003c/td\u003e\u003ctd\u003e2395\u003c/td\u003e\u003ctd\u003e2562.99\u003c/td\u003e\u003ctd\u003e0.0171188\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - tx2gene.tab (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  16 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eTXNAME\u003c/td\u003e\u003ctd\u003eGENEID\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_001168316\u003c/td\u003e\u003ctd\u003eDDX11L1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_174914\u003c/td\u003e\u003ctd\u003eDDX11L1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNR_031764\u003c/td\u003e\u003ctd\u003eDDX11L1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eNM_004503\u003c/td\u003e\u003ctd\u003eWASH7P\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 8 - RUVSeq covariate files on data 7, data 6, and others (HID - NAME) \n| Dataset Collection: {\u0027collection_type\u0027: \u0027list\u0027, \u0027deleted\u0027: False, \u0027element_count\u0027: 0, \u0027elements\u0027: [], \u0027hid\u0027: 8, \u0027history_content_type\u0027: \u0027dataset_collection\u0027, \u0027history_id\u0027: \u00272d785e48ad6d0148\u0027, \u0027id\u0027: \u002754f2a3a23292eb07\u0027, \u0027job_source_id\u0027: \u0027e7d42a199e1d2420\u0027, \u0027job_source_type\u0027: \u0027Job\u0027, \u0027model_class\u0027: \u0027HistoryDatasetCollectionAssociation\u0027, \u0027name\u0027: \u0027RUVSeq covariate files on data 7, data 6, and others\u0027, \u0027populated\u0027: True, \u0027populated_state\u0027: \u0027ok\u0027, \u0027populated_state_message\u0027: None, \u0027tags\u0027: [], \u0027type\u0027: \u0027collection\u0027, \u0027url\u0027: \u0027/api/histories/2d785e48ad6d0148/contents/dataset_collections/54f2a3a23292eb07\u0027, \u0027visible\u0027: True}\n|\n--------------------------------------\n| 9 - RUVSeq diagonstic plots on data 7, data 6, and others (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Error in file(filename, \"r\", encoding = encoding) :\n|  cannot open the connection\n|  Calls: source_local -\u003e source -\u003e file\n|  Warning message:\n|  In file(filename, \"r\", encoding = encoding) :\n|  cannot open file \u0027/home/runner/work/tools-iuc/tools-iuc/tools/ruvseq/get_deseq_dataset.R\u0027: No such file or directory\n|\n--------------------------------------\n| Job e7d42a199e1d2420\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:02:31.268295\n| Create Time:\n|  2020-01-23T17:02:17.611079\n|\n--------------------------------------\n| Job 566489874e32c504\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:02:16.170291\n| Create Time:\n|  2020-01-23T17:02:10.968286\n|\n--------------------------------------\n| Job 00c35829965cb81b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:02:08.836402\n| Create Time:\n|  2020-01-23T17:02:04.323938\n|\n--------------------------------------\n| Job a46b181036e98e92\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:02:02.554903\n| Create Time:\n|  2020-01-23T17:01:57.459556\n|\n--------------------------------------\n| Job 3654d500199ce491\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:55.637499\n| Create Time:\n|  2020-01-23T17:01:50.783782\n|\n--------------------------------------\n| Job 4942bca139ae755c\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:49.084638\n| Create Time:\n|  2020-01-23T17:01:44.118619\n|\n--------------------------------------\n| Job 69ce05044577833e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:43.060176\n| Create Time:\n|  2020-01-23T17:01:37.493433\n|\n--------------------------------------\n| Job bd73887c155f3ede\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:01:35.847313\n| Create Time:\n|  2020-01-23T17:01:30.818461\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 2, "time_seconds": 74.03467202186584, "tool_id": "ruvseq", "tool_version": "1.16.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_ruvseq.test_tool_000002"}, {"data": {"inputs": {"addref_cond|addref_select": "cached", "addref_cond|ref": "equCab2chrM", "input": {"id": "1ae74d26531588b0", "src": "hda"}, "parameter_regions|target_region": "entire_input_file"}, "job": {"command_line": "addthreads=${GALAXY_SLOTS:-1} \u0026\u0026 (( addthreads-- )) \u0026\u0026     ln -s \u0027/tmp/tmp4wbz4cg_/files/4/3/4/dataset_43428575-f605-4bef-9215-e8ef382f9f9f.dat\u0027 infile \u0026\u0026 ln -s \u0027/tmp/tmp4wbz4cg_/files/_metadata_files/7/9/d/metadata_79d57686-8762-45f8-ba26-b250bd46cc82.dat\u0027 infile.bai \u0026\u0026   samtools view  -@ $addthreads -C -h -o \u0027/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\u0027 -T \u0027/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta\u0027 -t \u0027/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta.fai\u0027 --output-fmt-option no_ref infile", "command_version": "Version: 1.9 (using htslib 1.9)\n", "create_time": "2020-01-23T16:48:31.712863", "exit_code": 1, "external_id": "5343", "galaxy_version": "20.01", "history_id": "7b55dbb89df8f4e5", "id": "440a6c2b5d9efe20", "inputs": {"input": {"id": "1ae74d26531588b0", "src": "hda", "uuid": "43428575-f605-4bef-9215-e8ef382f9f9f"}}, "job_messages": [{"code_desc": "Error", "desc": "Fatal error: Exit code 1 (Error)", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "WARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, get_cram_version Exception: bytearray index out of range\nWARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, set_index_file Exception: \u0027samtools returned with error 1: stdout=, stderr=samtools index: \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" is in a format that cannot be usefully indexed\\n\u0027\n", "job_stdout": "", "model_class": "Job", "outputs": {"output_alignment": {"id": "440a6c2b5d9efe20", "src": "hda", "uuid": "26b699d9-7294-4ff1-a93e-e760e88ee12d"}}, "params": {"__input_ext": "\"bam\"", "addref_cond": "{\"__current_case__\": 0, \"addref_select\": \"cached\", \"ref\": \"equCab2chrM\"}", "chromInfo": "\"/tmp/tmp4wbz4cg_/galaxy-dev/tool-data/shared/ucsc/chrom/equCab2.len\"", "dbkey": "\"equCab2\"", "parameter_regions": "{\"__current_case__\": 0, \"target_region\": \"entire_input_file\"}"}, "state": "error", "stderr": "/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta: No such file or directory\n[E::hts_open_format] Failed to open file /tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\nsamtools view: failed to open \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" for writing: Invalid argument\n\nWARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, get_cram_version Exception: bytearray index out of range\nWARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, set_index_file Exception: \u0027samtools returned with error 1: stdout=, stderr=samtools index: \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" is in a format that cannot be usefully indexed\\n\u0027\n", "stdout": "", "tool_id": "samtools_bam_to_cram", "tool_stderr": "/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta: No such file or directory\n[E::hts_open_format] Failed to open file /tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\nsamtools view: failed to open \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" for writing: Invalid argument\n", "tool_stdout": "", "update_time": "2020-01-23T16:48:35.936645", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp4wbz4cg_/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp4wbz4cg_/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp4wbz4cg_/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp4wbz4cg_/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp4wbz4cg_/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 7b55dbb89df8f4e5 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - sam_to_bam_out2.bam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  610 bytes\n| Dataset Info:\n|  uploaded bam file\n| Peek:\n|  Binary bam alignments file\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - samtools BAM to CRAM on data 1.cram (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file /tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file /tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\n|  samtools view: failed to open \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" for writing: Invalid argument\n|  WARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, get_cram_version Exception: bytearray index out of range\n|  WARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, set_index_file Exception: \u0027samtools returned with error 1: stdout=, stderr=samtools index: \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" is in a format that cannot be usefully indexed\\n\u0027\n|\n--------------------------------------\n| Job 440a6c2b5d9efe20\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:48:35.936645\n| Create Time:\n|  2020-01-23T16:48:31.712863\n|\n--------------------------------------\n| Job 1ae74d26531588b0\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:48:29.513686\n| Create Time:\n|  2020-01-23T16:48:25.099038\n|\n--------------------------------------\nProblem in history with id 7b55dbb89df8f4e5 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - sam_to_bam_out2.bam (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  610 bytes\n| Dataset Info:\n|  uploaded bam file\n| Peek:\n|  Binary bam alignments file\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - samtools BAM to CRAM on data 1.cram (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file /tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/bam_to_cram/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file /tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\n|  samtools view: failed to open \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" for writing: Invalid argument\n|  WARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, get_cram_version Exception: bytearray index out of range\n|  WARNING:galaxy.datatypes.binary:\u003cgalaxy.datatypes.binary.CRAM object at 0x7f1097211210\u003e, set_index_file Exception: \u0027samtools returned with error 1: stdout=, stderr=samtools index: \"/tmp/tmp4wbz4cg_/files/2/6/b/dataset_26b699d9-7294-4ff1-a93e-e760e88ee12d.dat\" is in a format that cannot be usefully indexed\\n\u0027\n|\n--------------------------------------\n| Job 440a6c2b5d9efe20\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:48:35.936645\n| Create Time:\n|  2020-01-23T16:48:31.712863\n|\n--------------------------------------\n| Job 1ae74d26531588b0\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:48:29.513686\n| Create Time:\n|  2020-01-23T16:48:25.099038\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 4, "time_seconds": 13.696571111679077, "tool_id": "samtools_bam_to_cram", "tool_version": "1.9"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_samtools_bam_to_cram.test_tool_000004"}, {"data": {"inputs": {"input_alignment": {"id": "3777da040b354424", "src": "hda"}, "parameter_regions|target_region": "entire_input_file", "reference_source|input_reference": "equCab2chrM", "reference_source|reference_source_selector": "cached"}, "job": {"command_line": "ln -f -s \u0027/tmp/tmpzdcvrx36/files/_metadata_files/0/d/8/metadata_0d8ca72d-9997-427f-9a73-2b100cdfb198.dat\u0027 \u0027/tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat.crai\u0027 \u0026\u0026   samtools view -@ ${GALAXY_SLOTS:-2} -b -T \u0027/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta\u0027 -o \u0027/tmp/tmpzdcvrx36/files/6/3/6/dataset_63628d59-8306-41cb-b7df-da0dfa230b1c.dat\u0027 \u0027/tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\u0027", "command_version": "Version: 1.9 (using htslib 1.9)\n", "create_time": "2020-01-23T16:52:00.547123", "exit_code": 1, "external_id": "8020", "galaxy_version": "20.01", "history_id": "80b8022ff3f677b7", "id": "2d785e48ad6d0148", "inputs": {"input_alignment": {"id": "3777da040b354424", "src": "hda", "uuid": "0777c6c0-fe94-4045-8cc6-bc0f8d831757"}}, "job_messages": [{"code_desc": "Error", "desc": "Fatal error: Exit code 1 (Error)", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output_alignment": {"id": "2d785e48ad6d0148", "src": "hda", "uuid": "63628d59-8306-41cb-b7df-da0dfa230b1c"}}, "params": {"__input_ext": "\"cram\"", "chromInfo": "\"/tmp/tmpzdcvrx36/galaxy-dev/tool-data/shared/ucsc/chrom/equCab2.len\"", "dbkey": "\"equCab2\"", "parameter_regions": "{\"__current_case__\": 0, \"target_region\": \"entire_input_file\"}", "reference_source": "{\"__current_case__\": 0, \"input_reference\": \"equCab2chrM\", \"reference_source_selector\": \"cached\"}"}, "state": "error", "stderr": "[samfaipath] fail to read file /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta.\n/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta: No such file or directory\n[E::hts_open_format] Failed to open file /tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\nsamtools view: failed to open \"/tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\" for reading: No such file or directory\n", "stdout": "", "tool_id": "samtools_cram_to_bam", "tool_stderr": "[samfaipath] fail to read file /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta.\n/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta: No such file or directory\n[E::hts_open_format] Failed to open file /tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\nsamtools view: failed to open \"/tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\" for reading: No such file or directory\n", "tool_stdout": "", "update_time": "2020-01-23T16:52:04.213803", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpzdcvrx36/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpzdcvrx36/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpzdcvrx36/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpzdcvrx36/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpzdcvrx36/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 80b8022ff3f677b7 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - test2.cram (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  binary data\n| Dataset Info:\n|  uploaded cram file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003eCRAM binary alignment file\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - samtools CRAM to BAM on data 1.bam (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [samfaipath] fail to read file /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta.\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fa\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [samfaipath] fail to read file /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta.\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file /tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\n|  samtools view: failed to open \"/tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\" for reading: No such file or directory\n|\n--------------------------------------\n| Job 2d785e48ad6d0148\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:52:04.213803\n| Create Time:\n|  2020-01-23T16:52:00.547123\n|\n--------------------------------------\n| Job 3777da040b354424\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:51:58.604488\n| Create Time:\n|  2020-01-23T16:51:53.966081\n|\n--------------------------------------\nProblem in history with id 80b8022ff3f677b7 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - test2.cram (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  binary data\n| Dataset Info:\n|  uploaded cram file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003eCRAM binary alignment file\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - samtools CRAM to BAM on data 1.bam (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  [samfaipath] fail to read file /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta.\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fa\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  [samfaipath] fail to read file /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta.\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/cram_to_bam/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file /tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\n|  samtools view: failed to open \"/tmp/tmpzdcvrx36/files/0/7/7/dataset_0777c6c0-fe94-4045-8cc6-bc0f8d831757.dat\" for reading: No such file or directory\n|\n--------------------------------------\n| Job 2d785e48ad6d0148\n| State: \n|  error\n| Update Time:\n|  2020-01-23T16:52:04.213803\n| Create Time:\n|  2020-01-23T16:52:00.547123\n|\n--------------------------------------\n| Job 3777da040b354424\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:51:58.604488\n| Create Time:\n|  2020-01-23T16:51:53.966081\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 3, "time_seconds": 13.67939805984497, "tool_id": "samtools_cram_to_bam", "tool_version": "1.9"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_samtools_cram_to_bam.test_tool_000003"}, {"data": {"inputs": {"addref_cond|addref_select": "cached", "addref_cond|ref": "equCab2chrM", "input": {"id": "c57c922a3f91222f", "src": "hda"}, "mode|filter_config|cond_region|select_region": "no", "mode|output_options|output_format|oformat": "bam", "mode|outtype": "selected_reads"}, "job": {"command_line": "addthreads=${GALAXY_SLOTS:-1} \u0026\u0026 (( addthreads-- )) \u0026\u0026     ln -s \u0027/tmp/tmpmzja49az/files/7/1/a/dataset_71ade6bb-1c8a-4616-a887-7f1b8e3271c6.dat\u0027 infile \u0026\u0026 ln -s \u0027/tmp/tmpmzja49az/files/_metadata_files/6/0/a/metadata_60afa2b7-fec4-4726-9fe1-348c177e3c13.dat\u0027 infile.crai \u0026\u0026        samtools view -@ $addthreads  -b      -o outfile    -T \u0027/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta\u0027 -t \u0027/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta.fai\u0027   infile    \u0026\u0026 samtools sort -@ $addthreads -m ${GALAXY_MEMORY_MB:-768}M -T \"${TMPDIR:-.}\" -O bam -o tmpsam outfile \u0026\u0026 mv tmpsam outfile", "command_version": "Version: 1.9 (using htslib 1.9)\n", "create_time": "2020-01-23T17:06:45.804930", "exit_code": 1, "external_id": "13161", "galaxy_version": "20.01", "history_id": "adba32ec9811aba7", "id": "4163d05c3cc243fc", "inputs": {"input": {"id": "c57c922a3f91222f", "src": "hda", "uuid": "71ade6bb-1c8a-4616-a887-7f1b8e3271c6"}}, "job_messages": [{"code_desc": "Error", "desc": "Fatal error: Exit code 1 (Error)", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"outputsam": {"id": "315992a54eac5ecc", "src": "hda", "uuid": "ae8ed3eb-fd61-40a4-87e3-b5b40ee21f88"}}, "params": {"__input_ext": "\"cram\"", "addref_cond": "{\"__current_case__\": 2, \"addref_select\": \"cached\", \"ref\": \"equCab2chrM\"}", "chromInfo": "\"/tmp/tmpmzja49az/galaxy-dev/tool-data/shared/ucsc/chrom/equCab2.len\"", "dbkey": "\"equCab2\"", "mode": "{\"__current_case__\": 1, \"filter_config\": {\"cigarcons\": \"\", \"cond_region\": {\"__current_case__\": 0, \"select_region\": \"no\"}, \"cond_rg\": {\"__current_case__\": 0, \"select_rg\": \"no\"}, \"exclusive_filter\": null, \"exclusive_filter_all\": null, \"inclusive_filter\": null, \"library\": \"\", \"quality\": \"\"}, \"output_options\": {\"__current_case__\": 0, \"adv_output\": {\"collapsecigar\": \"false\", \"readtags\": []}, \"complementary_output\": \"false\", \"output_format\": {\"__current_case__\": 1, \"fmtopt\": \"-b\", \"oformat\": \"bam\"}, \"reads_report_type\": \"retained\"}, \"outtype\": \"selected_reads\", \"subsample_config\": {\"subsampling_mode\": {\"__current_case__\": 0, \"factor\": \"1.0\", \"seed\": \"\", \"select_subsample\": \"fraction\"}}}"}, "state": "error", "stderr": "/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta: No such file or directory\n[E::hts_open_format] Failed to open file infile\nsamtools view: failed to open \"infile\" for reading: Invalid argument\n", "stdout": "", "tool_id": "samtools_view", "tool_stderr": "/home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta: No such file or directory\n[E::hts_open_format] Failed to open file infile\nsamtools view: failed to open \"infile\" for reading: Invalid argument\n", "tool_stdout": "", "update_time": "2020-01-23T17:06:50.229162", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpmzja49az/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpmzja49az/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpmzja49az/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpmzja49az/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpmzja49az/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id adba32ec9811aba7 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - test2.cram (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  binary data\n| Dataset Info:\n|  uploaded cram file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003eCRAM binary alignment file\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - Samtools view on data 1: filtered alignments (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file infile\n|  samtools view: failed to open \"infile\" for reading: Invalid argu\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file infile\n|  samtools view: failed to open \"infile\" for reading: Invalid argument\n|\n--------------------------------------\n| Job 4163d05c3cc243fc\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:50.229162\n| Create Time:\n|  2020-01-23T17:06:45.804930\n|\n--------------------------------------\n| Job a96f5ebdac2d8d75\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:44.661935\n| Create Time:\n|  2020-01-23T17:06:39.074487\n|\n--------------------------------------\nProblem in history with id adba32ec9811aba7 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - test2.cram (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  binary data\n| Dataset Info:\n|  uploaded cram file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003eCRAM binary alignment file\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - Samtools view on data 1: filtered alignments (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file infile\n|  samtools view: failed to open \"infile\" for reading: Invalid argu\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  /home/runner/work/tools-iuc/tools-iuc/tool_collections/samtools/samtools_view/test-data/cached_locally/chr_m.fasta: No such file or directory\n|  [E::hts_open_format] Failed to open file infile\n|  samtools view: failed to open \"infile\" for reading: Invalid argument\n|\n--------------------------------------\n| Job 4163d05c3cc243fc\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:50.229162\n| Create Time:\n|  2020-01-23T17:06:45.804930\n|\n--------------------------------------\n| Job a96f5ebdac2d8d75\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:44.661935\n| Create Time:\n|  2020-01-23T17:06:39.074487\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 13, "time_seconds": 13.977412223815918, "tool_id": "samtools_view", "tool_version": "1.9+galaxy1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_samtools_view.test_tool_000013"}, {"data": {"inputs": {"genome": {"id": "4e27b888e0f89616", "src": "hda"}, "maker_gff": {"id": "15839f6e830b1abe", "src": "hda"}}, "job": {"command_line": "cp \u0027/tmp/tmp5b1ldqcu/files/b/8/2/dataset_b8251995-0131-4565-9b45-db3f02cc2cf6.dat\u0027 input.gff3 \u0026\u0026 echo \"##FASTA\" \u003e\u003e input.gff3 \u0026\u0026 cat \u0027/tmp/tmp5b1ldqcu/files/a/4/7/dataset_a4753116-ee43-4c46-bc1a-03261d4c260d.dat\u0027 \u003e\u003e input.gff3 \u0026\u0026 maker2zff input.gff3 \u0026\u0026 fathom -categorize 1000 genome.ann genome.dna \u0026\u0026 fathom -export 1000 -plus uni.ann uni.dna \u0026\u0026 forge export.ann export.dna \u0026\u0026 hmm-assembler.pl snap_training . \u003e \u0027/tmp/tmp5b1ldqcu/files/5/2/9/dataset_52943bbb-1f40-48d1-8b33-8f874fd97482.dat\u0027", "command_version": "", "create_time": "2020-01-23T17:06:57.136937", "exit_code": 127, "external_id": "32985", "galaxy_version": "20.01", "history_id": "9a22496ee796fbfc", "id": "73de4610aed2490c", "inputs": {"genome": {"id": "4e27b888e0f89616", "src": "hda", "uuid": "a4753116-ee43-4c46-bc1a-03261d4c260d"}, "maker_gff": {"id": "15839f6e830b1abe", "src": "hda", "uuid": "b8251995-0131-4565-9b45-db3f02cc2cf6"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 127 ()", "error_level": 3, "exit_code": 127, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"output": {"id": "d760e1d92679b730", "src": "hda", "uuid": "52943bbb-1f40-48d1-8b33-8f874fd97482"}}, "params": {"__input_ext": "\"input\"", "chromInfo": "\"/tmp/tmp5b1ldqcu/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "dbkey": "\"?\"", "gene_num": "\"1000\""}, "state": "error", "stderr": "/usr/local/bin/hmm-assembler.pl: line 9: /usr/local/share/snap/bin/hmm-assembler.pl: not found\n", "stdout": "", "tool_id": "snap_training", "tool_stderr": "/usr/local/bin/hmm-assembler.pl: line 9: /usr/local/share/snap/bin/hmm-assembler.pl: not found\n", "tool_stdout": "", "update_time": "2020-01-23T17:08:32.141478", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp5b1ldqcu/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp5b1ldqcu/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp5b1ldqcu/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp5b1ldqcu/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp5b1ldqcu/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 9a22496ee796fbfc - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - human.fa (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  2 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;HS04636\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003egagctcacattaactatttacagggtaactgcttaggaccagtattatgaggagaattta\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ecctttcccgcctctctttccaagaaacaaggagggggtgaaggtacggagaacagtattt\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ecttctgttgaaagcaacttagctacaaagataaattacagctatgtacactgaaggtagc\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003etatttcattccacaaaataagagttttttaaaaagctatgtatgtatgtgctgcatatag\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - annot.gff3 (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  38 lines, 3 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003egene\u003c/td\u003e\u003ctd\u003e352\u003c/td\u003e\u003ctd\u003e1848\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0;Name=maker-HS08198-exonerate_est2genome-gene-0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003emRNA\u003c/td\u003e\u003ctd\u003e352\u003c/td\u003e\u003ctd\u003e1848\u003c/td\u003e\u003ctd\u003e2869\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1;Parent=maker-HS08198-exonerate_est2genome-gene-0.0;Name=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1;_AED=0.00;_eAED=0.00;_QI=70|1|1|1|0|0|7|0|192\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e352\u003c/td\u003e\u003ctd\u003e397\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1:exon:9;Parent=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e421\u003c/td\u003e\u003ctd\u003e582\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1:exon:10;Parent=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Train SNAP on data 2 and data 1: SNAP trained model (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/bin/hmm-assembler.pl: line 9: /usr/local/share/snap/bin/hmm-assembler.pl: not found\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  /usr/local/bin/hmm-assembler.pl: line 9: /usr/local/share/snap/bin/hmm-assembler.pl: not found\n|\n--------------------------------------\n| Job 73de4610aed2490c\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:08:32.141478\n| Create Time:\n|  2020-01-23T17:06:57.136937\n|\n--------------------------------------\n| Job f1e243926043ec87\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:56.051974\n| Create Time:\n|  2020-01-23T17:06:50.543520\n|\n--------------------------------------\n| Job 03229f6bce6358e8\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:48.751325\n| Create Time:\n|  2020-01-23T17:06:43.899649\n|\n--------------------------------------\nProblem in history with id 9a22496ee796fbfc - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - human.fa (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  2 sequences\n| Dataset Info:\n|  uploaded fasta file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003ctd\u003e\u0026gt;HS04636\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003egagctcacattaactatttacagggtaactgcttaggaccagtattatgaggagaattta\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ecctttcccgcctctctttccaagaaacaaggagggggtgaaggtacggagaacagtattt\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ecttctgttgaaagcaacttagctacaaagataaattacagctatgtacactgaaggtagc\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003etatttcattccacaaaataagagttttttaaaaagctatgtatgtatgtgctgcatatag\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - annot.gff3 (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  38 lines, 3 comments\n| Dataset Info:\n|  uploaded gff3 file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1.Seqid\u003c/th\u003e\u003cth\u003e2.Source\u003c/th\u003e\u003cth\u003e3.Type\u003c/th\u003e\u003cth\u003e4.Start\u003c/th\u003e\u003cth\u003e5.End\u003c/th\u003e\u003cth\u003e6.Score\u003c/th\u003e\u003cth\u003e7.Strand\u003c/th\u003e\u003cth\u003e8.Phase\u003c/th\u003e\u003cth\u003e9.Attributes\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e##gff-version 3\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003egene\u003c/td\u003e\u003ctd\u003e352\u003c/td\u003e\u003ctd\u003e1848\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0;Name=maker-HS08198-exonerate_est2genome-gene-0.0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003emRNA\u003c/td\u003e\u003ctd\u003e352\u003c/td\u003e\u003ctd\u003e1848\u003c/td\u003e\u003ctd\u003e2869\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1;Parent=maker-HS08198-exonerate_est2genome-gene-0.0;Name=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1;_AED=0.00;_eAED=0.00;_QI=70|1|1|1|0|0|7|0|192\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e352\u003c/td\u003e\u003ctd\u003e397\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1:exon:9;Parent=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eHS08198\u003c/td\u003e\u003ctd\u003emaker\u003c/td\u003e\u003ctd\u003eexon\u003c/td\u003e\u003ctd\u003e421\u003c/td\u003e\u003ctd\u003e582\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e.\u003c/td\u003e\u003ctd\u003eID=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1:exon:10;Parent=maker-HS08198-exonerate_est2genome-gene-0.0-mRNA-1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - Train SNAP on data 2 and data 1: SNAP trained model (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  /usr/local/bin/hmm-assembler.pl: line 9: /usr/local/share/snap/bin/hmm-assembler.pl: not found\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  /usr/local/bin/hmm-assembler.pl: line 9: /usr/local/share/snap/bin/hmm-assembler.pl: not found\n|\n--------------------------------------\n| Job 73de4610aed2490c\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:08:32.141478\n| Create Time:\n|  2020-01-23T17:06:57.136937\n|\n--------------------------------------\n| Job f1e243926043ec87\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:56.051974\n| Create Time:\n|  2020-01-23T17:06:50.543520\n|\n--------------------------------------\n| Job 03229f6bce6358e8\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:48.751325\n| Create Time:\n|  2020-01-23T17:06:43.899649\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 116.76263570785522, "tool_id": "snap_training", "tool_version": "2013_11_29+galaxy1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_snap_training.test_tool_000000"}, {"data": {"inputs": {"annotation": {"id": "08fbf3fbfe908f7c", "src": "hda"}, "burnin": "100", "mode|inputs": [{"id": "fed9b1622ba4118e", "src": "hda"}, {"id": "b323ca264f40001e", "src": "hda"}], "mode|replicates": "Replicates", "samples": "1000"}, "job": {"command_line": "ln -s \u0027/tmp/tmp8ys5njk4/files/a/b/4/dataset_ab4be582-ca3c-4569-a914-9e687bf59396.dat\u0027 input_file_0.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/0/7/e/dataset_07ea24c9-23f6-409d-8741-3071ba12707f.dat\u0027 input_file_1.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/2/6/b/dataset_26b52c14-ee7c-4bf2-ac4e-8c83eed1c244.dat\u0027 annotation.dat \u0026\u0026   transit gumbel input_file_0.wig,input_file_1.wig annotation.dat transit_out.txt  -iN 0.0 -tC 0.0 -n TTR -r Sum  -s 1000 -b 100 -m 1 -t 1", "command_version": "", "create_time": "2020-01-23T16:58:01.342910", "exit_code": 1, "external_id": "26128", "galaxy_version": "20.01", "history_id": "ea0b941dfbe636f8", "id": "616a01dfb111aa65", "inputs": {"annotation": {"id": "08fbf3fbfe908f7c", "src": "hda", "uuid": "26b52c14-ee7c-4bf2-ac4e-8c83eed1c244"}, "inputs": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "ab4be582-ca3c-4569-a914-9e687bf59396"}, "inputs1": {"id": "fed9b1622ba4118e", "src": "hda", "uuid": "ab4be582-ca3c-4569-a914-9e687bf59396"}, "inputs2": {"id": "b323ca264f40001e", "src": "hda", "uuid": "07ea24c9-23f6-409d-8741-3071ba12707f"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"sites": {"id": "ca4ebafec84c6506", "src": "hda", "uuid": "8dcc15df-0ec0-4c7c-839c-4ae0d94483dd"}}, "params": {"__input_ext": "\"wig\"", "burnin": "\"100\"", "chromInfo": "\"/tmp/tmp8ys5njk4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "cterm": "\"0.0\"", "dbkey": "\"?\"", "handle_replicates": "\"Sum\"", "mode": "{\"__current_case__\": 1, \"inputs\": {\"values\": [{\"id\": 69, \"src\": \"hda\"}, {\"id\": 70, \"src\": \"hda\"}]}, \"replicates\": \"Replicates\"}", "normalization": "\"TTR\"", "nterm": "\"0.0\"", "samples": "\"1000\"", "smallest": "\"1\"", "trim": "\"1\""}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "stdout": "", "tool_id": "transit_gumbel", "tool_stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "tool_stdout": "", "update_time": "2020-01-23T17:02:26.814304", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\n/home/runner/.planemo/involucro -v=3 -f /tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/deps/mulled/invfile.lua -set CHANNELS=\u0027conda-forge,bioconda\u0027 -set TARGETS=\u0027transit=3.0.1,python=3.7\u0027 -set REPO=\u0027quay.io/local/mulled-v2-98e94080f7dfc7de31b799237ba9a50efe3d7fd8:11668215c5ef47a7782abca5b2f699eb0e8133b1-0\u0027 -set BINDS=\u0027build/dist:/usr/local/\u0027 -set DEST_BASE_IMAGE=\u0027bgruening/busybox-bash:0.1\u0027 -set TEST=true build-and-test\n....\nProblem in history with id ea0b941dfbe636f8 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - Gumbel on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job 616a01dfb111aa65\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:02:26.814304\n| Create Time:\n|  2020-01-23T16:58:01.342910\n|\n--------------------------------------\n| Job 3f322b493a2c6407\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:00.066686\n| Create Time:\n|  2020-01-23T16:57:54.662750\n|\n--------------------------------------\n| Job 8c9b639396181d1e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:52.584335\n| Create Time:\n|  2020-01-23T16:57:47.974483\n|\n--------------------------------------\n| Job 244c246e3628fe4f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:46.563144\n| Create Time:\n|  2020-01-23T16:57:41.281052\n|\n--------------------------------------\nProblem in history with id ea0b941dfbe636f8 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - Gumbel on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job 616a01dfb111aa65\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:02:26.814304\n| Create Time:\n|  2020-01-23T16:58:01.342910\n|\n--------------------------------------\n| Job 3f322b493a2c6407\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:58:00.066686\n| Create Time:\n|  2020-01-23T16:57:54.662750\n|\n--------------------------------------\n| Job 8c9b639396181d1e\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:52.584335\n| Create Time:\n|  2020-01-23T16:57:47.974483\n|\n--------------------------------------\n| Job 244c246e3628fe4f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T16:57:46.563144\n| Create Time:\n|  2020-01-23T16:57:41.281052\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 431.2984879016876, "tool_id": "transit_gumbel", "tool_version": "3.0.1+galaxy2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_transit_gumbel.test_tool_000000"}, {"data": {"inputs": {"annotation": {"id": "4163d05c3cc243fc", "src": "hda"}, "mode|inputs": [{"id": "b31f4d82ada5e956", "src": "hda"}, {"id": "a96f5ebdac2d8d75", "src": "hda"}], "mode|replicates": "Replicates"}, "job": {"command_line": "ln -s \u0027/tmp/tmp8ys5njk4/files/5/3/6/dataset_53632e5e-06a3-4131-8848-2ade0637da58.dat\u0027 input_file_0.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/c/d/d/dataset_cdd2b6a3-4951-4e92-882d-dec86e3956e7.dat\u0027 input_file_1.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/c/c/8/dataset_cc8933cf-c325-4487-b90f-36db03607fd5.dat\u0027 annotation.dat \u0026\u0026   transit hmm input_file_0.wig,input_file_1.wig annotation.dat transit_out.txt  -iN 0.0 -tC 0.0 -n TTR -r Sum", "command_version": "", "create_time": "2020-01-23T17:05:12.648829", "exit_code": 1, "external_id": "26699", "galaxy_version": "20.01", "history_id": "27ee89e2e3d631e0", "id": "bab99c776d4b9391", "inputs": {"annotation": {"id": "4163d05c3cc243fc", "src": "hda", "uuid": "cc8933cf-c325-4487-b90f-36db03607fd5"}, "inputs": {"id": "b31f4d82ada5e956", "src": "hda", "uuid": "53632e5e-06a3-4131-8848-2ade0637da58"}, "inputs1": {"id": "b31f4d82ada5e956", "src": "hda", "uuid": "53632e5e-06a3-4131-8848-2ade0637da58"}, "inputs2": {"id": "a96f5ebdac2d8d75", "src": "hda", "uuid": "cdd2b6a3-4951-4e92-882d-dec86e3956e7"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"genes": {"id": "87dca8aa4ee9d3de", "src": "hda", "uuid": "8cbefcd7-d0fc-448c-a281-59628ef5a022"}, "sites": {"id": "0ed16744ccfa2592", "src": "hda", "uuid": "8431a0d9-5fe9-4047-ad85-4fcedfd24ca2"}}, "params": {"__input_ext": "\"wig\"", "chromInfo": "\"/tmp/tmp8ys5njk4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "cterm": "\"0.0\"", "dbkey": "\"?\"", "handle_replicates": "\"Sum\"", "loess": "\"false\"", "mode": "{\"__current_case__\": 1, \"inputs\": {\"values\": [{\"id\": 73, \"src\": \"hda\"}, {\"id\": 74, \"src\": \"hda\"}]}, \"replicates\": \"Replicates\"}", "normalization": "\"TTR\"", "nterm": "\"0.0\""}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "stdout": "", "tool_id": "transit_hmm", "tool_stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "tool_stdout": "", "update_time": "2020-01-23T17:05:16.690879", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 27ee89e2e3d631e0 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - HMM on data 3, data 2, and data 1 Genes (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| 5 - HMM on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job bab99c776d4b9391\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:05:16.690879\n| Create Time:\n|  2020-01-23T17:05:12.648829\n|\n--------------------------------------\n| Job a6b9cabbe61f73ca\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:10.981527\n| Create Time:\n|  2020-01-23T17:05:05.969987\n|\n--------------------------------------\n| Job 1c6e228e19b11e1b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:04.746691\n| Create Time:\n|  2020-01-23T17:04:59.276719\n|\n--------------------------------------\n| Job 25cc0305829779de\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:04:57.360239\n| Create Time:\n|  2020-01-23T17:04:52.613275\n|\n--------------------------------------\nProblem in history with id 27ee89e2e3d631e0 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - HMM on data 3, data 2, and data 1 Genes (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| 5 - HMM on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job bab99c776d4b9391\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:05:16.690879\n| Create Time:\n|  2020-01-23T17:05:12.648829\n|\n--------------------------------------\n| Job a6b9cabbe61f73ca\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:10.981527\n| Create Time:\n|  2020-01-23T17:05:05.969987\n|\n--------------------------------------\n| Job 1c6e228e19b11e1b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:04.746691\n| Create Time:\n|  2020-01-23T17:04:59.276719\n|\n--------------------------------------\n| Job 25cc0305829779de\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:04:57.360239\n| Create Time:\n|  2020-01-23T17:04:52.613275\n|\n--------------------------------------\nProblem in history with id 27ee89e2e3d631e0 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - HMM on data 3, data 2, and data 1 Genes (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| 5 - HMM on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job bab99c776d4b9391\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:05:16.690879\n| Create Time:\n|  2020-01-23T17:05:12.648829\n|\n--------------------------------------\n| Job a6b9cabbe61f73ca\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:10.981527\n| Create Time:\n|  2020-01-23T17:05:05.969987\n|\n--------------------------------------\n| Job 1c6e228e19b11e1b\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:04.746691\n| Create Time:\n|  2020-01-23T17:04:59.276719\n|\n--------------------------------------\n| Job 25cc0305829779de\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:04:57.360239\n| Create Time:\n|  2020-01-23T17:04:52.613275\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 27.88421392440796, "tool_id": "transit_hmm", "tool_version": "3.0.1+galaxy2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_transit_hmm.test_tool_000000"}, {"data": {"inputs": {"annotation": {"id": "315992a54eac5ecc", "src": "hda"}, "controls": [{"id": "945d4aa2c8920004", "src": "hda"}, {"id": "f276a8cc85beb7f0", "src": "hda"}, {"id": "03229f6bce6358e8", "src": "hda"}], "mode|inputs": [{"id": "a94ebc130565c5b1", "src": "hda"}, {"id": "c57c922a3f91222f", "src": "hda"}], "mode|replicates": "Replicates", "samples": "1000"}, "job": {"command_line": "ln -s \u0027/tmp/tmp8ys5njk4/files/c/d/b/dataset_cdb91132-c113-493d-87f0-2d10047badd4.dat\u0027 input_file_0.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/b/4/5/dataset_b45520d7-31a2-440f-a3da-5682bbe551cd.dat\u0027 input_file_1.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/6/7/9/dataset_6794c8cb-af36-4d9e-8e1f-ee41046e93bf.dat\u0027 annotation.dat \u0026\u0026   ln -s \u0027/tmp/tmp8ys5njk4/files/c/9/e/dataset_c9e6abcf-83e3-401e-ad9e-3d688d3cb545.dat\u0027 control_file_0.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/e/c/d/dataset_ecdf756a-165d-4268-b1e6-1ed92f2428e4.dat\u0027 control_file_1.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/d/c/7/dataset_dc726926-1526-46fb-a7d5-1a884a89fe47.dat\u0027 control_file_2.wig \u0026\u0026 transit resampling input_file_0.wig,input_file_1.wig control_file_0.wig,control_file_1.wig,control_file_2.wig annotation.dat transit_out.txt  -iN 0.0 -tC 0.0 -n TTR -r Sum  -s 1000", "command_version": "", "create_time": "2020-01-23T17:06:00.731971", "exit_code": 1, "external_id": "27181", "galaxy_version": "20.01", "history_id": "61f03d5eef6f1538", "id": "48f3e8809447c25a", "inputs": {"annotation": {"id": "315992a54eac5ecc", "src": "hda", "uuid": "6794c8cb-af36-4d9e-8e1f-ee41046e93bf"}, "controls": {"id": "945d4aa2c8920004", "src": "hda", "uuid": "c9e6abcf-83e3-401e-ad9e-3d688d3cb545"}, "controls1": {"id": "945d4aa2c8920004", "src": "hda", "uuid": "c9e6abcf-83e3-401e-ad9e-3d688d3cb545"}, "controls2": {"id": "f276a8cc85beb7f0", "src": "hda", "uuid": "ecdf756a-165d-4268-b1e6-1ed92f2428e4"}, "controls3": {"id": "03229f6bce6358e8", "src": "hda", "uuid": "dc726926-1526-46fb-a7d5-1a884a89fe47"}, "inputs": {"id": "a94ebc130565c5b1", "src": "hda", "uuid": "cdb91132-c113-493d-87f0-2d10047badd4"}, "inputs1": {"id": "a94ebc130565c5b1", "src": "hda", "uuid": "cdb91132-c113-493d-87f0-2d10047badd4"}, "inputs2": {"id": "c57c922a3f91222f", "src": "hda", "uuid": "b45520d7-31a2-440f-a3da-5682bbe551cd"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"sites": {"id": "f1e243926043ec87", "src": "hda", "uuid": "26c41f39-f392-4809-8f68-3bc33a95e8eb"}}, "params": {"__input_ext": "\"wig\"", "adaptive": "\"false\"", "chromInfo": "\"/tmp/tmp8ys5njk4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "cterm": "\"0.0\"", "dbkey": "\"?\"", "exclude_zero": "\"false\"", "handle_replicates": "\"Sum\"", "histogram": "\"false\"", "loess": "\"false\"", "mode": "{\"__current_case__\": 1, \"inputs\": {\"values\": [{\"id\": 78, \"src\": \"hda\"}, {\"id\": 79, \"src\": \"hda\"}]}, \"replicates\": \"Replicates\"}", "normalization": "\"TTR\"", "nterm": "\"0.0\"", "pseudo": "\"false\"", "samples": "\"1000\""}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "stdout": "", "tool_id": "transit_resampling", "tool_stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "tool_stdout": "", "update_time": "2020-01-23T17:06:04.731096", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id 61f03d5eef6f1538 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - transit-co1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - transit-co1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - transit-co1-rep3.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - Resampling on data 6, data 5, and others Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job 48f3e8809447c25a\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:04.731096\n| Create Time:\n|  2020-01-23T17:06:00.731971\n|\n--------------------------------------\n| Job baa1080356b320b7\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:59.218581\n| Create Time:\n|  2020-01-23T17:05:54.038317\n|\n--------------------------------------\n| Job 0e797101c95aea67\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:53.007445\n| Create Time:\n|  2020-01-23T17:05:47.377916\n|\n--------------------------------------\n| Job 0ff3854aee1a95dd\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:45.605872\n| Create Time:\n|  2020-01-23T17:05:40.698158\n|\n--------------------------------------\n| Job 5e6ad6ca0afeaf24\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:39.481364\n| Create Time:\n|  2020-01-23T17:05:33.997831\n|\n--------------------------------------\n| Job bcb334d054d0b7ff\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:31.971232\n| Create Time:\n|  2020-01-23T17:05:27.257042\n|\n--------------------------------------\n| Job dccb60510b24da2f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:25.611365\n| Create Time:\n|  2020-01-23T17:05:20.556362\n|\n--------------------------------------\nProblem in history with id 61f03d5eef6f1538 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit-in1.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1,000 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003cth\u003e11\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003echromosomal replication initiation protein \u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e1524\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e507\u003c/td\u003e\u003ctd\u003e15607143\u003c/td\u003e\u003ctd\u003e885041\u003c/td\u003e\u003ctd\u003ednaA\u003c/td\u003e\u003ctd\u003eRv0001\u003c/td\u003e\u003ctd\u003eCOG0593L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA polymerase III subunit beta \u003c/td\u003e\u003ctd\u003e2052\u003c/td\u003e\u003ctd\u003e3260\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e402\u003c/td\u003e\u003ctd\u003e15607144\u003c/td\u003e\u003ctd\u003e887092\u003c/td\u003e\u003ctd\u003ednaN\u003c/td\u003e\u003ctd\u003eRv0002\u003c/td\u003e\u003ctd\u003eCOG0592L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003erecombination protein F \u003c/td\u003e\u003ctd\u003e3280\u003c/td\u003e\u003ctd\u003e4437\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e385\u003c/td\u003e\u003ctd\u003e15607145\u003c/td\u003e\u003ctd\u003e887089\u003c/td\u003e\u003ctd\u003erecF\u003c/td\u003e\u003ctd\u003eRv0003\u003c/td\u003e\u003ctd\u003eCOG1195L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ehypothetical protein Rv0004 \u003c/td\u003e\u003ctd\u003e4434\u003c/td\u003e\u003ctd\u003e4997\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e187\u003c/td\u003e\u003ctd\u003e15607146\u003c/td\u003e\u003ctd\u003e887088\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eRv0004\u003c/td\u003e\u003ctd\u003eCOG5512R\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003eDNA gyrase subunit B \u003c/td\u003e\u003ctd\u003e5123\u003c/td\u003e\u003ctd\u003e7267\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e714\u003c/td\u003e\u003ctd\u003e15607147\u003c/td\u003e\u003ctd\u003e887081\u003c/td\u003e\u003ctd\u003egyrB\u003c/td\u003e\u003ctd\u003eRv0005\u003c/td\u003e\u003ctd\u003eCOG0187L\u003c/td\u003e\u003ctd\u003e\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - transit-co1-rep1.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 5 - transit-co1-rep2.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 6 - transit-co1-rep3.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  1 line, 1,999 comments\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd colspan=\"100%\"\u003e# from Griffin et al, (2011).  PLOS Pathogens, e1002251.\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003evariableStep chrom=H37Rv\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e60 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e72 0\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e102 0\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 7 - Resampling on data 6, data 5, and others Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job 48f3e8809447c25a\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:04.731096\n| Create Time:\n|  2020-01-23T17:06:00.731971\n|\n--------------------------------------\n| Job baa1080356b320b7\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:59.218581\n| Create Time:\n|  2020-01-23T17:05:54.038317\n|\n--------------------------------------\n| Job 0e797101c95aea67\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:53.007445\n| Create Time:\n|  2020-01-23T17:05:47.377916\n|\n--------------------------------------\n| Job 0ff3854aee1a95dd\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:45.605872\n| Create Time:\n|  2020-01-23T17:05:40.698158\n|\n--------------------------------------\n| Job 5e6ad6ca0afeaf24\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:39.481364\n| Create Time:\n|  2020-01-23T17:05:33.997831\n|\n--------------------------------------\n| Job bcb334d054d0b7ff\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:31.971232\n| Create Time:\n|  2020-01-23T17:05:27.257042\n|\n--------------------------------------\n| Job dccb60510b24da2f\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:05:25.611365\n| Create Time:\n|  2020-01-23T17:05:20.556362\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 47.91039514541626, "tool_id": "transit_resampling", "tool_version": "3.0.1+galaxy2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_transit_resampling.test_tool_000000"}, {"data": {"inputs": {"annotation": {"id": "19ec6936a7590160", "src": "hda"}, "mode|inputs": [{"id": "73de4610aed2490c", "src": "hda"}, {"id": "df545be7ad5cd764", "src": "hda"}], "mode|replicates": "Replicates"}, "job": {"command_line": "ln -s \u0027/tmp/tmp8ys5njk4/files/9/f/4/dataset_9f486774-12cd-4327-8e5c-73a94fcb592f.dat\u0027 input_file_0.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/1/0/9/dataset_109c1a2e-eabf-4b55-871b-02de73aee597.dat\u0027 input_file_1.wig \u0026\u0026 ln -s \u0027/tmp/tmp8ys5njk4/files/a/f/6/dataset_af6be553-f2c0-4f92-ae34-ec260fc7a3f5.dat\u0027 annotation.dat \u0026\u0026   transit tn5gaps input_file_0.wig,input_file_1.wig annotation.dat transit_out.txt  -iN 0.0 -tC 0.0 -n TTR -r Sum  -m 1", "command_version": "", "create_time": "2020-01-23T17:06:28.577106", "exit_code": 1, "external_id": "27536", "galaxy_version": "20.01", "history_id": "d5b3e51b7b790121", "id": "4ec7f8db939dbc03", "inputs": {"annotation": {"id": "19ec6936a7590160", "src": "hda", "uuid": "af6be553-f2c0-4f92-ae34-ec260fc7a3f5"}, "inputs": {"id": "73de4610aed2490c", "src": "hda", "uuid": "9f486774-12cd-4327-8e5c-73a94fcb592f"}, "inputs1": {"id": "73de4610aed2490c", "src": "hda", "uuid": "9f486774-12cd-4327-8e5c-73a94fcb592f"}, "inputs2": {"id": "df545be7ad5cd764", "src": "hda", "uuid": "109c1a2e-eabf-4b55-871b-02de73aee597"}}, "job_messages": [{"code_desc": "", "desc": "Fatal error: Exit code 1 ()", "error_level": 3, "exit_code": 1, "type": "exit_code"}], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"sites": {"id": "330fe5822b24a25f", "src": "hda", "uuid": "29a87e31-f485-4a44-9f5c-88c4a0151a93"}}, "params": {"__input_ext": "\"wig\"", "chromInfo": "\"/tmp/tmp8ys5njk4/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "cterm": "\"0.0\"", "dbkey": "\"?\"", "handle_replicates": "\"Sum\"", "mode": "{\"__current_case__\": 1, \"inputs\": {\"values\": [{\"id\": 85, \"src\": \"hda\"}, {\"id\": 86, \"src\": \"hda\"}]}, \"replicates\": \"Replicates\"}", "normalization": "\"TTR\"", "nterm": "\"0.0\"", "smallest": "\"1\""}, "state": "error", "stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "stdout": "", "tool_id": "transit_tn5gaps", "tool_stderr": "Traceback (most recent call last):\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n    ws.require(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n    needed = self.resolve(parse_requirements(requirements))\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n    raise VersionConflict(dist, req).with_context(dependent_req)\npkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n    from pkg_resources import load_entry_point\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n    @_call_aside\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n    f(*args, **kwargs)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n    working_set = WorkingSet._build_master()\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n    return cls._build_from_requirements(__requires__)\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n    dists = ws.resolve(reqs, Environment())\n  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n    raise DistributionNotFound(req, requirers)\npkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n", "tool_stdout": "", "update_time": "2020-01-23T17:06:32.841838", "user_email": "test@bx.psu.edu"}, "output_problems": ["Job in error state.", "Job in error state."], "problem_log": "  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8ys5njk4/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\nJob in error state.\nJob in error state.\n-------------------- \u003e\u003e begin captured stdout \u003c\u003c ---------------------\nProblem in history with id d5b3e51b7b790121 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in-tn5.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  2,000 lines\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e27\u003c/td\u003e\u003ctd\u003e3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e31\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e37\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e39\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e47\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in2-tn5.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  2,000 lines\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e27\u003c/td\u003e\u003ctd\u003e3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e33\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e58\u003c/td\u003e\u003ctd\u003e8\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e65\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit_tn5.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  500 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e190\u003c/td\u003e\u003ctd\u003e255\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e21\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrL\u003c/td\u003e\u003ctd\u003eBW25113_0001\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ebifunctional: aspartokinase I (N-terminal)%3B homoserine dehydrogenase I (C-terminal)\u003c/td\u003e\u003ctd\u003e337\u003c/td\u003e\u003ctd\u003e2799\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e820\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrA\u003c/td\u003e\u003ctd\u003eBW25113_0002\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e2801\u003c/td\u003e\u003ctd\u003e3733\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e310\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrB\u003c/td\u003e\u003ctd\u003eBW25113_0003\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e3734\u003c/td\u003e\u003ctd\u003e5020\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e428\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrC\u003c/td\u003e\u003ctd\u003eBW25113_0004\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e5234\u003c/td\u003e\u003ctd\u003e5530\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e98\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eyaaX\u003c/td\u003e\u003ctd\u003eBW25113_0005\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - Tn5Gaps on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job 4ec7f8db939dbc03\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:32.841838\n| Create Time:\n|  2020-01-23T17:06:28.577106\n|\n--------------------------------------\n| Job b908d9d86f8a3d36\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:27.289430\n| Create Time:\n|  2020-01-23T17:06:21.913336\n|\n--------------------------------------\n| Job f8cc3c3467afd773\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:19.830239\n| Create Time:\n|  2020-01-23T17:06:15.152945\n|\n--------------------------------------\n| Job 85f590476b78f8ed\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:13.684139\n| Create Time:\n|  2020-01-23T17:06:08.496429\n|\n--------------------------------------\nProblem in history with id d5b3e51b7b790121 - summary of history\u0027s datasets and jobs below.\n--------------------------------------\n| 1 - transit-in-tn5.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  2,000 lines\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e27\u003c/td\u003e\u003ctd\u003e3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e31\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e37\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e39\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e47\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 2 - transit-in2-tn5.wig (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  2,000 lines\n| Dataset Info:\n|  uploaded wig file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e1\u003c/td\u003e\u003ctd\u003e2\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e27\u003c/td\u003e\u003ctd\u003e3\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e33\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e58\u003c/td\u003e\u003ctd\u003e8\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e65\u003c/td\u003e\u003ctd\u003e1\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 3 - transit_tn5.prot (HID - NAME) \n| Dataset State:\n|  ok\n| Dataset Blurb:\n|  500 lines\n| Dataset Info:\n|  uploaded tabular file\n| Peek:\n|  \u003ctable cellspacing=\"0\" cellpadding=\"3\"\u003e\u003ctr\u003e\u003cth\u003e1\u003c/th\u003e\u003cth\u003e2\u003c/th\u003e\u003cth\u003e3\u003c/th\u003e\u003cth\u003e4\u003c/th\u003e\u003cth\u003e5\u003c/th\u003e\u003cth\u003e6\u003c/th\u003e\u003cth\u003e7\u003c/th\u003e\u003cth\u003e8\u003c/th\u003e\u003cth\u003e9\u003c/th\u003e\u003cth\u003e10\u003c/th\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e190\u003c/td\u003e\u003ctd\u003e255\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e21\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrL\u003c/td\u003e\u003ctd\u003eBW25113_0001\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003ebifunctional: aspartokinase I (N-terminal)%3B homoserine dehydrogenase I (C-terminal)\u003c/td\u003e\u003ctd\u003e337\u003c/td\u003e\u003ctd\u003e2799\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e820\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrA\u003c/td\u003e\u003ctd\u003eBW25113_0002\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e2801\u003c/td\u003e\u003ctd\u003e3733\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e310\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrB\u003c/td\u003e\u003ctd\u003eBW25113_0003\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e3734\u003c/td\u003e\u003ctd\u003e5020\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e428\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003ethrC\u003c/td\u003e\u003ctd\u003eBW25113_0004\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003ctr\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e5234\u003c/td\u003e\u003ctd\u003e5530\u003c/td\u003e\u003ctd\u003e+\u003c/td\u003e\u003ctd\u003e98\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003ctd\u003eyaaX\u003c/td\u003e\u003ctd\u003eBW25113_0005\u003c/td\u003e\u003ctd\u003e-\u003c/td\u003e\u003c/tr\u003e\u003c/table\u003e\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  *Standard error was empty.*\n|\n--------------------------------------\n| 4 - Tn5Gaps on data 3, data 2, and data 1 Sites (HID - NAME) \n| Dataset State:\n|  error\n| Dataset Blurb:\n|  error\n| Dataset Info:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in\n| Peek:\n|  *Peek unavilable.*\n| Dataset Job Standard Output:\n|  *Standard output was empty.*\n| Dataset Job Standard Error:\n|  Traceback (most recent call last):\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 584, in _build_master\n|  ws.require(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 901, in require\n|  needed = self.resolve(parse_requirements(requirements))\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 792, in resolve\n|  raise VersionConflict(dist, req).with_context(dependent_req)\n|  pkg_resources.ContextualVersionConflict: (Pillow 7.0.0 (/usr/local/lib/python3.7/site-packages), Requirement.parse(\u0027pillow~=6.0\u0027), {\u0027tnseq-transit\u0027})\n|  During handling of the above exception, another exception occurred:\n|  Traceback (most recent call last):\n|  File \"/usr/local/bin/transit\", line 6, in \u003cmodule\u003e\n|  from pkg_resources import load_entry_point\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3252, in \u003cmodule\u003e\n|  @_call_aside\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3236, in _call_aside\n|  f(*args, **kwargs)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 3265, in _initialize_master_working_set\n|  working_set = WorkingSet._build_master()\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 586, in _build_master\n|  return cls._build_from_requirements(__requires__)\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 599, in _build_from_requirements\n|  dists = ws.resolve(reqs, Environment())\n|  File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 787, in resolve\n|  raise DistributionNotFound(req, requirers)\n|  pkg_resources.DistributionNotFound: The \u0027pillow~=6.0\u0027 distribution was not found and is required by tnseq-transit\n|\n--------------------------------------\n| Job 4ec7f8db939dbc03\n| State: \n|  error\n| Update Time:\n|  2020-01-23T17:06:32.841838\n| Create Time:\n|  2020-01-23T17:06:28.577106\n|\n--------------------------------------\n| Job b908d9d86f8a3d36\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:27.289430\n| Create Time:\n|  2020-01-23T17:06:21.913336\n|\n--------------------------------------\n| Job f8cc3c3467afd773\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:19.830239\n| Create Time:\n|  2020-01-23T17:06:15.152945\n|\n--------------------------------------\n| Job 85f590476b78f8ed\n| State: \n|  ok\n| Update Time:\n|  2020-01-23T17:06:13.684139\n| Create Time:\n|  2020-01-23T17:06:08.496429\n|\n--------------------------------------\n\n--------------------- \u003e\u003e end captured stdout \u003c\u003c ----------------------", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 27.368898153305054, "tool_id": "transit_tn5gaps", "tool_version": "3.0.1+galaxy2"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_transit_tn5gaps.test_tool_000000"}, {"data": {"inputs": {"blastp": {"id": "513510c2b856e80e", "src": "hda"}, "blastx": {"id": "377e6a8287fa8b6a", "src": "hda"}, "gene_trans_map": {"id": "d60bc3c056ce6d02", "src": "hda"}, "hmmscan": {"id": "25de304670fe7732", "src": "hda"}, "signalp": {"id": "d57022ef22692475", "src": "hda"}, "tmhmm": {"id": "b09dd54432ee600c", "src": "hda"}, "transdecoder": {"id": "f66e4e467d639b46", "src": "hda"}, "trinity": {"id": "4b1441c8dda2d2fe", "src": "hda"}}, "job": {"command_line": "Build_Trinotate_Boilerplate_SQLite_db.pl Trinotate \u0026\u0026  Trinotate Trinotate.sqlite init --gene_trans_map \u0027/tmp/tmp8_jayk4s/files/7/1/3/dataset_71390e14-9a5e-49e1-874c-02874a8b4104.dat\u0027 --transcript_fasta /tmp/tmp8_jayk4s/files/c/e/a/dataset_cea7f91f-4335-428f-8d52-43edc4405b85.dat --transdecoder_pep \u0027/tmp/tmp8_jayk4s/files/d/6/8/dataset_d6845fdc-8f5d-412c-84d7-52bc01f5e8a6.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_swissprot_blastp \u0027/tmp/tmp8_jayk4s/files/c/4/d/dataset_c4d9eccc-17ca-49fa-8f3e-5212d209f39d.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_swissprot_blastx \u0027/tmp/tmp8_jayk4s/files/6/3/d/dataset_63dd4a82-239d-41c3-9799-dbb9729f43b8.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_pfam \u0027/tmp/tmp8_jayk4s/files/9/5/b/dataset_95b35c7c-d93b-496a-8a2e-119b17137f3d.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_tmhmm \u0027/tmp/tmp8_jayk4s/files/7/8/5/dataset_785afc7b-a6ba-4ce1-ae41-476f7aba932e.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_signalp \u0027/tmp/tmp8_jayk4s/files/3/e/b/dataset_3eb6d4c7-3a09-46c0-b663-2c866e25e46e.dat\u0027 \u0026\u0026  Trinotate Trinotate.sqlite report -E 1e-05 --pfam_cutoff DNC   \u003e \u0027/tmp/tmp8_jayk4s/files/c/1/8/dataset_c1819af0-a863-4ae9-aa0e-0613f91a9451.dat\u0027", "command_version": "Error, cannot locate version database file. Be sure it\u0027s in your current directory. at /usr/local/bin/Trinotate line 53.\n", "create_time": "2020-01-23T17:30:21.399124", "exit_code": 0, "external_id": "43934", "galaxy_version": "20.01", "history_id": "bcb334d054d0b7ff", "id": "2ff97a24d56d906f", "inputs": {"blastp": {"id": "513510c2b856e80e", "src": "hda", "uuid": "c4d9eccc-17ca-49fa-8f3e-5212d209f39d"}, "blastx": {"id": "377e6a8287fa8b6a", "src": "hda", "uuid": "63dd4a82-239d-41c3-9799-dbb9729f43b8"}, "gene_trans_map": {"id": "d60bc3c056ce6d02", "src": "hda", "uuid": "71390e14-9a5e-49e1-874c-02874a8b4104"}, "hmmscan": {"id": "25de304670fe7732", "src": "hda", "uuid": "95b35c7c-d93b-496a-8a2e-119b17137f3d"}, "signalp": {"id": "d57022ef22692475", "src": "hda", "uuid": "3eb6d4c7-3a09-46c0-b663-2c866e25e46e"}, "tmhmm": {"id": "b09dd54432ee600c", "src": "hda", "uuid": "785afc7b-a6ba-4ce1-ae41-476f7aba932e"}, "transdecoder": {"id": "f66e4e467d639b46", "src": "hda", "uuid": "d6845fdc-8f5d-412c-84d7-52bc01f5e8a6"}, "trinity": {"id": "4b1441c8dda2d2fe", "src": "hda", "uuid": "cea7f91f-4335-428f-8d52-43edc4405b85"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"trinotate_out": {"id": "7528831c3de506aa", "src": "hda", "uuid": "c1819af0-a863-4ae9-aa0e-0613f91a9451"}}, "params": {"__input_ext": "\"fasta\"", "chromInfo": "\"/tmp/tmp8_jayk4s/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "cond_download": "{\"__current_case__\": 1, \"select_download\": \"yes\"}", "dbkey": "\"?\"", "keep_sqlite": "\"false\"", "report_option": "{\"E\": \"1e-05\", \"incl_pep\": \"false\", \"incl_trans\": \"false\", \"pfam_cutoff\": \"DNC\"}"}, "state": "ok", "stderr": "* Running CMD: wget \"http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\"\n--2020-01-23 17:30:50--  http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\nResolving ftp.ebi.ac.uk... 193.62.193.138\nConnecting to ftp.ebi.ac.uk|193.62.193.138|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 582436772 (555M) [application/octet-stream]\nSaving to: \u0027uniprot_sprot.dat.gz\u0027\n\n     0K .......... .......... .......... .......... ..........  0%  183K 51m52s\n    50K .......... .......... .......... .......... ..........  0%  275K 43m11s\n   100K .......... .......... .......... .......... ..........  0%  509K 34m59s\n   150K .......... .......... .......... .......... ..........  0%  505K 30m55s\n   200K .......... .......... .......... .......... ..........  0% 69.4M 24m46s\n   250K .......... .......... .......... .......... ..........  0%  513K 23m43s\n   300K .......... .......... .......... .......... ..........  0%  518K 22m56s\n   350K .......... .......... .......... .......... ..........  0%  488K 22m30s\n   400K .......... .......... .......... .......... ..........  0%  518K 22m2s\n   450K .......... .......... .......... .......... ..........  0% 20.3M 19m52s\n   500K .......... .......... .......... .......... ..........  0%  522K 19m42s\n   550K .......... .......... .......... .......... ..........  0%  549K 19m30s\n   600K .......... .......... .......... .......... ..........  0%  565K 19m17s\n   650K .......... .......... .......... .......... ..........  0% 11.8M 17m58s\n   700K .......... .......... .......... .......... ..........  0%  559K 17m54s\n   750K .......... .......... .......... .......... ..........  0%  526K 17m54s\n   800K .......... .......... .......... .......... ..........  0%  514K 17m56s\n   850K .......... .......... .......... .......... ..........  0%  527K 17m56s\n   900K .......... .......... .......... .......... ..........  0% 24.3M 17m0s\n   950K .......... .......... .......... .......... ..........  0%  546K 17m1s\n  1000K .......... .......... .......... .......... ..........  0%  550K 17m2s\n  1050K .......... .......... .......... .......... ..........  0%  541K 17m3s\n  1100K .......... .......... .......... .......... ..........  0%  510K 17m7s\n  1150K .......... .......... .......... .......... ..........  0%  194M 16m24s\n  1200K .......... .......... .......... .......... ..........  0%  561K 16m25s\n  1250K .......... .......... .......... .......... ..........  0%  587K 16m24s\n  1300K .......... .......... .......... .......... ..........  0%  576K 16m24s\n  1350K .......... .......... .......... .......... ..........  0%  575K 16m24s\n  1400K .......... .......... .......... .......... ..........  0%  167M 15m50s\n  1450K .......... .......... .......... .......... ..........  0%  591K 15m51s\n  1500K .......... .......... .......... .......... ..........  0%  568K 15m52s\n  1550K .......... .......... .......... .......... ..........  0%  641K 15m50s\n  1600K .......... .......... .......... .......... ..........  0%  596K 15m50s\n  1650K .......... .......... .......... .......... ..........  0% 11.5M 15m23s\n  1700K .......... .......... .......... .......... ..........  0%  584K 15m24s\n  1750K .......... .......... .......... .......... ..........  0%  685K 15m22s\n  1800K .......... .......... .......... .......... ..........  0%  563K 15m24s\n  1850K .......... .......... .......... .......... ..........  0%  576K 15m25s\n  1900K .......... .......... .......... .......... ..........  0% 12.0M 15m3s\n  1950K .......... .......... .......... .......... ..........  0%  601K 15m4s\n  2000K .......... .......... .......... .......... ..........  0%  641K 15m3s\n  2050K .......... .......... .......... .......... ..........  0%  588K 15m4s\n  2100K .......... .......... .......... .......... ..........  0%  585K 15m6s\n  2150K .......... .......... .......... .......... ..........  0% 8.59M 14m47s\n  2200K .......... .......... .......... .......... ..........  0%  589K 14m48s\n  2250K .......... .......... .......... .......... ..........  0%  618K 14m49s\n  2300K .......... .......... .......... .......... ..........  0%  616K 14m49s\n  2350K .......... .......... .......... .......... ..........  0%  624K 14m50s\n  2400K .......... .......... .......... .......... ..........  0% 8.80M 14m33s\n  2450K .......... .......... .......... .......... ..........  0%  634K 14m33s\n  2500K .......... .......... .......... .......... ..........  0%  631K 14m34s\n  2550K .......... .......... .......... .......... ..........  0%  641K 14m34s\n  2600K .......... .......... .......... .......... ..........  0%  617K 14m34s\n  2650K .......... .......... .......... .......... ..........  0% 17.3M 14m19s\n  2700K .......... .......... .......... .......... ..........  0%  650K 14m19s\n  2750K .......... .......... .......... .......... ..........  0%  624K 14m20s\n  2800K .......... .......... .......... .......... ..........  0%  633K 14m20s\n  2850K .......... .......... .......... .......... ..........  0% 11.7M 14m6s\n  2900K .......... .......... .......... .......... ..........  0%  697K 14m5s\n  2950K .......... .......... .......... .......... ..........  0%  614K 14m7s\n  3000K .......... .......... .......... .......... ..........  0%  619K 14m8s\n  3050K .......... .......... .......... .......... ..........  0%  649K 14m8s\n  3100K .......... .......... .......... .......... ..........  0% 9.05M 13m55s\n  3150K .......... .......... .......... .......... ..........  0%  689K 13m55s\n  3200K .......... .......... .......... .......... ..........  0%  662K 13m55s\n  3250K .......... .......... .......... .......... ..........  0%  639K 13m56s\n  3300K .......... .......... .......... .......... ..........  0%  643K 13m57s\n  3350K .......... .......... .......... .......... ..........  0%  232M 13m44s\n  3400K .......... .......... .......... .......... ..........  0%  661K 13m45s\n  3450K .......... .......... .......... .......... ..........  0%  660K 13m45s\n  3500K .......... .......... .......... .......... ..........  0%  657K 13m45s\n  3550K .......... .......... .......... .......... ..........  0%  661K 13m46s\n  3600K .......... .......... .......... .......... ..........  0%  162M 13m34s\n  3650K .......... .......... .......... .......... ..........  0%  618K 13m36s\n  3700K .......... .......... .......... .......... ..........  0%  635K 13m37s\n  3750K .......... .......... .......... .......... ..........  0%  657K 13m37s\n  3800K .......... .......... .......... .......... ..........  0%  640K 13m38s\n  3850K .......... .......... .......... .......... ..........  0%  171M 13m27s\n  3900K .......... .......... .......... .......... ..........  0%  641K 13m28s\n  3950K .......... .......... .......... .......... ..........  0%  638K 13m29s\n  4000K .......... .......... .......... .......... ..........  0%  658K 13m30s\n  4050K .......... .......... .......... .......... ..........  0%  226M 13m20s\n  4100K .......... .......... .......... .......... ..........  0%  617K 13m21s\n  4150K .......... .......... .......... .......... ..........  0%  606K 13m23s\n  4200K .......... .......... .......... .......... ..........  0%  619K 13m24s\n  4250K .......... .......... .......... .......... ..........  0%  616K 13m25s\n  4300K .......... .......... .......... .......... ..........  0%  181M 13m16s\n  4350K .......... .......... .......... .......... ..........  0%  627K 13m17s\n  4400K .......... .......... .......... .......... ..........  0%  637K 13m18s\n  4450K .......... .......... .......... .......... ..........  0%  628K 13m19s\n  4500K .......... .......... .......... .......... ..........  0%  263M 13m10s\n  4550K .......... .......... .......... .......... ..........  0%  621K 13m11s\n  4600K .......... .......... .......... .......... ..........  0%  652K 13m12s\n  4650K .......... .......... .......... .......... ..........  0%  638K 13m13s\n  4700K .......... .......... .......... .......... ..........  0%  647K 13m14s\n  4750K .......... .......... .......... .......... ..........  0%  173M 13m5s\n  4800K .......... .......... .......... .......... ..........  0%  645K 13m6s\n  4850K .......... .......... .......... .......... ..........  0%  628K 13m7s\n  4900K .......... .......... .......... .......... ..........  0%  662K 13m8s\n  4950K .......... .......... .......... .......... ..........  0%  187M 13m0s\n  5000K .......... .......... .......... .......... ..........  0%  656K 13m1s\n  5050K .......... .......... .......... .......... ..........  0%  649K 13m1s\n  5100K .......... .......... .......... .......... ..........  0%  657K 13m2s\n  5150K .......... .......... .......... .......... ..........  0%  664K 13m3s\n  5200K .......... .......... .......... .......... ..........  0%  120M 12m55s\n  5250K .......... .......... .......... .......... ..........  0%  660K 12m56s\n  5300K .......... .......... .......... .......... ..........  0%  659K 12m57s\n  5350K .......... .......... .......... .......... ..........  0%  640K 12m57s\n  5400K .......... .......... .......... .......... ..........  0%  652K 12m58s\n  5450K .......... .......... .......... .......... ..........  0%  127M 12m51s\n  5500K .......... .......... .......... .......... ..........  0%  648K 12m52s\n  5550K .......... .......... .......... .......... ..........  0%  615K 12m53s\n  5600K .......... .......... .......... .......... ..........  0%  648K 12m54s\n  5650K .......... .......... .......... .......... ..........  1%  198M 12m47s\n  5700K .......... .......... .......... .......... ..........  1%  658K 12m48s\n  5750K .......... .......... .......... .......... ..........  1%  660K 12m48s\n  5800K .......... .......... .......... .......... ..........  1%  654K 12m49s\n  5850K .......... .......... .......... .......... ..........  1%  659K 12m50s\n  5900K .......... .......... .......... .......... ..........  1%  161M 12m43s\n  5950K .......... .......... .......... .......... ..........  1%  655K 12m44s\n  6000K .......... .......... .......... .......... ..........  1%  660K 12m45s\n  6050K .......... .......... .......... .......... ..........  1%  636K 12m46s\n  6100K .......... .......... .......... .......... ..........  1%  653K 12m46s\n  6150K .......... .......... .......... .......... ..........  1% 68.2M 12m40s\n  6200K .......... .......... .......... .......... ..........  1%  656K 12m41s\n  6250K .......... .......... .......... .......... ..........  1%  658K 12m42s\n  6300K .......... .......... .......... .......... ..........  1%  664K 12m42s\n  6350K .......... .......... .......... .......... ..........  1%  117M 12m36s\n  6400K .......... .......... .......... .......... ..........  1%  667K 12m37s\n  6450K .......... .......... .......... .......... ..........  1%  658K 12m37s\n  6500K .......... .......... .......... .......... ..........  1%  665K 12m38s\n  6550K .......... .......... .......... .......... ..........  1%  659K 12m39s\n  6600K .......... .......... .......... .......... ..........  1%  153M 12m33s\n  6650K .......... .......... .......... .......... ..........  1%  664K 12m34s\n  6700K .......... .......... .......... .......... ..........  1%  655K 12m34s\n  6750K .......... .......... .......... .......... ..........  1%  647K 12m35s\n  6800K .......... .......... .......... .......... ..........  1%  661K 12m36s\n  6850K .......... .......... .......... .......... ..........  1%  171M 12m30s\n  6900K .......... .......... .......... .......... ..........  1%  656K 12m31s\n  6950K .......... .......... .......... .......... ..........  1%  659K 12m32s\n  7000K .......... .......... .......... .......... ..........  1%  651K 12m32s\n  7050K .......... .......... .......... .......... ..........  1%  661K 12m33s\n  7100K .......... .......... .......... .......... ..........  1% 51.9M 12m28s\n  7150K .......... .......... .......... .......... ..........  1%  661K 12m28s\n  7200K .......... .......... .......... .......... ..........  1%  662K 12m29s\n  7250K .......... .......... .......... .......... ..........  1%  654K 12m30s\n  7300K .......... .......... .......... .......... ..........  1% 70.7M 12m24s\n  7350K .......... .......... .......... .......... ..........  1%  651K 12m25s\n  7400K .......... .......... .......... .......... ..........  1%  657K 12m26s\n  7450K .......... .......... .......... .......... ..........  1%  665K 12m26s\n  7500K .......... .......... .......... .......... ..........  1%  662K 12m27s\n  7550K .......... .......... .......... .......... ..........  1%  129M 12m22s\n  7600K .......... .......... .......... .......... ..........  1%  659K 12m23s\n  7650K .......... .......... .......... .......... ..........  1%  662K 12m23s\n  7700K .......... .......... .......... .......... ..........  1%  661K 12m24s\n  7750K .......... .......... .......... .......... ..........  1%  662K 12m25s\n  7800K .......... .......... .......... .......... ..........  1% 74.4M 12m20s\n  7850K .......... .......... .......... .......... ..........  1%  662K 12m20s\n  7900K .......... .......... .......... .......... ..........  1%  660K 12m21s\n  7950K .......... .......... .......... .......... ..........  1%  665K 12m22s\n  8000K .......... .......... .......... .......... ..........  1% 32.7M 12m17s\n  8050K .......... .......... .......... .......... ..........  1%  657K 12m18s\n  8100K .......... .......... .......... .......... ..........  1%  665K 12m18s\n  8150K .......... .......... .......... .......... ..........  1%  659K 12m19s\n  8200K .......... .......... .......... .......... ..........  1%  663K 12m19s\n  8250K .......... .......... .......... .......... ..........  1% 34.6M 12m15s\n  8300K .......... .......... .......... .......... ..........  1%  672K 12m16s\n  8350K .......... .......... .......... .......... ..........  1%  662K 12m16s\n  8400K .......... .......... .......... .......... ..........  1%  663K 12m17s\n  8450K .......... .......... .......... .......... ..........  1%  660K 12m17s\n  8500K .......... .......... .......... .......... ..........  1%  223M 12m13s\n  8550K .......... .......... .......... .......... ..........  1%  663K 12m14s\n  8600K .......... .......... .......... .......... ..........  1%  649K 12m14s\n  8650K .......... .......... .......... .......... ..........  1%  658K 12m15s\n  8700K .......... .......... .......... .......... ..........  1%  657K 12m15s\n  8750K .......... .......... .......... .......... ..........  1%  185M 12m11s\n  8800K .......... .......... .......... .......... ..........  1%  656K 12m12s\n  8850K .......... .......... .......... .......... ..........  1%  663K 12m12s\n  8900K .......... .......... .......... .......... ..........  1%  664K 12m13s\n  8950K .......... .......... .......... .......... ..........  1% 82.4M 12m9s\n  9000K .......... .......... .......... .......... ..........  1%  662K 12m9s\n  9050K .......... .......... .......... .......... ..........  1%  663K 12m10s\n  9100K .......... .......... .......... .......... ..........  1%  663K 12m11s\n  9150K .......... .......... .......... .......... ..........  1%  662K 12m11s\n  9200K .......... .......... .......... .......... ..........  1%  123M 12m7s\n  9250K .......... .......... .......... .......... ..........  1%  662K 12m8s\n  9300K .......... .......... .......... .......... ..........  1%  661K 12m8s\n  9350K .......... .......... .......... .......... ..........  1%  661K 12m9s\n  9400K .......... .......... .......... .......... ..........  1%  650K 12m9s\n  9450K .......... .......... .......... .......... ..........  1%  151M 12m6s\n  9500K .......... .......... .......... .......... ..........  1%  662K 12m6s\n  9550K .......... .......... .......... .......... ..........  1%  660K 12m7s\n  9600K .......... .......... .......... .......... ..........  1%  662K 12m7s\n  9650K .......... .......... .......... .......... ..........  1%  208M 12m3s\n  9700K .......... .......... .......... .......... ..........  1%  665K 12m4s\n  9750K .......... .......... .......... .......... ..........  1%  659K 12m5s\n  9800K .......... .......... .......... .......... ..........  1%  666K 12m5s\n  9850K .......... .......... .......... .......... ..........  1%  648K 12m6s\n  9900K .......... .......... .......... .......... ..........  1%  183M 12m2s\n  9950K ......\n..\n7s\n258900K .......... .......... .......... .......... .......... 97%  760K 7s\n258950K .......... .......... .......... .......... .......... 97%  839K 7s\n259000K .......... .......... .......... .......... .......... 97%  760K 7s\n259050K .......... .......... .......... .......... .......... 97%  681K 7s\n259100K .......... .......... .......... .......... .......... 97% 1.17M 7s\n259150K .......... .......... .......... .......... .......... 97% 1023K 7s\n259200K .......... .......... .......... .......... .......... 97%  763K 7s\n259250K .......... .......... .......... .......... .......... 97%  678K 6s\n259300K .......... .......... .......... .......... .......... 97%  930K 6s\n259350K .......... .......... .......... .......... .......... 98%  782K 6s\n259400K .......... .......... .......... .......... .......... 98%  998K 6s\n259450K .......... .......... .......... .......... .......... 98%  773K 6s\n259500K .......... .......... .......... .......... .......... 98%  686K 6s\n259550K .......... .......... .......... .......... .......... 98% 1.16M 6s\n259600K .......... .......... .......... .......... .......... 98%  767K 6s\n259650K .......... .......... .......... .......... .......... 98%  848K 6s\n259700K .......... .......... .......... .......... .......... 98%  744K 6s\n259750K .......... .......... .......... .......... .......... 98%  696K 6s\n259800K .......... .......... .......... .......... .......... 98% 1.11M 6s\n259850K .......... .......... .......... .......... .......... 98% 1014K 6s\n259900K .......... .......... .......... .......... .......... 98%  772K 6s\n259950K .......... .......... .......... .......... .......... 98%  679K 6s\n260000K .......... .......... .......... .......... .......... 98%  943K 6s\n260050K .......... .......... .......... .......... .......... 98%  761K 6s\n260100K .......... .......... .......... .......... .......... 98% 1006K 5s\n260150K .......... .......... .......... .......... .......... 98%  775K 5s\n260200K .......... .......... .......... .......... .......... 98%  693K 5s\n260250K .......... .......... .......... .......... .......... 98% 1.11M 5s\n260300K .......... .......... .......... .......... .......... 98%  789K 5s\n260350K .......... .......... .......... .......... .......... 98%  861K 5s\n260400K .......... .......... .......... .......... .......... 98%  735K 5s\n260450K .......... .......... .......... .......... .......... 98%  682K 5s\n260500K .......... .......... .......... .......... .......... 98% 1.12M 5s\n260550K .......... .......... .......... .......... .......... 98%  993K 5s\n260600K .......... .......... .......... .......... .......... 98%  793K 5s\n260650K .......... .......... .......... .......... .......... 98%  686K 5s\n260700K .......... .......... .......... .......... .......... 98%  945K 5s\n260750K .......... .......... .......... .......... .......... 98%  743K 5s\n260800K .......... .......... .......... .......... .......... 98%  996K 5s\n260850K .......... .......... .......... .......... .......... 98%  793K 5s\n260900K .......... .......... .......... .......... .......... 98%  687K 4s\n260950K .......... .......... .......... .......... .......... 98% 1.11M 4s\n261000K .......... .......... .......... .......... .......... 98%  797K 4s\n261050K .......... .......... .......... .......... .......... 98%  875K 4s\n261100K .......... .......... .......... .......... .......... 98%  736K 4s\n261150K .......... .......... .......... .......... .......... 98%  651K 4s\n261200K .......... .......... .......... .......... .......... 98% 1.17M 4s\n261250K .......... .......... .......... .......... .......... 98% 1000K 4s\n261300K .......... .......... .......... .......... .......... 98%  799K 4s\n261350K .......... .......... .......... .......... .......... 98%  659K 4s\n261400K .......... .......... .......... .......... .......... 98% 1.01M 4s\n261450K .......... .......... .......... .......... .......... 98%  714K 4s\n261500K .......... .......... .......... .......... .......... 98% 1001K 4s\n261550K .......... .......... .......... .......... .......... 98%  815K 4s\n261600K .......... .......... .......... .......... .......... 98%  651K 4s\n261650K .......... .......... .......... .......... .......... 98% 1.17M 4s\n261700K .......... .......... .......... .......... .......... 98%  816K 4s\n261750K .......... .......... .......... .......... .......... 98%  879K 3s\n261800K .......... .......... .......... .......... .......... 98%  710K 3s\n261850K .......... .......... .......... .......... .......... 98%  658K 3s\n261900K .......... .......... .......... .......... .......... 98% 1.13M 3s\n261950K .......... .......... .......... .......... .......... 99% 1017K 3s\n262000K .......... .......... .......... .......... .......... 99%  807K 3s\n262050K .......... .......... .......... .......... .......... 99%  653K 3s\n262100K .......... .......... .......... .......... .......... 99% 1.05M 3s\n262150K .......... .......... .......... .......... .......... 99%  692K 3s\n262200K .......... .......... .......... .......... .......... 99% 1015K 3s\n262250K .......... .......... .......... .......... .......... 99%  800K 3s\n262300K .......... .......... .......... .......... .......... 99%  664K 3s\n262350K .......... .......... .......... .......... .......... 99% 1.14M 3s\n262400K .......... .......... .......... .......... .......... 99%  822K 3s\n262450K .......... .......... .......... .......... .......... 99%  919K 3s\n262500K .......... .......... .......... .......... .......... 99%  688K 3s\n262550K .......... .......... .......... .......... .......... 99%  660K 2s\n262600K .......... .......... .......... .......... .......... 99% 1.15M 2s\n262650K .......... .......... .......... .......... .......... 99% 1023K 2s\n262700K .......... .......... .......... .......... .......... 99%  791K 2s\n262750K .......... .......... .......... .......... .......... 99%  652K 2s\n262800K .......... .......... .......... .......... .......... 99% 1.11M 2s\n262850K .......... .......... .......... .......... .......... 99%  673K 2s\n262900K .......... .......... .......... .......... .......... 99% 1.00M 2s\n262950K .......... .......... .......... .......... .......... 99%  789K 2s\n263000K .......... .......... .......... .......... .......... 99%  666K 2s\n263050K .......... .......... .......... .......... .......... 99% 1.14M 2s\n263100K .......... .......... .......... .......... .......... 99%  837K 2s\n263150K .......... .......... .......... .......... .......... 99%  804K 2s\n263200K .......... .......... .......... .......... .......... 99%  764K 2s\n263250K .......... .......... .......... .......... .......... 99%  661K 2s\n263300K .......... .......... .......... .......... .......... 99% 1.12M 2s\n263350K .......... .......... .......... .......... .......... 99% 1.01M 2s\n263400K .......... .......... .......... .......... .......... 99%  800K 1s\n263450K .......... .......... .......... .......... .......... 99%  652K 1s\n263500K .......... .......... .......... .......... .......... 99% 1.13M 1s\n263550K .......... .......... .......... .......... .......... 99%  655K 1s\n263600K .......... .......... .......... .......... .......... 99% 1.02M 1s\n263650K .......... .......... .......... .......... .......... 99%  799K 1s\n263700K .......... .......... .......... .......... .......... 99%  670K 1s\n263750K .......... .......... .......... .......... .......... 99% 1.08M 1s\n263800K .......... .......... .......... .......... .......... 99%  861K 1s\n263850K .......... .......... .......... .......... .......... 99%  810K 1s\n263900K .......... .......... .......... .......... .......... 99%  752K 1s\n263950K .......... .......... .......... .......... .......... 99%  668K 1s\n264000K .......... .......... .......... .......... .......... 99% 1.08M 1s\n264050K .......... .......... .......... .......... .......... 99% 1.04M 1s\n264100K .......... .......... .......... .......... .......... 99%  793K 1s\n264150K .......... .......... .......... .......... .......... 99%  654K 1s\n264200K .......... .......... .......... .......... .......... 99% 1.12M 0s\n264250K .......... .......... .......... .......... .......... 99%  669K 0s\n264300K .......... .......... .......... .......... .......... 99% 1019K 0s\n264350K .......... .......... .......... .......... .......... 99%  807K 0s\n264400K .......... .......... .......... .......... .......... 99%  656K 0s\n264450K .......... .......... .......... .......... .......... 99% 1.11M 0s\n264500K .......... .......... .......... .......... .......... 99%  876K 0s\n264550K .......... .......... .......... .......... .......... 99%  751K 0s\n264600K .......... .......... .......... .......... ....      100%  709K=5m21s\n\n2020-01-23 17:52:22 (825 KB/s) - \u0027Pfam-A.hmm.gz\u0027 saved [270995712/270995712]\n\n* Running CMD: /usr/local/bin/PFAM_dat_parser.pl Pfam-A.hmm.gz\n\r[100]  \r[200]  \r[300]  \r[400]  \r[500]  \r[600]  \r[700]  \r[800]  \r[900]  \r[1000]  \r[1100]  \r[1200]  \r[1300]  \r[1400]  \r[1500]  \r[1600]  \r[1700]  \r[1800]  \r[1900]  \r[2000]  \r[2100]  \r[2200]  \r[2300]  \r[2400]  \r[2500]  \r[2600]  \r[2700]  \r[2800]  \r[2900]  \r[3000]  \r[3100]  \r[3200]  \r[3300]  \r[3400]  \r[3500]  \r[3600]  \r[3700]  \r[3800]  \r[3900]  \r[4000]  \r[4100]  \r[4200]  \r[4300]  \r[4400]  \r[4500]  \r[4600]  \r[4700]  \r[4800]  \r[4900]  \r[5000]  \r[5100]  \r[5200]  \r[5300]  \r[5400]  \r[5500]  \r[5600]  \r[5700]  \r[5800]  \r[5900]  \r[6000]  \r[6100]  \r[6200]  \r[6300]  \r[6400]  \r[6500]  \r[6600]  \r[6700]  \r[6800]  \r[6900]  \r[7000]  \r[7100]  \r[7200]  \r[7300]  \r[7400]  \r[7500]  \r[7600]  \r[7700]  \r[7800]  \r[7900]  \r[8000]  \r[8100]  \r[8200]  \r[8300]  \r[8400]  \r[8500]  \r[8600]  \r[8700]  \r[8800]  \r[8900]  \r[9000]  \r[9100]  \r[9200]  \r[9300]  \r[9400]  \r[9500]  \r[9600]  \r[9700]  \r[9800]  \r[9900]  \r[10000]  \r[10100]  \r[10200]  \r[10300]  \r[10400]  \r[10500]  \r[10600]  \r[10700]  \r[10800]  \r[10900]  \r[11000]  \r[11100]  \r[11200]  \r[11300]  \r[11400]  \r[11500]  \r[11600]  \r[11700]  \r[11800]  \r[11900]  \r[12000]  \r[12100]  \r[12200]  \r[12300]  \r[12400]  \r[12500]  \r[12600]  \r[12700]  \r[12800]  \r[12900]  \r[13000]  \r[13100]  \r[13200]  \r[13300]  \r[13400]  \r[13500]  \r[13600]  \r[13700]  \r[13800]  \r[13900]  \r[14000]  \r[14100]  \r[14200]  \r[14300]  \r[14400]  \r[14500]  \r[14600]  \r[14700]  \r[14800]  \r[14900]  \r[15000]  \r[15100]  \r[15200]  \r[15300]  \r[15400]  \r[15500]  \r[15600]  \r[15700]  \r[15800]  \r[15900]  \r[16000]  \r[16100]  \r[16200]  \r[16300]  \r[16400]  \r[16500]  \r[16600]  \r[16700]  \r[16800]  \r[16900]  \r[17000]  \r[17100]  \r[17200]  \r[17300]  \r[17400]  \r[17500]  \r[17600]  \r[17700]  \r[17800]  \r[17900]  * Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam Pfam-A.hmm.gz.pfam_sqlite_bulk_load\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import Pfam-A.hmm.gz.pfam_sqlite_bulk_load PFAMreference\" | sqlite3 Trinotate.sqlite\n* Running CMD: wget \"http://www.geneontology.org/external2go/pfam2go\" \n--2020-01-23 17:52:34--  http://www.geneontology.org/external2go/pfam2go\nResolving www.geneontology.org... 107.23.42.138\nConnecting to www.geneontology.org|107.23.42.138|:80... connected.\nHTTP request sent, awaiting response... 301 Moved Permanently\nLocation: http://geneontology.org/external2go/pfam2go [following]\n--2020-01-23 17:52:34--  http://geneontology.org/external2go/pfam2go\nResolving geneontology.org... 107.23.42.138\nReusing existing connection to www.geneontology.org:80.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://purl.obolibrary.org/obo/go/external2go/pfam2go [following]\n--2020-01-23 17:52:34--  http://purl.obolibrary.org/obo/go/external2go/pfam2go\nResolving purl.obolibrary.org... 52.3.123.63\nConnecting to purl.obolibrary.org|52.3.123.63|:80... connected.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://current.geneontology.org/ontology/external2go/pfam2go [following]\n--2020-01-23 17:52:34--  http://current.geneontology.org/ontology/external2go/pfam2go\nResolving current.geneontology.org... 99.84.222.32, 99.84.222.131, 99.84.222.60, ...\nConnecting to current.geneontology.org|99.84.222.32|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 699985 (684K) [text/plain]\nSaving to: \u0027pfam2go\u0027\n\n     0K .......... .......... .......... .......... ..........  7% 80.3M 0s\n    50K .......... .......... .......... .......... .......... 14% 35.9M 0s\n   100K .......... .......... .......... .......... .......... 21%  140M 0s\n   150K .......... .......... .......... .......... .......... 29% 52.1M 0s\n   200K .......... .......... .......... .......... .......... 36% 60.0M 0s\n   250K .......... .......... .......... .......... .......... 43% 51.2M 0s\n   300K .......... .......... .......... .......... .......... 51% 51.4M 0s\n   350K .......... .......... .......... .......... .......... 58% 49.8M 0s\n   400K .......... .......... .......... .......... .......... 65% 42.5M 0s\n   450K .......... .......... .......... .......... .......... 73% 68.2M 0s\n   500K .......... .......... .......... .......... .......... 80%  136M 0s\n   550K .......... .......... .......... .......... .......... 87%  103M 0s\n   600K .......... .......... .......... .......... .......... 95%  140M 0s\n   650K .......... .......... .......... ...                  100%  144M=0.01s\n\n2020-01-23 17:52:35 (65.3 MB/s) - \u0027pfam2go\u0027 saved [699985/699985]\n\n* Running CMD: /usr/local/bin/PFAMtoGoParser.pl pfam2go \u003e pfam2go.tab\n* Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam2go pfam2go.tab\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import pfam2go.tab pfam2go\" | sqlite3 Trinotate.sqlite\nCMD: TrinotateSeqLoader.pl --sqlite Trinotate.sqlite --gene_trans_map /tmp/tmp8_jayk4s/files/7/1/3/dataset_71390e14-9a5e-49e1-874c-02874a8b4104.dat --transcript_fasta /tmp/tmp8_jayk4s/files/c/e/a/dataset_cea7f91f-4335-428f-8d52-43edc4405b85.dat --transdecoder_pep /tmp/tmp8_jayk4s/files/d/6/8/dataset_d6845fdc-8f5d-412c-84d7-52bc01f5e8a6.dat --bulk_load\n-parsing gene/trans map file.... done.\n-loading Transcripts.\n\r[100]   \r[200]   \ndone.\n-loading ORFs.\n\ndone.\n\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.Transcript.bulk_load Transcript\" | sqlite3 Trinotate.sqlite\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.ORF.bulk_load ORF\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/c/4/d/dataset_c4d9eccc-17ca-49fa-8f3e-5212d209f39d.dat --prog blastp --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.83 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/6/3/d/dataset_63dd4a82-239d-41c3-9799-dbb9729f43b8.dat --prog blastx --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.88 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_PFAM_loader.pl --sqlite Trinotate.sqlite --pfam /tmp/tmp8_jayk4s/files/9/5/b/dataset_95b35c7c-d93b-496a-8a2e-119b17137f3d.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.pfam_bulk_load.93 HMMERDbase\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_TMHMM_loader.pl --sqlite Trinotate.sqlite --tmhmm /tmp/tmp8_jayk4s/files/7/8/5/dataset_785afc7b-a6ba-4ce1-ae41-476f7aba932e.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.tmhmm_bulk_load.98 tmhmm\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_SIGNALP_loader.pl --sqlite Trinotate.sqlite --signalp /tmp/tmp8_jayk4s/files/3/e/b/dataset_3eb6d4c7-3a09-46c0-b663-2c866e25e46e.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.signalp_bulk_load.103 signalp\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_report_writer.pl --sqlite Trinotate.sqlite -E 1e-05 --pfam_cutoff DNC\n", "stdout": "memory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\n", "tool_id": "trinotate", "tool_stderr": "* Running CMD: wget \"http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\"\n--2020-01-23 17:30:50--  http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\nResolving ftp.ebi.ac.uk... 193.62.193.138\nConnecting to ftp.ebi.ac.uk|193.62.193.138|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 582436772 (555M) [application/octet-stream]\nSaving to: \u0027uniprot_sprot.dat.gz\u0027\n\n     0K .......... .......... .......... .......... ..........  0%  183K 51m52s\n    50K .......... .......... .......... .......... ..........  0%  275K 43m11s\n   100K .......... .......... .......... .......... ..........  0%  509K 34m59s\n   150K .......... .......... .......... .......... ..........  0%  505K 30m55s\n   200K .......... .......... .......... .......... ..........  0% 69.4M 24m46s\n   250K .......... .......... .......... .......... ..........  0%  513K 23m43s\n   300K .......... .......... .......... .......... ..........  0%  518K 22m56s\n   350K .......... .......... .......... .......... ..........  0%  488K 22m30s\n   400K .......... .......... .......... .......... ..........  0%  518K 22m2s\n   450K .......... .......... .......... .......... ..........  0% 20.3M 19m52s\n   500K .......... .......... .......... .......... ..........  0%  522K 19m42s\n   550K .......... .......... .......... .......... ..........  0%  549K 19m30s\n   600K .......... .......... .......... .......... ..........  0%  565K 19m17s\n   650K .......... .......... .......... .......... ..........  0% 11.8M 17m58s\n   700K .......... .......... .......... .......... ..........  0%  559K 17m54s\n   750K .......... .......... .......... .......... ..........  0%  526K 17m54s\n   800K .......... .......... .......... .......... ..........  0%  514K 17m56s\n   850K .......... .......... .......... .......... ..........  0%  527K 17m56s\n   900K .......... .......... .......... .......... ..........  0% 24.3M 17m0s\n   950K .......... .......... .......... .......... ..........  0%  546K 17m1s\n  1000K .......... .......... .......... .......... ..........  0%  550K 17m2s\n  1050K .......... .......... .......... .......... ..........  0%  541K 17m3s\n  1100K .......... .......... .......... .......... ..........  0%  510K 17m7s\n  1150K .......... .......... .......... .......... ..........  0%  194M 16m24s\n  1200K .......... .......... .......... .......... ..........  0%  561K 16m25s\n  1250K .......... .......... .......... .......... ..........  0%  587K 16m24s\n  1300K .......... .......... .......... .......... ..........  0%  576K 16m24s\n  1350K .......... .......... .......... .......... ..........  0%  575K 16m24s\n  1400K .......... .......... .......... .......... ..........  0%  167M 15m50s\n  1450K .......... .......... .......... .......... ..........  0%  591K 15m51s\n  1500K .......... .......... .......... .......... ..........  0%  568K 15m52s\n  1550K .......... .......... .......... .......... ..........  0%  641K 15m50s\n  1600K .......... .......... .......... .......... ..........  0%  596K 15m50s\n  1650K .......... .......... .......... .......... ..........  0% 11.5M 15m23s\n  1700K .......... .......... .......... .......... ..........  0%  584K 15m24s\n  1750K .......... .......... .......... .......... ..........  0%  685K 15m22s\n  1800K .......... .......... .......... .......... ..........  0%  563K 15m24s\n  1850K .......... .......... .......... .......... ..........  0%  576K 15m25s\n  1900K .......... .......... .......... .......... ..........  0% 12.0M 15m3s\n  1950K .......... .......... .......... .......... ..........  0%  601K 15m4s\n  2000K .......... .......... .......... .......... ..........  0%  641K 15m3s\n  2050K .......... .......... .......... .......... ..........  0%  588K 15m4s\n  2100K .......... .......... .......... .......... ..........  0%  585K 15m6s\n  2150K .......... .......... .......... .......... ..........  0% 8.59M 14m47s\n  2200K .......... .......... .......... .......... ..........  0%  589K 14m48s\n  2250K .......... .......... .......... .......... ..........  0%  618K 14m49s\n  2300K .......... .......... .......... .......... ..........  0%  616K 14m49s\n  2350K .......... .......... .......... .......... ..........  0%  624K 14m50s\n  2400K .......... .......... .......... .......... ..........  0% 8.80M 14m33s\n  2450K .......... .......... .......... .......... ..........  0%  634K 14m33s\n  2500K .......... .......... .......... .......... ..........  0%  631K 14m34s\n  2550K .......... .......... .......... .......... ..........  0%  641K 14m34s\n  2600K .......... .......... .......... .......... ..........  0%  617K 14m34s\n  2650K .......... .......... .......... .......... ..........  0% 17.3M 14m19s\n  2700K .......... .......... .......... .......... ..........  0%  650K 14m19s\n  2750K .......... .......... .......... .......... ..........  0%  624K 14m20s\n  2800K .......... .......... .......... .......... ..........  0%  633K 14m20s\n  2850K .......... .......... .......... .......... ..........  0% 11.7M 14m6s\n  2900K .......... .......... .......... .......... ..........  0%  697K 14m5s\n  2950K .......... .......... .......... .......... ..........  0%  614K 14m7s\n  3000K .......... .......... .......... .......... ..........  0%  619K 14m8s\n  3050K .......... .......... .......... .......... ..........  0%  649K 14m8s\n  3100K .......... .......... .......... .......... ..........  0% 9.05M 13m55s\n  3150K .......... .......... .......... .......... ..........  0%  689K 13m55s\n  3200K .......... .......... .......... .......... ..........  0%  662K 13m55s\n  3250K .......... .......... .......... .......... ..........  0%  639K 13m56s\n  3300K .......... .......... .......... .......... ..........  0%  643K 13m57s\n  3350K .......... .......... .......... .......... ..........  0%  232M 13m44s\n  3400K .......... .......... .......... .......... ..........  0%  661K 13m45s\n  3450K .......... .......... .......... .......... ..........  0%  660K 13m45s\n  3500K .......... .......... .......... .......... ..........  0%  657K 13m45s\n  3550K .......... .......... .......... .......... ..........  0%  661K 13m46s\n  3600K .......... .......... .......... .......... ..........  0%  162M 13m34s\n  3650K .......... .......... .......... .......... ..........  0%  618K 13m36s\n  3700K .......... .......... .......... .......... ..........  0%  635K 13m37s\n  3750K .......... .......... .......... .......... ..........  0%  657K 13m37s\n  3800K .......... .......... .......... .......... ..........  0%  640K 13m38s\n  3850K .......... .......... .......... .......... ..........  0%  171M 13m27s\n  3900K .......... .......... .......... .......... ..........  0%  641K 13m28s\n  3950K .......... .......... .......... .......... ..........  0%  638K 13m29s\n  4000K .......... .......... .......... .......... ..........  0%  658K 13m30s\n  4050K .......... .......... .......... .......... ..........  0%  226M 13m20s\n  4100K .......... .......... .......... .......... ..........  0%  617K 13m21s\n  4150K .......... .......... .......... .......... ..........  0%  606K 13m23s\n  4200K .......... .......... .......... .......... ..........  0%  619K 13m24s\n  4250K .......... .......... .......... .......... ..........  0%  616K 13m25s\n  4300K .......... .......... .......... .......... ..........  0%  181M 13m16s\n  4350K .......... .......... .......... .......... ..........  0%  627K 13m17s\n  4400K .......... .......... .......... .......... ..........  0%  637K 13m18s\n  4450K .......... .......... .......... .......... ..........  0%  628K 13m19s\n  4500K .......... .......... .......... .......... ..........  0%  263M 13m10s\n  4550K .......... .......... .......... .......... ..........  0%  621K 13m11s\n  4600K .......... .......... .......... .......... ..........  0%  652K 13m12s\n  4650K .......... .......... .......... .......... ..........  0%  638K 13m13s\n  4700K .......... .......... .......... .......... ..........  0%  647K 13m14s\n  4750K .......... .......... .......... .......... ..........  0%  173M 13m5s\n  4800K .......... .......... .......... .......... ..........  0%  645K 13m6s\n  4850K .......... .......... .......... .......... ..........  0%  628K 13m7s\n  4900K .......... .......... .......... .......... ..........  0%  662K 13m8s\n  4950K .......... .......... .......... .......... ..........  0%  187M 13m0s\n  5000K .......... .......... .......... .......... ..........  0%  656K 13m1s\n  5050K .......... .......... .......... .......... ..........  0%  649K 13m1s\n  5100K .......... .......... .......... .......... ..........  0%  657K 13m2s\n  5150K .......... .......... .......... .......... ..........  0%  664K 13m3s\n  5200K .......... .......... .......... .......... ..........  0%  120M 12m55s\n  5250K .......... .......... .......... .......... ..........  0%  660K 12m56s\n  5300K .......... .......... .......... .......... ..........  0%  659K 12m57s\n  5350K .......... .......... .......... .......... ..........  0%  640K 12m57s\n  5400K .......... .......... .......... .......... ..........  0%  652K 12m58s\n  5450K .......... .......... .......... .......... ..........  0%  127M 12m51s\n  5500K .......... .......... .......... .......... ..........  0%  648K 12m52s\n  5550K .......... .......... .......... .......... ..........  0%  615K 12m53s\n  5600K .......... .......... .......... .......... ..........  0%  648K 12m54s\n  5650K .......... .......... .......... .......... ..........  1%  198M 12m47s\n  5700K .......... .......... .......... .......... ..........  1%  658K 12m48s\n  5750K .......... .......... .......... .......... ..........  1%  660K 12m48s\n  5800K .......... .......... .......... .......... ..........  1%  654K 12m49s\n  5850K .......... .......... .......... .......... ..........  1%  659K 12m50s\n  5900K .......... .......... .......... .......... ..........  1%  161M 12m43s\n  5950K .......... .......... .......... .......... ..........  1%  655K 12m44s\n  6000K .......... .......... .......... .......... ..........  1%  660K 12m45s\n  6050K .......... .......... .......... .......... ..........  1%  636K 12m46s\n  6100K .......... .......... .......... .......... ..........  1%  653K 12m46s\n  6150K .......... .......... .......... .......... ..........  1% 68.2M 12m40s\n  6200K .......... .......... .......... .......... ..........  1%  656K 12m41s\n  6250K .......... .......... .......... .......... ..........  1%  658K 12m42s\n  6300K .......... .......... .......... .......... ..........  1%  664K 12m42s\n  6350K .......... .......... .......... .......... ..........  1%  117M 12m36s\n  6400K .......... .......... .......... .......... ..........  1%  667K 12m37s\n  6450K .......... .......... .......... .......... ..........  1%  658K 12m37s\n  6500K .......... .......... .......... .......... ..........  1%  665K 12m38s\n  6550K .......... .......... .......... .......... ..........  1%  659K 12m39s\n  6600K .......... .......... .......... .......... ..........  1%  153M 12m33s\n  6650K .......... .......... .......... .......... ..........  1%  664K 12m34s\n  6700K .......... .......... .......... .......... ..........  1%  655K 12m34s\n  6750K .......... .......... .......... .......... ..........  1%  647K 12m35s\n  6800K .......... .......... .......... .......... ..........  1%  661K 12m36s\n  6850K .......... .......... .......... .......... ..........  1%  171M 12m30s\n  6900K .......... .......... .......... .......... ..........  1%  656K 12m31s\n  6950K .......... .......... .......... .......... ..........  1%  659K 12m32s\n  7000K .......... .......... .......... .......... ..........  1%  651K 12m32s\n  7050K .......... .......... .......... .......... ..........  1%  661K 12m33s\n  7100K .......... .......... .......... .......... ..........  1% 51.9M 12m28s\n  7150K .......... .......... .......... .......... ..........  1%  661K 12m28s\n  7200K .......... .......... .......... .......... ..........  1%  662K 12m29s\n  7250K .......... .......... .......... .......... ..........  1%  654K 12m30s\n  7300K .......... .......... .......... .......... ..........  1% 70.7M 12m24s\n  7350K .......... .......... .......... .......... ..........  1%  651K 12m25s\n  7400K .......... .......... .......... .......... ..........  1%  657K 12m26s\n  7450K .......... .......... .......... .......... ..........  1%  665K 12m26s\n  7500K .......... .......... .......... .......... ..........  1%  662K 12m27s\n  7550K .......... .......... .......... .......... ..........  1%  129M 12m22s\n  7600K .......... .......... .......... .......... ..........  1%  659K 12m23s\n  7650K .......... .......... .......... .......... ..........  1%  662K 12m23s\n  7700K .......... .......... .......... .......... ..........  1%  661K 12m24s\n  7750K .......... .......... .......... .......... ..........  1%  662K 12m25s\n  7800K .......... .......... .......... .......... ..........  1% 74.4M 12m20s\n  7850K .......... .......... .......... .......... ..........  1%  662K 12m20s\n  7900K .......... .......... .......... .......... ..........  1%  660K 12m21s\n  7950K .......... .......... .......... .......... ..........  1%  665K 12m22s\n  8000K .......... .......... .......... .......... ..........  1% 32.7M 12m17s\n  8050K .......... .......... .......... .......... ..........  1%  657K 12m18s\n  8100K .......... .......... .......... .......... ..........  1%  665K 12m18s\n  8150K .......... .......... .......... .......... ..........  1%  659K 12m19s\n  8200K .......... .......... .......... .......... ..........  1%  663K 12m19s\n  8250K .......... .......... .......... .......... ..........  1% 34.6M 12m15s\n  8300K .......... .......... .......... .......... ..........  1%  672K 12m16s\n  8350K .......... .......... .......... .......... ..........  1%  662K 12m16s\n  8400K .......... .......... .......... .......... ..........  1%  663K 12m17s\n  8450K .......... .......... .......... .......... ..........  1%  660K 12m17s\n  8500K .......... .......... .......... .......... ..........  1%  223M 12m13s\n  8550K .......... .......... .......... .......... ..........  1%  663K 12m14s\n  8600K .......... .......... .......... .......... ..........  1%  649K 12m14s\n  8650K .......... .......... .......... .......... ..........  1%  658K 12m15s\n  8700K .......... .......... .......... .......... ..........  1%  657K 12m15s\n  8750K .......... .......... .......... .......... ..........  1%  185M 12m11s\n  8800K .......... .......... .......... .......... ..........  1%  656K 12m12s\n  8850K .......... .......... .......... .......... ..........  1%  663K 12m12s\n  8900K .......... .......... .......... .......... ..........  1%  664K 12m13s\n  8950K .......... .......... .......... .......... ..........  1% 82.4M 12m9s\n  9000K .......... .......... .......... .......... ..........  1%  662K 12m9s\n  9050K .......... .......... .......... .......... ..........  1%  663K 12m10s\n  9100K .......... .......... .......... .......... ..........  1%  663K 12m11s\n  9150K .......... .......... .......... .......... ..........  1%  662K 12m11s\n  9200K .......... .......... .......... .......... ..........  1%  123M 12m7s\n  9250K .......... .......... .......... .......... ..........  1%  662K 12m8s\n  9300K .......... .......... .......... .......... ..........  1%  661K 12m8s\n  9350K .......... .......... .......... .......... ..........  1%  661K 12m9s\n  9400K .......... .......... .......... .......... ..........  1%  650K 12m9s\n  9450K .......... .......... .......... .......... ..........  1%  151M 12m6s\n  9500K .......... .......... .......... .......... ..........  1%  662K 12m6s\n  9550K .......... .......... .......... .......... ..........  1%  660K 12m7s\n  9600K .......... .......... .......... .......... ..........  1%  662K 12m7s\n  9650K .......... .......... .......... .......... ..........  1%  208M 12m3s\n  9700K .......... .......... .......... .......... ..........  1%  665K 12m4s\n  9750K .......... .......... .......... .......... ..........  1%  659K 12m5s\n  9800K .......... .......... .......... .......... ..........  1%  666K 12m5s\n  9850K .......... .......... .......... .......... ..........  1%  648K 12m6s\n  9900K .......... .......... .......... .......... ..........  1%  183M 12m2s\n  9950K ......\n..\n7s\n258900K .......... .......... .......... .......... .......... 97%  760K 7s\n258950K .......... .......... .......... .......... .......... 97%  839K 7s\n259000K .......... .......... .......... .......... .......... 97%  760K 7s\n259050K .......... .......... .......... .......... .......... 97%  681K 7s\n259100K .......... .......... .......... .......... .......... 97% 1.17M 7s\n259150K .......... .......... .......... .......... .......... 97% 1023K 7s\n259200K .......... .......... .......... .......... .......... 97%  763K 7s\n259250K .......... .......... .......... .......... .......... 97%  678K 6s\n259300K .......... .......... .......... .......... .......... 97%  930K 6s\n259350K .......... .......... .......... .......... .......... 98%  782K 6s\n259400K .......... .......... .......... .......... .......... 98%  998K 6s\n259450K .......... .......... .......... .......... .......... 98%  773K 6s\n259500K .......... .......... .......... .......... .......... 98%  686K 6s\n259550K .......... .......... .......... .......... .......... 98% 1.16M 6s\n259600K .......... .......... .......... .......... .......... 98%  767K 6s\n259650K .......... .......... .......... .......... .......... 98%  848K 6s\n259700K .......... .......... .......... .......... .......... 98%  744K 6s\n259750K .......... .......... .......... .......... .......... 98%  696K 6s\n259800K .......... .......... .......... .......... .......... 98% 1.11M 6s\n259850K .......... .......... .......... .......... .......... 98% 1014K 6s\n259900K .......... .......... .......... .......... .......... 98%  772K 6s\n259950K .......... .......... .......... .......... .......... 98%  679K 6s\n260000K .......... .......... .......... .......... .......... 98%  943K 6s\n260050K .......... .......... .......... .......... .......... 98%  761K 6s\n260100K .......... .......... .......... .......... .......... 98% 1006K 5s\n260150K .......... .......... .......... .......... .......... 98%  775K 5s\n260200K .......... .......... .......... .......... .......... 98%  693K 5s\n260250K .......... .......... .......... .......... .......... 98% 1.11M 5s\n260300K .......... .......... .......... .......... .......... 98%  789K 5s\n260350K .......... .......... .......... .......... .......... 98%  861K 5s\n260400K .......... .......... .......... .......... .......... 98%  735K 5s\n260450K .......... .......... .......... .......... .......... 98%  682K 5s\n260500K .......... .......... .......... .......... .......... 98% 1.12M 5s\n260550K .......... .......... .......... .......... .......... 98%  993K 5s\n260600K .......... .......... .......... .......... .......... 98%  793K 5s\n260650K .......... .......... .......... .......... .......... 98%  686K 5s\n260700K .......... .......... .......... .......... .......... 98%  945K 5s\n260750K .......... .......... .......... .......... .......... 98%  743K 5s\n260800K .......... .......... .......... .......... .......... 98%  996K 5s\n260850K .......... .......... .......... .......... .......... 98%  793K 5s\n260900K .......... .......... .......... .......... .......... 98%  687K 4s\n260950K .......... .......... .......... .......... .......... 98% 1.11M 4s\n261000K .......... .......... .......... .......... .......... 98%  797K 4s\n261050K .......... .......... .......... .......... .......... 98%  875K 4s\n261100K .......... .......... .......... .......... .......... 98%  736K 4s\n261150K .......... .......... .......... .......... .......... 98%  651K 4s\n261200K .......... .......... .......... .......... .......... 98% 1.17M 4s\n261250K .......... .......... .......... .......... .......... 98% 1000K 4s\n261300K .......... .......... .......... .......... .......... 98%  799K 4s\n261350K .......... .......... .......... .......... .......... 98%  659K 4s\n261400K .......... .......... .......... .......... .......... 98% 1.01M 4s\n261450K .......... .......... .......... .......... .......... 98%  714K 4s\n261500K .......... .......... .......... .......... .......... 98% 1001K 4s\n261550K .......... .......... .......... .......... .......... 98%  815K 4s\n261600K .......... .......... .......... .......... .......... 98%  651K 4s\n261650K .......... .......... .......... .......... .......... 98% 1.17M 4s\n261700K .......... .......... .......... .......... .......... 98%  816K 4s\n261750K .......... .......... .......... .......... .......... 98%  879K 3s\n261800K .......... .......... .......... .......... .......... 98%  710K 3s\n261850K .......... .......... .......... .......... .......... 98%  658K 3s\n261900K .......... .......... .......... .......... .......... 98% 1.13M 3s\n261950K .......... .......... .......... .......... .......... 99% 1017K 3s\n262000K .......... .......... .......... .......... .......... 99%  807K 3s\n262050K .......... .......... .......... .......... .......... 99%  653K 3s\n262100K .......... .......... .......... .......... .......... 99% 1.05M 3s\n262150K .......... .......... .......... .......... .......... 99%  692K 3s\n262200K .......... .......... .......... .......... .......... 99% 1015K 3s\n262250K .......... .......... .......... .......... .......... 99%  800K 3s\n262300K .......... .......... .......... .......... .......... 99%  664K 3s\n262350K .......... .......... .......... .......... .......... 99% 1.14M 3s\n262400K .......... .......... .......... .......... .......... 99%  822K 3s\n262450K .......... .......... .......... .......... .......... 99%  919K 3s\n262500K .......... .......... .......... .......... .......... 99%  688K 3s\n262550K .......... .......... .......... .......... .......... 99%  660K 2s\n262600K .......... .......... .......... .......... .......... 99% 1.15M 2s\n262650K .......... .......... .......... .......... .......... 99% 1023K 2s\n262700K .......... .......... .......... .......... .......... 99%  791K 2s\n262750K .......... .......... .......... .......... .......... 99%  652K 2s\n262800K .......... .......... .......... .......... .......... 99% 1.11M 2s\n262850K .......... .......... .......... .......... .......... 99%  673K 2s\n262900K .......... .......... .......... .......... .......... 99% 1.00M 2s\n262950K .......... .......... .......... .......... .......... 99%  789K 2s\n263000K .......... .......... .......... .......... .......... 99%  666K 2s\n263050K .......... .......... .......... .......... .......... 99% 1.14M 2s\n263100K .......... .......... .......... .......... .......... 99%  837K 2s\n263150K .......... .......... .......... .......... .......... 99%  804K 2s\n263200K .......... .......... .......... .......... .......... 99%  764K 2s\n263250K .......... .......... .......... .......... .......... 99%  661K 2s\n263300K .......... .......... .......... .......... .......... 99% 1.12M 2s\n263350K .......... .......... .......... .......... .......... 99% 1.01M 2s\n263400K .......... .......... .......... .......... .......... 99%  800K 1s\n263450K .......... .......... .......... .......... .......... 99%  652K 1s\n263500K .......... .......... .......... .......... .......... 99% 1.13M 1s\n263550K .......... .......... .......... .......... .......... 99%  655K 1s\n263600K .......... .......... .......... .......... .......... 99% 1.02M 1s\n263650K .......... .......... .......... .......... .......... 99%  799K 1s\n263700K .......... .......... .......... .......... .......... 99%  670K 1s\n263750K .......... .......... .......... .......... .......... 99% 1.08M 1s\n263800K .......... .......... .......... .......... .......... 99%  861K 1s\n263850K .......... .......... .......... .......... .......... 99%  810K 1s\n263900K .......... .......... .......... .......... .......... 99%  752K 1s\n263950K .......... .......... .......... .......... .......... 99%  668K 1s\n264000K .......... .......... .......... .......... .......... 99% 1.08M 1s\n264050K .......... .......... .......... .......... .......... 99% 1.04M 1s\n264100K .......... .......... .......... .......... .......... 99%  793K 1s\n264150K .......... .......... .......... .......... .......... 99%  654K 1s\n264200K .......... .......... .......... .......... .......... 99% 1.12M 0s\n264250K .......... .......... .......... .......... .......... 99%  669K 0s\n264300K .......... .......... .......... .......... .......... 99% 1019K 0s\n264350K .......... .......... .......... .......... .......... 99%  807K 0s\n264400K .......... .......... .......... .......... .......... 99%  656K 0s\n264450K .......... .......... .......... .......... .......... 99% 1.11M 0s\n264500K .......... .......... .......... .......... .......... 99%  876K 0s\n264550K .......... .......... .......... .......... .......... 99%  751K 0s\n264600K .......... .......... .......... .......... ....      100%  709K=5m21s\n\n2020-01-23 17:52:22 (825 KB/s) - \u0027Pfam-A.hmm.gz\u0027 saved [270995712/270995712]\n\n* Running CMD: /usr/local/bin/PFAM_dat_parser.pl Pfam-A.hmm.gz\n\r[100]  \r[200]  \r[300]  \r[400]  \r[500]  \r[600]  \r[700]  \r[800]  \r[900]  \r[1000]  \r[1100]  \r[1200]  \r[1300]  \r[1400]  \r[1500]  \r[1600]  \r[1700]  \r[1800]  \r[1900]  \r[2000]  \r[2100]  \r[2200]  \r[2300]  \r[2400]  \r[2500]  \r[2600]  \r[2700]  \r[2800]  \r[2900]  \r[3000]  \r[3100]  \r[3200]  \r[3300]  \r[3400]  \r[3500]  \r[3600]  \r[3700]  \r[3800]  \r[3900]  \r[4000]  \r[4100]  \r[4200]  \r[4300]  \r[4400]  \r[4500]  \r[4600]  \r[4700]  \r[4800]  \r[4900]  \r[5000]  \r[5100]  \r[5200]  \r[5300]  \r[5400]  \r[5500]  \r[5600]  \r[5700]  \r[5800]  \r[5900]  \r[6000]  \r[6100]  \r[6200]  \r[6300]  \r[6400]  \r[6500]  \r[6600]  \r[6700]  \r[6800]  \r[6900]  \r[7000]  \r[7100]  \r[7200]  \r[7300]  \r[7400]  \r[7500]  \r[7600]  \r[7700]  \r[7800]  \r[7900]  \r[8000]  \r[8100]  \r[8200]  \r[8300]  \r[8400]  \r[8500]  \r[8600]  \r[8700]  \r[8800]  \r[8900]  \r[9000]  \r[9100]  \r[9200]  \r[9300]  \r[9400]  \r[9500]  \r[9600]  \r[9700]  \r[9800]  \r[9900]  \r[10000]  \r[10100]  \r[10200]  \r[10300]  \r[10400]  \r[10500]  \r[10600]  \r[10700]  \r[10800]  \r[10900]  \r[11000]  \r[11100]  \r[11200]  \r[11300]  \r[11400]  \r[11500]  \r[11600]  \r[11700]  \r[11800]  \r[11900]  \r[12000]  \r[12100]  \r[12200]  \r[12300]  \r[12400]  \r[12500]  \r[12600]  \r[12700]  \r[12800]  \r[12900]  \r[13000]  \r[13100]  \r[13200]  \r[13300]  \r[13400]  \r[13500]  \r[13600]  \r[13700]  \r[13800]  \r[13900]  \r[14000]  \r[14100]  \r[14200]  \r[14300]  \r[14400]  \r[14500]  \r[14600]  \r[14700]  \r[14800]  \r[14900]  \r[15000]  \r[15100]  \r[15200]  \r[15300]  \r[15400]  \r[15500]  \r[15600]  \r[15700]  \r[15800]  \r[15900]  \r[16000]  \r[16100]  \r[16200]  \r[16300]  \r[16400]  \r[16500]  \r[16600]  \r[16700]  \r[16800]  \r[16900]  \r[17000]  \r[17100]  \r[17200]  \r[17300]  \r[17400]  \r[17500]  \r[17600]  \r[17700]  \r[17800]  \r[17900]  * Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam Pfam-A.hmm.gz.pfam_sqlite_bulk_load\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import Pfam-A.hmm.gz.pfam_sqlite_bulk_load PFAMreference\" | sqlite3 Trinotate.sqlite\n* Running CMD: wget \"http://www.geneontology.org/external2go/pfam2go\" \n--2020-01-23 17:52:34--  http://www.geneontology.org/external2go/pfam2go\nResolving www.geneontology.org... 107.23.42.138\nConnecting to www.geneontology.org|107.23.42.138|:80... connected.\nHTTP request sent, awaiting response... 301 Moved Permanently\nLocation: http://geneontology.org/external2go/pfam2go [following]\n--2020-01-23 17:52:34--  http://geneontology.org/external2go/pfam2go\nResolving geneontology.org... 107.23.42.138\nReusing existing connection to www.geneontology.org:80.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://purl.obolibrary.org/obo/go/external2go/pfam2go [following]\n--2020-01-23 17:52:34--  http://purl.obolibrary.org/obo/go/external2go/pfam2go\nResolving purl.obolibrary.org... 52.3.123.63\nConnecting to purl.obolibrary.org|52.3.123.63|:80... connected.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://current.geneontology.org/ontology/external2go/pfam2go [following]\n--2020-01-23 17:52:34--  http://current.geneontology.org/ontology/external2go/pfam2go\nResolving current.geneontology.org... 99.84.222.32, 99.84.222.131, 99.84.222.60, ...\nConnecting to current.geneontology.org|99.84.222.32|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 699985 (684K) [text/plain]\nSaving to: \u0027pfam2go\u0027\n\n     0K .......... .......... .......... .......... ..........  7% 80.3M 0s\n    50K .......... .......... .......... .......... .......... 14% 35.9M 0s\n   100K .......... .......... .......... .......... .......... 21%  140M 0s\n   150K .......... .......... .......... .......... .......... 29% 52.1M 0s\n   200K .......... .......... .......... .......... .......... 36% 60.0M 0s\n   250K .......... .......... .......... .......... .......... 43% 51.2M 0s\n   300K .......... .......... .......... .......... .......... 51% 51.4M 0s\n   350K .......... .......... .......... .......... .......... 58% 49.8M 0s\n   400K .......... .......... .......... .......... .......... 65% 42.5M 0s\n   450K .......... .......... .......... .......... .......... 73% 68.2M 0s\n   500K .......... .......... .......... .......... .......... 80%  136M 0s\n   550K .......... .......... .......... .......... .......... 87%  103M 0s\n   600K .......... .......... .......... .......... .......... 95%  140M 0s\n   650K .......... .......... .......... ...                  100%  144M=0.01s\n\n2020-01-23 17:52:35 (65.3 MB/s) - \u0027pfam2go\u0027 saved [699985/699985]\n\n* Running CMD: /usr/local/bin/PFAMtoGoParser.pl pfam2go \u003e pfam2go.tab\n* Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam2go pfam2go.tab\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import pfam2go.tab pfam2go\" | sqlite3 Trinotate.sqlite\nCMD: TrinotateSeqLoader.pl --sqlite Trinotate.sqlite --gene_trans_map /tmp/tmp8_jayk4s/files/7/1/3/dataset_71390e14-9a5e-49e1-874c-02874a8b4104.dat --transcript_fasta /tmp/tmp8_jayk4s/files/c/e/a/dataset_cea7f91f-4335-428f-8d52-43edc4405b85.dat --transdecoder_pep /tmp/tmp8_jayk4s/files/d/6/8/dataset_d6845fdc-8f5d-412c-84d7-52bc01f5e8a6.dat --bulk_load\n-parsing gene/trans map file.... done.\n-loading Transcripts.\n\r[100]   \r[200]   \ndone.\n-loading ORFs.\n\ndone.\n\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.Transcript.bulk_load Transcript\" | sqlite3 Trinotate.sqlite\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.ORF.bulk_load ORF\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/c/4/d/dataset_c4d9eccc-17ca-49fa-8f3e-5212d209f39d.dat --prog blastp --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.83 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/6/3/d/dataset_63dd4a82-239d-41c3-9799-dbb9729f43b8.dat --prog blastx --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.88 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_PFAM_loader.pl --sqlite Trinotate.sqlite --pfam /tmp/tmp8_jayk4s/files/9/5/b/dataset_95b35c7c-d93b-496a-8a2e-119b17137f3d.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.pfam_bulk_load.93 HMMERDbase\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_TMHMM_loader.pl --sqlite Trinotate.sqlite --tmhmm /tmp/tmp8_jayk4s/files/7/8/5/dataset_785afc7b-a6ba-4ce1-ae41-476f7aba932e.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.tmhmm_bulk_load.98 tmhmm\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_SIGNALP_loader.pl --sqlite Trinotate.sqlite --signalp /tmp/tmp8_jayk4s/files/3/e/b/dataset_3eb6d4c7-3a09-46c0-b663-2c866e25e46e.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.signalp_bulk_load.103 signalp\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_report_writer.pl --sqlite Trinotate.sqlite -E 1e-05 --pfam_cutoff DNC\n", "tool_stdout": "memory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\n", "update_time": "2020-01-23T17:52:39.206581", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmpv_6mofybtrinotate_annotation_report.xls v. /tmp/tmpwafakizmtrinotate_annotation_report.xls )\n--- local_file\n+++ history_data\n@@ -1,29 +1,29 @@\n #gene_id\ttranscript_id\tsprot_Top_BLASTX_hit\tRNAMMER\tprot_id\tprot_coords\tsprot_Top_BLASTP_hit\tPfam\tSignalP\tTmHMM\teggnog\tKegg\tgene_ontology_blast\tgene_ontology_pfam\ttranscript\tpeptide\n-TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1E8.02\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n+TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\tGO:0005829^cellular_component^cytosol`GO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0031593^molecular_function^polyubiquitin modification-dependent protein binding`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n TRINITY_DN549_c0_g1\tTRINITY_DN549_c0_g1_i1\tGAA1_SCHPO^GAA1_SCHPO^Q:2-1363,H:121-574^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN549_c0_g1_i1|m.608\t2-1387[+]\tGAA1_SCHPO^GAA1_SCHPO^Q:1-461,H:121-581^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\tExpAA=140.37^PredHel=6^Topology=i247-269o284-302i309-331o346-368i381-403o428-450i\t.\tKEGG:spo:SPAC1002.11`KO:K05289\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0042765^cellular_component^GPI-anchor transamidase complex`GO:0016255^biological_process^attachment of GPI anchor to protein\t.\t.\t.\n-TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005737^cellular_component^cytoplasm`GO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0061630^molecular_function^ubiquitin protein ligase activity`GO:0031625^molecular_function^ubiquitin protein ligase binding`GO:0007031^biological_process^peroxisome organization`GO:0070647^biological_process^protein modification by small protein conjugation or removal\t.\t.\t.\n+TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0007031^biological_process^peroxisome organization\t.\t.\t.\n TRINITY_DN540_c0_g1\tTRINITY_DN540_c0_g1_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:26-1567,H:2-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g1_i1|m.650\t350-1570[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-406,H:110-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.651\t300-1847[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-515,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.652\t1-393[+]\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n-TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n TRINITY_DN547_c0_g1\tTRINITY_DN547_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n********\n*SNIP *\n********\n TRINITY_DN5472_c0_g1\tTRINITY_DN5472_c0_g1_i1\tYF11_SCHPO^YF11_SCHPO^Q:2-94,H:69-99^100%ID^E:3e-14^RecName: Full=Putative uncharacterized protein C9E9.01;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC9E9.01\tGO:0016021^cellular_component^integral component of membrane\t.\t.\t.\n-TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n-TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005622^cellular_component^intracellular`GO:0005634^cellular_component^nucleus`GO:0047433^molecular_function^branched-chain-2-oxoacid decarboxylase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0000949^biological_process^aromatic amino acid family catabolic process to alcohol via Ehrlich pathway`GO:0006067^biological_process^ethanol metabolic process`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006559^biological_process^L-phenylalanine catabolic process`GO:0006090^biological_process^pyruvate metabolic process`GO:0006569^biological_process^tryptophan catabolic process\t.\t.\t.\n+TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07`KO:K10992\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0062110^biological_process^negative regulation of DNA recombinase disassembly`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n+TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016831^molecular_function^carboxy-lyase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006090^biological_process^pyruvate metabolic process\t.\t.\t.\n TRINITY_DN5486_c0_g1\tTRINITY_DN5486_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5468_c0_g1\tTRINITY_DN5468_c0_g1_i1\tCDC37_SCHPO^CDC37_SCHPO^Q:280-5,H:209-300^98.91%ID^E:6e-41^RecName: Full=Hsp90 co-chaperone Cdc37;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPBC9B6.10`KO:K09554\tGO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0051087^molecular_function^chaperone binding`GO:0031072^molecular_function^heat shock protein binding`GO:0019901^molecular_function^protein kinase binding`GO:0051082^molecular_function^unfolded protein binding`GO:0007049^biological_process^cell cycle`GO:0051301^biological_process^cell division`GO:0061077^biological_process^chaperone-mediated protein folding`GO:0006457^biological_process^protein folding`GO:0050821^biological_process^protein stabilization\t.\t.\t.\n-TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n+TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n TRINITY_DN5400_c0_g1\tTRINITY_DN5400_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5476_c0_g1\tTRINITY_DN5476_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5490_c0_g1\tTRINITY_DN5490_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n+TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015171^molecular_function^amino acid transmembrane transporter activity`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0003333^biological_process^amino acid transmembrane transport`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n TRINITY_DN5471_c0_g1\tTRINITY_DN5471_c0_g1_i1\tRRG1_SCHPO^RRG1_SCHPO^Q:214-101,H:266-303^100%ID^E:3e-21^RecName: Full=Protein-lysine N-methyltransferase rrg1 {ECO:0000250|UniProtKB:P38347, ECO:0000305};^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC338.11c\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016279^molecular_function^protein-lysine N-methyltransferase activity`GO:2000765^biological_process^regulation of cytoplasmic translation\t.\t.\t.\n-TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC1851.03`KO:K03115\tGO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0043433^biological_process^negative regulation of DNA-binding transcription factor activity`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0007165^biological_process^signal transduction\t.\t.\t.\n+TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGO:0005737^cellular_component^cytoplasm`GO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0019887^molecular_function^protein kinase regulator activity`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0030291^molecular_function^protein serine/threonine kinase inhibitor activity`GO:0006974^biological_process^cellular response to DNA damage stimulus`GO:0035690^biological_process^cellular response to drug`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0018105^biological_process^peptidyl-serine phosphorylation`GO:0006356^biological_process^regulation of transcription by RNA polymerase I`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN5498_c0_g1\tTRINITY_DN5498_c0_g1_i1\tSRP2_SCHPO^SRP2_SCHPO^Q:213-145,H:22-44^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`SRP2_SCHPO^SRP2_SCHPO^Q:91-56,H:44-55^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC16.02c`KO:K12893\tGO:0005634^cellular_component^nucleus`GO:0003729^molecular_function^mRNA binding`GO:0008143^molecular_function^poly(A) binding`GO:0001069^molecular_function^regulatory region RNA binding`GO:0045292^biological_process^mRNA cis splicing, via spliceosome\t.\t.\t.\n-TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n-TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8862\t1057-1950[-]\tRNG3_SCHPO^RNG3_SCHPO^Q:1-297,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.04c`KO:K21991\tGO:0005826^cellular_component^actomyosin contractile ring`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0005844^cellular_component^polysome`GO:0017022^molecular_function^myosin binding`GO:0034613^biological_process^cellular protein localization`GO:0000917^biological_process^division septum assembly`GO:1903475^biological_process^mitotic actomyosin contractile ring assembly\t.\t.\t.\n-TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n+TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005793^cellular_component^endoplasmic reticulum-Golgi intermediate compartment`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n", "History item  different than expected, difference (using diff):\n( /tmp/tmph_zou6jftrinotate_annotation_report.xls v. /tmp/tmpqfm_99v0trinotate_annotation_report.xls )\n--- local_file\n+++ history_data\n@@ -1,29 +1,29 @@\n #gene_id\ttranscript_id\tsprot_Top_BLASTX_hit\tRNAMMER\tprot_id\tprot_coords\tsprot_Top_BLASTP_hit\tPfam\tSignalP\tTmHMM\teggnog\tKegg\tgene_ontology_blast\tgene_ontology_pfam\ttranscript\tpeptide\n-TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1E8.02\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n+TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\tGO:0005829^cellular_component^cytosol`GO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0031593^molecular_function^polyubiquitin modification-dependent protein binding`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n TRINITY_DN549_c0_g1\tTRINITY_DN549_c0_g1_i1\tGAA1_SCHPO^GAA1_SCHPO^Q:2-1363,H:121-574^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN549_c0_g1_i1|m.608\t2-1387[+]\tGAA1_SCHPO^GAA1_SCHPO^Q:1-461,H:121-581^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\tExpAA=140.37^PredHel=6^Topology=i247-269o284-302i309-331o346-368i381-403o428-450i\t.\tKEGG:spo:SPAC1002.11`KO:K05289\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0042765^cellular_component^GPI-anchor transamidase complex`GO:0016255^biological_process^attachment of GPI anchor to protein\t.\t.\t.\n-TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005737^cellular_component^cytoplasm`GO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0061630^molecular_function^ubiquitin protein ligase activity`GO:0031625^molecular_function^ubiquitin protein ligase binding`GO:0007031^biological_process^peroxisome organization`GO:0070647^biological_process^protein modification by small protein conjugation or removal\t.\t.\t.\n+TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0007031^biological_process^peroxisome organization\t.\t.\t.\n TRINITY_DN540_c0_g1\tTRINITY_DN540_c0_g1_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:26-1567,H:2-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g1_i1|m.650\t350-1570[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-406,H:110-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.651\t300-1847[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-515,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.652\t1-393[+]\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n-TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n TRINITY_DN547_c0_g1\tTRINITY_DN547_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n********\n*SNIP *\n********\n TRINITY_DN5472_c0_g1\tTRINITY_DN5472_c0_g1_i1\tYF11_SCHPO^YF11_SCHPO^Q:2-94,H:69-99^100%ID^E:3e-14^RecName: Full=Putative uncharacterized protein C9E9.01;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC9E9.01\tGO:0016021^cellular_component^integral component of membrane\t.\t.\t.\n-TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n-TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005622^cellular_component^intracellular`GO:0005634^cellular_component^nucleus`GO:0047433^molecular_function^branched-chain-2-oxoacid decarboxylase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0000949^biological_process^aromatic amino acid family catabolic process to alcohol via Ehrlich pathway`GO:0006067^biological_process^ethanol metabolic process`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006559^biological_process^L-phenylalanine catabolic process`GO:0006090^biological_process^pyruvate metabolic process`GO:0006569^biological_process^tryptophan catabolic process\t.\t.\t.\n+TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07`KO:K10992\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0062110^biological_process^negative regulation of DNA recombinase disassembly`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n+TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016831^molecular_function^carboxy-lyase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006090^biological_process^pyruvate metabolic process\t.\t.\t.\n TRINITY_DN5486_c0_g1\tTRINITY_DN5486_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5468_c0_g1\tTRINITY_DN5468_c0_g1_i1\tCDC37_SCHPO^CDC37_SCHPO^Q:280-5,H:209-300^98.91%ID^E:6e-41^RecName: Full=Hsp90 co-chaperone Cdc37;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPBC9B6.10`KO:K09554\tGO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0051087^molecular_function^chaperone binding`GO:0031072^molecular_function^heat shock protein binding`GO:0019901^molecular_function^protein kinase binding`GO:0051082^molecular_function^unfolded protein binding`GO:0007049^biological_process^cell cycle`GO:0051301^biological_process^cell division`GO:0061077^biological_process^chaperone-mediated protein folding`GO:0006457^biological_process^protein folding`GO:0050821^biological_process^protein stabilization\t.\t.\t.\n-TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n+TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n TRINITY_DN5400_c0_g1\tTRINITY_DN5400_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5476_c0_g1\tTRINITY_DN5476_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5490_c0_g1\tTRINITY_DN5490_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n+TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015171^molecular_function^amino acid transmembrane transporter activity`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0003333^biological_process^amino acid transmembrane transport`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n TRINITY_DN5471_c0_g1\tTRINITY_DN5471_c0_g1_i1\tRRG1_SCHPO^RRG1_SCHPO^Q:214-101,H:266-303^100%ID^E:3e-21^RecName: Full=Protein-lysine N-methyltransferase rrg1 {ECO:0000250|UniProtKB:P38347, ECO:0000305};^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC338.11c\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016279^molecular_function^protein-lysine N-methyltransferase activity`GO:2000765^biological_process^regulation of cytoplasmic translation\t.\t.\t.\n-TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC1851.03`KO:K03115\tGO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0043433^biological_process^negative regulation of DNA-binding transcription factor activity`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0007165^biological_process^signal transduction\t.\t.\t.\n+TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGO:0005737^cellular_component^cytoplasm`GO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0019887^molecular_function^protein kinase regulator activity`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0030291^molecular_function^protein serine/threonine kinase inhibitor activity`GO:0006974^biological_process^cellular response to DNA damage stimulus`GO:0035690^biological_process^cellular response to drug`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0018105^biological_process^peptidyl-serine phosphorylation`GO:0006356^biological_process^regulation of transcription by RNA polymerase I`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN5498_c0_g1\tTRINITY_DN5498_c0_g1_i1\tSRP2_SCHPO^SRP2_SCHPO^Q:213-145,H:22-44^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`SRP2_SCHPO^SRP2_SCHPO^Q:91-56,H:44-55^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC16.02c`KO:K12893\tGO:0005634^cellular_component^nucleus`GO:0003729^molecular_function^mRNA binding`GO:0008143^molecular_function^poly(A) binding`GO:0001069^molecular_function^regulatory region RNA binding`GO:0045292^biological_process^mRNA cis splicing, via spliceosome\t.\t.\t.\n-TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n-TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8862\t1057-1950[-]\tRNG3_SCHPO^RNG3_SCHPO^Q:1-297,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.04c`KO:K21991\tGO:0005826^cellular_component^actomyosin contractile ring`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0005844^cellular_component^polysome`GO:0017022^molecular_function^myosin binding`GO:0034613^biological_process^cellular protein localization`GO:0000917^biological_process^division septum assembly`GO:1903475^biological_process^mitotic actomyosin contractile ring assembly\t.\t.\t.\n-TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n+TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005793^cellular_component^endoplasmic reticulum-Golgi intermediate compartment`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmpv_6mofybtrinotate_annotation_report.xls v. /tmp/tmpwafakizmtrinotate_annotation_report.xls )\n--- local_file\n+++ history_data\n@@ -1,29 +1,29 @@\n #gene_id\ttranscript_id\tsprot_Top_BLASTX_hit\tRNAMMER\tprot_id\tprot_coords\tsprot_Top_BLASTP_hit\tPfam\tSignalP\tTmHMM\teggnog\tKegg\tgene_ontology_blast\tgene_ontology_pfam\ttranscript\tpeptide\n-TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1E8.02\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n+TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\tGO:0005829^cellular_component^cytosol`GO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0031593^molecular_function^polyubiquitin modification-dependent protein binding`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n TRINITY_DN549_c0_g1\tTRINITY_DN549_c0_g1_i1\tGAA1_SCHPO^GAA1_SCHPO^Q:2-1363,H:121-574^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN549_c0_g1_i1|m.608\t2-1387[+]\tGAA1_SCHPO^GAA1_SCHPO^Q:1-461,H:121-581^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\tExpAA=140.37^PredHel=6^Topology=i247-269o284-302i309-331o346-368i381-403o428-450i\t.\tKEGG:spo:SPAC1002.11`KO:K05289\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0042765^cellular_component^GPI-anchor transamidase complex`GO:0016255^biological_process^attachment of GPI anchor to protein\t.\t.\t.\n-TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005737^cellular_component^cytoplasm`GO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0061630^molecular_function^ubiquitin protein ligase activity`GO:0031625^molecular_function^ubiquitin protein ligase binding`GO:0007031^biological_process^peroxisome organization`GO:0070647^biological_process^protein modification by small protein conjugation or removal\t.\t.\t.\n+TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0007031^biological_process^peroxisome organization\t.\t.\t.\n TRINITY_DN540_c0_g1\tTRINITY_DN540_c0_g1_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:26-1567,H:2-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g1_i1|m.650\t350-1570[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-406,H:110-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.651\t300-1847[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-515,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.652\t1-393[+]\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n-TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n TRINITY_DN547_c0_g1\tTRINITY_DN547_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n********\n*SNIP *\n********\n TRINITY_DN5472_c0_g1\tTRINITY_DN5472_c0_g1_i1\tYF11_SCHPO^YF11_SCHPO^Q:2-94,H:69-99^100%ID^E:3e-14^RecName: Full=Putative uncharacterized protein C9E9.01;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC9E9.01\tGO:0016021^cellular_component^integral component of membrane\t.\t.\t.\n-TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n-TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005622^cellular_component^intracellular`GO:0005634^cellular_component^nucleus`GO:0047433^molecular_function^branched-chain-2-oxoacid decarboxylase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0000949^biological_process^aromatic amino acid family catabolic process to alcohol via Ehrlich pathway`GO:0006067^biological_process^ethanol metabolic process`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006559^biological_process^L-phenylalanine catabolic process`GO:0006090^biological_process^pyruvate metabolic process`GO:0006569^biological_process^tryptophan catabolic process\t.\t.\t.\n+TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07`KO:K10992\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0062110^biological_process^negative regulation of DNA recombinase disassembly`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n+TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016831^molecular_function^carboxy-lyase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006090^biological_process^pyruvate metabolic process\t.\t.\t.\n TRINITY_DN5486_c0_g1\tTRINITY_DN5486_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5468_c0_g1\tTRINITY_DN5468_c0_g1_i1\tCDC37_SCHPO^CDC37_SCHPO^Q:280-5,H:209-300^98.91%ID^E:6e-41^RecName: Full=Hsp90 co-chaperone Cdc37;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPBC9B6.10`KO:K09554\tGO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0051087^molecular_function^chaperone binding`GO:0031072^molecular_function^heat shock protein binding`GO:0019901^molecular_function^protein kinase binding`GO:0051082^molecular_function^unfolded protein binding`GO:0007049^biological_process^cell cycle`GO:0051301^biological_process^cell division`GO:0061077^biological_process^chaperone-mediated protein folding`GO:0006457^biological_process^protein folding`GO:0050821^biological_process^protein stabilization\t.\t.\t.\n-TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n+TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n TRINITY_DN5400_c0_g1\tTRINITY_DN5400_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5476_c0_g1\tTRINITY_DN5476_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5490_c0_g1\tTRINITY_DN5490_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n+TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015171^molecular_function^amino acid transmembrane transporter activity`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0003333^biological_process^amino acid transmembrane transport`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n TRINITY_DN5471_c0_g1\tTRINITY_DN5471_c0_g1_i1\tRRG1_SCHPO^RRG1_SCHPO^Q:214-101,H:266-303^100%ID^E:3e-21^RecName: Full=Protein-lysine N-methyltransferase rrg1 {ECO:0000250|UniProtKB:P38347, ECO:0000305};^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC338.11c\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016279^molecular_function^protein-lysine N-methyltransferase activity`GO:2000765^biological_process^regulation of cytoplasmic translation\t.\t.\t.\n-TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC1851.03`KO:K03115\tGO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0043433^biological_process^negative regulation of DNA-binding transcription factor activity`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0007165^biological_process^signal transduction\t.\t.\t.\n+TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGO:0005737^cellular_component^cytoplasm`GO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0019887^molecular_function^protein kinase regulator activity`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0030291^molecular_function^protein serine/threonine kinase inhibitor activity`GO:0006974^biological_process^cellular response to DNA damage stimulus`GO:0035690^biological_process^cellular response to drug`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0018105^biological_process^peptidyl-serine phosphorylation`GO:0006356^biological_process^regulation of transcription by RNA polymerase I`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN5498_c0_g1\tTRINITY_DN5498_c0_g1_i1\tSRP2_SCHPO^SRP2_SCHPO^Q:213-145,H:22-44^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`SRP2_SCHPO^SRP2_SCHPO^Q:91-56,H:44-55^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC16.02c`KO:K12893\tGO:0005634^cellular_component^nucleus`GO:0003729^molecular_function^mRNA binding`GO:0008143^molecular_function^poly(A) binding`GO:0001069^molecular_function^regulatory region RNA binding`GO:0045292^biological_process^mRNA cis splicing, via spliceosome\t.\t.\t.\n-TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n-TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8862\t1057-1950[-]\tRNG3_SCHPO^RNG3_SCHPO^Q:1-297,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.04c`KO:K21991\tGO:0005826^cellular_component^actomyosin contractile ring`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0005844^cellular_component^polysome`GO:0017022^molecular_function^myosin binding`GO:0034613^biological_process^cellular protein localization`GO:0000917^biological_process^division septum assembly`GO:1903475^biological_process^mitotic actomyosin contractile ring assembly\t.\t.\t.\n-TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n+TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005793^cellular_component^endoplasmic reticulum-Golgi intermediate compartment`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n\nHistory item  different than expected, difference (using diff):\n( /tmp/tmph_zou6jftrinotate_annotation_report.xls v. /tmp/tmpqfm_99v0trinotate_annotation_report.xls )\n--- local_file\n+++ history_data\n@@ -1,29 +1,29 @@\n #gene_id\ttranscript_id\tsprot_Top_BLASTX_hit\tRNAMMER\tprot_id\tprot_coords\tsprot_Top_BLASTP_hit\tPfam\tSignalP\tTmHMM\teggnog\tKegg\tgene_ontology_blast\tgene_ontology_pfam\ttranscript\tpeptide\n-TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1E8.02\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n+TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\tGO:0005829^cellular_component^cytosol`GO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0031593^molecular_function^polyubiquitin modification-dependent protein binding`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\t.\t.\n TRINITY_DN549_c0_g1\tTRINITY_DN549_c0_g1_i1\tGAA1_SCHPO^GAA1_SCHPO^Q:2-1363,H:121-574^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN549_c0_g1_i1|m.608\t2-1387[+]\tGAA1_SCHPO^GAA1_SCHPO^Q:1-461,H:121-581^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\tExpAA=140.37^PredHel=6^Topology=i247-269o284-302i309-331o346-368i381-403o428-450i\t.\tKEGG:spo:SPAC1002.11`KO:K05289\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0042765^cellular_component^GPI-anchor transamidase complex`GO:0016255^biological_process^attachment of GPI anchor to protein\t.\t.\t.\n-TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005737^cellular_component^cytoplasm`GO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0061630^molecular_function^ubiquitin protein ligase activity`GO:0031625^molecular_function^ubiquitin protein ligase binding`GO:0007031^biological_process^peroxisome organization`GO:0070647^biological_process^protein modification by small protein conjugation or removal\t.\t.\t.\n+TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\t.\t.\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0007031^biological_process^peroxisome organization\t.\t.\t.\n TRINITY_DN540_c0_g1\tTRINITY_DN540_c0_g1_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:26-1567,H:2-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g1_i1|m.650\t350-1570[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-406,H:110-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.651\t300-1847[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-515,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.652\t1-393[+]\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n-TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n+TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\t.\t.\n TRINITY_DN547_c0_g1\tTRINITY_DN547_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n********\n*SNIP *\n********\n TRINITY_DN5472_c0_g1\tTRINITY_DN5472_c0_g1_i1\tYF11_SCHPO^YF11_SCHPO^Q:2-94,H:69-99^100%ID^E:3e-14^RecName: Full=Putative uncharacterized protein C9E9.01;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC9E9.01\tGO:0016021^cellular_component^integral component of membrane\t.\t.\t.\n-TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n-TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005622^cellular_component^intracellular`GO:0005634^cellular_component^nucleus`GO:0047433^molecular_function^branched-chain-2-oxoacid decarboxylase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0000949^biological_process^aromatic amino acid family catabolic process to alcohol via Ehrlich pathway`GO:0006067^biological_process^ethanol metabolic process`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006559^biological_process^L-phenylalanine catabolic process`GO:0006090^biological_process^pyruvate metabolic process`GO:0006569^biological_process^tryptophan catabolic process\t.\t.\t.\n+TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07`KO:K10992\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0062110^biological_process^negative regulation of DNA recombinase disassembly`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\t.\t.\n+TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016831^molecular_function^carboxy-lyase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006090^biological_process^pyruvate metabolic process\t.\t.\t.\n TRINITY_DN5486_c0_g1\tTRINITY_DN5486_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5468_c0_g1\tTRINITY_DN5468_c0_g1_i1\tCDC37_SCHPO^CDC37_SCHPO^Q:280-5,H:209-300^98.91%ID^E:6e-41^RecName: Full=Hsp90 co-chaperone Cdc37;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPBC9B6.10`KO:K09554\tGO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0051087^molecular_function^chaperone binding`GO:0031072^molecular_function^heat shock protein binding`GO:0019901^molecular_function^protein kinase binding`GO:0051082^molecular_function^unfolded protein binding`GO:0007049^biological_process^cell cycle`GO:0051301^biological_process^cell division`GO:0061077^biological_process^chaperone-mediated protein folding`GO:0006457^biological_process^protein folding`GO:0050821^biological_process^protein stabilization\t.\t.\t.\n-TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n+TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\t.\t.\n TRINITY_DN5400_c0_g1\tTRINITY_DN5400_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5476_c0_g1\tTRINITY_DN5476_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n TRINITY_DN5490_c0_g1\tTRINITY_DN5490_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\n-TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n+TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015171^molecular_function^amino acid transmembrane transporter activity`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0003333^biological_process^amino acid transmembrane transport`GO:0098713^biological_process^leucine import across plasma membrane\t.\t.\t.\n TRINITY_DN5471_c0_g1\tTRINITY_DN5471_c0_g1_i1\tRRG1_SCHPO^RRG1_SCHPO^Q:214-101,H:266-303^100%ID^E:3e-21^RecName: Full=Protein-lysine N-methyltransferase rrg1 {ECO:0000250|UniProtKB:P38347, ECO:0000305};^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC338.11c\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016279^molecular_function^protein-lysine N-methyltransferase activity`GO:2000765^biological_process^regulation of cytoplasmic translation\t.\t.\t.\n-TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC1851.03`KO:K03115\tGO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0043433^biological_process^negative regulation of DNA-binding transcription factor activity`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0007165^biological_process^signal transduction\t.\t.\t.\n+TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGO:0005737^cellular_component^cytoplasm`GO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0019887^molecular_function^protein kinase regulator activity`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0030291^molecular_function^protein serine/threonine kinase inhibitor activity`GO:0006974^biological_process^cellular response to DNA damage stimulus`GO:0035690^biological_process^cellular response to drug`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0018105^biological_process^peptidyl-serine phosphorylation`GO:0006356^biological_process^regulation of transcription by RNA polymerase I`GO:0007165^biological_process^signal transduction\t.\t.\t.\n TRINITY_DN5498_c0_g1\tTRINITY_DN5498_c0_g1_i1\tSRP2_SCHPO^SRP2_SCHPO^Q:213-145,H:22-44^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`SRP2_SCHPO^SRP2_SCHPO^Q:91-56,H:44-55^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC16.02c`KO:K12893\tGO:0005634^cellular_component^nucleus`GO:0003729^molecular_function^mRNA binding`GO:0008143^molecular_function^poly(A) binding`GO:0001069^molecular_function^regulatory region RNA binding`GO:0045292^biological_process^mRNA cis splicing, via spliceosome\t.\t.\t.\n-TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n-TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n+TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\t.\t.\n TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8862\t1057-1950[-]\tRNG3_SCHPO^RNG3_SCHPO^Q:1-297,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.04c`KO:K21991\tGO:0005826^cellular_component^actomyosin contractile ring`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0005844^cellular_component^polysome`GO:0017022^molecular_function^myosin binding`GO:0034613^biological_process^cellular protein localization`GO:0000917^biological_process^division septum assembly`GO:1903475^biological_process^mitotic actomyosin contractile ring assembly\t.\t.\t.\n-TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n+TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005793^cellular_component^endoplasmic reticulum-Golgi intermediate compartment`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\t.\t.\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 1693.7140629291534, "tool_id": "trinotate", "tool_version": "3.1.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_trinotate.test_tool_000000"}, {"data": {"inputs": {"blastp": {"id": "24a1da3a8e5693fe", "src": "hda"}, "blastx": {"id": "266b4c416078dc76", "src": "hda"}, "gene_trans_map": {"id": "c64bfc8445bed47e", "src": "hda"}, "hmmscan": {"id": "0b62b0b7271d572a", "src": "hda"}, "keep_sqlite": true, "report_option|incl_pep": true, "report_option|incl_trans": true, "transdecoder": {"id": "14f44a11533277c1", "src": "hda"}, "trinity": {"id": "9b36a3aa47b79bf2", "src": "hda"}}, "job": {"command_line": "Build_Trinotate_Boilerplate_SQLite_db.pl Trinotate \u0026\u0026  Trinotate Trinotate.sqlite init --gene_trans_map \u0027/tmp/tmp8_jayk4s/files/4/3/0/dataset_43030d4b-e681-405b-805e-1fe6e3b39b4b.dat\u0027 --transcript_fasta /tmp/tmp8_jayk4s/files/4/5/a/dataset_45aeb2f5-af12-4711-8868-26c6ec9d3ff4.dat --transdecoder_pep \u0027/tmp/tmp8_jayk4s/files/5/a/3/dataset_5a30da26-8da3-480f-89f5-5bf7088e8d11.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_swissprot_blastp \u0027/tmp/tmp8_jayk4s/files/6/a/2/dataset_6a23846f-e710-4ce6-a43b-dd22f87cacee.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_swissprot_blastx \u0027/tmp/tmp8_jayk4s/files/c/3/4/dataset_c3463f12-6a71-4d37-8519-d3bb250afd9b.dat\u0027 \u0026\u0026 Trinotate Trinotate.sqlite LOAD_pfam \u0027/tmp/tmp8_jayk4s/files/c/f/3/dataset_cf38b685-d33f-49f2-80d0-b0f9bde64b09.dat\u0027 \u0026\u0026  Trinotate Trinotate.sqlite report -E 1e-05 --pfam_cutoff DNC --incl_pep --incl_trans \u003e \u0027/tmp/tmp8_jayk4s/files/d/0/4/dataset_d0414029-af34-46b5-9295-0c5db8df99c0.dat\u0027", "command_version": "Error, cannot locate version database file. Be sure it\u0027s in your current directory. at /usr/local/bin/Trinotate line 53.\n", "create_time": "2020-01-23T17:58:21.169481", "exit_code": 0, "external_id": "47374", "galaxy_version": "20.01", "history_id": "5e6ad6ca0afeaf24", "id": "26f324d382d6d8c1", "inputs": {"blastp": {"id": "24a1da3a8e5693fe", "src": "hda", "uuid": "6a23846f-e710-4ce6-a43b-dd22f87cacee"}, "blastx": {"id": "266b4c416078dc76", "src": "hda", "uuid": "c3463f12-6a71-4d37-8519-d3bb250afd9b"}, "gene_trans_map": {"id": "c64bfc8445bed47e", "src": "hda", "uuid": "43030d4b-e681-405b-805e-1fe6e3b39b4b"}, "hmmscan": {"id": "0b62b0b7271d572a", "src": "hda", "uuid": "cf38b685-d33f-49f2-80d0-b0f9bde64b09"}, "transdecoder": {"id": "14f44a11533277c1", "src": "hda", "uuid": "5a30da26-8da3-480f-89f5-5bf7088e8d11"}, "trinity": {"id": "9b36a3aa47b79bf2", "src": "hda", "uuid": "45aeb2f5-af12-4711-8868-26c6ec9d3ff4"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"trinotate_out": {"id": "8543024575f6f30b", "src": "hda", "uuid": "d0414029-af34-46b5-9295-0c5db8df99c0"}, "trinotate_sqlite": {"id": "74219090cb2d0112", "src": "hda", "uuid": "e04ab60a-074e-4e2f-b695-6527385ade0a"}}, "params": {"__input_ext": "\"fasta\"", "chromInfo": "\"/tmp/tmp8_jayk4s/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "cond_download": "{\"__current_case__\": 1, \"select_download\": \"yes\"}", "dbkey": "\"?\"", "keep_sqlite": "\"true\"", "report_option": "{\"E\": \"1e-05\", \"incl_pep\": \"true\", \"incl_trans\": \"true\", \"pfam_cutoff\": \"DNC\"}", "signalp": "null", "tmhmm": "null"}, "state": "ok", "stderr": "* Running CMD: wget \"http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\"\n--2020-01-23 17:58:22--  http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\nResolving ftp.ebi.ac.uk... 193.62.193.138\nConnecting to ftp.ebi.ac.uk|193.62.193.138|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 582436772 (555M) [application/octet-stream]\nSaving to: \u0027uniprot_sprot.dat.gz\u0027\n\n     0K .......... .......... .......... .......... ..........  0%  167K 56m40s\n    50K .......... .......... .......... .......... ..........  0%  261K 46m28s\n   100K .......... .......... .......... .......... ..........  0%  551K 36m42s\n   150K .......... .......... .......... .......... ..........  0% 46.8M 27m35s\n   200K .......... .......... .......... .......... ..........  0%  534K 25m36s\n   250K .......... .......... .......... .......... ..........  0%  532K 24m18s\n   300K .......... .......... .......... .......... ..........  0%  537K 23m21s\n   350K .......... .......... .......... .......... ..........  0%  524K 22m42s\n   400K .......... .......... .......... .......... ..........  0% 57.7M 20m11s\n   450K .......... .......... .......... .......... ..........  0%  529K 19m57s\n   500K .......... .......... .......... .......... ..........  0%  500K 19m52s\n   550K .......... .......... .......... .......... ..........  0%  526K 19m42s\n   600K .......... .......... .......... .......... ..........  0%  551K 19m31s\n   650K .......... .......... .......... .......... ..........  0% 9.74M 18m11s\n   700K .......... .......... .......... .......... ..........  0%  524K 18m11s\n   750K .......... .......... .......... .......... ..........  0%  533K 18m9s\n   800K .......... .......... .......... .......... ..........  0%  523K 18m9s\n   850K .......... .......... .......... .......... ..........  0%  535K 18m7s\n   900K .......... .......... .......... .......... ..........  0% 14.8M 17m12s\n   950K .......... .......... .......... .......... ..........  0%  478K 17m19s\n  1000K .......... .......... .......... .......... ..........  0%  566K 17m18s\n  1050K .......... .......... .......... .......... ..........  0%  535K 17m19s\n  1100K .......... .......... .......... .......... ..........  0% 13.1M 16m35s\n  1150K .......... .......... .......... .......... ..........  0%  519K 16m39s\n  1200K .......... .......... .......... .......... ..........  0%  531K 16m42s\n  1250K .......... .......... .......... .......... ..........  0%  540K 16m44s\n  1300K .......... .......... .......... .......... ..........  0%  526K 16m46s\n  1350K .......... .......... .......... .......... ..........  0% 8.69M 16m13s\n  1400K .......... .......... .......... .......... ..........  0%  543K 16m15s\n  1450K .......... .......... .......... .......... ..........  0%  550K 16m17s\n  1500K .......... .......... .......... .......... ..........  0%  571K 16m17s\n  1550K .......... .......... .......... .......... ..........  0%  545K 16m19s\n  1600K .......... .......... .......... .......... ..........  0% 4.91M 15m53s\n  1650K .......... .......... .......... .......... ..........  0%  511K 15m57s\n  1700K .......... .......... .......... .......... ..........  0%  496K 16m3s\n  1750K .......... .......... .......... .......... ..........  0%  502K 16m7s\n  1800K .......... .......... .......... .......... ..........  0% 4.52M 15m44s\n  1850K .......... .......... .......... .......... ..........  0%  494K 15m50s\n  1900K .......... .......... .......... .......... ..........  0%  536K 15m52s\n  1950K .......... .......... .......... .......... ..........  0%  496K 15m57s\n  2000K .......... .......... .......... .......... ..........  0%  659K 15m55s\n  2050K .......... .......... .......... .......... ..........  0% 4.51M 15m35s\n  2100K .......... .......... .......... .......... ..........  0%  587K 15m35s\n  2150K .......... .......... .......... .......... ..........  0%  603K 15m35s\n  2200K .......... .......... .......... .......... ..........  0%  612K 15m35s\n  2250K .......... .......... .......... .......... ..........  0%  190M 15m15s\n  2300K .......... .......... .......... .......... ..........  0%  584K 15m16s\n  2350K .......... .......... .......... .......... ..........  0%  551K 15m18s\n  2400K .......... .......... .......... .......... ..........  0%  573K 15m19s\n  2450K .......... .......... .......... .......... ..........  0%  523K 15m23s\n  2500K .......... .......... .......... .......... ..........  0%  222M 15m4s\n  2550K .......... .......... .......... .......... ..........  0%  540K 15m7s\n  2600K .......... .......... .......... .......... ..........  0%  559K 15m9s\n  2650K .......... .......... .......... .......... ..........  0%  549K 15m11s\n  2700K .......... .......... .......... .......... ..........  0%  538K 15m14s\n  2750K .......... .......... .......... .......... ..........  0%  148M 14m57s\n  2800K .......... .......... .......... .......... ..........  0%  534K 15m0s\n  2850K .......... .......... .......... .......... ..........  0%  558K 15m2s\n  2900K .......... .......... .......... .......... ..........  0%  531K 15m5s\n  2950K .......... .......... .......... .......... ..........  0%  178M 14m50s\n  3000K .......... .......... .......... .......... ..........  0%  563K 14m51s\n  3050K .......... .......... .......... .......... ..........  0%  558K 14m53s\n  3100K .......... .......... .......... .......... ..........  0%  516K 14m56s\n  3150K .......... .......... .......... .......... ..........  0%  543K 14m59s\n  3200K .......... .......... .......... .......... ..........  0%  193M 14m45s\n  3250K .......... .......... .......... .......... ..........  0%  523K 14m48s\n  3300K .......... .......... .......... .......... ..........  0%  521K 14m51s\n  3350K .......... .......... .......... .......... ..........  0%  558K 14m52s\n  3400K .......... .......... .......... .......... ..........  0%  199M 14m39s\n  3450K .......... .......... .......... .......... ..........  0%  531K 14m42s\n  3500K .......... .......... .......... .......... ..........  0%  554K 14m44s\n  3550K .......... .......... .......... .......... ..........  0%  537K 14m46s\n  3600K .......... .......... .......... .......... ..........  0%  540K 14m48s\n  3650K .......... .......... .......... .......... ..........  0%  165M 14m36s\n  3700K .......... .......... .......... .......... ..........  0%  574K 14m37s\n  3750K .......... .......... .......... .......... ..........  0%  574K 14m39s\n  3800K .......... .......... .......... .......... ..........  0%  532K 14m41s\n  3850K .......... .......... .......... .......... ..........  0% 30.8M 14m30s\n  3900K .......... .......... .......... .......... ..........  0%  572K 14m31s\n  3950K .......... .......... .......... .......... ..........  0%  599K 14m32s\n  4000K .......... .......... .......... .......... ..........  0%  568K 14m34s\n  4050K .......... .......... .......... .......... ..........  0%  585K 14m35s\n  4100K .......... .......... .......... .......... ..........  0%  236M 14m24s\n  4150K .......... .......... .......... .......... ..........  0%  602K 14m25s\n  4200K .......... .......... .......... .......... ..........  0%  515K 14m28s\n  4250K .......... .......... .......... .......... ..........  0%  580K 14m29s\n  4300K .......... .......... .......... .......... ..........  0%  578K 14m30s\n  4350K .......... .......... .......... .......... ..........  0% 52.4M 14m20s\n  4400K .......... .......... .......... .......... ..........  0%  554K 14m22s\n  4450K .......... .......... .......... .......... ..........  0%  587K 14m23s\n  4500K .......... .......... .......... .......... ..........  0%  585K 14m24s\n  4550K .......... .......... .......... .......... ..........  0% 26.9M 14m15s\n  4600K .......... .......... .......... .......... ..........  0%  597K 14m15s\n  4650K .......... .......... .......... .......... ..........  0%  563K 14m17s\n  4700K .......... .......... .......... .......... ..........  0%  579K 14m18s\n  4750K .......... .......... .......... .......... ..........  0%  567K 14m19s\n  4800K .......... .......... .......... .......... ..........  0% 28.7M 14m11s\n  4850K .......... .......... .......... .......... ..........  0%  564K 14m12s\n  4900K .......... .......... .......... .......... ..........  0%  551K 14m14s\n  4950K .......... .......... .......... .......... ..........  0%  544K 14m16s\n  5000K .......... .......... .......... .......... ..........  0%  550K 14m17s\n  5050K .......... .......... .......... .......... ..........  0% 11.4M 14m9s\n  5100K .......... .......... .......... .......... ..........  0%  565K 14m10s\n  5150K .......... .......... .......... .......... ..........  0%  537K 14m12s\n  5200K .......... .......... .......... .......... ..........  0%  563K 14m14s\n  5250K .......... .......... .......... .......... ..........  0%  568K 14m15s\n  5300K .......... .......... .......... .......... ..........  0% 12.3M 14m7s\n  5350K .......... .......... .......... .......... ..........  0%  594K 14m8s\n  5400K .......... .......... .......... .......... ..........  0%  615K 14m9s\n  5450K .......... .......... .......... .......... ..........  0%  581K 14m10s\n  5500K .......... .......... .......... .......... ..........  0%  610K 14m10s\n  5550K .......... .......... .......... .......... ..........  0% 8.07M 14m3s\n  5600K .......... .......... .......... .......... ..........  0%  676K 14m3s\n  5650K .......... .......... .......... .......... ..........  1%  594K 14m4s\n  5700K .......... .......... .......... .......... ..........  1%  613K 14m4s\n  5750K .......... .......... .......... .......... ..........  1%  620K 14m5s\n  5800K .......... .......... .......... .......... ..........  1% 10.9M 13m58s\n  5850K .......... .......... .......... .......... ..........  1%  651K 13m58s\n  5900K .......... .......... .......... .......... ..........  1%  653K 13m58s\n  5950K .......... .......... .......... .......... ..........  1%  647K 13m59s\n  6000K .......... .......... .......... .......... ..........  1%  623K 13m59s\n  6050K .......... .......... .......... .......... ..........  1% 43.2M 13m52s\n  6100K .......... .......... .......... .......... ..........  1%  630K 13m53s\n  6150K .......... .......... .......... .......... ..........  1%  619K 13m53s\n  6200K .......... .......... .......... .......... ..........  1%  564K 13m54s\n  6250K .......... .......... .......... .......... ..........  1%  584K 13m55s\n  6300K .......... .......... .......... .......... ..........  1%  199M 13m49s\n  6350K .......... .......... .......... .......... ..........  1%  544K 13m50s\n  6400K .......... .......... .......... .......... ..........  1%  550K 13m52s\n  6450K .......... .......... .......... .......... ..........  1%  548K 13m53s\n  6500K .......... .......... .......... .......... ..........  1%  241M 13m47s\n  6550K .......... .......... .......... .......... ..........  1%  522K 13m48s\n  6600K .......... .......... .......... .......... ..........  1%  576K 13m50s\n  6650K .......... .......... .......... .......... ..........  1%  579K 13m50s\n  6700K .......... .......... .......... .......... ..........  1%  573K 13m52s\n  6750K .......... .......... .......... .......... ..........  1%  122M 13m45s\n  6800K .......... .......... .......... .......... ..........  1%  567K 13m47s\n  6850K .......... .......... .......... .......... ..........  1%  567K 13m48s\n  6900K .......... .......... .......... .......... ..........  1%  570K 13m49s\n  6950K .......... .......... .......... .......... ..........  1%  599K 13m49s\n  7000K .......... .......... .......... .......... ..........  1% 17.7M 13m44s\n  7050K .......... .......... .......... .......... ..........  1%  599K 13m44s\n  7100K .......... .......... .......... .......... ..........  1%  563K 13m46s\n  7150K .......... .......... .......... .......... ..........  1%  586K 13m46s\n  7200K .......... .......... .......... .......... ..........  1%  136M 13m41s\n  7250K .......... .......... .......... .......... ..........  1%  620K 13m41s\n  7300K .......... .......... .......... .......... ..........  1%  634K 13m42s\n  7350K .......... .......... .......... .......... ..........  1%  629K 13m42s\n  7400K .......... .......... .......... .......... ..........  1%  606K 13m43s\n  7450K .......... .......... .......... .......... ..........  1%  234M 13m37s\n  7500K .......... .......... .......... .......... ..........  1%  632K 13m37s\n  7550K .......... .......... .......... .......... ..........  1%  636K 13m38s\n  7600K .......... .......... .......... .......... ..........  1%  640K 13m38s\n  7650K .......... .......... .......... .......... ..........  1%  193M 13m33s\n  7700K .......... .......... .......... .......... ..........  1%  658K 13m33s\n  7750K .......... .......... .......... .......... ..........  1%  657K 13m33s\n  7800K .......... .......... .......... .......... ..........  1%  635K 13m33s\n  7850K .......... .......... .......... .......... ..........  1%  660K 13m34s\n  7900K .......... .......... .......... .......... ..........  1%  160M 13m28s\n  7950K .......... .......... .......... .......... ..........  1%  610K 13m29s\n  8000K .......... .......... .......... .......... ..........  1%  656K 13m29s\n  8050K .......... .......... .......... .......... ..........  1%  659K 13m29s\n  8100K .......... .......... .......... .......... ..........  1% 53.7M 13m25s\n  8150K .......... .......... .......... .......... ..........  1%  639K 13m25s\n  8200K .......... .......... .......... .......... ..........  1%  628K 13m25s\n  8250K .......... .......... .......... .......... ..........  1%  618K 13m26s\n  8300K .......... .......... .......... .......... ..........  1%  605K 13m27s\n  8350K .......... .......... .......... .......... ..........  1% 82.1M 13m22s\n  8400K .......... .......... .......... .......... ..........  1%  656K 13m22s\n  8450K .......... .......... .......... .......... ..........  1%  657K 13m22s\n  8500K .......... .......... .......... .......... ..........  1%  667K 13m22s\n  8550K .......... .......... .......... .......... ..........  1%  651K 13m23s\n  8600K .......... .......... .......... .......... ..........  1% 35.1M 13m18s\n  8650K .......... .......... .......... .......... ..........  1%  655K 13m18s\n  8700K .......... .......... .......... .......... ..........  1%  606K 13m19s\n  8750K .......... .......... .......... .......... ..........  1%  657K 13m19s\n  8800K .......... .......... .......... .......... ..........  1% 17.8M 13m15s\n  8850K .......... .......... .......... .......... ..........  1%  646K 13m15s\n  8900K .......... .......... .......... .......... ..........  1%  608K 13m16s\n  8950K .......... .......... .......... .......... ..........  1%  631K 13m16s\n  9000K .......... .......... .......... .......... ..........  1%  664K 13m16s\n  9050K .......... .......... .......... .......... ..........  1% 15.7M 13m12s\n  9100K .......... .......... .......... .......... ..........  1%  618K 13m13s\n  9150K .......... .......... .......... .......... ..........  1%  663K 13m13s\n  9200K .......... .......... .......... .......... ..........  1%  608K 13m13s\n  9250K .......... .......... .......... .......... ..........  1%  625K 13m14s\n  9300K .......... .......... .......... .......... ..........  1% 39.7M 13m10s\n  9350K .......... .......... .......... .......... ..........  1%  630K 13m10s\n  9400K .......... .......... .......... .......... ..........  1%  633K 13m11s\n  9450K .......... .......... .......... .......... ..........  1%  639K 13m11s\n  9500K .......... .......... .......... .......... ..........  1%  144M 13m7s\n  9550K .......... .......... .......... .......... ..........  1%  642K 13m7s\n  9600K .......... .......... .......... .......... ..........  1%  613K 13m8s\n  9650K .......... .......... .......... .......... ..........  1%  605K 13m8s\n  9700K .......... .......... .......... .......... ..........  1%  617K 13m9s\n  9750K .......... .......... .......... .......... ..........  1% 83.5M 13m5s\n  9800K .......... .......... .......... .......... ..........  1%  650K 13m5s\n  9850K .......... .......... .......... .......... ..........  1%  656K 13m5s\n  9900K .......... .......... .......... .......... ..........  1%  661K 13m6s\n \n..\n... .......... .......... 97% 1000K 7s\n258500K .......... .......... .......... .......... .......... 97%  650K 7s\n258550K .......... .......... .......... .......... .......... 97%  724K 7s\n258600K .......... .......... .......... .......... .......... 97% 1.41M 7s\n258650K .......... .......... .......... .......... .......... 97%  724K 7s\n258700K .......... .......... .......... .......... .......... 97%  862K 7s\n258750K .......... .......... .......... .......... .......... 97%  652K 7s\n258800K .......... .......... .......... .......... .......... 97%  789K 7s\n258850K .......... .......... .......... .......... .......... 97% 1.24M 7s\n258900K .......... .......... .......... .......... .......... 97%  779K 7s\n258950K .......... .......... .......... .......... .......... 97%  784K 7s\n259000K .......... .......... .......... .......... .......... 97%  687K 7s\n259050K .......... .......... .......... .......... .......... 97% 1.58M 7s\n259100K .......... .......... .......... .......... .......... 97%  665K 7s\n259150K .......... .......... .......... .......... .......... 97%  968K 7s\n259200K .......... .......... .......... .......... .......... 97%  655K 7s\n259250K .......... .......... .......... .......... .......... 97%  742K 6s\n259300K .......... .......... .......... .......... .......... 97% 1.38M 6s\n259350K .......... .......... .......... .......... .......... 98%  739K 6s\n259400K .......... .......... .......... .......... .......... 98%  826K 6s\n259450K .......... .......... .......... .......... .......... 98%  655K 6s\n259500K .......... .......... .......... .......... .......... 98%  818K 6s\n259550K .......... .......... .......... .......... .......... 98% 1.18M 6s\n259600K .......... .......... .......... .......... .......... 98%  797K 6s\n259650K .......... .......... .......... .......... .......... 98%  764K 6s\n259700K .......... .......... .......... .......... .......... 98%  714K 6s\n259750K .......... .......... .......... .......... .......... 98% 1.53M 6s\n259800K .......... .......... .......... .......... .......... 98%  675K 6s\n259850K .......... .......... .......... .......... .......... 98%  921K 6s\n259900K .......... .......... .......... .......... .......... 98%  651K 6s\n259950K .......... .......... .......... .......... .......... 98%  766K 6s\n260000K .......... .......... .......... .......... .......... 98% 1.32M 6s\n260050K .......... .......... .......... .......... .......... 98%  755K 5s\n260100K .......... .......... .......... .......... .......... 98%  806K 5s\n260150K .......... .......... .......... .......... .......... 98%  650K 5s\n260200K .......... .......... .......... .......... .......... 98%  862K 5s\n260250K .......... .......... .......... .......... .......... 98% 1.12M 5s\n260300K .......... .......... .......... .......... .......... 98%  814K 5s\n260350K .......... .......... .......... .......... .......... 98%  737K 5s\n260400K .......... .......... .......... .......... .......... 98%  691K 5s\n260450K .......... .......... .......... .......... .......... 98% 1.69M 5s\n260500K .......... .......... .......... .......... .......... 98%  698K 5s\n260550K .......... .......... .......... .......... .......... 98%  858K 5s\n260600K .......... .......... .......... .......... .......... 98%  650K 5s\n260650K .......... .......... .......... .......... .......... 98%  808K 5s\n260700K .......... .......... .......... .......... .......... 98% 1.27M 5s\n260750K .......... .......... .......... .......... .......... 98%  766K 5s\n260800K .......... .......... .......... .......... .......... 98%  774K 5s\n260850K .......... .......... .......... .......... .......... 98%  651K 5s\n260900K .......... .......... .......... .......... .......... 98%  922K 4s\n260950K .......... .......... .......... .......... .......... 98% 1.08M 4s\n261000K .......... .......... .......... .......... .......... 98%  828K 4s\n261050K .......... .......... .......... .......... .......... 98%  714K 4s\n261100K .......... .......... .......... .......... .......... 98%  719K 4s\n261150K .......... .......... .......... .......... .......... 98% 1.57M 4s\n261200K .......... .......... .......... .......... .......... 98%  726K 4s\n261250K .......... .......... .......... .......... .......... 98%  822K 4s\n261300K .......... .......... .......... .......... .......... 98%  654K 4s\n261350K .......... .......... .......... .......... .......... 98%  820K 4s\n261400K .......... .......... .......... .......... .......... 98% 1.23M 4s\n261450K .......... .......... .......... .......... .......... 98%  777K 4s\n261500K .......... .......... .......... .......... .......... 98%  764K 4s\n261550K .......... .......... .......... .......... .......... 98%  652K 4s\n261600K .......... .......... .......... .......... .......... 98%  949K 4s\n261650K .......... .......... .......... .......... .......... 98% 1.02M 4s\n261700K .......... .......... .......... .......... .......... 98%  853K 3s\n261750K .......... .......... .......... .......... .......... 98%  707K 3s\n261800K .......... .......... .......... .......... .......... 98%  726K 3s\n261850K .......... .......... .......... .......... .......... 98% 1.53M 3s\n261900K .......... .......... .......... .......... .......... 98%  740K 3s\n261950K .......... .......... .......... .......... .......... 99%  808K 3s\n262000K .......... .......... .......... .......... .......... 99%  655K 3s\n262050K .......... .......... .......... .......... .......... 99%  823K 3s\n262100K .......... .......... .......... .......... .......... 99% 1.22M 3s\n262150K .......... .......... .......... .......... .......... 99%  788K 3s\n262200K .......... .......... .......... .......... .......... 99%  755K 3s\n262250K .......... .......... .......... .......... .......... 99%  651K 3s\n262300K .......... .......... .......... .......... .......... 99%  969K 3s\n262350K .......... .......... .......... .......... .......... 99% 1.01M 3s\n262400K .......... .......... .......... .......... .......... 99%  865K 3s\n262450K .......... .......... .......... .......... .......... 99%  695K 3s\n262500K .......... .......... .......... .......... .......... 99%  744K 3s\n262550K .......... .......... .......... .......... .......... 99% 1.48M 2s\n262600K .......... .......... .......... .......... .......... 99%  746K 2s\n262650K .......... .......... .......... .......... .......... 99%  798K 2s\n262700K .......... .......... .......... .......... .......... 99%  653K 2s\n262750K .......... .......... .......... .......... .......... 99%  838K 2s\n262800K .......... .......... .......... .......... .......... 99% 1.22M 2s\n262850K .......... .......... .......... .......... .......... 99%  772K 2s\n262900K .......... .......... .......... .......... .......... 99%  761K 2s\n262950K .......... .......... .......... .......... .......... 99%  653K 2s\n263000K .......... .......... .......... .......... .......... 99%  980K 2s\n263050K .......... .......... .......... .......... .......... 99% 1.01M 2s\n263100K .......... .......... .......... .......... .......... 99%  858K 2s\n263150K .......... .......... .......... .......... .......... 99%  689K 2s\n263200K .......... .......... .......... .......... .......... 99%  757K 2s\n263250K .......... .......... .......... .......... .......... 99% 1.47M 2s\n263300K .......... .......... .......... .......... .......... 99%  747K 2s\n263350K .......... .......... .......... .......... .......... 99%  788K 2s\n263400K .......... .......... .......... .......... .......... 99%  653K 1s\n263450K .......... .......... .......... .......... .......... 99%  834K 1s\n263500K .......... .......... .......... .......... .......... 99% 1.23M 1s\n263550K .......... .......... .......... .......... .......... 99%  764K 1s\n263600K .......... .......... .......... .......... .......... 99%  762K 1s\n263650K .......... .......... .......... .......... .......... 99%  655K 1s\n263700K .......... .......... .......... .......... .......... 99% 1014K 1s\n263750K .......... .......... .......... .......... .......... 99% 1010K 1s\n263800K .......... .......... .......... .......... .......... 99%  867K 1s\n263850K .......... .......... .......... .......... .......... 99%  672K 1s\n263900K .......... .......... .......... .......... .......... 99%  707K 1s\n263950K .......... .......... .......... .......... .......... 99% 1.77M 1s\n264000K .......... .......... .......... .......... .......... 99%  762K 1s\n264050K .......... .......... .......... .......... .......... 99%  750K 1s\n264100K .......... .......... .......... .......... .......... 99%  652K 1s\n264150K .......... .......... .......... .......... .......... 99%  866K 1s\n264200K .......... .......... .......... .......... .......... 99% 1.20M 0s\n264250K .......... .......... .......... .......... .......... 99%  771K 0s\n264300K .......... .......... .......... .......... .......... 99%  752K 0s\n264350K .......... .......... .......... .......... .......... 99%  663K 0s\n264400K .......... .......... .......... .......... .......... 99% 1.02M 0s\n264450K .......... .......... .......... .......... .......... 99%  961K 0s\n264500K .......... .......... .......... .......... .......... 99%  895K 0s\n264550K .......... .......... .......... .......... .......... 99%  663K 0s\n264600K .......... .......... .......... .......... ....      100%  705K=5m19s\n\n2020-01-23 18:19:11 (829 KB/s) - \u0027Pfam-A.hmm.gz\u0027 saved [270995712/270995712]\n\n* Running CMD: /usr/local/bin/PFAM_dat_parser.pl Pfam-A.hmm.gz\n\r[100]  \r[200]  \r[300]  \r[400]  \r[500]  \r[600]  \r[700]  \r[800]  \r[900]  \r[1000]  \r[1100]  \r[1200]  \r[1300]  \r[1400]  \r[1500]  \r[1600]  \r[1700]  \r[1800]  \r[1900]  \r[2000]  \r[2100]  \r[2200]  \r[2300]  \r[2400]  \r[2500]  \r[2600]  \r[2700]  \r[2800]  \r[2900]  \r[3000]  \r[3100]  \r[3200]  \r[3300]  \r[3400]  \r[3500]  \r[3600]  \r[3700]  \r[3800]  \r[3900]  \r[4000]  \r[4100]  \r[4200]  \r[4300]  \r[4400]  \r[4500]  \r[4600]  \r[4700]  \r[4800]  \r[4900]  \r[5000]  \r[5100]  \r[5200]  \r[5300]  \r[5400]  \r[5500]  \r[5600]  \r[5700]  \r[5800]  \r[5900]  \r[6000]  \r[6100]  \r[6200]  \r[6300]  \r[6400]  \r[6500]  \r[6600]  \r[6700]  \r[6800]  \r[6900]  \r[7000]  \r[7100]  \r[7200]  \r[7300]  \r[7400]  \r[7500]  \r[7600]  \r[7700]  \r[7800]  \r[7900]  \r[8000]  \r[8100]  \r[8200]  \r[8300]  \r[8400]  \r[8500]  \r[8600]  \r[8700]  \r[8800]  \r[8900]  \r[9000]  \r[9100]  \r[9200]  \r[9300]  \r[9400]  \r[9500]  \r[9600]  \r[9700]  \r[9800]  \r[9900]  \r[10000]  \r[10100]  \r[10200]  \r[10300]  \r[10400]  \r[10500]  \r[10600]  \r[10700]  \r[10800]  \r[10900]  \r[11000]  \r[11100]  \r[11200]  \r[11300]  \r[11400]  \r[11500]  \r[11600]  \r[11700]  \r[11800]  \r[11900]  \r[12000]  \r[12100]  \r[12200]  \r[12300]  \r[12400]  \r[12500]  \r[12600]  \r[12700]  \r[12800]  \r[12900]  \r[13000]  \r[13100]  \r[13200]  \r[13300]  \r[13400]  \r[13500]  \r[13600]  \r[13700]  \r[13800]  \r[13900]  \r[14000]  \r[14100]  \r[14200]  \r[14300]  \r[14400]  \r[14500]  \r[14600]  \r[14700]  \r[14800]  \r[14900]  \r[15000]  \r[15100]  \r[15200]  \r[15300]  \r[15400]  \r[15500]  \r[15600]  \r[15700]  \r[15800]  \r[15900]  \r[16000]  \r[16100]  \r[16200]  \r[16300]  \r[16400]  \r[16500]  \r[16600]  \r[16700]  \r[16800]  \r[16900]  \r[17000]  \r[17100]  \r[17200]  \r[17300]  \r[17400]  \r[17500]  \r[17600]  \r[17700]  \r[17800]  \r[17900]  * Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam Pfam-A.hmm.gz.pfam_sqlite_bulk_load\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import Pfam-A.hmm.gz.pfam_sqlite_bulk_load PFAMreference\" | sqlite3 Trinotate.sqlite\n* Running CMD: wget \"http://www.geneontology.org/external2go/pfam2go\" \n--2020-01-23 18:19:23--  http://www.geneontology.org/external2go/pfam2go\nResolving www.geneontology.org... 107.23.42.138\nConnecting to www.geneontology.org|107.23.42.138|:80... connected.\nHTTP request sent, awaiting response... 301 Moved Permanently\nLocation: http://geneontology.org/external2go/pfam2go [following]\n--2020-01-23 18:19:23--  http://geneontology.org/external2go/pfam2go\nResolving geneontology.org... 107.23.42.138\nReusing existing connection to www.geneontology.org:80.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://purl.obolibrary.org/obo/go/external2go/pfam2go [following]\n--2020-01-23 18:19:23--  http://purl.obolibrary.org/obo/go/external2go/pfam2go\nResolving purl.obolibrary.org... 52.3.123.63\nConnecting to purl.obolibrary.org|52.3.123.63|:80... connected.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://current.geneontology.org/ontology/external2go/pfam2go [following]\n--2020-01-23 18:19:23--  http://current.geneontology.org/ontology/external2go/pfam2go\nResolving current.geneontology.org... 99.84.222.34, 99.84.222.60, 99.84.222.32, ...\nConnecting to current.geneontology.org|99.84.222.34|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 699985 (684K) [text/plain]\nSaving to: \u0027pfam2go\u0027\n\n     0K .......... .......... .......... .......... ..........  7% 74.5M 0s\n    50K .......... .......... .......... .......... .......... 14% 49.0M 0s\n   100K .......... .......... .......... .......... .......... 21%  193M 0s\n   150K .......... .......... .......... .......... .......... 29% 64.4M 0s\n   200K .......... .......... .......... .......... .......... 36%  149M 0s\n   250K .......... .......... .......... .......... .......... 43%  112M 0s\n   300K .......... .......... .......... .......... .......... 51%  231M 0s\n   350K .......... .......... .......... .......... .......... 58%  108M 0s\n   400K .......... .......... .......... .......... .......... 65%  137M 0s\n   450K .......... .......... .......... .......... .......... 73% 92.8M 0s\n   500K .......... .......... .......... .......... .......... 80%  181M 0s\n   550K .......... .......... .......... .......... .......... 87%  115M 0s\n   600K .......... .......... .......... .......... .......... 95%  190M 0s\n   650K .......... .......... .......... ...                  100%  130M=0.006s\n\n2020-01-23 18:19:23 (108 MB/s) - \u0027pfam2go\u0027 saved [699985/699985]\n\n* Running CMD: /usr/local/bin/PFAMtoGoParser.pl pfam2go \u003e pfam2go.tab\n* Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam2go pfam2go.tab\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import pfam2go.tab pfam2go\" | sqlite3 Trinotate.sqlite\nCMD: TrinotateSeqLoader.pl --sqlite Trinotate.sqlite --gene_trans_map /tmp/tmp8_jayk4s/files/4/3/0/dataset_43030d4b-e681-405b-805e-1fe6e3b39b4b.dat --transcript_fasta /tmp/tmp8_jayk4s/files/4/5/a/dataset_45aeb2f5-af12-4711-8868-26c6ec9d3ff4.dat --transdecoder_pep /tmp/tmp8_jayk4s/files/5/a/3/dataset_5a30da26-8da3-480f-89f5-5bf7088e8d11.dat --bulk_load\n-parsing gene/trans map file.... done.\n-loading Transcripts.\n\r[100]   \r[200]   \ndone.\n-loading ORFs.\n\ndone.\n\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.Transcript.bulk_load Transcript\" | sqlite3 Trinotate.sqlite\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.ORF.bulk_load ORF\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/6/a/2/dataset_6a23846f-e710-4ce6-a43b-dd22f87cacee.dat --prog blastp --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.82 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/c/3/4/dataset_c3463f12-6a71-4d37-8519-d3bb250afd9b.dat --prog blastx --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.87 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_PFAM_loader.pl --sqlite Trinotate.sqlite --pfam /tmp/tmp8_jayk4s/files/c/f/3/dataset_cf38b685-d33f-49f2-80d0-b0f9bde64b09.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.pfam_bulk_load.92 HMMERDbase\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_report_writer.pl --sqlite Trinotate.sqlite -E 1e-05 --pfam_cutoff DNC --incl_pep --incl_trans\n", "stdout": "memory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\n", "tool_id": "trinotate", "tool_stderr": "* Running CMD: wget \"http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\"\n--2020-01-23 17:58:22--  http://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.dat.gz\nResolving ftp.ebi.ac.uk... 193.62.193.138\nConnecting to ftp.ebi.ac.uk|193.62.193.138|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 582436772 (555M) [application/octet-stream]\nSaving to: \u0027uniprot_sprot.dat.gz\u0027\n\n     0K .......... .......... .......... .......... ..........  0%  167K 56m40s\n    50K .......... .......... .......... .......... ..........  0%  261K 46m28s\n   100K .......... .......... .......... .......... ..........  0%  551K 36m42s\n   150K .......... .......... .......... .......... ..........  0% 46.8M 27m35s\n   200K .......... .......... .......... .......... ..........  0%  534K 25m36s\n   250K .......... .......... .......... .......... ..........  0%  532K 24m18s\n   300K .......... .......... .......... .......... ..........  0%  537K 23m21s\n   350K .......... .......... .......... .......... ..........  0%  524K 22m42s\n   400K .......... .......... .......... .......... ..........  0% 57.7M 20m11s\n   450K .......... .......... .......... .......... ..........  0%  529K 19m57s\n   500K .......... .......... .......... .......... ..........  0%  500K 19m52s\n   550K .......... .......... .......... .......... ..........  0%  526K 19m42s\n   600K .......... .......... .......... .......... ..........  0%  551K 19m31s\n   650K .......... .......... .......... .......... ..........  0% 9.74M 18m11s\n   700K .......... .......... .......... .......... ..........  0%  524K 18m11s\n   750K .......... .......... .......... .......... ..........  0%  533K 18m9s\n   800K .......... .......... .......... .......... ..........  0%  523K 18m9s\n   850K .......... .......... .......... .......... ..........  0%  535K 18m7s\n   900K .......... .......... .......... .......... ..........  0% 14.8M 17m12s\n   950K .......... .......... .......... .......... ..........  0%  478K 17m19s\n  1000K .......... .......... .......... .......... ..........  0%  566K 17m18s\n  1050K .......... .......... .......... .......... ..........  0%  535K 17m19s\n  1100K .......... .......... .......... .......... ..........  0% 13.1M 16m35s\n  1150K .......... .......... .......... .......... ..........  0%  519K 16m39s\n  1200K .......... .......... .......... .......... ..........  0%  531K 16m42s\n  1250K .......... .......... .......... .......... ..........  0%  540K 16m44s\n  1300K .......... .......... .......... .......... ..........  0%  526K 16m46s\n  1350K .......... .......... .......... .......... ..........  0% 8.69M 16m13s\n  1400K .......... .......... .......... .......... ..........  0%  543K 16m15s\n  1450K .......... .......... .......... .......... ..........  0%  550K 16m17s\n  1500K .......... .......... .......... .......... ..........  0%  571K 16m17s\n  1550K .......... .......... .......... .......... ..........  0%  545K 16m19s\n  1600K .......... .......... .......... .......... ..........  0% 4.91M 15m53s\n  1650K .......... .......... .......... .......... ..........  0%  511K 15m57s\n  1700K .......... .......... .......... .......... ..........  0%  496K 16m3s\n  1750K .......... .......... .......... .......... ..........  0%  502K 16m7s\n  1800K .......... .......... .......... .......... ..........  0% 4.52M 15m44s\n  1850K .......... .......... .......... .......... ..........  0%  494K 15m50s\n  1900K .......... .......... .......... .......... ..........  0%  536K 15m52s\n  1950K .......... .......... .......... .......... ..........  0%  496K 15m57s\n  2000K .......... .......... .......... .......... ..........  0%  659K 15m55s\n  2050K .......... .......... .......... .......... ..........  0% 4.51M 15m35s\n  2100K .......... .......... .......... .......... ..........  0%  587K 15m35s\n  2150K .......... .......... .......... .......... ..........  0%  603K 15m35s\n  2200K .......... .......... .......... .......... ..........  0%  612K 15m35s\n  2250K .......... .......... .......... .......... ..........  0%  190M 15m15s\n  2300K .......... .......... .......... .......... ..........  0%  584K 15m16s\n  2350K .......... .......... .......... .......... ..........  0%  551K 15m18s\n  2400K .......... .......... .......... .......... ..........  0%  573K 15m19s\n  2450K .......... .......... .......... .......... ..........  0%  523K 15m23s\n  2500K .......... .......... .......... .......... ..........  0%  222M 15m4s\n  2550K .......... .......... .......... .......... ..........  0%  540K 15m7s\n  2600K .......... .......... .......... .......... ..........  0%  559K 15m9s\n  2650K .......... .......... .......... .......... ..........  0%  549K 15m11s\n  2700K .......... .......... .......... .......... ..........  0%  538K 15m14s\n  2750K .......... .......... .......... .......... ..........  0%  148M 14m57s\n  2800K .......... .......... .......... .......... ..........  0%  534K 15m0s\n  2850K .......... .......... .......... .......... ..........  0%  558K 15m2s\n  2900K .......... .......... .......... .......... ..........  0%  531K 15m5s\n  2950K .......... .......... .......... .......... ..........  0%  178M 14m50s\n  3000K .......... .......... .......... .......... ..........  0%  563K 14m51s\n  3050K .......... .......... .......... .......... ..........  0%  558K 14m53s\n  3100K .......... .......... .......... .......... ..........  0%  516K 14m56s\n  3150K .......... .......... .......... .......... ..........  0%  543K 14m59s\n  3200K .......... .......... .......... .......... ..........  0%  193M 14m45s\n  3250K .......... .......... .......... .......... ..........  0%  523K 14m48s\n  3300K .......... .......... .......... .......... ..........  0%  521K 14m51s\n  3350K .......... .......... .......... .......... ..........  0%  558K 14m52s\n  3400K .......... .......... .......... .......... ..........  0%  199M 14m39s\n  3450K .......... .......... .......... .......... ..........  0%  531K 14m42s\n  3500K .......... .......... .......... .......... ..........  0%  554K 14m44s\n  3550K .......... .......... .......... .......... ..........  0%  537K 14m46s\n  3600K .......... .......... .......... .......... ..........  0%  540K 14m48s\n  3650K .......... .......... .......... .......... ..........  0%  165M 14m36s\n  3700K .......... .......... .......... .......... ..........  0%  574K 14m37s\n  3750K .......... .......... .......... .......... ..........  0%  574K 14m39s\n  3800K .......... .......... .......... .......... ..........  0%  532K 14m41s\n  3850K .......... .......... .......... .......... ..........  0% 30.8M 14m30s\n  3900K .......... .......... .......... .......... ..........  0%  572K 14m31s\n  3950K .......... .......... .......... .......... ..........  0%  599K 14m32s\n  4000K .......... .......... .......... .......... ..........  0%  568K 14m34s\n  4050K .......... .......... .......... .......... ..........  0%  585K 14m35s\n  4100K .......... .......... .......... .......... ..........  0%  236M 14m24s\n  4150K .......... .......... .......... .......... ..........  0%  602K 14m25s\n  4200K .......... .......... .......... .......... ..........  0%  515K 14m28s\n  4250K .......... .......... .......... .......... ..........  0%  580K 14m29s\n  4300K .......... .......... .......... .......... ..........  0%  578K 14m30s\n  4350K .......... .......... .......... .......... ..........  0% 52.4M 14m20s\n  4400K .......... .......... .......... .......... ..........  0%  554K 14m22s\n  4450K .......... .......... .......... .......... ..........  0%  587K 14m23s\n  4500K .......... .......... .......... .......... ..........  0%  585K 14m24s\n  4550K .......... .......... .......... .......... ..........  0% 26.9M 14m15s\n  4600K .......... .......... .......... .......... ..........  0%  597K 14m15s\n  4650K .......... .......... .......... .......... ..........  0%  563K 14m17s\n  4700K .......... .......... .......... .......... ..........  0%  579K 14m18s\n  4750K .......... .......... .......... .......... ..........  0%  567K 14m19s\n  4800K .......... .......... .......... .......... ..........  0% 28.7M 14m11s\n  4850K .......... .......... .......... .......... ..........  0%  564K 14m12s\n  4900K .......... .......... .......... .......... ..........  0%  551K 14m14s\n  4950K .......... .......... .......... .......... ..........  0%  544K 14m16s\n  5000K .......... .......... .......... .......... ..........  0%  550K 14m17s\n  5050K .......... .......... .......... .......... ..........  0% 11.4M 14m9s\n  5100K .......... .......... .......... .......... ..........  0%  565K 14m10s\n  5150K .......... .......... .......... .......... ..........  0%  537K 14m12s\n  5200K .......... .......... .......... .......... ..........  0%  563K 14m14s\n  5250K .......... .......... .......... .......... ..........  0%  568K 14m15s\n  5300K .......... .......... .......... .......... ..........  0% 12.3M 14m7s\n  5350K .......... .......... .......... .......... ..........  0%  594K 14m8s\n  5400K .......... .......... .......... .......... ..........  0%  615K 14m9s\n  5450K .......... .......... .......... .......... ..........  0%  581K 14m10s\n  5500K .......... .......... .......... .......... ..........  0%  610K 14m10s\n  5550K .......... .......... .......... .......... ..........  0% 8.07M 14m3s\n  5600K .......... .......... .......... .......... ..........  0%  676K 14m3s\n  5650K .......... .......... .......... .......... ..........  1%  594K 14m4s\n  5700K .......... .......... .......... .......... ..........  1%  613K 14m4s\n  5750K .......... .......... .......... .......... ..........  1%  620K 14m5s\n  5800K .......... .......... .......... .......... ..........  1% 10.9M 13m58s\n  5850K .......... .......... .......... .......... ..........  1%  651K 13m58s\n  5900K .......... .......... .......... .......... ..........  1%  653K 13m58s\n  5950K .......... .......... .......... .......... ..........  1%  647K 13m59s\n  6000K .......... .......... .......... .......... ..........  1%  623K 13m59s\n  6050K .......... .......... .......... .......... ..........  1% 43.2M 13m52s\n  6100K .......... .......... .......... .......... ..........  1%  630K 13m53s\n  6150K .......... .......... .......... .......... ..........  1%  619K 13m53s\n  6200K .......... .......... .......... .......... ..........  1%  564K 13m54s\n  6250K .......... .......... .......... .......... ..........  1%  584K 13m55s\n  6300K .......... .......... .......... .......... ..........  1%  199M 13m49s\n  6350K .......... .......... .......... .......... ..........  1%  544K 13m50s\n  6400K .......... .......... .......... .......... ..........  1%  550K 13m52s\n  6450K .......... .......... .......... .......... ..........  1%  548K 13m53s\n  6500K .......... .......... .......... .......... ..........  1%  241M 13m47s\n  6550K .......... .......... .......... .......... ..........  1%  522K 13m48s\n  6600K .......... .......... .......... .......... ..........  1%  576K 13m50s\n  6650K .......... .......... .......... .......... ..........  1%  579K 13m50s\n  6700K .......... .......... .......... .......... ..........  1%  573K 13m52s\n  6750K .......... .......... .......... .......... ..........  1%  122M 13m45s\n  6800K .......... .......... .......... .......... ..........  1%  567K 13m47s\n  6850K .......... .......... .......... .......... ..........  1%  567K 13m48s\n  6900K .......... .......... .......... .......... ..........  1%  570K 13m49s\n  6950K .......... .......... .......... .......... ..........  1%  599K 13m49s\n  7000K .......... .......... .......... .......... ..........  1% 17.7M 13m44s\n  7050K .......... .......... .......... .......... ..........  1%  599K 13m44s\n  7100K .......... .......... .......... .......... ..........  1%  563K 13m46s\n  7150K .......... .......... .......... .......... ..........  1%  586K 13m46s\n  7200K .......... .......... .......... .......... ..........  1%  136M 13m41s\n  7250K .......... .......... .......... .......... ..........  1%  620K 13m41s\n  7300K .......... .......... .......... .......... ..........  1%  634K 13m42s\n  7350K .......... .......... .......... .......... ..........  1%  629K 13m42s\n  7400K .......... .......... .......... .......... ..........  1%  606K 13m43s\n  7450K .......... .......... .......... .......... ..........  1%  234M 13m37s\n  7500K .......... .......... .......... .......... ..........  1%  632K 13m37s\n  7550K .......... .......... .......... .......... ..........  1%  636K 13m38s\n  7600K .......... .......... .......... .......... ..........  1%  640K 13m38s\n  7650K .......... .......... .......... .......... ..........  1%  193M 13m33s\n  7700K .......... .......... .......... .......... ..........  1%  658K 13m33s\n  7750K .......... .......... .......... .......... ..........  1%  657K 13m33s\n  7800K .......... .......... .......... .......... ..........  1%  635K 13m33s\n  7850K .......... .......... .......... .......... ..........  1%  660K 13m34s\n  7900K .......... .......... .......... .......... ..........  1%  160M 13m28s\n  7950K .......... .......... .......... .......... ..........  1%  610K 13m29s\n  8000K .......... .......... .......... .......... ..........  1%  656K 13m29s\n  8050K .......... .......... .......... .......... ..........  1%  659K 13m29s\n  8100K .......... .......... .......... .......... ..........  1% 53.7M 13m25s\n  8150K .......... .......... .......... .......... ..........  1%  639K 13m25s\n  8200K .......... .......... .......... .......... ..........  1%  628K 13m25s\n  8250K .......... .......... .......... .......... ..........  1%  618K 13m26s\n  8300K .......... .......... .......... .......... ..........  1%  605K 13m27s\n  8350K .......... .......... .......... .......... ..........  1% 82.1M 13m22s\n  8400K .......... .......... .......... .......... ..........  1%  656K 13m22s\n  8450K .......... .......... .......... .......... ..........  1%  657K 13m22s\n  8500K .......... .......... .......... .......... ..........  1%  667K 13m22s\n  8550K .......... .......... .......... .......... ..........  1%  651K 13m23s\n  8600K .......... .......... .......... .......... ..........  1% 35.1M 13m18s\n  8650K .......... .......... .......... .......... ..........  1%  655K 13m18s\n  8700K .......... .......... .......... .......... ..........  1%  606K 13m19s\n  8750K .......... .......... .......... .......... ..........  1%  657K 13m19s\n  8800K .......... .......... .......... .......... ..........  1% 17.8M 13m15s\n  8850K .......... .......... .......... .......... ..........  1%  646K 13m15s\n  8900K .......... .......... .......... .......... ..........  1%  608K 13m16s\n  8950K .......... .......... .......... .......... ..........  1%  631K 13m16s\n  9000K .......... .......... .......... .......... ..........  1%  664K 13m16s\n  9050K .......... .......... .......... .......... ..........  1% 15.7M 13m12s\n  9100K .......... .......... .......... .......... ..........  1%  618K 13m13s\n  9150K .......... .......... .......... .......... ..........  1%  663K 13m13s\n  9200K .......... .......... .......... .......... ..........  1%  608K 13m13s\n  9250K .......... .......... .......... .......... ..........  1%  625K 13m14s\n  9300K .......... .......... .......... .......... ..........  1% 39.7M 13m10s\n  9350K .......... .......... .......... .......... ..........  1%  630K 13m10s\n  9400K .......... .......... .......... .......... ..........  1%  633K 13m11s\n  9450K .......... .......... .......... .......... ..........  1%  639K 13m11s\n  9500K .......... .......... .......... .......... ..........  1%  144M 13m7s\n  9550K .......... .......... .......... .......... ..........  1%  642K 13m7s\n  9600K .......... .......... .......... .......... ..........  1%  613K 13m8s\n  9650K .......... .......... .......... .......... ..........  1%  605K 13m8s\n  9700K .......... .......... .......... .......... ..........  1%  617K 13m9s\n  9750K .......... .......... .......... .......... ..........  1% 83.5M 13m5s\n  9800K .......... .......... .......... .......... ..........  1%  650K 13m5s\n  9850K .......... .......... .......... .......... ..........  1%  656K 13m5s\n  9900K .......... .......... .......... .......... ..........  1%  661K 13m6s\n \n..\n... .......... .......... 97% 1000K 7s\n258500K .......... .......... .......... .......... .......... 97%  650K 7s\n258550K .......... .......... .......... .......... .......... 97%  724K 7s\n258600K .......... .......... .......... .......... .......... 97% 1.41M 7s\n258650K .......... .......... .......... .......... .......... 97%  724K 7s\n258700K .......... .......... .......... .......... .......... 97%  862K 7s\n258750K .......... .......... .......... .......... .......... 97%  652K 7s\n258800K .......... .......... .......... .......... .......... 97%  789K 7s\n258850K .......... .......... .......... .......... .......... 97% 1.24M 7s\n258900K .......... .......... .......... .......... .......... 97%  779K 7s\n258950K .......... .......... .......... .......... .......... 97%  784K 7s\n259000K .......... .......... .......... .......... .......... 97%  687K 7s\n259050K .......... .......... .......... .......... .......... 97% 1.58M 7s\n259100K .......... .......... .......... .......... .......... 97%  665K 7s\n259150K .......... .......... .......... .......... .......... 97%  968K 7s\n259200K .......... .......... .......... .......... .......... 97%  655K 7s\n259250K .......... .......... .......... .......... .......... 97%  742K 6s\n259300K .......... .......... .......... .......... .......... 97% 1.38M 6s\n259350K .......... .......... .......... .......... .......... 98%  739K 6s\n259400K .......... .......... .......... .......... .......... 98%  826K 6s\n259450K .......... .......... .......... .......... .......... 98%  655K 6s\n259500K .......... .......... .......... .......... .......... 98%  818K 6s\n259550K .......... .......... .......... .......... .......... 98% 1.18M 6s\n259600K .......... .......... .......... .......... .......... 98%  797K 6s\n259650K .......... .......... .......... .......... .......... 98%  764K 6s\n259700K .......... .......... .......... .......... .......... 98%  714K 6s\n259750K .......... .......... .......... .......... .......... 98% 1.53M 6s\n259800K .......... .......... .......... .......... .......... 98%  675K 6s\n259850K .......... .......... .......... .......... .......... 98%  921K 6s\n259900K .......... .......... .......... .......... .......... 98%  651K 6s\n259950K .......... .......... .......... .......... .......... 98%  766K 6s\n260000K .......... .......... .......... .......... .......... 98% 1.32M 6s\n260050K .......... .......... .......... .......... .......... 98%  755K 5s\n260100K .......... .......... .......... .......... .......... 98%  806K 5s\n260150K .......... .......... .......... .......... .......... 98%  650K 5s\n260200K .......... .......... .......... .......... .......... 98%  862K 5s\n260250K .......... .......... .......... .......... .......... 98% 1.12M 5s\n260300K .......... .......... .......... .......... .......... 98%  814K 5s\n260350K .......... .......... .......... .......... .......... 98%  737K 5s\n260400K .......... .......... .......... .......... .......... 98%  691K 5s\n260450K .......... .......... .......... .......... .......... 98% 1.69M 5s\n260500K .......... .......... .......... .......... .......... 98%  698K 5s\n260550K .......... .......... .......... .......... .......... 98%  858K 5s\n260600K .......... .......... .......... .......... .......... 98%  650K 5s\n260650K .......... .......... .......... .......... .......... 98%  808K 5s\n260700K .......... .......... .......... .......... .......... 98% 1.27M 5s\n260750K .......... .......... .......... .......... .......... 98%  766K 5s\n260800K .......... .......... .......... .......... .......... 98%  774K 5s\n260850K .......... .......... .......... .......... .......... 98%  651K 5s\n260900K .......... .......... .......... .......... .......... 98%  922K 4s\n260950K .......... .......... .......... .......... .......... 98% 1.08M 4s\n261000K .......... .......... .......... .......... .......... 98%  828K 4s\n261050K .......... .......... .......... .......... .......... 98%  714K 4s\n261100K .......... .......... .......... .......... .......... 98%  719K 4s\n261150K .......... .......... .......... .......... .......... 98% 1.57M 4s\n261200K .......... .......... .......... .......... .......... 98%  726K 4s\n261250K .......... .......... .......... .......... .......... 98%  822K 4s\n261300K .......... .......... .......... .......... .......... 98%  654K 4s\n261350K .......... .......... .......... .......... .......... 98%  820K 4s\n261400K .......... .......... .......... .......... .......... 98% 1.23M 4s\n261450K .......... .......... .......... .......... .......... 98%  777K 4s\n261500K .......... .......... .......... .......... .......... 98%  764K 4s\n261550K .......... .......... .......... .......... .......... 98%  652K 4s\n261600K .......... .......... .......... .......... .......... 98%  949K 4s\n261650K .......... .......... .......... .......... .......... 98% 1.02M 4s\n261700K .......... .......... .......... .......... .......... 98%  853K 3s\n261750K .......... .......... .......... .......... .......... 98%  707K 3s\n261800K .......... .......... .......... .......... .......... 98%  726K 3s\n261850K .......... .......... .......... .......... .......... 98% 1.53M 3s\n261900K .......... .......... .......... .......... .......... 98%  740K 3s\n261950K .......... .......... .......... .......... .......... 99%  808K 3s\n262000K .......... .......... .......... .......... .......... 99%  655K 3s\n262050K .......... .......... .......... .......... .......... 99%  823K 3s\n262100K .......... .......... .......... .......... .......... 99% 1.22M 3s\n262150K .......... .......... .......... .......... .......... 99%  788K 3s\n262200K .......... .......... .......... .......... .......... 99%  755K 3s\n262250K .......... .......... .......... .......... .......... 99%  651K 3s\n262300K .......... .......... .......... .......... .......... 99%  969K 3s\n262350K .......... .......... .......... .......... .......... 99% 1.01M 3s\n262400K .......... .......... .......... .......... .......... 99%  865K 3s\n262450K .......... .......... .......... .......... .......... 99%  695K 3s\n262500K .......... .......... .......... .......... .......... 99%  744K 3s\n262550K .......... .......... .......... .......... .......... 99% 1.48M 2s\n262600K .......... .......... .......... .......... .......... 99%  746K 2s\n262650K .......... .......... .......... .......... .......... 99%  798K 2s\n262700K .......... .......... .......... .......... .......... 99%  653K 2s\n262750K .......... .......... .......... .......... .......... 99%  838K 2s\n262800K .......... .......... .......... .......... .......... 99% 1.22M 2s\n262850K .......... .......... .......... .......... .......... 99%  772K 2s\n262900K .......... .......... .......... .......... .......... 99%  761K 2s\n262950K .......... .......... .......... .......... .......... 99%  653K 2s\n263000K .......... .......... .......... .......... .......... 99%  980K 2s\n263050K .......... .......... .......... .......... .......... 99% 1.01M 2s\n263100K .......... .......... .......... .......... .......... 99%  858K 2s\n263150K .......... .......... .......... .......... .......... 99%  689K 2s\n263200K .......... .......... .......... .......... .......... 99%  757K 2s\n263250K .......... .......... .......... .......... .......... 99% 1.47M 2s\n263300K .......... .......... .......... .......... .......... 99%  747K 2s\n263350K .......... .......... .......... .......... .......... 99%  788K 2s\n263400K .......... .......... .......... .......... .......... 99%  653K 1s\n263450K .......... .......... .......... .......... .......... 99%  834K 1s\n263500K .......... .......... .......... .......... .......... 99% 1.23M 1s\n263550K .......... .......... .......... .......... .......... 99%  764K 1s\n263600K .......... .......... .......... .......... .......... 99%  762K 1s\n263650K .......... .......... .......... .......... .......... 99%  655K 1s\n263700K .......... .......... .......... .......... .......... 99% 1014K 1s\n263750K .......... .......... .......... .......... .......... 99% 1010K 1s\n263800K .......... .......... .......... .......... .......... 99%  867K 1s\n263850K .......... .......... .......... .......... .......... 99%  672K 1s\n263900K .......... .......... .......... .......... .......... 99%  707K 1s\n263950K .......... .......... .......... .......... .......... 99% 1.77M 1s\n264000K .......... .......... .......... .......... .......... 99%  762K 1s\n264050K .......... .......... .......... .......... .......... 99%  750K 1s\n264100K .......... .......... .......... .......... .......... 99%  652K 1s\n264150K .......... .......... .......... .......... .......... 99%  866K 1s\n264200K .......... .......... .......... .......... .......... 99% 1.20M 0s\n264250K .......... .......... .......... .......... .......... 99%  771K 0s\n264300K .......... .......... .......... .......... .......... 99%  752K 0s\n264350K .......... .......... .......... .......... .......... 99%  663K 0s\n264400K .......... .......... .......... .......... .......... 99% 1.02M 0s\n264450K .......... .......... .......... .......... .......... 99%  961K 0s\n264500K .......... .......... .......... .......... .......... 99%  895K 0s\n264550K .......... .......... .......... .......... .......... 99%  663K 0s\n264600K .......... .......... .......... .......... ....      100%  705K=5m19s\n\n2020-01-23 18:19:11 (829 KB/s) - \u0027Pfam-A.hmm.gz\u0027 saved [270995712/270995712]\n\n* Running CMD: /usr/local/bin/PFAM_dat_parser.pl Pfam-A.hmm.gz\n\r[100]  \r[200]  \r[300]  \r[400]  \r[500]  \r[600]  \r[700]  \r[800]  \r[900]  \r[1000]  \r[1100]  \r[1200]  \r[1300]  \r[1400]  \r[1500]  \r[1600]  \r[1700]  \r[1800]  \r[1900]  \r[2000]  \r[2100]  \r[2200]  \r[2300]  \r[2400]  \r[2500]  \r[2600]  \r[2700]  \r[2800]  \r[2900]  \r[3000]  \r[3100]  \r[3200]  \r[3300]  \r[3400]  \r[3500]  \r[3600]  \r[3700]  \r[3800]  \r[3900]  \r[4000]  \r[4100]  \r[4200]  \r[4300]  \r[4400]  \r[4500]  \r[4600]  \r[4700]  \r[4800]  \r[4900]  \r[5000]  \r[5100]  \r[5200]  \r[5300]  \r[5400]  \r[5500]  \r[5600]  \r[5700]  \r[5800]  \r[5900]  \r[6000]  \r[6100]  \r[6200]  \r[6300]  \r[6400]  \r[6500]  \r[6600]  \r[6700]  \r[6800]  \r[6900]  \r[7000]  \r[7100]  \r[7200]  \r[7300]  \r[7400]  \r[7500]  \r[7600]  \r[7700]  \r[7800]  \r[7900]  \r[8000]  \r[8100]  \r[8200]  \r[8300]  \r[8400]  \r[8500]  \r[8600]  \r[8700]  \r[8800]  \r[8900]  \r[9000]  \r[9100]  \r[9200]  \r[9300]  \r[9400]  \r[9500]  \r[9600]  \r[9700]  \r[9800]  \r[9900]  \r[10000]  \r[10100]  \r[10200]  \r[10300]  \r[10400]  \r[10500]  \r[10600]  \r[10700]  \r[10800]  \r[10900]  \r[11000]  \r[11100]  \r[11200]  \r[11300]  \r[11400]  \r[11500]  \r[11600]  \r[11700]  \r[11800]  \r[11900]  \r[12000]  \r[12100]  \r[12200]  \r[12300]  \r[12400]  \r[12500]  \r[12600]  \r[12700]  \r[12800]  \r[12900]  \r[13000]  \r[13100]  \r[13200]  \r[13300]  \r[13400]  \r[13500]  \r[13600]  \r[13700]  \r[13800]  \r[13900]  \r[14000]  \r[14100]  \r[14200]  \r[14300]  \r[14400]  \r[14500]  \r[14600]  \r[14700]  \r[14800]  \r[14900]  \r[15000]  \r[15100]  \r[15200]  \r[15300]  \r[15400]  \r[15500]  \r[15600]  \r[15700]  \r[15800]  \r[15900]  \r[16000]  \r[16100]  \r[16200]  \r[16300]  \r[16400]  \r[16500]  \r[16600]  \r[16700]  \r[16800]  \r[16900]  \r[17000]  \r[17100]  \r[17200]  \r[17300]  \r[17400]  \r[17500]  \r[17600]  \r[17700]  \r[17800]  \r[17900]  * Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam Pfam-A.hmm.gz.pfam_sqlite_bulk_load\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import Pfam-A.hmm.gz.pfam_sqlite_bulk_load PFAMreference\" | sqlite3 Trinotate.sqlite\n* Running CMD: wget \"http://www.geneontology.org/external2go/pfam2go\" \n--2020-01-23 18:19:23--  http://www.geneontology.org/external2go/pfam2go\nResolving www.geneontology.org... 107.23.42.138\nConnecting to www.geneontology.org|107.23.42.138|:80... connected.\nHTTP request sent, awaiting response... 301 Moved Permanently\nLocation: http://geneontology.org/external2go/pfam2go [following]\n--2020-01-23 18:19:23--  http://geneontology.org/external2go/pfam2go\nResolving geneontology.org... 107.23.42.138\nReusing existing connection to www.geneontology.org:80.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://purl.obolibrary.org/obo/go/external2go/pfam2go [following]\n--2020-01-23 18:19:23--  http://purl.obolibrary.org/obo/go/external2go/pfam2go\nResolving purl.obolibrary.org... 52.3.123.63\nConnecting to purl.obolibrary.org|52.3.123.63|:80... connected.\nHTTP request sent, awaiting response... 302 Found\nLocation: http://current.geneontology.org/ontology/external2go/pfam2go [following]\n--2020-01-23 18:19:23--  http://current.geneontology.org/ontology/external2go/pfam2go\nResolving current.geneontology.org... 99.84.222.34, 99.84.222.60, 99.84.222.32, ...\nConnecting to current.geneontology.org|99.84.222.34|:80... connected.\nHTTP request sent, awaiting response... 200 OK\nLength: 699985 (684K) [text/plain]\nSaving to: \u0027pfam2go\u0027\n\n     0K .......... .......... .......... .......... ..........  7% 74.5M 0s\n    50K .......... .......... .......... .......... .......... 14% 49.0M 0s\n   100K .......... .......... .......... .......... .......... 21%  193M 0s\n   150K .......... .......... .......... .......... .......... 29% 64.4M 0s\n   200K .......... .......... .......... .......... .......... 36%  149M 0s\n   250K .......... .......... .......... .......... .......... 43%  112M 0s\n   300K .......... .......... .......... .......... .......... 51%  231M 0s\n   350K .......... .......... .......... .......... .......... 58%  108M 0s\n   400K .......... .......... .......... .......... .......... 65%  137M 0s\n   450K .......... .......... .......... .......... .......... 73% 92.8M 0s\n   500K .......... .......... .......... .......... .......... 80%  181M 0s\n   550K .......... .......... .......... .......... .......... 87%  115M 0s\n   600K .......... .......... .......... .......... .......... 95%  190M 0s\n   650K .......... .......... .......... ...                  100%  130M=0.006s\n\n2020-01-23 18:19:23 (108 MB/s) - \u0027pfam2go\u0027 saved [699985/699985]\n\n* Running CMD: /usr/local/bin/PFAMtoGoParser.pl pfam2go \u003e pfam2go.tab\n* Running CMD: /usr/local/bin/EMBL_dat_to_Trinotate_sqlite_resourceDB.pl --sqlite Trinotate.sqlite --pfam2go pfam2go.tab\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import pfam2go.tab pfam2go\" | sqlite3 Trinotate.sqlite\nCMD: TrinotateSeqLoader.pl --sqlite Trinotate.sqlite --gene_trans_map /tmp/tmp8_jayk4s/files/4/3/0/dataset_43030d4b-e681-405b-805e-1fe6e3b39b4b.dat --transcript_fasta /tmp/tmp8_jayk4s/files/4/5/a/dataset_45aeb2f5-af12-4711-8868-26c6ec9d3ff4.dat --transdecoder_pep /tmp/tmp8_jayk4s/files/5/a/3/dataset_5a30da26-8da3-480f-89f5-5bf7088e8d11.dat --bulk_load\n-parsing gene/trans map file.... done.\n-loading Transcripts.\n\r[100]   \r[200]   \ndone.\n-loading ORFs.\n\ndone.\n\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.Transcript.bulk_load Transcript\" | sqlite3 Trinotate.sqlite\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.ORF.bulk_load ORF\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/6/a/2/dataset_6a23846f-e710-4ce6-a43b-dd22f87cacee.dat --prog blastp --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.82 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_BLAST_loader.pl --sqlite Trinotate.sqlite --outfmt6 /tmp/tmp8_jayk4s/files/c/3/4/dataset_c3463f12-6a71-4d37-8519-d3bb250afd9b.dat --prog blastx --dbtype Swissprot\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.blast_bulk_load.87 BlastDbase\" | sqlite3 Trinotate.sqlite\n\n\nBlastDbase loading complete..\n\nCMD: Trinotate_PFAM_loader.pl --sqlite Trinotate.sqlite --pfam /tmp/tmp8_jayk4s/files/c/f/3/dataset_cf38b685-d33f-49f2-80d0-b0f9bde64b09.dat\nCMD: echo \"pragma journal_mode=memory;\npragma synchronous=0;\npragma cache_size=4000000;\n.mode tabs\n.import tmp.pfam_bulk_load.92 HMMERDbase\" | sqlite3 Trinotate.sqlite\n\n\nLoading complete..\n\nCMD: Trinotate_report_writer.pl --sqlite Trinotate.sqlite -E 1e-05 --pfam_cutoff DNC --incl_pep --incl_trans\n", "tool_stdout": "memory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\nmemory\n", "update_time": "2020-01-23T18:19:32.983531", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmp4vhthgkjtrinotate_annotation_report_minus_tmhmm_signalp.xls v. /tmp/tmpm8gt2so9trinotate_annotation_report_minus_tmhmm_signalp.xls )\n--- local_file\n+++ history_data\n@@ -1,29 +1,29 @@\n #gene_id\ttranscript_id\tsprot_Top_BLASTX_hit\tRNAMMER\tprot_id\tprot_coords\tsprot_Top_BLASTP_hit\tPfam\tSignalP\tTmHMM\teggnog\tKegg\tgene_ontology_blast\tgene_ontology_pfam\ttranscript\tpeptide\n-TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1E8.02\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\tTGTGATTCGTGCCGCTCAACATTTCTTCAAGTGATTCCTAACATTGCTGTACATTCTCTTGAAAAGCGTTTTAACCTTTTCAATTCTGAAACTTGTAAAGAATGTCAGAGTATCGTATTCGAGTGACAACTGTGGACCAGAAGGTTGGTATTTTTCAAGTTCCTAGGACAAAAACGGTTTTAGAATTAAAGGAACTGATTGCTGTAACATTTGAAGCTCCTGCAGATCGATTAAAGTTGATTCACGCTGGTCGTGTATTGAGGAATGAAACTCCCCTCGAGGAAATTTTGCATGACGCTACTGATCTGGTCACTTTTCATCTTGTAATTGCGGTTTTTAATTCTACAAGCACAACCTTGCCATCAGCGACATCATCTTCTGTTCCTCAAAGTAGGACCAGCGAATTATCATCGACTAATTCAATTCCAACTCCTCGAATCACCAGTTTGAACCCGGAAGAATTGTCAAGACGTGAACGTGCCCAACGTTTATTACAGACATACAACAGTTTCCATGGATCTGGCTTAGGAGGACTTTTTCCAAACATACACCGGGAACTAGAAAGTCACGGATTTTCATTGCCAACCCATGAGCAATCATCACCCGTTGCTGAATCTCTGGATAATTCGGTATCGTCTGCTTTATCTCCACATTTGGAAACGCTGCGTCGCCGCAATCTCAGTATTCATCATCAACATATACAAGCTCACGAAATGGCACAAGAGTCCTTGGAGACCAGAAATCCAGGAAACATTTCTTCCTCGAGTGCTCCTTTAGCTAGTGATCAATCCCCGACTGTATCATCTAACCATATCCATGCATCCGGTAATTTGGCTTTAGGATCAAATTCGGGGTTAAATCCACGAAGCCCTAATTCTTTTTCTTCCCCATTGGACAATCCTGCACTTCATACTGTTGACTCAACCAACGTCAATGGTTCGCTTTCACCTCTATCAAATTCTTCATCTATCAATCAAGTTCACCAAAATGAAACTCATGGTTCTACCATCTCAGTACCTAACCCGAACTTAAGTCAAATGGGCCCTTCACACTCTTCTTCTGTGCCCTCAAATCTGTCACCAAATCCTGCCCAAAATGAAAATCCTAGTACAACCTCTATTCCATCAATAAATAATCAACCATTCCCGTCTGGTCTTTCTGCTTCTAACTCTAATTTTGCCTCTTCCTCGTTTATCCCTCAGAGCGTTCCGCAACTATTGCCTATATATTATCAAACTATCTTTTACAACGGTAACTATTATTTACAGCAGCTCCCATCTGCATCGCCTCCCACTATGTTTCGAGATCACTCATTTGCTCCTTTAGTTTCCCCTTCTATTGTTTCTCCGTATGGTGTTTTAGAAAATGAAGAAACTGGCGAATGTGCCTTTTTGTTTTCACCAAATGCCTCCCAACCCCATTTTCAGCCGAGAGCCCCCACGTTTGGCATTCCACGAAATGTGAGATCCCTCTTCACCTTACCGTTCTTCCATACTATCCGTAATATTGAGAGACATTTCCGATTATTTATTCGTTTAGCCCTGTTTTGTGTATTAACGACTTATAATGTTTCTCTCTCACAAACAATTTTGCTAACATCTATAATGTCTGTTGTGTTTTTGTTACAGACCGGTGCTCTAGCTCCATTCATCAATGACAATCCACTCATCCAGTCTGGTATGCGTCATATTCGTAATCTTCAAGATGAATACCGGCGGCGTCGAAACCGAACGGCTCAAAGAGTAGTTGAAATTCCTAATGAGACACAGACTGAAGATGAACAAGATGGTACAAATACGCCGGATAATAGGGCTGACGCTGAAGAACGCGAACTAACTCGAAGTCAAAGAATATACCGGACGGTAGTAAGAACCATAGTTGCCTTCGCATTAAGCTTTGTACCTCGAGCGTAACATTTTTTTCTTAAAAACAAAAGTGATAATATAGCTGTAAAGTCATTTTTCATTACTAATTAGAAATATTGTCTAAGCTTGCTATTAATAAAAGAGTTCTAGCTAGACTAAATTTTGACAGTGTCGAAATATCATTCCAAGGTACCCCATCAGATTAACAAATTTATAA\tMSEYRIRVTTVDQKVGIFQVPRTKTVLELKELIAVTFEAPADRLKLIHAGRVLRNETPLEEILHDATDLVTFHLVIAVFNSTSTTLPSATSSSVPQSRTSELSSTNSIPTPRITSLNPEELSRRERAQRLLQTYNSFHGSGLGGLFPNIHRELESHGFSLPTHEQSSPVAESLDNSVSSALSPHLETLRRRNLSIHHQHIQAHEMAQESLETRNPGNISSSSAPLASDQSPTVSSNHIHASGNLALGSNSGLNPRSPNSFSSPLDNPALHTVDSTNVNGSLSPLSNSSSINQVHQNETHGSTISVPNPNLSQMGPSHSSSVPSNLSPNPAQNENPSTTSIPSINNQPFPSGLSASNSNFASSSFIPQSVPQLLPIYYQTIFYNGNYYLQQLPSASPPTMFRDHSFAPLVSPSIVSPYGVLENEETGECAFLFSPNASQPHFQPRAPTFGIPRNVRSLFTLPFFHTIRNIERHFRLFIRLALFCVLTTYNVSLSQTILLTSIMSVVFLLQTGALAPFINDNPLIQSGMRHIRNLQDEYRRRRNRTAQRVVEIPNETQTEDEQDGTNTPDNRADAEERELTRSQRIYRTVVRTIVAFALSFVPRA*\n+TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\tGO:0005829^cellular_component^cytosol`GO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0031593^molecular_function^polyubiquitin modification-dependent protein binding`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\tTGTGATTCGTGCCGCTCAACATTTCTTCAAGTGATTCCTAACATTGCTGTACATTCTCTTGAAAAGCGTTTTAACCTTTTCAATTCTGAAACTTGTAAAGAATGTCAGAGTATCGTATTCGAGTGACAACTGTGGACCAGAAGGTTGGTATTTTTCAAGTTCCTAGGACAAAAACGGTTTTAGAATTAAAGGAACTGATTGCTGTAACATTTGAAGCTCCTGCAGATCGATTAAAGTTGATTCACGCTGGTCGTGTATTGAGGAATGAAACTCCCCTCGAGGAAATTTTGCATGACGCTACTGATCTGGTCACTTTTCATCTTGTAATTGCGGTTTTTAATTCTACAAGCACAACCTTGCCATCAGCGACATCATCTTCTGTTCCTCAAAGTAGGACCAGCGAATTATCATCGACTAATTCAATTCCAACTCCTCGAATCACCAGTTTGAACCCGGAAGAATTGTCAAGACGTGAACGTGCCCAACGTTTATTACAGACATACAACAGTTTCCATGGATCTGGCTTAGGAGGACTTTTTCCAAACATACACCGGGAACTAGAAAGTCACGGATTTTCATTGCCAACCCATGAGCAATCATCACCCGTTGCTGAATCTCTGGATAATTCGGTATCGTCTGCTTTATCTCCACATTTGGAAACGCTGCGTCGCCGCAATCTCAGTATTCATCATCAACATATACAAGCTCACGAAATGGCACAAGAGTCCTTGGAGACCAGAAATCCAGGAAACATTTCTTCCTCGAGTGCTCCTTTAGCTAGTGATCAATCCCCGACTGTATCATCTAACCATATCCATGCATCCGGTAATTTGGCTTTAGGATCAAATTCGGGGTTAAATCCACGAAGCCCTAATTCTTTTTCTTCCCCATTGGACAATCCTGCACTTCATACTGTTGACTCAACCAACGTCAATGGTTCGCTTTCACCTCTATCAAATTCTTCATCTATCAATCAAGTTCACCAAAATGAAACTCATGGTTCTACCATCTCAGTACCTAACCCGAACTTAAGTCAAATGGGCCCTTCACACTCTTCTTCTGTGCCCTCAAATCTGTCACCAAATCCTGCCCAAAATGAAAATCCTAGTACAACCTCTATTCCATCAATAAATAATCAACCATTCCCGTCTGGTCTTTCTGCTTCTAACTCTAATTTTGCCTCTTCCTCGTTTATCCCTCAGAGCGTTCCGCAACTATTGCCTATATATTATCAAACTATCTTTTACAACGGTAACTATTATTTACAGCAGCTCCCATCTGCATCGCCTCCCACTATGTTTCGAGATCACTCATTTGCTCCTTTAGTTTCCCCTTCTATTGTTTCTCCGTATGGTGTTTTAGAAAATGAAGAAACTGGCGAATGTGCCTTTTTGTTTTCACCAAATGCCTCCCAACCCCATTTTCAGCCGAGAGCCCCCACGTTTGGCATTCCACGAAATGTGAGATCCCTCTTCACCTTACCGTTCTTCCATACTATCCGTAATATTGAGAGACATTTCCGATTATTTATTCGTTTAGCCCTGTTTTGTGTATTAACGACTTATAATGTTTCTCTCTCACAAACAATTTTGCTAACATCTATAATGTCTGTTGTGTTTTTGTTACAGACCGGTGCTCTAGCTCCATTCATCAATGACAATCCACTCATCCAGTCTGGTATGCGTCATATTCGTAATCTTCAAGATGAATACCGGCGGCGTCGAAACCGAACGGCTCAAAGAGTAGTTGAAATTCCTAATGAGACACAGACTGAAGATGAACAAGATGGTACAAATACGCCGGATAATAGGGCTGACGCTGAAGAACGCGAACTAACTCGAAGTCAAAGAATATACCGGACGGTAGTAAGAACCATAGTTGCCTTCGCATTAAGCTTTGTACCTCGAGCGTAACATTTTTTTCTTAAAAACAAAAGTGATAATATAGCTGTAAAGTCATTTTTCATTACTAATTAGAAATATTGTCTAAGCTTGCTATTAATAAAAGAGTTCTAGCTAGACTAAATTTTGACAGTGTCGAAATATCATTCCAAGGTACCCCATCAGATTAACAAATTTATAA\tMSEYRIRVTTVDQKVGIFQVPRTKTVLELKELIAVTFEAPADRLKLIHAGRVLRNETPLEEILHDATDLVTFHLVIAVFNSTSTTLPSATSSSVPQSRTSELSSTNSIPTPRITSLNPEELSRRERAQRLLQTYNSFHGSGLGGLFPNIHRELESHGFSLPTHEQSSPVAESLDNSVSSALSPHLETLRRRNLSIHHQHIQAHEMAQESLETRNPGNISSSSAPLASDQSPTVSSNHIHASGNLALGSNSGLNPRSPNSFSSPLDNPALHTVDSTNVNGSLSPLSNSSSINQVHQNETHGSTISVPNPNLSQMGPSHSSSVPSNLSPNPAQNENPSTTSIPSINNQPFPSGLSASNSNFASSSFIPQSVPQLLPIYYQTIFYNGNYYLQQLPSASPPTMFRDHSFAPLVSPSIVSPYGVLENEETGECAFLFSPNASQPHFQPRAPTFGIPRNVRSLFTLPFFHTIRNIERHFRLFIRLALFCVLTTYNVSLSQTILLTSIMSVVFLLQTGALAPFINDNPLIQSGMRHIRNLQDEYRRRRNRTAQRVVEIPNETQTEDEQDGTNTPDNRADAEERELTRSQRIYRTVVRTIVAFALSFVPRA*\n TRINITY_DN549_c0_g1\tTRINITY_DN549_c0_g1_i1\tGAA1_SCHPO^GAA1_SCHPO^Q:2-1363,H:121-574^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN549_c0_g1_i1|m.608\t2-1387[+]\tGAA1_SCHPO^GAA1_SCHPO^Q:1-461,H:121-581^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC1002.11`KO:K05289\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0042765^cellular_component^GPI-anchor transamidase complex`GO:0016255^biological_process^attachment of GPI anchor to protein\t.\tGTTTAACGGCTCGAATTTTATTACAACTTTACGTGCTCCTAGAGGTGATGCTACTGAGTCTCTACTCCTATGTGTCCCTTGGAAAGATCATATTGGGCAATATAATGAGGCCGGCGTTGCTTTAGCCATTAGTTTATTGAAATACTTCCAGGGATGGTCTCTTTGGTCGAAAGACATCATCTTGGTTATTTTTGATGATCCTGTTTATGGTCCATCGAGTTTTCTCACGTCTTATTTTGACCAAACGACGCCATATATATCTTACACACCATTGAAAATTCGTTCTGGTTCCATTCAAGCTGGTTTATCGCTTGAGCTAGTTACTACTGAAAATAACTCGGACGTGCTTGAAGTCCTTTACCAAGCTACCAATGGACAGTTGCCCAACCTTGACCTTTTCAATACTATTTCGAGAATCTTTATGCAACATTTTAATTATCCATTACGACTACAAGGATATGATTTCCATGCTAATTCGGGGTCCTCATACACATCCCGTCTAAAATCTTTGTGGATGGGTATGTTGACACAAGCAGTTTCGAATGTTACTAGTGCGCATGCACTATTTCCTCAATATAGAATTGACATGTTGACCTTGCGAATGAAAGTGAAAGACCCGTTTTCATTTGACATGTTTAGATTTGGTCAAGCCATTGAAAGCACATTTCGATCTCTTAACAATCTTCTTGAACACTTGCATCAATCTTTTTTCTTTTACTTCATCCTAGATCATTTGCATTTTATTAGTATCGGAAATTATATGCCTTCCATTTTAATTCTGGCTGCTAGTTTCATGCTCGGAGCTTACCGACATTGGATCAACCATGAAAAAAAAATAGATCTTTGGAGACCATTTTCTTTTTGGCTTTTTAGTATTTTTTGTACGATCGCCGCATATTATTTGGTATCTTCTTCTACTAAGATAACCGTTTTTATATTCCTCTATTTAATGCTCACTTTTATTGGTATTATCTTTTCAACTTTTATGACGAGTGAAGACGCTGAACTGGTTCTTTCTTACGATTTAATGTCAAAATCCCTTTTCATTTCCGTAGTGAGTACGTTAAATTTTTCTCTTTCTTTTGTTGTTGCAATTTTGCTTGTACCATTACAGTTTATTTCATTTAGATTTAATCGACGTCTTTCGTTACTTTTCGCAGTCTTGACATACTTTTCTACATTTATATTCCTTTGTTCTCTGTCTAAAATTTTGAATGGACCTTTGGTACCATTTTGGTTGTGGGCAAAAGAATATGAGTTGTTCAACTCTTGGTTAATGCCATCCGTGTTTATGATTTTAGTTTTACCTGAAATCATATTCTCAGTTACAAGTTTTTTTTCGCTTTGGAATGAACCGTCTGTGAAAACCAAGACAAAGACTTTGTGAGGGACTAAAAGGTACTATTGTATGAGTACTTATCATTATTTATTTTACGAACATCATTAAAAACTGTCTATAAAAAATATTACCGCAGAATTGTTACGTTCGCGGAAAGCTTTTAGAGGTAGAAATGCATTTCCG\tFNGSNFITTLRAPRGDATESLLLCVPWKDHIGQYNEAGVALAISLLKYFQGWSLWSKDIILVIFDDPVYGPSSFLTSYFDQTTPYISYTPLKIRSGSIQAGLSLELVTTENNSDVLEVLYQATNGQLPNLDLFNTISRIFMQHFNYPLRLQGYDFHANSGSSYTSRLKSLWMGMLTQAVSNVTSAHALFPQYRIDMLTLRMKVKDPFSFDMFRFGQAIESTFRSLNNLLEHLHQSFFFYFILDHLHFISIGNYMPSILILAASFMLGAYRHWINHEKKIDLWRPFSFWLFSIFCTIAAYYLVSSSTKITVFIFLYLMLTFIGIIFSTFMTSEDAELVLSYDLMSKSLFISVVSTLNFSLSFVVAILLVPLQFISFRFNRRLSLLFAVLTYFSTFIFLCSLSKILNGPLVPFWLWAKEYELFNSWLMPSVFMILVLPEIIFSVTSFFSLWNEPSVKTKTKTL*\n-TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGGTATTGTTCAAAAGGGAGTCCCTTTTCAAGTTCTGCATAGTGTAACACATCTAACAAACGCTTTGAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSSKRLLDVLHYAELEKGLPFEQYLVHPGLCTTNMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSCPSRTMYY*\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tYPIQKKHLTTEMLRQIPHLRLRTAKQGEIFRLRSDSLKALRQFFSSKDFTETNPPIITSSDCEGAGEVFTLTPQETHKNKSFERDDQKHFFDRPAFLTVSTQLHLEALALGLSRVYTISPAFRAEQSHTSRHLAEFWMLEAEVAFMTSLSQLTSLMEDMIKYTLNSLMEQNYHRDHWDQLLKPWKCMTYSEAIEELSAVKKTWKYPPKWGNDLSSEHEKYLCEILHKTPVFVTDYPQKIKPFYMKSSGPDTVAAVDLLVPQVGELAGGSLRKDHLDEYKTYPPELQWYLDLMKYSNAPHGGFGLGIERLIAFLEGENTNVKETIPFPRSVGSIFA*\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005737^cellular_component^cytoplasm`GO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0061630^molecular_function^ubiquitin protein ligase activity`GO:0031625^molecular_function^ubiquitin protein ligase binding`GO:0007031^biological_process^peroxisome organization`GO:0070647^biological_process^protein modification by small protein conjugation or removal\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tMNSSIKRLHKEYASFQAGDISGLLLLKPVTDVDMFHWKAVIEGPTETPYEGGQWVLDIHVHEGYPISPPSVYFQTKIVHPNISWTNGEVCMDILKTHWSPAWSLQSACLAIISLLSNYDASSPLNVDAAKLLRTGDKTAYNSLVRCTTYLYAKGKIEDLS*\n+TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGGTATTGTTCAAAAGGGAGTCCCTTTTCAAGTTCTGCATAGTGTAACACATCTAACAAACGCTTTGAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSSKRLLDVLHYAELEKGLPFEQYLVHPGLCTTNMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSCPSRTMYY*\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tYPIQKKHLTTEMLRQIPHLRLRTAKQGEIFRLRSDSLKALRQFFSSKDFTETNPPIITSSDCEGAGEVFTLTPQETHKNKSFERDDQKHFFDRPAFLTVSTQLHLEALALGLSRVYTISPAFRAEQSHTSRHLAEFWMLEAEVAFMTSLSQLTSLMEDMIKYTLNSLMEQNYHRDHWDQLLKPWKCMTYSEAIEELSAVKKTWKYPPKWGNDLSSEHEKYLCEILHKTPVFVTDYPQKIKPFYMKSSGPDTVAAVDLLVPQVGELAGGSLRKDHLDEYKTYPPELQWYLDLMKYSNAPHGGFGLGIERLIAFLEGENTNVKETIPFPRSVGSIFA*\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0007031^biological_process^peroxisome organization\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tMNSSIKRLHKEYASFQAGDISGLLLLKPVTDVDMFHWKAVIEGPTETPYEGGQWVLDIHVHEGYPISPPSVYFQTKIVHPNISWTNGEVCMDILKTHWSPAWSLQSACLAIISLLSNYDASSPLNVDAAKLLRTGDKTAYNSLVRCTTYLYAKGKIEDLS*\n TRINITY_DN540_c0_g1\tTRINITY_DN540_c0_g1_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:26-1567,H:2-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g1_i1|m.650\t350-1570[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-406,H:110-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\tATCTACCATTAATTGAAATACAAGGGGACAGGGGAGCAGTAAACACGCCGACAGCAAGCTCGATTCCTACCCATCCTTTTCTCGGTCAGACACTCAAGGATCCATTAAGTCTTTAAAGTCTCTGAAAACAGTCCTGGGGAAGGGCAAAGACTCGAATCATGATCGCCGAACGTCTACTGATACAACCCATAGTCGTCACCGGTATCCCGAAACACCACCAAGCCTTCCTCCTCCTCCTTCTCCTGGGATATTAGCTACATCTCCGGCTGTGTTGCAAAAGCATCAACAGGAGGATTCGGGAAATTCTTCTCAATCTCCGACGTCGCCGCATCCTTCTAACCAGCCTGCCATGCTTAGTCCCTCTACGGCCGCTTCCCAGCATCATCATCATCATTCTTCTTCTTCTTCTTATGCCGTTTCTCCAACATCACCTACCTCACCAACCAGCTCTGGCCCAATTGGCTCCAACTTTGATTCAGCTTCTGAGCATAATGGTCCTGTGTATCCACAAGATCAGCAGGGGCCCGTAATTATCCCTAATTCGGCAATTTCTTCTACGGATCCTGATGATCCTGAAACAGTTGTTAGCCTTAACGTAGATGAAATGATTCAAAGATTGATCCATGTTGGCTATTCTCGAAAAAGTAGTAAAAGTGTGTGTCTAAAGAATGCTGAAATTACCAGCATTTGTATGGCCGTACGAGAAATATTTCTTTCGCAGCCGACACTTTTAGAATTGACCCCGCCGGTGAAAATCGTTGGTGACGTTCATGGTCAATATTCCGATTTAATTCGACTGTTTGAAATGTGTGGGTTTCCTCCGAGCTCTAATTATTTATTCTTAGGCGATTATGTCGATCGTGGTAAGCAGAGTCTTGAAACCATCCTCTTGCTCTTTTTGTACAAGATCAGATATCCAGAAAACTTTTTCTTGCTACGGGGCAATCATGAATGCGCTAATATCACTCGCGTTTACGGTTTTTACGATGAGTGTAAAAGACGTTGCAATATCAAAATTTGGAAGACTTTCATAAACACCTTTAATTGCTTACCCATTGCGTCCGTTGTAGCCGGTAAAATTTTTTGCGTTCATGGTGGTCTTTCTCCATCGTTAAGTCATATGGATGACATACGGGAGATTCCTCGTCCTACTGACGTTCCTGACTATGGATTGCTGAATGATTTGTTATGGTCAGATCCTGCTGATACTGAAAACGATTGGGAAGATAATGAACGAGGTGTTTCTTTTGTTTTCAACAAAAACGTTATCCGGCAATTTTTGGCCAAGCACGACTTTGATTTAATCTGTCGTGCTCACATGGTTGTTGAAGACGGTTATGAATTTTTTAATGACAGAACTTTATGTACGGTGTTTTCTGCCCCTAATTACTGTGGGGAATTTGATAATTGGGGGGCCGTGATGAGTGTTAATTCAGAGCTGCTTTGTTCCTTTGAACTTATAAAGCCTCTCGACCAGGCAGCAATTCGACGAGAATTAAAAAAGAGCAAGCGTAGTGGTATGGCTATTTATCAATCTCCCCCTGCTGAGCAGGTTACCCAAAGCGTGTAAGCATTCTATTTTTTCTATTTTGGTCTCCATTTGAAATTATCTAGCGGTTTTATTGGGACACTATTTTATTCTCTTAAATGAGTATTATCTGTTATTATGTTATAGTCACTATGGTGATGGATCAAGTCCTTGGTTCACTTTATCGGTTCGAGGGCAATTCCATCCTGACACGTTTAGTATAAGTTTCTGTTTCTTTTAATAAAAACACAATCATTTCCATATAGTGACAGTTAACTTGCTTTTTATTTTTGCTAACCGACACCTGCTGAACATCGAAAGAGTAGGTAAAAAATTTACATGAGGAGGAAATAAACAATTATTATAGTATTATTATTAGCATTAAATAAATTTAAAGCGAAAATGT\tMLSPSTAASQHHHHHSSSSSYAVSPTSPTSPTSSGPIGSNFDSASEHNGPVYPQDQQGPVIIPNSAISSTDPDDPETVVSLNVDEMIQRLIHVGYSRKSSKSVCLKNAEITSICMAVREIFLSQPTLLELTPPVKIVGDVHGQYSDLIRLFEMCGFPPSSNYLFLGDYVDRGKQSLETILLLFLYKIRYPENFFLLRGNHECANITRVYGFYDECKRRCNIKIWKTFINTFNCLPIASVVAGKIFCVHGGLSPSLSHMDDIREIPRPTDVPDYGLLNDLLWSDPADTENDWEDNERGVSFVFNKNVIRQFLAKHDFDLICRAHMVVEDGYEFFNDRTLCTVFSAPNYCGEFDNWGAVMSVNSELLCSFELIKPLDQAAIRRELKKSKRSGMAIYQSPPAEQVTQSV*\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.651\t300-1847[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-515,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\tTTATACTACACTTCCCATCACTTGTTAACCCATTCTTCACCACGACCTTACAAATTTTTAAACGATTGTGTTTTGTCAGCAAAGCAAAACATTTTTATAGACCAGGATTACTTGAAGAAGAGAAAATCTACTAATTTACGGAAAATATCGAAAAATAGAAAAAATCTAGGTACAATTGGGAAACCTTTTAATTTTGGCACTAAATGCAAGCAACGAAGTCTCGATTTTGAAGTAAAAAAGAAAATAAAAATAGAACAAAGTGGAATAGTTTCCCGGAATCTACCATTAATTGAAATACAATGGGACAGGGGAGCAGTAAACACGCCGACAGCAAGCTCGATTCCTACCCATCCTTTTCTCGGTCAGACACTCAAGGATCCATTAAGTCTTTAAAGTCTCTGAAAACAGTCCTGGGGAAGGGCAAAGACTCGAATCATGATCGCCGAACGTCTACTGATACAACCCATAGTCGTCACCGGTATCCCGAAACACCACCAAGCCTTCCTCCTCCTCCTTCTCCTGGGATATTAGCTACATCTCCGGCTGTGTTGCAAAAGCATCAACAGGAGGATTCGGGAAATTCTTCTCAATCTCCGACGTCGCCGCATCCTTCTAACCAGCCTGCCATGCTTAGTCCCTCTACGGCCGCTTCCCAGCATCATCATCATCATTCTTCTTCTTCTTCTTATGCCGTTTCTCCAACATCACCTACCTCACCAACCAGCTCTGGCCCAATTGGCTCCAACTTTGATTCAGCTTCTGAGCATAATGGTCCTGTGTATCCACAAGATCAGCAGGGGCCCGTAATTATCCCTAATTCGGCAATTTCTTCTACGGATCCTGATGATCCTGAAACAGTTGTTAGCCTTAACGTAGATGAAATGATTCAAAGATTGATCCATGTTGGCTATTCTCGAAAAAGTAGTAAAAGTGTGTGTCTAAAGAATGCTGAAATTACCAGCATTTGTATGGCCGTACGAGAAATATTTCTTTCGCAGCCGACACTTTTAGAATTGACCCCGCCGGTGAAAATCGTTGGTGACGTTCATGGTCAATATTCCGATTTAATTCGACTGTTTGAAATGTGTGGGTTTCCTCCGAGCTCTAATTATTTATTCTTAGGCGATTATGTCGATCGTGGTAAGCAGAGTCTTGAAACCATCCTCTTGCTCTTTTTGTACAAGATCAGATATCCAGAAAACTTTTTCTTGCTACGGGGCAATCATGAATGCGCTAATATCACTCGCGTTTACGGTTTTTACGATGAGTGTAAAAGACGTTGCAATATCAAAATTTGGAAGACTTTCATAAACACCTTTAATTGCTTACCCATTGCGTCCGTTGTAGCCGGTAAAATTTTTTGCGTTCATGGTGGTCTTTCTCCATCGTTAAGTCATATGGATGACATACGGGAGATTCCTCGTCCTACTGACGTTCCTGACTATGGATTGCTGAATGATTTGTTATGGTCAGATCCTGCTGATACTGAAAACGATTGGGAAGATAATGAACGAGGTGTTTCTTTTGTTTTCAACAAAAACGTTATCCGGCAATTTTTGGCCAAGCACGACTTTGATTTAATCTGTCGTGCTCACATGGTTGTTGAAGACGGTTATGAATTTTTTAATGACAGAACTTTATGTACGGTGTTTTCTGCCCCTAATTACTGTGGGGAATTTGATAATTGGGGGGCCGTGATGAGTGTTAATTCAGAGCTGCTTTGTTCCTTTGAACTTATAAAGCCTCTCGACCAGGCAGCAATTCGACGAGAATTAAAAAAGAGCAAGCGTAGTGGTATGGCTATTTATCAATCTCCCCCTGCTGAGCAGGTTACCCAAAGCGTGTAAGCATTCTATTTTTTCTATTTTGGTCTCCATTTGAAATTATCTAGCGGTTTTATTGGGACACTATTTTATTCTCTTAAATGAGTATTATCTGTTATTATGTTATAGTCACTATGGTGATGGATCAAGTCCTTGGTTCACTTTATCGGTTCGAGGGCAATTCCATCCTGACACGTTTAGTATAAGTTTCTGTTTCTTTTAATAAAAACACAATCATTTCCATATAGTGACAGTTAACTTGCTTTTTATTTTTGCTAACCGACACCTGCTGAACATCGAAAGAGTAGGTAAAAAATTTACATGAGGAGGAAATAAACAATTATTATAGTATTATTATTAGCATTAAATAAATTTAAAGCGAAAATGT\tMGQGSSKHADSKLDSYPSFSRSDTQGSIKSLKSLKTVLGKGKDSNHDRRTSTDTTHSRHRYPETPPSLPPPPSPGILATSPAVLQKHQQEDSGNSSQSPTSPHPSNQPAMLSPSTAASQHHHHHSSSSSYAVSPTSPTSPTSSGPIGSNFDSASEHNGPVYPQDQQGPVIIPNSAISSTDPDDPETVVSLNVDEMIQRLIHVGYSRKSSKSVCLKNAEITSICMAVREIFLSQPTLLELTPPVKIVGDVHGQYSDLIRLFEMCGFPPSSNYLFLGDYVDRGKQSLETILLLFLYKIRYPENFFLLRGNHECANITRVYGFYDECKRRCNIKIWKTFINTFNCLPIASVVAGKIFCVHGGLSPSLSHMDDIREIPRPTDVPDYGLLNDLLWSDPADTENDWEDNERGVSFVFNKNVIRQFLAKHDFDLICRAHMVVEDGYEFFNDRTLCTVFSAPNYCGEFDNWGAVMSVNSELLCSFELIKPLDQAAIRRELKKSKRSGMAIYQSPPAEQVTQSV*\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.652\t1-393[+]\t.\t.\t.\t.\t.\t.\t.\t.\tTTATACTACACTTCCCATCACTTGTTAACCCATTCTTCACCACGACCTTACAAATTTTTAAACGATTGTGTTTTGTCAGCAAAGCAAAACATTTTTATAGACCAGGATTACTTGAAGAAGAGAAAATCTACTAATTTACGGAAAATATCGAAAAATAGAAAAAATCTAGGTACAATTGGGAAACCTTTTAATTTTGGCACTAAATGCAAGCAACGAAGTCTCGATTTTGAAGTAAAAAAGAAAATAAAAATAGAACAAAGTGGAATAGTTTCCCGGAATCTACCATTAATTGAAATACAATGGGACAGGGGAGCAGTAAACACGCCGACAGCAAGCTCGATTCCTACCCATCCTTTTCTCGGTCAGACACTCAAGGATCCATTAAGTCTTTAAAGTCTCTGAAAACAGTCCTGGGGAAGGGCAAAGACTCGAATCATGATCGCCGAACGTCTACTGATACAACCCATAGTCGTCACCGGTATCCCGAAACACCACCAAGCCTTCCTCCTCCTCCTTCTCCTGGGATATTAGCTACATCTCCGGCTGTGTTGCAAAAGCATCAACAGGAGGATTCGGGAAATTCTTCTCAATCTCCGACGTCGCCGCATCCTTCTAACCAGCCTGCCATGCTTAGTCCCTCTACGGCCGCTTCCCAGCATCATCATCATCATTCTTCTTCTTCTTCTTATGCCGTTTCTCCAACATCACCTACCTCACCAACCAGCTCTGGCCCAATTGGCTCCAACTTTGATTCAGCTTCTGAGCATAATGGTCCTGTGTATCCACAAGATCAGCAGGGGCCCGTAATTATCCCTAATTCGGCAATTTCTTCTACGGATCCTGATGATCCTGAAACAGTTGTTAGCCTTAACGTAGATGAAATGATTCAAAGATTGATCCATGTTGGCTATTCTCGAAAAAGTAGTAAAAGTGTGTGTCTAAAGAATGCTGAAATTACCAGCATTTGTATGGCCGTACGAGAAATATTTCTTTCGCAGCCGACACTTTTAGAATTGACCCCGCCGGTGAAAATCGTTGGTGACGTTCATGGTCAATATTCCGATTTAATTCGACTGTTTGAAATGTGTGGGTTTCCTCCGAGCTCTAATTATTTATTCTTAGGCGATTATGTCGATCGTGGTAAGCAGAGTCTTGAAACCATCCTCTTGCTCTTTTTGTACAAGATCAGATATCCAGAAAACTTTTTCTTGCTACGGGGCAATCATGAATGCGCTAATATCACTCGCGTTTACGGTTTTTACGATGAGTGTAAAAGACGTTGCAATATCAAAATTTGGAAGACTTTCATAAACACCTTTAATTGCTTACCCATTGCGTCCGTTGTAGCCGGTAAAATTTTTTGCGTTCATGGTGGTCTTTCTCCATCGTTAAGTCATATGGATGACATACGGGAGATTCCTCGTCCTACTGACGTTCCTGACTATGGATTGCTGAATGATTTGTTATGGTCAGATCCTGCTGATACTGAAAACGATTGGGAAGATAATGAACGAGGTGTTTCTTTTGTTTTCAACAAAAACGTTATCCGGCAATTTTTGGCCAAGCACGACTTTGATTTAATCTGTCGTGCTCACATGGTTGTTGAAGACGGTTATGAATTTTTTAATGACAGAACTTTATGTACGGTGTTTTCTGCCCCTAATTACTGTGGGGAATTTGATAATTGGGGGGCCGTGATGAGTGTTAATTCAGAGCTGCTTTGTTCCTTTGAACTTATAAAGCCTCTCGACCAGGCAGCAATTCGACGAGAATTAAAAAAGAGCAAGCGTAGTGGTATGGCTATTTATCAATCTCCCCCTGCTGAGCAGGTTACCCAAAGCGTGTAAGCATTCTATTTTTTCTATTTTGGTCTCCATTTGAAATTATCTAGCGGTTTTATTGGGACACTATTTTATTCTCTTAAATGAGTATTATCTGTTATTATGTTATAGTCACTATGGTGATGGATCAAGTCCTTGGTTCACTTTATCGGTTCGAGGGCAATTCCATCCTGACACGTTTAGTATAAGTTTCTGTTTCTTTTAATAAAAACACAATCATTTCCATATAGTGACAGTTAACTTGCTTTTTATTTTTGCTAACCGACACCTGCTGAACATCGAAAGAGTAGGTAAAAAATTTACATGAGGAGGAAATAAACAATTATTATAGTATTATTATTAGCATTAAATAAATTTAAAGCGAAAATGT\tLYYTSHHLLTHSSPRPYKFLNDCVLSAKQNIFIDQDYLKKRKSTNLRKISKNRKNLGTIGKPFNFGTKCKQRSLDFEVKKKIKIEQSGIVSRNLPLIEIQWDRGAVNTPTASSIPTHPFLGQTLKDPLSL*\n-TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTTCGTACGGATCATGACAGTTATTTTATCGTTCTCTTCATCCTGTTTTTTTTTGTTTTTTAATCGATATTTGATATCGTCAAATATTTCTATCTATATGGAGAGTTCTTATATTTGTTTTTTGAATAATCACACACTCTTCAAAACAGCAGCTGCATTCAATAATGTACTTGGCATTGGAGTAACGCACATTATACAAGAATTACCGTGGTCCGTAAACACTTGCAAGCATGAAATACTTAGAAAGCTTTAGGCAGACGCTAGACGTGGGGAAAGGGATGAAGTGTTAAAAATTCAAGAGTGTGGATGTTTTGTATGCAAACTGAGTGAGTTCTCGGTTTTTGTAGTTACGTGACATAACTTTAAAAATTTTATGAGAAATTGACTAATATCAAGAATTGAGAATAGTTTTTTAATACAATACACAAAATTTTTAC\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n-TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTCGTACGGATCATGA\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n+TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTTCGTACGGATCATGACAGTTATTTTATCGTTCTCTTCATCCTGTTTTTTTTTGTTTTTTAATCGATATTTGATATCGTCAAATATTTCTATCTATATGGAGAGTTCTTATATTTGTTTTTTGAATAATCACACACTCTTCAAAACAGCAGCTGCATTCAATAATGTACTTGGCATTGGAGTAACGCACATTATACAAGAATTACCGTGGTCCGTAAACACTTGCAAGCATGAAATACTTAGAAAGCTTTAGGCAGACGCTAGACGTGGGGAAAGGGATGAAGTGTTAAAAATTCAAGAGTGTGGATGTTTTGTATGCAAACTGAGTGAGTTCTCGGTTTTTGTAGTTACGTGACATAACTTTAAAAATTTTATGAGAAATTGACTAATATCAAGAATTGAGAATAGTTTTTTAATACAATACACAAAATTTTTAC\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n+TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTCGTACGGATCATGA\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n TRINITY_DN547_c0_g1\tTRINITY_DN547_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tCTTCAACAATGAGTTTATAAGAACAACGCAATAATATTGCGGTTGAAATTGTAATTGTTAATCATGTCAGGAGATCTACTTTTAAATTGGTGGTTGTACTAGCTCGAGTAGTTTTCTTTTTAGACTTCCTTTTGTATGGAAACGTGCAATTTAATGCATGTGAATATAACCAGAGTTTTAGTGATGGAGGAATAGCCTTCCGATCTCCATAGCTTAATAGAAATGTTATTTTTA\t.\n********\n*SNIP *\n********\n TRINITY_DN5472_c0_g1\tTRINITY_DN5472_c0_g1_i1\tYF11_SCHPO^YF11_SCHPO^Q:2-94,H:69-99^100%ID^E:3e-14^RecName: Full=Putative uncharacterized protein C9E9.01;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC9E9.01\tGO:0016021^cellular_component^integral component of membrane\t.\tAGACAATGAAGATCGTTGTCGGTTATACCAAATTTACCGGGAAAGCAAACTCAAGGAATTTGCGAGATACAATGTCCTTTCAAGGGTTAATTGGTAAAGTAAACGACTATGAAATGAAAGAAATATCGTCGTTTAATTGAAATGGAGGAAGGATATTTCTCTTCTTTGTGCTAGTAAAGAGAAGACGGATATTTTTGGAAGAGATGTAAAAAGACAAGCAAACAAGAAAATGAAAATGGACGTCTAGCGATTTTACAAGAAAAAATTAGACGGGAAAAGGGAAAAGAAAAACTAAAGCTTATAG\t.\n-TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\tTAGAAAACCCTCCTAAATGCAATAACAGTGGCAATCACTCAGATAACCTAGGTTTTATCGAGCAGTCAGAAACAGTACATCCAGAGAATGAGAAGGCTTTGACGCCGGACCTGCGAGATACAAAAATACATACTTCTTTACCAATTACTACACCTTTTTCAAAGAAAAGGGCTCGAGAAGCCAAAAATATACTTCTTAAGCCGTTCAAAAGCCCCTTAAGACAAACTGCCAGTCCTCAAGTTGCAGACACTAATCTAAAACCATCGTTAGCTGTAACAAATCTTAACTCTGATGAAACAAACACTTCTTCAGAACCTGTTACTTCTCCCTTAAGAACAACGCCTAATTCAATAAAACGGCAAAAACGACTTTTCAAGTCACCTATCTCTAATTGCCTTAATCCAAAGTCCGACCCTGAAATCACACAGTTGCTTTCTCGTCGTTTAAAACTTGAAAAAGAAGTCAGAAATTTACAAGAGCAGCTCATAACCGCCGAAACTGCACGAAAGGTTGAAGCCAAAAATGAAGATAAAGACTTGCAAACGTTGATACAGAAATGGAAAAATGCTGCTCAGCAAGCCA\tENPPKCNNSGNHSDNLGFIEQSETVHPENEKALTPDLRDTKIHTSLPITTPFSKKRAREAKNILLKPFKSPLRQTASPQVADTNLKPSLAVTNLNSDETNTSSEPVTSPLRTTPNSIKRQKRLFKSPISNCLNPKSDPEITQLLSRRLKLEKEVRNLQEQLITAETARKVEAKNEDKDLQTLIQKWKNAAQQA\n-TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005622^cellular_component^intracellular`GO:0005634^cellular_component^nucleus`GO:0047433^molecular_function^branched-chain-2-oxoacid decarboxylase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0000949^biological_process^aromatic amino acid family catabolic process to alcohol via Ehrlich pathway`GO:0006067^biological_process^ethanol metabolic process`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006559^biological_process^L-phenylalanine catabolic process`GO:0006090^biological_process^pyruvate metabolic process`GO:0006569^biological_process^tryptophan catabolic process\t.\tGAATTTTTGGCACCATAGCCTTTAAGAAGGTTTTGATAGTCCCATTCAGTGTTGATGTCGTTGTAAACAGCGTGAAGACCGTGAATGAGACGTTCAATTGTATATCCCTTATTGTTAATTACAAATATGATAGGGGTGAGACCGTTACGGATGGTAGCAGAAATTTCCTGAACAGTTAAATGAAGGGAACCGTCACCAACAATCAATATGGAACGACGG\t.\n+TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07`KO:K10992\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0062110^biological_process^negative regulation of DNA recombinase disassembly`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\tTAGAAAACCCTCCTAAATGCAATAACAGTGGCAATCACTCAGATAACCTAGGTTTTATCGAGCAGTCAGAAACAGTACATCCAGAGAATGAGAAGGCTTTGACGCCGGACCTGCGAGATACAAAAATACATACTTCTTTACCAATTACTACACCTTTTTCAAAGAAAAGGGCTCGAGAAGCCAAAAATATACTTCTTAAGCCGTTCAAAAGCCCCTTAAGACAAACTGCCAGTCCTCAAGTTGCAGACACTAATCTAAAACCATCGTTAGCTGTAACAAATCTTAACTCTGATGAAACAAACACTTCTTCAGAACCTGTTACTTCTCCCTTAAGAACAACGCCTAATTCAATAAAACGGCAAAAACGACTTTTCAAGTCACCTATCTCTAATTGCCTTAATCCAAAGTCCGACCCTGAAATCACACAGTTGCTTTCTCGTCGTTTAAAACTTGAAAAAGAAGTCAGAAATTTACAAGAGCAGCTCATAACCGCCGAAACTGCACGAAAGGTTGAAGCCAAAAATGAAGATAAAGACTTGCAAACGTTGATACAGAAATGGAAAAATGCTGCTCAGCAAGCCA\tENPPKCNNSGNHSDNLGFIEQSETVHPENEKALTPDLRDTKIHTSLPITTPFSKKRAREAKNILLKPFKSPLRQTASPQVADTNLKPSLAVTNLNSDETNTSSEPVTSPLRTTPNSIKRQKRLFKSPISNCLNPKSDPEITQLLSRRLKLEKEVRNLQEQLITAETARKVEAKNEDKDLQTLIQKWKNAAQQA\n+TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016831^molecular_function^carboxy-lyase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006090^biological_process^pyruvate metabolic process\t.\tGAATTTTTGGCACCATAGCCTTTAAGAAGGTTTTGATAGTCCCATTCAGTGTTGATGTCGTTGTAAACAGCGTGAAGACCGTGAATGAGACGTTCAATTGTATATCCCTTATTGTTAATTACAAATATGATAGGGGTGAGACCGTTACGGATGGTAGCAGAAATTTCCTGAACAGTTAAATGAAGGGAACCGTCACCAACAATCAATATGGAACGACGG\t.\n TRINITY_DN5486_c0_g1\tTRINITY_DN5486_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tCGCTTTATCGGAAAGACTTTGGCAATTTTACCGATTCTCTAGAATCATGTAAAGTGACGAAAACAGAAGGCAGAAAGTCCAACCAAATATGGGCAAAAAAAACAACCACTTCAATAAATTGTTCCAGTTAACAGCAGTTTAGTTGTGTTCAAGTCAATTACCAGCGTTTCGCGTGAAACGTTACAGAAGTGTAAATAATGAAATACTGTTGCTTTCTCTTTGACTCTCA\t.\n TRINITY_DN5468_c0_g1\tTRINITY_DN5468_c0_g1_i1\tCDC37_SCHPO^CDC37_SCHPO^Q:280-5,H:209-300^98.91%ID^E:6e-41^RecName: Full=Hsp90 co-chaperone Cdc37;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPBC9B6.10`KO:K09554\tGO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0051087^molecular_function^chaperone binding`GO:0031072^molecular_function^heat shock protein binding`GO:0019901^molecular_function^protein kinase binding`GO:0051082^molecular_function^unfolded protein binding`GO:0007049^biological_process^cell cycle`GO:0051301^biological_process^cell division`GO:0061077^biological_process^chaperone-mediated protein folding`GO:0006457^biological_process^protein folding`GO:0050821^biological_process^protein stabilization\t.\tCCGGTGCTAAGGACGGCTTCCCTTCTAGTTCAGCATTAAAGGCTTCCAAAAGAATAGCATCAGATTCTTCCTCATCTGCCAAGATATTTAAATGTTCTTTCAAAAACTCTTCCGATGAAGAATAATCACCAAAATCAATATTGGCAAACTTCTCACCATCTTCAGAAAGAGACAACCCTTCTTCATCTTCCGACTCATTCTCATTGACAGGATTTTTAGGGGCATTACCAGTAGAAGCTTGCTCTTTAGCGCCGGGAAAATCAGTAGGAGGTGTAGGGTT\t.\n-TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\tTTTCACCTCTCACAAACCATTGGACCATATGGCTTACATCAAATAAGCCAGAATGCTCCCGTGTCCACTTATGACTGGCACTAATGGTTTGACCTTTATACTGTACTGGCATTGAAAAACCAGCAAAGGGGACTATGGTAGCTCCTTCTTTTAAATGCAGATCATATAAAGGAGTTCTCTTGAGTGAAGAAGAAGCTGCCGTAGAACTTTGGCGCTTTAAGATTGATAAAGCAGCAGAA\t.\n+TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\tTTTCACCTCTCACAAACCATTGGACCATATGGCTTACATCAAATAAGCCAGAATGCTCCCGTGTCCACTTATGACTGGCACTAATGGTTTGACCTTTATACTGTACTGGCATTGAAAAACCAGCAAAGGGGACTATGGTAGCTCCTTCTTTTAAATGCAGATCATATAAAGGAGTTCTCTTGAGTGAAGAAGAAGCTGCCGTAGAACTTTGGCGCTTTAAGATTGATAAAGCAGCAGAA\t.\n TRINITY_DN5400_c0_g1\tTRINITY_DN5400_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tAAAGTCCTCCTTATTTTTTTTGCGTTGAAATGTAGTTTGTAGTGTCAAGCTGCTTTGTGCTATTTATCGAGCAATGAGAAATAGGAGAGGAGTTATGGTAGCAGGGTATACTTAGTAATGTTTTCATGGACCAAGCGGGGGAGGGGCGAAAAGCATGTCTAGTTCGCTGTATTCAGTTTCAAGACCCCATTTCATGTTTAACATTTGTTCTAGATACGCAAAAATCCCTTTTTTCTAGATTGTTTGCAAGTCCC\t.\n TRINITY_DN5476_c0_g1\tTRINITY_DN5476_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGTTTGTTTACACCTATGTGTGATCCACTTTTGAGGTATATACTGGTTCAAAAACTCGAAAACTCTTCTTTTATCCATCTTGTCATAACTTTTGCTTTTTTATTTTTATTTTTACCCATTCCCACATTTCTTATACTAAAAGGAGAGGTTTACTATTGACATTCTTGACGTTTATTAGACCCTTGATTCTAACGCTGAAGGGTGGTTATCTTTGGTTTTCTTGTCAGGTATACGAGATAAGTAGCCTTTTTATAATTCTCCCTGAATTCATTACTTCTCCTCTGCCTTAATATACTATTTTGTTTAGTCTTGTATGCAAGGCGCTTAATCCTCTTCTTCAAGTATTATTCCTCACAGTTCATTTCGCTTTTTGGTTTAAGTTGACCTAATTCATGCCC\t.\n TRINITY_DN5490_c0_g1\tTRINITY_DN5490_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tCATGTCTGCTGCATTCACGATCTCCAAGAAGAGAAAAGTAAGTAATCTTTTATTAGAAAAAAAATCGATGTTTGTGTAGGGATATATATGATGATTTCCAATATCTGGCACCTCTTGTCTTATGGGATTCTTAAAAGTTTCTCGATGAACTTAACCTTTTGGAACTATATCTGATATCAACGTATATTGTTAAACGATAATAATGTAAAATTATATTTTTCTACGCATAAGATCTTTTTTCCTAATATCGTTCACTTTGTGCTTCTTTCTAAAGTACTGACCAATTTAGTTCGTTGCTGATGGTGTTTTCTACGCCGAGTTG\t.\n-TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0098713^biological_process^leucine import across plasma membrane\t.\tGCCGACAATTAGACCCACCGGACCACCAATATGCAGGGCACTGCCCATACCAATGAAGACACCTGGACCGATGATACCAGCAAGGGCCATCATCGATACATGACGAGTCTTCAATCCACGTTTGATTCCATTTGAATTATTTTCCGCTGCGTCAACTGTTACAACTTCGGAAACTCCAATGTCGAGGTTTTCTGAGGAGGATAGCTTCTTGGCAGGAACTTTCTCAGAAGGCGCTGCCTCCATTTTCTTAATTAACTTAATTCACTTTATTCTTAATATTAGAAATTACAATTGGTATTGTGATCTATATATGTACAACAGAATTGATAAATGTAATCAATTCTAATAATGTTAGCTGCAGAATCAACATGAATTCTACCGCTAA\t.\n+TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015171^molecular_function^amino acid transmembrane transporter activity`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0003333^biological_process^amino acid transmembrane transport`GO:0098713^biological_process^leucine import across plasma membrane\t.\tGCCGACAATTAGACCCACCGGACCACCAATATGCAGGGCACTGCCCATACCAATGAAGACACCTGGACCGATGATACCAGCAAGGGCCATCATCGATACATGACGAGTCTTCAATCCACGTTTGATTCCATTTGAATTATTTTCCGCTGCGTCAACTGTTACAACTTCGGAAACTCCAATGTCGAGGTTTTCTGAGGAGGATAGCTTCTTGGCAGGAACTTTCTCAGAAGGCGCTGCCTCCATTTTCTTAATTAACTTAATTCACTTTATTCTTAATATTAGAAATTACAATTGGTATTGTGATCTATATATGTACAACAGAATTGATAAATGTAATCAATTCTAATAATGTTAGCTGCAGAATCAACATGAATTCTACCGCTAA\t.\n TRINITY_DN5471_c0_g1\tTRINITY_DN5471_c0_g1_i1\tRRG1_SCHPO^RRG1_SCHPO^Q:214-101,H:266-303^100%ID^E:3e-21^RecName: Full=Protein-lysine N-methyltransferase rrg1 {ECO:0000250|UniProtKB:P38347, ECO:0000305};^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC338.11c\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016279^molecular_function^protein-lysine N-methyltransferase activity`GO:2000765^biological_process^regulation of cytoplasmic translation\t.\tAAATTATTCAAACAAGGAAAAAAATAAAACCCAATTATTTTGGCTCGCTTTTTCTAGTTTGCAACATGGTTTGACCTATAGATTGATACATTTTAAGTTATCCATGGTATTTCCATCTACTCCAACGACAAGTTACTGGGTATAGAGAGCCAAAGTCTTCTTCACCAATTTCTTCTCCCATTTGTCTTTCAAATCCGGCGGCGTCCATTCCCTTTT\t.\n-TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC1851.03`KO:K03115\tGO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0043433^biological_process^negative regulation of DNA-binding transcription factor activity`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0007165^biological_process^signal transduction\t.\tGCGATCCTGGATGAAGTCTTCGTCGACTTCGCAAAAAAATTCGTTTCCTTTTAGTCCTAAAAACCAATCAACCCAATATTGAGAATCATCGGATTCACTGCAATTAATGATTAATAAACGGTCCAATATGGCCAAAATGTAAAATATTTTTTTATTTAACCCTAATTTGCACAAACCTTTCCGAAGAGTATAGCTGCATGCTTCGTTCCG\t.\n+TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGO:0005737^cellular_component^cytoplasm`GO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0019887^molecular_function^protein kinase regulator activity`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0030291^molecular_function^protein serine/threonine kinase inhibitor activity`GO:0006974^biological_process^cellular response to DNA damage stimulus`GO:0035690^biological_process^cellular response to drug`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0018105^biological_process^peptidyl-serine phosphorylation`GO:0006356^biological_process^regulation of transcription by RNA polymerase I`GO:0007165^biological_process^signal transduction\t.\tGCGATCCTGGATGAAGTCTTCGTCGACTTCGCAAAAAAATTCGTTTCCTTTTAGTCCTAAAAACCAATCAACCCAATATTGAGAATCATCGGATTCACTGCAATTAATGATTAATAAACGGTCCAATATGGCCAAAATGTAAAATATTTTTTTATTTAACCCTAATTTGCACAAACCTTTCCGAAGAGTATAGCTGCATGCTTCGTTCCG\t.\n TRINITY_DN5498_c0_g1\tTRINITY_DN5498_c0_g1_i1\tSRP2_SCHPO^SRP2_SCHPO^Q:213-145,H:22-44^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`SRP2_SCHPO^SRP2_SCHPO^Q:91-56,H:44-55^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC16.02c`KO:K12893\tGO:0005634^cellular_component^nucleus`GO:0003729^molecular_function^mRNA binding`GO:0008143^molecular_function^poly(A) binding`GO:0001069^molecular_function^regulatory region RNA binding`GO:0045292^biological_process^mRNA cis splicing, via spliceosome\t.\tAAGCATACGAAAGTCTAAACTGTTAGCAAAAGTTTTAAGATGGTATGAACGTACCATTTACGATATCACGAGCATCGCGAGCGTCTTCAACCTACATCAAGTTAGTCACATAGGGATTACAAAAAGAAGGCTAATGGTTTTATTTACCTCAACAAAGCCAAATCCATTCATCAGCTTGCAATCAAGGATTTGACCGTAACCTTTAAAAAAATCCA\t.\n-TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFLIKEVTHLLKLWHEVSVRAGVAQTSKPSFDAPLVLEAIEKYKNALGAPDTATKSCKENLISLDSEIRQTY*\n-TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFPNKRSYPSIKIMA*\n+TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFLIKEVTHLLKLWHEVSVRAGVAQTSKPSFDAPLVLEAIEKYKNALGAPDTATKSCKENLISLDSEIRQTY*\n+TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFPNKRSYPSIKIMA*\n TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8862\t1057-1950[-]\tRNG3_SCHPO^RNG3_SCHPO^Q:1-297,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.04c`KO:K21991\tGO:0005826^cellular_component^actomyosin contractile ring`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0005844^cellular_component^polysome`GO:0017022^molecular_function^myosin binding`GO:0034613^biological_process^cellular protein localization`GO:0000917^biological_process^division septum assembly`GO:1903475^biological_process^mitotic actomyosin contractile ring assembly\t.\tAGAACTTGACCGCAGCGCACATCAACTACACCTTGAGGCAATTCAAAATATGAAGTTTTCTACCGTAGGGTTTTTGTTTTCAACAATTTTATTCAAAAGTGCGTTTGCAGGATGGATGGATACACACATGAAAGATGAACATCACATTGATAAGTACACAGATGAATCTTTTTTTCGTCTTCACGATTTAGGAAAAAAGGGATACTGGTCAGATCAAGATATACTTAGTTTATATGGATTATTTGAAAATGATGAGGTCCCATTCGTCAAAAAGAATGAGGTTTTGGTTGATGTTCTAAAAAAATGCGATCCATCGGGCAATCGCCGGATAACACTTGATGAATTTCTTGCATTTCGTAAAAATGGAGGAGAGTTAACGGATTTTGGATTTCCAGGGCACCATGGTGATGAGGAAGAGGAATTTGAAATGCATCATGTGGAAAAATATCACCCAGCTGGCCTCGATGAACCAGATGAAAACTGGAATCATCCTGAAGATATCGAACATTTTCAAAAACACGATGAGATATTTCATGGTGATAAGAAACCTGAGGAACGACGCAAACATTTTGTAAAATACAATAATATCCCTGACAAGTATCGTCGTGTGAGCATTTAAATTTTCCCATTGATAGATTTACTTGCTTCTCCTTGGGACGTTTTATTTTTATTTTTATTTTTCCGTTTATTTTTGGATTGTCGTGTTCCGATATATATCCGTGTTTACAAACAATAGCTTCAACATTTTTGAACGGGTCCTTCCAATACTAATGCCATATCTTATTTCTTTGTCAATACAAAATCACTTTTGTTCGTTAAACTCATTTTGGTTTGCAGATATTATCATATTCTTACCAATTATACATACACTCACAATTTGTTTTACATTTCCGCTCCCTTCTTCAATAAGCTGATTGCTTATAAAAAATCAAACATTCATATGATTCAAGTTCAAATATTATCATCTGAAAACTGACAAGTTCCATTCATTACTATATTAATATCATTTATTCGTTAAACCGATATCATTAAAAATGGAAAATTAAAGAAATCAATTCATTTGCTTCCTTGAAGTCTAGAGTCGATCATAACGAGAGCTTCATGTTGAAGCTCTTTAAGTGCAGCTTGCTTAGTAATATAAGTGCGTAATGTTTCAACAGCCTTTGGTGTTTTTATGAACTTGTTAAATATTTCTTGATCTTTGGAATATACGATATTGCATATGCAAACTAATCCACGATGTTGAATTCCTTCATCTTGGTCGGTTAACATTCTGACTATGTAGTTAAAATCGTTCTGGAGACTTAAAATTTTCTTGCACCCTTCATCAACACTAGTAATTTGAACAAGAATTCCGCATGCTGCTAATCTCGTGGGAGTGTCTTCGGTATCTGAGAGTGCTAGTACAATGTGTAAACGAGTATTTTCAAAATCCGAGTCCTTGTCTCCTATAAACTTAATTAAGCAATAAGGTGATAAAGAAAGGTTGTTAATAAGTTCCGTGGTTGCCCTCTGAATTAAAGGATTTGTTTCTATAATTAGTTCATCCAATTCTCTCCAACATTCTTGAACTATGGCTTGCCGACTTTCTTCATCATGTGAAGCAAGATTTGTTAGCGCAAGTAACACTTCAAATTTCCCCAAGAGAATTGGATACTCAGTATCAGCAGAATTAGTACTTAGTAACTTTGACATAGGATGAATTGCTCGGTTCGATGGAAACGCCTTAGTAAATATACTATGAGGAGCCACACTGATAAGAATTTTCGCTAATGCAAAAGCCGCATTCGAATCCTGGGTTTGGATAAAACATTGTCTAGTAAGGAATTTCAATGCACCTATTTGTACTAGCTTGCGACGTGCCAGCTTGTAATTAGCCATGTGTACAAGTATCATTGCTAATAACCCGTTTAGGCTATTATAGTGCTTGGCCGCTGAAAAGAGTTTTGAC\tSKLFSAAKHYNSLNGLLAMILVHMANYKLARRKLVQIGALKFLTRQCFIQTQDSNAAFALAKILISVAPHSIFTKAFPSNRAIHPMSKLLSTNSADTEYPILLGKFEVLLALTNLASHDEESRQAIVQECWRELDELIIETNPLIQRATTELINNLSLSPYCLIKFIGDKDSDFENTRLHIVLALSDTEDTPTRLAACGILVQITSVDEGCKKILSLQNDFNYIVRMLTDQDEGIQHRGLVCICNIVYSKDQEIFNKFIKTPKAVETLRTYITKQAALKELQHEALVMIDSRLQGSK*\n-TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\tAGAACTTGACCGCAGCGCACATCAACTACACCTTGAGGCAATTCAAAATATGAAGTTTTCTACCGTAGGGTTTTTGTTTTCAACAATTTTATTCAAAAGTGCGTTTGCAGGATGGATGGATACACACATGAAAGATGAACATCACATTGATAAGTACACAGATGAATCTTTTTTTCGTCTTCACGATTTAGGAAAAAAGGGATACTGGTCAGATCAAGATATACTTAGTTTATATGGATTATTTGAAAATGATGAGGTCCCATTCGTCAAAAAGAATGAGGTTTTGGTTGATGTTCTAAAAAAATGCGATCCATCGGGCAATCGCCGGATAACACTTGATGAATTTCTTGCATTTCGTAAAAATGGAGGAGAGTTAACGGATTTTGGATTTCCAGGGCACCATGGTGATGAGGAAGAGGAATTTGAAATGCATCATGTGGAAAAATATCACCCAGCTGGCCTCGATGAACCAGATGAAAACTGGAATCATCCTGAAGATATCGAACATTTTCAAAAACACGATGAGATATTTCATGGTGATAAGAAACCTGAGGAACGACGCAAACATTTTGTAAAATACAATAATATCCCTGACAAGTATCGTCGTGTGAGCATTTAAATTTTCCCATTGATAGATTTACTTGCTTCTCCTTGGGACGTTTTATTTTTATTTTTATTTTTCCGTTTATTTTTGGATTGTCGTGTTCCGATATATATCCGTGTTTACAAACAATAGCTTCAACATTTTTGAACGGGTCCTTCCAATACTAATGCCATATCTTATTTCTTTGTCAATACAAAATCACTTTTGTTCGTTAAACTCATTTTGGTTTGCAGATATTATCATATTCTTACCAATTATACATACACTCACAATTTGTTTTACATTTCCGCTCCCTTCTTCAATAAGCTGATTGCTTATAAAAAATCAAACATTCATATGATTCAAGTTCAAATATTATCATCTGAAAACTGACAAGTTCCATTCATTACTATATTAATATCATTTATTCGTTAAACCGATATCATTAAAAATGGAAAATTAAAGAAATCAATTCATTTGCTTCCTTGAAGTCTAGAGTCGATCATAACGAGAGCTTCATGTTGAAGCTCTTTAAGTGCAGCTTGCTTAGTAATATAAGTGCGTAATGTTTCAACAGCCTTTGGTGTTTTTATGAACTTGTTAAATATTTCTTGATCTTTGGAATATACGATATTGCATATGCAAACTAATCCACGATGTTGAATTCCTTCATCTTGGTCGGTTAACATTCTGACTATGTAGTTAAAATCGTTCTGGAGACTTAAAATTTTCTTGCACCCTTCATCAACACTAGTAATTTGAACAAGAATTCCGCATGCTGCTAATCTCGTGGGAGTGTCTTCGGTATCTGAGAGTGCTAGTACAATGTGTAAACGAGTATTTTCAAAATCCGAGTCCTTGTCTCCTATAAACTTAATTAAGCAATAAGGTGATAAAGAAAGGTTGTTAATAAGTTCCGTGGTTGCCCTCTGAATTAAAGGATTTGTTTCTATAATTAGTTCATCCAATTCTCTCCAACATTCTTGAACTATGGCTTGCCGACTTTCTTCATCATGTGAAGCAAGATTTGTTAGCGCAAGTAACACTTCAAATTTCCCCAAGAGAATTGGATACTCAGTATCAGCAGAATTAGTACTTAGTAACTTTGACATAGGATGAATTGCTCGGTTCGATGGAAACGCCTTAGTAAATATACTATGAGGAGCCACACTGATAAGAATTTTCGCTAATGCAAAAGCCGCATTCGAATCCTGGGTTTGGATAAAACATTGTCTAGTAAGGAATTTCAATGCACCTATTTGTACTAGCTTGCGACGTGCCAGCTTGTAATTAGCCATGTGTACAAGTATCATTGCTAATAACCCGTTTAGGCTATTATAGTGCTTGGCCGCTGAAAAGAGTTTTGAC\tELDRSAHQLHLEAIQNMKFSTVGFLFSTILFKSAFAGWMDTHMKDEHHIDKYTDESFFRLHDLGKKGYWSDQDILSLYGLFENDEVPFVKKNEVLVDVLKKCDPSGNRRITLDEFLAFRKNGGELTDFGFPGHHGDEEEEFEMHHVEKYHPAGLDEPDENWNHPEDIEHFQKHDEIFHGDKKPEERRKHFVKYNNIPDKYRRVSI*\n+TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005793^cellular_component^endoplasmic reticulum-Golgi intermediate compartment`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\tAGAACTTGACCGCAGCGCACATCAACTACACCTTGAGGCAATTCAAAATATGAAGTTTTCTACCGTAGGGTTTTTGTTTTCAACAATTTTATTCAAAAGTGCGTTTGCAGGATGGATGGATACACACATGAAAGATGAACATCACATTGATAAGTACACAGATGAATCTTTTTTTCGTCTTCACGATTTAGGAAAAAAGGGATACTGGTCAGATCAAGATATACTTAGTTTATATGGATTATTTGAAAATGATGAGGTCCCATTCGTCAAAAAGAATGAGGTTTTGGTTGATGTTCTAAAAAAATGCGATCCATCGGGCAATCGCCGGATAACACTTGATGAATTTCTTGCATTTCGTAAAAATGGAGGAGAGTTAACGGATTTTGGATTTCCAGGGCACCATGGTGATGAGGAAGAGGAATTTGAAATGCATCATGTGGAAAAATATCACCCAGCTGGCCTCGATGAACCAGATGAAAACTGGAATCATCCTGAAGATATCGAACATTTTCAAAAACACGATGAGATATTTCATGGTGATAAGAAACCTGAGGAACGACGCAAACATTTTGTAAAATACAATAATATCCCTGACAAGTATCGTCGTGTGAGCATTTAAATTTTCCCATTGATAGATTTACTTGCTTCTCCTTGGGACGTTTTATTTTTATTTTTATTTTTCCGTTTATTTTTGGATTGTCGTGTTCCGATATATATCCGTGTTTACAAACAATAGCTTCAACATTTTTGAACGGGTCCTTCCAATACTAATGCCATATCTTATTTCTTTGTCAATACAAAATCACTTTTGTTCGTTAAACTCATTTTGGTTTGCAGATATTATCATATTCTTACCAATTATACATACACTCACAATTTGTTTTACATTTCCGCTCCCTTCTTCAATAAGCTGATTGCTTATAAAAAATCAAACATTCATATGATTCAAGTTCAAATATTATCATCTGAAAACTGACAAGTTCCATTCATTACTATATTAATATCATTTATTCGTTAAACCGATATCATTAAAAATGGAAAATTAAAGAAATCAATTCATTTGCTTCCTTGAAGTCTAGAGTCGATCATAACGAGAGCTTCATGTTGAAGCTCTTTAAGTGCAGCTTGCTTAGTAATATAAGTGCGTAATGTTTCAACAGCCTTTGGTGTTTTTATGAACTTGTTAAATATTTCTTGATCTTTGGAATATACGATATTGCATATGCAAACTAATCCACGATGTTGAATTCCTTCATCTTGGTCGGTTAACATTCTGACTATGTAGTTAAAATCGTTCTGGAGACTTAAAATTTTCTTGCACCCTTCATCAACACTAGTAATTTGAACAAGAATTCCGCATGCTGCTAATCTCGTGGGAGTGTCTTCGGTATCTGAGAGTGCTAGTACAATGTGTAAACGAGTATTTTCAAAATCCGAGTCCTTGTCTCCTATAAACTTAATTAAGCAATAAGGTGATAAAGAAAGGTTGTTAATAAGTTCCGTGGTTGCCCTCTGAATTAAAGGATTTGTTTCTATAATTAGTTCATCCAATTCTCTCCAACATTCTTGAACTATGGCTTGCCGACTTTCTTCATCATGTGAAGCAAGATTTGTTAGCGCAAGTAACACTTCAAATTTCCCCAAGAGAATTGGATACTCAGTATCAGCAGAATTAGTACTTAGTAACTTTGACATAGGATGAATTGCTCGGTTCGATGGAAACGCCTTAGTAAATATACTATGAGGAGCCACACTGATAAGAATTTTCGCTAATGCAAAAGCCGCATTCGAATCCTGGGTTTGGATAAAACATTGTCTAGTAAGGAATTTCAATGCACCTATTTGTACTAGCTTGCGACGTGCCAGCTTGTAATTAGCCATGTGTACAAGTATCATTGCTAATAACCCGTTTAGGCTATTATAGTGCTTGGCCGCTGAAAAGAGTTTTGAC\tELDRSAHQLHLEAIQNMKFSTVGFLFSTILFKSAFAGWMDTHMKDEHHIDKYTDESFFRLHDLGKKGYWSDQDILSLYGLFENDEVPFVKKNEVLVDVLKKCDPSGNRRITLDEFLAFRKNGGELTDFGFPGHHGDEEEEFEMHHVEKYHPAGLDEPDENWNHPEDIEHFQKHDEIFHGDKKPEERRKHFVKYNNIPDKYRRVSI*\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmp8_jayk4s/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmp4vhthgkjtrinotate_annotation_report_minus_tmhmm_signalp.xls v. /tmp/tmpm8gt2so9trinotate_annotation_report_minus_tmhmm_signalp.xls )\n--- local_file\n+++ history_data\n@@ -1,29 +1,29 @@\n #gene_id\ttranscript_id\tsprot_Top_BLASTX_hit\tRNAMMER\tprot_id\tprot_coords\tsprot_Top_BLASTP_hit\tPfam\tSignalP\tTmHMM\teggnog\tKegg\tgene_ontology_blast\tgene_ontology_pfam\ttranscript\tpeptide\n-TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1E8.02\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\tTGTGATTCGTGCCGCTCAACATTTCTTCAAGTGATTCCTAACATTGCTGTACATTCTCTTGAAAAGCGTTTTAACCTTTTCAATTCTGAAACTTGTAAAGAATGTCAGAGTATCGTATTCGAGTGACAACTGTGGACCAGAAGGTTGGTATTTTTCAAGTTCCTAGGACAAAAACGGTTTTAGAATTAAAGGAACTGATTGCTGTAACATTTGAAGCTCCTGCAGATCGATTAAAGTTGATTCACGCTGGTCGTGTATTGAGGAATGAAACTCCCCTCGAGGAAATTTTGCATGACGCTACTGATCTGGTCACTTTTCATCTTGTAATTGCGGTTTTTAATTCTACAAGCACAACCTTGCCATCAGCGACATCATCTTCTGTTCCTCAAAGTAGGACCAGCGAATTATCATCGACTAATTCAATTCCAACTCCTCGAATCACCAGTTTGAACCCGGAAGAATTGTCAAGACGTGAACGTGCCCAACGTTTATTACAGACATACAACAGTTTCCATGGATCTGGCTTAGGAGGACTTTTTCCAAACATACACCGGGAACTAGAAAGTCACGGATTTTCATTGCCAACCCATGAGCAATCATCACCCGTTGCTGAATCTCTGGATAATTCGGTATCGTCTGCTTTATCTCCACATTTGGAAACGCTGCGTCGCCGCAATCTCAGTATTCATCATCAACATATACAAGCTCACGAAATGGCACAAGAGTCCTTGGAGACCAGAAATCCAGGAAACATTTCTTCCTCGAGTGCTCCTTTAGCTAGTGATCAATCCCCGACTGTATCATCTAACCATATCCATGCATCCGGTAATTTGGCTTTAGGATCAAATTCGGGGTTAAATCCACGAAGCCCTAATTCTTTTTCTTCCCCATTGGACAATCCTGCACTTCATACTGTTGACTCAACCAACGTCAATGGTTCGCTTTCACCTCTATCAAATTCTTCATCTATCAATCAAGTTCACCAAAATGAAACTCATGGTTCTACCATCTCAGTACCTAACCCGAACTTAAGTCAAATGGGCCCTTCACACTCTTCTTCTGTGCCCTCAAATCTGTCACCAAATCCTGCCCAAAATGAAAATCCTAGTACAACCTCTATTCCATCAATAAATAATCAACCATTCCCGTCTGGTCTTTCTGCTTCTAACTCTAATTTTGCCTCTTCCTCGTTTATCCCTCAGAGCGTTCCGCAACTATTGCCTATATATTATCAAACTATCTTTTACAACGGTAACTATTATTTACAGCAGCTCCCATCTGCATCGCCTCCCACTATGTTTCGAGATCACTCATTTGCTCCTTTAGTTTCCCCTTCTATTGTTTCTCCGTATGGTGTTTTAGAAAATGAAGAAACTGGCGAATGTGCCTTTTTGTTTTCACCAAATGCCTCCCAACCCCATTTTCAGCCGAGAGCCCCCACGTTTGGCATTCCACGAAATGTGAGATCCCTCTTCACCTTACCGTTCTTCCATACTATCCGTAATATTGAGAGACATTTCCGATTATTTATTCGTTTAGCCCTGTTTTGTGTATTAACGACTTATAATGTTTCTCTCTCACAAACAATTTTGCTAACATCTATAATGTCTGTTGTGTTTTTGTTACAGACCGGTGCTCTAGCTCCATTCATCAATGACAATCCACTCATCCAGTCTGGTATGCGTCATATTCGTAATCTTCAAGATGAATACCGGCGGCGTCGAAACCGAACGGCTCAAAGAGTAGTTGAAATTCCTAATGAGACACAGACTGAAGATGAACAAGATGGTACAAATACGCCGGATAATAGGGCTGACGCTGAAGAACGCGAACTAACTCGAAGTCAAAGAATATACCGGACGGTAGTAAGAACCATAGTTGCCTTCGCATTAAGCTTTGTACCTCGAGCGTAACATTTTTTTCTTAAAAACAAAAGTGATAATATAGCTGTAAAGTCATTTTTCATTACTAATTAGAAATATTGTCTAAGCTTGCTATTAATAAAAGAGTTCTAGCTAGACTAAATTTTGACAGTGTCGAAATATCATTCCAAGGTACCCCATCAGATTAACAAATTTATAA\tMSEYRIRVTTVDQKVGIFQVPRTKTVLELKELIAVTFEAPADRLKLIHAGRVLRNETPLEEILHDATDLVTFHLVIAVFNSTSTTLPSATSSSVPQSRTSELSSTNSIPTPRITSLNPEELSRRERAQRLLQTYNSFHGSGLGGLFPNIHRELESHGFSLPTHEQSSPVAESLDNSVSSALSPHLETLRRRNLSIHHQHIQAHEMAQESLETRNPGNISSSSAPLASDQSPTVSSNHIHASGNLALGSNSGLNPRSPNSFSSPLDNPALHTVDSTNVNGSLSPLSNSSSINQVHQNETHGSTISVPNPNLSQMGPSHSSSVPSNLSPNPAQNENPSTTSIPSINNQPFPSGLSASNSNFASSSFIPQSVPQLLPIYYQTIFYNGNYYLQQLPSASPPTMFRDHSFAPLVSPSIVSPYGVLENEETGECAFLFSPNASQPHFQPRAPTFGIPRNVRSLFTLPFFHTIRNIERHFRLFIRLALFCVLTTYNVSLSQTILLTSIMSVVFLLQTGALAPFINDNPLIQSGMRHIRNLQDEYRRRRNRTAQRVVEIPNETQTEDEQDGTNTPDNRADAEERELTRSQRIYRTVVRTIVAFALSFVPRA*\n+TRINITY_DN596_c0_g1\tTRINITY_DN596_c0_g1_i1\tYB92_SCHPO^YB92_SCHPO^Q:102-1910,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN596_c0_g1_i1|m.607\t102-1913[+]\tYB92_SCHPO^YB92_SCHPO^Q:1-603,H:1-603^100%ID^E:0^RecName: Full=Uncharacterized ubiquitin-like protein C1E8.02;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\tGO:0005829^cellular_component^cytosol`GO:0005783^cellular_component^endoplasmic reticulum`GO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0016021^cellular_component^integral component of membrane`GO:0031593^molecular_function^polyubiquitin modification-dependent protein binding`GO:0032446^biological_process^protein modification by small protein conjugation`GO:0006511^biological_process^ubiquitin-dependent protein catabolic process\t.\tTGTGATTCGTGCCGCTCAACATTTCTTCAAGTGATTCCTAACATTGCTGTACATTCTCTTGAAAAGCGTTTTAACCTTTTCAATTCTGAAACTTGTAAAGAATGTCAGAGTATCGTATTCGAGTGACAACTGTGGACCAGAAGGTTGGTATTTTTCAAGTTCCTAGGACAAAAACGGTTTTAGAATTAAAGGAACTGATTGCTGTAACATTTGAAGCTCCTGCAGATCGATTAAAGTTGATTCACGCTGGTCGTGTATTGAGGAATGAAACTCCCCTCGAGGAAATTTTGCATGACGCTACTGATCTGGTCACTTTTCATCTTGTAATTGCGGTTTTTAATTCTACAAGCACAACCTTGCCATCAGCGACATCATCTTCTGTTCCTCAAAGTAGGACCAGCGAATTATCATCGACTAATTCAATTCCAACTCCTCGAATCACCAGTTTGAACCCGGAAGAATTGTCAAGACGTGAACGTGCCCAACGTTTATTACAGACATACAACAGTTTCCATGGATCTGGCTTAGGAGGACTTTTTCCAAACATACACCGGGAACTAGAAAGTCACGGATTTTCATTGCCAACCCATGAGCAATCATCACCCGTTGCTGAATCTCTGGATAATTCGGTATCGTCTGCTTTATCTCCACATTTGGAAACGCTGCGTCGCCGCAATCTCAGTATTCATCATCAACATATACAAGCTCACGAAATGGCACAAGAGTCCTTGGAGACCAGAAATCCAGGAAACATTTCTTCCTCGAGTGCTCCTTTAGCTAGTGATCAATCCCCGACTGTATCATCTAACCATATCCATGCATCCGGTAATTTGGCTTTAGGATCAAATTCGGGGTTAAATCCACGAAGCCCTAATTCTTTTTCTTCCCCATTGGACAATCCTGCACTTCATACTGTTGACTCAACCAACGTCAATGGTTCGCTTTCACCTCTATCAAATTCTTCATCTATCAATCAAGTTCACCAAAATGAAACTCATGGTTCTACCATCTCAGTACCTAACCCGAACTTAAGTCAAATGGGCCCTTCACACTCTTCTTCTGTGCCCTCAAATCTGTCACCAAATCCTGCCCAAAATGAAAATCCTAGTACAACCTCTATTCCATCAATAAATAATCAACCATTCCCGTCTGGTCTTTCTGCTTCTAACTCTAATTTTGCCTCTTCCTCGTTTATCCCTCAGAGCGTTCCGCAACTATTGCCTATATATTATCAAACTATCTTTTACAACGGTAACTATTATTTACAGCAGCTCCCATCTGCATCGCCTCCCACTATGTTTCGAGATCACTCATTTGCTCCTTTAGTTTCCCCTTCTATTGTTTCTCCGTATGGTGTTTTAGAAAATGAAGAAACTGGCGAATGTGCCTTTTTGTTTTCACCAAATGCCTCCCAACCCCATTTTCAGCCGAGAGCCCCCACGTTTGGCATTCCACGAAATGTGAGATCCCTCTTCACCTTACCGTTCTTCCATACTATCCGTAATATTGAGAGACATTTCCGATTATTTATTCGTTTAGCCCTGTTTTGTGTATTAACGACTTATAATGTTTCTCTCTCACAAACAATTTTGCTAACATCTATAATGTCTGTTGTGTTTTTGTTACAGACCGGTGCTCTAGCTCCATTCATCAATGACAATCCACTCATCCAGTCTGGTATGCGTCATATTCGTAATCTTCAAGATGAATACCGGCGGCGTCGAAACCGAACGGCTCAAAGAGTAGTTGAAATTCCTAATGAGACACAGACTGAAGATGAACAAGATGGTACAAATACGCCGGATAATAGGGCTGACGCTGAAGAACGCGAACTAACTCGAAGTCAAAGAATATACCGGACGGTAGTAAGAACCATAGTTGCCTTCGCATTAAGCTTTGTACCTCGAGCGTAACATTTTTTTCTTAAAAACAAAAGTGATAATATAGCTGTAAAGTCATTTTTCATTACTAATTAGAAATATTGTCTAAGCTTGCTATTAATAAAAGAGTTCTAGCTAGACTAAATTTTGACAGTGTCGAAATATCATTCCAAGGTACCCCATCAGATTAACAAATTTATAA\tMSEYRIRVTTVDQKVGIFQVPRTKTVLELKELIAVTFEAPADRLKLIHAGRVLRNETPLEEILHDATDLVTFHLVIAVFNSTSTTLPSATSSSVPQSRTSELSSTNSIPTPRITSLNPEELSRRERAQRLLQTYNSFHGSGLGGLFPNIHRELESHGFSLPTHEQSSPVAESLDNSVSSALSPHLETLRRRNLSIHHQHIQAHEMAQESLETRNPGNISSSSAPLASDQSPTVSSNHIHASGNLALGSNSGLNPRSPNSFSSPLDNPALHTVDSTNVNGSLSPLSNSSSINQVHQNETHGSTISVPNPNLSQMGPSHSSSVPSNLSPNPAQNENPSTTSIPSINNQPFPSGLSASNSNFASSSFIPQSVPQLLPIYYQTIFYNGNYYLQQLPSASPPTMFRDHSFAPLVSPSIVSPYGVLENEETGECAFLFSPNASQPHFQPRAPTFGIPRNVRSLFTLPFFHTIRNIERHFRLFIRLALFCVLTTYNVSLSQTILLTSIMSVVFLLQTGALAPFINDNPLIQSGMRHIRNLQDEYRRRRNRTAQRVVEIPNETQTEDEQDGTNTPDNRADAEERELTRSQRIYRTVVRTIVAFALSFVPRA*\n TRINITY_DN549_c0_g1\tTRINITY_DN549_c0_g1_i1\tGAA1_SCHPO^GAA1_SCHPO^Q:2-1363,H:121-574^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN549_c0_g1_i1|m.608\t2-1387[+]\tGAA1_SCHPO^GAA1_SCHPO^Q:1-461,H:121-581^100%ID^E:0^RecName: Full=GPI transamidase component gaa1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC1002.11`KO:K05289\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0042765^cellular_component^GPI-anchor transamidase complex`GO:0016255^biological_process^attachment of GPI anchor to protein\t.\tGTTTAACGGCTCGAATTTTATTACAACTTTACGTGCTCCTAGAGGTGATGCTACTGAGTCTCTACTCCTATGTGTCCCTTGGAAAGATCATATTGGGCAATATAATGAGGCCGGCGTTGCTTTAGCCATTAGTTTATTGAAATACTTCCAGGGATGGTCTCTTTGGTCGAAAGACATCATCTTGGTTATTTTTGATGATCCTGTTTATGGTCCATCGAGTTTTCTCACGTCTTATTTTGACCAAACGACGCCATATATATCTTACACACCATTGAAAATTCGTTCTGGTTCCATTCAAGCTGGTTTATCGCTTGAGCTAGTTACTACTGAAAATAACTCGGACGTGCTTGAAGTCCTTTACCAAGCTACCAATGGACAGTTGCCCAACCTTGACCTTTTCAATACTATTTCGAGAATCTTTATGCAACATTTTAATTATCCATTACGACTACAAGGATATGATTTCCATGCTAATTCGGGGTCCTCATACACATCCCGTCTAAAATCTTTGTGGATGGGTATGTTGACACAAGCAGTTTCGAATGTTACTAGTGCGCATGCACTATTTCCTCAATATAGAATTGACATGTTGACCTTGCGAATGAAAGTGAAAGACCCGTTTTCATTTGACATGTTTAGATTTGGTCAAGCCATTGAAAGCACATTTCGATCTCTTAACAATCTTCTTGAACACTTGCATCAATCTTTTTTCTTTTACTTCATCCTAGATCATTTGCATTTTATTAGTATCGGAAATTATATGCCTTCCATTTTAATTCTGGCTGCTAGTTTCATGCTCGGAGCTTACCGACATTGGATCAACCATGAAAAAAAAATAGATCTTTGGAGACCATTTTCTTTTTGGCTTTTTAGTATTTTTTGTACGATCGCCGCATATTATTTGGTATCTTCTTCTACTAAGATAACCGTTTTTATATTCCTCTATTTAATGCTCACTTTTATTGGTATTATCTTTTCAACTTTTATGACGAGTGAAGACGCTGAACTGGTTCTTTCTTACGATTTAATGTCAAAATCCCTTTTCATTTCCGTAGTGAGTACGTTAAATTTTTCTCTTTCTTTTGTTGTTGCAATTTTGCTTGTACCATTACAGTTTATTTCATTTAGATTTAATCGACGTCTTTCGTTACTTTTCGCAGTCTTGACATACTTTTCTACATTTATATTCCTTTGTTCTCTGTCTAAAATTTTGAATGGACCTTTGGTACCATTTTGGTTGTGGGCAAAAGAATATGAGTTGTTCAACTCTTGGTTAATGCCATCCGTGTTTATGATTTTAGTTTTACCTGAAATCATATTCTCAGTTACAAGTTTTTTTTCGCTTTGGAATGAACCGTCTGTGAAAACCAAGACAAAGACTTTGTGAGGGACTAAAAGGTACTATTGTATGAGTACTTATCATTATTTATTTTACGAACATCATTAAAAACTGTCTATAAAAAATATTACCGCAGAATTGTTACGTTCGCGGAAAGCTTTTAGAGGTAGAAATGCATTTCCG\tFNGSNFITTLRAPRGDATESLLLCVPWKDHIGQYNEAGVALAISLLKYFQGWSLWSKDIILVIFDDPVYGPSSFLTSYFDQTTPYISYTPLKIRSGSIQAGLSLELVTTENNSDVLEVLYQATNGQLPNLDLFNTISRIFMQHFNYPLRLQGYDFHANSGSSYTSRLKSLWMGMLTQAVSNVTSAHALFPQYRIDMLTLRMKVKDPFSFDMFRFGQAIESTFRSLNNLLEHLHQSFFFYFILDHLHFISIGNYMPSILILAASFMLGAYRHWINHEKKIDLWRPFSFWLFSIFCTIAAYYLVSSSTKITVFIFLYLMLTFIGIIFSTFMTSEDAELVLSYDLMSKSLFISVVSTLNFSLSFVVAILLVPLQFISFRFNRRLSLLFAVLTYFSTFIFLCSLSKILNGPLVPFWLWAKEYELFNSWLMPSVFMILVLPEIIFSVTSFFSLWNEPSVKTKTKTL*\n-TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGGTATTGTTCAAAAGGGAGTCCCTTTTCAAGTTCTGCATAGTGTAACACATCTAACAAACGCTTTGAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSSKRLLDVLHYAELEKGLPFEQYLVHPGLCTTNMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n-TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0005741^cellular_component^mitochondrial outer membrane`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSCPSRTMYY*\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tYPIQKKHLTTEMLRQIPHLRLRTAKQGEIFRLRSDSLKALRQFFSSKDFTETNPPIITSSDCEGAGEVFTLTPQETHKNKSFERDDQKHFFDRPAFLTVSTQLHLEALALGLSRVYTISPAFRAEQSHTSRHLAEFWMLEAEVAFMTSLSQLTSLMEDMIKYTLNSLMEQNYHRDHWDQLLKPWKCMTYSEAIEELSAVKKTWKYPPKWGNDLSSEHEKYLCEILHKTPVFVTDYPQKIKPFYMKSSGPDTVAAVDLLVPQVGELAGGSLRKDHLDEYKTYPPELQWYLDLMKYSNAPHGGFGLGIERLIAFLEGENTNVKETIPFPRSVGSIFA*\n-TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005737^cellular_component^cytoplasm`GO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0061630^molecular_function^ubiquitin protein ligase activity`GO:0031625^molecular_function^ubiquitin protein ligase binding`GO:0007031^biological_process^peroxisome organization`GO:0070647^biological_process^protein modification by small protein conjugation or removal\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tMNSSIKRLHKEYASFQAGDISGLLLLKPVTDVDMFHWKAVIEGPTETPYEGGQWVLDIHVHEGYPISPPSVYFQTKIVHPNISWTNGEVCMDILKTHWSPAWSLQSACLAIISLLSNYDASSPLNVDAAKLLRTGDKTAYNSLVRCTTYLYAKGKIEDLS*\n+TRINITY_DN594_c0_g1\tTRINITY_DN594_c0_g1_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1275-349,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g1_i1|m.612\t346-1275[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-309,H:46-338^94.82%ID^E:0^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGGTATTGTTCAAAAGGGAGTCCCTTTTCAAGTTCTGCATAGTGTAACACATCTAACAAACGCTTTGAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSSKRLLDVLHYAELEKGLPFEQYLVHPGLCTTNMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.614\t346-651[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-101,H:238-338^100%ID^E:7e-68^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tMYETFLGPILVMCAKLGFYICRLLGSPWHTISPYVAAFAFLWTALHATKEDQSIKWGAAVTRFGHERVLSTPVELILPSEQEKALEYMTKLYQEWKKKLVS*\n+TRINITY_DN594_c0_g2\tTRINITY_DN594_c0_g2_i1\tERG27_SCHPO^ERG27_SCHPO^Q:1207-680,H:46-205^90.91%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`ERG27_SCHPO^ERG27_SCHPO^Q:681-349,H:228-338^100%ID^E:2e-141^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN594_c0_g2_i1|m.613\t653-1207[-]\tERG27_SCHPO^ERG27_SCHPO^Q:1-176,H:46-205^90.91%ID^E:2e-108^RecName: Full=3-keto-steroid reductase;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1709.07`KO:K09827\tGO:0005789^cellular_component^endoplasmic reticulum membrane`GO:0005811^cellular_component^lipid droplet`GO:0000253^molecular_function^3-keto sterol reductase activity`GO:0006696^biological_process^ergosterol biosynthetic process\t.\tGGTCTTATAGCTAGCATGAGACAAAGACGGCAACTCGCGGAATTCTTCATATAAACTTTTCCGAGTCCGAGGTTCTAATAAATTCGTATGTCTGTATTAACTAAATTGAGACATAAACTATATTCCAGACTTGGTTAAAATCCCCTTATCTATTCATTTTGTCAAGATATCTATACCGATTGTAGCAAAATTTGATAAAAGCAATTTTCAAAATGTTTTACTGAGAATGCAATGAATCAGGGATTGTTCTTTCATAACTCCCCTCGTCTCGTGCACTTTAAAATTAACTGGTACTCCAGATTTTTATCAAGGCCTCCGCGCAAGAGAATATGTAAATGTTTCCCTTTACGAAACCAGCTTCTTTTTCCATTCCTGGTAAAGTTTTGTCATATATTCTAAAGCTTTCTCTTGCTCTGAAGGCAAAATTAATTCCACAGGAGTAGAGAGGACTCTTTCATGACCAAAACGAGTCACGGCTGCTCCCCACTTGATAGATTGATCTTCTTTGGTAGCATGTAAAGCAGTCCAAAGAAAAGCAAAAGCAGCAACGTATGGAGAAATTGTATGCCAAGGACTTCCAAGTAAGCGACAAATATAGAAACCGAGCTTTGCACACATAACCAATATAGGACCGAGGAAAGTTTCATACATGTTAGTAGTACATAGTCCGGGATGGACAAGATTGATAGGGTTGCTCGCCGTGAAAACATTGTAGGTCTTCAAAATCAAGGGATTTTTTTTCTGCTACTAAACTGCTTGTAAGGACTACCTTTGTTGACGAACGGAGCACTTTTAGTTCGGCCAGGCGATTTTTCTTATGTAAATTAGTACTAACATACGGAATTAAGGTGAAAAAACTTACCAAATAAAAATGACCAAAAACATTACTTTCAAATATGTAACCGAGGGAGTCATTGCTCACACGTCCAGCAGTCTCTTTGTAAAAGGTTGGATGCGTCAGTGCCTGGATAGGGTTGATCAAAGTAGAAAAGATAGCCTTTAGCCATTGAATACCTTCCAGGTCCCAAGCACCAGCATTTAGGTAAACAAAATCTAATTTAGGGAATCTAGTAGCGATATCTTGCACCGCAGCTTCCACTGATGCCATATTCGACAAGTCTAACAAAACGTATTCCAACCGTATTTTCCTATCGGGAAAGAATTCTTTCAATCTTCTACAGGCATCTTCTGCCTTTTCACGACTCCTG\tRSREKAEDACRRLKEFFPDRKIRLEYVLLDLSNMASVEAAVQDIATRFPKLDFVYLNAGAWDLEGIQWLKAIFSTLINPIQALTHPTFYKETAGRVSNDSLGYIFESNVFGHFYLVSFFTLIPYVSTNLHKKNRLAELKVLRSSTKVVLTSSLVAEKKSLDFEDLQCFHGEQPYQSCPSRTMYY*\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.617\t735-1742[-]\tSYNM_SCHPO^SYNM_SCHPO^Q:1-335,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.10c`KO:K01893\tGO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004816^molecular_function^asparagine-tRNA ligase activity`GO:0005524^molecular_function^ATP binding`GO:0000049^molecular_function^tRNA binding`GO:0006421^biological_process^asparaginyl-tRNA aminoacylation`GO:0032543^biological_process^mitochondrial translation\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tYPIQKKHLTTEMLRQIPHLRLRTAKQGEIFRLRSDSLKALRQFFSSKDFTETNPPIITSSDCEGAGEVFTLTPQETHKNKSFERDDQKHFFDRPAFLTVSTQLHLEALALGLSRVYTISPAFRAEQSHTSRHLAEFWMLEAEVAFMTSLSQLTSLMEDMIKYTLNSLMEQNYHRDHWDQLLKPWKCMTYSEAIEELSAVKKTWKYPPKWGNDLSSEHEKYLCEILHKTPVFVTDYPQKIKPFYMKSSGPDTVAAVDLLVPQVGELAGGSLRKDHLDEYKTYPPELQWYLDLMKYSNAPHGGFGLGIERLIAFLEGENTNVKETIPFPRSVGSIFA*\n+TRINITY_DN591_c0_g1\tTRINITY_DN591_c0_g1_i1\tSYNM_SCHPO^SYNM_SCHPO^Q:1742-738,H:107-441^100%ID^E:0^RecName: Full=Asparagine--tRNA ligase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN591_c0_g1_i1|m.618\t251-733[+]\tUBC16_SCHPO^UBC16_SCHPO^Q:1-160,H:1-160^100%ID^E:2e-116^RecName: Full=Ubiquitin-conjugating enzyme E2 16;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC1198.09`KO:K10689\tGO:0005524^molecular_function^ATP binding`GO:0061631^molecular_function^ubiquitin conjugating enzyme activity`GO:0007031^biological_process^peroxisome organization\t.\tACTGAGCCTTAATCAAATTTCGGTTACCTTCCGTTGAATAAGGCTAATTAAGGATCGCTTTGAGTGCATTTATGTATTAACTTCTTTTTACACAACTTAGTGAGTACACCTGGTTTTGTTTGCTTCCAGCAACACCTGCGAAACCGGTAATCCTTGAAAATTTAAATCTTCTTAGATTGACAAGCTTTGCTATCTAGAGTTGATGCATGCTAGAACCTAAAATAGAAACGAGTGTTATGAATCTTGACAAATGAATAGTTCAATTAAACGACTTCACAAAGAATATGCCAGTTTCCAGGCTGGAGATATATCTGGTCTCTTACTTTTAAAACCGGTCACTGATGTTGATATGTTCCATTGGAAGGCAGTCATTGAAGGACCTACTGAAACACCGTATGAAGGGGGGCAATGGGTTCTCGATATTCATGTTCACGAAGGGTATCCTATTAGTCCCCCGTCTGTGTATTTTCAAACAAAAATTGTACATCCGAATATTTCATGGACGAACGGCGAAGTATGTATGGACATATTAAAGACACATTGGTCGCCGGCCTGGAGTCTACAATCTGCTTGTTTAGCAATTATCTCTCTTTTAAGTAATTACGATGCTAGTTCTCCATTAAATGTAGATGCCGCCAAGCTGTTGAGAACCGGGGACAAAACGGCTTATAATTCATTAGTTCGATGTACAACCTATCTTTACGCCAAAGGTAAGATAGAAGATCTTTCATAAATCATGCAAAAATGGATCCTACTGAACGGGGGAAGGGTATAGTTTCTTTTACGTTCGTGTTTTCTCCTTCTAGAAAAGCAATCAGTCGTTCTATCCCTAAGCCAAACCCTCCGTGTGGCGCATTGCTGTACTTCATTAGGTCTAAATACCATTGAAGTTCTGGTGGGTAAGTTTTATACTCATCTAAATGGTCTTTACGTAAAGAACCTCCAGCAAGTTCACCCACCTGTGGAACGAGTAAATCAACAGCAGCAACTGTATCGGGCCCAGAGGATTTCATATAAAACGGTTTAATTTTTTGTGGATAGTCCGTCACGAATACCGGAGTTTTATGTAAGATTTCACAAAGATATTTTTCATGTTCACTACTTAAATCGTTTCCCCATTTAGGAGGGTATTTCCATGTTTTTTTCACAGCGCTTAATTCTTCTATTGCTTCAGAATATGTCATGCATTTCCACGGTTTAAGCAGTTGGTCCCAATGATCTCTGTGATAGTTCTGCTCCATCAATGAATTAAGAGTATATTTAATCATATCCTCCATAAGTGATGTTAGTTGTGAAAGCGAAGTCATGAACGCAACTTCTGCTTCCAGCATCCAAAATTCTGCCAAATGTCGACTAGTATGACTTTGTTCTGCTCTAAATGCTGGCGATATAGTATACACTCTGGACAGTCCTAATGCTAGAGCCTCTAGATGCAATTGAGTGGATACAGTTAAAAATGCAGGTCGGTCAAAAAAATGCTTTTGATCATCTCTTTCAAAACTTTTATTTTTATGCGTCTCCTGGGGGGTCAAGGTAAATACTTCACCAGCTCCTTCACAATCAGAAGAAGTTATTATCGGGGGGTTGGTCTCTGTAAAATCTTTGCTTGAAAAAAATTGTCGCAATGCTTTTAATGAATCTGAACGGAGACGAAAAATCTCCCCTTGTTTTGCTGTTCGTAATCGAAGATGGGGTATTTGTCGTAACATTTCAGTAGTCAAATGTTTTTTCTGAATAGGGTAGT\tMNSSIKRLHKEYASFQAGDISGLLLLKPVTDVDMFHWKAVIEGPTETPYEGGQWVLDIHVHEGYPISPPSVYFQTKIVHPNISWTNGEVCMDILKTHWSPAWSLQSACLAIISLLSNYDASSPLNVDAAKLLRTGDKTAYNSLVRCTTYLYAKGKIEDLS*\n TRINITY_DN540_c0_g1\tTRINITY_DN540_c0_g1_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:26-1567,H:2-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g1_i1|m.650\t350-1570[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-406,H:110-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\tATCTACCATTAATTGAAATACAAGGGGACAGGGGAGCAGTAAACACGCCGACAGCAAGCTCGATTCCTACCCATCCTTTTCTCGGTCAGACACTCAAGGATCCATTAAGTCTTTAAAGTCTCTGAAAACAGTCCTGGGGAAGGGCAAAGACTCGAATCATGATCGCCGAACGTCTACTGATACAACCCATAGTCGTCACCGGTATCCCGAAACACCACCAAGCCTTCCTCCTCCTCCTTCTCCTGGGATATTAGCTACATCTCCGGCTGTGTTGCAAAAGCATCAACAGGAGGATTCGGGAAATTCTTCTCAATCTCCGACGTCGCCGCATCCTTCTAACCAGCCTGCCATGCTTAGTCCCTCTACGGCCGCTTCCCAGCATCATCATCATCATTCTTCTTCTTCTTCTTATGCCGTTTCTCCAACATCACCTACCTCACCAACCAGCTCTGGCCCAATTGGCTCCAACTTTGATTCAGCTTCTGAGCATAATGGTCCTGTGTATCCACAAGATCAGCAGGGGCCCGTAATTATCCCTAATTCGGCAATTTCTTCTACGGATCCTGATGATCCTGAAACAGTTGTTAGCCTTAACGTAGATGAAATGATTCAAAGATTGATCCATGTTGGCTATTCTCGAAAAAGTAGTAAAAGTGTGTGTCTAAAGAATGCTGAAATTACCAGCATTTGTATGGCCGTACGAGAAATATTTCTTTCGCAGCCGACACTTTTAGAATTGACCCCGCCGGTGAAAATCGTTGGTGACGTTCATGGTCAATATTCCGATTTAATTCGACTGTTTGAAATGTGTGGGTTTCCTCCGAGCTCTAATTATTTATTCTTAGGCGATTATGTCGATCGTGGTAAGCAGAGTCTTGAAACCATCCTCTTGCTCTTTTTGTACAAGATCAGATATCCAGAAAACTTTTTCTTGCTACGGGGCAATCATGAATGCGCTAATATCACTCGCGTTTACGGTTTTTACGATGAGTGTAAAAGACGTTGCAATATCAAAATTTGGAAGACTTTCATAAACACCTTTAATTGCTTACCCATTGCGTCCGTTGTAGCCGGTAAAATTTTTTGCGTTCATGGTGGTCTTTCTCCATCGTTAAGTCATATGGATGACATACGGGAGATTCCTCGTCCTACTGACGTTCCTGACTATGGATTGCTGAATGATTTGTTATGGTCAGATCCTGCTGATACTGAAAACGATTGGGAAGATAATGAACGAGGTGTTTCTTTTGTTTTCAACAAAAACGTTATCCGGCAATTTTTGGCCAAGCACGACTTTGATTTAATCTGTCGTGCTCACATGGTTGTTGAAGACGGTTATGAATTTTTTAATGACAGAACTTTATGTACGGTGTTTTCTGCCCCTAATTACTGTGGGGAATTTGATAATTGGGGGGCCGTGATGAGTGTTAATTCAGAGCTGCTTTGTTCCTTTGAACTTATAAAGCCTCTCGACCAGGCAGCAATTCGACGAGAATTAAAAAAGAGCAAGCGTAGTGGTATGGCTATTTATCAATCTCCCCCTGCTGAGCAGGTTACCCAAAGCGTGTAAGCATTCTATTTTTTCTATTTTGGTCTCCATTTGAAATTATCTAGCGGTTTTATTGGGACACTATTTTATTCTCTTAAATGAGTATTATCTGTTATTATGTTATAGTCACTATGGTGATGGATCAAGTCCTTGGTTCACTTTATCGGTTCGAGGGCAATTCCATCCTGACACGTTTAGTATAAGTTTCTGTTTCTTTTAATAAAAACACAATCATTTCCATATAGTGACAGTTAACTTGCTTTTTATTTTTGCTAACCGACACCTGCTGAACATCGAAAGAGTAGGTAAAAAATTTACATGAGGAGGAAATAAACAATTATTATAGTATTATTATTAGCATTAAATAAATTTAAAGCGAAAATGT\tMLSPSTAASQHHHHHSSSSSYAVSPTSPTSPTSSGPIGSNFDSASEHNGPVYPQDQQGPVIIPNSAISSTDPDDPETVVSLNVDEMIQRLIHVGYSRKSSKSVCLKNAEITSICMAVREIFLSQPTLLELTPPVKIVGDVHGQYSDLIRLFEMCGFPPSSNYLFLGDYVDRGKQSLETILLLFLYKIRYPENFFLLRGNHECANITRVYGFYDECKRRCNIKIWKTFINTFNCLPIASVVAGKIFCVHGGLSPSLSHMDDIREIPRPTDVPDYGLLNDLLWSDPADTENDWEDNERGVSFVFNKNVIRQFLAKHDFDLICRAHMVVEDGYEFFNDRTLCTVFSAPNYCGEFDNWGAVMSVNSELLCSFELIKPLDQAAIRRELKKSKRSGMAIYQSPPAEQVTQSV*\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.651\t300-1847[+]\tPPZ_SCHPO^PPZ_SCHPO^Q:1-515,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC57A7.08`KO:K06269\tGO:0005737^cellular_component^cytoplasm`GO:0005783^cellular_component^endoplasmic reticulum`GO:0000324^cellular_component^fungal-type vacuole`GO:0005634^cellular_component^nucleus`GO:0048037^molecular_function^cofactor binding`GO:0004724^molecular_function^magnesium-dependent protein serine/threonine phosphatase activity`GO:0046872^molecular_function^metal ion binding`GO:0016791^molecular_function^phosphatase activity`GO:0004722^molecular_function^protein serine/threonine phosphatase activity`GO:0007165^biological_process^signal transduction\t.\tTTATACTACACTTCCCATCACTTGTTAACCCATTCTTCACCACGACCTTACAAATTTTTAAACGATTGTGTTTTGTCAGCAAAGCAAAACATTTTTATAGACCAGGATTACTTGAAGAAGAGAAAATCTACTAATTTACGGAAAATATCGAAAAATAGAAAAAATCTAGGTACAATTGGGAAACCTTTTAATTTTGGCACTAAATGCAAGCAACGAAGTCTCGATTTTGAAGTAAAAAAGAAAATAAAAATAGAACAAAGTGGAATAGTTTCCCGGAATCTACCATTAATTGAAATACAATGGGACAGGGGAGCAGTAAACACGCCGACAGCAAGCTCGATTCCTACCCATCCTTTTCTCGGTCAGACACTCAAGGATCCATTAAGTCTTTAAAGTCTCTGAAAACAGTCCTGGGGAAGGGCAAAGACTCGAATCATGATCGCCGAACGTCTACTGATACAACCCATAGTCGTCACCGGTATCCCGAAACACCACCAAGCCTTCCTCCTCCTCCTTCTCCTGGGATATTAGCTACATCTCCGGCTGTGTTGCAAAAGCATCAACAGGAGGATTCGGGAAATTCTTCTCAATCTCCGACGTCGCCGCATCCTTCTAACCAGCCTGCCATGCTTAGTCCCTCTACGGCCGCTTCCCAGCATCATCATCATCATTCTTCTTCTTCTTCTTATGCCGTTTCTCCAACATCACCTACCTCACCAACCAGCTCTGGCCCAATTGGCTCCAACTTTGATTCAGCTTCTGAGCATAATGGTCCTGTGTATCCACAAGATCAGCAGGGGCCCGTAATTATCCCTAATTCGGCAATTTCTTCTACGGATCCTGATGATCCTGAAACAGTTGTTAGCCTTAACGTAGATGAAATGATTCAAAGATTGATCCATGTTGGCTATTCTCGAAAAAGTAGTAAAAGTGTGTGTCTAAAGAATGCTGAAATTACCAGCATTTGTATGGCCGTACGAGAAATATTTCTTTCGCAGCCGACACTTTTAGAATTGACCCCGCCGGTGAAAATCGTTGGTGACGTTCATGGTCAATATTCCGATTTAATTCGACTGTTTGAAATGTGTGGGTTTCCTCCGAGCTCTAATTATTTATTCTTAGGCGATTATGTCGATCGTGGTAAGCAGAGTCTTGAAACCATCCTCTTGCTCTTTTTGTACAAGATCAGATATCCAGAAAACTTTTTCTTGCTACGGGGCAATCATGAATGCGCTAATATCACTCGCGTTTACGGTTTTTACGATGAGTGTAAAAGACGTTGCAATATCAAAATTTGGAAGACTTTCATAAACACCTTTAATTGCTTACCCATTGCGTCCGTTGTAGCCGGTAAAATTTTTTGCGTTCATGGTGGTCTTTCTCCATCGTTAAGTCATATGGATGACATACGGGAGATTCCTCGTCCTACTGACGTTCCTGACTATGGATTGCTGAATGATTTGTTATGGTCAGATCCTGCTGATACTGAAAACGATTGGGAAGATAATGAACGAGGTGTTTCTTTTGTTTTCAACAAAAACGTTATCCGGCAATTTTTGGCCAAGCACGACTTTGATTTAATCTGTCGTGCTCACATGGTTGTTGAAGACGGTTATGAATTTTTTAATGACAGAACTTTATGTACGGTGTTTTCTGCCCCTAATTACTGTGGGGAATTTGATAATTGGGGGGCCGTGATGAGTGTTAATTCAGAGCTGCTTTGTTCCTTTGAACTTATAAAGCCTCTCGACCAGGCAGCAATTCGACGAGAATTAAAAAAGAGCAAGCGTAGTGGTATGGCTATTTATCAATCTCCCCCTGCTGAGCAGGTTACCCAAAGCGTGTAAGCATTCTATTTTTTCTATTTTGGTCTCCATTTGAAATTATCTAGCGGTTTTATTGGGACACTATTTTATTCTCTTAAATGAGTATTATCTGTTATTATGTTATAGTCACTATGGTGATGGATCAAGTCCTTGGTTCACTTTATCGGTTCGAGGGCAATTCCATCCTGACACGTTTAGTATAAGTTTCTGTTTCTTTTAATAAAAACACAATCATTTCCATATAGTGACAGTTAACTTGCTTTTTATTTTTGCTAACCGACACCTGCTGAACATCGAAAGAGTAGGTAAAAAATTTACATGAGGAGGAAATAAACAATTATTATAGTATTATTATTAGCATTAAATAAATTTAAAGCGAAAATGT\tMGQGSSKHADSKLDSYPSFSRSDTQGSIKSLKSLKTVLGKGKDSNHDRRTSTDTTHSRHRYPETPPSLPPPPSPGILATSPAVLQKHQQEDSGNSSQSPTSPHPSNQPAMLSPSTAASQHHHHHSSSSSYAVSPTSPTSPTSSGPIGSNFDSASEHNGPVYPQDQQGPVIIPNSAISSTDPDDPETVVSLNVDEMIQRLIHVGYSRKSSKSVCLKNAEITSICMAVREIFLSQPTLLELTPPVKIVGDVHGQYSDLIRLFEMCGFPPSSNYLFLGDYVDRGKQSLETILLLFLYKIRYPENFFLLRGNHECANITRVYGFYDECKRRCNIKIWKTFINTFNCLPIASVVAGKIFCVHGGLSPSLSHMDDIREIPRPTDVPDYGLLNDLLWSDPADTENDWEDNERGVSFVFNKNVIRQFLAKHDFDLICRAHMVVEDGYEFFNDRTLCTVFSAPNYCGEFDNWGAVMSVNSELLCSFELIKPLDQAAIRRELKKSKRSGMAIYQSPPAEQVTQSV*\n TRINITY_DN540_c0_g2\tTRINITY_DN540_c0_g2_i1\tPPZ_SCHPO^PPZ_SCHPO^Q:300-1844,H:1-515^100%ID^E:0^RecName: Full=Serine/threonine-protein phosphatase PP-Z;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN540_c0_g2_i1|m.652\t1-393[+]\t.\t.\t.\t.\t.\t.\t.\t.\tTTATACTACACTTCCCATCACTTGTTAACCCATTCTTCACCACGACCTTACAAATTTTTAAACGATTGTGTTTTGTCAGCAAAGCAAAACATTTTTATAGACCAGGATTACTTGAAGAAGAGAAAATCTACTAATTTACGGAAAATATCGAAAAATAGAAAAAATCTAGGTACAATTGGGAAACCTTTTAATTTTGGCACTAAATGCAAGCAACGAAGTCTCGATTTTGAAGTAAAAAAGAAAATAAAAATAGAACAAAGTGGAATAGTTTCCCGGAATCTACCATTAATTGAAATACAATGGGACAGGGGAGCAGTAAACACGCCGACAGCAAGCTCGATTCCTACCCATCCTTTTCTCGGTCAGACACTCAAGGATCCATTAAGTCTTTAAAGTCTCTGAAAACAGTCCTGGGGAAGGGCAAAGACTCGAATCATGATCGCCGAACGTCTACTGATACAACCCATAGTCGTCACCGGTATCCCGAAACACCACCAAGCCTTCCTCCTCCTCCTTCTCCTGGGATATTAGCTACATCTCCGGCTGTGTTGCAAAAGCATCAACAGGAGGATTCGGGAAATTCTTCTCAATCTCCGACGTCGCCGCATCCTTCTAACCAGCCTGCCATGCTTAGTCCCTCTACGGCCGCTTCCCAGCATCATCATCATCATTCTTCTTCTTCTTCTTATGCCGTTTCTCCAACATCACCTACCTCACCAACCAGCTCTGGCCCAATTGGCTCCAACTTTGATTCAGCTTCTGAGCATAATGGTCCTGTGTATCCACAAGATCAGCAGGGGCCCGTAATTATCCCTAATTCGGCAATTTCTTCTACGGATCCTGATGATCCTGAAACAGTTGTTAGCCTTAACGTAGATGAAATGATTCAAAGATTGATCCATGTTGGCTATTCTCGAAAAAGTAGTAAAAGTGTGTGTCTAAAGAATGCTGAAATTACCAGCATTTGTATGGCCGTACGAGAAATATTTCTTTCGCAGCCGACACTTTTAGAATTGACCCCGCCGGTGAAAATCGTTGGTGACGTTCATGGTCAATATTCCGATTTAATTCGACTGTTTGAAATGTGTGGGTTTCCTCCGAGCTCTAATTATTTATTCTTAGGCGATTATGTCGATCGTGGTAAGCAGAGTCTTGAAACCATCCTCTTGCTCTTTTTGTACAAGATCAGATATCCAGAAAACTTTTTCTTGCTACGGGGCAATCATGAATGCGCTAATATCACTCGCGTTTACGGTTTTTACGATGAGTGTAAAAGACGTTGCAATATCAAAATTTGGAAGACTTTCATAAACACCTTTAATTGCTTACCCATTGCGTCCGTTGTAGCCGGTAAAATTTTTTGCGTTCATGGTGGTCTTTCTCCATCGTTAAGTCATATGGATGACATACGGGAGATTCCTCGTCCTACTGACGTTCCTGACTATGGATTGCTGAATGATTTGTTATGGTCAGATCCTGCTGATACTGAAAACGATTGGGAAGATAATGAACGAGGTGTTTCTTTTGTTTTCAACAAAAACGTTATCCGGCAATTTTTGGCCAAGCACGACTTTGATTTAATCTGTCGTGCTCACATGGTTGTTGAAGACGGTTATGAATTTTTTAATGACAGAACTTTATGTACGGTGTTTTCTGCCCCTAATTACTGTGGGGAATTTGATAATTGGGGGGCCGTGATGAGTGTTAATTCAGAGCTGCTTTGTTCCTTTGAACTTATAAAGCCTCTCGACCAGGCAGCAATTCGACGAGAATTAAAAAAGAGCAAGCGTAGTGGTATGGCTATTTATCAATCTCCCCCTGCTGAGCAGGTTACCCAAAGCGTGTAAGCATTCTATTTTTTCTATTTTGGTCTCCATTTGAAATTATCTAGCGGTTTTATTGGGACACTATTTTATTCTCTTAAATGAGTATTATCTGTTATTATGTTATAGTCACTATGGTGATGGATCAAGTCCTTGGTTCACTTTATCGGTTCGAGGGCAATTCCATCCTGACACGTTTAGTATAAGTTTCTGTTTCTTTTAATAAAAACACAATCATTTCCATATAGTGACAGTTAACTTGCTTTTTATTTTTGCTAACCGACACCTGCTGAACATCGAAAGAGTAGGTAAAAAATTTACATGAGGAGGAAATAAACAATTATTATAGTATTATTATTAGCATTAAATAAATTTAAAGCGAAAATGT\tLYYTSHHLLTHSSPRPYKFLNDCVLSAKQNIFIDQDYLKKRKSTNLRKISKNRKNLGTIGKPFNFGTKCKQRSLDFEVKKKIKIEQSGIVSRNLPLIEIQWDRGAVNTPTASSIPTHPFLGQTLKDPLSL*\n-TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTTCGTACGGATCATGACAGTTATTTTATCGTTCTCTTCATCCTGTTTTTTTTTGTTTTTTAATCGATATTTGATATCGTCAAATATTTCTATCTATATGGAGAGTTCTTATATTTGTTTTTTGAATAATCACACACTCTTCAAAACAGCAGCTGCATTCAATAATGTACTTGGCATTGGAGTAACGCACATTATACAAGAATTACCGTGGTCCGTAAACACTTGCAAGCATGAAATACTTAGAAAGCTTTAGGCAGACGCTAGACGTGGGGAAAGGGATGAAGTGTTAAAAATTCAAGAGTGTGGATGTTTTGTATGCAAACTGAGTGAGTTCTCGGTTTTTGTAGTTACGTGACATAACTTTAAAAATTTTATGAGAAATTGACTAATATCAAGAATTGAGAATAGTTTTTTAATACAATACACAAAATTTTTAC\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n-TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0030552^molecular_function^cAMP binding`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0110034^biological_process^negative regulation of adenylate cyclase-activating glucose-activated G-protein coupled receptor signaling pathway`GO:2000480^biological_process^negative regulation of cAMP-dependent protein kinase activity`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTCGTACGGATCATGA\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n+TRINITY_DN597_c0_g1\tTRINITY_DN597_c0_g1_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g1_i1|m.662\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTTCGTACGGATCATGACAGTTATTTTATCGTTCTCTTCATCCTGTTTTTTTTTGTTTTTTAATCGATATTTGATATCGTCAAATATTTCTATCTATATGGAGAGTTCTTATATTTGTTTTTTGAATAATCACACACTCTTCAAAACAGCAGCTGCATTCAATAATGTACTTGGCATTGGAGTAACGCACATTATACAAGAATTACCGTGGTCCGTAAACACTTGCAAGCATGAAATACTTAGAAAGCTTTAGGCAGACGCTAGACGTGGGGAAAGGGATGAAGTGTTAAAAATTCAAGAGTGTGGATGTTTTGTATGCAAACTGAGTGAGTTCTCGGTTTTTGTAGTTACGTGACATAACTTTAAAAATTTTATGAGAAATTGACTAATATCAAGAATTGAGAATAGTTTTTTAATACAATACACAAAATTTTTAC\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n+TRINITY_DN597_c0_g2\tTRINITY_DN597_c0_g2_i1\tKAPR_SCHPO^KAPR_SCHPO^Q:202-1437,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN597_c0_g2_i1|m.663\t202-1440[+]\tKAPR_SCHPO^KAPR_SCHPO^Q:1-412,H:1-412^100%ID^E:0^RecName: Full=cAMP-dependent protein kinase regulatory subunit;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC8C9.03`KO:K04739\tGO:0005952^cellular_component^cAMP-dependent protein kinase complex`GO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0047555^molecular_function^3\u0027,5\u0027-cyclic-GMP phosphodiesterase activity`GO:0030552^molecular_function^cAMP binding`GO:0004862^molecular_function^cAMP-dependent protein kinase inhibitor activity`GO:0008603^molecular_function^cAMP-dependent protein kinase regulator activity`GO:0019933^biological_process^cAMP-mediated signaling`GO:0042149^biological_process^cellular response to glucose starvation`GO:0019934^biological_process^cGMP-mediated signaling`GO:0051447^biological_process^negative regulation of meiotic cell cycle`GO:0046827^biological_process^positive regulation of protein export from nucleus\t.\tTCTCAACCGTAACAAACGAAGGTAAATAACTGATTTGTATAGCTGAATTCAGCTCTTTTATTTTGGAAAATTTACCTCTTGTGGTTGATTAAAGTGTCAGGAAAAGCTGAAGCGAACAAAAACCGCGAGATTGTGATTCACTGAGAGTAAGAAACAAAAAACCGCATTTTTTGAAAGAAAGGCACGTAAACTTTAGGAGACATGTCTTTCGAAGAAGTATATGAGGAGCTAAAAGCCCTTGTTGATGAGCAAAATCCTTCAGATGTACTTCAATTTTGCTATGACTTTTTTGGAGAAAAGCTGAAAGCTGAGCGCTCAGTATTTCGTCGAGGCGACACCATTACAGAGTCTTTTAGCGACGGAGATGAAAGCGATTTCCTGTCTGAGCTCAACGATATGGTTGCTGGCCCCGAGGCAATTGGTCCGGATGCCAAATACGTTCCTGAATTGGGAGGATTAAAAGAAATGAACGTTTCATATCCGCAAAACTACAATCTGTTGCGTCGCCAATCTGTTTCCACTGAGTCGATGAATCCTAGTGCATTTGCACTAGAGACCAAACGAACCTTTCCTCCGAAGGATCCTGAAGATTTGAAAAGACTAAAGCGCTCAGTTGCTGGGAACTTTTTGTTTAAGAATTTGGATGAAGAACATTACAACGAGGTTCTGAACGCTATGACTGAAAAACGGATTGGAGAGGCTGGTGTTGCTGTCATTGTCCAAGGAGCAGTTGGAGATTACTTTTATATTGTGGAGCAAGGTGAATTCGATGTATACAAAAGGCCCGAACTAAATATTACCCCTGAAGAAGTGCTTTCGAGTGGTTATGGTAACTACATAACTACGATTTCTCCCGGTGAATATTTTGGAGAGTTGGCACTCATGTATAATGCTCCACGAGCTGCATCGGTGGTCTCCAAAACACCAAATAATGTAATTTACGCATTAGATCGTACAAGTTTCCGTCGTATCGTATTTGAAAACGCATATCGCCAGAGGATGCTGTATGAAAGCCTCCTTGAAGAAGTTCCCATTCTCTCAAGCCTTGACAAATATCAACGACAAAAAATTGCCGATGCTCTTCAGACAGTCGTGTATCAAGCCGGTAGCATCGTTATTCGACAAGGCGACATTGGTAACCAGTTTTATCTGATAGAGGATGGCGAGGCTGAAGTAGTGAAGAATGGCAAAGGCGTAGTCGTTACCCTAACCAAGGGAGATTACTTTGGTGAACTAGCTTTAATTCACGAAACAGTGAGAAATGCGACGGTACAGGCAAAAACTCGCCTGAAACTTGCTACATTTGATAAACCTACATTTAATAGACTACTTGGTAACGCGATCGATTTGATGCGTAATCAACCTCGTGCCAGGATGGGAATGGATAATGAGTATGGTGACCAATCTCTTCATAGGTCACCTCCATCAACTAAAGCATAAAATTTCTATAATCTGTTTTCTTTGAAACTATCTTTGTAATAATGACCTGTCATGATATGCTAAAGAATTTCCCTGTTGTTTGTTTGCCTTCTAACTTTTTCATATTGGTCATGCATGCATGTGATTGTTTACCATTATCCCTGGAATAATTTACTTATTCATGTTGAACATGTTTTGTTTGGATTACTTATCTAATTCAGCATTATATGTGATAATGTATATATACGTATTATTCTTTTTTATTTCTTTGAAAACCTTTATGTCTTGTTCTAGCATGGCACTGGATATGGTTTGTTAATGTTTTAATTCCAATTATATTCGTGATTACCACTTCACTGTTCTTATTTTGTTTTTTTTTTTCTTTTTGAAATTTCCTTTTTTTTTTATTATTTTTATCTGTCGAATATCGCAGCATTTGATGTTTGGAAGCTTACCGTTTAGTTTGTGTTTTTTTTTCGTACGGATCATGA\tMSFEEVYEELKALVDEQNPSDVLQFCYDFFGEKLKAERSVFRRGDTITESFSDGDESDFLSELNDMVAGPEAIGPDAKYVPELGGLKEMNVSYPQNYNLLRRQSVSTESMNPSAFALETKRTFPPKDPEDLKRLKRSVAGNFLFKNLDEEHYNEVLNAMTEKRIGEAGVAVIVQGAVGDYFYIVEQGEFDVYKRPELNITPEEVLSSGYGNYITTISPGEYFGELALMYNAPRAASVVSKTPNNVIYALDRTSFRRIVFENAYRQRMLYESLLEEVPILSSLDKYQRQKIADALQTVVYQAGSIVIRQGDIGNQFYLIEDGEAEVVKNGKGVVVTLTKGDYFGELALIHETVRNATVQAKTRLKLATFDKPTFNRLLGNAIDLMRNQPRARMGMDNEYGDQSLHRSPPSTKA*\n TRINITY_DN547_c0_g1\tTRINITY_DN547_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tCTTCAACAATGAGTTTATAAGAACAACGCAATAATATTGCGGTTGAAATTGTAATTGTTAATCATGTCAGGAGATCTACTTTTAAATTGGTGGTTGTACTAGCTCGAGTAGTTTTCTTTTTAGACTTCCTTTTGTATGGAAACGTGCAATTTAATGCATGTGAATATAACCAGAGTTTTAGTGATGGAGGAATAGCCTTCCGATCTCCATAGCTTAATAGAAATGTTATTTTTA\t.\n********\n*SNIP *\n********\n TRINITY_DN5472_c0_g1\tTRINITY_DN5472_c0_g1_i1\tYF11_SCHPO^YF11_SCHPO^Q:2-94,H:69-99^100%ID^E:3e-14^RecName: Full=Putative uncharacterized protein C9E9.01;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC9E9.01\tGO:0016021^cellular_component^integral component of membrane\t.\tAGACAATGAAGATCGTTGTCGGTTATACCAAATTTACCGGGAAAGCAAACTCAAGGAATTTGCGAGATACAATGTCCTTTCAAGGGTTAATTGGTAAAGTAAACGACTATGAAATGAAAGAAATATCGTCGTTTAATTGAAATGGAGGAAGGATATTTCTCTTCTTTGTGCTAGTAAAGAGAAGACGGATATTTTTGGAAGAGATGTAAAAAGACAAGCAAACAAGAAAATGAAAATGGACGTCTAGCGATTTTACAAGAAAAAATTAGACGGGAAAAGGGAAAAGAAAAACTAAAGCTTATAG\t.\n-TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\tTAGAAAACCCTCCTAAATGCAATAACAGTGGCAATCACTCAGATAACCTAGGTTTTATCGAGCAGTCAGAAACAGTACATCCAGAGAATGAGAAGGCTTTGACGCCGGACCTGCGAGATACAAAAATACATACTTCTTTACCAATTACTACACCTTTTTCAAAGAAAAGGGCTCGAGAAGCCAAAAATATACTTCTTAAGCCGTTCAAAAGCCCCTTAAGACAAACTGCCAGTCCTCAAGTTGCAGACACTAATCTAAAACCATCGTTAGCTGTAACAAATCTTAACTCTGATGAAACAAACACTTCTTCAGAACCTGTTACTTCTCCCTTAAGAACAACGCCTAATTCAATAAAACGGCAAAAACGACTTTTCAAGTCACCTATCTCTAATTGCCTTAATCCAAAGTCCGACCCTGAAATCACACAGTTGCTTTCTCGTCGTTTAAAACTTGAAAAAGAAGTCAGAAATTTACAAGAGCAGCTCATAACCGCCGAAACTGCACGAAAGGTTGAAGCCAAAAATGAAGATAAAGACTTGCAAACGTTGATACAGAAATGGAAAAATGCTGCTCAGCAAGCCA\tENPPKCNNSGNHSDNLGFIEQSETVHPENEKALTPDLRDTKIHTSLPITTPFSKKRAREAKNILLKPFKSPLRQTASPQVADTNLKPSLAVTNLNSDETNTSSEPVTSPLRTTPNSIKRQKRLFKSPISNCLNPKSDPEITQLLSRRLKLEKEVRNLQEQLITAETARKVEAKNEDKDLQTLIQKWKNAAQQA\n-TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005622^cellular_component^intracellular`GO:0005634^cellular_component^nucleus`GO:0047433^molecular_function^branched-chain-2-oxoacid decarboxylase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0000949^biological_process^aromatic amino acid family catabolic process to alcohol via Ehrlich pathway`GO:0006067^biological_process^ethanol metabolic process`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006559^biological_process^L-phenylalanine catabolic process`GO:0006090^biological_process^pyruvate metabolic process`GO:0006569^biological_process^tryptophan catabolic process\t.\tGAATTTTTGGCACCATAGCCTTTAAGAAGGTTTTGATAGTCCCATTCAGTGTTGATGTCGTTGTAAACAGCGTGAAGACCGTGAATGAGACGTTCAATTGTATATCCCTTATTGTTAATTACAAATATGATAGGGGTGAGACCGTTACGGATGGTAGCAGAAATTTCCTGAACAGTTAAATGAAGGGAACCGTCACCAACAATCAATATGGAACGACGG\t.\n+TRINITY_DN5497_c0_g1\tTRINITY_DN5497_c0_g1_i1\tSFR1_SCHPO^SFR1_SCHPO^Q:3-581,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN5497_c0_g1_i1|m.6946\t3-581[+]\tSFR1_SCHPO^SFR1_SCHPO^Q:1-193,H:40-232^100%ID^E:3e-135^RecName: Full=Swi5-dependent recombination DNA repair protein 1;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPBC28F2.07`KO:K10992\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0035861^cellular_component^site of double-strand break`GO:0032798^cellular_component^Swi5-Sfr1 complex`GO:0003690^molecular_function^double-stranded DNA binding`GO:0003697^molecular_function^single-stranded DNA binding`GO:0007059^biological_process^chromosome segregation`GO:0000730^biological_process^DNA recombinase assembly`GO:1990918^biological_process^double-strand break repair involved in meiotic recombination`GO:0000724^biological_process^double-strand break repair via homologous recombination`GO:0031573^biological_process^intra-S DNA damage checkpoint`GO:0010772^biological_process^meiotic DNA recombinase assembly involved in reciprocal meiotic recombination`GO:0062110^biological_process^negative regulation of DNA recombinase disassembly`GO:0043388^biological_process^positive regulation of DNA binding`GO:0010520^biological_process^regulation of reciprocal meiotic recombination`GO:0042148^biological_process^strand invasion\t.\tTAGAAAACCCTCCTAAATGCAATAACAGTGGCAATCACTCAGATAACCTAGGTTTTATCGAGCAGTCAGAAACAGTACATCCAGAGAATGAGAAGGCTTTGACGCCGGACCTGCGAGATACAAAAATACATACTTCTTTACCAATTACTACACCTTTTTCAAAGAAAAGGGCTCGAGAAGCCAAAAATATACTTCTTAAGCCGTTCAAAAGCCCCTTAAGACAAACTGCCAGTCCTCAAGTTGCAGACACTAATCTAAAACCATCGTTAGCTGTAACAAATCTTAACTCTGATGAAACAAACACTTCTTCAGAACCTGTTACTTCTCCCTTAAGAACAACGCCTAATTCAATAAAACGGCAAAAACGACTTTTCAAGTCACCTATCTCTAATTGCCTTAATCCAAAGTCCGACCCTGAAATCACACAGTTGCTTTCTCGTCGTTTAAAACTTGAAAAAGAAGTCAGAAATTTACAAGAGCAGCTCATAACCGCCGAAACTGCACGAAAGGTTGAAGCCAAAAATGAAGATAAAGACTTGCAAACGTTGATACAGAAATGGAAAAATGCTGCTCAGCAAGCCA\tENPPKCNNSGNHSDNLGFIEQSETVHPENEKALTPDLRDTKIHTSLPITTPFSKKRAREAKNILLKPFKSPLRQTASPQVADTNLKPSLAVTNLNSDETNTSSEPVTSPLRTTPNSIKRQKRLFKSPISNCLNPKSDPEITQLLSRRLKLEKEVRNLQEQLITAETARKVEAKNEDKDLQTLIQKWKNAAQQA\n+TRINITY_DN5426_c0_g1\tTRINITY_DN5426_c0_g1_i1\tPDC4_SCHPO^PDC4_SCHPO^Q:218-3,H:443-514^100%ID^E:6e-43^RecName: Full=Putative pyruvate decarboxylase C3G9.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC3G9.11c`KO:K01568\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016831^molecular_function^carboxy-lyase activity`GO:0000287^molecular_function^magnesium ion binding`GO:0004737^molecular_function^pyruvate decarboxylase activity`GO:0030976^molecular_function^thiamine pyrophosphate binding`GO:0006091^biological_process^generation of precursor metabolites and energy`GO:0006090^biological_process^pyruvate metabolic process\t.\tGAATTTTTGGCACCATAGCCTTTAAGAAGGTTTTGATAGTCCCATTCAGTGTTGATGTCGTTGTAAACAGCGTGAAGACCGTGAATGAGACGTTCAATTGTATATCCCTTATTGTTAATTACAAATATGATAGGGGTGAGACCGTTACGGATGGTAGCAGAAATTTCCTGAACAGTTAAATGAAGGGAACCGTCACCAACAATCAATATGGAACGACGG\t.\n TRINITY_DN5486_c0_g1\tTRINITY_DN5486_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tCGCTTTATCGGAAAGACTTTGGCAATTTTACCGATTCTCTAGAATCATGTAAAGTGACGAAAACAGAAGGCAGAAAGTCCAACCAAATATGGGCAAAAAAAACAACCACTTCAATAAATTGTTCCAGTTAACAGCAGTTTAGTTGTGTTCAAGTCAATTACCAGCGTTTCGCGTGAAACGTTACAGAAGTGTAAATAATGAAATACTGTTGCTTTCTCTTTGACTCTCA\t.\n TRINITY_DN5468_c0_g1\tTRINITY_DN5468_c0_g1_i1\tCDC37_SCHPO^CDC37_SCHPO^Q:280-5,H:209-300^98.91%ID^E:6e-41^RecName: Full=Hsp90 co-chaperone Cdc37;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPBC9B6.10`KO:K09554\tGO:0005737^cellular_component^cytoplasm`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0051087^molecular_function^chaperone binding`GO:0031072^molecular_function^heat shock protein binding`GO:0019901^molecular_function^protein kinase binding`GO:0051082^molecular_function^unfolded protein binding`GO:0007049^biological_process^cell cycle`GO:0051301^biological_process^cell division`GO:0061077^biological_process^chaperone-mediated protein folding`GO:0006457^biological_process^protein folding`GO:0050821^biological_process^protein stabilization\t.\tCCGGTGCTAAGGACGGCTTCCCTTCTAGTTCAGCATTAAAGGCTTCCAAAAGAATAGCATCAGATTCTTCCTCATCTGCCAAGATATTTAAATGTTCTTTCAAAAACTCTTCCGATGAAGAATAATCACCAAAATCAATATTGGCAAACTTCTCACCATCTTCAGAAAGAGACAACCCTTCTTCATCTTCCGACTCATTCTCATTGACAGGATTTTTAGGGGCATTACCAGTAGAAGCTTGCTCTTTAGCGCCGGGAAAATCAGTAGGAGGTGTAGGGTT\t.\n-TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\tTTTCACCTCTCACAAACCATTGGACCATATGGCTTACATCAAATAAGCCAGAATGCTCCCGTGTCCACTTATGACTGGCACTAATGGTTTGACCTTTATACTGTACTGGCATTGAAAAACCAGCAAAGGGGACTATGGTAGCTCCTTCTTTTAAATGCAGATCATATAAAGGAGTTCTCTTGAGTGAAGAAGAAGCTGCCGTAGAACTTTGGCGCTTTAAGATTGATAAAGCAGCAGAA\t.\n+TRINITY_DN5434_c0_g1\tTRINITY_DN5434_c0_g1_i1\tGCST_SCHPO^GCST_SCHPO^Q:238-2,H:4-82^100%ID^E:2e-49^RecName: Full=Probable aminomethyltransferase, mitochondrial;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC31G5.14`KO:K00605\tGO:0005960^cellular_component^glycine cleavage complex`GO:0005759^cellular_component^mitochondrial matrix`GO:0005739^cellular_component^mitochondrion`GO:0004047^molecular_function^aminomethyltransferase activity`GO:0008483^molecular_function^transaminase activity`GO:0019464^biological_process^glycine decarboxylation via glycine cleavage system\t.\tTTTCACCTCTCACAAACCATTGGACCATATGGCTTACATCAAATAAGCCAGAATGCTCCCGTGTCCACTTATGACTGGCACTAATGGTTTGACCTTTATACTGTACTGGCATTGAAAAACCAGCAAAGGGGACTATGGTAGCTCCTTCTTTTAAATGCAGATCATATAAAGGAGTTCTCTTGAGTGAAGAAGAAGCTGCCGTAGAACTTTGGCGCTTTAAGATTGATAAAGCAGCAGAA\t.\n TRINITY_DN5400_c0_g1\tTRINITY_DN5400_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tAAAGTCCTCCTTATTTTTTTTGCGTTGAAATGTAGTTTGTAGTGTCAAGCTGCTTTGTGCTATTTATCGAGCAATGAGAAATAGGAGAGGAGTTATGGTAGCAGGGTATACTTAGTAATGTTTTCATGGACCAAGCGGGGGAGGGGCGAAAAGCATGTCTAGTTCGCTGTATTCAGTTTCAAGACCCCATTTCATGTTTAACATTTGTTCTAGATACGCAAAAATCCCTTTTTTCTAGATTGTTTGCAAGTCCC\t.\n TRINITY_DN5476_c0_g1\tTRINITY_DN5476_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGTTTGTTTACACCTATGTGTGATCCACTTTTGAGGTATATACTGGTTCAAAAACTCGAAAACTCTTCTTTTATCCATCTTGTCATAACTTTTGCTTTTTTATTTTTATTTTTACCCATTCCCACATTTCTTATACTAAAAGGAGAGGTTTACTATTGACATTCTTGACGTTTATTAGACCCTTGATTCTAACGCTGAAGGGTGGTTATCTTTGGTTTTCTTGTCAGGTATACGAGATAAGTAGCCTTTTTATAATTCTCCCTGAATTCATTACTTCTCCTCTGCCTTAATATACTATTTTGTTTAGTCTTGTATGCAAGGCGCTTAATCCTCTTCTTCAAGTATTATTCCTCACAGTTCATTTCGCTTTTTGGTTTAAGTTGACCTAATTCATGCCC\t.\n TRINITY_DN5490_c0_g1\tTRINITY_DN5490_c0_g1_i1\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\t.\tCATGTCTGCTGCATTCACGATCTCCAAGAAGAGAAAAGTAAGTAATCTTTTATTAGAAAAAAAATCGATGTTTGTGTAGGGATATATATGATGATTTCCAATATCTGGCACCTCTTGTCTTATGGGATTCTTAAAAGTTTCTCGATGAACTTAACCTTTTGGAACTATATCTGATATCAACGTATATTGTTAAACGATAATAATGTAAAATTATATTTTTCTACGCATAAGATCTTTTTTCCTAATATCGTTCACTTTGTGCTTCTTTCTAAAGTACTGACCAATTTAGTTCGTTGCTGATGGTGTTTTCTACGCCGAGTTG\t.\n-TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0098713^biological_process^leucine import across plasma membrane\t.\tGCCGACAATTAGACCCACCGGACCACCAATATGCAGGGCACTGCCCATACCAATGAAGACACCTGGACCGATGATACCAGCAAGGGCCATCATCGATACATGACGAGTCTTCAATCCACGTTTGATTCCATTTGAATTATTTTCCGCTGCGTCAACTGTTACAACTTCGGAAACTCCAATGTCGAGGTTTTCTGAGGAGGATAGCTTCTTGGCAGGAACTTTCTCAGAAGGCGCTGCCTCCATTTTCTTAATTAACTTAATTCACTTTATTCTTAATATTAGAAATTACAATTGGTATTGTGATCTATATATGTACAACAGAATTGATAAATGTAATCAATTCTAATAATGTTAGCTGCAGAATCAACATGAATTCTACCGCTAA\t.\n+TRINITY_DN5443_c0_g1\tTRINITY_DN5443_c0_g1_i1\tYCUB_SCHPO^YCUB_SCHPO^Q:243-1,H:1-81^100%ID^E:2e-46^RecName: Full=Uncharacterized amino-acid permease C965.11c;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC965.11c`KO:K16261\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005794^cellular_component^Golgi apparatus`GO:0016021^cellular_component^integral component of membrane`GO:0015171^molecular_function^amino acid transmembrane transporter activity`GO:0015190^molecular_function^L-leucine transmembrane transporter activity`GO:0003333^biological_process^amino acid transmembrane transport`GO:0098713^biological_process^leucine import across plasma membrane\t.\tGCCGACAATTAGACCCACCGGACCACCAATATGCAGGGCACTGCCCATACCAATGAAGACACCTGGACCGATGATACCAGCAAGGGCCATCATCGATACATGACGAGTCTTCAATCCACGTTTGATTCCATTTGAATTATTTTCCGCTGCGTCAACTGTTACAACTTCGGAAACTCCAATGTCGAGGTTTTCTGAGGAGGATAGCTTCTTGGCAGGAACTTTCTCAGAAGGCGCTGCCTCCATTTTCTTAATTAACTTAATTCACTTTATTCTTAATATTAGAAATTACAATTGGTATTGTGATCTATATATGTACAACAGAATTGATAAATGTAATCAATTCTAATAATGTTAGCTGCAGAATCAACATGAATTCTACCGCTAA\t.\n TRINITY_DN5471_c0_g1\tTRINITY_DN5471_c0_g1_i1\tRRG1_SCHPO^RRG1_SCHPO^Q:214-101,H:266-303^100%ID^E:3e-21^RecName: Full=Protein-lysine N-methyltransferase rrg1 {ECO:0000250|UniProtKB:P38347, ECO:0000305};^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPCC338.11c\tGO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0016279^molecular_function^protein-lysine N-methyltransferase activity`GO:2000765^biological_process^regulation of cytoplasmic translation\t.\tAAATTATTCAAACAAGGAAAAAAATAAAACCCAATTATTTTGGCTCGCTTTTTCTAGTTTGCAACATGGTTTGACCTATAGATTGATACATTTTAAGTTATCCATGGTATTTCCATCTACTCCAACGACAAGTTACTGGGTATAGAGAGCCAAAGTCTTCTTCACCAATTTCTTCTCCCATTTGTCTTTCAAATCCGGCGGCGTCCATTCCCTTTT\t.\n-TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC1851.03`KO:K03115\tGO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0043433^biological_process^negative regulation of DNA-binding transcription factor activity`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0007165^biological_process^signal transduction\t.\tGCGATCCTGGATGAAGTCTTCGTCGACTTCGCAAAAAAATTCGTTTCCTTTTAGTCCTAAAAACCAATCAACCCAATATTGAGAATCATCGGATTCACTGCAATTAATGATTAATAAACGGTCCAATATGGCCAAAATGTAAAATATTTTTTTATTTAACCCTAATTTGCACAAACCTTTCCGAAGAGTATAGCTGCATGCTTCGTTCCG\t.\n+TRINITY_DN5407_c0_g1\tTRINITY_DN5407_c0_g1_i1\tCSK2B_SCHPO^CSK2B_SCHPO^Q:81-1,H:14-40^100%ID^E:2e-14^RecName: Full=Casein kinase II subunit beta;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\t.\tGO:0005737^cellular_component^cytoplasm`GO:0044732^cellular_component^mitotic spindle pole body`GO:0005634^cellular_component^nucleus`GO:0005956^cellular_component^protein kinase CK2 complex`GO:0034456^cellular_component^UTP-C complex`GO:0019887^molecular_function^protein kinase regulator activity`GO:0043539^molecular_function^protein serine/threonine kinase activator activity`GO:0030291^molecular_function^protein serine/threonine kinase inhibitor activity`GO:0006974^biological_process^cellular response to DNA damage stimulus`GO:0035690^biological_process^cellular response to drug`GO:0000122^biological_process^negative regulation of transcription by RNA polymerase II`GO:0018105^biological_process^peptidyl-serine phosphorylation`GO:0006356^biological_process^regulation of transcription by RNA polymerase I`GO:0007165^biological_process^signal transduction\t.\tGCGATCCTGGATGAAGTCTTCGTCGACTTCGCAAAAAAATTCGTTTCCTTTTAGTCCTAAAAACCAATCAACCCAATATTGAGAATCATCGGATTCACTGCAATTAATGATTAATAAACGGTCCAATATGGCCAAAATGTAAAATATTTTTTTATTTAACCCTAATTTGCACAAACCTTTCCGAAGAGTATAGCTGCATGCTTCGTTCCG\t.\n TRINITY_DN5498_c0_g1\tTRINITY_DN5498_c0_g1_i1\tSRP2_SCHPO^SRP2_SCHPO^Q:213-145,H:22-44^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`SRP2_SCHPO^SRP2_SCHPO^Q:91-56,H:44-55^100%ID^E:9e-10^RecName: Full=Pre-mRNA-splicing factor srp2;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\t.\t.\t.\t.\tKEGG:spo:SPAC16.02c`KO:K12893\tGO:0005634^cellular_component^nucleus`GO:0003729^molecular_function^mRNA binding`GO:0008143^molecular_function^poly(A) binding`GO:0001069^molecular_function^regulatory region RNA binding`GO:0045292^biological_process^mRNA cis splicing, via spliceosome\t.\tAAGCATACGAAAGTCTAAACTGTTAGCAAAAGTTTTAAGATGGTATGAACGTACCATTTACGATATCACGAGCATCGCGAGCGTCTTCAACCTACATCAAGTTAGTCACATAGGGATTACAAAAAGAAGGCTAATGGTTTTATTTACCTCAACAAAGCCAAATCCATTCATCAGCTTGCAATCAAGGATTTGACCGTAACCTTTAAAAAAATCCA\t.\n-TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFLIKEVTHLLKLWHEVSVRAGVAQTSKPSFDAPLVLEAIEKYKNALGAPDTATKSCKENLISLDSEIRQTY*\n-TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006405^biological_process^RNA export from nucleus`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFPNKRSYPSIKIMA*\n+TRINITY_DN54_c0_g1\tTRINITY_DN54_c0_g1_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3993,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g1_i1|m.8839\t1-3996[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1331,H:1-1315^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFLIKEVTHLLKLWHEVSVRAGVAQTSKPSFDAPLVLEAIEKYKNALGAPDTATKSCKENLISLDSEIRQTY*\n+TRINITY_DN54_c0_g2\tTRINITY_DN54_c0_g2_i1\tNG06_SCHPO^NG06_SCHPO^Q:49-3783,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces`NG06_SCHPO^NG06_SCHPO^Q:3758-3994,H:1239-1315^93.67%ID^E:2e-35^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN54_c0_g2_i1|m.8842\t1-3825[+]\tNG06_SCHPO^NG06_SCHPO^Q:17-1261,H:1-1245^100%ID^E:0^RecName: Full=Probable nucleoporin C890.06;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPAC890.06`KO:K14312\tGO:0005737^cellular_component^cytoplasm`GO:0005635^cellular_component^nuclear envelope`GO:0005643^cellular_component^nuclear pore`GO:0044611^cellular_component^nuclear pore inner ring`GO:0017056^molecular_function^structural constituent of nuclear pore`GO:0006606^biological_process^protein import into nucleus`GO:0036228^biological_process^protein localization to nuclear inner membrane`GO:0006407^biological_process^rRNA export from nucleus`GO:0000972^biological_process^transcription-dependent tethering of RNA polymerase II gene DNA at nuclear periphery\t.\tCACCAGCAGCCAACATTTCCAGCTGTCCTTCATTTGGAGCTTGGGAAAATGGACATCAATGATCCATCATTTTCATCGACTGAAAATGGTATTTATGAACTTCTATCACGTGGCTATGCTGCGATTGAAGCTGCAACGGAGCGTGATTTAAAATTGCCAGAGTTAGGAGACATTATTGGTCAAGTACACGCTCCAGAATACATATCACAAGTTCTGAGTGGTTGGAAGCCGTTTTACCTTCGTTCTGTTGTAAATATTCCTGACCGTATATTTGAACAATACAATCGTACGGAATGTTTTACTCAAATGGGGCTTTTTGCTGAAATCCAACGTGCTTGGATAACTGTGGACAACCGTTTGTTCTTATGGGATTATCTTTCGGGTCAAAACTTTCAAGCTTACGAAGATTTATCTCATACCATTGTCAATGTAAAGTTAGTGAGACCAAAGGCCAATGTTTTCGTCTCGGAAATCCAGCATTTACTCGTAATAGCAACTTCACAAGAAATGCTCTTATTAGGTGTTACAATCGATGAGAAAACTGGTGAATTATCCTTCTTCTCAACTGGTATCCAAATATCTGTTCAAGGGATTAATGTTAATTGTATCGTTTCTTCGGAGGATGGACGTATTTTTTTTTCCGGAAACAAAGATCCCAATTTGTATGAATTTTCTTACCAACTCGAAGAAGGCTGGTTTTCTCGTAGATGCAGCAAAATTAATATTACCGGTTCTGTTTTTGATAATTTTATTCCTTCTTTCTTTTCTTTTGGTACTCATGGTGATGGCATCAAGCAGATTGCTGTTGATGACTCCCGTAGCTTGCTTTATGTTTTGCGAGAGACCTCTTCTGTTTCTTGTTATGAACTTACGAAAAACGGCGTAAATCGTTGTGTCTTTTATTCTTTCAGCTCTATGATCTCTCAAGCTCAAATGCTAAATGCAACCTCTCCCTTATTGGATCCTCGGACTACTCAAATAGTTTCCATAGTTCCTATTCCAGCATATGAATCACAGCAAATATACTGTGTCGCAATTACCTCTACAGGATGTCGGTTTTACATGAGAGGTGGCCGGGGCCCCATTTCTCACTATGCACCGTCTAATTCGACTTTGTCGTCCACACCTCCTAGTACTTTACAACTTACTTTCGTTCGTTTTCCTCCTCCTATGCAAGTGGAGAATTATGCCTCTTCCCGCAATTATCCTGCTAACCCTTTCTTCCTGCAAAATCAATCTACATCTCAACAACAACCTGAAAGGTCAAGTGCTGTAAAAACTACTCCTATGAAATGTTCTTCTCTGAGTAATATTTATACATCCGATTTGTTTTTCGCCATAAGCTCCTCCAATACGAATGAGGGCGACGTCGTCTGTTGTACTGCTCCTGAAGTTGGTAGAATTGCCAATGCTTGGCAGAGTGGTACGCAACCGTCTCTCATAGAGTCTTCCATGTACGTTCCTATTAAGGGTTTTGTACAAGATATTAAATGCATTCAAAATTCTCGAGAACGCAATGAGTTGGTTTCTCAATTTAACACACCTCCTCCTACTTTTGCCATACTTACGAATACAGGGGTTTACGTGGTAGTTCATCGTCGTCCGATCGATGTCCTCGCTTCTGCTATTCGTATGGGACCTTCCCTATCGTCCGGAATCGATGGCCAAGTTCAACTGTTTTTTGAGAGTGTTGGTCGCGCTGAAGGTTGTGCAACTTGTTTGGGTATTGTAAGTGGATGTCTTGATCAGGGGGATTTTAGTCATGCTGCAGCAAACTTCTCTGGCTCCACTACTAAACTTGCACAAGCGGATCTTTTGGACATAGTAAAGAAATATTATATTGAGTTTGGCGGTAAAGCTTTTATTGATCAATCTCGCTATAACAACCAGTATGATTCTTCTTCTTTAGAATTCGTTCGTTTGTCTGGTTGTCATGATGGTTTGGCGTCCTCCATTTCGCGTATTATTAGAAATGTTTGGAAAAATCATGTCATTATAGCTAAAAAAATGCAAAATAAACGTATCCATTACGCGCCCGCATTTAACGCTACTGAAATTTTAAAAATACAATCTGGCTTGTTATATTTATCTACGTTCCTTGAAAATAACAAAAGTTTTATTGAAGGCCTTAATAGCCCAAATACCCTCATCGGCTCATCCAATGTTGCTGATGAAATTGCTGTACAAGCTGAACATAGAGCTTTGTCTGCATTGCTCTTGGTCCTCCAACAAATTGTTGAGGGGATATCGTTTTTGTTATTCTTAAACGATACCGGTGTATCCGATTTTCATGAAATTGTTTCTTCTACCTCCATTGATATTCAAAAATCGTGTTCAAACATGACCTTTGGTGAGTTTTTTACTTCGAAACGGGGTCGAGAAGTAACTAAGGAACTCGTAAATTCTCTTGTCAATAGGCATCTGCAGTCTGGAGGGAATATTGATATGGTTAGTCAGCTTTTACGTAAAAAGTGTGGAAGTTTTTGCAGCGCTGATGATGTTTTAATTTTTAAAGCCGTGGAGTCTTTAAAAAAGGCTAAGGATACGGTCGACATTGAAGAGCGACAATCTCTTATAGAGTTATCCTACACCCTCTTTAAAAAGGCTGCACATGTCTTTACTCCCGAAGACCTTCGTTTAGCAGTTGAGGAATACAAGTCTTTGAACGCCTATACAACCGCGGTAAACTTAGCATTACACGTAGCAAGTGCGAGAGACGATCGAAATCAAGCTTTGTCTTACCTTGTCGATGGAATGCCTGAGAACGATCCTAGGCGTGAACCTTTCGAGAGCCGTACAAAGTGTTATTCGTACATTTTTGAGATTTTGGATTCTTTGGAATCTCAAATGTCAAATGATTCGAGTGCAATTAAGGTTGATGTATACGATACTATACAGCGTTCAAAGGATGAACTTTTTCATTATTGCTTTTATGATTGGTATTCATTTAAGGGGCTTACAGATCGCTTGATTGAGATAGATTCCCCTTACATACAGAGTTATCTGGAACGTAACAGCACCAAAGACATGAAAATTGCTGACTTACTATGGCAGTATTATGCTAAACGAGAACAGTATTATCAAGCTTCAATAGTCTTGTATGATTTGGCTACTACCCACCTTGCATTTAGCCTTGAACAACGCATTGAATATCTTACACGAGCCAAAGGTTTTGGTAGCTGCCACGTCCCTAATTCTCTTAGGCATAAAATGAATAAGGTTATGCTTAGTGTACTAGAGCAGCTAGACGTTGCTAGTATCCAAGATGATGTTTTAATAGCAATCCGTGGTGATATGCGTATTCCTACTTCAAAAAGAGAAGAATTGAGCAAACAATTGGATGGGGAAATTATACCCCTTAGCGATTTATTTAATAATTATGCTGATCCTCTGGGTTACGGTGAGATATGCTTGTCCATATTTCAGTGTGCTGATTACAGAGGAATTAACGAAATCTTAAACTGTTGGGAGTCTATTATTAAAACTACTCATGAGAATGCTATAATATCACCAGTTGGGTCATCCCCTGTCGAAGCAGTTTCCTCTACATTAAAAAATTTGACGCTGAGATTTTCACAATCTGAAAACGTTTTCCCTATTGAGCAAATAATTGACATTACTGAAAGATATGCGTTTGATCAACAAGGTGAAGCGGTAGCTACTGGTTGGGTTATTGATACATTCTTGGGAGCCGGCGTTTCTCATGAATTAATATTTATTGTACTAAACCAGCTCTACGATCGCAGAGAAAAGCCTTGGCAAGGAAAAGATAGGCTTTTTTTTCCTAATAAAAGAAGTTACCCATCTATTAAAATTATGGCATGAAGTGAGTGTAAGAGCCGGAGTTGCTCAAACCTCCAAACCCAGTTTTGATGCTCCTCTTGTTTTGGAAGCAATAGAGAAATACAAAAATGCGCTAGGTGCACCCGATACTGCCACGAAAAGTTGCAAAGAAAATTTAATATCTTTAGATTCTGAAATTCGTCAGACATACTAGTGAAAAGATAGCTTTTTATAACAACATTTTTTAAAAAGATGGTGTTTGGTATTTAATAAATGCAATGATTATATTGGATGGATTTGAAACCTTCACATAATTGGATGTTGTTAACTATCGCTATGACTGTTACATATAAAAAAACAATTTAGAAAATACTCTTTTACAGCACTTGCA\tHQQPTFPAVLHLELGKMDINDPSFSSTENGIYELLSRGYAAIEAATERDLKLPELGDIIGQVHAPEYISQVLSGWKPFYLRSVVNIPDRIFEQYNRTECFTQMGLFAEIQRAWITVDNRLFLWDYLSGQNFQAYEDLSHTIVNVKLVRPKANVFVSEIQHLLVIATSQEMLLLGVTIDEKTGELSFFSTGIQISVQGINVNCIVSSEDGRIFFSGNKDPNLYEFSYQLEEGWFSRRCSKINITGSVFDNFIPSFFSFGTHGDGIKQIAVDDSRSLLYVLRETSSVSCYELTKNGVNRCVFYSFSSMISQAQMLNATSPLLDPRTTQIVSIVPIPAYESQQIYCVAITSTGCRFYMRGGRGPISHYAPSNSTLSSTPPSTLQLTFVRFPPPMQVENYASSRNYPANPFFLQNQSTSQQQPERSSAVKTTPMKCSSLSNIYTSDLFFAISSSNTNEGDVVCCTAPEVGRIANAWQSGTQPSLIESSMYVPIKGFVQDIKCIQNSRERNELVSQFNTPPPTFAILTNTGVYVVVHRRPIDVLASAIRMGPSLSSGIDGQVQLFFESVGRAEGCATCLGIVSGCLDQGDFSHAAANFSGSTTKLAQADLLDIVKKYYIEFGGKAFIDQSRYNNQYDSSSLEFVRLSGCHDGLASSISRIIRNVWKNHVIIAKKMQNKRIHYAPAFNATEILKIQSGLLYLSTFLENNKSFIEGLNSPNTLIGSSNVADEIAVQAEHRALSALLLVLQQIVEGISFLLFLNDTGVSDFHEIVSSTSIDIQKSCSNMTFGEFFTSKRGREVTKELVNSLVNRHLQSGGNIDMVSQLLRKKCGSFCSADDVLIFKAVESLKKAKDTVDIEERQSLIELSYTLFKKAAHVFTPEDLRLAVEEYKSLNAYTTAVNLALHVASARDDRNQALSYLVDGMPENDPRREPFESRTKCYSYIFEILDSLESQMSNDSSAIKVDVYDTIQRSKDELFHYCFYDWYSFKGLTDRLIEIDSPYIQSYLERNSTKDMKIADLLWQYYAKREQYYQASIVLYDLATTHLAFSLEQRIEYLTRAKGFGSCHVPNSLRHKMNKVMLSVLEQLDVASIQDDVLIAIRGDMRIPTSKREELSKQLDGEIIPLSDLFNNYADPLGYGEICLSIFQCADYRGINEILNCWESIIKTTHENAIISPVGSSPVEAVSSTLKNLTLRFSQSENVFPIEQIIDITERYAFDQQGEAVATGWVIDTFLGAGVSHELIFIVLNQLYDRREKPWQGKDRLFFPNKRSYPSIKIMA*\n TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8862\t1057-1950[-]\tRNG3_SCHPO^RNG3_SCHPO^Q:1-297,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.04c`KO:K21991\tGO:0005826^cellular_component^actomyosin contractile ring`GO:0005829^cellular_component^cytosol`GO:0005634^cellular_component^nucleus`GO:0005844^cellular_component^polysome`GO:0017022^molecular_function^myosin binding`GO:0034613^biological_process^cellular protein localization`GO:0000917^biological_process^division septum assembly`GO:1903475^biological_process^mitotic actomyosin contractile ring assembly\t.\tAGAACTTGACCGCAGCGCACATCAACTACACCTTGAGGCAATTCAAAATATGAAGTTTTCTACCGTAGGGTTTTTGTTTTCAACAATTTTATTCAAAAGTGCGTTTGCAGGATGGATGGATACACACATGAAAGATGAACATCACATTGATAAGTACACAGATGAATCTTTTTTTCGTCTTCACGATTTAGGAAAAAAGGGATACTGGTCAGATCAAGATATACTTAGTTTATATGGATTATTTGAAAATGATGAGGTCCCATTCGTCAAAAAGAATGAGGTTTTGGTTGATGTTCTAAAAAAATGCGATCCATCGGGCAATCGCCGGATAACACTTGATGAATTTCTTGCATTTCGTAAAAATGGAGGAGAGTTAACGGATTTTGGATTTCCAGGGCACCATGGTGATGAGGAAGAGGAATTTGAAATGCATCATGTGGAAAAATATCACCCAGCTGGCCTCGATGAACCAGATGAAAACTGGAATCATCCTGAAGATATCGAACATTTTCAAAAACACGATGAGATATTTCATGGTGATAAGAAACCTGAGGAACGACGCAAACATTTTGTAAAATACAATAATATCCCTGACAAGTATCGTCGTGTGAGCATTTAAATTTTCCCATTGATAGATTTACTTGCTTCTCCTTGGGACGTTTTATTTTTATTTTTATTTTTCCGTTTATTTTTGGATTGTCGTGTTCCGATATATATCCGTGTTTACAAACAATAGCTTCAACATTTTTGAACGGGTCCTTCCAATACTAATGCCATATCTTATTTCTTTGTCAATACAAAATCACTTTTGTTCGTTAAACTCATTTTGGTTTGCAGATATTATCATATTCTTACCAATTATACATACACTCACAATTTGTTTTACATTTCCGCTCCCTTCTTCAATAAGCTGATTGCTTATAAAAAATCAAACATTCATATGATTCAAGTTCAAATATTATCATCTGAAAACTGACAAGTTCCATTCATTACTATATTAATATCATTTATTCGTTAAACCGATATCATTAAAAATGGAAAATTAAAGAAATCAATTCATTTGCTTCCTTGAAGTCTAGAGTCGATCATAACGAGAGCTTCATGTTGAAGCTCTTTAAGTGCAGCTTGCTTAGTAATATAAGTGCGTAATGTTTCAACAGCCTTTGGTGTTTTTATGAACTTGTTAAATATTTCTTGATCTTTGGAATATACGATATTGCATATGCAAACTAATCCACGATGTTGAATTCCTTCATCTTGGTCGGTTAACATTCTGACTATGTAGTTAAAATCGTTCTGGAGACTTAAAATTTTCTTGCACCCTTCATCAACACTAGTAATTTGAACAAGAATTCCGCATGCTGCTAATCTCGTGGGAGTGTCTTCGGTATCTGAGAGTGCTAGTACAATGTGTAAACGAGTATTTTCAAAATCCGAGTCCTTGTCTCCTATAAACTTAATTAAGCAATAAGGTGATAAAGAAAGGTTGTTAATAAGTTCCGTGGTTGCCCTCTGAATTAAAGGATTTGTTTCTATAATTAGTTCATCCAATTCTCTCCAACATTCTTGAACTATGGCTTGCCGACTTTCTTCATCATGTGAAGCAAGATTTGTTAGCGCAAGTAACACTTCAAATTTCCCCAAGAGAATTGGATACTCAGTATCAGCAGAATTAGTACTTAGTAACTTTGACATAGGATGAATTGCTCGGTTCGATGGAAACGCCTTAGTAAATATACTATGAGGAGCCACACTGATAAGAATTTTCGCTAATGCAAAAGCCGCATTCGAATCCTGGGTTTGGATAAAACATTGTCTAGTAAGGAATTTCAATGCACCTATTTGTACTAGCTTGCGACGTGCCAGCTTGTAATTAGCCATGTGTACAAGTATCATTGCTAATAACCCGTTTAGGCTATTATAGTGCTTGGCCGCTGAAAAGAGTTTTGAC\tSKLFSAAKHYNSLNGLLAMILVHMANYKLARRKLVQIGALKFLTRQCFIQTQDSNAAFALAKILISVAPHSIFTKAFPSNRAIHPMSKLLSTNSADTEYPILLGKFEVLLALTNLASHDEESRQAIVQECWRELDELIIETNPLIQRATTELINNLSLSPYCLIKFIGDKDSDFENTRLHIVLALSDTEDTPTRLAACGILVQITSVDEGCKKILSLQNDFNYIVRMLTDQDEGIQHRGLVCICNIVYSKDQEIFNKFIKTPKAVETLRTYITKQAALKELQHEALVMIDSRLQGSK*\n-TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\tAGAACTTGACCGCAGCGCACATCAACTACACCTTGAGGCAATTCAAAATATGAAGTTTTCTACCGTAGGGTTTTTGTTTTCAACAATTTTATTCAAAAGTGCGTTTGCAGGATGGATGGATACACACATGAAAGATGAACATCACATTGATAAGTACACAGATGAATCTTTTTTTCGTCTTCACGATTTAGGAAAAAAGGGATACTGGTCAGATCAAGATATACTTAGTTTATATGGATTATTTGAAAATGATGAGGTCCCATTCGTCAAAAAGAATGAGGTTTTGGTTGATGTTCTAAAAAAATGCGATCCATCGGGCAATCGCCGGATAACACTTGATGAATTTCTTGCATTTCGTAAAAATGGAGGAGAGTTAACGGATTTTGGATTTCCAGGGCACCATGGTGATGAGGAAGAGGAATTTGAAATGCATCATGTGGAAAAATATCACCCAGCTGGCCTCGATGAACCAGATGAAAACTGGAATCATCCTGAAGATATCGAACATTTTCAAAAACACGATGAGATATTTCATGGTGATAAGAAACCTGAGGAACGACGCAAACATTTTGTAAAATACAATAATATCCCTGACAAGTATCGTCGTGTGAGCATTTAAATTTTCCCATTGATAGATTTACTTGCTTCTCCTTGGGACGTTTTATTTTTATTTTTATTTTTCCGTTTATTTTTGGATTGTCGTGTTCCGATATATATCCGTGTTTACAAACAATAGCTTCAACATTTTTGAACGGGTCCTTCCAATACTAATGCCATATCTTATTTCTTTGTCAATACAAAATCACTTTTGTTCGTTAAACTCATTTTGGTTTGCAGATATTATCATATTCTTACCAATTATACATACACTCACAATTTGTTTTACATTTCCGCTCCCTTCTTCAATAAGCTGATTGCTTATAAAAAATCAAACATTCATATGATTCAAGTTCAAATATTATCATCTGAAAACTGACAAGTTCCATTCATTACTATATTAATATCATTTATTCGTTAAACCGATATCATTAAAAATGGAAAATTAAAGAAATCAATTCATTTGCTTCCTTGAAGTCTAGAGTCGATCATAACGAGAGCTTCATGTTGAAGCTCTTTAAGTGCAGCTTGCTTAGTAATATAAGTGCGTAATGTTTCAACAGCCTTTGGTGTTTTTATGAACTTGTTAAATATTTCTTGATCTTTGGAATATACGATATTGCATATGCAAACTAATCCACGATGTTGAATTCCTTCATCTTGGTCGGTTAACATTCTGACTATGTAGTTAAAATCGTTCTGGAGACTTAAAATTTTCTTGCACCCTTCATCAACACTAGTAATTTGAACAAGAATTCCGCATGCTGCTAATCTCGTGGGAGTGTCTTCGGTATCTGAGAGTGCTAGTACAATGTGTAAACGAGTATTTTCAAAATCCGAGTCCTTGTCTCCTATAAACTTAATTAAGCAATAAGGTGATAAAGAAAGGTTGTTAATAAGTTCCGTGGTTGCCCTCTGAATTAAAGGATTTGTTTCTATAATTAGTTCATCCAATTCTCTCCAACATTCTTGAACTATGGCTTGCCGACTTTCTTCATCATGTGAAGCAAGATTTGTTAGCGCAAGTAACACTTCAAATTTCCCCAAGAGAATTGGATACTCAGTATCAGCAGAATTAGTACTTAGTAACTTTGACATAGGATGAATTGCTCGGTTCGATGGAAACGCCTTAGTAAATATACTATGAGGAGCCACACTGATAAGAATTTTCGCTAATGCAAAAGCCGCATTCGAATCCTGGGTTTGGATAAAACATTGTCTAGTAAGGAATTTCAATGCACCTATTTGTACTAGCTTGCGACGTGCCAGCTTGTAATTAGCCATGTGTACAAGTATCATTGCTAATAACCCGTTTAGGCTATTATAGTGCTTGGCCGCTGAAAAGAGTTTTGAC\tELDRSAHQLHLEAIQNMKFSTVGFLFSTILFKSAFAGWMDTHMKDEHHIDKYTDESFFRLHDLGKKGYWSDQDILSLYGLFENDEVPFVKKNEVLVDVLKKCDPSGNRRITLDEFLAFRKNGGELTDFGFPGHHGDEEEEFEMHHVEKYHPAGLDEPDENWNHPEDIEHFQKHDEIFHGDKKPEERRKHFVKYNNIPDKYRRVSI*\n+TRINITY_DN59_c0_g1\tTRINITY_DN59_c0_g1_i1\tRNG3_SCHPO^RNG3_SCHPO^Q:1950-1060,H:450-746^100%ID^E:0^RecName: Full=Ring assembly protein 3;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\tTRINITY_DN59_c0_g1_i1|m.8863\t2-619[+]\tYCS3_SCHPO^YCS3_SCHPO^Q:17-205,H:1-189^100%ID^E:3e-137^RecName: Full=Uncharacterized calcium-binding protein C613.03;^Eukaryota; Fungi; Dikarya; Ascomycota; Taphrinomycotina; Schizosaccharomycetes; Schizosaccharomycetales; Schizosaccharomycetaceae; Schizosaccharomyces\t.\t.\t.\t.\tKEGG:spo:SPCC613.03\tGO:0005783^cellular_component^endoplasmic reticulum`GO:0005788^cellular_component^endoplasmic reticulum lumen`GO:0005793^cellular_component^endoplasmic reticulum-Golgi intermediate compartment`GO:0005794^cellular_component^Golgi apparatus`GO:0005796^cellular_component^Golgi lumen`GO:0005509^molecular_function^calcium ion binding\t.\tAGAACTTGACCGCAGCGCACATCAACTACACCTTGAGGCAATTCAAAATATGAAGTTTTCTACCGTAGGGTTTTTGTTTTCAACAATTTTATTCAAAAGTGCGTTTGCAGGATGGATGGATACACACATGAAAGATGAACATCACATTGATAAGTACACAGATGAATCTTTTTTTCGTCTTCACGATTTAGGAAAAAAGGGATACTGGTCAGATCAAGATATACTTAGTTTATATGGATTATTTGAAAATGATGAGGTCCCATTCGTCAAAAAGAATGAGGTTTTGGTTGATGTTCTAAAAAAATGCGATCCATCGGGCAATCGCCGGATAACACTTGATGAATTTCTTGCATTTCGTAAAAATGGAGGAGAGTTAACGGATTTTGGATTTCCAGGGCACCATGGTGATGAGGAAGAGGAATTTGAAATGCATCATGTGGAAAAATATCACCCAGCTGGCCTCGATGAACCAGATGAAAACTGGAATCATCCTGAAGATATCGAACATTTTCAAAAACACGATGAGATATTTCATGGTGATAAGAAACCTGAGGAACGACGCAAACATTTTGTAAAATACAATAATATCCCTGACAAGTATCGTCGTGTGAGCATTTAAATTTTCCCATTGATAGATTTACTTGCTTCTCCTTGGGACGTTTTATTTTTATTTTTATTTTTCCGTTTATTTTTGGATTGTCGTGTTCCGATATATATCCGTGTTTACAAACAATAGCTTCAACATTTTTGAACGGGTCCTTCCAATACTAATGCCATATCTTATTTCTTTGTCAATACAAAATCACTTTTGTTCGTTAAACTCATTTTGGTTTGCAGATATTATCATATTCTTACCAATTATACATACACTCACAATTTGTTTTACATTTCCGCTCCCTTCTTCAATAAGCTGATTGCTTATAAAAAATCAAACATTCATATGATTCAAGTTCAAATATTATCATCTGAAAACTGACAAGTTCCATTCATTACTATATTAATATCATTTATTCGTTAAACCGATATCATTAAAAATGGAAAATTAAAGAAATCAATTCATTTGCTTCCTTGAAGTCTAGAGTCGATCATAACGAGAGCTTCATGTTGAAGCTCTTTAAGTGCAGCTTGCTTAGTAATATAAGTGCGTAATGTTTCAACAGCCTTTGGTGTTTTTATGAACTTGTTAAATATTTCTTGATCTTTGGAATATACGATATTGCATATGCAAACTAATCCACGATGTTGAATTCCTTCATCTTGGTCGGTTAACATTCTGACTATGTAGTTAAAATCGTTCTGGAGACTTAAAATTTTCTTGCACCCTTCATCAACACTAGTAATTTGAACAAGAATTCCGCATGCTGCTAATCTCGTGGGAGTGTCTTCGGTATCTGAGAGTGCTAGTACAATGTGTAAACGAGTATTTTCAAAATCCGAGTCCTTGTCTCCTATAAACTTAATTAAGCAATAAGGTGATAAAGAAAGGTTGTTAATAAGTTCCGTGGTTGCCCTCTGAATTAAAGGATTTGTTTCTATAATTAGTTCATCCAATTCTCTCCAACATTCTTGAACTATGGCTTGCCGACTTTCTTCATCATGTGAAGCAAGATTTGTTAGCGCAAGTAACACTTCAAATTTCCCCAAGAGAATTGGATACTCAGTATCAGCAGAATTAGTACTTAGTAACTTTGACATAGGATGAATTGCTCGGTTCGATGGAAACGCCTTAGTAAATATACTATGAGGAGCCACACTGATAAGAATTTTCGCTAATGCAAAAGCCGCATTCGAATCCTGGGTTTGGATAAAACATTGTCTAGTAAGGAATTTCAATGCACCTATTTGTACTAGCTTGCGACGTGCCAGCTTGTAATTAGCCATGTGTACAAGTATCATTGCTAATAACCCGTTTAGGCTATTATAGTGCTTGGCCGCTGAAAAGAGTTTTGAC\tELDRSAHQLHLEAIQNMKFSTVGFLFSTILFKSAFAGWMDTHMKDEHHIDKYTDESFFRLHDLGKKGYWSDQDILSLYGLFENDEVPFVKKNEVLVDVLKKCDPSGNRRITLDEFLAFRKNGGELTDFGFPGHHGDEEEEFEMHHVEKYHPAGLDEPDENWNHPEDIEHFQKHDEIFHGDKKPEERRKHFVKYNNIPDKYRRVSI*\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 1, "time_seconds": 1679.67990899086, "tool_id": "trinotate", "tool_version": "3.1.1"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_trinotate.test_tool_000001"}, {"data": {"inputs": {"assembly_0|assembly_fasta": {"id": "e8a2248405ed0640", "src": "hda"}, "assembly_0|assembly_names": "reference", "assembly_1|assembly_fasta": {"id": "df2cbca38520d0f1", "src": "hda"}, "assembly_1|assembly_names": "duplication", "assembly_2|assembly_fasta": {"id": "26f324d382d6d8c1", "src": "hda"}, "assembly_2|assembly_names": "relocation", "assembly_3|assembly_fasta": {"id": "168b8fc92b5c87b9", "src": "hda"}, "assembly_3|assembly_names": "reloc-dup", "breakpoint_bin": "50", "coverage_file": {"id": "6f4bdab7a73c92f2", "src": "hda"}, "coverage_multiplier": "0", "ignore_ends": "0", "input_reads|orientation": "fr", "input_reads|paired_input_reads|paired_reads_0|mate_1": {"id": "8877579cefa7ba1d", "src": "hda"}, "input_reads|paired_input_reads|paired_reads_0|mate_2": {"id": "f28875befd5fbf72", "src": "hda"}, "input_reads|paired_input_reads|paired_reads_0|maxins": "500", "input_reads|paired_input_reads|paired_reads_0|minins": "0", "input_reads|paired_input_reads|type": "--fastq", "input_reads|type": "paired", "kmer": "15", "max_alignments": "10000", "min_contig_length": "1000", "min_coverage": "0", "min_suspicious": "2", "suspicious_flank_size": "2000", "window_size": "501"}, "job": {"command_line": "valet.py         --assembly-fasta /tmp/tmpaj3oz7xj/files/f/3/5/dataset_f35a3a00-d0a8-4a12-9e70-654a5cd65066.dat,/tmp/tmpaj3oz7xj/files/a/a/7/dataset_aa741a7f-7448-4628-ba2b-f0c52fbad2f9.dat,/tmp/tmpaj3oz7xj/files/c/3/a/dataset_c3a0fd42-28bb-4930-bb4b-f91e9da9542a.dat,/tmp/tmpaj3oz7xj/files/1/c/4/dataset_1c47dfdc-666d-47d4-bc43-62fb59895e2b.dat --assembly-names reference,duplication,relocation,reloc-dup     --1 /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat --2 /tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --minins 0 --maxins 500 --fastq --orientation \u0027fr\u0027 --output-dir output --window-size \u0027501\u0027 --threads ${GALAXY_SLOTS:-4} --max-alignments \u002710000\u0027 --min-coverage \u00270\u0027 --coverage-multiplier \u00270.0\u0027 --min-suspicious \u00272\u0027 --suspicious-flank-size \u00272000\u0027 --min-contig-length \u00271000\u0027 --ignore-ends \u00270\u0027 --breakpoint-bin \u002750\u0027 --coverage-file \u0027/tmp/tmpaj3oz7xj/files/1/8/d/dataset_18d8861a-de1b-4c5c-b6e3-bc45eab1c877.dat\u0027 --kmer \u002715\u0027 --skip-reapr \u0026\u0026 mv output/reference/summary.bed output/reference_summary.bed \u0026\u0026 mv output/reference/summary.tsv output/reference_summary.tsv \u0026\u0026 mv output/reference/suspicious.bed output/reference_suspicious.bed \u0026\u0026 mv output/duplication/summary.bed output/duplication_summary.bed \u0026\u0026 mv output/duplication/summary.tsv output/duplication_summary.tsv \u0026\u0026 mv output/duplication/suspicious.bed output/duplication_suspicious.bed \u0026\u0026 mv output/relocation/summary.bed output/relocation_summary.bed \u0026\u0026 mv output/relocation/summary.tsv output/relocation_summary.tsv \u0026\u0026 mv output/relocation/suspicious.bed output/relocation_suspicious.bed \u0026\u0026 mv output/reloc-dup/summary.bed output/reloc-dup_summary.bed \u0026\u0026 mv output/reloc-dup/summary.tsv output/reloc-dup_summary.tsv \u0026\u0026 mv output/reloc-dup/suspicious.bed output/reloc-dup_suspicious.bed", "command_version": "1.0\n", "create_time": "2020-01-23T17:17:45.920654", "exit_code": 0, "external_id": "55013", "galaxy_version": "20.01", "history_id": "244c246e3628fe4f", "id": "1fb9c7b21f94ef77", "inputs": {"assembly_0|assembly_fasta": {"id": "e8a2248405ed0640", "src": "hda", "uuid": "f35a3a00-d0a8-4a12-9e70-654a5cd65066"}, "assembly_1|assembly_fasta": {"id": "df2cbca38520d0f1", "src": "hda", "uuid": "aa741a7f-7448-4628-ba2b-f0c52fbad2f9"}, "assembly_2|assembly_fasta": {"id": "26f324d382d6d8c1", "src": "hda", "uuid": "c3a0fd42-28bb-4930-bb4b-f91e9da9542a"}, "assembly_3|assembly_fasta": {"id": "168b8fc92b5c87b9", "src": "hda", "uuid": "1c47dfdc-666d-47d4-bc43-62fb59895e2b"}, "coverage_file": {"id": "6f4bdab7a73c92f2", "src": "hda", "uuid": "18d8861a-de1b-4c5c-b6e3-bc45eab1c877"}, "input_reads|paired_input_reads|paired_reads_0|mate_1": {"id": "8877579cefa7ba1d", "src": "hda", "uuid": "792636fb-2f41-4594-b2be-3d1876ad5685"}, "input_reads|paired_input_reads|paired_reads_0|mate_2": {"id": "f28875befd5fbf72", "src": "hda", "uuid": "29e7409b-9f20-4f13-829d-8d1d3e602fd9"}}, "job_messages": [], "job_metrics": [], "job_stderr": "", "job_stdout": "", "model_class": "Job", "outputs": {"__new_primary_file_flagged|duplication__": {"id": "fc3d4e085ebafe27", "src": "hda", "uuid": "0e668f8a-e559-40e3-b08e-df44585165ee"}, "__new_primary_file_flagged|reference__": {"id": "dee39598aab00098", "src": "hda", "uuid": "1fd4fe8c-8cf9-4d89-a675-0f3563d39c52"}, "__new_primary_file_flagged|reloc-dup__": {"id": "ea1cd3f6f7bfdbc8", "src": "hda", "uuid": "0e026b6c-f88b-42ff-823b-def856a8b9d3"}, "__new_primary_file_flagged|relocation__": {"id": "a6abe2125965f40e", "src": "hda", "uuid": "b9c24f49-f841-439a-8df7-38667ba50016"}, "__new_primary_file_summary|duplication__": {"id": "068effd7f708a791", "src": "hda", "uuid": "3e08da5e-c7a4-4927-bf32-677a1892dba5"}, "__new_primary_file_summary|reference__": {"id": "6b84be122359b58b", "src": "hda", "uuid": "16c6e129-8297-4f90-9edf-af55e3968cf8"}, "__new_primary_file_summary|reloc-dup__": {"id": "e4dd3c0d281b4301", "src": "hda", "uuid": "aaec6ef7-d078-466a-a9b2-3c023e898939"}, "__new_primary_file_summary|relocation__": {"id": "b9562402199c9a4d", "src": "hda", "uuid": "cabb58d3-912d-4386-9c5a-5a03159d8241"}, "__new_primary_file_suspicious|duplication__": {"id": "eb96e852c39cc9f1", "src": "hda", "uuid": "5660529b-f9f8-4822-8476-737e7bd97f7b"}, "__new_primary_file_suspicious|reference__": {"id": "42aacd38bfe2dec7", "src": "hda", "uuid": "7e7b7bac-f123-401a-ad68-dda7ab30a87d"}, "__new_primary_file_suspicious|reloc-dup__": {"id": "835e531c7ecd0d52", "src": "hda", "uuid": "dd3f1232-9372-4b69-b331-c795e2b41d71"}, "__new_primary_file_suspicious|relocation__": {"id": "e63663c4a976dc8c", "src": "hda", "uuid": "dc02b021-2ff9-4338-8272-88f90ecc6a31"}, "comparison_plot": {"id": "0805ac7051fb8901", "src": "hda", "uuid": "bf1c6174-f6d9-4bdf-a75f-3449aea1ea6c"}}, "params": {"__input_ext": "\"fasta\"", "assembly": "[{\"__index__\": 0, \"assembly_fasta\": {\"values\": [{\"id\": 179, \"src\": \"hda\"}]}, \"assembly_names\": \"reference\"}, {\"__index__\": 1, \"assembly_fasta\": {\"values\": [{\"id\": 180, \"src\": \"hda\"}]}, \"assembly_names\": \"duplication\"}, {\"__index__\": 2, \"assembly_fasta\": {\"values\": [{\"id\": 181, \"src\": \"hda\"}]}, \"assembly_names\": \"relocation\"}, {\"__index__\": 3, \"assembly_fasta\": {\"values\": [{\"id\": 182, \"src\": \"hda\"}]}, \"assembly_names\": \"reloc-dup\"}]", "breakpoint_bin": "\"50\"", "chromInfo": "\"/tmp/tmpaj3oz7xj/galaxy-dev/tool-data/shared/ucsc/chrom/?.len\"", "coverage_multiplier": "\"0.0\"", "dbkey": "\"?\"", "ignore_ends": "\"0\"", "input_reads": "{\"__current_case__\": 1, \"orientation\": \"fr\", \"paired_input_reads\": {\"__current_case__\": 1, \"paired_reads\": [{\"__index__\": 0, \"mate_1\": {\"values\": [{\"id\": 183, \"src\": \"hda\"}]}, \"mate_2\": {\"values\": [{\"id\": 184, \"src\": \"hda\"}]}, \"maxins\": \"500\", \"minins\": \"0\"}], \"type\": \"--fastq\"}, \"type\": \"paired\"}", "kmer": "\"15\"", "max_alignments": "\"10000\"", "min_contig_length": "\"1000\"", "min_coverage": "\"0\"", "min_suspicious": "\"2\"", "orf_file": "null", "suspicious_flank_size": "\"2000\"", "window_size": "\"501\""}, "state": "ok", "stderr": "###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: reference (/tmp/tmpaj3oz7xj/files/f/3/5/dataset_f35a3a00-d0a8-4a12-9e70-654a5cd65066.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/reference/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/indexes/temp_iiNzAj\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/indexes/temp_iiNzAj -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/unaligned_reads/unaligned.reads -S output/reference/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/reference/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/reference/bam/sorted_library.bam output/reference/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/reference/filtered_assembly.fasta output/reference/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/reference/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/reference/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/reference/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/reference/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/reference/coverage/mpileup_output.out -w 501 -o output/reference/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reference/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/unaligned_reads/ -o output/reference/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/reference/filtered_assembly.fasta -r output/reference/breakpoint/split_reads/ -b 50 -o output/reference/breakpoint/ -c output/reference/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reference/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/reference/summary.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: duplication (/tmp/tmpaj3oz7xj/files/a/a/7/dataset_aa741a7f-7448-4628-ba2b-f0c52fbad2f9.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/indexes/temp_QK73gc\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/indexes/temp_QK73gc -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/unaligned_reads/unaligned.reads -S output/duplication/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/duplication/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/duplication/bam/sorted_library.bam output/duplication/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/duplication/filtered_assembly.fasta output/duplication/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/duplication/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/duplication/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/duplication/coverage/mpileup_output.out -w 501 -o output/duplication/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/duplication/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/unaligned_reads/ -o output/duplication/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/duplication/filtered_assembly.fasta -r output/duplication/breakpoint/split_reads/ -b 50 -o output/duplication/breakpoint/ -c output/duplication/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/duplication/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/summary.bed\n\u001b[94mCOMMAND:\t\u001b[0m bedtools merge -i output/duplication/tmp.bed -d 2000 -o distinct -c 4\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: relocation (/tmp/tmpaj3oz7xj/files/c/3/a/dataset_c3a0fd42-28bb-4930-bb4b-f91e9da9542a.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/indexes/temp_l3jSxT\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/indexes/temp_l3jSxT -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/unaligned_reads/unaligned.reads -S output/relocation/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/relocation/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/relocation/bam/sorted_library.bam output/relocation/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/relocation/filtered_assembly.fasta output/relocation/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/relocation/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/relocation/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/relocation/coverage/mpileup_output.out -w 501 -o output/relocation/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/relocation/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/unaligned_reads/ -o output/relocation/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/relocation/filtered_assembly.fasta -r output/relocation/breakpoint/split_reads/ -b 50 -o output/relocation/breakpoint/ -c output/relocation/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/relocation/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/summary.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: reloc-dup (/tmp/tmpaj3oz7xj/files/1/c/4/dataset_1c47dfdc-666d-47d4-bc43-62fb59895e2b.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/indexes/temp_R4mXh5\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/indexes/temp_R4mXh5 -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/unaligned_reads/unaligned.reads -S output/reloc-dup/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/reloc-dup/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/reloc-dup/bam/sorted_library.bam output/reloc-dup/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/reloc-dup/filtered_assembly.fasta output/reloc-dup/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/reloc-dup/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/reloc-dup/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/reloc-dup/coverage/mpileup_output.out -w 501 -o output/reloc-dup/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reloc-dup/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/unaligned_reads/ -o output/reloc-dup/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/reloc-dup/filtered_assembly.fasta -r output/reloc-dup/breakpoint/split_reads/ -b 50 -o output/reloc-dup/breakpoint/ -c output/reloc-dup/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reloc-dup/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/summary.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mGENERATING ASSEMBLY COMPARISON PLOTS\u001b[0m\n###########################################################################\n\u001b[94mCOMMAND:\t\u001b[0m Rscript /usr/local/bin/src/R/compare_assemblies.R output/reference/summary.tsv,output/duplication/summary.tsv,output/relocation/summary.tsv,output/reloc-dup/summary.tsv reference,duplication,relocation,reloc-dup output/comparison_plots\n\u001b[93mRESULTS:\t\u001b[0m output/comparison_plots.pdf\n", "stdout": "", "tool_id": "valet", "tool_stderr": "###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: reference (/tmp/tmpaj3oz7xj/files/f/3/5/dataset_f35a3a00-d0a8-4a12-9e70-654a5cd65066.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/reference/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/indexes/temp_iiNzAj\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/indexes/temp_iiNzAj -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/unaligned_reads/unaligned.reads -S output/reference/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/reference/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/reference/bam/sorted_library.bam output/reference/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/reference/filtered_assembly.fasta output/reference/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/reference/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/reference/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/reference/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/reference/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/reference/coverage/mpileup_output.out -w 501 -o output/reference/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reference/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reference/unaligned_reads/ -o output/reference/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/reference/filtered_assembly.fasta -r output/reference/breakpoint/split_reads/ -b 50 -o output/reference/breakpoint/ -c output/reference/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reference/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/reference/summary.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reference/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: duplication (/tmp/tmpaj3oz7xj/files/a/a/7/dataset_aa741a7f-7448-4628-ba2b-f0c52fbad2f9.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/indexes/temp_QK73gc\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/indexes/temp_QK73gc -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/unaligned_reads/unaligned.reads -S output/duplication/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/duplication/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/duplication/bam/sorted_library.bam output/duplication/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/duplication/filtered_assembly.fasta output/duplication/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/duplication/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/duplication/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/duplication/coverage/mpileup_output.out -w 501 -o output/duplication/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/duplication/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/duplication/unaligned_reads/ -o output/duplication/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/duplication/filtered_assembly.fasta -r output/duplication/breakpoint/split_reads/ -b 50 -o output/duplication/breakpoint/ -c output/duplication/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/duplication/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/summary.bed\n\u001b[94mCOMMAND:\t\u001b[0m bedtools merge -i output/duplication/tmp.bed -d 2000 -o distinct -c 4\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/duplication/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: relocation (/tmp/tmpaj3oz7xj/files/c/3/a/dataset_c3a0fd42-28bb-4930-bb4b-f91e9da9542a.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/indexes/temp_l3jSxT\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/indexes/temp_l3jSxT -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/unaligned_reads/unaligned.reads -S output/relocation/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/relocation/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/relocation/bam/sorted_library.bam output/relocation/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/relocation/filtered_assembly.fasta output/relocation/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/relocation/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/relocation/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/relocation/coverage/mpileup_output.out -w 501 -o output/relocation/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/relocation/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/relocation/unaligned_reads/ -o output/relocation/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/relocation/filtered_assembly.fasta -r output/relocation/breakpoint/split_reads/ -b 50 -o output/relocation/breakpoint/ -c output/relocation/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/relocation/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/summary.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/relocation/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mPROCESSING ASSEMBLY: reloc-dup (/tmp/tmpaj3oz7xj/files/1/c/4/dataset_1c47dfdc-666d-47d4-bc43-62fb59895e2b.dat)\u001b[0m\n###########################################################################\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m FILTERING ASSEMBLY CONTIGS LESS THAN 1000 BPs\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/filtered_assembly.fasta\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m ALIGNING READS\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2-build /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/filtered_assembly.fasta /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/indexes/temp_R4mXh5\n\u001b[94mCOMMAND:\t\u001b[0m bowtie2 -a -x /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/indexes/temp_R4mXh5 -q -U /tmp/tmpaj3oz7xj/files/7/9/2/dataset_792636fb-2f41-4594-b2be-3d1876ad5685.dat,/tmp/tmpaj3oz7xj/files/2/9/e/dataset_29e7409b-9f20-4f13-829d-8d1d3e602fd9.dat --very-sensitive -a --reorder -p 1 --un /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/unaligned_reads/unaligned.reads -S output/reloc-dup/sam/library.sam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m RUNNING SAMTOOLS\n\u001b[94mCOMMAND:\t\u001b[0m samtools view -F 0x100 -bS output/reloc-dup/sam/library.sam\n\u001b[94mCOMMAND:\t\u001b[0m samtools sort -@ 16 -o output/reloc-dup/bam/sorted_library.bam output/reloc-dup/bam/library.bam\n\u001b[94mCOMMAND:\t\u001b[0m samtools mpileup -C50 -A -f output/reloc-dup/filtered_assembly.fasta output/reloc-dup/bam/sorted_library.bam\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/coverage/mpileup_output.out\n\u001b[94mCOMMAND:\t\u001b[0m samtools index output/reloc-dup/bam/sorted_library.bam\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m CALCULATING CONTIG COVERAGE\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/coverage/temp.cvg\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m PARTITIONING COVERAGE FILE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/split_pileup.py -p output/reloc-dup/coverage/mpileup_output.out -c 1\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m DEPTH OF COVERAGE\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/depth_of_coverage.py -m output/reloc-dup/coverage/mpileup_output.out -w 501 -o output/reloc-dup/coverage/errors_cov.bed -g -e -c 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reloc-dup/coverage/errors_cov.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/coverage.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m BREAKPOINT\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_splitter.py -u /tmp/tmpaj3oz7xj/job_working_directory/000/114/working/output/reloc-dup/unaligned_reads/ -o output/reloc-dup/breakpoint/split_reads/\n\u001b[94mCOMMAND:\t\u001b[0m /usr/local/bin/src/py/breakpoint_finder.py -a output/reloc-dup/filtered_assembly.fasta -r output/reloc-dup/breakpoint/split_reads/ -b 50 -o output/reloc-dup/breakpoint/ -c output/reloc-dup/coverage/temp.cvg -p 1\n\u001b[94mCOMMAND:\t\u001b[0m bedtools sort -i output/reloc-dup/breakpoint/interesting_bins.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/breakpoint/../breakpoints.bed\n---------------------------------------------------------------------------\n\u001b[95mSTEP:\t\u001b[0m SUMMARY\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/summary.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/suspicious.bed\n\u001b[93mRESULTS:\t\u001b[0m output/reloc-dup/summary.tsv\n###########################################################################\n\u001b[1m\u001b[95mGENERATING ASSEMBLY COMPARISON PLOTS\u001b[0m\n###########################################################################\n\u001b[94mCOMMAND:\t\u001b[0m Rscript /usr/local/bin/src/R/compare_assemblies.R output/reference/summary.tsv,output/duplication/summary.tsv,output/relocation/summary.tsv,output/reloc-dup/summary.tsv reference,duplication,relocation,reloc-dup output/comparison_plots\n\u001b[93mRESULTS:\t\u001b[0m output/comparison_plots.pdf\n", "tool_stdout": "", "update_time": "2020-01-23T17:18:36.793707", "user_email": "test@bx.psu.edu"}, "output_problems": ["History item  different than expected, difference (using diff):\n( /tmp/tmp6hp5gr09flagged_reference.bed v. /tmp/tmpqntkd_dkflagged_reference.bed )\n--- local_file\n+++ history_data\n@@ -1,2 +0,0 @@\n-gi|116334902|ref|NC_008512.1|\t1\t50\tBreakpoint_finder\n-gi|116334902|ref|NC_008512.1|\t159578\t159628\tBreakpoint_finder\n", "History item  different than expected, difference (using diff):\n( /tmp/tmp1zpkd6o0summary_reference.tabular v. /tmp/tmp82axz17ysummary_reference.tabular )\n--- local_file\n+++ history_data\n@@ -1,2 +1,2 @@\n contig_name\tcontig_length\tabundance\tlow_cov\tlow_cov_bps\thigh_cov\thigh_cov_bps\treapr\treapr_bps\tbreakpoints\tbreakpoints_bps\n-gi|116334902|ref|NC_008512.1|\t159662\t5\t0\t0\t0\t0\t0\t0\t2\t101\n+gi|116334902|ref|NC_008512.1|\t159662\tNA\tNA\tNA\tNA\tNA\tNA\tNA\tNA\tNA\n"], "problem_log": "Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 59, in testPartExecutor\n    yield\n  File \"/opt/hostedtoolcache/Python/3.7.6/x64/lib/python3.7/unittest/case.py\", line 628, in run\n    testMethod()\n  File \"/tmp/tmpaj3oz7xj/galaxy-dev/test/functional/test_toolbox.py\", line 99, in test_tool\n    self.do_it(tool_version=tool_version, test_index=test_index)\n  File \"/tmp/tmpaj3oz7xj/galaxy-dev/test/functional/test_toolbox.py\", line 36, in do_it\n    verify_tool(tool_id, self.galaxy_interactor, resource_parameters=resource_parameters, test_index=test_index, tool_version=tool_version, register_job_data=register_job_data)\n  File \"/tmp/tmpaj3oz7xj/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 789, in verify_tool\n    raise e\n  File \"/tmp/tmpaj3oz7xj/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 785, in verify_tool\n    job_stdio = _verify_outputs(testdef, test_history, jobs, tool_id, data_list, data_collection_list, galaxy_interactor, quiet=quiet)\n  File \"/tmp/tmpaj3oz7xj/galaxy-dev/lib/galaxy/tool_util/verify/interactor.py\", line 1006, in _verify_outputs\n    raise JobOutputsError(found_exceptions, job_stdio)\ngalaxy.tool_util.verify.interactor.JobOutputsError: History item  different than expected, difference (using diff):\n( /tmp/tmp6hp5gr09flagged_reference.bed v. /tmp/tmpqntkd_dkflagged_reference.bed )\n--- local_file\n+++ history_data\n@@ -1,2 +0,0 @@\n-gi|116334902|ref|NC_008512.1|\t1\t50\tBreakpoint_finder\n-gi|116334902|ref|NC_008512.1|\t159578\t159628\tBreakpoint_finder\n\nHistory item  different than expected, difference (using diff):\n( /tmp/tmp1zpkd6o0summary_reference.tabular v. /tmp/tmp82axz17ysummary_reference.tabular )\n--- local_file\n+++ history_data\n@@ -1,2 +1,2 @@\n contig_name\tcontig_length\tabundance\tlow_cov\tlow_cov_bps\thigh_cov\thigh_cov_bps\treapr\treapr_bps\tbreakpoints\tbreakpoints_bps\n-gi|116334902|ref|NC_008512.1|\t159662\t5\t0\t0\t0\t0\t0\t0\t2\t101\n+gi|116334902|ref|NC_008512.1|\t159662\tNA\tNA\tNA\tNA\tNA\tNA\tNA\tNA\tNA\n\n", "problem_type": "galaxy.tool_util.verify.interactor.JobOutputsError", "status": "failure", "test_index": 0, "time_seconds": 120.24953293800354, "tool_id": "valet", "tool_version": "1.0.0"}, "has_data": true, "id": "functional.test_toolbox.TestForTool_valet.test_tool_000000"}]};
        renderTestResults(test_data);
      }
    </script>
  </body>
</html>